<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8" />
		<meta http_equiv="Content-Type" content="text/html; charset=utf-8" />
		<title>Workflow Registry Test results</title>
		<style>table, td {padding: 10px;border: 1px solid black;border-collapse: collapse}
thead {font-weight: bold;background-color: lightgray}</style>
	</head>
	<body>
		<h1 id="wf-reg-test-results">
			<a href="#wf-reg-test-results">wf-reg-test results</a>
		</h1>
		<span>Time: 2023-02-27 14:43:35
		</span>
		<h2 id="stats">
			<a href="#stats">Stats</a>
		</h2>
		<table>
			<thead>
				<tr>
					<td>Stat</td>
					<td>Total</td>
					<td>nextflow</td>
					<td>snakemake</td>
				</tr>
			</thead>
			<tbody>
				<tr>
					<td>N workflows</td>
					<td>217</td>
					<td>75</td>
					<td>142</td>
				</tr>
				<tr>
					<td>revisions / workflows</td>
					<td>2.7</td>
					<td>3.3</td>
					<td>2.3</td>
				</tr>
				<tr>
					<td>N revisions</td>
					<td>584</td>
					<td>251</td>
					<td>333</td>
				</tr>
				<tr>
					<td>executions / revision</td>
					<td>100%</td>
					<td>100%</td>
					<td>100%</td>
				</tr>
				<tr>
					<td>N executions</td>
					<td>584</td>
					<td>251</td>
					<td>333</td>
				</tr>
				<tr>
					<td>working executions / executions</td>
					<td>28%</td>
					<td>51%</td>
					<td>11%</td>
				</tr>
				<tr>
					<td>N working executions</td>
					<td>164</td>
					<td>129</td>
					<td>35</td>
				</tr>
				<tr>
					<td>error executions / executions</td>
					<td>72%</td>
					<td>49%</td>
					<td>89%</td>
				</tr>
				<tr>
					<td>N error executions</td>
					<td>420</td>
					<td>122</td>
					<td>298</td>
				</tr>
				<tr>
					<td>classified error executions / error executions</td>
					<td>100%</td>
					<td>100%</td>
					<td>100%</td>
				</tr>
				<tr>
					<td>N classified error executions</td>
					<td>419</td>
					<td>122</td>
					<td>297</td>
				</tr>
				<tr>
					<td>wall time hours</td>
					<td>10307</td>
					<td>6645</td>
					<td>3662</td>
				</tr>
				<tr>
					<td>cpu time hours</td>
					<td>5223</td>
					<td>3320</td>
					<td>1902</td>
				</tr>
			</tbody>
		</table>
		<h2 id="errors">
			<a href="#errors">Errors</a>
		</h2>
		<div>
			<table>
				<thead>
					<tr>
						<td>Error</td>
						<td>Count</td>
						<td>Fraction of all errors</td>
						<td>Instances</td>
					</tr>
				</thead>
				<tbody>
					<tr>
						<td>
							<code>missing input</code>
						</td>
						<td>
							<span>188</span>
						</td>
						<td>
							<span>45%</span>
						</td>
						<td>
							<details>
								<summary>See individual instances</summary>
								<ul>
									<li>
										<span>
											<a href="#140010746842512">Execution of Revision 1.2.0 of Workflow ampliseq</a>
											<code></code>
											<code>Warning message:
In read.table(file = file, header = header, sep = sep, quote = quote,  :
  incomplete final line found by readTableHeader on 'Metadata.tsv'
Error in 1:ncol(data) : argument of length 0
Execution halted
											</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010746877680">Execution of Revision 1.1.3 of Workflow ampliseq</a>
											<code></code>
											<code>Warning message:
In read.table(file = file, header = header, sep = sep, quote = quote,  :
  incomplete final line found by readTableHeader on 'Metadata.tsv'
Error in 2:ncol(data) : argument of length 0
Execution halted
											</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010746961808">Execution of Revision 1.1.2 of Workflow ampliseq</a>
											<code></code>
											<code>Warning message:
In read.table(file = file, header = header, sep = sep, quote = quote,  :
  incomplete final line found by readTableHeader on 'Metadata.tsv'
Error in 1:ncol(data) : argument of length 0
Execution halted
											</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010746997168">Execution of Revision 1.1.1 of Workflow ampliseq</a>
											<code></code>
											<code>Warning message:
In read.table(file = file, header = header, sep = sep, quote = quote,  :
  incomplete final line found by readTableHeader on 'Metadata.tsv'
Error in 1:ncol(data) : argument of length 0
Execution halted
											</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010744967760">Execution of Revision 1.1.0 of Workflow ampliseq</a>
											<code></code>
											<code>Warning message:
In read.table(file = file, header = header, sep = sep, quote = quote,  :
  incomplete final line found by readTableHeader on 'Metadata.tsv'
Error in 1:ncol(data) : argument of length 0
Execution halted
											</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010745002688">Execution of Revision 1.0.0 of Workflow ampliseq</a>
											<code></code>
											<code>Warning message:
In read.table(file = file, header = header, sep = sep, quote = quote,  :
  incomplete final line found by readTableHeader on 'Metadata.tsv'
Error in 1:ncol(data) : argument of length 0
Execution halted
											</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743104944">Execution of Revision 2.3.5 of Workflow eager</a>
											<code>[nf-core/eager] error: --input was not supplied! Please check '--help' or documentation under 'running the pipeline' for details</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743105184">Execution of Revision 2.3.4 of Workflow eager</a>
											<code>[nf-core/eager] error: --input was not supplied! Please check '--help' or documentation under 'running the pipeline' for details</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743105424">Execution of Revision 2.3.3 of Workflow eager</a>
											<code>[nf-core/eager] error: --input was not supplied! Please check '--help' or documentation under 'running the pipeline' for details</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743105664">Execution of Revision 2.3.2 of Workflow eager</a>
											<code>ERROR: Validation of pipeline parameters failed!</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743105904">Execution of Revision 2.3.1 of Workflow eager</a>
											<code>[nf-core/eager] error: --input was not supplied! Please check '--help' or documentation under 'running the pipeline' for details</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743106144">Execution of Revision 2.3.0 of Workflow eager</a>
											<code>[nf-core/eager] error: --input was not supplied! Please check '--help' or documentation under 'running the pipeline' for details</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743106384">Execution of Revision 2.2.2 of Workflow eager</a>
											<code>[nf-core/eager] error: --input was not supplied! Please check '--help' or documentation under 'running the pipeline' for details</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743106624">Execution of Revision 2.2.1 of Workflow eager</a>
											<code>[nf-core/eager] error: --input was not supplied! Please check '--help' or documentation under 'running the pipeline' for details</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743106864">Execution of Revision 2.2.0 of Workflow eager</a>
											<code>[nf-core/eager] error: --input was not supplied! Please check '--help' or documentation under 'running the pipeline' for details</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743107104">Execution of Revision 2.1.0 of Workflow eager</a>
											<code>Please specify either --single_end, --paired_end to execute the pipeline on FastQ files and --bam for previously processed BAM files!</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743107344">Execution of Revision 2.0.7 of Workflow eager</a>
											<code>Please specify either --singleEnd, --pairedEnd to execute the pipeline on FastQ files and --bam for previously processed BAM files!</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743107584">Execution of Revision 2.0.6 of Workflow eager</a>
											<code>Please specify either --singleEnd, --pairedEnd to execute the pipeline on FastQ files and --bam for previously processed BAM files!</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743107824">Execution of Revision 2.0.5 of Workflow eager</a>
											<code>Please specify either --singleEnd, --pairedEnd to execute the pipeline on FastQ files and --bam for previously processed BAM files!</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743108064">Execution of Revision 2.0.4 of Workflow eager</a>
											<code>Please specify either --singleEnd or --pairedEnd to execute the pipeline!</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743108304">Execution of Revision 2.0.3 of Workflow eager</a>
											<code>Please specify either --singleEnd or --pairedEnd to execute the pipeline!</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743108544">Execution of Revision 2.0.2 of Workflow eager</a>
											<code>Please specify either --singleEnd or --pairedEnd to execute the pipeline!</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743108784">Execution of Revision 2.0.1 of Workflow eager</a>
											<code>Please specify either --singleEnd or --pairedEnd to execute the pipeline!</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743109024">Execution of Revision 2.0.0 of Workflow eager</a>
											<code>Please specify either --singleEnd or --pairedEnd to execute the pipeline!</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743109504">Execution of Revision 2.0.0 of Workflow epitopeprediction</a>
											<code>samplesheet</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743138672">Execution of Revision 1.6.1 of Workflow methylseq</a>
											<code>SRR389222_sub3.fastq.gz</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743138912">Execution of Revision 1.6 of Workflow methylseq</a>
											<code>SRR389222_sub2.fastq.gz</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743140352">Execution of Revision 1.0 of Workflow methylseq</a>
											<code>singularity profile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743140592">Execution of Revision 2.4.0 of Workflow mhcquant</a>
											<code>test.fasta</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743140832">Execution of Revision 2.3.1 of Workflow mhcquant</a>
											<code>test.fasta</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743141072">Execution of Revision 2.3.0 of Workflow mhcquant</a>
											<code>test.fasta</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743141312">Execution of Revision 2.2.0 of Workflow mhcquant</a>
											<code>test.fasta</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743141552">Execution of Revision 2.1.0 of Workflow mhcquant</a>
											<code>test.fasta</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743144672">Execution of Revision 1.2.0 of Workflow mhcquant</a>
											<code>Script compilation error</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743146592">Execution of Revision 1.0.0 of Workflow nanoseq</a>
											<code>samplesheet</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743146832">Execution of Revision 2.1.0 of Workflow nascent</a>
											<code>GRCh38_chr21.fa</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743147552">Execution of Revision 1.0.0 of Workflow neutronstar</a>
											<code>@unknown</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743148992">Execution of Revision 2.1.0 of Workflow rnafusion</a>
											<code>Unable to read script: '/home/azureuser/tmp/auooxnsnkf/code/./workflows/rnafusion.nf' -- cause: /home/azureuser/tmp/auooxnsnkf/out/references/ensembl/Homo_sapiens.GRCh38.102.all.fa
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743149232">Execution of Revision 2.0.0 of Workflow rnafusion</a>
											<code>Unable to read script: '/home/azureuser/tmp/cn9oowh3ia/code/./workflows/rnafusion.nf' -- cause: /home/azureuser/tmp/cn9oowh3ia/out/references/ensembl/Homo_sapiens.GRCh38.102.all.fa
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743149952">Execution of Revision 1.0.2 of Workflow rnafusion</a>
											<code>Cannot find any reads matching: tests/reads_{1,2}.fq.gz</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743150192">Execution of Revision 1.0.1 of Workflow rnafusion</a>
											<code>Fasta file not found: tests/genome.fa</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743150432">Execution of Revision 1.0 of Workflow rnafusion</a>
											<code>Fasta file not found: tests/genome.fa</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743252592">Execution of Revision 23.02.0 of Workflow seankmartin/atn-sub-lfp-workflow</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010741895216">Execution of Revision v1.0.0 of Workflow simonmoulds/uk-decadal-flood-prediction</a>
											<code>configfile config/config.yml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742126992">Execution of Revision v0.0.2 of Workflow johnne/map_metaT</a>
											<code>configfile config/config.yml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742160672">Execution of Revision v0.0.1 of Workflow johnne/map_metaT</a>
											<code>configfile config/config.yml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742194928">Execution of Revision v1.0.0 of Workflow crs4/fair-crcc-get-data</a>
											<code>property recipient_key</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742362176">Execution of Revision v1.2.0.1 of Workflow solida-core/musta</a>
											<code>get_sample_names  in line 2</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742494640">Execution of Revision v1.2 of Workflow solida-core/musta</a>
											<code>get_sample_names  in line 2</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742578048">Execution of Revision 1.0 of Workflow solida-core/musta</a>
											<code>get_sample_names  in line 2</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742611728">Execution of Revision v1.1.1 of Workflow solida-core/musta</a>
											<code>get_sample_names  in line 2</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742694000">Execution of Revision v1.1 of Workflow solida-core/musta</a>
											<code>get_sample_names  in line 2</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742777264">Execution of Revision v1.0 of Workflow solida-core/musta</a>
											<code>configfile ../config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010741028592">Execution of Revision v0.1.0 of Workflow SilkeAllmannLab/pacbio_snakemake</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010741229328">Execution of Revision v1.2.1 of Workflow plycrsk/rna-seq-star-deseq2</a>
											<code>all  in line 36</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010741281120">Execution of Revision v0.1.0 of Workflow thomasbtf/sars-cov-2-wbe</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010741332144">Execution of Revision v0.1.3 of Workflow aryarm/admixtools</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010741465776">Execution of Revision v0.1.2 of Workflow aryarm/admixtools</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010741534096">Execution of Revision v0.1.1 of Workflow aryarm/admixtools</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010741585792">Execution of Revision v0.1.0 of Workflow aryarm/admixtools</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010741621200">Execution of Revision v0.0.3 of Workflow aryarm/admixtools</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739788832">Execution of Revision v0.0.2 of Workflow aryarm/admixtools</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739693216">Execution of Revision v0.0.1 of Workflow aryarm/admixtools</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739876144">Execution of Revision v1.1.0 of Workflow snakemake-workflows/cyrcular-calling</a>
											<code>/home/azureuser/tmp/k18gvqyyul/code/workflow/rules/../envs/pigz.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740080768">Execution of Revision v2.0 of Workflow solida-core/dima</a>
											<code>pre_rename_fastq_se  in line 2</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740419072">Execution of Revision v1.0.1 of Workflow TLenfers/multispecies_mitochondrial_variant_analysis</a>
											<code>rule bwa_mem</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740503680">Execution of Revision v1.0.0 of Workflow TLenfers/multispecies_mitochondrial_variant_analysis</a>
											<code>rule bwa_mem</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740539088">Execution of Revision v0.12-alpha of Workflow mrvollger/Rhodonite</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740639840">Execution of Revision v0.7-alpha of Workflow mrvollger/Rhodonite</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740708016">Execution of Revision v0.6-alpha of Workflow mrvollger/Rhodonite</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738695424">Execution of Revision v0.5-alpha of Workflow mrvollger/Rhodonite</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738812816">Execution of Revision v0.4-alpha of Workflow mrvollger/Rhodonite</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738864560">Execution of Revision v0.3-alpha of Workflow mrvollger/Rhodonite</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738932688">Execution of Revision v0.2-alpha of Workflow mrvollger/Rhodonite</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739033728">Execution of Revision v0.1-alpha of Workflow mrvollger/Rhodonite</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739085520">Execution of Revision v2.0.1 of Workflow IMS-Bio2Core-Facility/polya_liftover</a>
											<code>clean_names  in line 24</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739186464">Execution of Revision v2.0.0 of Workflow IMS-Bio2Core-Facility/polya_liftover</a>
											<code>clean_names  in line 24</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739287472">Execution of Revision v1.1.0 of Workflow IMS-Bio2Core-Facility/polya_liftover</a>
											<code>fastqc  in line 1</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739322880">Execution of Revision v1.0.1 of Workflow IMS-Bio2Core-Facility/polya_liftover</a>
											<code>fastqc  in line 1</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739440208">Execution of Revision v1.0.0 of Workflow IMS-Bio2Core-Facility/polya_liftover</a>
											<code>fastqc  in line 1</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739508384">Execution of Revision v0.2.0 of Workflow IMS-Bio2Core-Facility/polya_liftover</a>
											<code>clean_names  in line 56</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010737902272">Execution of Revision v0.1.2 of Workflow IMS-Bio2Core-Facility/single_snake_sequencing</a>
											<code>clean_names  in line 56</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010737937392">Execution of Revision v0.1.1 of Workflow IMS-Bio2Core-Facility/single_snake_sequencing</a>
											<code>clean_names  in line 56</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738054720">Execution of Revision v0.1.0 of Workflow IMS-Bio2Core-Facility/single_snake_sequencing</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738164416">Execution of Revision v1.3.4 of Workflow snakemake-workflows/dna-seq-mtb</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738164656">Execution of Revision v1.3.3 of Workflow snakemake-workflows/dna-seq-mtb</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738164896">Execution of Revision v1.3.2 of Workflow snakemake-workflows/dna-seq-mtb</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738165136">Execution of Revision v1.3.1 of Workflow snakemake-workflows/dna-seq-mtb</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738165376">Execution of Revision v1.3.0 of Workflow snakemake-workflows/dna-seq-mtb</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738165616">Execution of Revision v1.2.0 of Workflow snakemake-workflows/dna-seq-mtb</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738165856">Execution of Revision v1.1.0 of Workflow snakemake-workflows/dna-seq-mtb</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738166096">Execution of Revision v1.0.1 of Workflow snakemake-workflows/dna-seq-mtb</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738166336">Execution of Revision v1.0.0 of Workflow snakemake-workflows/dna-seq-mtb</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738168496">Execution of Revision 1.2.1 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code>rule bwa_index</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738168736">Execution of Revision 1.2.0 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code>rule bwa_index</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738168976">Execution of Revision 1.1 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code>rule bwa_strandseq_to_reference_alignment</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738169216">Execution of Revision 1.0 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code>rule bwa_strandseq_to_reference_alignment</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738209728">Execution of Revision v1.3.2 of Workflow snakemake-workflows/benchmark-giab</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738214768">Execution of Revision v1.3.2 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738216448">Execution of Revision v1.0 of Workflow tahiri-lab/aPhyloGeo-pipeline</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738216928">Execution of Revision v1.0.1 of Workflow dbespiatykh/RDscan</a>
											<code>rule bwa_mem</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738217408">Execution of Revision v0.1.0 of Workflow epigen/dea_limma</a>
											<code>/research/home/sreichl/projects/macroIC/config/CC001_dea_limma_annotation.csv</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738217648">Execution of Revision v0.1.0 of Workflow epigen/genome_tracks</a>
											<code>/path/to/myData_genome_tracks_annotation.csv</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738218128">Execution of Revision v1.0.0 of Workflow GooglingTheCancerGenome/sv-gen</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738218368">Execution of Revision v0.1.1 of Workflow sreichl/atacseq_pipeline</a>
											<code>/research/home/sreichl/projects/atacseq_pipeline/test/BSA_0000_test_hg38_atac_atacseq_config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738218608">Execution of Revision v0.1.0 of Workflow sreichl/atacseq_pipeline</a>
											<code>/research/home/sreichl/projects/atacseq_pipeline/test/BSA_0000_test_hg38_atac_atacseq_config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738218848">Execution of Revision v.1.1.0 of Workflow akcorut/kGWASflow</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738219088">Execution of Revision v1.0.0 of Workflow akcorut/kGWASflow</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738219328">Execution of Revision v0.1.0 of Workflow epigen/enrichment_analysis</a>
											<code>/path/to/enrichment_analysis_annotation.csv</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738219808">Execution of Revision v0.1.0 of Workflow epigen/unsupervised_analysis</a>
											<code>/research/home/sreichl/projects/macroIC/config/AKsmall_unsupervised_analysis_annotation.csv</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738256240">Execution of Revision 1.3 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738256720">Execution of Revision v0.15.1 of Workflow IKIM-Essen/uncovar</a>
											<code>fastp_pe  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738256960">Execution of Revision v0.15.0 of Workflow IKIM-Essen/uncovar</a>
											<code>fastp_pe  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738257200">Execution of Revision v0.14.0 of Workflow IKIM-Essen/uncovar</a>
											<code>fastp_pe  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738257440">Execution of Revision v0.13.2 of Workflow IKIM-Essen/uncovar</a>
											<code>fastp_pe  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738257680">Execution of Revision v0.13.1 of Workflow IKIM-Essen/uncovar</a>
											<code>fastp_pe  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738257920">Execution of Revision v0.13.0 of Workflow IKIM-Essen/uncovar</a>
											<code>fastp_pe  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738258160">Execution of Revision v0.12.0 of Workflow IKIM-Essen/uncovar</a>
											<code>fastqc  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738258400">Execution of Revision v0.11.0 of Workflow IKIM-Essen/uncovar</a>
											<code>fastqc  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738258640">Execution of Revision v0.10.2 of Workflow IKIM-Essen/uncovar</a>
											<code>fastqc  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738258880">Execution of Revision v0.10.1 of Workflow IKIM-Essen/uncovar</a>
											<code>fastqc  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738259120">Execution of Revision v0.10.0 of Workflow IKIM-Essen/uncovar</a>
											<code>fastqc  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738259360">Execution of Revision v0.9.0 of Workflow IKIM-Essen/uncovar</a>
											<code>fastqc  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738259600">Execution of Revision v0.8.0 of Workflow IKIM-Essen/uncovar</a>
											<code>fastqc  in line 7</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738261040">Execution of Revision v0.3.1 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738261280">Execution of Revision v0.3.0 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738261520">Execution of Revision v0.2.5 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738261760">Execution of Revision v0.2.4 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738262000">Execution of Revision v0.2.3 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738262240">Execution of Revision v0.2.2 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738262480">Execution of Revision v0.2.1 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738262720">Execution of Revision v0.2.0 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738262960">Execution of Revision v0.1.8 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738263200">Execution of Revision v0.1.7 of Workflow IKIM-Essen/uncovar</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738263440">Execution of Revision v0.1.6 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738263680">Execution of Revision v0.1.5 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738263920">Execution of Revision v0.1.4 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738264160">Execution of Revision v0.1.3 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738264400">Execution of Revision v0.1.2 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738264640">Execution of Revision v0.1.1 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738264880">Execution of Revision v0.1.0 of Workflow IKIM-Essen/uncovar</a>
											<code>env GISAID_API_TOKEN</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738265120">Execution of Revision v0.1.1 of Workflow epigen/atacseq_pipeline</a>
											<code>/research/home/sreichl/projects/atacseq_pipeline/test/BSA_0000_test_hg38_atac_atacseq_config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738265360">Execution of Revision v0.1.0 of Workflow epigen/atacseq_pipeline</a>
											<code>/research/home/sreichl/projects/atacseq_pipeline/test/BSA_0000_test_hg38_atac_atacseq_config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738265600">Execution of Revision v0.3.0 of Workflow sneuensc/mapache</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738266320">Execution of Revision v0.4.1 of Workflow zavolanlab/zarp</a>
											<code>start  in line 178</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738268960">Execution of Revision v1.1.2 of Workflow GooglingTheCancerGenome/sv-callers</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738269200">Execution of Revision v1.1.1 of Workflow GooglingTheCancerGenome/sv-callers</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738269440">Execution of Revision v1.1.0 of Workflow GooglingTheCancerGenome/sv-callers</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738269680">Execution of Revision v1.0.0 of Workflow GooglingTheCancerGenome/sv-callers</a>
											<code>snakefile</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738290208">Execution of Revision v1.1.0 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>cutadapt_pe  in line 1</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738290448">Execution of Revision v1.0.0 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>configfile config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738290688">Execution of Revision v3.20.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738290928">Execution of Revision v3.20.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738291168">Execution of Revision v3.19.6 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738291408">Execution of Revision v3.19.5 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738291648">Execution of Revision v3.19.4 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738291888">Execution of Revision v3.19.3 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738292128">Execution of Revision v3.19.2 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738292368">Execution of Revision v3.19.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738292608">Execution of Revision v3.19.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738292848">Execution of Revision v3.18.7 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738293088">Execution of Revision v3.18.6 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738293328">Execution of Revision v3.18.5 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738293568">Execution of Revision v3.18.4 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738293808">Execution of Revision v3.18.3 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738294048">Execution of Revision v3.18.2 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738294288">Execution of Revision v3.18.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738294528">Execution of Revision v3.18.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738294768">Execution of Revision v3.17.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738295008">Execution of Revision v3.17.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738295248">Execution of Revision v3.16.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738295488">Execution of Revision v3.15.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738295728">Execution of Revision v3.14.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738295968">Execution of Revision v3.14.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738296208">Execution of Revision v3.13.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738296448">Execution of Revision v3.13.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738296688">Execution of Revision v3.12.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738296928">Execution of Revision v3.11.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738297168">Execution of Revision v3.10.2 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738297408">Execution of Revision v3.10.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738297648">Execution of Revision v3.10.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738297888">Execution of Revision v3.9.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>configfile config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738298128">Execution of Revision v3.8.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>config/samples.tsv</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738298368">Execution of Revision v3.8.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>config/samples.tsv</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738321776">Execution of Revision v1.0.3 of Workflow franciscozorrilla/metaGEM</a>
											<code>configfile ../config/config.yaml</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738324176">Execution of Revision v1.0.1 of Workflow snakemake-workflows/dna-seq-gatk-variant-calling</a>
											<code>data/ref/genome.chr21.fa.fai</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738324416">Execution of Revision v1.0.0 of Workflow snakemake-workflows/dna-seq-gatk-variant-calling</a>
											<code>data/ref/genome.chr21.fa.fai</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738325616">Execution of Revision v1.0.0 of Workflow snakemake-workflows/rna-seq-star-deseq2</a>
											<code>cutadapt_pe  in line 5</code>
											<code>None</code>
										</span>
									</li>
								</ul>
							</details>
						</td>
					</tr>
					<tr>
						<td>
							<code>conda</code>
						</td>
						<td>
							<span>63</span>
						</td>
						<td>
							<span>15%</span>
						</td>
						<td>
							<details>
								<summary>See individual instances</summary>
								<ul>
									<li>
										<span>
											<a href="#140010740928176">Execution of Revision v1.1.0 of Workflow vembrane/vembrane-benchmark</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740945472">Execution of Revision v1.0.0 of Workflow vembrane/vembrane-benchmark</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010737664976">Execution of Revision v1.1.1 of Workflow IMS-Bio2Core-Facility/GTExSnake</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010737749536">Execution of Revision v1.1.0 of Workflow IMS-Bio2Core-Facility/GTExSnake</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010737817712">Execution of Revision v1.0.0 of Workflow IMS-Bio2Core-Facility/GTExSnake</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738163456">Execution of Revision v0.3.1 of Workflow nikostr/dna-seq-deepvariant-glnexus-variant-calling</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738163696">Execution of Revision v0.3.0 of Workflow nikostr/dna-seq-deepvariant-glnexus-variant-calling</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738163936">Execution of Revision v0.2.0 of Workflow nikostr/dna-seq-deepvariant-glnexus-variant-calling</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738164176">Execution of Revision v0.1.0-alpha of Workflow nikostr/dna-seq-deepvariant-glnexus-variant-calling</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738208048">Execution of Revision v1.5.0 of Workflow snakemake-workflows/benchmark-giab</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738208288">Execution of Revision v1.4.1 of Workflow snakemake-workflows/benchmark-giab</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738208528">Execution of Revision v1.4.0 of Workflow snakemake-workflows/benchmark-giab</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738210208">Execution of Revision v1.3.0 of Workflow snakemake-workflows/benchmark-giab</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738210688">Execution of Revision v1.1.0 of Workflow snakemake-workflows/benchmark-giab</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738267040">Execution of Revision v0.4 of Workflow mrvollger/StainedGlass</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738267280">Execution of Revision v0.3 of Workflow mrvollger/StainedGlass</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738267520">Execution of Revision v0.2 of Workflow mrvollger/StainedGlass</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738267760">Execution of Revision v0.1 of Workflow mrvollger/StainedGlass</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738287088">Execution of Revision v2.4.0 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738287328">Execution of Revision v2.3.2 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738287568">Execution of Revision v2.3.1 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738287808">Execution of Revision v2.3.0 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738288048">Execution of Revision v2.2.1 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738288288">Execution of Revision v2.2.0 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738288528">Execution of Revision v2.1.1 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738288768">Execution of Revision v2.1.0 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738289008">Execution of Revision v2.0.4 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738289248">Execution of Revision v2.0.3 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738289488">Execution of Revision v2.0.2 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738289728">Execution of Revision v2.0.1 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738289968">Execution of Revision v2.0.0 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738298608">Execution of Revision v3.7.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738298848">Execution of Revision v3.7.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738299088">Execution of Revision v3.6.3 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738299328">Execution of Revision v3.6.2 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738299568">Execution of Revision v3.6.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738299808">Execution of Revision v3.6.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738300048">Execution of Revision v3.5.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738300288">Execution of Revision v3.4.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738300528">Execution of Revision v3.3.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738300768">Execution of Revision v3.2.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738301008">Execution of Revision v3.1.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738301248">Execution of Revision v3.0.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738301488">Execution of Revision v2.0.6 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738301728">Execution of Revision v2.0.5 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738301968">Execution of Revision v2.0.4 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738302208">Execution of Revision v2.0.3 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738302448">Execution of Revision v2.0.2 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738302688">Execution of Revision v2.0.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738302928">Execution of Revision v2.0.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738319616">Execution of Revision v1.5.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738319856">Execution of Revision v1.4.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738320096">Execution of Revision v1.3.2 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738320336">Execution of Revision v1.3.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738320576">Execution of Revision v1.3.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738320816">Execution of Revision v1.2.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738321056">Execution of Revision v1.1.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738321296">Execution of Revision v1.0.1 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738321536">Execution of Revision v1.0.0 of Workflow snakemake-workflows/dna-seq-varlociraptor</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738324656">Execution of Revision v1.2.0 of Workflow snakemake-workflows/rna-seq-star-deseq2</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738324896">Execution of Revision v1.1.2 of Workflow snakemake-workflows/rna-seq-star-deseq2</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738325136">Execution of Revision v1.1.1 of Workflow snakemake-workflows/rna-seq-star-deseq2</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738325376">Execution of Revision v1.1.0 of Workflow snakemake-workflows/rna-seq-star-deseq2</a>
											<code>Could not create conda environment from </code>
											<code>None</code>
										</span>
									</li>
								</ul>
							</details>
						</td>
					</tr>
					<tr>
						<td>
							<code>unclassified</code>
						</td>
						<td>
							<span>45</span>
						</td>
						<td>
							<span>11%</span>
						</td>
						<td>
							<details>
								<summary>See individual instances</summary>
								<ul>
									<li>
										<span>
											<a href="#140010743142032">Execution of Revision 1.6.0 of Workflow mhcquant</a>
											<code>NextflowCommandError(kind='CalledProcessError', process='db_search_comet (MM16_Melanom)', command_executed="CometAdapter  -in MM15_Melanom_1_B_1.mzML \\\n              -out MM16_Melanom_B_3.idXML \\\n              -threads 2 \\\n              -database test_decoy.fasta \\\n              -precursor_mass_tolerance 5 \\\n              -fragment_bin_tolerance 0.02 \\\n              -fragment_bin_offset 0 \\\n              -num_hits 1 \\\n              -digest_mass_range 800:2500 \\\n              -m
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743192624">Execution of Revision 1.1.0 of Workflow scrnaseq</a>
											<code>NextflowCommandError(kind='CalledProcessError', process='extract_transcriptome (GRCm38.p6.genome.chr19.fa)', command_executed='gffread -F gencode.vM19.annotation.chr19.gtf -w "GRCm38.p6.genome.chr19.fa.transcriptome.fa" -g GRCm38.p6.genome.chr19.fa', exit_status=1, output='(empty)', error='FASTA index file GRCm38.p6.genome.chr19.fa.fai created.\nError: cannot open input file gencode.vM19.annotation.chr19.gtf!', work_dir=PosixPath('work/81/e544f7f0ad42859bd158e5c433d503'))
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743193584">Execution of Revision 2.0.0 of Workflow smrnaseq</a>
											<code>NextflowCommandError(kind='CalledProcessError', process='NFCORE_SMRNASEQ:SMRNASEQ:MIRTRACE:MIRTRACE_RUN', command_executed='export mirtracejar=$(dirname $(which mirtrace))\nfor i in C1-N1-R1_S4_L001_R1_001.fastq.gz C1-N3-R1_S6_L001_R1_001.fastq.gz C9-N1-R1_S7_L001_R1_001.fastq.gz C9-N2-R1_S8_L001_R1_001.fastq.gz C9-N3-R1_S9_L001_R1_001.fastq.gz Ctl-N1-R1_S1_L001_R1_001.fastq.gz Ctl-N2-R1_S2_L001_R1_001.fastq.gz Ctl-N3-R1_S3_L001_R1_001.fastq.gz\ndo\n    path=$(realpath ${i})\n    prefix=$(echo $
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743193824">Execution of Revision 1.1.0 of Workflow smrnaseq</a>
											<code>NextflowCommandError(kind='CalledProcessError', process='edgeR_mirna', command_executed='edgeR_miRBase.r sample_3.mature.stats sample_2.mature.stats sample_2.hairpin.stats sample_3.hairpin.stats sample_1.mature.stats sample_1.hairpin.stats', exit_status=1, output='$mature\n[1] "sample_3.mature.stats" "sample_2.mature.stats" "sample_1.mature.stats"\n\n$hairpin\n[1] "sample_2.hairpin.stats" "sample_3.hairpin.stats" "sample_1.hairpin.stats"', error='Loading required package: limma\nLoading required
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743194064">Execution of Revision 1.0.0 of Workflow smrnaseq</a>
											<code>NextflowCommandError(kind='CalledProcessError', process='edgeR_mirna', command_executed='edgeR_miRBase.r sample_2.mature.stats sample_3.mature.stats sample_1.mature.stats sample_2.hairpin.stats sample_1.hairpin.stats sample_3.hairpin.stats', exit_status=1, output='$mature\n[1] "sample_2.mature.stats" "sample_3.mature.stats" "sample_1.mature.stats"\n\n$hairpin\n[1] "sample_2.hairpin.stats" "sample_1.hairpin.stats" "sample_3.hairpin.stats"', error='Loading required package: limma\nLoading required
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739977136">Execution of Revision v2.0.2 of Workflow solida-core/dima</a>
											<code>SnakemakePythonError(kind='AttributeError', line_no=137, file=PosixPath('workflow/rules/picard.smk'), rest='\'NoneType\' object has no attribute \'get\'\n  File "/home/azureuser/tmp/0qq4gk4oeu/code/workflow/Snakefile", line 17, in <module>\n  File "/home/azureuser/tmp/0qq4gk4oeu/code/workflow/rules/picard.smk", line 137, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740012592">Execution of Revision v2.0.1 of Workflow solida-core/dima</a>
											<code>SnakemakePythonError(kind='AttributeError', line_no=79, file=PosixPath('workflow/rules/picard.smk'), rest='\'NoneType\' object has no attribute \'get\'\n  File "/home/azureuser/tmp/q09gyw8enl/code/workflow/Snakefile", line 19, in <module>\n  File "/home/azureuser/tmp/q09gyw8enl/code/workflow/rules/picard.smk", line 79, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740181712">Execution of Revision v1.2 of Workflow solida-core/dima</a>
											<code>SnakemakePythonError(kind='KeyError', line_no=10, file=PosixPath('Snakefile'), rest='\'samples\'\n  File "/home/azureuser/tmp/6qzkdnl3ka/code/Snakefile", line 10, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740233504">Execution of Revision v1.1 of Workflow solida-core/dima</a>
											<code>SnakemakePythonError(kind='KeyError', line_no=10, file=PosixPath('Snakefile'), rest='\'samples\'\n  File "/home/azureuser/tmp/qfg6f1oqkt/code/Snakefile", line 10, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010740285296">Execution of Revision v1.0 of Workflow solida-core/dima</a>
											<code>SnakemakePythonError(kind='KeyError', line_no=10, file=PosixPath('Snakefile'), rest='\'samples\'\n  File "/home/azureuser/tmp/yh4t2wej0l/code/Snakefile", line 10, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738171136">Execution of Revision 1.4.8 of Workflow CVUA-RRW/FooDMe</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=21, file=PosixPath('/home/azureuser/tmp/byo6mnr9a7/code/workflow/rules/reports.smk'), rest='Command \'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate \'/home/azureuser/tmp/byo6mnr9a7/code/.tests/.snakemake/conda/ea2703fc29c7a0a2f7082ca0c475e3ea_\'; set -euo pipefail;  python /home/azureuser/tmp/byo6mnr9a7/code/.tests/.snakemake/scripts/tmpbrxmup1r.krona_table.py
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738171376">Execution of Revision 1.4.7 of Workflow CVUA-RRW/FooDMe</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=21, file=PosixPath('/home/azureuser/tmp/uwm26axoff/code/workflow/rules/reports.smk'), rest='Command \'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate \'/home/azureuser/tmp/uwm26axoff/code/.tests/.snakemake/conda/9a6c34784b2b6310268dab4e3dcc52f7_\'; set -euo pipefail;  python /home/azureuser/tmp/uwm26axoff/code/.tests/.snakemake/scripts/tmpjzoh0zxc.krona_table.py
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738171616">Execution of Revision 1.4.6 of Workflow CVUA-RRW/FooDMe</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=21, file=PosixPath('/home/azureuser/tmp/6p1dgqi88h/code/workflow/rules/reports.smk'), rest='Command \'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate \'/home/azureuser/tmp/6p1dgqi88h/code/.tests/.snakemake/conda/f544522396b62211963f91c7025f4913_\'; set -euo pipefail;  python /home/azureuser/tmp/6p1dgqi88h/code/.tests/.snakemake/scripts/tmpvah3cjew.krona_table.py
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738171856">Execution of Revision 1.4.5 of Workflow CVUA-RRW/FooDMe</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=21, file=PosixPath('/home/azureuser/tmp/3lm11z5wl1/code/workflow/rules/reports.smk'), rest='Command \'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate \'/home/azureuser/tmp/3lm11z5wl1/code/.tests/.snakemake/conda/088e7876344b50ce0ddee547533112d2_\'; set -euo pipefail;  python /home/azureuser/tmp/3lm11z5wl1/code/.tests/.snakemake/scripts/tmp7gs2fmv_.krona_table.py
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738204928">Execution of Revision 1.4.4 of Workflow CVUA-RRW/FooDMe</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=21, file=PosixPath('/home/azureuser/tmp/qrvm0tk9ay/code/workflow/rules/reports.smk'), rest='Command \'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate \'/home/azureuser/tmp/qrvm0tk9ay/code/.tests/.snakemake/conda/60836a4e660ddf763a775a295dde8a21_\'; set -euo pipefail;  python /home/azureuser/tmp/qrvm0tk9ay/code/.tests/.snakemake/scripts/tmpf6rde94y.krona_table.py
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738205168">Execution of Revision 1.4.2 of Workflow CVUA-RRW/FooDMe</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=20, file=PosixPath('/home/azureuser/tmp/oa352xnwpi/code/workflow/rules/blast.smk'), rest='Command \'set -euo pipefail;  /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/python3.10 /home/azureuser/tmp/oa352xnwpi/code/.tests/.snakemake/scripts/tmppy98skma.filter_taxonomy.py\' returned non-zero exit status 1.\n  File "/home/azureuser/tmp/oa352xnwpi/code/workflow/rules/blast.smk", l
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738205408">Execution of Revision 1.4.1 of Workflow CVUA-RRW/FooDMe</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=20, file=PosixPath('/home/azureuser/tmp/10tx4adb6o/code/workflow/rules/blast.smk'), rest='Command \'set -euo pipefail;  /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/python3.10 /home/azureuser/tmp/10tx4adb6o/code/.tests/.snakemake/scripts/tmpt0nee8rk.filter_taxonomy.py\' returned non-zero exit status 1.\n  File "/home/azureuser/tmp/10tx4adb6o/code/workflow/rules/blast.smk", l
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738205648">Execution of Revision 1.4.0 of Workflow CVUA-RRW/FooDMe</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=70, file=PosixPath('/home/azureuser/tmp/8s3hf4cn6o/code/rules/blast.smk'), rest='Command \'set -euo pipefail;  /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/python3.10 /home/azureuser/tmp/8s3hf4cn6o/code/tests/.snakemake/scripts/tmpsabsl5xf.make_blast_mask.py\' returned non-zero exit status 1.\n  File "/home/azureuser/tmp/8s3hf4cn6o/code/rules/blast.smk", line 70, in __rule_c
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738205888">Execution of Revision 1.3.5 of Workflow CVUA-RRW/FooDMe</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=148, file=PosixPath('/home/azureuser/tmp/dtbjnxj7wj/code/rules/blast.smk'), rest='Command \'set -euo pipefail;  /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/python3.10 /home/azureuser/tmp/dtbjnxj7wj/code/tests/.snakemake/scripts/tmpwi8m0sd4.blast_to_lca.py\' returned non-zero exit status 1.\n  File "/home/azureuser/tmp/dtbjnxj7wj/code/rules/blast.smk", line 148, in __rule_bl
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738206368">Execution of Revision v0.1-alpha of Workflow mrvollger/fastCN-smk</a>
											<code>SnakemakePythonError(kind='KeyError', line_no=15, file=PosixPath('workflow/Snakefile'), rest='\'manifest\'\n  File "/home/azureuser/tmp/1t1x0ihhd4/code/workflow/Snakefile", line 15, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738207328">Execution of Revision v1.6.1 of Workflow snakemake-workflows/benchmark-giab</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=161, file=PosixPath('/home/azureuser/tmp/bxhdgkmyui/code/workflow/rules/eval.smk'), rest='Command \'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate \'/home/azureuser/tmp/bxhdgkmyui/code/.test/.snakemake/conda/170f7b276dafd14c1f9cee6ea57cff9c_\'; set -euo pipefail;  python /home/azureuser/tmp/bxhdgkmyui/code/.test/.snakemake/scripts/tmpsikgjmo6.collect-precision-
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738212368">Execution of Revision v1.6.1 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=161, file=PosixPath('/home/azureuser/tmp/clc5qtlcfs/code/workflow/rules/eval.smk'), rest='Command \'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate \'/home/azureuser/tmp/clc5qtlcfs/code/.test/.snakemake/conda/5987075987fe13661dc2a0ebfbd253f9_\'; set -euo pipefail;  python /home/azureuser/tmp/clc5qtlcfs/code/.test/.snakemake/scripts/tmp98n4pgw1.collect-precision-
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738217888">Execution of Revision v1.1.0 of Workflow GooglingTheCancerGenome/sv-gen</a>
											<code>SnakemakePythonError(kind='ModuleNotFoundError', line_no=1, file=PosixPath('workflow/Snakefile'), rest='No module named \'pyfaidx\'\n  File "/home/azureuser/tmp/8j15m8kvzk/code/workflow/Snakefile", line 1, in <module>\n  File "/home/azureuser/tmp/8j15m8kvzk/code/workflow/helper_functions.py", line 5, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738219568">Execution of Revision v0.2.0 of Workflow epigen/unsupervised_analysis</a>
											<code>SnakemakeInternalError(kind='SnakemakeInternalError', rest='File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/conda", line 12, in <module>\n    from conda.cli import main\nModuleNotFoundError: No module named \'conda\'\n[Sun Feb 19 16:16:45 2023]\nError in rule env_export:\n    jobid: 16\n    output: .test/results/envs/unsupervised_analysis/sklearn.yaml\n    log: logs/rules/env_sklearn.log (check log file(s) for error message)\n
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738220048">Execution of Revision v1.2.0 of Workflow SchlossLab/mikropml-snakemake-workflow</a>
											<code>SnakemakeWorkflowError(kind='SnakemakeError', rest="\nUnable to find environment in container image. Maybe a conda environment was modified without containerizing again (see snakemake --containerize)?\nDetails:\nCommand 'SINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/cbecbdf8-40ce-43a9-91b8-8379c0b0ac72 singularity --quiet --silent exec --home /home/azureuser/tmp/qo58fw9v0x/code  /home/azureuser/tmp/qo58fw9v0x/code/.snakemake/singularity/0c5b3ddbad4071ce8cdb3cdee75d09f0.simg sh -c '[ -d '\\''/conda-e
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738220288">Execution of Revision v1.1.0 of Workflow SchlossLab/mikropml-snakemake-workflow</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=68, file=PosixPath('workflow/rules/learn.smk'), rest='Command \'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate \'/home/azureuser/tmp/mzfmy02ilc/code/.snakemake/conda/380f5f8d8fcb955186f723a8f3ea1525_\'; set -euo pipefail;  Rscript --vanilla /home/azureuser/tmp/mzfmy02ilc/code/.snakemake/scripts/tmpd4b78l1z.find_feature_importance.R\' returned non-zero exit stat
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738220528">Execution of Revision v1.0.0 of Workflow SchlossLab/mikropml-snakemake-workflow</a>
											<code>SnakemakePythonError(kind='CalledProcessError', line_no=31, file=PosixPath('Snakefile'), rest='Command \'set -euo pipefail;  Rscript --vanilla /home/azureuser/tmp/8p6uk30bag/code/.snakemake/scripts/tmpg13k7t7c.preproc.R\' returned non-zero exit status 127.\n  File "/home/azureuser/tmp/8p6uk30bag/code/Snakefile", line 31, in __rule_preprocess_data\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738255520">Execution of Revision 1.6.0 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code>SnakemakeWorkflowError(kind='SnakemakeError', rest="\nUnable to find environment in container image. Maybe a conda environment was modified without containerizing again (see snakemake --containerize)?\nDetails:\nCommand 'SINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/f7e4bc8e-ca2a-4e7f-9411-7541731567db singularity --quiet --silent exec --home /home/azureuser/tmp/mpx9s0vdfg/code  /home/azureuser/tmp/mpx9s0vdfg/code/.snakemake/singularity/05142899ca28933c11364ad46012383f.simg sh -c '[ -d '\\''/conda-e
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738256000">Execution of Revision 1.5.1 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code>SnakemakePythonError(kind='ModuleNotFoundError', line_no=71, file=PosixPath('Snakefile'), rest='No module named \'pysam\'\n  File "/home/azureuser/tmp/hmj0rcn4cl/code/Snakefile", line 71, in <module>\n  File "/home/azureuser/tmp/hmj0rcn4cl/code/workflow/scripts/utils/handle_input.py", line 5, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738256480">Execution of Revision 1.0 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code>SnakemakePythonError(kind='IndexError', line_no=362, file=PosixPath('Snakefile'), rest='list index out of range\n  File "/home/azureuser/tmp/9n28odktia/code/Snakefile", line 362, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738260560">Execution of Revision v0.4.1 of Workflow IKIM-Essen/uncovar</a>
											<code>SnakemakePythonError(kind='ValueError', line_no=37, file=PosixPath('workflow/rules/common.smk'), rest='The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()\n  File "/home/azureuser/tmp/zakmzlx2v8/code/workflow/Snakefile", line 33, in <module>\n  File "/home/azureuser/tmp/zakmzlx2v8/code/workflow/rules/common.smk", line 37, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738260800">Execution of Revision v0.4.0 of Workflow IKIM-Essen/uncovar</a>
											<code>SnakemakePythonError(kind='ValueError', line_no=37, file=PosixPath('workflow/rules/common.smk'), rest='The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()\n  File "/home/azureuser/tmp/i55b9wj2lb/code/workflow/Snakefile", line 33, in <module>\n  File "/home/azureuser/tmp/i55b9wj2lb/code/workflow/rules/common.smk", line 37, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738265840">Execution of Revision v0.2.0 of Workflow sneuensc/mapache</a>
											<code>SnakemakePythonError(kind='IndexError', line_no=665, file=PosixPath('workflow/scripts/utils.py'), rest='list index out of range\n  File "/home/azureuser/tmp/rjpmx8d3pr/code/workflow/Snakefile", line 179, in <module>\n  File "/home/azureuser/tmp/rjpmx8d3pr/code/workflow/scripts/utils.py", line 665, in set_sex_inference')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738266080">Execution of Revision v0.1.0 of Workflow sneuensc/mapache</a>
											<code>None</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738266560">Execution of Revision v0.4.0 of Workflow zavolanlab/zarp</a>
											<code>SnakemakePythonError(kind='KeyError', line_no=13, file=PosixPath('workflow/Snakefile'), rest='\'samples\'\n  File "/home/azureuser/tmp/w1zzblg6nc/code/workflow/Snakefile", line 13, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738268000">Execution of Revision v1.2.2 of Workflow GooglingTheCancerGenome/sv-callers</a>
											<code>SnakemakePythonError(kind='ModuleNotFoundError', line_no=7, file=PosixPath('workflow/Snakefile'), rest='No module named \'yatiml\'\n  File "/home/azureuser/tmp/7qfk4uy5mm/code/workflow/Snakefile", line 7, in <module>\n  File "/home/azureuser/tmp/7qfk4uy5mm/code/workflow/helper_functions.py", line 5, in <module>\n  File "/home/azureuser/tmp/7qfk4uy5mm/code/workflow/validator/__init__.py", line 8, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738268240">Execution of Revision v1.2.1 of Workflow GooglingTheCancerGenome/sv-callers</a>
											<code>SnakemakePythonError(kind='ModuleNotFoundError', line_no=7, file=PosixPath('workflow/Snakefile'), rest='No module named \'yatiml\'\n  File "/home/azureuser/tmp/lp153dey0x/code/workflow/Snakefile", line 7, in <module>\n  File "/home/azureuser/tmp/lp153dey0x/code/workflow/helper_functions.py", line 5, in <module>\n  File "/home/azureuser/tmp/lp153dey0x/code/workflow/validator/__init__.py", line 8, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738268480">Execution of Revision v1.2.0 of Workflow GooglingTheCancerGenome/sv-callers</a>
											<code>SnakemakePythonError(kind='ModuleNotFoundError', line_no=7, file=PosixPath('workflow/Snakefile'), rest='No module named \'yatiml\'\n  File "/home/azureuser/tmp/5018p5k5a0/code/workflow/Snakefile", line 7, in <module>\n  File "/home/azureuser/tmp/5018p5k5a0/code/workflow/helper_functions.py", line 5, in <module>\n  File "/home/azureuser/tmp/5018p5k5a0/code/workflow/validator/__init__.py", line 8, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738268720">Execution of Revision v1.1.3 of Workflow GooglingTheCancerGenome/sv-callers</a>
											<code>SnakemakePythonError(kind='ModuleNotFoundError', line_no=7, file=PosixPath('workflow/Snakefile'), rest='No module named \'yatiml\'\n  File "/home/azureuser/tmp/wrhfb27wax/code/workflow/Snakefile", line 7, in <module>\n  File "/home/azureuser/tmp/wrhfb27wax/code/workflow/helper_functions.py", line 5, in <module>\n  File "/home/azureuser/tmp/wrhfb27wax/code/workflow/validator/__init__.py", line 8, in <module>')
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738322736">Execution of Revision v2.1.1 of Workflow snakemake-workflows/dna-seq-gatk-variant-calling</a>
											<code>SnakemakeInternalError(kind='SnakemakeInternalError', rest='File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py", line 747, in snakemake\n    success = workflow.execute(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py", line 926, in execute\n    dag.create_conda_envs(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.s
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738322976">Execution of Revision v2.1.0 of Workflow snakemake-workflows/dna-seq-gatk-variant-calling</a>
											<code>SnakemakeInternalError(kind='SnakemakeInternalError', rest='File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py", line 747, in snakemake\n    success = workflow.execute(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py", line 926, in execute\n    dag.create_conda_envs(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.s
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738323216">Execution of Revision v2.0.3 of Workflow snakemake-workflows/dna-seq-gatk-variant-calling</a>
											<code>SnakemakeInternalError(kind='SnakemakeInternalError', rest='File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py", line 747, in snakemake\n    success = workflow.execute(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py", line 926, in execute\n    dag.create_conda_envs(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.s
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738323456">Execution of Revision v2.0.2 of Workflow snakemake-workflows/dna-seq-gatk-variant-calling</a>
											<code>SnakemakeInternalError(kind='SnakemakeInternalError', rest='File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py", line 747, in snakemake\n    success = workflow.execute(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py", line 926, in execute\n    dag.create_conda_envs(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.s
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738323696">Execution of Revision v2.0.1 of Workflow snakemake-workflows/dna-seq-gatk-variant-calling</a>
											<code>SnakemakeInternalError(kind='SnakemakeInternalError', rest='File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py", line 747, in snakemake\n    success = workflow.execute(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py", line 926, in execute\n    dag.create_conda_envs(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.s
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738323936">Execution of Revision v2.0.0 of Workflow snakemake-workflows/dna-seq-gatk-variant-calling</a>
											<code>SnakemakeInternalError(kind='SnakemakeInternalError', rest='File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py", line 747, in snakemake\n    success = workflow.execute(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py", line 926, in execute\n    dag.create_conda_envs(\n  File "/home/azureuser/spack/var/spack/environments/wf-reg-test/.s
											</code>
											<code>None</code>
										</span>
									</li>
								</ul>
							</details>
						</td>
					</tr>
					<tr>
						<td>
							<code>timeout</code>
						</td>
						<td>
							<span>41</span>
						</td>
						<td>
							<span>10%</span>
						</td>
						<td>
							<details>
								<summary>See individual instances</summary>
								<ul>
									<li>
										<span>
											<a href="#140010763176784">Execution of Revision 2.3.0 of Workflow airrflow</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010761613616">Execution of Revision 2.2.0 of Workflow airrflow</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010763103024">Execution of Revision 2.1.0 of Workflow airrflow</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010762896384">Execution of Revision 2.0.0 of Workflow airrflow</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010746065440">Execution of Revision 2.4.1 of Workflow ampliseq</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010746369488">Execution of Revision 2.3.1 of Workflow ampliseq</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010746488208">Execution of Revision 2.2.0 of Workflow ampliseq</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010746555904">Execution of Revision 2.1.1 of Workflow ampliseq</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743050688">Execution of Revision 1.0.0 of Workflow cutandrun</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743111904">Execution of Revision 1.2 of Workflow fetchngs</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743114784">Execution of Revision 1.0.0 of Workflow hicar</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743115984">Execution of Revision 1.1.3 of Workflow hlatyping</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743118864">Execution of Revision 2.1.1 of Workflow mag</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743119104">Execution of Revision 2.1.0 of Workflow mag</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743145632">Execution of Revision 3.0.0 of Workflow nanoseq</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743146112">Execution of Revision 2.0.0 of Workflow nanoseq</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743152352">Execution of Revision 3.5 of Workflow rnaseq</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743194544">Execution of Revision 2.4.1 of Workflow viralrecon</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743195024">Execution of Revision 2.3.1 of Workflow viralrecon</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743195264">Execution of Revision 2.3 of Workflow viralrecon</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743195504">Execution of Revision 2.2 of Workflow viralrecon</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743195744">Execution of Revision 2.1 of Workflow viralrecon</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738206128">Execution of Revision 1.3.4 of Workflow CVUA-RRW/FooDMe</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738208768">Execution of Revision v1.3.6 of Workflow snakemake-workflows/benchmark-giab</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738209008">Execution of Revision v1.3.5 of Workflow snakemake-workflows/benchmark-giab</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738209248">Execution of Revision v1.3.4 of Workflow snakemake-workflows/benchmark-giab</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738209488">Execution of Revision v1.3.3 of Workflow snakemake-workflows/benchmark-giab</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738209968">Execution of Revision v1.3.1 of Workflow snakemake-workflows/benchmark-giab</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738210448">Execution of Revision v1.2.0 of Workflow snakemake-workflows/benchmark-giab</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738210928">Execution of Revision v1.0.1 of Workflow snakemake-workflows/benchmark-giab</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738211168">Execution of Revision v1.0.0 of Workflow snakemake-workflows/benchmark-giab</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738213808">Execution of Revision v1.3.6 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738214048">Execution of Revision v1.3.5 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738214288">Execution of Revision v1.3.4 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738214528">Execution of Revision v1.3.3 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738215008">Execution of Revision v1.3.1 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738215248">Execution of Revision v1.3.0 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738215488">Execution of Revision v1.2.0 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738215728">Execution of Revision v1.1.0 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738215968">Execution of Revision v1.0.1 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738216208">Execution of Revision v1.0.0 of Workflow snakemake-workflows/dna-seq-benchmark</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
								</ul>
							</details>
						</td>
					</tr>
					<tr>
						<td>
							<code>singularity</code>
						</td>
						<td>
							<span>35</span>
						</td>
						<td>
							<span>8%</span>
						</td>
						<td>
							<details>
								<summary>See individual instances</summary>
								<ul>
									<li>
										<span>
											<a href="#140010762278688">Execution of Revision 2.4.0 of Workflow airrflow</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010746334080">Execution of Revision 2.3.2 of Workflow ampliseq</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010745791584">Execution of Revision 1.1.1 of Workflow bacass</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743050448">Execution of Revision 1.1 of Workflow cutandrun</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743112144">Execution of Revision 1.1 of Workflow fetchngs</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743117184">Execution of Revision 1.0.0 of Workflow imcyto</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743119584">Execution of Revision 1.2.0 of Workflow mag</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743119824">Execution of Revision 1.1.2 of Workflow mag</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743136512">Execution of Revision 1.1.1 of Workflow mag</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743136752">Execution of Revision 1.1.0 of Workflow mag</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743145872">Execution of Revision 2.0.1 of Workflow nanoseq</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743152112">Execution of Revision 3.6 of Workflow rnaseq</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743194784">Execution of Revision 2.4 of Workflow viralrecon</a>
											<code>failed to pull singularity image</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739592944">Execution of Revision v1.2.0 of Workflow IMS-Bio2Core-Facility/GTExSnake</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010739661120">Execution of Revision v1.1.2 of Workflow IMS-Bio2Core-Facility/GTExSnake</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738166576">Execution of Revision 1.4.1 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738166816">Execution of Revision 1.3.6 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738167056">Execution of Revision 1.3.5 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738167296">Execution of Revision 1.3.4 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738167536">Execution of Revision 1.3.3 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code>singularity failed to pull image</code>
											<code>docker://weber8thomas/ashleys-qc-pipeline:1.3.3</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738167776">Execution of Revision 1.3.2 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738168016">Execution of Revision 1.3.1 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738168256">Execution of Revision 1.3.0 of Workflow friendsofstrandseq/ashleys-qc-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738220768">Execution of Revision 1.9.1 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738221008">Execution of Revision 1.9.0 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738254080">Execution of Revision 1.8.5 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738254320">Execution of Revision 1.8.4 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738254560">Execution of Revision 1.8.3 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738254800">Execution of Revision 1.8.2 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738255040">Execution of Revision 1.8.1 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738255280">Execution of Revision 1.7.0 of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738255760">Execution of Revision 1.5.1-dev of Workflow friendsofstrandseq/mosaicatcher-pipeline</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738269920">Execution of Revision v2.4.3 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738270160">Execution of Revision v2.4.2 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738286848">Execution of Revision v2.4.1 of Workflow snakemake-workflows/rna-seq-kallisto-sleuth</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
								</ul>
							</details>
						</td>
					</tr>
					<tr>
						<td>
							<code>workflow script error</code>
						</td>
						<td>
							<span>33</span>
						</td>
						<td>
							<span>8%</span>
						</td>
						<td>
							<details>
								<summary>See individual instances</summary>
								<ul>
									<li>
										<span>
											<a href="#140010746420512">Execution of Revision 2.3.0 of Workflow ampliseq</a>
											<code>java.lang.NullPointerException</code>
											<code>DataflowHelper.groovy:285</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010746658192">Execution of Revision 2.1.0 of Workflow ampliseq</a>
											<code>java.lang.NullPointerException</code>
											<code>DataflowHelper.groovy:285</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010746724944">Execution of Revision 2.0.0 of Workflow ampliseq</a>
											<code>Include statement is not allowed within a workflow definition</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010745351424">Execution of Revision 1.2.2 of Workflow atacseq</a>
											<code>nextflow.dag.MultipleInputChannelException</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010745419312">Execution of Revision 1.2.1 of Workflow atacseq</a>
											<code>nextflow.dag.MultipleInputChannelException</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010745487008">Execution of Revision 1.2.0 of Workflow atacseq</a>
											<code>nextflow.dag.MultipleInputChannelException</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010745538800">Execution of Revision 1.1.0 of Workflow atacseq</a>
											<code>nextflow.dag.MultipleInputChannelException</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743914288">Execution of Revision 1.0.0 of Workflow bactmap</a>
											<code>java.lang.NullPointerException</code>
											<code>BaseScript.groovy:170</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743112384">Execution of Revision 1.0 of Workflow fetchngs</a>
											<code>Include statement is not allowed within a workflow definition</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743119344">Execution of Revision 2.0.0 of Workflow mag</a>
											<code>Include statement is not allowed within a workflow definition</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743138192">Execution of Revision 2.1.0 of Workflow methylseq</a>
											<code>input file name collision</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743138432">Execution of Revision 2.0.0 of Workflow methylseq</a>
											<code>input file name collision</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743144432">Execution of Revision 1.2.1 of Workflow mhcquant</a>
											<code>java.io.IOException</code>
											<code>XFileSystemProvider.groovy:463</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743144912">Execution of Revision 1.1.0 of Workflow mhcquant</a>
											<code>java.io.IOException</code>
											<code>XFileSystemProvider.groovy:463</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743145152">Execution of Revision 1.0.0 of Workflow mhcquant</a>
											<code>java.io.IOException</code>
											<code>XFileSystemProvider.groovy:463</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743145392">Execution of Revision 1.0.0 of Workflow mnaseseq</a>
											<code>nextflow.dag.MultipleInputChannelException</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743147312">Execution of Revision 1.0 of Workflow nascent</a>
											<code>nextflow.exception.IllegalDirectiveException</code>
											<code>ProcessConfig.groovy:218</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743148752">Execution of Revision 1.0 of Workflow quantms</a>
											<code>nextflow.exception.AbortOperationException</code>
											<code>ConfigBuilder.groovy:511</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743149712">Execution of Revision 1.1.0 of Workflow rnafusion</a>
											<code>java.nio.file.NoSuchFileException</code>
											<code>ConfigBase.groovy:136</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743152592">Execution of Revision 3.4 of Workflow rnaseq</a>
											<code>java.lang.NullPointerException</code>
											<code>BaseScript.groovy:170</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743185664">Execution of Revision 3.3 of Workflow rnaseq</a>
											<code>java.lang.NullPointerException</code>
											<code>BaseScript.groovy:170</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743185904">Execution of Revision 3.2 of Workflow rnaseq</a>
											<code>java.lang.NullPointerException</code>
											<code>BaseScript.groovy:170</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743186144">Execution of Revision 3.1 of Workflow rnaseq</a>
											<code>Include statement is not allowed within a workflow definition</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743186384">Execution of Revision 3.0 of Workflow rnaseq</a>
											<code>Include statement is not allowed within a workflow definition</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743186624">Execution of Revision 2.0 of Workflow rnaseq</a>
											<code>Include statement is not allowed within a workflow definition</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743190224">Execution of Revision 2.7.2 of Workflow sarek</a>
											<code>nextflow.exception.AbortOperationException</code>
											<code>GraphObserver.groovy:77</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743190464">Execution of Revision 2.7.1 of Workflow sarek</a>
											<code>nextflow.exception.AbortOperationException</code>
											<code>GraphObserver.groovy:77</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743195984">Execution of Revision 2.0 of Workflow viralrecon</a>
											<code>Include statement is not allowed within a workflow definition</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738155728">Execution of Revision v1.0.0 of Workflow huzuner/Tumor-microbiome-calling</a>
											<code>caching multiple outputs</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738211408">Execution of Revision v1.0.0 of Workflow snakemake-workflows/species-quantification</a>
											<code>caching multiple outputs</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738259840">Execution of Revision v0.7.0 of Workflow IKIM-Essen/uncovar</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738260080">Execution of Revision v0.6.0 of Workflow IKIM-Essen/uncovar</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738260320">Execution of Revision v0.5.0 of Workflow IKIM-Essen/uncovar</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
								</ul>
							</details>
						</td>
					</tr>
					<tr>
						<td>
							<code>workflow step error</code>
						</td>
						<td>
							<span>7</span>
						</td>
						<td>
							<span>2%</span>
						</td>
						<td>
							<details>
								<summary>See individual instances</summary>
								<ul>
									<li>
										<span>
											<a href="#140010746032480">Execution of Revision 1.0.0 of Workflow airrflow</a>
											<code>x</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010745299920">Execution of Revision 2.0 of Workflow atacseq</a>
											<code>Missing output file(s)</code>
											<code>WARNING: Skipping mount /home/azureuser/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/singularityce-3.10.3-z5qu45ho72nvsy4rpcyd4eqrop35tnbe/var/singularity/mnt/session/etc/resolv.conf [files]: /etc/resolv.conf doesn't exist in container
Skipping 'OSMOTIC_STRESS_T100_SE_REP1_T1.gz' which didn't exist, or couldn't be read
											</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010745859520">Execution of Revision 1.1.0 of Workflow bacass</a>
											<code>/usr/bin/env: Rscript not found</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010745943696">Execution of Revision 1.0.0 of Workflow bacass</a>
											<code>tbl2asn -V b -a r10k -l paired-ends -M n -N 1 -y 'Annotated using prokka 1.13.3 from https://github.com/tseemann/prokka' -Z ERR044595_annotation\/ERR044595\.err -i ERR044595_annotation\/ERR044595\.fsa 2>
											</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742866096">Execution of Revision 1.1 of Workflow coproid</a>
											<code></code>
											<code>sqlite3.IntegrityError: UNIQUE constraint failed: synonym.spname, synonym.taxid</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010742917552">Execution of Revision 1.0 of Workflow coproid</a>
											<code></code>
											<code>sqlite3.IntegrityError: UNIQUE constraint failed: synonym.spname, synonym.taxid</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743143472">Execution of Revision 1.2.5 of Workflow mhcquant</a>
											<code>comet.exe</code>
											<code>None</code>
										</span>
									</li>
								</ul>
							</details>
						</td>
					</tr>
					<tr>
						<td>
							<code>network resource changed</code>
						</td>
						<td>
							<span>4</span>
						</td>
						<td>
							<span>1%</span>
						</td>
						<td>
							<details>
								<summary>See individual instances</summary>
								<ul>
									<li>
										<span>
											<a href="#140010743110944">Execution of Revision 1.6 of Workflow fetchngs</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743111184">Execution of Revision 1.5 of Workflow fetchngs</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743111424">Execution of Revision 1.4 of Workflow fetchngs</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010743111664">Execution of Revision 1.3 of Workflow fetchngs</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
								</ul>
							</details>
						</td>
					</tr>
					<tr>
						<td>
							<code>missing dep</code>
						</td>
						<td>
							<span>3</span>
						</td>
						<td>
							<span>1%</span>
						</td>
						<td>
							<details>
								<summary>See individual instances</summary>
								<ul>
									<li>
										<span>
											<a href="#140010743826128">Execution of Revision v1.0.1 of Workflow dbespiatykh/TBvar</a>
											<code>biopython</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010741990400">Execution of Revision v1.0.0 of Workflow dbespiatykh/TBvar</a>
											<code>biopython</code>
											<code>None</code>
										</span>
									</li>
									<li>
										<span>
											<a href="#140010738217168">Execution of Revision v1.0.0 of Workflow dbespiatykh/RDscan</a>
											<code>biopython</code>
											<code>None</code>
										</span>
									</li>
								</ul>
							</details>
						</td>
					</tr>
					<tr>
						<td>
							<code>experiment error</code>
						</td>
						<td>
							<span>1</span>
						</td>
						<td>
							<span>0%</span>
						</td>
						<td>
							<details>
								<summary>See individual instances</summary>
								<ul>
									<li>
										<span>
											<a href="#140010743050208">Execution of Revision 2.0 of Workflow cutandrun</a>
											<code></code>
											<code>None</code>
										</span>
									</li>
								</ul>
							</details>
						</td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
				</tbody>
			</table>
		</div>
		<h2 id="workflows">
			<a href="#workflows">Workflows</a>
		</h2>
		<table>
			<thead>
				<tr>
					<td>Workflow</td>
					<td>Engine</td>
					<td>Repo</td>
					<td>Interesting?</td>
					<td>Revisions</td>
				</tr>
			</thead>
			<tbody>
				<tr>
					<td>
						<a href="https://nf-co.re/airrflow">airrflow</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/airrflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/airrflow/tree/2.4.0">2.4.0</a>
										</td>
										<td>2022-12-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010762278688">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/airrflow/tree/2.3.0">2.3.0</a>
										</td>
										<td>2022-09-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010763176784">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/airrflow/tree/2.2.0">2.2.0</a>
										</td>
										<td>2022-06-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010761613616">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/airrflow/tree/2.1.0">2.1.0</a>
										</td>
										<td>2022-05-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010763103024">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/airrflow/tree/2.0.0">2.0.0</a>
										</td>
										<td>2021-07-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010762896384">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/airrflow/tree/1.2.0">1.2.0</a>
										</td>
										<td>2020-01-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010762702656">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/airrflow/tree/1.1.0">1.1.0</a>
										</td>
										<td>2019-11-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010763149536">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/airrflow/tree/1.0.0">1.0.0</a>
										</td>
										<td>2019-04-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746032480">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/ampliseq">ampliseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/ampliseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/2.4.1">2.4.1</a>
										</td>
										<td>2022-12-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746065440">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/2.4.0">2.4.0</a>
										</td>
										<td>2022-09-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746200704">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/2.3.2">2.3.2</a>
										</td>
										<td>2022-05-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746334080">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/2.3.1">2.3.1</a>
										</td>
										<td>2022-04-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746369488">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/2.3.0">2.3.0</a>
										</td>
										<td>2022-04-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746420512">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/2.2.0">2.2.0</a>
										</td>
										<td>2022-01-31</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746488208">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/2.1.1">2.1.1</a>
										</td>
										<td>2021-10-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746555904">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/2.1.0">2.1.0</a>
										</td>
										<td>2021-09-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746658192">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/2.0.0">2.0.0</a>
										</td>
										<td>2021-06-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746724944">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/1.2.0">1.2.0</a>
										</td>
										<td>2021-02-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746842512">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/1.1.3">1.1.3</a>
										</td>
										<td>2020-11-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746877680">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/1.1.2">1.1.2</a>
										</td>
										<td>2019-12-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746961808">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/1.1.1">1.1.1</a>
										</td>
										<td>2019-12-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010746997168">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/1.1.0">1.1.0</a>
										</td>
										<td>2019-07-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744967760">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/ampliseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2018-12-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745002688">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/atacseq">atacseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/atacseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/atacseq/tree/2.0">2.0</a>
										</td>
										<td>2022-11-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745299920">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/atacseq/tree/1.2.2">1.2.2</a>
										</td>
										<td>2022-05-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745351424">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/atacseq/tree/1.2.1">1.2.1</a>
										</td>
										<td>2020-07-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745419312">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/atacseq/tree/1.2.0">1.2.0</a>
										</td>
										<td>2020-07-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745487008">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/atacseq/tree/1.1.0">1.1.0</a>
										</td>
										<td>2019-11-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745538800">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/atacseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2019-04-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745623072">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/bacass">bacass</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/bacass?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/bacass/tree/2.0.0">2.0.0</a>
										</td>
										<td>2021-08-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745674624">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/bacass/tree/1.1.1">1.1.1</a>
										</td>
										<td>2020-11-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745791584">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/bacass/tree/1.1.0">1.1.0</a>
										</td>
										<td>2019-12-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745859520">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/bacass/tree/1.0.0">1.0.0</a>
										</td>
										<td>2019-05-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010745943696">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/bactmap">bactmap</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/bactmap?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/bactmap/tree/1.0.0">1.0.0</a>
										</td>
										<td>2021-06-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743914288">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/cageseq">cageseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/cageseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/cageseq/tree/1.0.2">1.0.2</a>
										</td>
										<td>2021-01-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743965792">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/cageseq/tree/1.0.1">1.0.1</a>
										</td>
										<td>2020-11-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744033968">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/cageseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2020-10-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744085424">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/callingcards">callingcards</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/callingcards?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/chipseq">chipseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/chipseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/chipseq/tree/2.0.0">2.0.0</a>
										</td>
										<td>2022-10-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744202528">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/chipseq/tree/1.2.2">1.2.2</a>
										</td>
										<td>2021-04-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744270512">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/chipseq/tree/1.2.1">1.2.1</a>
										</td>
										<td>2020-07-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744354976">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/chipseq/tree/1.2.0">1.2.0</a>
										</td>
										<td>2020-07-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744422672">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/chipseq/tree/1.1.0">1.1.0</a>
										</td>
										<td>2019-11-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744507232">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/chipseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2019-06-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744558832">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/circdna">circdna</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/circdna?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/circdna/tree/1.0.1">1.0.1</a>
										</td>
										<td>2022-06-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744626624">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/circdna/tree/1.0.0">1.0.0</a>
										</td>
										<td>2022-05-31</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744727152">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/circrna">circrna</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/circrna?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/clipseq">clipseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/clipseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/clipseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2021-04-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744795040">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/coproid">coproid</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/coproid?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/coproid/tree/1.1.1">1.1.1</a>
										</td>
										<td>2022-11-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010744879120">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/coproid/tree/1.1">1.1</a>
										</td>
										<td>2020-04-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742866096">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/coproid/tree/1.0">1.0</a>
										</td>
										<td>2019-04-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742917552">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/crisprseq">crisprseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/crisprseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/crisprseq/tree/1.0">1.0</a>
										</td>
										<td>2023-02-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743002112">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/crisprvar">crisprvar</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/crisprvar?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/cutandrun">cutandrun</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/cutandrun?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/cutandrun/tree/3.0">3.0</a>
										</td>
										<td>2022-10-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743049968">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/cutandrun/tree/2.0">2.0</a>
										</td>
										<td>2022-06-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743050208">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/cutandrun/tree/1.1">1.1</a>
										</td>
										<td>2022-01-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743050448">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/cutandrun/tree/1.0.0">1.0.0</a>
										</td>
										<td>2021-11-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743050688">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/ddamsproteomics">ddamsproteomics</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/ddamsproteomics?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/deepvariant">deepvariant</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/deepvariant?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/deepvariant/tree/1.0">1.0</a>
										</td>
										<td>2018-11-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743050928">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/demultiplex">demultiplex</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/demultiplex?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/demultiplex/tree/1.1.0">1.1.0</a>
										</td>
										<td>2023-01-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743051168">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/demultiplex/tree/1.0.0">1.0.0</a>
										</td>
										<td>2022-10-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743051408">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/denovohybrid">denovohybrid</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/denovohybrid?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/diaproteomics">diaproteomics</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/diaproteomics?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/diaproteomics/tree/1.2.4">1.2.4</a>
										</td>
										<td>2021-04-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743051648">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/diaproteomics/tree/1.2.3">1.2.3</a>
										</td>
										<td>2021-04-26</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743051888">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/diaproteomics/tree/1.2.2">1.2.2</a>
										</td>
										<td>2021-02-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743052128">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/diaproteomics/tree/1.2.1">1.2.1</a>
										</td>
										<td>2021-02-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743052368">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/diaproteomics/tree/1.2.0">1.2.0</a>
										</td>
										<td>2020-12-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743052608">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/diaproteomics/tree/1.1.0">1.1.0</a>
										</td>
										<td>2020-12-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743052848">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/diaproteomics/tree/1.0.0">1.0.0</a>
										</td>
										<td>2020-11-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743053088">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/differentialabundance">differentialabundance</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/differentialabundance?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/differentialabundance/tree/1.0.1">1.0.1</a>
										</td>
										<td>2023-01-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743053328">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/differentialabundance/tree/1.0.0">1.0.0</a>
										</td>
										<td>2023-01-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743053568">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/dualrnaseq">dualrnaseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/dualrnaseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/dualrnaseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2021-02-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743053808">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/eager">eager</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/eager?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.4.6">2.4.6</a>
										</td>
										<td>2022-11-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743054048">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.4.5">2.4.5</a>
										</td>
										<td>2022-08-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743054288">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.4.4">2.4.4</a>
										</td>
										<td>2022-04-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743103744">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.4.3">2.4.3</a>
										</td>
										<td>2022-03-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743103984">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.4.2">2.4.2</a>
										</td>
										<td>2022-01-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743104224">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.4.1">2.4.1</a>
										</td>
										<td>2021-11-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743104464">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.4.0">2.4.0</a>
										</td>
										<td>2021-09-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743104704">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.3.5">2.3.5</a>
										</td>
										<td>2021-06-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743104944">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.3.4">2.3.4</a>
										</td>
										<td>2021-05-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743105184">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.3.3">2.3.3</a>
										</td>
										<td>2021-04-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743105424">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.3.2">2.3.2</a>
										</td>
										<td>2021-03-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743105664">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.3.1">2.3.1</a>
										</td>
										<td>2021-01-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743105904">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.3.0">2.3.0</a>
										</td>
										<td>2021-01-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743106144">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.2.2">2.2.2</a>
										</td>
										<td>2020-12-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743106384">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.2.1">2.2.1</a>
										</td>
										<td>2020-10-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743106624">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.2.0">2.2.0</a>
										</td>
										<td>2020-10-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743106864">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.1.0">2.1.0</a>
										</td>
										<td>2020-03-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743107104">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.0.7">2.0.7</a>
										</td>
										<td>2019-06-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743107344">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.0.6">2.0.6</a>
										</td>
										<td>2019-03-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743107584">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.0.5">2.0.5</a>
										</td>
										<td>2019-01-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743107824">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.0.4">2.0.4</a>
										</td>
										<td>2019-01-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743108064">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.0.3">2.0.3</a>
										</td>
										<td>2018-12-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743108304">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.0.2">2.0.2</a>
										</td>
										<td>2018-11-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743108544">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.0.1">2.0.1</a>
										</td>
										<td>2018-11-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743108784">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/eager/tree/2.0.0">2.0.0</a>
										</td>
										<td>2018-10-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743109024">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/epitopeprediction">epitopeprediction</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/epitopeprediction?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/epitopeprediction/tree/2.1.0">2.1.0</a>
										</td>
										<td>2022-08-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743109264">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/epitopeprediction/tree/2.0.0">2.0.0</a>
										</td>
										<td>2021-12-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743109504">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/epitopeprediction/tree/1.1.0">1.1.0</a>
										</td>
										<td>2020-10-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743109744">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/epitopeprediction/tree/1.0.0">1.0.0</a>
										</td>
										<td>2019-12-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743109984">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/exoseq">exoseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/exoseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/fastquorum">fastquorum</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/fastquorum?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/fetchngs">fetchngs</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/fetchngs?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/fetchngs/tree/1.9">1.9</a>
										</td>
										<td>2022-12-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743110224">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/fetchngs/tree/1.8">1.8</a>
										</td>
										<td>2022-11-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743110464">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/fetchngs/tree/1.7">1.7</a>
										</td>
										<td>2022-07-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743110704">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/fetchngs/tree/1.6">1.6</a>
										</td>
										<td>2022-05-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743110944">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/fetchngs/tree/1.5">1.5</a>
										</td>
										<td>2021-12-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743111184">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/fetchngs/tree/1.4">1.4</a>
										</td>
										<td>2021-11-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743111424">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/fetchngs/tree/1.3">1.3</a>
										</td>
										<td>2021-09-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743111664">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/fetchngs/tree/1.2">1.2</a>
										</td>
										<td>2021-07-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743111904">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/fetchngs/tree/1.1">1.1</a>
										</td>
										<td>2021-06-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743112144">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/fetchngs/tree/1.0">1.0</a>
										</td>
										<td>2021-06-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743112384">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/funcscan">funcscan</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/funcscan?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/funcscan/tree/1.0.0">1.0.0</a>
										</td>
										<td>2023-02-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743112624">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/genomeannotator">genomeannotator</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/genomeannotator?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/genomeassembler">genomeassembler</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/genomeassembler?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/genomeskim">genomeskim</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/genomeskim?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/gwas">gwas</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/gwas?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/hgtseq">hgtseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/hgtseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hgtseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2022-10-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743112864">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/hic">hic</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/hic?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hic/tree/2.0.0">2.0.0</a>
										</td>
										<td>2023-01-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743113104">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hic/tree/1.3.0">1.3.0</a>
										</td>
										<td>2021-05-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743113344">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hic/tree/1.2.2">1.2.2</a>
										</td>
										<td>2020-09-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743113584">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hic/tree/1.2.1">1.2.1</a>
										</td>
										<td>2020-07-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743113824">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hic/tree/1.2.0">1.2.0</a>
										</td>
										<td>2020-06-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743114064">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hic/tree/1.1.0">1.1.0</a>
										</td>
										<td>2019-10-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743114304">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hic/tree/1.0.0">1.0.0</a>
										</td>
										<td>2019-05-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743114544">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/hicar">hicar</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/hicar?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hicar/tree/1.0.0">1.0.0</a>
										</td>
										<td>2022-05-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743114784">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/hlatyping">hlatyping</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/hlatyping?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hlatyping/tree/2.0.0">2.0.0</a>
										</td>
										<td>2022-10-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743115024">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hlatyping/tree/1.2.0">1.2.0</a>
										</td>
										<td>2020-08-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743115264">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hlatyping/tree/1.1.5">1.1.5</a>
										</td>
										<td>2019-06-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743115504">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hlatyping/tree/1.1.4">1.1.4</a>
										</td>
										<td>2019-03-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743115744">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hlatyping/tree/1.1.3">1.1.3</a>
										</td>
										<td>2019-02-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743115984">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hlatyping/tree/1.1.2">1.1.2</a>
										</td>
										<td>2018-12-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743116224">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hlatyping/tree/1.1.1">1.1.1</a>
										</td>
										<td>2018-08-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743116464">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hlatyping/tree/1.1.0">1.1.0</a>
										</td>
										<td>2018-08-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743116704">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/hlatyping/tree/1.0.0">1.0.0</a>
										</td>
										<td>2018-07-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743116944">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/imcyto">imcyto</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/imcyto?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/imcyto/tree/1.0.0">1.0.0</a>
										</td>
										<td>2020-05-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743117184">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/isoseq">isoseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/isoseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/isoseq/tree/1.1.2">1.1.2</a>
										</td>
										<td>2023-01-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743117424">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/isoseq/tree/1.1.1">1.1.1</a>
										</td>
										<td>2022-09-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743117664">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/isoseq/tree/1.1.0">1.1.0</a>
										</td>
										<td>2022-07-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743117904">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/isoseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2022-06-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743118144">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/kmermaid">kmermaid</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/kmermaid?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/liverctanalysis">liverctanalysis</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/liverctanalysis?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/lncpipe">lncpipe</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/lncpipe?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/mag">mag</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/mag?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mag/tree/2.2.1">2.2.1</a>
										</td>
										<td>2022-08-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743118384">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mag/tree/2.2.0">2.2.0</a>
										</td>
										<td>2022-06-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743118624">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mag/tree/2.1.1">2.1.1</a>
										</td>
										<td>2021-11-26</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743118864">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mag/tree/2.1.0">2.1.0</a>
										</td>
										<td>2021-07-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743119104">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mag/tree/2.0.0">2.0.0</a>
										</td>
										<td>2021-06-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743119344">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mag/tree/1.2.0">1.2.0</a>
										</td>
										<td>2021-02-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743119584">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mag/tree/1.1.2">1.1.2</a>
										</td>
										<td>2020-11-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743119824">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mag/tree/1.1.1">1.1.1</a>
										</td>
										<td>2020-11-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743136512">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mag/tree/1.1.0">1.1.0</a>
										</td>
										<td>2020-10-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743136752">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mag/tree/1.0.0">1.0.0</a>
										</td>
										<td>2019-12-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743136992">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/metaboigniter">metaboigniter</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/metaboigniter?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/metaboigniter/tree/1.0.1">1.0.1</a>
										</td>
										<td>2021-05-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743137232">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/metaboigniter/tree/1.0.0">1.0.0</a>
										</td>
										<td>2021-05-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743137472">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/metapep">metapep</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/metapep?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/methylseq">methylseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/methylseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/2.3.0">2.3.0</a>
										</td>
										<td>2022-12-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743137712">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/2.2.0">2.2.0</a>
										</td>
										<td>2022-11-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743137952">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/2.1.0">2.1.0</a>
										</td>
										<td>2022-11-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743138192">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/2.0.0">2.0.0</a>
										</td>
										<td>2022-11-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743138432">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/1.6.1">1.6.1</a>
										</td>
										<td>2021-05-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743138672">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/1.6">1.6</a>
										</td>
										<td>2021-03-26</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743138912">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/1.5">1.5</a>
										</td>
										<td>2020-04-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743139152">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/1.4">1.4</a>
										</td>
										<td>2019-11-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743139392">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/1.3">1.3</a>
										</td>
										<td>2019-02-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743139632">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/1.2">1.2</a>
										</td>
										<td>2019-01-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743139872">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/1.1">1.1</a>
										</td>
										<td>2018-08-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743140112">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/methylseq/tree/1.0">1.0</a>
										</td>
										<td>2018-04-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743140352">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/mhcquant">mhcquant</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/mhcquant?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/2.4.0">2.4.0</a>
										</td>
										<td>2022-12-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743140592">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/2.3.1">2.3.1</a>
										</td>
										<td>2022-05-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743140832">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/2.3.0">2.3.0</a>
										</td>
										<td>2022-04-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743141072">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/2.2.0">2.2.0</a>
										</td>
										<td>2022-01-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743141312">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/2.1.0">2.1.0</a>
										</td>
										<td>2021-12-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743141552">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/2.0.0">2.0.0</a>
										</td>
										<td>2021-09-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743141792">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.6.0">1.6.0</a>
										</td>
										<td>2020-10-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743142032">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.5.1">1.5.1</a>
										</td>
										<td>2020-04-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743142272">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.5">1.5</a>
										</td>
										<td>2020-04-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743142512">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.4">1.4</a>
										</td>
										<td>2020-03-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743142752">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.3">1.3</a>
										</td>
										<td>2019-08-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743142992">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.2.6">1.2.6</a>
										</td>
										<td>2019-03-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743143232">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.2.5">1.2.5</a>
										</td>
										<td>2019-03-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743143472">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.2.4">1.2.4</a>
										</td>
										<td>2019-02-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743143712">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.2.3">1.2.3</a>
										</td>
										<td>2019-02-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743143952">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.2.2">1.2.2</a>
										</td>
										<td>2019-01-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743144192">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.2.1">1.2.1</a>
										</td>
										<td>2019-01-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743144432">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.2.0">1.2.0</a>
										</td>
										<td>2019-01-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743144672">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.1.0">1.1.0</a>
										</td>
										<td>2019-01-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743144912">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mhcquant/tree/1.0.0">1.0.0</a>
										</td>
										<td>2018-11-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743145152">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/mnaseseq">mnaseseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/mnaseseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/mnaseseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2022-05-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743145392">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/nanoseq">nanoseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/nanoseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/nanoseq/tree/3.0.0">3.0.0</a>
										</td>
										<td>2022-06-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743145632">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/nanoseq/tree/2.0.1">2.0.1</a>
										</td>
										<td>2021-11-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743145872">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/nanoseq/tree/2.0.0">2.0.0</a>
										</td>
										<td>2021-11-26</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743146112">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/nanoseq/tree/1.1.0">1.1.0</a>
										</td>
										<td>2020-11-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743146352">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/nanoseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2020-03-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743146592">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/nanostring">nanostring</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/nanostring?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/nascent">nascent</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/nascent?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/nascent/tree/2.1.0">2.1.0</a>
										</td>
										<td>2023-02-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743146832">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/nascent/tree/2.0.0">2.0.0</a>
										</td>
										<td>2022-10-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743147072">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/nascent/tree/1.0">1.0</a>
										</td>
										<td>2019-04-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743147312">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/neutronstar">neutronstar</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/neutronstar?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/neutronstar/tree/1.0.0">1.0.0</a>
										</td>
										<td>2019-12-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743147552">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/pangenome">pangenome</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/pangenome?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/pgdb">pgdb</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/pgdb?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/pgdb/tree/1.0.0">1.0.0</a>
										</td>
										<td>2021-04-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743147792">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/phyloplace">phyloplace</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/phyloplace?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/phyloplace/tree/1.0.0">1.0.0</a>
										</td>
										<td>2023-02-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743148032">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/proteinfold">proteinfold</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/proteinfold?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/proteinfold/tree/1.0.0">1.0.0</a>
										</td>
										<td>2023-02-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743148272">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/proteomicslfq">proteomicslfq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/proteomicslfq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/proteomicslfq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2020-10-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743148512">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/quantms">quantms</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/quantms?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/quantms/tree/1.0">1.0</a>
										</td>
										<td>2022-05-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743148752">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/raredisease">raredisease</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/raredisease?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/rnadnavar">rnadnavar</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/rnadnavar?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/rnafusion">rnafusion</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/rnafusion?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnafusion/tree/2.1.0">2.1.0</a>
										</td>
										<td>2022-07-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743148992">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnafusion/tree/2.0.0">2.0.0</a>
										</td>
										<td>2022-05-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743149232">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnafusion/tree/1.2.0">1.2.0</a>
										</td>
										<td>2020-07-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743149472">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnafusion/tree/1.1.0">1.1.0</a>
										</td>
										<td>2020-02-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743149712">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnafusion/tree/1.0.2">1.0.2</a>
										</td>
										<td>2019-05-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743149952">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnafusion/tree/1.0.1">1.0.1</a>
										</td>
										<td>2019-04-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743150192">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnafusion/tree/1.0">1.0</a>
										</td>
										<td>2019-02-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743150432">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/rnaseq">rnaseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/rnaseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.10.1">3.10.1</a>
										</td>
										<td>2023-01-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743150672">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.10">3.10</a>
										</td>
										<td>2022-12-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743150912">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.9">3.9</a>
										</td>
										<td>2022-09-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743151152">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.8.1">3.8.1</a>
										</td>
										<td>2022-05-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743151392">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.8">3.8</a>
										</td>
										<td>2022-05-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743151632">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.7">3.7</a>
										</td>
										<td>2022-05-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743151872">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.6">3.6</a>
										</td>
										<td>2022-03-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743152112">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.5">3.5</a>
										</td>
										<td>2021-12-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743152352">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.4">3.4</a>
										</td>
										<td>2021-10-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743152592">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.3">3.3</a>
										</td>
										<td>2021-07-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743185664">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.2">3.2</a>
										</td>
										<td>2021-06-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743185904">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.1">3.1</a>
										</td>
										<td>2021-05-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743186144">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/3.0">3.0</a>
										</td>
										<td>2020-12-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743186384">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/2.0">2.0</a>
										</td>
										<td>2020-11-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743186624">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/1.4.2">1.4.2</a>
										</td>
										<td>2019-10-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743186864">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/1.4.1">1.4.1</a>
										</td>
										<td>2019-10-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743187104">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/1.4">1.4</a>
										</td>
										<td>2019-10-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743187344">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/1.3">1.3</a>
										</td>
										<td>2019-03-26</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743187584">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/1.2">1.2</a>
										</td>
										<td>2018-12-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743187824">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/1.1">1.1</a>
										</td>
										<td>2018-10-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743188064">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnaseq/tree/1.0">1.0</a>
										</td>
										<td>2018-08-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743188304">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/rnasplice">rnasplice</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/rnasplice?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/rnavar">rnavar</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/rnavar?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/rnavar/tree/1.0.0">1.0.0</a>
										</td>
										<td>2022-06-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743188544">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/sarek">sarek</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/sarek?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/3.1.2">3.1.2</a>
										</td>
										<td>2023-01-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743188784">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/3.1.1">3.1.1</a>
										</td>
										<td>2022-11-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743189024">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/3.1">3.1</a>
										</td>
										<td>2022-11-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743189264">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/3.0.2">3.0.2</a>
										</td>
										<td>2022-09-26</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743189504">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/3.0.1">3.0.1</a>
										</td>
										<td>2022-08-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743189744">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/3.0">3.0</a>
										</td>
										<td>2022-07-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743189984">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/2.7.2">2.7.2</a>
										</td>
										<td>2022-06-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743190224">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/2.7.1">2.7.1</a>
										</td>
										<td>2021-06-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743190464">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/2.7">2.7</a>
										</td>
										<td>2021-01-26</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743190704">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/2.6.1">2.6.1</a>
										</td>
										<td>2020-06-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743190944">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/2.6">2.6</a>
										</td>
										<td>2020-05-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743191184">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/2.5.2">2.5.2</a>
										</td>
										<td>2019-12-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743191424">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/2.5.1">2.5.1</a>
										</td>
										<td>2019-10-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743191664">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/sarek/tree/2.5">2.5</a>
										</td>
										<td>2019-10-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743191904">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/scflow">scflow</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/scflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/scrnaseq">scrnaseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/scrnaseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/scrnaseq/tree/2.1.0">2.1.0</a>
										</td>
										<td>2022-10-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743192144">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/scrnaseq/tree/2.0.0">2.0.0</a>
										</td>
										<td>2022-06-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743192384">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/scrnaseq/tree/1.1.0">1.1.0</a>
										</td>
										<td>2021-03-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743192624">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/scrnaseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2019-12-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743192864">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/slamseq">slamseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/slamseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/slamseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2020-05-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743193104">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/smartseq2">smartseq2</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/smartseq2?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/smrnaseq">smrnaseq</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/smrnaseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/smrnaseq/tree/2.1.0">2.1.0</a>
										</td>
										<td>2022-10-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743193344">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/smrnaseq/tree/2.0.0">2.0.0</a>
										</td>
										<td>2022-05-31</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743193584">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/smrnaseq/tree/1.1.0">1.1.0</a>
										</td>
										<td>2021-06-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743193824">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/smrnaseq/tree/1.0.0">1.0.0</a>
										</td>
										<td>2019-09-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743194064">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/spatialtranscriptomics">spatialtranscriptomics</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/spatialtranscriptomics?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/ssds">ssds</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/ssds?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/taxprofiler">taxprofiler</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/taxprofiler?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/vipr">vipr</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/vipr?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/viralintegration">viralintegration</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/viralintegration?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://nf-co.re/viralrecon">viralrecon</a>
					</td>
					<td>nextflow</td>
					<td>
						<a href="https://github.com/nf-core/viralrecon?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nf-core/viralrecon/tree/2.5">2.5</a>
										</td>
										<td>2022-07-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743194304">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/viralrecon/tree/2.4.1">2.4.1</a>
										</td>
										<td>2022-03-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743194544">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/viralrecon/tree/2.4">2.4</a>
										</td>
										<td>2022-02-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743194784">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/viralrecon/tree/2.3.1">2.3.1</a>
										</td>
										<td>2022-02-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743195024">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/viralrecon/tree/2.3">2.3</a>
										</td>
										<td>2022-02-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743195264">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/viralrecon/tree/2.2">2.2</a>
										</td>
										<td>2021-07-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743195504">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/viralrecon/tree/2.1">2.1</a>
										</td>
										<td>2021-06-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743195744">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/viralrecon/tree/2.0">2.0</a>
										</td>
										<td>2021-05-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743195984">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/viralrecon/tree/1.1.0">1.1.0</a>
										</td>
										<td>2020-06-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743196224">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nf-core/viralrecon/tree/1.0.0">1.0.0</a>
										</td>
										<td>2020-06-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743196464">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/simonmoulds/gwm-analysis">simonmoulds/gwm-analysis</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/simonmoulds/gwm-analysis?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/seankmartin/atn-sub-lfp-workflow">seankmartin/atn-sub-lfp-workflow</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/seankmartin/atn-sub-lfp-workflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/seankmartin/atn-sub-lfp-workflow/tree/23.02.0">23.02.0</a>
										</td>
										<td>2023-02-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743252592">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/hzi-bifo/phylogeny-of-single-cells">hzi-bifo/phylogeny-of-single-cells</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/hzi-bifo/phylogeny-of-single-cells?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/DSiB-G2/G2-CovPipeline">DSiB-G2/G2-CovPipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/DSiB-G2/G2-CovPipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/DSiB-G2/G2-CovPipeline/tree/v1.0">v1.0</a>
										</td>
										<td>2023-02-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743201264">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/dbespiatykh/TBvar">dbespiatykh/TBvar</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/dbespiatykh/TBvar?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/dbespiatykh/TBvar/tree/v1.1.4">v1.1.4</a>
										</td>
										<td>2023-02-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743477968">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/dbespiatykh/TBvar/tree/v1.1.3">v1.1.3</a>
										</td>
										<td>2023-01-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743528416">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/dbespiatykh/TBvar/tree/v1.1.2">v1.1.2</a>
										</td>
										<td>2022-12-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743577280">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/dbespiatykh/TBvar/tree/v1.1.1">v1.1.1</a>
										</td>
										<td>2022-12-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743694336">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/dbespiatykh/TBvar/tree/v1.0.1">v1.0.1</a>
										</td>
										<td>2022-12-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010743826128">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/dbespiatykh/TBvar/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2022-07-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741990400">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/NaKent/metagenome_workflow">NaKent/metagenome_workflow</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/NaKent/metagenome_workflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/cjgunnar/snake-dreem">cjgunnar/snake-dreem</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/cjgunnar/snake-dreem?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/simonmoulds/uk-decadal-flood-prediction">simonmoulds/uk-decadal-flood-prediction</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/simonmoulds/uk-decadal-flood-prediction?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/simonmoulds/uk-decadal-flood-prediction/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2023-01-31</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741895216">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/EichlerLab/ONT_pipelines">EichlerLab/ONT_pipelines</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/EichlerLab/ONT_pipelines?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/nixonlab/snakemake_rule_library">nixonlab/snakemake_rule_library</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/nixonlab/snakemake_rule_library?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/metagenome-atlas/metagenome-assembly">metagenome-atlas/metagenome-assembly</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/metagenome-atlas/metagenome-assembly?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/WanlinLi2021/demo">WanlinLi2021/demo</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/WanlinLi2021/demo?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/tbenavi1/RetroDetector">tbenavi1/RetroDetector</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/tbenavi1/RetroDetector?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/matinnuhamunada/interproscan_helper">matinnuhamunada/interproscan_helper</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/matinnuhamunada/interproscan_helper?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/valentynbez/FuncMi">valentynbez/FuncMi</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/valentynbez/FuncMi?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/dna-seq-short-read-circle-map">snakemake-workflows/dna-seq-short-read-circle-map</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/dna-seq-short-read-circle-map?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/simonmoulds/jules-mosart">simonmoulds/jules-mosart</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/simonmoulds/jules-mosart?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/nate-d-olson/defrabb">nate-d-olson/defrabb</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/nate-d-olson/defrabb?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/zifornd/arrays">zifornd/arrays</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/zifornd/arrays?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/zifornd/arrays/tree/1.0.0">1.0.0</a>
										</td>
										<td>2022-12-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741945760">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/sandragodinhosilva/melange">sandragodinhosilva/melange</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/sandragodinhosilva/melange?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/johnne/map_metaT">johnne/map_metaT</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/johnne/map_metaT?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/johnne/map_metaT/tree/v0.0.2">v0.0.2</a>
										</td>
										<td>2022-05-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742126992">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/johnne/map_metaT/tree/v0.0.1">v0.0.1</a>
										</td>
										<td>2022-05-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742160672">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Zilong-Li/array-imputation-workflow">Zilong-Li/array-imputation-workflow</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Zilong-Li/array-imputation-workflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/ArsenaultResearch/RNAseq_Pipeline">ArsenaultResearch/RNAseq_Pipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/ArsenaultResearch/RNAseq_Pipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Sara-Morsy/metaomic-workflow">Sara-Morsy/metaomic-workflow</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Sara-Morsy/metaomic-workflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Wytamma/snk-basic-pipeline">Wytamma/snk-basic-pipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Wytamma/snk-basic-pipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/simonmoulds/decadal-prediction-data-preparation">simonmoulds/decadal-prediction-data-preparation</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/simonmoulds/decadal-prediction-data-preparation?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/SichongP/atac-seq">SichongP/atac-seq</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/SichongP/atac-seq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/delpropo/structural-variant-analysis">delpropo/structural-variant-analysis</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/delpropo/structural-variant-analysis?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/rmFlynn/snakemake_example">rmFlynn/snakemake_example</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/rmFlynn/snakemake_example?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/trickovicmatija/kingfisher_workflow">trickovicmatija/kingfisher_workflow</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/trickovicmatija/kingfisher_workflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/AmberFu/snakemake_BAMmetrics_mosdepth">AmberFu/snakemake_BAMmetrics_mosdepth</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/AmberFu/snakemake_BAMmetrics_mosdepth?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/delpropo/bam-fq">delpropo/bam-fq</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/delpropo/bam-fq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/simonmoulds/jules-ganges-data-preparation">simonmoulds/jules-ganges-data-preparation</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/simonmoulds/jules-ganges-data-preparation?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/crs4/fair-crcc-get-data">crs4/fair-crcc-get-data</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/crs4/fair-crcc-get-data?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/crs4/fair-crcc-get-data/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2022-10-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742194928">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/solida-core/musta">solida-core/musta</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/solida-core/musta?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/solida-core/musta/tree/v1.2.2">v1.2.2</a>
										</td>
										<td>2022-07-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742245568">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/musta/tree/v1.2.1">v1.2.1</a>
										</td>
										<td>2022-07-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742311776">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/musta/tree/v1.2.0.1">v1.2.0.1</a>
										</td>
										<td>2022-04-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742362176">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/musta/tree/v1.2">v1.2</a>
										</td>
										<td>2022-04-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742494640">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/musta/tree/1.0">1.0</a>
										</td>
										<td>2022-04-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742578048">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/musta/tree/v1.1.1">v1.1.1</a>
										</td>
										<td>2022-02-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742611728">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/musta/tree/v1.1">v1.1</a>
										</td>
										<td>2022-02-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742694000">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/musta/tree/v1.0">v1.0</a>
										</td>
										<td>2022-02-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010742777264">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/omidalam/can_vac">omidalam/can_vac</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/omidalam/can_vac?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/rmFlynn/dram_validataion_pipline">rmFlynn/dram_validataion_pipline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/rmFlynn/dram_validataion_pipline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/myousif9/snsx_surface_analysis">myousif9/snsx_surface_analysis</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/myousif9/snsx_surface_analysis?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/MaNyh/NTRFlows">MaNyh/NTRFlows</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/MaNyh/NTRFlows?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/simonmoulds/us-streamflow-prediction">simonmoulds/us-streamflow-prediction</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/simonmoulds/us-streamflow-prediction?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/panguangze/trio_call_phase_pipeline">panguangze/trio_call_phase_pipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/panguangze/trio_call_phase_pipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/kvntrc/snakemake_template">kvntrc/snakemake_template</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/kvntrc/snakemake_template?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/zifornd/array">zifornd/array</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/zifornd/array?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/zifornd/array/tree/1.0.0">1.0.0</a>
										</td>
										<td>2022-12-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740861632">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/simonmoulds/aquacrop-vegmon-analysis">simonmoulds/aquacrop-vegmon-analysis</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/simonmoulds/aquacrop-vegmon-analysis?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/yeyuan98/rna_fish_analysis">yeyuan98/rna_fish_analysis</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/yeyuan98/rna_fish_analysis?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/seung1yoo/snakemake-s3-prototype">seung1yoo/snakemake-s3-prototype</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/seung1yoo/snakemake-s3-prototype?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Yasas1994/metagenome_assembly">Yasas1994/metagenome_assembly</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Yasas1994/metagenome_assembly?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/simonmoulds/uk-decadal-flood-prediction-lstm">simonmoulds/uk-decadal-flood-prediction-lstm</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/simonmoulds/uk-decadal-flood-prediction-lstm?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/knutdrand/bionumpy-benchmarks">knutdrand/bionumpy-benchmarks</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/knutdrand/bionumpy-benchmarks?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Cesconlab/ExomeSeq">Cesconlab/ExomeSeq</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Cesconlab/ExomeSeq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/trickovicmatija/assign-MAGs">trickovicmatija/assign-MAGs</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/trickovicmatija/assign-MAGs?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/vembrane/vembrane-benchmark">vembrane/vembrane-benchmark</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/vembrane/vembrane-benchmark?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/vembrane/vembrane-benchmark/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2022-08-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740928176">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/vembrane/vembrane-benchmark/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2022-07-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740945472">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/elsamah/ExomeSeq_wNorm">elsamah/ExomeSeq_wNorm</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/elsamah/ExomeSeq_wNorm?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Xavster838/get_and_annotate_locus">Xavster838/get_and_annotate_locus</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Xavster838/get_and_annotate_locus?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/SchlossLab/MiSeq-SOP-demo">SchlossLab/MiSeq-SOP-demo</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/SchlossLab/MiSeq-SOP-demo?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/KonstantinBurkin/cbai">KonstantinBurkin/cbai</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/KonstantinBurkin/cbai?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Albinam1/bioinformatics_school">Albinam1/bioinformatics_school</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Albinam1/bioinformatics_school?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Ovewh/Climaso">Ovewh/Climaso</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Ovewh/Climaso?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/shengwang/TransposonSnakemake">shengwang/TransposonSnakemake</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/shengwang/TransposonSnakemake?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Xavster838/coreDupGraph">Xavster838/coreDupGraph</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Xavster838/coreDupGraph?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/osvaldoreisss/rnaseq_analysis_kallisto_deseq2">osvaldoreisss/rnaseq_analysis_kallisto_deseq2</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/osvaldoreisss/rnaseq_analysis_kallisto_deseq2?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/dwils152/dePCRM2-snakemake">dwils152/dePCRM2-snakemake</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/dwils152/dePCRM2-snakemake?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/elsamah/ExomeSeq">elsamah/ExomeSeq</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/elsamah/ExomeSeq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/t-i-r/snakemake-workflow-template">t-i-r/snakemake-workflow-template</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/t-i-r/snakemake-workflow-template?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Xavster838/run_NucFreq">Xavster838/run_NucFreq</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Xavster838/run_NucFreq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/rmFlynn/DRAM_testing_pipline">rmFlynn/DRAM_testing_pipline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/rmFlynn/DRAM_testing_pipline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/radlinsky/snakemake_scrape_house_data">radlinsky/snakemake_scrape_house_data</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/radlinsky/snakemake_scrape_house_data?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/davidlorenz/snakemake-workflow-template">davidlorenz/snakemake-workflow-template</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/davidlorenz/snakemake-workflow-template?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/SilkeAllmannLab/pacbio_snakemake">SilkeAllmannLab/pacbio_snakemake</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/SilkeAllmannLab/pacbio_snakemake?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/SilkeAllmannLab/pacbio_snakemake/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2022-07-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741028592">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/gbucci/CNVkit">gbucci/CNVkit</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/gbucci/CNVkit?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/CBIbigA/hicstuff_pipeline">CBIbigA/hicstuff_pipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/CBIbigA/hicstuff_pipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/clarajegousse/hanoxy">clarajegousse/hanoxy</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/clarajegousse/hanoxy?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/hzi-bifo/snakemake-workflow-univec-removal">hzi-bifo/snakemake-workflow-univec-removal</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/hzi-bifo/snakemake-workflow-univec-removal?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/hzi-bifo/snakemake-workflow-univec-removal/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2022-02-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741144576">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/WestGermanGenomeCenter/circrna_detection">WestGermanGenomeCenter/circrna_detection</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/WestGermanGenomeCenter/circrna_detection?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/plycrsk/rna-seq-star-deseq2">plycrsk/rna-seq-star-deseq2</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/plycrsk/rna-seq-star-deseq2?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/plycrsk/rna-seq-star-deseq2/tree/v1.2.1">v1.2.1</a>
										</td>
										<td>2022-01-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741229328">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/thomasbtf/sars-cov-2-wbe">thomasbtf/sars-cov-2-wbe</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/thomasbtf/sars-cov-2-wbe?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/thomasbtf/sars-cov-2-wbe/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2021-12-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741281120">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/T3choH4ck3r/array">T3choH4ck3r/array</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/T3choH4ck3r/array?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/matinnuhamunada/genome_assembly_tryouts">matinnuhamunada/genome_assembly_tryouts</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/matinnuhamunada/genome_assembly_tryouts?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/rochevin/RNA-seq-from-SRR-single">rochevin/RNA-seq-from-SRR-single</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/rochevin/RNA-seq-from-SRR-single?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/wharvey31/megalodon-smk">wharvey31/megalodon-smk</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/wharvey31/megalodon-smk?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Xavster838/assembly_validation">Xavster838/assembly_validation</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Xavster838/assembly_validation?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/aryarm/admixtools">aryarm/admixtools</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/aryarm/admixtools?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/aryarm/admixtools/tree/v0.1.3">v0.1.3</a>
										</td>
										<td>2023-02-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741332144">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/aryarm/admixtools/tree/v0.1.2">v0.1.2</a>
										</td>
										<td>2023-02-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741465776">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/aryarm/admixtools/tree/v0.1.1">v0.1.1</a>
										</td>
										<td>2023-02-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741534096">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/aryarm/admixtools/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2023-02-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741585792">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/aryarm/admixtools/tree/v0.0.3">v0.0.3</a>
										</td>
										<td>2022-11-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010741621200">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/aryarm/admixtools/tree/v0.0.2">v0.0.2</a>
										</td>
										<td>2022-11-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739788832">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/aryarm/admixtools/tree/v0.0.1">v0.0.1</a>
										</td>
										<td>2022-11-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739693216">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/jma1991/rnaseq">jma1991/rnaseq</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/jma1991/rnaseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/lacklab/MuSTARRd">lacklab/MuSTARRd</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/lacklab/MuSTARRd?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Zilong-Li/lcWGS-imputation-workflow">Zilong-Li/lcWGS-imputation-workflow</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Zilong-Li/lcWGS-imputation-workflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/ETH-NEXUS/gExcite_pipeline">ETH-NEXUS/gExcite_pipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/ETH-NEXUS/gExcite_pipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/pauldrinn/MuSTARRd">pauldrinn/MuSTARRd</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/pauldrinn/MuSTARRd?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/Jain-Joyce-Labs/snakemake-chipseq">Jain-Joyce-Labs/snakemake-chipseq</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/Jain-Joyce-Labs/snakemake-chipseq?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/jjjoelle/learn_rna">jjjoelle/learn_rna</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/jjjoelle/learn_rna?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/cyrcular-calling">snakemake-workflows/cyrcular-calling</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/cyrcular-calling?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/cyrcular-calling/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2022-09-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739876144">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/SilasK/Snakebread">SilasK/Snakebread</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/SilasK/Snakebread?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/currocam/RawIlluminaPipeline">currocam/RawIlluminaPipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/currocam/RawIlluminaPipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/wharvey31/denovo_sv_validation">wharvey31/denovo_sv_validation</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/wharvey31/denovo_sv_validation?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/solida-core/dima">solida-core/dima</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/solida-core/dima?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/solida-core/dima/tree/v2.0.2">v2.0.2</a>
										</td>
										<td>2022-07-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739977136">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/dima/tree/v2.0.1">v2.0.1</a>
										</td>
										<td>2022-07-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740012592">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/dima/tree/v2.0">v2.0</a>
										</td>
										<td>2022-07-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740080768">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/dima/tree/v1.2">v1.2</a>
										</td>
										<td>2020-10-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740181712">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/dima/tree/v1.1">v1.1</a>
										</td>
										<td>2020-06-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740233504">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/solida-core/dima/tree/v1.0">v1.0</a>
										</td>
										<td>2020-02-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740285296">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/TLenfers/multispecies_mitochondrial_variant_analysis">TLenfers/multispecies_mitochondrial_variant_analysis</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/TLenfers/multispecies_mitochondrial_variant_analysis?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/TLenfers/multispecies_mitochondrial_variant_analysis/tree/v1.0.1">v1.0.1</a>
										</td>
										<td>2022-03-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740419072">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/TLenfers/multispecies_mitochondrial_variant_analysis/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2022-03-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740503680">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/mrvollger/Rhodonite">mrvollger/Rhodonite</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/mrvollger/Rhodonite?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/Rhodonite/tree/v0.12-alpha">v0.12-alpha</a>
										</td>
										<td>2022-02-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740539088">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/Rhodonite/tree/v0.7-alpha">v0.7-alpha</a>
										</td>
										<td>2022-01-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740639840">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/Rhodonite/tree/v0.6-alpha">v0.6-alpha</a>
										</td>
										<td>2021-11-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010740708016">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/Rhodonite/tree/v0.5-alpha">v0.5-alpha</a>
										</td>
										<td>2021-11-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738695424">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/Rhodonite/tree/v0.4-alpha">v0.4-alpha</a>
										</td>
										<td>2021-11-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738812816">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/Rhodonite/tree/v0.3-alpha">v0.3-alpha</a>
										</td>
										<td>2021-10-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738864560">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/Rhodonite/tree/v0.2-alpha">v0.2-alpha</a>
										</td>
										<td>2021-09-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738932688">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/Rhodonite/tree/v0.1-alpha">v0.1-alpha</a>
										</td>
										<td>2021-09-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739033728">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/mrvollger/sd-divergence">mrvollger/sd-divergence</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/mrvollger/sd-divergence?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover">IMS-Bio2Core-Facility/polya_liftover</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v2.0.1">v2.0.1</a>
										</td>
										<td>2022-01-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739085520">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v2.0.0">v2.0.0</a>
										</td>
										<td>2021-12-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739186464">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2021-12-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739287472">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v1.0.1">v1.0.1</a>
										</td>
										<td>2021-12-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739322880">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2021-12-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739440208">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v0.2.0">v0.2.0</a>
										</td>
										<td>2021-11-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739508384">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake">IMS-Bio2Core-Facility/GTExSnake</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake/tree/v1.2.0">v1.2.0</a>
										</td>
										<td>2021-10-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739592944">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake/tree/v1.1.2">v1.1.2</a>
										</td>
										<td>2021-07-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010739661120">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake/tree/v1.1.1">v1.1.1</a>
										</td>
										<td>2021-07-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010737664976">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2021-07-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010737749536">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2021-07-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010737817712">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing">IMS-Bio2Core-Facility/single_snake_sequencing</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing/tree/v0.1.2">v0.1.2</a>
										</td>
										<td>2022-01-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010737902272">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing/tree/v0.1.1">v0.1.1</a>
										</td>
										<td>2021-11-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010737937392">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2021-10-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738054720">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/koesterlab/ngs-fastq-to-cram">koesterlab/ngs-fastq-to-cram</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/koesterlab/ngs-fastq-to-cram?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/huzuner/Tumor-microbiome-calling">huzuner/Tumor-microbiome-calling</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/huzuner/Tumor-microbiome-calling?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/huzuner/Tumor-microbiome-calling/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2021-09-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738155728">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/jma1991/chromium">jma1991/chromium</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/jma1991/chromium?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling">nikostr/dna-seq-deepvariant-glnexus-variant-calling</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling/tree/v0.3.1">v0.3.1</a>
										</td>
										<td>2021-06-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738163456">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling/tree/v0.3.0">v0.3.0</a>
										</td>
										<td>2021-06-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738163696">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling/tree/v0.2.0">v0.2.0</a>
										</td>
										<td>2021-06-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738163936">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling/tree/v0.1.0-alpha">v0.1.0-alpha</a>
										</td>
										<td>2021-05-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738164176">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/dna-seq-mtb">snakemake-workflows/dna-seq-mtb</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/dna-seq-mtb?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.3.4">v1.3.4</a>
										</td>
										<td>2023-02-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738164416">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.3.3">v1.3.3</a>
										</td>
										<td>2023-01-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738164656">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.3.2">v1.3.2</a>
										</td>
										<td>2023-01-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738164896">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.3.1">v1.3.1</a>
										</td>
										<td>2022-11-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738165136">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.3.0">v1.3.0</a>
										</td>
										<td>2022-10-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738165376">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.2.0">v1.2.0</a>
										</td>
										<td>2022-06-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738165616">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2022-05-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738165856">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.0.1">v1.0.1</a>
										</td>
										<td>2022-05-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738166096">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2022-05-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738166336">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/fiberseq/fiberseq-smk">fiberseq/fiberseq-smk</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/fiberseq/fiberseq-smk?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.4.1">1.4.1</a>
										</td>
										<td>2023-01-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738166576">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.6">1.3.6</a>
										</td>
										<td>2022-12-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738166816">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.5">1.3.5</a>
										</td>
										<td>2022-11-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738167056">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.4">1.3.4</a>
										</td>
										<td>2022-11-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738167296">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.3">1.3.3</a>
										</td>
										<td>2022-11-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738167536">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.2">1.3.2</a>
										</td>
										<td>2022-11-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738167776">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.1">1.3.1</a>
										</td>
										<td>2022-10-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738168016">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.0">1.3.0</a>
										</td>
										<td>2022-10-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738168256">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.2.1">1.2.1</a>
										</td>
										<td>2022-08-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738168496">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.2.0">1.2.0</a>
										</td>
										<td>2022-07-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738168736">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.1">1.1</a>
										</td>
										<td>2022-07-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738168976">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.0">1.0</a>
										</td>
										<td>2022-07-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738169216">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/CVUA-RRW/FooDMe?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.6.3">1.6.3</a>
										</td>
										<td>2022-09-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738169456">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.6.2">1.6.2</a>
										</td>
										<td>2022-06-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738169696">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.6.1">1.6.1</a>
										</td>
										<td>2022-06-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738169936">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.6.0">1.6.0</a>
										</td>
										<td>2022-06-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738170176">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.5.1">1.5.1</a>
										</td>
										<td>2022-06-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738170416">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.5.0">1.5.0</a>
										</td>
										<td>2022-06-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738170656">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.9">1.4.9</a>
										</td>
										<td>2022-05-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738170896">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.8">1.4.8</a>
										</td>
										<td>2022-05-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738171136">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.7">1.4.7</a>
										</td>
										<td>2022-05-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738171376">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.6">1.4.6</a>
										</td>
										<td>2022-05-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738171616">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.5">1.4.5</a>
										</td>
										<td>2022-04-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738171856">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.4">1.4.4</a>
										</td>
										<td>2022-04-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738204928">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.2">1.4.2</a>
										</td>
										<td>2021-12-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738205168">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.1">1.4.1</a>
										</td>
										<td>2021-12-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738205408">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.0">1.4.0</a>
										</td>
										<td>2021-11-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738205648">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.3.5">1.3.5</a>
										</td>
										<td>2021-07-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738205888">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.3.4">1.3.4</a>
										</td>
										<td>2021-01-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738206128">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/epigen/mixscape_seurat">epigen/mixscape_seurat</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/epigen/mixscape_seurat?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/mrvollger/fastCN-smk">mrvollger/fastCN-smk</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/mrvollger/fastCN-smk?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/fastCN-smk/tree/v0.1-alpha">v0.1-alpha</a>
										</td>
										<td>2021-09-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738206368">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/benchmark-giab?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.6.4">v1.6.4</a>
										</td>
										<td>2023-02-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738206608">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.6.3">v1.6.3</a>
										</td>
										<td>2023-02-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738206848">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.6.2">v1.6.2</a>
										</td>
										<td>2023-01-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738207088">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.6.1">v1.6.1</a>
										</td>
										<td>2023-01-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738207328">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.6.0">v1.6.0</a>
										</td>
										<td>2023-01-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738207568">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.5.1">v1.5.1</a>
										</td>
										<td>2022-11-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738207808">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.5.0">v1.5.0</a>
										</td>
										<td>2022-10-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738208048">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.4.1">v1.4.1</a>
										</td>
										<td>2022-08-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738208288">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.4.0">v1.4.0</a>
										</td>
										<td>2022-08-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738208528">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.6">v1.3.6</a>
										</td>
										<td>2022-07-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738208768">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.5">v1.3.5</a>
										</td>
										<td>2022-06-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738209008">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.4">v1.3.4</a>
										</td>
										<td>2022-05-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738209248">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.3">v1.3.3</a>
										</td>
										<td>2022-05-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738209488">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.2">v1.3.2</a>
										</td>
										<td>2022-05-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738209728">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.1">v1.3.1</a>
										</td>
										<td>2022-05-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738209968">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.0">v1.3.0</a>
										</td>
										<td>2022-05-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738210208">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.2.0">v1.2.0</a>
										</td>
										<td>2022-03-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738210448">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2022-02-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738210688">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.0.1">v1.0.1</a>
										</td>
										<td>2021-12-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738210928">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2021-12-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738211168">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/species-quantification">snakemake-workflows/species-quantification</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/species-quantification?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/species-quantification/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2021-09-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738211408">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/dna-seq-benchmark?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.6.4">v1.6.4</a>
										</td>
										<td>2023-02-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738211648">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.6.3">v1.6.3</a>
										</td>
										<td>2023-02-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738211888">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.6.2">v1.6.2</a>
										</td>
										<td>2023-01-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738212128">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.6.1">v1.6.1</a>
										</td>
										<td>2023-01-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738212368">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.6.0">v1.6.0</a>
										</td>
										<td>2023-01-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738212608">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.5.1">v1.5.1</a>
										</td>
										<td>2022-11-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738212848">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.5.0">v1.5.0</a>
										</td>
										<td>2022-10-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738213088">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.4.1">v1.4.1</a>
										</td>
										<td>2022-08-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738213328">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.4.0">v1.4.0</a>
										</td>
										<td>2022-08-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738213568">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.6">v1.3.6</a>
										</td>
										<td>2022-07-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738213808">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.5">v1.3.5</a>
										</td>
										<td>2022-06-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738214048">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.4">v1.3.4</a>
										</td>
										<td>2022-05-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738214288">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.3">v1.3.3</a>
										</td>
										<td>2022-05-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738214528">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.2">v1.3.2</a>
										</td>
										<td>2022-05-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738214768">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.1">v1.3.1</a>
										</td>
										<td>2022-05-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738215008">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.0">v1.3.0</a>
										</td>
										<td>2022-05-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738215248">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.2.0">v1.2.0</a>
										</td>
										<td>2022-03-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738215488">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2022-02-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738215728">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.0.1">v1.0.1</a>
										</td>
										<td>2021-12-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738215968">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2021-12-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738216208">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/tahiri-lab/aPhyloGeo-pipeline">tahiri-lab/aPhyloGeo-pipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/tahiri-lab/aPhyloGeo-pipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/tahiri-lab/aPhyloGeo-pipeline/tree/v1.0">v1.0</a>
										</td>
										<td>2023-01-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738216448">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/dbespiatykh/RDscan">dbespiatykh/RDscan</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/dbespiatykh/RDscan?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/dbespiatykh/RDscan/tree/v1.0.3r">v1.0.3r</a>
										</td>
										<td>2023-01-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738216688">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/dbespiatykh/RDscan/tree/v1.0.1">v1.0.1</a>
										</td>
										<td>2023-01-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738216928">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/dbespiatykh/RDscan/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2022-08-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738217168">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/clami66/AF_server">clami66/AF_server</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/clami66/AF_server?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/SchlossLab/mothur-snakemake-workflow">SchlossLab/mothur-snakemake-workflow</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/SchlossLab/mothur-snakemake-workflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/epigen/dea_limma">epigen/dea_limma</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/epigen/dea_limma?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/epigen/dea_limma/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2023-01-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738217408">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/epigen/genome_tracks">epigen/genome_tracks</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/epigen/genome_tracks?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/epigen/genome_tracks/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2023-01-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738217648">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/nismod/open-gira">nismod/open-gira</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/nismod/open-gira?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/GooglingTheCancerGenome/sv-gen">GooglingTheCancerGenome/sv-gen</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/GooglingTheCancerGenome/sv-gen?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/GooglingTheCancerGenome/sv-gen/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2023-01-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738217888">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/GooglingTheCancerGenome/sv-gen/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2020-03-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738218128">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/epigen/dea_seurat">epigen/dea_seurat</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/epigen/dea_seurat?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/epigen/scrnaseq_processing_seurat">epigen/scrnaseq_processing_seurat</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/epigen/scrnaseq_processing_seurat?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/sreichl/atacseq_pipeline">sreichl/atacseq_pipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/sreichl/atacseq_pipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/sreichl/atacseq_pipeline/tree/v0.1.1">v0.1.1</a>
										</td>
										<td>2022-03-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738218368">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/sreichl/atacseq_pipeline/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2022-03-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738218608">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/akcorut/kGWASflow">akcorut/kGWASflow</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/akcorut/kGWASflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/akcorut/kGWASflow/tree/v.1.1.0">v.1.1.0</a>
										</td>
										<td>2023-01-26</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738218848">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/akcorut/kGWASflow/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2022-11-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738219088">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/epigen/enrichment_analysis">epigen/enrichment_analysis</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/epigen/enrichment_analysis?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/epigen/enrichment_analysis/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2023-01-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738219328">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/epigen/unsupervised_analysis">epigen/unsupervised_analysis</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/epigen/unsupervised_analysis?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/epigen/unsupervised_analysis/tree/v0.2.0">v0.2.0</a>
										</td>
										<td>2022-10-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738219568">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/epigen/unsupervised_analysis/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2022-09-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738219808">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow">SchlossLab/mikropml-snakemake-workflow</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow/tree/v1.2.0">v1.2.0</a>
										</td>
										<td>2023-01-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738220048">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2022-11-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738220288">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2021-04-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738220528">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.9.1">1.9.1</a>
										</td>
										<td>2023-01-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738220768">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.9.0">1.9.0</a>
										</td>
										<td>2023-01-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738221008">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.8.5">1.8.5</a>
										</td>
										<td>2022-11-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738254080">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.8.4">1.8.4</a>
										</td>
										<td>2022-11-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738254320">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.8.3">1.8.3</a>
										</td>
										<td>2022-11-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738254560">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.8.2">1.8.2</a>
										</td>
										<td>2022-10-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738254800">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.8.1">1.8.1</a>
										</td>
										<td>2022-10-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738255040">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.7.0">1.7.0</a>
										</td>
										<td>2022-09-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738255280">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.6.0">1.6.0</a>
										</td>
										<td>2022-07-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738255520">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.5.1-dev">1.5.1-dev</a>
										</td>
										<td>2022-06-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738255760">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.5.1">1.5.1</a>
										</td>
										<td>2022-07-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738256000">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.3">1.3</a>
										</td>
										<td>2022-06-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738256240">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.0">1.0</a>
										</td>
										<td>2020-09-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738256480">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/IKIM-Essen/uncovar?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.15.1">v0.15.1</a>
										</td>
										<td>2022-12-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738256720">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.15.0">v0.15.0</a>
										</td>
										<td>2022-03-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738256960">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.14.0">v0.14.0</a>
										</td>
										<td>2022-03-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738257200">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.13.2">v0.13.2</a>
										</td>
										<td>2022-01-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738257440">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.13.1">v0.13.1</a>
										</td>
										<td>2022-01-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738257680">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.13.0">v0.13.0</a>
										</td>
										<td>2022-01-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738257920">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.12.0">v0.12.0</a>
										</td>
										<td>2021-12-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738258160">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.11.0">v0.11.0</a>
										</td>
										<td>2021-12-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738258400">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.10.2">v0.10.2</a>
										</td>
										<td>2021-12-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738258640">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.10.1">v0.10.1</a>
										</td>
										<td>2021-12-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738258880">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.10.0">v0.10.0</a>
										</td>
										<td>2021-12-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738259120">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.9.0">v0.9.0</a>
										</td>
										<td>2021-12-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738259360">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.8.0">v0.8.0</a>
										</td>
										<td>2021-12-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738259600">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.7.0">v0.7.0</a>
										</td>
										<td>2021-12-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738259840">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.6.0">v0.6.0</a>
										</td>
										<td>2021-12-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738260080">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.5.0">v0.5.0</a>
										</td>
										<td>2021-12-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738260320">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.4.1">v0.4.1</a>
										</td>
										<td>2021-12-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738260560">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.4.0">v0.4.0</a>
										</td>
										<td>2021-12-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738260800">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.3.1">v0.3.1</a>
										</td>
										<td>2021-11-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738261040">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.3.0">v0.3.0</a>
										</td>
										<td>2021-11-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738261280">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.5">v0.2.5</a>
										</td>
										<td>2021-09-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738261520">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.4">v0.2.4</a>
										</td>
										<td>2021-09-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738261760">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.3">v0.2.3</a>
										</td>
										<td>2021-09-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738262000">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.2">v0.2.2</a>
										</td>
										<td>2021-09-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738262240">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.1">v0.2.1</a>
										</td>
										<td>2021-09-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738262480">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.0">v0.2.0</a>
										</td>
										<td>2021-09-22</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738262720">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.8">v0.1.8</a>
										</td>
										<td>2021-09-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738262960">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.7">v0.1.7</a>
										</td>
										<td>2021-09-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738263200">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.6">v0.1.6</a>
										</td>
										<td>2021-09-16</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738263440">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.5">v0.1.5</a>
										</td>
										<td>2021-09-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738263680">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.4">v0.1.4</a>
										</td>
										<td>2021-09-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738263920">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.3">v0.1.3</a>
										</td>
										<td>2021-09-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738264160">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.2">v0.1.2</a>
										</td>
										<td>2021-09-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738264400">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.1">v0.1.1</a>
										</td>
										<td>2021-09-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738264640">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2021-09-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738264880">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/epigen/atacseq_pipeline">epigen/atacseq_pipeline</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/epigen/atacseq_pipeline?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/epigen/atacseq_pipeline/tree/v0.1.1">v0.1.1</a>
										</td>
										<td>2022-03-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738265120">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/epigen/atacseq_pipeline/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2022-03-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738265360">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/mrvollger/SmkTemplate">mrvollger/SmkTemplate</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/mrvollger/SmkTemplate?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/sneuensc/mapache">sneuensc/mapache</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/sneuensc/mapache?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/sneuensc/mapache/tree/v0.3.0">v0.3.0</a>
										</td>
										<td>2022-12-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738265600">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/sneuensc/mapache/tree/v0.2.0">v0.2.0</a>
										</td>
										<td>2022-09-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738265840">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/sneuensc/mapache/tree/v0.1.0">v0.1.0</a>
										</td>
										<td>2021-11-05</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738266080">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/snakemake-workflow-template">snakemake-workflows/snakemake-workflow-template</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/snakemake-workflow-template?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<tbody>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/zavolanlab/zarp">zavolanlab/zarp</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/zavolanlab/zarp?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/zavolanlab/zarp/tree/v0.4.1">v0.4.1</a>
										</td>
										<td>2022-12-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738266320">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/zavolanlab/zarp/tree/v0.4.0">v0.4.0</a>
										</td>
										<td>2022-11-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738266560">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/mrvollger/StainedGlass">mrvollger/StainedGlass</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/mrvollger/StainedGlass?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/StainedGlass/tree/v0.5">v0.5</a>
										</td>
										<td>2022-08-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738266800">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/StainedGlass/tree/v0.4">v0.4</a>
										</td>
										<td>2021-12-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738267040">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/StainedGlass/tree/v0.3">v0.3</a>
										</td>
										<td>2021-11-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738267280">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/StainedGlass/tree/v0.2">v0.2</a>
										</td>
										<td>2021-11-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738267520">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/mrvollger/StainedGlass/tree/v0.1">v0.1</a>
										</td>
										<td>2021-08-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738267760">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/GooglingTheCancerGenome/sv-callers">GooglingTheCancerGenome/sv-callers</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/GooglingTheCancerGenome/sv-callers?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.2.2">v1.2.2</a>
										</td>
										<td>2022-11-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738268000">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.2.1">v1.2.1</a>
										</td>
										<td>2022-09-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738268240">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.2.0">v1.2.0</a>
										</td>
										<td>2022-06-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738268480">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.1.3">v1.1.3</a>
										</td>
										<td>2022-02-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738268720">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.1.2">v1.1.2</a>
										</td>
										<td>2020-10-26</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738268960">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.1.1">v1.1.1</a>
										</td>
										<td>2020-02-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738269200">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2019-05-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738269440">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2018-04-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738269680">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.4.3">v2.4.3</a>
										</td>
										<td>2023-02-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738269920">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.4.2">v2.4.2</a>
										</td>
										<td>2022-12-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738270160">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.4.1">v2.4.1</a>
										</td>
										<td>2022-11-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738286848">here</a>
														</td>
														<td>2023-02-20</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.4.0">v2.4.0</a>
										</td>
										<td>2022-03-29</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738287088">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.3.2">v2.3.2</a>
										</td>
										<td>2021-12-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738287328">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.3.1">v2.3.1</a>
										</td>
										<td>2021-11-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738287568">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.3.0">v2.3.0</a>
										</td>
										<td>2021-10-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738287808">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.2.1">v2.2.1</a>
										</td>
										<td>2021-08-31</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738288048">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.2.0">v2.2.0</a>
										</td>
										<td>2021-08-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738288288">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.1.1">v2.1.1</a>
										</td>
										<td>2021-05-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738288528">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.1.0">v2.1.0</a>
										</td>
										<td>2021-05-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738288768">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.0.4">v2.0.4</a>
										</td>
										<td>2021-05-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738289008">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.0.3">v2.0.3</a>
										</td>
										<td>2021-05-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738289248">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.0.2">v2.0.2</a>
										</td>
										<td>2021-04-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738289488">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.0.1">v2.0.1</a>
										</td>
										<td>2021-01-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738289728">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.0.0">v2.0.0</a>
										</td>
										<td>2019-10-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738289968">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2019-04-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738290208">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2019-04-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738290448">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.20.1">v3.20.1</a>
										</td>
										<td>2023-02-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738290688">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.20.0">v3.20.0</a>
										</td>
										<td>2023-02-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738290928">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.6">v3.19.6</a>
										</td>
										<td>2023-02-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738291168">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.5">v3.19.5</a>
										</td>
										<td>2023-01-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738291408">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.4">v3.19.4</a>
										</td>
										<td>2023-01-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738291648">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.3">v3.19.3</a>
										</td>
										<td>2023-01-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738291888">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.2">v3.19.2</a>
										</td>
										<td>2023-01-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738292128">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.1">v3.19.1</a>
										</td>
										<td>2022-12-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738292368">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.0">v3.19.0</a>
										</td>
										<td>2022-12-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738292608">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.7">v3.18.7</a>
										</td>
										<td>2022-11-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738292848">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.6">v3.18.6</a>
										</td>
										<td>2022-11-15</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738293088">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.5">v3.18.5</a>
										</td>
										<td>2022-11-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738293328">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.4">v3.18.4</a>
										</td>
										<td>2022-11-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738293568">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.3">v3.18.3</a>
										</td>
										<td>2022-11-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738293808">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.2">v3.18.2</a>
										</td>
										<td>2022-10-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738294048">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.1">v3.18.1</a>
										</td>
										<td>2022-09-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738294288">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.0">v3.18.0</a>
										</td>
										<td>2022-09-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738294528">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.17.1">v3.17.1</a>
										</td>
										<td>2022-09-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738294768">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.17.0">v3.17.0</a>
										</td>
										<td>2022-09-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738295008">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.16.0">v3.16.0</a>
										</td>
										<td>2022-09-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738295248">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.15.0">v3.15.0</a>
										</td>
										<td>2022-08-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738295488">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.14.1">v3.14.1</a>
										</td>
										<td>2022-07-25</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738295728">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.14.0">v3.14.0</a>
										</td>
										<td>2022-05-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738295968">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.13.1">v3.13.1</a>
										</td>
										<td>2022-05-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738296208">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.13.0">v3.13.0</a>
										</td>
										<td>2022-05-12</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738296448">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.12.0">v3.12.0</a>
										</td>
										<td>2022-05-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738296688">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.11.0">v3.11.0</a>
										</td>
										<td>2022-05-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738296928">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.10.2">v3.10.2</a>
										</td>
										<td>2022-05-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738297168">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.10.1">v3.10.1</a>
										</td>
										<td>2022-04-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738297408">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.10.0">v3.10.0</a>
										</td>
										<td>2022-04-26</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738297648">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.9.0">v3.9.0</a>
										</td>
										<td>2022-04-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738297888">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.8.1">v3.8.1</a>
										</td>
										<td>2022-03-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738298128">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.8.0">v3.8.0</a>
										</td>
										<td>2022-03-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738298368">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.7.1">v3.7.1</a>
										</td>
										<td>2022-02-23</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738298608">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.7.0">v3.7.0</a>
										</td>
										<td>2022-02-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738298848">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.6.3">v3.6.3</a>
										</td>
										<td>2022-02-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738299088">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.6.2">v3.6.2</a>
										</td>
										<td>2022-01-20</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738299328">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.6.1">v3.6.1</a>
										</td>
										<td>2022-01-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738299568">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.6.0">v3.6.0</a>
										</td>
										<td>2022-01-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738299808">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.5.0">v3.5.0</a>
										</td>
										<td>2022-01-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738300048">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.4.0">v3.4.0</a>
										</td>
										<td>2021-12-17</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738300288">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.3.0">v3.3.0</a>
										</td>
										<td>2021-11-18</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738300528">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.2.0">v3.2.0</a>
										</td>
										<td>2021-11-11</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738300768">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.1.0">v3.1.0</a>
										</td>
										<td>2021-11-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738301008">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.0.0">v3.0.0</a>
										</td>
										<td>2021-09-03</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738301248">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.6">v2.0.6</a>
										</td>
										<td>2021-07-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738301488">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.5">v2.0.5</a>
										</td>
										<td>2021-07-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738301728">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.4">v2.0.4</a>
										</td>
										<td>2021-07-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738301968">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.3">v2.0.3</a>
										</td>
										<td>2021-07-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738302208">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.2">v2.0.2</a>
										</td>
										<td>2021-07-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738302448">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.1">v2.0.1</a>
										</td>
										<td>2021-07-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738302688">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.0">v2.0.0</a>
										</td>
										<td>2021-07-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738302928">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.5.0">v1.5.0</a>
										</td>
										<td>2021-07-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738319616">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.4.0">v1.4.0</a>
										</td>
										<td>2021-06-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738319856">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.3.2">v1.3.2</a>
										</td>
										<td>2021-05-28</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738320096">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.3.1">v1.3.1</a>
										</td>
										<td>2021-05-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738320336">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.3.0">v1.3.0</a>
										</td>
										<td>2021-05-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738320576">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.2.0">v1.2.0</a>
										</td>
										<td>2021-05-19</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738320816">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2021-05-07</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738321056">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.0.1">v1.0.1</a>
										</td>
										<td>2021-04-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738321296">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2021-04-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738321536">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/franciscozorrilla/metaGEM">franciscozorrilla/metaGEM</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/franciscozorrilla/metaGEM?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/franciscozorrilla/metaGEM/tree/v1.0.3">v1.0.3</a>
										</td>
										<td>2023-01-27</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738321776">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/franciscozorrilla/metaGEM/tree/v1.0.2">v1.0.2</a>
										</td>
										<td>2022-11-10</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738322016">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/franciscozorrilla/metaGEM/tree/v1.0.1">v1.0.1</a>
										</td>
										<td>2021-04-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738322256">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/franciscozorrilla/metaGEM/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2021-04-21</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738322496">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling">snakemake-workflows/dna-seq-gatk-variant-calling</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.1.1">v2.1.1</a>
										</td>
										<td>2021-05-02</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738322736">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.1.0">v2.1.0</a>
										</td>
										<td>2021-04-30</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738322976">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.0.3">v2.0.3</a>
										</td>
										<td>2021-04-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738323216">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.0.2">v2.0.2</a>
										</td>
										<td>2021-04-09</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738323456">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.0.1">v2.0.1</a>
										</td>
										<td>2021-01-14</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738323696">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.0.0">v2.0.0</a>
										</td>
										<td>2020-06-01</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738323936">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v1.0.1">v1.0.1</a>
										</td>
										<td>2019-02-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738324176">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2019-02-08</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738324416">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
				<tr>
					<td>
						<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2">snakemake-workflows/rna-seq-star-deseq2</a>
					</td>
					<td>snakemake</td>
					<td>
						<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2?revisions=releases&skip-drafts=1&skip-preleases=1">repo</a>
					</td>
					<td>
						<span></span>
					</td>
					<td>
						<details>
							<summary>Revisions</summary>
							<table>
								<thead>
									<tr>
										<td>Revision</td>
										<td>Date/time</td>
										<td>Executions</td>
									</tr>
								</thead>
								<tbody>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.2.0">v1.2.0</a>
										</td>
										<td>2021-08-24</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738324656">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.1.2">v1.1.2</a>
										</td>
										<td>2021-05-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738324896">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.1.1">v1.1.1</a>
										</td>
										<td>2021-05-06</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738325136">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.1.0">v1.1.0</a>
										</td>
										<td>2021-05-04</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738325376">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
									<tr>
										<td>
											<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.0.0">v1.0.0</a>
										</td>
										<td>2019-03-13</td>
										<td>
											<table>
												<thead>
													<tr>
														<td>Link</td>
														<td>Date/time</td>
														<td>Success</td>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>
															<a href="#140010738325616">here</a>
														</td>
														<td>2023-02-19</td>
														<td>
															<span></span>
														</td>
													</tr>
												</tbody>
											</table>
										</td>
									</tr>
								</tbody>
							</table>
						</details>
					</td>
				</tr>
			</tbody>
		</table>
		<h2 id="executions">
			<a href="#executions">Executions</a>
		</h2>
		<div>
			<span>
				<button onclick="Array.from(this.parentElement.parentElement.getElementsByTagName('details')).forEach(elem => {elem.open = true; });">Expand all</button>
				<button onclick="Array.from(this.parentElement.parentElement.getElementsByTagName('details')).forEach(elem => {elem.open = false;});">Collapse all</button>
			</span>
			<table>
				<thead>
					<tr>
						<td>Workflow</td>
						<td>Revision</td>
						<td>Engine</td>
						<td>Staleness</td>
						<td>Execution date</td>
						<td>Success</td>
						<td>Logs</td>
						<td>Outputs</td>
						<td>Max RAM</td>
						<td>CPU Time</td>
						<td>Wall Time</td>
					</tr>
				</thead>
				<tbody>
					<tr>
						<td>
							<span id="140010738269680">
								<a href="https://github.com/GooglingTheCancerGenome/sv-callers">GooglingTheCancerGenome/sv-callers</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>1774 days</td>
						<td>2023-02-19 03:33:12</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2Fzai45es0%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2Fzai45es0%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743140352">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/1.0">1.0</a>
						</td>
						<td>nextflow</td>
						<td>1769 days</td>
						<td>2023-02-19 04:41:45</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: nextflow.exception.AbortOperationException
msg: 'Unknown configuration profile: ''singularity'''
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.config.ConfigBuilder.checkValidProfile(ConfigBuilder.groovy:511)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.config.ConfigBuilder.buildConfig0(ConfigBuilder.groovy:385)\n\tat\
  \ java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat nextflow.config.ConfigBuilder.buildGivenFiles(ConfigBuilder.groovy:317)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.config.ConfigBuilder.buildConfigObject(ConfigBuilder.groovy:780)\n\
  \tat nextflow.config.ConfigBuilder.build(ConfigBuilder.groovy:793)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:307)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Ff15rzt2j%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Ff15rzt2j%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.262GiB</td>
						<td>12.0 seconds</td>
						<td>8.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743116944">
								<a href="https://nf-co.re/hlatyping">hlatyping</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hlatyping/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1677 days</td>
						<td>2023-02-19 06:24:03</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fqhljhhvo%2Flogs.tar.xz">0.037GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fqhljhhvo%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.245GiB</td>
						<td>425.0 seconds</td>
						<td>389.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743140112">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/1.1">1.1</a>
						</td>
						<td>nextflow</td>
						<td>1654 days</td>
						<td>2023-02-19 04:32:39</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fee2d8s2q%2Flogs.tar.xz">0.030GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fee2d8s2q%2Foutput.tar.xz">0.005GiB</a>
						</td>
						<td>1.568GiB</td>
						<td>582.8 seconds</td>
						<td>470.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743116704">
								<a href="https://nf-co.re/hlatyping">hlatyping</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hlatyping/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>1649 days</td>
						<td>2023-02-19 07:00:43</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2F3p3bitw3%2Flogs.tar.xz">0.037GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2F3p3bitw3%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.631GiB</td>
						<td>48.2 seconds</td>
						<td>37.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743188304">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/1.0">1.0</a>
						</td>
						<td>nextflow</td>
						<td>1644 days</td>
						<td>2023-02-19 06:07:09</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F2rj3slrh%2Flogs.tar.xz">0.023GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F2rj3slrh%2Foutput.tar.xz">0.004GiB</a>
						</td>
						<td>2.078GiB</td>
						<td>925.5 seconds</td>
						<td>672.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743116464">
								<a href="https://nf-co.re/hlatyping">hlatyping</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hlatyping/tree/1.1.1">1.1.1</a>
						</td>
						<td>nextflow</td>
						<td>1643 days</td>
						<td>2023-02-19 06:40:20</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fcgxdwm1w%2Flogs.tar.xz">0.038GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fcgxdwm1w%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.385GiB</td>
						<td>481.3 seconds</td>
						<td>238.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743188064">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/1.1">1.1</a>
						</td>
						<td>nextflow</td>
						<td>1597 days</td>
						<td>2023-02-19 04:39:42</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fwlsbjkh4%2Flogs.tar.xz">0.023GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fwlsbjkh4%2Foutput.tar.xz">0.004GiB</a>
						</td>
						<td>2.134GiB</td>
						<td>920.5 seconds</td>
						<td>787.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743109024">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1585 days</td>
						<td>2023-02-19 03:31:07</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 03:31:28.288
kind: NextflowMiscError
process: main
rest: Please specify either --singleEnd or --pairedEnd to execute the pipeline!

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fkkap72hv%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fkkap72hv%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.510GiB</td>
						<td>21.9 seconds</td>
						<td>21.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743108784">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.0.1">2.0.1</a>
						</td>
						<td>nextflow</td>
						<td>1570 days</td>
						<td>2023-02-19 03:32:19</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 03:32:34.609
kind: NextflowMiscError
process: main
rest: Please specify either --singleEnd or --pairedEnd to execute the pipeline!

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fkyyrd1al%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fkyyrd1al%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.483GiB</td>
						<td>19.9 seconds</td>
						<td>14.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743108544">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.0.2">2.0.2</a>
						</td>
						<td>nextflow</td>
						<td>1569 days</td>
						<td>2023-02-19 03:31:05</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 03:31:27.775
kind: NextflowMiscError
process: main
rest: Please specify either --singleEnd or --pairedEnd to execute the pipeline!

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fnevsvak9%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fnevsvak9%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.453GiB</td>
						<td>20.8 seconds</td>
						<td>22.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743050928">
								<a href="https://nf-co.re/deepvariant">deepvariant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/deepvariant/tree/1.0">1.0</a>
						</td>
						<td>nextflow</td>
						<td>1553 days</td>
						<td>2023-02-19 04:24:14</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/deepvariant%2Fipcpcp1p%2Flogs.tar.xz">0.023GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/deepvariant%2Fipcpcp1p%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.632GiB</td>
						<td>573.3 seconds</td>
						<td>354.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743145152">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1545 days</td>
						<td>2023-02-19 04:31:21</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.io.IOException
msg: 'Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta'
rest: "at nextflow.file.http.XFileSystemProvider.readAttributes(XFileSystemProvider.groovy:463)\n\
  \tat java.base/java.nio.file.Files.readAttributes(Files.java:1764)\n\tat nextflow.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:156)\n\
  \tat nextflow.file.FileHelper.copyPath(FileHelper.groovy:971)\n\tat nextflow.file.FilePorter$FileTransfer.stageForeignFile0(FilePorter.groovy:299)\n\
  \tat nextflow.file.FilePorter$FileTransfer.stageForeignFile(FilePorter.groovy:266)\n\
  \tat nextflow.file.FilePorter$FileTransfer.run(FilePorter.groovy:252)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\
  \tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-19 04:31:40.454 [FileTransfer-1]\
  \ DEBUG nextflow.file.FilePorter - Copying foreign file https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\
  \ to work dir: /home/azureuser/tmp/ny7j3hbnpt/code/work/stage/e9/6e1520aa7757a645e2ccc565bb62f7/HUMAN_2014_concat.fasta\n\
  Feb-19 04:31:40.472 [FileTransfer-1] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 250ms (attempt=1)\nFeb-19 04:31:40.739 [FileTransfer-1]\
  \ DEBUG n.file.http.XFileSystemProvider - Got HTTP error=404 waiting for 750ms (attempt=2)\n\
  Feb-19 04:31:41.510 [FileTransfer-1] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 2250ms (attempt=3)\nFeb-19 04:31:43.779 [FileTransfer-1]\
  \ WARN  nextflow.file.FilePorter - Unable to stage foreign file: https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\
  \ (try 2) -- Cause: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\n\
  java.io.IOException: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\n\
  \tat nextflow.file.http.XFileSystemProvider.readAttributes(XFileSystemProvider.groovy:463)\n\
  \tat java.base/java.nio.file.Files.readAttributes(Files.java:1764)\n\tat nextflow.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:156)\n\
  \tat nextflow.file.FileHelper.copyPath(FileHelper.groovy:971)\n\tat nextflow.file.FilePorter$FileTransfer.stageForeignFile0(FilePorter.groovy:299)\n\
  \tat nextflow.file.FilePorter$FileTransfer.stageForeignFile(FilePorter.groovy:266)\n\
  \tat nextflow.file.FilePorter$FileTransfer.run(FilePorter.groovy:252)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\
  \tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-19 04:31:43.995 [FileTransfer-1]\
  \ DEBUG nextflow.file.FilePorter - Copying foreign file https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\
  \ to work dir: /home/azureuser/tmp/ny7j3hbnpt/code/work/stage/e9/6e1520aa7757a645e2ccc565bb62f7/HUMAN_2014_concat.fasta\n\
  Feb-19 04:31:44.009 [FileTransfer-1] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 250ms (attempt=1)\nFeb-19 04:31:44.276 [FileTransfer-1]\
  \ DEBUG n.file.http.XFileSystemProvider - Got HTTP error=404 waiting for 750ms (attempt=2)\n\
  Feb-19 04:31:45.044 [FileTransfer-1] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 2250ms (attempt=3)\nFeb-19 04:31:47.321 [FileTransfer-1]\
  \ WARN  nextflow.file.FilePorter - Unable to stage foreign file: https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\
  \ (try 3) -- Cause: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\n\
  java.io.IOException: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\n\
  \tat nextflow.file.http.XFileSystemProvider.readAttributes(XFileSystemProvider.groovy:463)\n\
  \tat java.base/java.nio.file.Files.readAttributes(Files.java:1764)\n\tat nextflow.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:156)\n\
  \tat nextflow.file.FileHelper.copyPath(FileHelper.groovy:971)\n\tat nextflow.file.FilePorter$FileTransfer.stageForeignFile0(FilePorter.groovy:299)\n\
  \tat nextflow.file.FilePorter$FileTransfer.stageForeignFile(FilePorter.groovy:266)\n\
  \tat nextflow.file.FilePorter$FileTransfer.run(FilePorter.groovy:252)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\
  \tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-19 04:31:47.420 [FileTransfer-1]\
  \ DEBUG nextflow.file.FilePorter - Copying foreign file https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\
  \ to work dir: /home/azureuser/tmp/ny7j3hbnpt/code/work/stage/e9/6e1520aa7757a645e2ccc565bb62f7/HUMAN_2014_concat.fasta\n\
  Feb-19 04:31:47.436 [FileTransfer-1] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 250ms (attempt=1)\nFeb-19 04:31:47.708 [FileTransfer-1]\
  \ DEBUG n.file.http.XFileSystemProvider - Got HTTP error=404 waiting for 750ms (attempt=2)\n\
  Feb-19 04:31:48.476 [FileTransfer-1] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 2250ms (attempt=3)\nFeb-19 04:31:50.766 [Actor Thread\
  \ 4] ERROR nextflow.processor.TaskProcessor - Error executing process > 'generate_decoy_database\
  \ (1)'\n\nCaused by:\n  Can't stage file https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\
  \ -- reason: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/HUMAN_2014_concat.fasta\n\
  \n\nTip: you can try to figure out what's wrong by changing to the process work\
  \ dir and showing the script file named `.command.sh`\nFeb-19 04:31:50.772 [Actor\
  \ Thread 4] INFO  nextflow.Session - Execution cancelled -- Finishing pending tasks\
  \ before exit\nFeb-19 04:31:50.785 [main] DEBUG nextflow.Session - Session await\
  \ > all processes finished\nFeb-19 04:31:50.792 [main] DEBUG nextflow.Session -\
  \ Session await > all barriers passed\nFeb-19 04:31:50.923 [main] INFO  nextflow.Nextflow\
  \ - [nf-core/mhcquant] Pipeline Complete\nFeb-19 04:31:50.927 [main] DEBUG nextflow.trace.WorkflowStatsObserver\
  \ - Workflow completed > WorkflowStats[succeededCount=0; failedCount=0; ignoredCount=0;\
  \ cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0;\
  \ abortedCount=0; succeedDuration=0ms; failedDuration=0ms; cachedDuration=0ms;loadCpus=0;\
  \ loadMemory=0; peakRunning=0; peakCpus=0; peakMemory=0; ]\nFeb-19 04:31:50.927\
  \ [main] DEBUG nextflow.trace.TraceFileObserver - Workflow completed -- saving trace\
  \ file\nFeb-19 04:31:50.939 [main] DEBUG nextflow.trace.ReportObserver - Workflow\
  \ completed -- rendering execution report\nFeb-19 04:31:50.943 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Execution report summary data:\n  []\nFeb-19 04:31:51.641 [main] DEBUG nextflow.trace.TimelineObserver\
  \ - Workflow completed -- rendering execution timeline\nFeb-19 04:31:51.803 [main]\
  \ DEBUG nextflow.cache.CacheDB - Closing CacheDB done\nFeb-19 04:31:51.840 [main]\
  \ DEBUG nextflow.util.ThreadPoolManager - Thread pool 'FileTransfer' shutdown completed\
  \ (hard=false)\nFeb-19 04:31:51.840 [main] DEBUG nextflow.script.ScriptRunner -\
  \ > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Ffowb4jti%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Ffowb4jti%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.591GiB</td>
						<td>22.5 seconds</td>
						<td>30.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743116224">
								<a href="https://nf-co.re/hlatyping">hlatyping</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hlatyping/tree/1.1.2">1.1.2</a>
						</td>
						<td>nextflow</td>
						<td>1530 days</td>
						<td>2023-02-19 03:43:31</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fp5omcqur%2Flogs.tar.xz">0.038GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fp5omcqur%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.520GiB</td>
						<td>516.2 seconds</td>
						<td>299.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743187824">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/1.2">1.2</a>
						</td>
						<td>nextflow</td>
						<td>1529 days</td>
						<td>2023-02-19 04:34:12</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F3ljk9ucd%2Flogs.tar.xz">0.023GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F3ljk9ucd%2Foutput.tar.xz">0.004GiB</a>
						</td>
						<td>2.132GiB</td>
						<td>990.5 seconds</td>
						<td>650.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743108304">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.0.3">2.0.3</a>
						</td>
						<td>nextflow</td>
						<td>1529 days</td>
						<td>2023-02-19 06:38:54</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 06:39:03.965
kind: NextflowMiscError
process: main
rest: Please specify either --singleEnd or --pairedEnd to execute the pipeline!

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F88s31kfz%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F88s31kfz%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.513GiB</td>
						<td>21.4 seconds</td>
						<td>9.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745002688">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1529 days</td>
						<td>2023-02-19 04:19:06</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: metadataCategoryPairwise.r Metadata.tsv
error: "Warning message:\nIn read.table(file = file, header = header, sep = sep, quote\
  \ = quote,  :\n  incomplete final line found by readTableHeader on 'Metadata.tsv'\n\
  Error in 1:ncol(data) : argument of length 0\nExecution halted"
exit_status: 1
kind: CalledProcessError
output: (empty)
process: metadata_category_pairwise (1)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- 0a
- 3817f9c55134c306dcd31c3cedf069

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fmwfoplpz%2Flogs.tar.xz">0.004GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fmwfoplpz%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>1.787GiB</td>
						<td>1150.1 seconds</td>
						<td>660.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743139872">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/1.2">1.2</a>
						</td>
						<td>nextflow</td>
						<td>1509 days</td>
						<td>2023-02-19 06:21:39</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fxcl343pc%2Flogs.tar.xz">0.018GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fxcl343pc%2Foutput.tar.xz">0.004GiB</a>
						</td>
						<td>1.662GiB</td>
						<td>576.4 seconds</td>
						<td>396.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743144912">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>1507 days</td>
						<td>2023-02-19 03:31:12</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.io.IOException
msg: 'Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv'
rest: "at nextflow.file.http.XFileSystemProvider.readAttributes(XFileSystemProvider.groovy:463)\n\
  \tat java.base/java.nio.file.Files.readAttributes(Files.java:1764)\n\tat nextflow.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:156)\n\
  \tat nextflow.file.FileHelper.copyPath(FileHelper.groovy:971)\n\tat nextflow.file.FilePorter$FileTransfer.stageForeignFile0(FilePorter.groovy:299)\n\
  \tat nextflow.file.FilePorter$FileTransfer.stageForeignFile(FilePorter.groovy:266)\n\
  \tat nextflow.file.FilePorter$FileTransfer.run(FilePorter.groovy:252)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\
  \tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-19 04:00:17.617 [FileTransfer-10]\
  \ DEBUG nextflow.file.FilePorter - Copying foreign file https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ to work dir: /home/azureuser/tmp/7dtgvtgp48/code/work/stage/70/69b1780b1020cd96ef20a0adf1ba66/alleles.tsv\n\
  Feb-19 04:00:17.634 [FileTransfer-10] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 250ms (attempt=1)\nFeb-19 04:00:17.898 [FileTransfer-10]\
  \ DEBUG n.file.http.XFileSystemProvider - Got HTTP error=404 waiting for 750ms (attempt=2)\n\
  Feb-19 04:00:18.661 [FileTransfer-10] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 2250ms (attempt=3)\nFeb-19 04:00:20.925 [FileTransfer-10]\
  \ WARN  nextflow.file.FilePorter - Unable to stage foreign file: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ (try 2) -- Cause: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\n\
  java.io.IOException: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\n\
  \tat nextflow.file.http.XFileSystemProvider.readAttributes(XFileSystemProvider.groovy:463)\n\
  \tat java.base/java.nio.file.Files.readAttributes(Files.java:1764)\n\tat nextflow.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:156)\n\
  \tat nextflow.file.FileHelper.copyPath(FileHelper.groovy:971)\n\tat nextflow.file.FilePorter$FileTransfer.stageForeignFile0(FilePorter.groovy:299)\n\
  \tat nextflow.file.FilePorter$FileTransfer.stageForeignFile(FilePorter.groovy:266)\n\
  \tat nextflow.file.FilePorter$FileTransfer.run(FilePorter.groovy:252)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\
  \tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-19 04:00:21.085 [FileTransfer-10]\
  \ DEBUG nextflow.file.FilePorter - Copying foreign file https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ to work dir: /home/azureuser/tmp/7dtgvtgp48/code/work/stage/70/69b1780b1020cd96ef20a0adf1ba66/alleles.tsv\n\
  Feb-19 04:00:21.098 [FileTransfer-10] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 250ms (attempt=1)\nFeb-19 04:00:21.362 [FileTransfer-10]\
  \ DEBUG n.file.http.XFileSystemProvider - Got HTTP error=404 waiting for 750ms (attempt=2)\n\
  Feb-19 04:00:22.126 [FileTransfer-10] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 2250ms (attempt=3)\nFeb-19 04:00:24.393 [FileTransfer-10]\
  \ WARN  nextflow.file.FilePorter - Unable to stage foreign file: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ (try 3) -- Cause: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\n\
  java.io.IOException: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\n\
  \tat nextflow.file.http.XFileSystemProvider.readAttributes(XFileSystemProvider.groovy:463)\n\
  \tat java.base/java.nio.file.Files.readAttributes(Files.java:1764)\n\tat nextflow.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:156)\n\
  \tat nextflow.file.FileHelper.copyPath(FileHelper.groovy:971)\n\tat nextflow.file.FilePorter$FileTransfer.stageForeignFile0(FilePorter.groovy:299)\n\
  \tat nextflow.file.FilePorter$FileTransfer.stageForeignFile(FilePorter.groovy:266)\n\
  \tat nextflow.file.FilePorter$FileTransfer.run(FilePorter.groovy:252)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\
  \tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-19 04:00:24.465 [FileTransfer-10]\
  \ DEBUG nextflow.file.FilePorter - Copying foreign file https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ to work dir: /home/azureuser/tmp/7dtgvtgp48/code/work/stage/70/69b1780b1020cd96ef20a0adf1ba66/alleles.tsv\n\
  Feb-19 04:00:24.480 [FileTransfer-10] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 250ms (attempt=1)\nFeb-19 04:00:24.745 [FileTransfer-10]\
  \ DEBUG n.file.http.XFileSystemProvider - Got HTTP error=404 waiting for 750ms (attempt=2)\n\
  Feb-19 04:00:25.509 [FileTransfer-10] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 2250ms (attempt=3)\nFeb-19 04:00:27.786 [Actor Thread\
  \ 11] ERROR nextflow.processor.TaskProcessor - Error executing process > 'predict_peptides\
  \ (1)'\n\nCaused by:\n  Can't stage file https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ -- reason: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\n\
  \n\nTip: you can replicate the issue by changing to the process work dir and entering\
  \ the command `bash .command.run`\nFeb-19 04:00:27.827 [Actor Thread 11] INFO  nextflow.Session\
  \ - Execution cancelled -- Finishing pending tasks before exit\nFeb-19 04:00:27.832\
  \ [main] DEBUG nextflow.Session - Session await > all processes finished\nFeb-19\
  \ 04:00:27.832 [main] DEBUG nextflow.Session - Session await > all barriers passed\n\
  Feb-19 04:00:27.835 [main] DEBUG nextflow.util.ThreadPoolManager - Thread pool 'PublishDir'\
  \ shutdown completed (hard=false)\nFeb-19 04:00:28.065 [main] INFO  nextflow.Nextflow\
  \ - [nf-core/mhcquant] Pipeline Complete\nFeb-19 04:00:28.068 [main] DEBUG nextflow.trace.WorkflowStatsObserver\
  \ - Workflow completed > WorkflowStats[succeededCount=39; failedCount=0; ignoredCount=0;\
  \ cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0;\
  \ abortedCount=0; succeedDuration=1h 2m 38s; failedDuration=0ms; cachedDuration=0ms;loadCpus=0;\
  \ loadMemory=0; peakRunning=2; peakCpus=4; peakMemory=12 GB; ]\nFeb-19 04:00:28.069\
  \ [main] DEBUG nextflow.trace.TraceFileObserver - Workflow completed -- saving trace\
  \ file\nFeb-19 04:00:28.076 [main] DEBUG nextflow.trace.ReportObserver - Workflow\
  \ completed -- rendering execution report\nFeb-19 04:00:28.192 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Execution report summary data:\n  [{\"cpuUsage\":{\"mean\":20.75,\"min\":20.75,\"\
  q1\":20.75,\"q2\":20.75,\"q3\":20.75,\"max\":20.75,\"minLabel\":\"output_documentation\"\
  ,\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\"\
  :\"output_documentation\",\"q3Label\":\"output_documentation\"},\"process\":\"output_documentation\"\
  ,\"mem\":{\"mean\":11485184,\"min\":11485184,\"q1\":11485184,\"q2\":11485184,\"\
  q3\":11485184,\"max\":11485184,\"minLabel\":\"output_documentation\",\"maxLabel\"\
  :\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\":\"output_documentation\"\
  ,\"q3Label\":\"output_documentation\"},\"memUsage\":{\"mean\":0.18,\"min\":0.18,\"\
  q1\":0.18,\"q2\":0.18,\"q3\":0.18,\"max\":0.18,\"minLabel\":\"output_documentation\"\
  ,\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\"\
  :\"output_documentation\",\"q3Label\":\"output_documentation\"},\"timeUsage\":{\"\
  mean\":0.02,\"min\":0.02,\"q1\":0.02,\"q2\":0.02,\"q3\":0.02,\"max\":0.02,\"minLabel\"\
  :\"output_documentation\",\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\"\
  ,\"q2Label\":\"output_documentation\",\"q3Label\":\"output_documentation\"},\"vmem\"\
  :{\"mean\":166858752,\"min\":166858752,\"q1\":166858752,\"q2\":166858752,\"q3\"\
  :166858752,\"max\":166858752,\"minLabel\":\"output_documentation\",\"maxLabel\"\
  :\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\":\"output_documentation\"\
  ,\"q3Label\":\"output_documentation\"},\"reads\":{\"mean\":5772448,\"min\":5772448,\"\
  q1\":5772448,\"q2\":5772448,\"q3\":5772448,\"max\":5772448,\"minLabel\":\"output_documentation\"\
  ,\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\"\
  :\"output_documentation\",\"q3Label\":\"output_documentation\"},\"cpu\":{\"mean\"\
  :41.5,\"min\":41.5,\"q1\":41.5,\"q2\":41.5,\"q3\":41.5,\"max\":41.5,\"minLabel\"\
  :\"output_documentation\",\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\"\
  ,\"q2Label\":\"output_documentation\",\"q3Label\":\"output_documentation\"},\"time\"\
  :{\"mean\":1316,\"min\":1316,\"q1\":1316,\"q2\":1316,\"q3\":1316,\"max\":1316,\"\
  minLabel\":\"output_documentation\",\"maxLabel\":\"output_documentation\",\"q1Label\"\
  :\"output_documentation\",\"q2Label\":\"output_documentation\",\"q3Label\":\"output_documentation\"\
  },\"writes\":{\"mean\":7035,\"min\":7035,\"q1\":7035,\"q2\":7035,\"q3\":7035,\"\
  max\":7035,\"minLabel\":\"output_documentation\",\"maxLabel\":\"output_documentation\"\
  ,\"q1Label\":\"output_documentation\",\"q2Label\":\"output_documentation\",\"q3Label\"\
  :\"output_documentation\"}},{\"cpuUsage\":{\"mean\":22.6,\"min\":22.6,\"q1\":22.6,\"\
  q2\":22.6,\"q3\":22.6,\"max\":22.6,\"minLabel\":\"generate_decoy_database (1)\"\
  ,\"maxLabel\":\"generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database\
  \ (1)\",\"q2Label\":\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database\
  \ (1)\"},\"process\":\"generate_decoy_database\",\"mem\":{\"mean\":40095744,\"min\"\
  :40095744,\"q1\":40095744,\"q2\":40095744,\"q3\":40095744,\"max\":40095744,\"minLabel\"\
  :\"generate_decoy_database (1)\",\"maxLabel\":\"generate_decoy_database (1)\",\"\
  q1Label\":\"generate_decoy_database (1)\",\"q2Label\":\"generate_decoy_database\
  \ (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"memUsage\":{\"mean\":0.62,\"\
  min\":0.62,\"q1\":0.62,\"q2\":0.62,\"q3\":0.62,\"max\":0.62,\"minLabel\":\"generate_decoy_database\
  \ (1)\",\"maxLabel\":\"generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database\
  \ (1)\",\"q2Label\":\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database\
  \ (1)\"},\"timeUsage\":{\"mean\":0.2,\"min\":0.2,\"q1\":0.2,\"q2\":0.2,\"q3\":0.2,\"\
  max\":0.2,\"minLabel\":\"generate_decoy_database (1)\",\"maxLabel\":\"generate_decoy_database\
  \ (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"q2Label\":\"generate_decoy_database\
  \ (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"vmem\":{\"mean\":182812672,\"\
  min\":182812672,\"q1\":182812672,\"q2\":182812672,\"q3\":182812672,\"max\":182812672,\"\
  minLabel\":\"generate_decoy_database (1)\",\"maxLabel\":\"generate_decoy_database\
  \ (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"q2Label\":\"generate_decoy_database\
  \ (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"reads\":{\"mean\":16754664,\"\
  min\":16754664,\"q1\":16754664,\"q2\":16754664,\"q3\":16754664,\"max\":16754664,\"\
  minLabel\":\"generate_decoy_database (1)\",\"maxLabel\":\"generate_decoy_database\
  \ (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"q2Label\":\"generate_decoy_database\
  \ (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"cpu\":{\"mean\":45.2,\"min\"\
  :45.2,\"q1\":45.2,\"q2\":45.2,\"q3\":45.2,\"max\":45.2,\"minLabel\":\"generate_decoy_database\
  \ (1)\",\"maxLabel\":\"generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database\
  \ (1)\",\"q2Label\":\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database\
  \ (1)\"},\"time\":{\"mean\":14578,\"min\":14578,\"q1\":14578,\"q2\":14578,\"q3\"\
  :14578,\"max\":14578,\"minLabel\":\"generate_decoy_database (1)\",\"maxLabel\":\"\
  generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"q2Label\"\
  :\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"\
  writes\":{\"mean\":26798722,\"min\":26798722,\"q1\":26798722,\"q2\":26798722,\"\
  q3\":26798722,\"max\":26798722,\"minLabel\":\"generate_decoy_database (1)\",\"maxLabel\"\
  :\"generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"\
  q2Label\":\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database\
  \ (1)\"}},{\"cpuUsage\":{\"mean\":55.7,\"min\":45.15,\"q1\":46.61,\"q2\":55.88,\"\
  q3\":64.96,\"max\":65.9,\"minLabel\":\"db_search_comet (1)\",\"maxLabel\":\"db_search_comet\
  \ (2)\",\"q1Label\":\"db_search_comet (1)\",\"q2Label\":\"db_search_comet (4)\"\
  ,\"q3Label\":\"db_search_comet (3)\"},\"process\":\"db_search_comet\",\"mem\":{\"\
  mean\":283415552,\"min\":262615040,\"q1\":263564288,\"q2\":264005632,\"q3\":283856896,\"\
  max\":343035904,\"minLabel\":\"db_search_comet (4)\",\"maxLabel\":\"db_search_comet\
  \ (2)\",\"q1Label\":\"db_search_comet (4)\",\"q2Label\":\"db_search_comet (3)\"\
  ,\"q3Label\":\"db_search_comet (1)\"},\"memUsage\":{\"mean\":4.4,\"min\":4.08,\"\
  q1\":4.09,\"q2\":4.1,\"q3\":4.41,\"max\":5.32,\"minLabel\":\"db_search_comet (4)\"\
  ,\"maxLabel\":\"db_search_comet (2)\",\"q1Label\":\"db_search_comet (4)\",\"q2Label\"\
  :\"db_search_comet (3)\",\"q3Label\":\"db_search_comet (1)\"},\"timeUsage\":{\"\
  mean\":0.51,\"min\":0.41,\"q1\":0.42,\"q2\":0.5,\"q3\":0.59,\"max\":0.63,\"minLabel\"\
  :\"db_search_comet (2)\",\"maxLabel\":\"db_search_comet (4)\",\"q1Label\":\"db_search_comet\
  \ (2)\",\"q2Label\":\"db_search_comet (3)\",\"q3Label\":\"db_search_comet (1)\"\
  },\"vmem\":{\"mean\":856925184,\"min\":802451456,\"q1\":802469888,\"q2\":844374016,\"\
  q3\":898829312,\"max\":936501248,\"minLabel\":\"db_search_comet (3)\",\"maxLabel\"\
  :\"db_search_comet (4)\",\"q1Label\":\"db_search_comet (3)\",\"q2Label\":\"db_search_comet\
  \ (1)\",\"q3Label\":\"db_search_comet (2)\"},\"reads\":{\"mean\":307167001.75,\"\
  min\":301047678,\"q1\":303283535.25,\"q2\":307076793.5,\"q3\":310960260,\"max\"\
  :313466742,\"minLabel\":\"db_search_comet (4)\",\"maxLabel\":\"db_search_comet (2)\"\
  ,\"q1Label\":\"db_search_comet (4)\",\"q2Label\":\"db_search_comet (3)\",\"q3Label\"\
  :\"db_search_comet (1)\"},\"cpu\":{\"mean\":111.4,\"min\":90.3,\"q1\":93.23,\"q2\"\
  :111.75,\"q3\":129.93,\"max\":131.8,\"minLabel\":\"db_search_comet (1)\",\"maxLabel\"\
  :\"db_search_comet (2)\",\"q1Label\":\"db_search_comet (1)\",\"q2Label\":\"db_search_comet\
  \ (4)\",\"q3Label\":\"db_search_comet (3)\"},\"time\":{\"mean\":439883.75,\"min\"\
  :350172,\"q1\":360545.25,\"q2\":434527,\"q3\":513865.5,\"max\":540309,\"minLabel\"\
  :\"db_search_comet (2)\",\"maxLabel\":\"db_search_comet (4)\",\"q1Label\":\"db_search_comet\
  \ (2)\",\"q2Label\":\"db_search_comet (3)\",\"q3Label\":\"db_search_comet (1)\"\
  },\"writes\":{\"mean\":3496292.5,\"min\":3250070,\"q1\":3306671,\"q2\":3505962.5,\"\
  q3\":3695584,\"max\":3723175,\"minLabel\":\"db_search_comet (4)\",\"maxLabel\":\"\
  db_search_comet (2)\",\"q1Label\":\"db_search_comet (4)\",\"q2Label\":\"db_search_comet\
  \ (3)\",\"q3Label\":\"db_search_comet (1)\"}},{\"cpuUsage\":{\"mean\":43.58,\"min\"\
  :23.35,\"q1\":40.9,\"q2\":46.9,\"q3\":49.58,\"max\":57.15,\"minLabel\":\"index_peptides\
  \ (1)\",\"maxLabel\":\"index_peptides (4)\",\"q1Label\":\"index_peptides (1)\",\"\
  q2Label\":\"index_peptides (2)\",\"q3Label\":\"index_peptides (3)\"},\"process\"\
  :\"index_peptides\",\"mem\":{\"mean\":58960896,\"min\":15073280,\"q1\":48158720,\"\
  q2\":69931008,\"q3\":80733184,\"max\":80908288,\"minLabel\":\"index_peptides (4)\"\
  ,\"maxLabel\":\"index_peptides (2)\",\"q1Label\":\"index_peptides (4)\",\"q2Label\"\
  :\"index_peptides (1)\",\"q3Label\":\"index_peptides (3)\"},\"memUsage\":{\"mean\"\
  :0.92,\"min\":0.23,\"q1\":0.75,\"q2\":1.09,\"q3\":1.25,\"max\":1.26,\"minLabel\"\
  :\"index_peptides (4)\",\"maxLabel\":\"index_peptides (2)\",\"q1Label\":\"index_peptides\
  \ (4)\",\"q2Label\":\"index_peptides (1)\",\"q3Label\":\"index_peptides (3)\"},\"\
  timeUsage\":{\"mean\":0.03,\"min\":0.02,\"q1\":0.02,\"q2\":0.03,\"q3\":0.03,\"max\"\
  :0.05,\"minLabel\":\"index_peptides (4)\",\"maxLabel\":\"index_peptides (1)\",\"\
  q1Label\":\"index_peptides (4)\",\"q2Label\":\"index_peptides (3)\",\"q3Label\"\
  :\"index_peptides (2)\"},\"vmem\":{\"mean\":371334144,\"min\":173944832,\"q1\":322927616,\"\
  q2\":377350144,\"q3\":425756672,\"max\":556691456,\"minLabel\":\"index_peptides\
  \ (4)\",\"maxLabel\":\"index_peptides (1)\",\"q1Label\":\"index_peptides (4)\",\"\
  q2Label\":\"index_peptides (3)\",\"q3Label\":\"index_peptides (2)\"},\"reads\":{\"\
  mean\":31945421.25,\"min\":31811995,\"q1\":31843438.75,\"q2\":31955144.5,\"q3\"\
  :32057127,\"max\":32059401,\"minLabel\":\"index_peptides (2)\",\"maxLabel\":\"index_peptides\
  \ (1)\",\"q1Label\":\"index_peptides (2)\",\"q2Label\":\"index_peptides (3)\",\"\
  q3Label\":\"index_peptides (4)\"},\"cpu\":{\"mean\":87.15,\"min\":46.7,\"q1\":81.8,\"\
  q2\":93.8,\"q3\":99.15,\"max\":114.3,\"minLabel\":\"index_peptides (1)\",\"maxLabel\"\
  :\"index_peptides (4)\",\"q1Label\":\"index_peptides (1)\",\"q2Label\":\"index_peptides\
  \ (2)\",\"q3Label\":\"index_peptides (3)\"},\"time\":{\"mean\":2201.5,\"min\":1148,\"\
  q1\":1669.25,\"q2\":1893,\"q3\":2425.25,\"max\":3872,\"minLabel\":\"index_peptides\
  \ (4)\",\"maxLabel\":\"index_peptides (1)\",\"q1Label\":\"index_peptides (4)\",\"\
  q2Label\":\"index_peptides (3)\",\"q3Label\":\"index_peptides (2)\"},\"writes\"\
  :{\"mean\":2213653.25,\"min\":2056962,\"q1\":2096397.75,\"q2\":2225222,\"q3\":2342477.5,\"\
  max\":2347207,\"minLabel\":\"index_peptides (2)\",\"maxLabel\":\"index_peptides\
  \ (4)\",\"q1Label\":\"index_peptides (2)\",\"q2Label\":\"index_peptides (3)\",\"\
  q3Label\":\"index_peptides (1)\"}},{\"cpuUsage\":{\"mean\":33.01,\"min\":18.1,\"\
  q1\":28,\"q2\":32.88,\"q3\":37.89,\"max\":48.2,\"minLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"maxLabel\":\"calculate_fdr_for_idalignment (4)\",\"q1Label\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q2Label\":\"calculate_fdr_for_idalignment (2)\",\"q3Label\":\"calculate_fdr_for_idalignment\
  \ (3)\"},\"process\":\"calculate_fdr_for_idalignment\",\"mem\":{\"mean\":22989824,\"\
  min\":10391552,\"q1\":19106816,\"q2\":22132736,\"q3\":26015744,\"max\":37302272,\"\
  minLabel\":\"calculate_fdr_for_idalignment (4)\",\"maxLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"calculate_fdr_for_idalignment (4)\",\"q2Label\":\"calculate_fdr_for_idalignment\
  \ (3)\",\"q3Label\":\"calculate_fdr_for_idalignment (2)\"},\"memUsage\":{\"mean\"\
  :0.36,\"min\":0.16,\"q1\":0.3,\"q2\":0.34,\"q3\":0.4,\"max\":0.58,\"minLabel\":\"\
  calculate_fdr_for_idalignment (4)\",\"maxLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"calculate_fdr_for_idalignment (4)\",\"q2Label\":\"calculate_fdr_for_idalignment\
  \ (3)\",\"q3Label\":\"calculate_fdr_for_idalignment (2)\"},\"timeUsage\":{\"mean\"\
  :0.02,\"min\":0.01,\"q1\":0.02,\"q2\":0.02,\"q3\":0.03,\"max\":0.04,\"minLabel\"\
  :\"calculate_fdr_for_idalignment (4)\",\"maxLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"calculate_fdr_for_idalignment (4)\",\"q2Label\":\"calculate_fdr_for_idalignment\
  \ (3)\",\"q3Label\":\"calculate_fdr_for_idalignment (2)\"},\"vmem\":{\"mean\":269411328,\"\
  min\":173666304,\"q1\":173779968,\"q2\":173817856,\"q3\":269449216,\"max\":556343296,\"\
  minLabel\":\"calculate_fdr_for_idalignment (4)\",\"maxLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"calculate_fdr_for_idalignment (4)\",\"q2Label\":\"calculate_fdr_for_idalignment\
  \ (2)\",\"q3Label\":\"calculate_fdr_for_idalignment (3)\"},\"reads\":{\"mean\":5568485.25,\"\
  min\":5406213,\"q1\":5445629.25,\"q2\":5577599.5,\"q3\":5700455.5,\"max\":5712529,\"\
  minLabel\":\"calculate_fdr_for_idalignment (2)\",\"maxLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"calculate_fdr_for_idalignment (2)\",\"q2Label\":\"calculate_fdr_for_idalignment\
  \ (3)\",\"q3Label\":\"calculate_fdr_for_idalignment (4)\"},\"cpu\":{\"mean\":66.03,\"\
  min\":36.2,\"q1\":56,\"q2\":65.75,\"q3\":75.78,\"max\":96.4,\"minLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"maxLabel\":\"calculate_fdr_for_idalignment (4)\",\"q1Label\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q2Label\":\"calculate_fdr_for_idalignment (2)\",\"q3Label\":\"calculate_fdr_for_idalignment\
  \ (3)\"},\"time\":{\"mean\":1679.75,\"min\":1034,\"q1\":1243.25,\"q2\":1436,\"q3\"\
  :1872.5,\"max\":2813,\"minLabel\":\"calculate_fdr_for_idalignment (4)\",\"maxLabel\"\
  :\"calculate_fdr_for_idalignment (1)\",\"q1Label\":\"calculate_fdr_for_idalignment\
  \ (4)\",\"q2Label\":\"calculate_fdr_for_idalignment (3)\",\"q3Label\":\"calculate_fdr_for_idalignment\
  \ (2)\"},\"writes\":{\"mean\":1580569.5,\"min\":1385109,\"q1\":1460340.75,\"q2\"\
  :1603451.5,\"q3\":1723680.25,\"max\":1730266,\"minLabel\":\"calculate_fdr_for_idalignment\
  \ (2)\",\"maxLabel\":\"calculate_fdr_for_idalignment (1)\",\"q1Label\":\"calculate_fdr_for_idalignment\
  \ (2)\",\"q2Label\":\"calculate_fdr_for_idalignment (3)\",\"q3Label\":\"calculate_fdr_for_idalignment\
  \ (4)\"}},{\"cpuUsage\":{\"mean\":38.67,\"min\":17.9,\"q1\":36.91,\"q2\":44.5,\"\
  q3\":46.26,\"max\":47.8,\"minLabel\":\"filter_fdr_for_idalignment (1)\",\"maxLabel\"\
  :\"filter_fdr_for_idalignment (4)\",\"q1Label\":\"filter_fdr_for_idalignment (1)\"\
  ,\"q2Label\":\"filter_fdr_for_idalignment (2)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (3)\"},\"process\":\"filter_fdr_for_idalignment\",\"mem\":{\"mean\":23363584,\"\
  min\":7380992,\"q1\":14968832,\"q2\":22874112,\"q3\":31268864,\"max\":40325120,\"\
  minLabel\":\"filter_fdr_for_idalignment (4)\",\"maxLabel\":\"filter_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"filter_fdr_for_idalignment (4)\",\"q2Label\":\"filter_fdr_for_idalignment\
  \ (2)\",\"q3Label\":\"filter_fdr_for_idalignment (3)\"},\"memUsage\":{\"mean\":0.36,\"\
  min\":0.11,\"q1\":0.23,\"q2\":0.36,\"q3\":0.49,\"max\":0.63,\"minLabel\":\"filter_fdr_for_idalignment\
  \ (4)\",\"maxLabel\":\"filter_fdr_for_idalignment (1)\",\"q1Label\":\"filter_fdr_for_idalignment\
  \ (4)\",\"q2Label\":\"filter_fdr_for_idalignment (2)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (3)\"},\"timeUsage\":{\"mean\":0.02,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"q3\"\
  :0.02,\"max\":0.04,\"minLabel\":\"filter_fdr_for_idalignment (3)\",\"maxLabel\"\
  :\"filter_fdr_for_idalignment (1)\",\"q1Label\":\"filter_fdr_for_idalignment (3)\"\
  ,\"q2Label\":\"filter_fdr_for_idalignment (4)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (2)\"},\"vmem\":{\"mean\":229115904,\"min\":164855808,\"q1\":171632640,\"q2\"\
  :174233600,\"q3\":231716864,\"max\":403140608,\"minLabel\":\"filter_fdr_for_idalignment\
  \ (4)\",\"maxLabel\":\"filter_fdr_for_idalignment (1)\",\"q1Label\":\"filter_fdr_for_idalignment\
  \ (4)\",\"q2Label\":\"filter_fdr_for_idalignment (2)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (3)\"},\"reads\":{\"mean\":4962818.25,\"min\":4761767,\"q1\":4836996.5,\"q2\"\
  :4980106.5,\"q3\":5105928.25,\"max\":5129293,\"minLabel\":\"filter_fdr_for_idalignment\
  \ (2)\",\"maxLabel\":\"filter_fdr_for_idalignment (1)\",\"q1Label\":\"filter_fdr_for_idalignment\
  \ (2)\",\"q2Label\":\"filter_fdr_for_idalignment (3)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (4)\"},\"cpu\":{\"mean\":77.35,\"min\":35.8,\"q1\":73.83,\"q2\":89,\"q3\":92.53,\"\
  max\":95.6,\"minLabel\":\"filter_fdr_for_idalignment (1)\",\"maxLabel\":\"filter_fdr_for_idalignment\
  \ (4)\",\"q1Label\":\"filter_fdr_for_idalignment (1)\",\"q2Label\":\"filter_fdr_for_idalignment\
  \ (2)\",\"q3Label\":\"filter_fdr_for_idalignment (3)\"},\"time\":{\"mean\":1323.25,\"\
  min\":867,\"q1\":871.5,\"q2\":889,\"q3\":1340.75,\"max\":2648,\"minLabel\":\"filter_fdr_for_idalignment\
  \ (3)\",\"maxLabel\":\"filter_fdr_for_idalignment (1)\",\"q1Label\":\"filter_fdr_for_idalignment\
  \ (3)\",\"q2Label\":\"filter_fdr_for_idalignment (4)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (2)\"},\"writes\":{\"mean\":520655,\"min\":357883,\"q1\":405503.5,\"q2\":504867.5,\"\
  q3\":620019,\"max\":715002,\"minLabel\":\"filter_fdr_for_idalignment (2)\",\"maxLabel\"\
  :\"filter_fdr_for_idalignment (1)\",\"q1Label\":\"filter_fdr_for_idalignment (2)\"\
  ,\"q2Label\":\"filter_fdr_for_idalignment (3)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (4)\"}},{\"cpuUsage\":{\"mean\":17.5,\"min\":17.5,\"q1\":17.5,\"q2\":17.5,\"q3\"\
  :17.5,\"max\":17.5,\"minLabel\":\"align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\"\
  :\"align_ids\",\"q2Label\":\"align_ids\",\"q3Label\":\"align_ids\"},\"process\"\
  :\"align_ids\",\"mem\":{\"mean\":34283520,\"min\":34283520,\"q1\":34283520,\"q2\"\
  :34283520,\"q3\":34283520,\"max\":34283520,\"minLabel\":\"align_ids\",\"maxLabel\"\
  :\"align_ids\",\"q1Label\":\"align_ids\",\"q2Label\":\"align_ids\",\"q3Label\":\"\
  align_ids\"},\"memUsage\":{\"mean\":0.53,\"min\":0.53,\"q1\":0.53,\"q2\":0.53,\"\
  q3\":0.53,\"max\":0.53,\"minLabel\":\"align_ids\",\"maxLabel\":\"align_ids\",\"\
  q1Label\":\"align_ids\",\"q2Label\":\"align_ids\",\"q3Label\":\"align_ids\"},\"\
  timeUsage\":{\"mean\":0.03,\"min\":0.03,\"q1\":0.03,\"q2\":0.03,\"q3\":0.03,\"max\"\
  :0.03,\"minLabel\":\"align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\"\
  ,\"q2Label\":\"align_ids\",\"q3Label\":\"align_ids\"},\"vmem\":{\"mean\":397688832,\"\
  min\":397688832,\"q1\":397688832,\"q2\":397688832,\"q3\":397688832,\"max\":397688832,\"\
  minLabel\":\"align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\",\"\
  q2Label\":\"align_ids\",\"q3Label\":\"align_ids\"},\"reads\":{\"mean\":5478256,\"\
  min\":5478256,\"q1\":5478256,\"q2\":5478256,\"q3\":5478256,\"max\":5478256,\"minLabel\"\
  :\"align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\",\"q2Label\"\
  :\"align_ids\",\"q3Label\":\"align_ids\"},\"cpu\":{\"mean\":35,\"min\":35,\"q1\"\
  :35,\"q2\":35,\"q3\":35,\"max\":35,\"minLabel\":\"align_ids\",\"maxLabel\":\"align_ids\"\
  ,\"q1Label\":\"align_ids\",\"q2Label\":\"align_ids\",\"q3Label\":\"align_ids\"},\"\
  time\":{\"mean\":2017,\"min\":2017,\"q1\":2017,\"q2\":2017,\"q3\":2017,\"max\":2017,\"\
  minLabel\":\"align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\",\"\
  q2Label\":\"align_ids\",\"q3Label\":\"align_ids\"},\"writes\":{\"mean\":56604,\"\
  min\":56604,\"q1\":56604,\"q2\":56604,\"q3\":56604,\"max\":56604,\"minLabel\":\"\
  align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\",\"q2Label\":\"\
  align_ids\",\"q3Label\":\"align_ids\"}},{\"cpuUsage\":{\"mean\":38.7,\"min\":20.25,\"\
  q1\":35.74,\"q2\":43.78,\"q3\":46.74,\"max\":47,\"minLabel\":\"align_idxml_files\
  \ (4)\",\"maxLabel\":\"align_idxml_files (3)\",\"q1Label\":\"align_idxml_files (4)\"\
  ,\"q2Label\":\"align_idxml_files (1)\",\"q3Label\":\"align_idxml_files (2)\"},\"\
  process\":\"align_idxml_files\",\"mem\":{\"mean\":18051072,\"min\":5500928,\"q1\"\
  :5706752,\"q2\":9099264,\"q3\":21443584,\"max\":48504832,\"minLabel\":\"align_idxml_files\
  \ (3)\",\"maxLabel\":\"align_idxml_files (4)\",\"q1Label\":\"align_idxml_files (3)\"\
  ,\"q2Label\":\"align_idxml_files (2)\",\"q3Label\":\"align_idxml_files (1)\"},\"\
  memUsage\":{\"mean\":0.28,\"min\":0.09,\"q1\":0.09,\"q2\":0.14,\"q3\":0.33,\"max\"\
  :0.75,\"minLabel\":\"align_idxml_files (3)\",\"maxLabel\":\"align_idxml_files (4)\"\
  ,\"q1Label\":\"align_idxml_files (3)\",\"q2Label\":\"align_idxml_files (2)\",\"\
  q3Label\":\"align_idxml_files (1)\"},\"timeUsage\":{\"mean\":0.02,\"min\":0.01,\"\
  q1\":0.01,\"q2\":0.01,\"q3\":0.02,\"max\":0.03,\"minLabel\":\"align_idxml_files\
  \ (3)\",\"maxLabel\":\"align_idxml_files (4)\",\"q1Label\":\"align_idxml_files (3)\"\
  ,\"q2Label\":\"align_idxml_files (1)\",\"q3Label\":\"align_idxml_files (2)\"},\"\
  vmem\":{\"mean\":229075968,\"min\":88113152,\"q1\":95621120,\"q2\":135917568,\"\
  q3\":269372416,\"max\":556355584,\"minLabel\":\"align_idxml_files (3)\",\"maxLabel\"\
  :\"align_idxml_files (4)\",\"q1Label\":\"align_idxml_files (3)\",\"q2Label\":\"\
  align_idxml_files (2)\",\"q3Label\":\"align_idxml_files (1)\"},\"reads\":{\"mean\"\
  :5590551.25,\"min\":5430382,\"q1\":5467314.25,\"q2\":5596530,\"q3\":5719767,\"max\"\
  :5738763,\"minLabel\":\"align_idxml_files (2)\",\"maxLabel\":\"align_idxml_files\
  \ (4)\",\"q1Label\":\"align_idxml_files (2)\",\"q2Label\":\"align_idxml_files (3)\"\
  ,\"q3Label\":\"align_idxml_files (1)\"},\"cpu\":{\"mean\":77.4,\"min\":40.5,\"q1\"\
  :71.47,\"q2\":87.55,\"q3\":93.48,\"max\":94,\"minLabel\":\"align_idxml_files (4)\"\
  ,\"maxLabel\":\"align_idxml_files (3)\",\"q1Label\":\"align_idxml_files (4)\",\"\
  q2Label\":\"align_idxml_files (1)\",\"q3Label\":\"align_idxml_files (2)\"},\"time\"\
  :{\"mean\":1216.25,\"min\":731,\"q1\":808.25,\"q2\":879,\"q3\":1287,\"max\":2376,\"\
  minLabel\":\"align_idxml_files (3)\",\"maxLabel\":\"align_idxml_files (4)\",\"q1Label\"\
  :\"align_idxml_files (3)\",\"q2Label\":\"align_idxml_files (1)\",\"q3Label\":\"\
  align_idxml_files (2)\"},\"writes\":{\"mean\":2230244,\"min\":2072241,\"q1\":2111970,\"\
  q2\":2241529,\"q3\":2359803,\"max\":2365677,\"minLabel\":\"align_idxml_files (2)\"\
  ,\"maxLabel\":\"align_idxml_files (4)\",\"q1Label\":\"align_idxml_files (2)\",\"\
  q2Label\":\"align_idxml_files (3)\",\"q3Label\":\"align_idxml_files (1)\"}},{\"\
  cpuUsage\":{\"mean\":53.84,\"min\":52.45,\"q1\":53.16,\"q2\":53.65,\"q3\":54.33,\"\
  max\":55.6,\"minLabel\":\"align_mzml_files (2)\",\"maxLabel\":\"align_mzml_files\
  \ (4)\",\"q1Label\":\"align_mzml_files (2)\",\"q2Label\":\"align_mzml_files (1)\"\
  ,\"q3Label\":\"align_mzml_files (3)\"},\"process\":\"align_mzml_files\",\"mem\"\
  :{\"mean\":63121408,\"min\":61132800,\"q1\":62671872,\"q2\":63238144,\"q3\":63687680,\"\
  max\":64876544,\"minLabel\":\"align_mzml_files (4)\",\"maxLabel\":\"align_mzml_files\
  \ (2)\",\"q1Label\":\"align_mzml_files (4)\",\"q2Label\":\"align_mzml_files (3)\"\
  ,\"q3Label\":\"align_mzml_files (1)\"},\"memUsage\":{\"mean\":0.98,\"min\":0.95,\"\
  q1\":0.97,\"q2\":0.98,\"q3\":0.99,\"max\":1.01,\"minLabel\":\"align_mzml_files (4)\"\
  ,\"maxLabel\":\"align_mzml_files (2)\",\"q1Label\":\"align_mzml_files (4)\",\"q2Label\"\
  :\"align_mzml_files (3)\",\"q3Label\":\"align_mzml_files (1)\"},\"timeUsage\":{\"\
  mean\":0.03,\"min\":0.02,\"q1\":0.03,\"q2\":0.03,\"q3\":0.03,\"max\":0.03,\"minLabel\"\
  :\"align_mzml_files (4)\",\"maxLabel\":\"align_mzml_files (2)\",\"q1Label\":\"align_mzml_files\
  \ (4)\",\"q2Label\":\"align_mzml_files (1)\",\"q3Label\":\"align_mzml_files (3)\"\
  },\"vmem\":{\"mean\":327940096,\"min\":327901184,\"q1\":327904256,\"q2\":327919616,\"\
  q3\":327955456,\"max\":328019968,\"minLabel\":\"align_mzml_files (4)\",\"maxLabel\"\
  :\"align_mzml_files (2)\",\"q1Label\":\"align_mzml_files (4)\",\"q2Label\":\"align_mzml_files\
  \ (3)\",\"q3Label\":\"align_mzml_files (1)\"},\"reads\":{\"mean\":27956709.75,\"\
  min\":27127105,\"q1\":27438835,\"q2\":27916184.5,\"q3\":28434059.25,\"max\":28867365,\"\
  minLabel\":\"align_mzml_files (4)\",\"maxLabel\":\"align_mzml_files (2)\",\"q1Label\"\
  :\"align_mzml_files (4)\",\"q2Label\":\"align_mzml_files (3)\",\"q3Label\":\"align_mzml_files\
  \ (1)\"},\"cpu\":{\"mean\":107.68,\"min\":104.9,\"q1\":106.33,\"q2\":107.3,\"q3\"\
  :108.65,\"max\":111.2,\"minLabel\":\"align_mzml_files (2)\",\"maxLabel\":\"align_mzml_files\
  \ (4)\",\"q1Label\":\"align_mzml_files (2)\",\"q2Label\":\"align_mzml_files (1)\"\
  ,\"q3Label\":\"align_mzml_files (3)\"},\"time\":{\"mean\":2230.75,\"min\":1721,\"\
  q1\":2195,\"q2\":2372,\"q3\":2407.75,\"max\":2458,\"minLabel\":\"align_mzml_files\
  \ (4)\",\"maxLabel\":\"align_mzml_files (2)\",\"q1Label\":\"align_mzml_files (4)\"\
  ,\"q2Label\":\"align_mzml_files (1)\",\"q3Label\":\"align_mzml_files (3)\"},\"writes\"\
  :{\"mean\":23667045.75,\"min\":22841870,\"q1\":23151818,\"q2\":23626274,\"q3\":24141501.75,\"\
  max\":24573765,\"minLabel\":\"align_mzml_files (4)\",\"maxLabel\":\"align_mzml_files\
  \ (2)\",\"q1Label\":\"align_mzml_files (4)\",\"q2Label\":\"align_mzml_files (3)\"\
  ,\"q3Label\":\"align_mzml_files (1)\"}},{\"cpuUsage\":{\"mean\":27.25,\"min\":27.25,\"\
  q1\":27.25,\"q2\":27.25,\"q3\":27.25,\"max\":27.25,\"minLabel\":\"merge_aligned_idxml_files\"\
  ,\"maxLabel\":\"merge_aligned_idxml_files\",\"q1Label\":\"merge_aligned_idxml_files\"\
  ,\"q2Label\":\"merge_aligned_idxml_files\",\"q3Label\":\"merge_aligned_idxml_files\"\
  },\"process\":\"merge_aligned_idxml_files\",\"mem\":{\"mean\":51851264,\"min\":51851264,\"\
  q1\":51851264,\"q2\":51851264,\"q3\":51851264,\"max\":51851264,\"minLabel\":\"merge_aligned_idxml_files\"\
  ,\"maxLabel\":\"merge_aligned_idxml_files\",\"q1Label\":\"merge_aligned_idxml_files\"\
  ,\"q2Label\":\"merge_aligned_idxml_files\",\"q3Label\":\"merge_aligned_idxml_files\"\
  },\"memUsage\":{\"mean\":0.8,\"min\":0.8,\"q1\":0.8,\"q2\":0.8,\"q3\":0.8,\"max\"\
  :0.8,\"minLabel\":\"merge_aligned_idxml_files\",\"maxLabel\":\"merge_aligned_idxml_files\"\
  ,\"q1Label\":\"merge_aligned_idxml_files\",\"q2Label\":\"merge_aligned_idxml_files\"\
  ,\"q3Label\":\"merge_aligned_idxml_files\"},\"timeUsage\":{\"mean\":0.04,\"min\"\
  :0.04,\"q1\":0.04,\"q2\":0.04,\"q3\":0.04,\"max\":0.04,\"minLabel\":\"merge_aligned_idxml_files\"\
  ,\"maxLabel\":\"merge_aligned_idxml_files\",\"q1Label\":\"merge_aligned_idxml_files\"\
  ,\"q2Label\":\"merge_aligned_idxml_files\",\"q3Label\":\"merge_aligned_idxml_files\"\
  },\"vmem\":{\"mean\":506486784,\"min\":506486784,\"q1\":506486784,\"q2\":506486784,\"\
  q3\":506486784,\"max\":506486784,\"minLabel\":\"merge_aligned_idxml_files\",\"maxLabel\"\
  :\"merge_aligned_idxml_files\",\"q1Label\":\"merge_aligned_idxml_files\",\"q2Label\"\
  :\"merge_aligned_idxml_files\",\"q3Label\":\"merge_aligned_idxml_files\"},\"reads\"\
  :{\"mean\":12317751,\"min\":12317751,\"q1\":12317751,\"q2\":12317751,\"q3\":12317751,\"\
  max\":12317751,\"minLabel\":\"merge_aligned_idxml_files\",\"maxLabel\":\"merge_aligned_idxml_files\"\
  ,\"q1Label\":\"merge_aligned_idxml_files\",\"q2Label\":\"merge_aligned_idxml_files\"\
  ,\"q3Label\":\"merge_aligned_idxml_files\"},\"cpu\":{\"mean\":54.5,\"min\":54.5,\"\
  q1\":54.5,\"q2\":54.5,\"q3\":54.5,\"max\":54.5,\"minLabel\":\"merge_aligned_idxml_files\"\
  ,\"maxLabel\":\"merge_aligned_idxml_files\",\"q1Label\":\"merge_aligned_idxml_files\"\
  ,\"q2Label\":\"merge_aligned_idxml_files\",\"q3Label\":\"merge_aligned_idxml_files\"\
  },\"time\":{\"mean\":3014,\"min\":3014,\"q1\":3014,\"q2\":3014,\"q3\":3014,\"max\"\
  :3014,\"minLabel\":\"merge_aligned_idxml_files\",\"maxLabel\":\"merge_aligned_idxml_files\"\
  ,\"q1Label\":\"merge_aligned_idxml_files\",\"q2Label\":\"merge_aligned_idxml_files\"\
  ,\"q3Label\":\"merge_aligned_idxml_files\"},\"writes\":{\"mean\":9593737,\"min\"\
  :9593737,\"q1\":9593737,\"q2\":9593737,\"q3\":9593737,\"max\":9593737,\"minLabel\"\
  :\"merge_aligned_idxml_files\",\"maxLabel\":\"merge_aligned_idxml_files\",\"q1Label\"\
  :\"merge_aligned_idxml_files\",\"q2Label\":\"merge_aligned_idxml_files\",\"q3Label\"\
  :\"merge_aligned_idxml_files\"}},{\"cpuUsage\":{\"mean\":30.4,\"min\":30.4,\"q1\"\
  :30.4,\"q2\":30.4,\"q3\":30.4,\"max\":30.4,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"process\":\"extract_psm_features_for_percolator\",\"mem\":{\"mean\":76201984,\"\
  min\":76201984,\"q1\":76201984,\"q2\":76201984,\"q3\":76201984,\"max\":76201984,\"\
  minLabel\":\"extract_psm_features_for_percolator\",\"maxLabel\":\"extract_psm_features_for_percolator\"\
  ,\"q1Label\":\"extract_psm_features_for_percolator\",\"q2Label\":\"extract_psm_features_for_percolator\"\
  ,\"q3Label\":\"extract_psm_features_for_percolator\"},\"memUsage\":{\"mean\":1.18,\"\
  min\":1.18,\"q1\":1.18,\"q2\":1.18,\"q3\":1.18,\"max\":1.18,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"timeUsage\":{\"mean\":0.05,\"min\":0.05,\"q1\":0.05,\"q2\":0.05,\"q3\":0.05,\"\
  max\":0.05,\"minLabel\":\"extract_psm_features_for_percolator\",\"maxLabel\":\"\
  extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"vmem\":{\"mean\":531144704,\"min\":531144704,\"q1\":531144704,\"q2\":531144704,\"\
  q3\":531144704,\"max\":531144704,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"reads\":{\"mean\":12966557,\"min\":12966557,\"q1\":12966557,\"q2\":12966557,\"\
  q3\":12966557,\"max\":12966557,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"cpu\":{\"mean\":60.8,\"min\":60.8,\"q1\":60.8,\"q2\":60.8,\"q3\":60.8,\"max\"\
  :60.8,\"minLabel\":\"extract_psm_features_for_percolator\",\"maxLabel\":\"extract_psm_features_for_percolator\"\
  ,\"q1Label\":\"extract_psm_features_for_percolator\",\"q2Label\":\"extract_psm_features_for_percolator\"\
  ,\"q3Label\":\"extract_psm_features_for_percolator\"},\"time\":{\"mean\":3285,\"\
  min\":3285,\"q1\":3285,\"q2\":3285,\"q3\":3285,\"max\":3285,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"writes\":{\"mean\":11655798,\"min\":11655798,\"q1\":11655798,\"q2\":11655798,\"\
  q3\":11655798,\"max\":11655798,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  }},{\"cpuUsage\":{\"mean\":46,\"min\":46,\"q1\":46,\"q2\":46,\"q3\":46,\"max\":46,\"\
  minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\",\"q1Label\":\"run_percolator\"\
  ,\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"},\"process\":\"run_percolator\"\
  ,\"mem\":{\"mean\":95981568,\"min\":95981568,\"q1\":95981568,\"q2\":95981568,\"\
  q3\":95981568,\"max\":95981568,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\"\
  ,\"q1Label\":\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  },\"memUsage\":{\"mean\":1.49,\"min\":1.49,\"q1\":1.49,\"q2\":1.49,\"q3\":1.49,\"\
  max\":1.49,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\",\"q1Label\"\
  :\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  },\"timeUsage\":{\"mean\":0.01,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"q3\":0.01,\"\
  max\":0.01,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\",\"q1Label\"\
  :\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  },\"vmem\":{\"mean\":822321152,\"min\":822321152,\"q1\":822321152,\"q2\":822321152,\"\
  q3\":822321152,\"max\":822321152,\"minLabel\":\"run_percolator\",\"maxLabel\":\"\
  run_percolator\",\"q1Label\":\"run_percolator\",\"q2Label\":\"run_percolator\",\"\
  q3Label\":\"run_percolator\"},\"reads\":{\"mean\":17537849,\"min\":17537849,\"q1\"\
  :17537849,\"q2\":17537849,\"q3\":17537849,\"max\":17537849,\"minLabel\":\"run_percolator\"\
  ,\"maxLabel\":\"run_percolator\",\"q1Label\":\"run_percolator\",\"q2Label\":\"run_percolator\"\
  ,\"q3Label\":\"run_percolator\"},\"cpu\":{\"mean\":92,\"min\":92,\"q1\":92,\"q2\"\
  :92,\"q3\":92,\"max\":92,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\"\
  ,\"q1Label\":\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  },\"time\":{\"mean\":9392,\"min\":9392,\"q1\":9392,\"q2\":9392,\"q3\":9392,\"max\"\
  :9392,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\",\"q1Label\"\
  :\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  },\"writes\":{\"mean\":16530177,\"min\":16530177,\"q1\":16530177,\"q2\":16530177,\"\
  q3\":16530177,\"max\":16530177,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\"\
  ,\"q1Label\":\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  }},{\"cpuUsage\":{\"mean\":37.8,\"min\":37.8,\"q1\":37.8,\"q2\":37.8,\"q3\":37.8,\"\
  max\":37.8,\"minLabel\":\"filter_q_value\",\"maxLabel\":\"filter_q_value\",\"q1Label\"\
  :\"filter_q_value\",\"q2Label\":\"filter_q_value\",\"q3Label\":\"filter_q_value\"\
  },\"process\":\"filter_q_value\",\"mem\":{\"mean\":60792832,\"min\":60792832,\"\
  q1\":60792832,\"q2\":60792832,\"q3\":60792832,\"max\":60792832,\"minLabel\":\"filter_q_value\"\
  ,\"maxLabel\":\"filter_q_value\",\"q1Label\":\"filter_q_value\",\"q2Label\":\"filter_q_value\"\
  ,\"q3Label\":\"filter_q_value\"},\"memUsage\":{\"mean\":0.94,\"min\":0.94,\"q1\"\
  :0.94,\"q2\":0.94,\"q3\":0.94,\"max\":0.94,\"minLabel\":\"filter_q_value\",\"maxLabel\"\
  :\"filter_q_value\",\"q1Label\":\"filter_q_value\",\"q2Label\":\"filter_q_value\"\
  ,\"q3Label\":\"filter_q_value\"},\"timeUsage\":{\"mean\":0.05,\"min\":0.05,\"q1\"\
  :0.05,\"q2\":0.05,\"q3\":0.05,\"max\":0.05,\"minLabel\":\"filter_q_value\",\"maxLabel\"\
  :\"filter_q_value\",\"q1Label\":\"filter_q_value\",\"q2Label\":\"filter_q_value\"\
  ,\"q3Label\":\"filter_q_value\"},\"vmem\":{\"mean\":202928128,\"min\":202928128,\"\
  q1\":202928128,\"q2\":202928128,\"q3\":202928128,\"max\":202928128,\"minLabel\"\
  :\"filter_q_value\",\"maxLabel\":\"filter_q_value\",\"q1Label\":\"filter_q_value\"\
  ,\"q2Label\":\"filter_q_value\",\"q3Label\":\"filter_q_value\"},\"reads\":{\"mean\"\
  :16374778,\"min\":16374778,\"q1\":16374778,\"q2\":16374778,\"q3\":16374778,\"max\"\
  :16374778,\"minLabel\":\"filter_q_value\",\"maxLabel\":\"filter_q_value\",\"q1Label\"\
  :\"filter_q_value\",\"q2Label\":\"filter_q_value\",\"q3Label\":\"filter_q_value\"\
  },\"cpu\":{\"mean\":75.6,\"min\":75.6,\"q1\":75.6,\"q2\":75.6,\"q3\":75.6,\"max\"\
  :75.6,\"minLabel\":\"filter_q_value\",\"maxLabel\":\"filter_q_value\",\"q1Label\"\
  :\"filter_q_value\",\"q2Label\":\"filter_q_value\",\"q3Label\":\"filter_q_value\"\
  },\"time\":{\"mean\":3525,\"min\":3525,\"q1\":3525,\"q2\":3525,\"q3\":3525,\"max\"\
  :3525,\"minLabel\":\"filter_q_value\",\"maxLabel\":\"filter_q_value\",\"q1Label\"\
  :\"filter_q_value\",\"q2Label\":\"filter_q_value\",\"q3Label\":\"filter_q_value\"\
  },\"writes\":{\"mean\":1031971,\"min\":1031971,\"q1\":1031971,\"q2\":1031971,\"\
  q3\":1031971,\"max\":1031971,\"minLabel\":\"filter_q_value\",\"maxLabel\":\"filter_q_value\"\
  ,\"q1Label\":\"filter_q_value\",\"q2Label\":\"filter_q_value\",\"q3Label\":\"filter_q_value\"\
  }},{\"cpuUsage\":{\"mean\":36.62,\"min\":30.95,\"q1\":34.78,\"q2\":37.2,\"q3\":39.05,\"\
  max\":41.15,\"minLabel\":\"quantify_identifications_targeted (2)\",\"maxLabel\"\
  :\"quantify_identifications_targeted (4)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (2)\",\"q2Label\":\"quantify_identifications_targeted (1)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (3)\"},\"process\":\"quantify_identifications_targeted\",\"mem\":{\"mean\":101968896,\"\
  min\":97316864,\"q1\":97519616,\"q2\":99399680,\"q3\":103848960,\"max\":111759360,\"\
  minLabel\":\"quantify_identifications_targeted (2)\",\"maxLabel\":\"quantify_identifications_targeted\
  \ (1)\",\"q1Label\":\"quantify_identifications_targeted (2)\",\"q2Label\":\"quantify_identifications_targeted\
  \ (3)\",\"q3Label\":\"quantify_identifications_targeted (4)\"},\"memUsage\":{\"\
  mean\":1.58,\"min\":1.51,\"q1\":1.51,\"q2\":1.54,\"q3\":1.61,\"max\":1.73,\"minLabel\"\
  :\"quantify_identifications_targeted (2)\",\"maxLabel\":\"quantify_identifications_targeted\
  \ (1)\",\"q1Label\":\"quantify_identifications_targeted (2)\",\"q2Label\":\"quantify_identifications_targeted\
  \ (3)\",\"q3Label\":\"quantify_identifications_targeted (4)\"},\"timeUsage\":{\"\
  mean\":0.01,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"q3\":0.01,\"max\":0.01,\"minLabel\"\
  :\"quantify_identifications_targeted (3)\",\"maxLabel\":\"quantify_identifications_targeted\
  \ (2)\",\"q1Label\":\"quantify_identifications_targeted (3)\",\"q2Label\":\"quantify_identifications_targeted\
  \ (4)\",\"q3Label\":\"quantify_identifications_targeted (1)\"},\"vmem\":{\"mean\"\
  :389684224,\"min\":330092544,\"q1\":330304512,\"q2\":330446848,\"q3\":389826560,\"\
  max\":567750656,\"minLabel\":\"quantify_identifications_targeted (2)\",\"maxLabel\"\
  :\"quantify_identifications_targeted (1)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (2)\",\"q2Label\":\"quantify_identifications_targeted (3)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (4)\"},\"reads\":{\"mean\":30139401.25,\"min\":29331533,\"q1\":29624173.25,\"\
  q2\":30092860.5,\"q3\":30608088.5,\"max\":31040351,\"minLabel\":\"quantify_identifications_targeted\
  \ (1)\",\"maxLabel\":\"quantify_identifications_targeted (4)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (1)\",\"q2Label\":\"quantify_identifications_targeted (2)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (3)\"},\"cpu\":{\"mean\":73.25,\"min\":61.9,\"q1\":69.55,\"q2\":74.4,\"q3\":78.1,\"\
  max\":82.3,\"minLabel\":\"quantify_identifications_targeted (2)\",\"maxLabel\":\"\
  quantify_identifications_targeted (4)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (2)\",\"q2Label\":\"quantify_identifications_targeted (1)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (3)\"},\"time\":{\"mean\":8368.75,\"min\":6966,\"q1\":7073.25,\"q2\":8222.5,\"\
  q3\":9518,\"max\":10064,\"minLabel\":\"quantify_identifications_targeted (3)\",\"\
  maxLabel\":\"quantify_identifications_targeted (2)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (3)\",\"q2Label\":\"quantify_identifications_targeted (4)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (1)\"},\"writes\":{\"mean\":5452771,\"min\":5126920,\"q1\":5368278.25,\"q2\":5492269.5,\"\
  q3\":5576762.25,\"max\":5699625,\"minLabel\":\"quantify_identifications_targeted\
  \ (3)\",\"maxLabel\":\"quantify_identifications_targeted (4)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (3)\",\"q2Label\":\"quantify_identifications_targeted (2)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (1)\"}},{\"cpuUsage\":{\"mean\":34.4,\"min\":34.4,\"q1\":34.4,\"q2\":34.4,\"q3\"\
  :34.4,\"max\":34.4,\"minLabel\":\"link_extracted_features\",\"maxLabel\":\"link_extracted_features\"\
  ,\"q1Label\":\"link_extracted_features\",\"q2Label\":\"link_extracted_features\"\
  ,\"q3Label\":\"link_extracted_features\"},\"process\":\"link_extracted_features\"\
  ,\"mem\":{\"mean\":51679232,\"min\":51679232,\"q1\":51679232,\"q2\":51679232,\"\
  q3\":51679232,\"max\":51679232,\"minLabel\":\"link_extracted_features\",\"maxLabel\"\
  :\"link_extracted_features\",\"q1Label\":\"link_extracted_features\",\"q2Label\"\
  :\"link_extracted_features\",\"q3Label\":\"link_extracted_features\"},\"memUsage\"\
  :{\"mean\":0.8,\"min\":0.8,\"q1\":0.8,\"q2\":0.8,\"q3\":0.8,\"max\":0.8,\"minLabel\"\
  :\"link_extracted_features\",\"maxLabel\":\"link_extracted_features\",\"q1Label\"\
  :\"link_extracted_features\",\"q2Label\":\"link_extracted_features\",\"q3Label\"\
  :\"link_extracted_features\"},\"timeUsage\":{\"mean\":0.01,\"min\":0.01,\"q1\":0.01,\"\
  q2\":0.01,\"q3\":0.01,\"max\":0.01,\"minLabel\":\"link_extracted_features\",\"maxLabel\"\
  :\"link_extracted_features\",\"q1Label\":\"link_extracted_features\",\"q2Label\"\
  :\"link_extracted_features\",\"q3Label\":\"link_extracted_features\"},\"vmem\":{\"\
  mean\":556359680,\"min\":556359680,\"q1\":556359680,\"q2\":556359680,\"q3\":556359680,\"\
  max\":556359680,\"minLabel\":\"link_extracted_features\",\"maxLabel\":\"link_extracted_features\"\
  ,\"q1Label\":\"link_extracted_features\",\"q2Label\":\"link_extracted_features\"\
  ,\"q3Label\":\"link_extracted_features\"},\"reads\":{\"mean\":25265069,\"min\":25265069,\"\
  q1\":25265069,\"q2\":25265069,\"q3\":25265069,\"max\":25265069,\"minLabel\":\"link_extracted_features\"\
  ,\"maxLabel\":\"link_extracted_features\",\"q1Label\":\"link_extracted_features\"\
  ,\"q2Label\":\"link_extracted_features\",\"q3Label\":\"link_extracted_features\"\
  },\"cpu\":{\"mean\":68.8,\"min\":68.8,\"q1\":68.8,\"q2\":68.8,\"q3\":68.8,\"max\"\
  :68.8,\"minLabel\":\"link_extracted_features\",\"maxLabel\":\"link_extracted_features\"\
  ,\"q1Label\":\"link_extracted_features\",\"q2Label\":\"link_extracted_features\"\
  ,\"q3Label\":\"link_extracted_features\"},\"time\":{\"mean\":4579,\"min\":4579,\"\
  q1\":4579,\"q2\":4579,\"q3\":4579,\"max\":4579,\"minLabel\":\"link_extracted_features\"\
  ,\"maxLabel\":\"link_extracted_features\",\"q1Label\":\"link_extracted_features\"\
  ,\"q2Label\":\"link_extracted_features\",\"q3Label\":\"link_extracted_features\"\
  },\"writes\":{\"mean\":4826047,\"min\":4826047,\"q1\":4826047,\"q2\":4826047,\"\
  q3\":4826047,\"max\":4826047,\"minLabel\":\"link_extracted_features\",\"maxLabel\"\
  :\"link_extracted_features\",\"q1Label\":\"link_extracted_features\",\"q2Label\"\
  :\"link_extracted_features\",\"q3Label\":\"link_extracted_features\"}},{\"cpuUsage\"\
  :{\"mean\":27.8,\"min\":27.8,\"q1\":27.8,\"q2\":27.8,\"q3\":27.8,\"max\":27.8,\"\
  minLabel\":\"resolve_conflicts\",\"maxLabel\":\"resolve_conflicts\",\"q1Label\"\
  :\"resolve_conflicts\",\"q2Label\":\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"\
  },\"process\":\"resolve_conflicts\",\"mem\":{\"mean\":51884032,\"min\":51884032,\"\
  q1\":51884032,\"q2\":51884032,\"q3\":51884032,\"max\":51884032,\"minLabel\":\"resolve_conflicts\"\
  ,\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\",\"q2Label\"\
  :\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"},\"memUsage\":{\"mean\"\
  :0.81,\"min\":0.81,\"q1\":0.81,\"q2\":0.81,\"q3\":0.81,\"max\":0.81,\"minLabel\"\
  :\"resolve_conflicts\",\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\"\
  ,\"q2Label\":\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"},\"timeUsage\"\
  :{\"mean\":0.04,\"min\":0.04,\"q1\":0.04,\"q2\":0.04,\"q3\":0.04,\"max\":0.04,\"\
  minLabel\":\"resolve_conflicts\",\"maxLabel\":\"resolve_conflicts\",\"q1Label\"\
  :\"resolve_conflicts\",\"q2Label\":\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"\
  },\"vmem\":{\"mean\":556310528,\"min\":556310528,\"q1\":556310528,\"q2\":556310528,\"\
  q3\":556310528,\"max\":556310528,\"minLabel\":\"resolve_conflicts\",\"maxLabel\"\
  :\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\",\"q2Label\":\"resolve_conflicts\"\
  ,\"q3Label\":\"resolve_conflicts\"},\"reads\":{\"mean\":8214260,\"min\":8214260,\"\
  q1\":8214260,\"q2\":8214260,\"q3\":8214260,\"max\":8214260,\"minLabel\":\"resolve_conflicts\"\
  ,\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\",\"q2Label\"\
  :\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"},\"cpu\":{\"mean\":55.6,\"\
  min\":55.6,\"q1\":55.6,\"q2\":55.6,\"q3\":55.6,\"max\":55.6,\"minLabel\":\"resolve_conflicts\"\
  ,\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\",\"q2Label\"\
  :\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"},\"time\":{\"mean\":2973,\"\
  min\":2973,\"q1\":2973,\"q2\":2973,\"q3\":2973,\"max\":2973,\"minLabel\":\"resolve_conflicts\"\
  ,\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\",\"q2Label\"\
  :\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"},\"writes\":{\"mean\":4980833,\"\
  min\":4980833,\"q1\":4980833,\"q2\":4980833,\"q3\":4980833,\"max\":4980833,\"minLabel\"\
  :\"resolve_conflicts\",\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\"\
  ,\"q2Label\":\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"}},{\"cpuUsage\"\
  :{\"mean\":22.3,\"min\":22.3,\"q1\":22.3,\"q2\":22.3,\"q3\":22.3,\"max\":22.3,\"\
  minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\",\"q1Label\":\"export_mztab\"\
  ,\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"},\"process\":\"export_mztab\"\
  ,\"mem\":{\"mean\":47886336,\"min\":47886336,\"q1\":47886336,\"q2\":47886336,\"\
  q3\":47886336,\"max\":47886336,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\"\
  ,\"q1Label\":\"export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"\
  },\"memUsage\":{\"mean\":0.74,\"min\":0.74,\"q1\":0.74,\"q2\":0.74,\"q3\":0.74,\"\
  max\":0.74,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\",\"q1Label\"\
  :\"export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"},\"\
  timeUsage\":{\"mean\":0.05,\"min\":0.05,\"q1\":0.05,\"q2\":0.05,\"q3\":0.05,\"max\"\
  :0.05,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\",\"q1Label\":\"\
  export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"},\"vmem\"\
  :{\"mean\":556503040,\"min\":556503040,\"q1\":556503040,\"q2\":556503040,\"q3\"\
  :556503040,\"max\":556503040,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\"\
  ,\"q1Label\":\"export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"\
  },\"reads\":{\"mean\":8369554,\"min\":8369554,\"q1\":8369554,\"q2\":8369554,\"q3\"\
  :8369554,\"max\":8369554,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\"\
  ,\"q1Label\":\"export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"\
  },\"cpu\":{\"mean\":44.6,\"min\":44.6,\"q1\":44.6,\"q2\":44.6,\"q3\":44.6,\"max\"\
  :44.6,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\",\"q1Label\":\"\
  export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"},\"time\"\
  :{\"mean\":3420,\"min\":3420,\"q1\":3420,\"q2\":3420,\"q3\":3420,\"max\":3420,\"\
  minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\",\"q1Label\":\"export_mztab\"\
  ,\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"},\"writes\":{\"mean\"\
  :191147,\"min\":191147,\"q1\":191147,\"q2\":191147,\"q3\":191147,\"max\":191147,\"\
  minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\",\"q1Label\":\"export_mztab\"\
  ,\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"}},{\"cpuUsage\":{\"mean\"\
  :21.45,\"min\":21.45,\"q1\":21.45,\"q2\":21.45,\"q3\":21.45,\"max\":21.45,\"minLabel\"\
  :\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\":\"export_text\",\"q2Label\"\
  :\"export_text\",\"q3Label\":\"export_text\"},\"process\":\"export_text\",\"mem\"\
  :{\"mean\":48459776,\"min\":48459776,\"q1\":48459776,\"q2\":48459776,\"q3\":48459776,\"\
  max\":48459776,\"minLabel\":\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\"\
  :\"export_text\",\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"},\"memUsage\"\
  :{\"mean\":0.75,\"min\":0.75,\"q1\":0.75,\"q2\":0.75,\"q3\":0.75,\"max\":0.75,\"\
  minLabel\":\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\":\"export_text\"\
  ,\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"},\"timeUsage\":{\"mean\"\
  :0.05,\"min\":0.05,\"q1\":0.05,\"q2\":0.05,\"q3\":0.05,\"max\":0.05,\"minLabel\"\
  :\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\":\"export_text\",\"q2Label\"\
  :\"export_text\",\"q3Label\":\"export_text\"},\"vmem\":{\"mean\":556429312,\"min\"\
  :556429312,\"q1\":556429312,\"q2\":556429312,\"q3\":556429312,\"max\":556429312,\"\
  minLabel\":\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\":\"export_text\"\
  ,\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"},\"reads\":{\"mean\":8369649,\"\
  min\":8369649,\"q1\":8369649,\"q2\":8369649,\"q3\":8369649,\"max\":8369649,\"minLabel\"\
  :\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\":\"export_text\",\"q2Label\"\
  :\"export_text\",\"q3Label\":\"export_text\"},\"cpu\":{\"mean\":42.9,\"min\":42.9,\"\
  q1\":42.9,\"q2\":42.9,\"q3\":42.9,\"max\":42.9,\"minLabel\":\"export_text\",\"maxLabel\"\
  :\"export_text\",\"q1Label\":\"export_text\",\"q2Label\":\"export_text\",\"q3Label\"\
  :\"export_text\"},\"time\":{\"mean\":3514,\"min\":3514,\"q1\":3514,\"q2\":3514,\"\
  q3\":3514,\"max\":3514,\"minLabel\":\"export_text\",\"maxLabel\":\"export_text\"\
  ,\"q1Label\":\"export_text\",\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"\
  },\"writes\":{\"mean\":1021262,\"min\":1021262,\"q1\":1021262,\"q2\":1021262,\"\
  q3\":1021262,\"max\":1021262,\"minLabel\":\"export_text\",\"maxLabel\":\"export_text\"\
  ,\"q1Label\":\"export_text\",\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"\
  }}]\nFeb-19 04:00:29.245 [main] DEBUG nextflow.trace.TimelineObserver - Workflow\
  \ completed -- rendering execution timeline\nFeb-19 04:00:29.682 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-19 04:00:29.802 [main] DEBUG nextflow.util.ThreadPoolManager\
  \ - Thread pool 'FileTransfer' shutdown completed (hard=false)\nFeb-19 04:00:30.132\
  \ [main] DEBUG nextflow.script.ScriptRunner - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2F0lfg0v1p%2Flogs.tar.xz">0.078GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2F0lfg0v1p%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.694GiB</td>
						<td>2688.2 seconds</td>
						<td>1757.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743108064">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.0.4">2.0.4</a>
						</td>
						<td>nextflow</td>
						<td>1502 days</td>
						<td>2023-02-19 04:02:53</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 04:03:10.751
kind: NextflowMiscError
process: main
rest: Please specify either --singleEnd or --pairedEnd to execute the pipeline!

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fip2dq775%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fip2dq775%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.532GiB</td>
						<td>21.9 seconds</td>
						<td>17.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743144672">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.2.0">1.2.0</a>
						</td>
						<td>nextflow</td>
						<td>1490 days</td>
						<td>2023-02-19 04:41:46</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.cli.Launcher
date: Feb-19 04:41:57.430
kind: NextflowMiscError
process: main
rest: Script compilation error

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fm61rl1p4%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fm61rl1p4%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.437GiB</td>
						<td>17.5 seconds</td>
						<td>11.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743144432">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.2.1">1.2.1</a>
						</td>
						<td>nextflow</td>
						<td>1486 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.io.IOException
msg: 'Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv'
rest: "at nextflow.file.http.XFileSystemProvider.readAttributes(XFileSystemProvider.groovy:463)\n\
  \tat java.base/java.nio.file.Files.readAttributes(Files.java:1764)\n\tat nextflow.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:156)\n\
  \tat nextflow.file.FileHelper.copyPath(FileHelper.groovy:971)\n\tat nextflow.file.FilePorter$FileTransfer.stageForeignFile0(FilePorter.groovy:299)\n\
  \tat nextflow.file.FilePorter$FileTransfer.stageForeignFile(FilePorter.groovy:266)\n\
  \tat nextflow.file.FilePorter$FileTransfer.run(FilePorter.groovy:252)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\
  \tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-19 04:03:42.526 [FileTransfer-2]\
  \ DEBUG nextflow.file.FilePorter - Copying foreign file https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ to work dir: /home/azureuser/tmp/g089ah8qkt/code/work/stage/f5/072f8c116a1f544f5d80d31c8d60c7/alleles.tsv\n\
  Feb-19 04:03:42.548 [FileTransfer-2] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 250ms (attempt=1)\nFeb-19 04:03:42.814 [FileTransfer-2]\
  \ DEBUG n.file.http.XFileSystemProvider - Got HTTP error=404 waiting for 750ms (attempt=2)\n\
  Feb-19 04:03:43.596 [FileTransfer-2] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 2250ms (attempt=3)\nFeb-19 04:03:45.861 [FileTransfer-2]\
  \ WARN  nextflow.file.FilePorter - Unable to stage foreign file: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ (try 2) -- Cause: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\n\
  java.io.IOException: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\n\
  \tat nextflow.file.http.XFileSystemProvider.readAttributes(XFileSystemProvider.groovy:463)\n\
  \tat java.base/java.nio.file.Files.readAttributes(Files.java:1764)\n\tat nextflow.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:156)\n\
  \tat nextflow.file.FileHelper.copyPath(FileHelper.groovy:971)\n\tat nextflow.file.FilePorter$FileTransfer.stageForeignFile0(FilePorter.groovy:299)\n\
  \tat nextflow.file.FilePorter$FileTransfer.stageForeignFile(FilePorter.groovy:266)\n\
  \tat nextflow.file.FilePorter$FileTransfer.run(FilePorter.groovy:252)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\
  \tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-19 04:03:45.991 [FileTransfer-2]\
  \ DEBUG nextflow.file.FilePorter - Copying foreign file https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ to work dir: /home/azureuser/tmp/g089ah8qkt/code/work/stage/f5/072f8c116a1f544f5d80d31c8d60c7/alleles.tsv\n\
  Feb-19 04:03:46.022 [FileTransfer-2] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 250ms (attempt=1)\nFeb-19 04:03:46.356 [FileTransfer-2]\
  \ DEBUG n.file.http.XFileSystemProvider - Got HTTP error=404 waiting for 750ms (attempt=2)\n\
  Feb-19 04:03:47.123 [FileTransfer-2] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 2250ms (attempt=3)\nFeb-19 04:03:49.396 [FileTransfer-2]\
  \ WARN  nextflow.file.FilePorter - Unable to stage foreign file: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ (try 3) -- Cause: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\n\
  java.io.IOException: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\n\
  \tat nextflow.file.http.XFileSystemProvider.readAttributes(XFileSystemProvider.groovy:463)\n\
  \tat java.base/java.nio.file.Files.readAttributes(Files.java:1764)\n\tat nextflow.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:156)\n\
  \tat nextflow.file.FileHelper.copyPath(FileHelper.groovy:971)\n\tat nextflow.file.FilePorter$FileTransfer.stageForeignFile0(FilePorter.groovy:299)\n\
  \tat nextflow.file.FilePorter$FileTransfer.stageForeignFile(FilePorter.groovy:266)\n\
  \tat nextflow.file.FilePorter$FileTransfer.run(FilePorter.groovy:252)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\
  \tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-19 04:03:49.527 [FileTransfer-2]\
  \ DEBUG nextflow.file.FilePorter - Copying foreign file https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ to work dir: /home/azureuser/tmp/g089ah8qkt/code/work/stage/f5/072f8c116a1f544f5d80d31c8d60c7/alleles.tsv\n\
  Feb-19 04:03:49.542 [FileTransfer-2] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 250ms (attempt=1)\nFeb-19 04:03:49.806 [FileTransfer-2]\
  \ DEBUG n.file.http.XFileSystemProvider - Got HTTP error=404 waiting for 750ms (attempt=2)\n\
  Feb-19 04:03:50.570 [FileTransfer-2] DEBUG n.file.http.XFileSystemProvider - Got\
  \ HTTP error=404 waiting for 2250ms (attempt=3)\nFeb-19 04:03:52.859 [Actor Thread\
  \ 14] ERROR nextflow.processor.TaskProcessor - Error executing process > 'predict_peptides\
  \ (1)'\n\nCaused by:\n  Can't stage file https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\
  \ -- reason: Unable to access path: https://github.com/nf-core/test-datasets/raw/mhcquant/alleles.tsv\n\
  \n\nTip: you can try to figure out what's wrong by changing to the process work\
  \ dir and showing the script file named `.command.sh`\nFeb-19 04:03:52.862 [Actor\
  \ Thread 14] INFO  nextflow.Session - Execution cancelled -- Finishing pending tasks\
  \ before exit\nFeb-19 04:03:52.882 [main] DEBUG nextflow.Session - Session await\
  \ > all processes finished\nFeb-19 04:03:52.882 [main] DEBUG nextflow.Session -\
  \ Session await > all barriers passed\nFeb-19 04:03:52.885 [main] DEBUG nextflow.util.ThreadPoolManager\
  \ - Thread pool 'PublishDir' shutdown completed (hard=false)\nFeb-19 04:03:52.992\
  \ [main] INFO  nextflow.Nextflow - [nf-core/mhcquant] Pipeline Complete\nFeb-19\
  \ 04:03:52.996 [main] DEBUG nextflow.trace.WorkflowStatsObserver - Workflow completed\
  \ > WorkflowStats[succeededCount=40; failedCount=0; ignoredCount=0; cachedCount=0;\
  \ pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0; abortedCount=0;\
  \ succeedDuration=1h 32m 52s; failedDuration=0ms; cachedDuration=0ms;loadCpus=0;\
  \ loadMemory=0; peakRunning=2; peakCpus=4; peakMemory=12 GB; ]\nFeb-19 04:03:52.996\
  \ [main] DEBUG nextflow.trace.TraceFileObserver - Workflow completed -- saving trace\
  \ file\nFeb-19 04:03:53.005 [main] DEBUG nextflow.trace.ReportObserver - Workflow\
  \ completed -- rendering execution report\nFeb-19 04:03:53.171 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Execution report summary data:\n  [{\"cpuUsage\":{\"mean\":13.6,\"min\":13.6,\"\
  q1\":13.6,\"q2\":13.6,\"q3\":13.6,\"max\":13.6,\"minLabel\":\"output_documentation\"\
  ,\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\"\
  :\"output_documentation\",\"q3Label\":\"output_documentation\"},\"process\":\"output_documentation\"\
  ,\"mem\":{\"mean\":7446528,\"min\":7446528,\"q1\":7446528,\"q2\":7446528,\"q3\"\
  :7446528,\"max\":7446528,\"minLabel\":\"output_documentation\",\"maxLabel\":\"output_documentation\"\
  ,\"q1Label\":\"output_documentation\",\"q2Label\":\"output_documentation\",\"q3Label\"\
  :\"output_documentation\"},\"memUsage\":{\"mean\":0.12,\"min\":0.12,\"q1\":0.12,\"\
  q2\":0.12,\"q3\":0.12,\"max\":0.12,\"minLabel\":\"output_documentation\",\"maxLabel\"\
  :\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\":\"output_documentation\"\
  ,\"q3Label\":\"output_documentation\"},\"timeUsage\":{\"mean\":0.03,\"min\":0.03,\"\
  q1\":0.03,\"q2\":0.03,\"q3\":0.03,\"max\":0.03,\"minLabel\":\"output_documentation\"\
  ,\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\"\
  :\"output_documentation\",\"q3Label\":\"output_documentation\"},\"vmem\":{\"mean\"\
  :108957696,\"min\":108957696,\"q1\":108957696,\"q2\":108957696,\"q3\":108957696,\"\
  max\":108957696,\"minLabel\":\"output_documentation\",\"maxLabel\":\"output_documentation\"\
  ,\"q1Label\":\"output_documentation\",\"q2Label\":\"output_documentation\",\"q3Label\"\
  :\"output_documentation\"},\"reads\":{\"mean\":5775098,\"min\":5775098,\"q1\":5775098,\"\
  q2\":5775098,\"q3\":5775098,\"max\":5775098,\"minLabel\":\"output_documentation\"\
  ,\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\"\
  :\"output_documentation\",\"q3Label\":\"output_documentation\"},\"cpu\":{\"mean\"\
  :27.2,\"min\":27.2,\"q1\":27.2,\"q2\":27.2,\"q3\":27.2,\"max\":27.2,\"minLabel\"\
  :\"output_documentation\",\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\"\
  ,\"q2Label\":\"output_documentation\",\"q3Label\":\"output_documentation\"},\"time\"\
  :{\"mean\":1820,\"min\":1820,\"q1\":1820,\"q2\":1820,\"q3\":1820,\"max\":1820,\"\
  minLabel\":\"output_documentation\",\"maxLabel\":\"output_documentation\",\"q1Label\"\
  :\"output_documentation\",\"q2Label\":\"output_documentation\",\"q3Label\":\"output_documentation\"\
  },\"writes\":{\"mean\":7036,\"min\":7036,\"q1\":7036,\"q2\":7036,\"q3\":7036,\"\
  max\":7036,\"minLabel\":\"output_documentation\",\"maxLabel\":\"output_documentation\"\
  ,\"q1Label\":\"output_documentation\",\"q2Label\":\"output_documentation\",\"q3Label\"\
  :\"output_documentation\"}},{\"cpuUsage\":{\"mean\":9.5,\"min\":9.5,\"q1\":9.5,\"\
  q2\":9.5,\"q3\":9.5,\"max\":9.5,\"minLabel\":\"generate_proteins_from_vcf (1)\"\
  ,\"maxLabel\":\"generate_proteins_from_vcf (1)\",\"q1Label\":\"generate_proteins_from_vcf\
  \ (1)\",\"q2Label\":\"generate_proteins_from_vcf (1)\",\"q3Label\":\"generate_proteins_from_vcf\
  \ (1)\"},\"process\":\"generate_proteins_from_vcf\",\"mem\":{\"mean\":83947520,\"\
  min\":83947520,\"q1\":83947520,\"q2\":83947520,\"q3\":83947520,\"max\":83947520,\"\
  minLabel\":\"generate_proteins_from_vcf (1)\",\"maxLabel\":\"generate_proteins_from_vcf\
  \ (1)\",\"q1Label\":\"generate_proteins_from_vcf (1)\",\"q2Label\":\"generate_proteins_from_vcf\
  \ (1)\",\"q3Label\":\"generate_proteins_from_vcf (1)\"},\"memUsage\":{\"mean\":1.3,\"\
  min\":1.3,\"q1\":1.3,\"q2\":1.3,\"q3\":1.3,\"max\":1.3,\"minLabel\":\"generate_proteins_from_vcf\
  \ (1)\",\"maxLabel\":\"generate_proteins_from_vcf (1)\",\"q1Label\":\"generate_proteins_from_vcf\
  \ (1)\",\"q2Label\":\"generate_proteins_from_vcf (1)\",\"q3Label\":\"generate_proteins_from_vcf\
  \ (1)\"},\"timeUsage\":{\"mean\":0.08,\"min\":0.08,\"q1\":0.08,\"q2\":0.08,\"q3\"\
  :0.08,\"max\":0.08,\"minLabel\":\"generate_proteins_from_vcf (1)\",\"maxLabel\"\
  :\"generate_proteins_from_vcf (1)\",\"q1Label\":\"generate_proteins_from_vcf (1)\"\
  ,\"q2Label\":\"generate_proteins_from_vcf (1)\",\"q3Label\":\"generate_proteins_from_vcf\
  \ (1)\"},\"vmem\":{\"mean\":311312384,\"min\":311312384,\"q1\":311312384,\"q2\"\
  :311312384,\"q3\":311312384,\"max\":311312384,\"minLabel\":\"generate_proteins_from_vcf\
  \ (1)\",\"maxLabel\":\"generate_proteins_from_vcf (1)\",\"q1Label\":\"generate_proteins_from_vcf\
  \ (1)\",\"q2Label\":\"generate_proteins_from_vcf (1)\",\"q3Label\":\"generate_proteins_from_vcf\
  \ (1)\"},\"reads\":{\"mean\":25806475,\"min\":25806475,\"q1\":25806475,\"q2\":25806475,\"\
  q3\":25806475,\"max\":25806475,\"minLabel\":\"generate_proteins_from_vcf (1)\",\"\
  maxLabel\":\"generate_proteins_from_vcf (1)\",\"q1Label\":\"generate_proteins_from_vcf\
  \ (1)\",\"q2Label\":\"generate_proteins_from_vcf (1)\",\"q3Label\":\"generate_proteins_from_vcf\
  \ (1)\"},\"cpu\":{\"mean\":19,\"min\":19,\"q1\":19,\"q2\":19,\"q3\":19,\"max\":19,\"\
  minLabel\":\"generate_proteins_from_vcf (1)\",\"maxLabel\":\"generate_proteins_from_vcf\
  \ (1)\",\"q1Label\":\"generate_proteins_from_vcf (1)\",\"q2Label\":\"generate_proteins_from_vcf\
  \ (1)\",\"q3Label\":\"generate_proteins_from_vcf (1)\"},\"time\":{\"mean\":5871,\"\
  min\":5871,\"q1\":5871,\"q2\":5871,\"q3\":5871,\"max\":5871,\"minLabel\":\"generate_proteins_from_vcf\
  \ (1)\",\"maxLabel\":\"generate_proteins_from_vcf (1)\",\"q1Label\":\"generate_proteins_from_vcf\
  \ (1)\",\"q2Label\":\"generate_proteins_from_vcf (1)\",\"q3Label\":\"generate_proteins_from_vcf\
  \ (1)\"},\"writes\":{\"mean\":13394572,\"min\":13394572,\"q1\":13394572,\"q2\":13394572,\"\
  q3\":13394572,\"max\":13394572,\"minLabel\":\"generate_proteins_from_vcf (1)\",\"\
  maxLabel\":\"generate_proteins_from_vcf (1)\",\"q1Label\":\"generate_proteins_from_vcf\
  \ (1)\",\"q2Label\":\"generate_proteins_from_vcf (1)\",\"q3Label\":\"generate_proteins_from_vcf\
  \ (1)\"}},{\"cpuUsage\":{\"mean\":20.7,\"min\":20.7,\"q1\":20.7,\"q2\":20.7,\"q3\"\
  :20.7,\"max\":20.7,\"minLabel\":\"generate_decoy_database (1)\",\"maxLabel\":\"\
  generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"q2Label\"\
  :\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"\
  process\":\"generate_decoy_database\",\"mem\":{\"mean\":35647488,\"min\":35647488,\"\
  q1\":35647488,\"q2\":35647488,\"q3\":35647488,\"max\":35647488,\"minLabel\":\"generate_decoy_database\
  \ (1)\",\"maxLabel\":\"generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database\
  \ (1)\",\"q2Label\":\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database\
  \ (1)\"},\"memUsage\":{\"mean\":0.55,\"min\":0.55,\"q1\":0.55,\"q2\":0.55,\"q3\"\
  :0.55,\"max\":0.55,\"minLabel\":\"generate_decoy_database (1)\",\"maxLabel\":\"\
  generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"q2Label\"\
  :\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"\
  timeUsage\":{\"mean\":0.19,\"min\":0.19,\"q1\":0.19,\"q2\":0.19,\"q3\":0.19,\"max\"\
  :0.19,\"minLabel\":\"generate_decoy_database (1)\",\"maxLabel\":\"generate_decoy_database\
  \ (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"q2Label\":\"generate_decoy_database\
  \ (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"vmem\":{\"mean\":132689920,\"\
  min\":132689920,\"q1\":132689920,\"q2\":132689920,\"q3\":132689920,\"max\":132689920,\"\
  minLabel\":\"generate_decoy_database (1)\",\"maxLabel\":\"generate_decoy_database\
  \ (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"q2Label\":\"generate_decoy_database\
  \ (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"reads\":{\"mean\":16758181,\"\
  min\":16758181,\"q1\":16758181,\"q2\":16758181,\"q3\":16758181,\"max\":16758181,\"\
  minLabel\":\"generate_decoy_database (1)\",\"maxLabel\":\"generate_decoy_database\
  \ (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"q2Label\":\"generate_decoy_database\
  \ (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"cpu\":{\"mean\":41.4,\"min\"\
  :41.4,\"q1\":41.4,\"q2\":41.4,\"q3\":41.4,\"max\":41.4,\"minLabel\":\"generate_decoy_database\
  \ (1)\",\"maxLabel\":\"generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database\
  \ (1)\",\"q2Label\":\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database\
  \ (1)\"},\"time\":{\"mean\":13392,\"min\":13392,\"q1\":13392,\"q2\":13392,\"q3\"\
  :13392,\"max\":13392,\"minLabel\":\"generate_decoy_database (1)\",\"maxLabel\":\"\
  generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"q2Label\"\
  :\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database (1)\"},\"\
  writes\":{\"mean\":26807501,\"min\":26807501,\"q1\":26807501,\"q2\":26807501,\"\
  q3\":26807501,\"max\":26807501,\"minLabel\":\"generate_decoy_database (1)\",\"maxLabel\"\
  :\"generate_decoy_database (1)\",\"q1Label\":\"generate_decoy_database (1)\",\"\
  q2Label\":\"generate_decoy_database (1)\",\"q3Label\":\"generate_decoy_database\
  \ (1)\"}},{\"cpuUsage\":{\"mean\":35.36,\"min\":30.3,\"q1\":30.41,\"q2\":35.25,\"\
  q3\":40.2,\"max\":40.65,\"minLabel\":\"db_search_comet (4)\",\"maxLabel\":\"db_search_comet\
  \ (2)\",\"q1Label\":\"db_search_comet (4)\",\"q2Label\":\"db_search_comet (3)\"\
  ,\"q3Label\":\"db_search_comet (1)\"},\"process\":\"db_search_comet\",\"mem\":{\"\
  mean\":316678144,\"min\":262938624,\"q1\":314566656,\"q2\":333621248,\"q3\":335732736,\"\
  max\":336531456,\"minLabel\":\"db_search_comet (2)\",\"maxLabel\":\"db_search_comet\
  \ (3)\",\"q1Label\":\"db_search_comet (2)\",\"q2Label\":\"db_search_comet (1)\"\
  ,\"q3Label\":\"db_search_comet (4)\"},\"memUsage\":{\"mean\":4.92,\"min\":4.08,\"\
  q1\":4.88,\"q2\":5.18,\"q3\":5.21,\"max\":5.22,\"minLabel\":\"db_search_comet (2)\"\
  ,\"maxLabel\":\"db_search_comet (3)\",\"q1Label\":\"db_search_comet (2)\",\"q2Label\"\
  :\"db_search_comet (1)\",\"q3Label\":\"db_search_comet (4)\"},\"timeUsage\":{\"\
  mean\":0.77,\"min\":0.62,\"q1\":0.64,\"q2\":0.77,\"q3\":0.89,\"max\":0.91,\"minLabel\"\
  :\"db_search_comet (1)\",\"maxLabel\":\"db_search_comet (4)\",\"q1Label\":\"db_search_comet\
  \ (1)\",\"q2Label\":\"db_search_comet (2)\",\"q3Label\":\"db_search_comet (3)\"\
  },\"vmem\":{\"mean\":869655552,\"min\":752295936,\"q1\":852848640,\"q2\":886413312,\"\
  q3\":903220224,\"max\":953499648,\"minLabel\":\"db_search_comet (2)\",\"maxLabel\"\
  :\"db_search_comet (3)\",\"q1Label\":\"db_search_comet (2)\",\"q2Label\":\"db_search_comet\
  \ (4)\",\"q3Label\":\"db_search_comet (1)\"},\"reads\":{\"mean\":307203514.75,\"\
  min\":301084227,\"q1\":303320079.75,\"q2\":307113340.5,\"q3\":310996775.5,\"max\"\
  :313503151,\"minLabel\":\"db_search_comet (4)\",\"maxLabel\":\"db_search_comet (2)\"\
  ,\"q1Label\":\"db_search_comet (4)\",\"q2Label\":\"db_search_comet (3)\",\"q3Label\"\
  :\"db_search_comet (1)\"},\"cpu\":{\"mean\":70.72,\"min\":60.6,\"q1\":60.83,\"q2\"\
  :70.5,\"q3\":80.4,\"max\":81.3,\"minLabel\":\"db_search_comet (4)\",\"maxLabel\"\
  :\"db_search_comet (2)\",\"q1Label\":\"db_search_comet (4)\",\"q2Label\":\"db_search_comet\
  \ (3)\",\"q3Label\":\"db_search_comet (1)\"},\"time\":{\"mean\":663231.5,\"min\"\
  :537536,\"q1\":556694,\"q2\":665182,\"q3\":771719.5,\"max\":785026,\"minLabel\"\
  :\"db_search_comet (1)\",\"maxLabel\":\"db_search_comet (4)\",\"q1Label\":\"db_search_comet\
  \ (1)\",\"q2Label\":\"db_search_comet (2)\",\"q3Label\":\"db_search_comet (3)\"\
  },\"writes\":{\"mean\":3496145,\"min\":3249877,\"q1\":3306580,\"q2\":3505860.5,\"\
  q3\":3695425.5,\"max\":3722982,\"minLabel\":\"db_search_comet (4)\",\"maxLabel\"\
  :\"db_search_comet (2)\",\"q1Label\":\"db_search_comet (4)\",\"q2Label\":\"db_search_comet\
  \ (3)\",\"q3Label\":\"db_search_comet (1)\"}},{\"cpuUsage\":{\"mean\":35.56,\"min\"\
  :23.45,\"q1\":31.96,\"q2\":36.7,\"q3\":40.3,\"max\":45.4,\"minLabel\":\"index_peptides\
  \ (1)\",\"maxLabel\":\"index_peptides (4)\",\"q1Label\":\"index_peptides (1)\",\"\
  q2Label\":\"index_peptides (2)\",\"q3Label\":\"index_peptides (3)\"},\"process\"\
  :\"index_peptides\",\"mem\":{\"mean\":71135232,\"min\":60461056,\"q1\":62679040,\"\
  q2\":70029312,\"q3\":78485504,\"max\":84021248,\"minLabel\":\"index_peptides (3)\"\
  ,\"maxLabel\":\"index_peptides (1)\",\"q1Label\":\"index_peptides (3)\",\"q2Label\"\
  :\"index_peptides (2)\",\"q3Label\":\"index_peptides (4)\"},\"memUsage\":{\"mean\"\
  :1.1,\"min\":0.94,\"q1\":0.97,\"q2\":1.09,\"q3\":1.22,\"max\":1.3,\"minLabel\":\"\
  index_peptides (3)\",\"maxLabel\":\"index_peptides (1)\",\"q1Label\":\"index_peptides\
  \ (3)\",\"q2Label\":\"index_peptides (2)\",\"q3Label\":\"index_peptides (4)\"},\"\
  timeUsage\":{\"mean\":0.04,\"min\":0.03,\"q1\":0.03,\"q2\":0.03,\"q3\":0.04,\"max\"\
  :0.05,\"minLabel\":\"index_peptides (4)\",\"maxLabel\":\"index_peptides (1)\",\"\
  q1Label\":\"index_peptides (4)\",\"q2Label\":\"index_peptides (3)\",\"q3Label\"\
  :\"index_peptides (2)\"},\"vmem\":{\"mean\":290391040,\"min\":165351424,\"q1\":167981056,\"\
  q2\":245868544,\"q3\":368278528,\"max\":504475648,\"minLabel\":\"index_peptides\
  \ (3)\",\"maxLabel\":\"index_peptides (1)\",\"q1Label\":\"index_peptides (3)\",\"\
  q2Label\":\"index_peptides (2)\",\"q3Label\":\"index_peptides (4)\"},\"reads\":{\"\
  mean\":31953341.75,\"min\":31819846,\"q1\":31868675.5,\"q2\":31964593.5,\"q3\":32049259.75,\"\
  max\":32064334,\"minLabel\":\"index_peptides (2)\",\"maxLabel\":\"index_peptides\
  \ (4)\",\"q1Label\":\"index_peptides (2)\",\"q2Label\":\"index_peptides (1)\",\"\
  q3Label\":\"index_peptides (3)\"},\"cpu\":{\"mean\":71.13,\"min\":46.9,\"q1\":63.93,\"\
  q2\":73.4,\"q3\":80.6,\"max\":90.8,\"minLabel\":\"index_peptides (1)\",\"maxLabel\"\
  :\"index_peptides (4)\",\"q1Label\":\"index_peptides (1)\",\"q2Label\":\"index_peptides\
  \ (2)\",\"q3Label\":\"index_peptides (3)\"},\"time\":{\"mean\":2554,\"min\":1867,\"\
  q1\":2162.5,\"q2\":2338.5,\"q3\":2730,\"max\":3672,\"minLabel\":\"index_peptides\
  \ (4)\",\"maxLabel\":\"index_peptides (1)\",\"q1Label\":\"index_peptides (4)\",\"\
  q2Label\":\"index_peptides (3)\",\"q3Label\":\"index_peptides (2)\"},\"writes\"\
  :{\"mean\":2214028.5,\"min\":2057377,\"q1\":2097285.25,\"q2\":2225553.5,\"q3\":2342296.75,\"\
  max\":2347630,\"minLabel\":\"index_peptides (2)\",\"maxLabel\":\"index_peptides\
  \ (4)\",\"q1Label\":\"index_peptides (2)\",\"q2Label\":\"index_peptides (1)\",\"\
  q3Label\":\"index_peptides (3)\"}},{\"cpuUsage\":{\"mean\":30.99,\"min\":18.7,\"\
  q1\":28.08,\"q2\":32.58,\"q3\":35.49,\"max\":40.1,\"minLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"maxLabel\":\"calculate_fdr_for_idalignment (2)\",\"q1Label\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q2Label\":\"calculate_fdr_for_idalignment (4)\",\"q3Label\":\"calculate_fdr_for_idalignment\
  \ (3)\"},\"process\":\"calculate_fdr_for_idalignment\",\"mem\":{\"mean\":18041856,\"\
  min\":6115328,\"q1\":13451264,\"q2\":18173952,\"q3\":22764544,\"max\":29704192,\"\
  minLabel\":\"calculate_fdr_for_idalignment (2)\",\"maxLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"calculate_fdr_for_idalignment (2)\",\"q2Label\":\"calculate_fdr_for_idalignment\
  \ (3)\",\"q3Label\":\"calculate_fdr_for_idalignment (4)\"},\"memUsage\":{\"mean\"\
  :0.28,\"min\":0.09,\"q1\":0.21,\"q2\":0.28,\"q3\":0.35,\"max\":0.46,\"minLabel\"\
  :\"calculate_fdr_for_idalignment (2)\",\"maxLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"calculate_fdr_for_idalignment (2)\",\"q2Label\":\"calculate_fdr_for_idalignment\
  \ (3)\",\"q3Label\":\"calculate_fdr_for_idalignment (4)\"},\"timeUsage\":{\"mean\"\
  :0.02,\"min\":0.02,\"q1\":0.02,\"q2\":0.02,\"q3\":0.03,\"max\":0.03,\"minLabel\"\
  :\"calculate_fdr_for_idalignment (2)\",\"maxLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"calculate_fdr_for_idalignment (2)\",\"q2Label\":\"calculate_fdr_for_idalignment\
  \ (3)\",\"q3Label\":\"calculate_fdr_for_idalignment (4)\"},\"vmem\":{\"mean\":169658368,\"\
  min\":85213184,\"q1\":114071552,\"q2\":123965440,\"q3\":179552256,\"max\":345489408,\"\
  minLabel\":\"calculate_fdr_for_idalignment (2)\",\"maxLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"calculate_fdr_for_idalignment (2)\",\"q2Label\":\"calculate_fdr_for_idalignment\
  \ (3)\",\"q3Label\":\"calculate_fdr_for_idalignment (4)\"},\"reads\":{\"mean\":5568052.5,\"\
  min\":5405803,\"q1\":5462506.75,\"q2\":5585175.5,\"q3\":5690721.25,\"max\":5696056,\"\
  minLabel\":\"calculate_fdr_for_idalignment (2)\",\"maxLabel\":\"calculate_fdr_for_idalignment\
  \ (4)\",\"q1Label\":\"calculate_fdr_for_idalignment (2)\",\"q2Label\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q3Label\":\"calculate_fdr_for_idalignment (3)\"},\"cpu\":{\"mean\":61.98,\"\
  min\":37.4,\"q1\":56.15,\"q2\":65.15,\"q3\":70.98,\"max\":80.2,\"minLabel\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"maxLabel\":\"calculate_fdr_for_idalignment (2)\",\"q1Label\":\"calculate_fdr_for_idalignment\
  \ (1)\",\"q2Label\":\"calculate_fdr_for_idalignment (4)\",\"q3Label\":\"calculate_fdr_for_idalignment\
  \ (3)\"},\"time\":{\"mean\":1656.25,\"min\":1159,\"q1\":1355.5,\"q2\":1504,\"q3\"\
  :1804.75,\"max\":2458,\"minLabel\":\"calculate_fdr_for_idalignment (2)\",\"maxLabel\"\
  :\"calculate_fdr_for_idalignment (1)\",\"q1Label\":\"calculate_fdr_for_idalignment\
  \ (2)\",\"q2Label\":\"calculate_fdr_for_idalignment (3)\",\"q3Label\":\"calculate_fdr_for_idalignment\
  \ (4)\"},\"writes\":{\"mean\":1583427,\"min\":1388962,\"q1\":1463183.5,\"q2\":1606077.5,\"\
  q3\":1726321,\"max\":1732591,\"minLabel\":\"calculate_fdr_for_idalignment (2)\"\
  ,\"maxLabel\":\"calculate_fdr_for_idalignment (3)\",\"q1Label\":\"calculate_fdr_for_idalignment\
  \ (2)\",\"q2Label\":\"calculate_fdr_for_idalignment (1)\",\"q3Label\":\"calculate_fdr_for_idalignment\
  \ (4)\"}},{\"cpuUsage\":{\"mean\":30.83,\"min\":15.7,\"q1\":28.86,\"q2\":34.65,\"\
  q3\":36.61,\"max\":38.3,\"minLabel\":\"filter_fdr_for_idalignment (1)\",\"maxLabel\"\
  :\"filter_fdr_for_idalignment (2)\",\"q1Label\":\"filter_fdr_for_idalignment (1)\"\
  ,\"q2Label\":\"filter_fdr_for_idalignment (3)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (4)\"},\"process\":\"filter_fdr_for_idalignment\",\"mem\":{\"mean\":21085184,\"\
  min\":13803520,\"q1\":13981696,\"q2\":17352704,\"q3\":24456192,\"max\":35831808,\"\
  minLabel\":\"filter_fdr_for_idalignment (4)\",\"maxLabel\":\"filter_fdr_for_idalignment\
  \ (1)\",\"q1Label\":\"filter_fdr_for_idalignment (4)\",\"q2Label\":\"filter_fdr_for_idalignment\
  \ (3)\",\"q3Label\":\"filter_fdr_for_idalignment (2)\"},\"memUsage\":{\"mean\":0.33,\"\
  min\":0.21,\"q1\":0.22,\"q2\":0.27,\"q3\":0.38,\"max\":0.56,\"minLabel\":\"filter_fdr_for_idalignment\
  \ (4)\",\"maxLabel\":\"filter_fdr_for_idalignment (1)\",\"q1Label\":\"filter_fdr_for_idalignment\
  \ (4)\",\"q2Label\":\"filter_fdr_for_idalignment (3)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (2)\"},\"timeUsage\":{\"mean\":0.02,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"q3\"\
  :0.02,\"max\":0.04,\"minLabel\":\"filter_fdr_for_idalignment (2)\",\"maxLabel\"\
  :\"filter_fdr_for_idalignment (1)\",\"q1Label\":\"filter_fdr_for_idalignment (2)\"\
  ,\"q2Label\":\"filter_fdr_for_idalignment (4)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (3)\"},\"vmem\":{\"mean\":180678656,\"min\":123645952,\"q1\":123735040,\"q2\"\
  :124039168,\"q3\":180982784,\"max\":350990336,\"minLabel\":\"filter_fdr_for_idalignment\
  \ (4)\",\"maxLabel\":\"filter_fdr_for_idalignment (1)\",\"q1Label\":\"filter_fdr_for_idalignment\
  \ (4)\",\"q2Label\":\"filter_fdr_for_idalignment (3)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (2)\"},\"reads\":{\"mean\":4964853.25,\"min\":4787835,\"q1\":4844281.5,\"q2\"\
  :4981578.5,\"q3\":5102150.25,\"max\":5108421,\"minLabel\":\"filter_fdr_for_idalignment\
  \ (1)\",\"maxLabel\":\"filter_fdr_for_idalignment (3)\",\"q1Label\":\"filter_fdr_for_idalignment\
  \ (1)\",\"q2Label\":\"filter_fdr_for_idalignment (2)\",\"q3Label\":\"filter_fdr_for_idalignment\
  \ (4)\"},\"cpu\":{\"mean\":61.65,\"min\":31.4,\"q1\":57.73,\"q2\":69.3,\"q3\":73.22,\"\
  max\":76.6,\"minLabel\":\"filter_fdr_for_idalignment (1)\",\"maxLabel\":\"filter_fdr_for_idalignment\
  \ (2)\",\"q1Label\":\"filter_fdr_for_idalignment (1)\",\"q2Label\":\"filter_fdr_for_idalignment\
  \ (3)\",\"q3Label\":\"filter_fdr_for_idalignment (4)\"},\"time\":{\"mean\":1412.75,\"\
  min\":966,\"q1\":1008,\"q2\":1076.5,\"q3\":1481.25,\"max\":2532,\"minLabel\":\"\
  filter_fdr_for_idalignment (2)\",\"maxLabel\":\"filter_fdr_for_idalignment (1)\"\
  ,\"q1Label\":\"filter_fdr_for_idalignment (2)\",\"q2Label\":\"filter_fdr_for_idalignment\
  \ (4)\",\"q3Label\":\"filter_fdr_for_idalignment (3)\"},\"writes\":{\"mean\":524337.5,\"\
  min\":362156,\"q1\":409923.5,\"q2\":508578.5,\"q3\":622992.5,\"max\":718037,\"minLabel\"\
  :\"filter_fdr_for_idalignment (1)\",\"maxLabel\":\"filter_fdr_for_idalignment (3)\"\
  ,\"q1Label\":\"filter_fdr_for_idalignment (1)\",\"q2Label\":\"filter_fdr_for_idalignment\
  \ (2)\",\"q3Label\":\"filter_fdr_for_idalignment (4)\"}},{\"cpuUsage\":{\"mean\"\
  :16.8,\"min\":16.8,\"q1\":16.8,\"q2\":16.8,\"q3\":16.8,\"max\":16.8,\"minLabel\"\
  :\"align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\",\"q2Label\"\
  :\"align_ids\",\"q3Label\":\"align_ids\"},\"process\":\"align_ids\",\"mem\":{\"\
  mean\":32768000,\"min\":32768000,\"q1\":32768000,\"q2\":32768000,\"q3\":32768000,\"\
  max\":32768000,\"minLabel\":\"align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\"\
  :\"align_ids\",\"q2Label\":\"align_ids\",\"q3Label\":\"align_ids\"},\"memUsage\"\
  :{\"mean\":0.51,\"min\":0.51,\"q1\":0.51,\"q2\":0.51,\"q3\":0.51,\"max\":0.51,\"\
  minLabel\":\"align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\",\"\
  q2Label\":\"align_ids\",\"q3Label\":\"align_ids\"},\"timeUsage\":{\"mean\":0.04,\"\
  min\":0.04,\"q1\":0.04,\"q2\":0.04,\"q3\":0.04,\"max\":0.04,\"minLabel\":\"align_ids\"\
  ,\"maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\",\"q2Label\":\"align_ids\"\
  ,\"q3Label\":\"align_ids\"},\"vmem\":{\"mean\":500097024,\"min\":500097024,\"q1\"\
  :500097024,\"q2\":500097024,\"q3\":500097024,\"max\":500097024,\"minLabel\":\"align_ids\"\
  ,\"maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\",\"q2Label\":\"align_ids\"\
  ,\"q3Label\":\"align_ids\"},\"reads\":{\"mean\":5492141,\"min\":5492141,\"q1\":5492141,\"\
  q2\":5492141,\"q3\":5492141,\"max\":5492141,\"minLabel\":\"align_ids\",\"maxLabel\"\
  :\"align_ids\",\"q1Label\":\"align_ids\",\"q2Label\":\"align_ids\",\"q3Label\":\"\
  align_ids\"},\"cpu\":{\"mean\":33.6,\"min\":33.6,\"q1\":33.6,\"q2\":33.6,\"q3\"\
  :33.6,\"max\":33.6,\"minLabel\":\"align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\"\
  :\"align_ids\",\"q2Label\":\"align_ids\",\"q3Label\":\"align_ids\"},\"time\":{\"\
  mean\":2569,\"min\":2569,\"q1\":2569,\"q2\":2569,\"q3\":2569,\"max\":2569,\"minLabel\"\
  :\"align_ids\",\"maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\",\"q2Label\"\
  :\"align_ids\",\"q3Label\":\"align_ids\"},\"writes\":{\"mean\":56820,\"min\":56820,\"\
  q1\":56820,\"q2\":56820,\"q3\":56820,\"max\":56820,\"minLabel\":\"align_ids\",\"\
  maxLabel\":\"align_ids\",\"q1Label\":\"align_ids\",\"q2Label\":\"align_ids\",\"\
  q3Label\":\"align_ids\"}},{\"cpuUsage\":{\"mean\":25.78,\"min\":15.7,\"q1\":24.03,\"\
  q2\":27.33,\"q3\":29.08,\"max\":32.75,\"minLabel\":\"align_idxml_files (4)\",\"\
  maxLabel\":\"align_idxml_files (1)\",\"q1Label\":\"align_idxml_files (4)\",\"q2Label\"\
  :\"align_idxml_files (2)\",\"q3Label\":\"align_idxml_files (3)\"},\"process\":\"\
  align_idxml_files\",\"mem\":{\"mean\":22575104,\"min\":13893632,\"q1\":15288320,\"\
  q2\":15941632,\"q3\":23228416,\"max\":44523520,\"minLabel\":\"align_idxml_files\
  \ (1)\",\"maxLabel\":\"align_idxml_files (4)\",\"q1Label\":\"align_idxml_files (1)\"\
  ,\"q2Label\":\"align_idxml_files (2)\",\"q3Label\":\"align_idxml_files (3)\"},\"\
  memUsage\":{\"mean\":0.35,\"min\":0.22,\"q1\":0.24,\"q2\":0.25,\"q3\":0.36,\"max\"\
  :0.69,\"minLabel\":\"align_idxml_files (1)\",\"maxLabel\":\"align_idxml_files (4)\"\
  ,\"q1Label\":\"align_idxml_files (1)\",\"q2Label\":\"align_idxml_files (2)\",\"\
  q3Label\":\"align_idxml_files (3)\"},\"timeUsage\":{\"mean\":0.03,\"min\":0.02,\"\
  q1\":0.02,\"q2\":0.02,\"q3\":0.03,\"max\":0.04,\"minLabel\":\"align_idxml_files\
  \ (1)\",\"maxLabel\":\"align_idxml_files (4)\",\"q1Label\":\"align_idxml_files (1)\"\
  ,\"q2Label\":\"align_idxml_files (2)\",\"q3Label\":\"align_idxml_files (3)\"},\"\
  vmem\":{\"mean\":217762816,\"min\":123584512,\"q1\":123673600,\"q2\":123703296,\"\
  q3\":217792512,\"max\":500060160,\"minLabel\":\"align_idxml_files (1)\",\"maxLabel\"\
  :\"align_idxml_files (4)\",\"q1Label\":\"align_idxml_files (1)\",\"q2Label\":\"\
  align_idxml_files (3)\",\"q3Label\":\"align_idxml_files (2)\"},\"reads\":{\"mean\"\
  :5590175.5,\"min\":5430027,\"q1\":5469291,\"q2\":5596117,\"q3\":5717001.5,\"max\"\
  :5738441,\"minLabel\":\"align_idxml_files (2)\",\"maxLabel\":\"align_idxml_files\
  \ (4)\",\"q1Label\":\"align_idxml_files (2)\",\"q2Label\":\"align_idxml_files (1)\"\
  ,\"q3Label\":\"align_idxml_files (3)\"},\"cpu\":{\"mean\":51.55,\"min\":31.4,\"\
  q1\":48.05,\"q2\":54.65,\"q3\":58.15,\"max\":65.5,\"minLabel\":\"align_idxml_files\
  \ (4)\",\"maxLabel\":\"align_idxml_files (1)\",\"q1Label\":\"align_idxml_files (4)\"\
  ,\"q2Label\":\"align_idxml_files (2)\",\"q3Label\":\"align_idxml_files (3)\"},\"\
  time\":{\"mean\":1968,\"min\":1496,\"q1\":1559.75,\"q2\":1593.5,\"q3\":2001.75,\"\
  max\":3189,\"minLabel\":\"align_idxml_files (1)\",\"maxLabel\":\"align_idxml_files\
  \ (4)\",\"q1Label\":\"align_idxml_files (1)\",\"q2Label\":\"align_idxml_files (2)\"\
  ,\"q3Label\":\"align_idxml_files (3)\"},\"writes\":{\"mean\":2230638.5,\"min\":2072667,\"\
  q1\":2112392.25,\"q2\":2241886.5,\"q3\":2360132.75,\"max\":2366114,\"minLabel\"\
  :\"align_idxml_files (2)\",\"maxLabel\":\"align_idxml_files (4)\",\"q1Label\":\"\
  align_idxml_files (2)\",\"q2Label\":\"align_idxml_files (1)\",\"q3Label\":\"align_idxml_files\
  \ (3)\"}},{\"cpuUsage\":{\"mean\":34.38,\"min\":32.05,\"q1\":33.03,\"q2\":33.38,\"\
  q3\":34.73,\"max\":38.7,\"minLabel\":\"align_mzml_files (3)\",\"maxLabel\":\"align_mzml_files\
  \ (4)\",\"q1Label\":\"align_mzml_files (3)\",\"q2Label\":\"align_mzml_files (2)\"\
  ,\"q3Label\":\"align_mzml_files (1)\"},\"process\":\"align_mzml_files\",\"mem\"\
  :{\"mean\":58518528,\"min\":57819136,\"q1\":57828352,\"q2\":58191872,\"q3\":58882048,\"\
  max\":59871232,\"minLabel\":\"align_mzml_files (4)\",\"maxLabel\":\"align_mzml_files\
  \ (2)\",\"q1Label\":\"align_mzml_files (4)\",\"q2Label\":\"align_mzml_files (3)\"\
  ,\"q3Label\":\"align_mzml_files (1)\"},\"memUsage\":{\"mean\":0.91,\"min\":0.9,\"\
  q1\":0.9,\"q2\":0.9,\"q3\":0.91,\"max\":0.93,\"minLabel\":\"align_mzml_files (4)\"\
  ,\"maxLabel\":\"align_mzml_files (2)\",\"q1Label\":\"align_mzml_files (4)\",\"q2Label\"\
  :\"align_mzml_files (3)\",\"q3Label\":\"align_mzml_files (1)\"},\"timeUsage\":{\"\
  mean\":0.05,\"min\":0.05,\"q1\":0.05,\"q2\":0.06,\"q3\":0.06,\"max\":0.06,\"minLabel\"\
  :\"align_mzml_files (4)\",\"maxLabel\":\"align_mzml_files (2)\",\"q1Label\":\"align_mzml_files\
  \ (4)\",\"q2Label\":\"align_mzml_files (3)\",\"q3Label\":\"align_mzml_files (1)\"\
  },\"vmem\":{\"mean\":277813248,\"min\":277774336,\"q1\":277777408,\"q2\":277792768,\"\
  q3\":277828608,\"max\":277893120,\"minLabel\":\"align_mzml_files (4)\",\"maxLabel\"\
  :\"align_mzml_files (2)\",\"q1Label\":\"align_mzml_files (4)\",\"q2Label\":\"align_mzml_files\
  \ (3)\",\"q3Label\":\"align_mzml_files (1)\"},\"reads\":{\"mean\":27955940.25,\"\
  min\":27126337,\"q1\":27438066.25,\"q2\":27915415,\"q3\":28433289,\"max\":28866594,\"\
  minLabel\":\"align_mzml_files (4)\",\"maxLabel\":\"align_mzml_files (2)\",\"q1Label\"\
  :\"align_mzml_files (4)\",\"q2Label\":\"align_mzml_files (3)\",\"q3Label\":\"align_mzml_files\
  \ (1)\"},\"cpu\":{\"mean\":68.75,\"min\":64.1,\"q1\":66.05,\"q2\":66.75,\"q3\":69.45,\"\
  max\":77.4,\"minLabel\":\"align_mzml_files (3)\",\"maxLabel\":\"align_mzml_files\
  \ (4)\",\"q1Label\":\"align_mzml_files (3)\",\"q2Label\":\"align_mzml_files (2)\"\
  ,\"q3Label\":\"align_mzml_files (1)\"},\"time\":{\"mean\":3921.5,\"min\":3376,\"\
  q1\":3813.25,\"q2\":4043.5,\"q3\":4151.75,\"max\":4223,\"minLabel\":\"align_mzml_files\
  \ (4)\",\"maxLabel\":\"align_mzml_files (2)\",\"q1Label\":\"align_mzml_files (4)\"\
  ,\"q2Label\":\"align_mzml_files (3)\",\"q3Label\":\"align_mzml_files (1)\"},\"writes\"\
  :{\"mean\":23667079,\"min\":22841920,\"q1\":23151834.25,\"q2\":23626284,\"q3\":24141528.75,\"\
  max\":24573828,\"minLabel\":\"align_mzml_files (4)\",\"maxLabel\":\"align_mzml_files\
  \ (2)\",\"q1Label\":\"align_mzml_files (4)\",\"q2Label\":\"align_mzml_files (3)\"\
  ,\"q3Label\":\"align_mzml_files (1)\"}},{\"cpuUsage\":{\"mean\":27.25,\"min\":27.25,\"\
  q1\":27.25,\"q2\":27.25,\"q3\":27.25,\"max\":27.25,\"minLabel\":\"merge_aligned_idxml_files\"\
  ,\"maxLabel\":\"merge_aligned_idxml_files\",\"q1Label\":\"merge_aligned_idxml_files\"\
  ,\"q2Label\":\"merge_aligned_idxml_files\",\"q3Label\":\"merge_aligned_idxml_files\"\
  },\"process\":\"merge_aligned_idxml_files\",\"mem\":{\"mean\":72650752,\"min\":72650752,\"\
  q1\":72650752,\"q2\":72650752,\"q3\":72650752,\"max\":72650752,\"minLabel\":\"merge_aligned_idxml_files\"\
  ,\"maxLabel\":\"merge_aligned_idxml_files\",\"q1Label\":\"merge_aligned_idxml_files\"\
  ,\"q2Label\":\"merge_aligned_idxml_files\",\"q3Label\":\"merge_aligned_idxml_files\"\
  },\"memUsage\":{\"mean\":1.13,\"min\":1.13,\"q1\":1.13,\"q2\":1.13,\"q3\":1.13,\"\
  max\":1.13,\"minLabel\":\"merge_aligned_idxml_files\",\"maxLabel\":\"merge_aligned_idxml_files\"\
  ,\"q1Label\":\"merge_aligned_idxml_files\",\"q2Label\":\"merge_aligned_idxml_files\"\
  ,\"q3Label\":\"merge_aligned_idxml_files\"},\"timeUsage\":{\"mean\":0.07,\"min\"\
  :0.07,\"q1\":0.07,\"q2\":0.07,\"q3\":0.07,\"max\":0.07,\"minLabel\":\"merge_aligned_idxml_files\"\
  ,\"maxLabel\":\"merge_aligned_idxml_files\",\"q1Label\":\"merge_aligned_idxml_files\"\
  ,\"q2Label\":\"merge_aligned_idxml_files\",\"q3Label\":\"merge_aligned_idxml_files\"\
  },\"vmem\":{\"mean\":500084736,\"min\":500084736,\"q1\":500084736,\"q2\":500084736,\"\
  q3\":500084736,\"max\":500084736,\"minLabel\":\"merge_aligned_idxml_files\",\"maxLabel\"\
  :\"merge_aligned_idxml_files\",\"q1Label\":\"merge_aligned_idxml_files\",\"q2Label\"\
  :\"merge_aligned_idxml_files\",\"q3Label\":\"merge_aligned_idxml_files\"},\"reads\"\
  :{\"mean\":12318499,\"min\":12318499,\"q1\":12318499,\"q2\":12318499,\"q3\":12318499,\"\
  max\":12318499,\"minLabel\":\"merge_aligned_idxml_files\",\"maxLabel\":\"merge_aligned_idxml_files\"\
  ,\"q1Label\":\"merge_aligned_idxml_files\",\"q2Label\":\"merge_aligned_idxml_files\"\
  ,\"q3Label\":\"merge_aligned_idxml_files\"},\"cpu\":{\"mean\":54.5,\"min\":54.5,\"\
  q1\":54.5,\"q2\":54.5,\"q3\":54.5,\"max\":54.5,\"minLabel\":\"merge_aligned_idxml_files\"\
  ,\"maxLabel\":\"merge_aligned_idxml_files\",\"q1Label\":\"merge_aligned_idxml_files\"\
  ,\"q2Label\":\"merge_aligned_idxml_files\",\"q3Label\":\"merge_aligned_idxml_files\"\
  },\"time\":{\"mean\":4752,\"min\":4752,\"q1\":4752,\"q2\":4752,\"q3\":4752,\"max\"\
  :4752,\"minLabel\":\"merge_aligned_idxml_files\",\"maxLabel\":\"merge_aligned_idxml_files\"\
  ,\"q1Label\":\"merge_aligned_idxml_files\",\"q2Label\":\"merge_aligned_idxml_files\"\
  ,\"q3Label\":\"merge_aligned_idxml_files\"},\"writes\":{\"mean\":9595349,\"min\"\
  :9595349,\"q1\":9595349,\"q2\":9595349,\"q3\":9595349,\"max\":9595349,\"minLabel\"\
  :\"merge_aligned_idxml_files\",\"maxLabel\":\"merge_aligned_idxml_files\",\"q1Label\"\
  :\"merge_aligned_idxml_files\",\"q2Label\":\"merge_aligned_idxml_files\",\"q3Label\"\
  :\"merge_aligned_idxml_files\"}},{\"cpuUsage\":{\"mean\":28.4,\"min\":28.4,\"q1\"\
  :28.4,\"q2\":28.4,\"q3\":28.4,\"max\":28.4,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"process\":\"extract_psm_features_for_percolator\",\"mem\":{\"mean\":70111232,\"\
  min\":70111232,\"q1\":70111232,\"q2\":70111232,\"q3\":70111232,\"max\":70111232,\"\
  minLabel\":\"extract_psm_features_for_percolator\",\"maxLabel\":\"extract_psm_features_for_percolator\"\
  ,\"q1Label\":\"extract_psm_features_for_percolator\",\"q2Label\":\"extract_psm_features_for_percolator\"\
  ,\"q3Label\":\"extract_psm_features_for_percolator\"},\"memUsage\":{\"mean\":1.09,\"\
  min\":1.09,\"q1\":1.09,\"q2\":1.09,\"q3\":1.09,\"max\":1.09,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"timeUsage\":{\"mean\":0.06,\"min\":0.06,\"q1\":0.06,\"q2\":0.06,\"q3\":0.06,\"\
  max\":0.06,\"minLabel\":\"extract_psm_features_for_percolator\",\"maxLabel\":\"\
  extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"vmem\":{\"mean\":500056064,\"min\":500056064,\"q1\":500056064,\"q2\":500056064,\"\
  q3\":500056064,\"max\":500056064,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"reads\":{\"mean\":12967344,\"min\":12967344,\"q1\":12967344,\"q2\":12967344,\"\
  q3\":12967344,\"max\":12967344,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"cpu\":{\"mean\":56.8,\"min\":56.8,\"q1\":56.8,\"q2\":56.8,\"q3\":56.8,\"max\"\
  :56.8,\"minLabel\":\"extract_psm_features_for_percolator\",\"maxLabel\":\"extract_psm_features_for_percolator\"\
  ,\"q1Label\":\"extract_psm_features_for_percolator\",\"q2Label\":\"extract_psm_features_for_percolator\"\
  ,\"q3Label\":\"extract_psm_features_for_percolator\"},\"time\":{\"mean\":4388,\"\
  min\":4388,\"q1\":4388,\"q2\":4388,\"q3\":4388,\"max\":4388,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  },\"writes\":{\"mean\":11630478,\"min\":11630478,\"q1\":11630478,\"q2\":11630478,\"\
  q3\":11630478,\"max\":11630478,\"minLabel\":\"extract_psm_features_for_percolator\"\
  ,\"maxLabel\":\"extract_psm_features_for_percolator\",\"q1Label\":\"extract_psm_features_for_percolator\"\
  ,\"q2Label\":\"extract_psm_features_for_percolator\",\"q3Label\":\"extract_psm_features_for_percolator\"\
  }},{\"cpuUsage\":{\"mean\":40,\"min\":40,\"q1\":40,\"q2\":40,\"q3\":40,\"max\":40,\"\
  minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\",\"q1Label\":\"run_percolator\"\
  ,\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"},\"process\":\"run_percolator\"\
  ,\"mem\":{\"mean\":91062272,\"min\":91062272,\"q1\":91062272,\"q2\":91062272,\"\
  q3\":91062272,\"max\":91062272,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\"\
  ,\"q1Label\":\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  },\"memUsage\":{\"mean\":1.41,\"min\":1.41,\"q1\":1.41,\"q2\":1.41,\"q3\":1.41,\"\
  max\":1.41,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\",\"q1Label\"\
  :\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  },\"timeUsage\":{\"mean\":0.01,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"q3\":0.01,\"\
  max\":0.01,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\",\"q1Label\"\
  :\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  },\"vmem\":{\"mean\":759771136,\"min\":759771136,\"q1\":759771136,\"q2\":759771136,\"\
  q3\":759771136,\"max\":759771136,\"minLabel\":\"run_percolator\",\"maxLabel\":\"\
  run_percolator\",\"q1Label\":\"run_percolator\",\"q2Label\":\"run_percolator\",\"\
  q3Label\":\"run_percolator\"},\"reads\":{\"mean\":17521231,\"min\":17521231,\"q1\"\
  :17521231,\"q2\":17521231,\"q3\":17521231,\"max\":17521231,\"minLabel\":\"run_percolator\"\
  ,\"maxLabel\":\"run_percolator\",\"q1Label\":\"run_percolator\",\"q2Label\":\"run_percolator\"\
  ,\"q3Label\":\"run_percolator\"},\"cpu\":{\"mean\":80,\"min\":80,\"q1\":80,\"q2\"\
  :80,\"q3\":80,\"max\":80,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\"\
  ,\"q1Label\":\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  },\"time\":{\"mean\":10320,\"min\":10320,\"q1\":10320,\"q2\":10320,\"q3\":10320,\"\
  max\":10320,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\",\"q1Label\"\
  :\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  },\"writes\":{\"mean\":16519315,\"min\":16519315,\"q1\":16519315,\"q2\":16519315,\"\
  q3\":16519315,\"max\":16519315,\"minLabel\":\"run_percolator\",\"maxLabel\":\"run_percolator\"\
  ,\"q1Label\":\"run_percolator\",\"q2Label\":\"run_percolator\",\"q3Label\":\"run_percolator\"\
  }},{\"cpuUsage\":{\"mean\":41.55,\"min\":41.55,\"q1\":41.55,\"q2\":41.55,\"q3\"\
  :41.55,\"max\":41.55,\"minLabel\":\"filter_by_q_value\",\"maxLabel\":\"filter_by_q_value\"\
  ,\"q1Label\":\"filter_by_q_value\",\"q2Label\":\"filter_by_q_value\",\"q3Label\"\
  :\"filter_by_q_value\"},\"process\":\"filter_by_q_value\",\"mem\":{\"mean\":56090624,\"\
  min\":56090624,\"q1\":56090624,\"q2\":56090624,\"q3\":56090624,\"max\":56090624,\"\
  minLabel\":\"filter_by_q_value\",\"maxLabel\":\"filter_by_q_value\",\"q1Label\"\
  :\"filter_by_q_value\",\"q2Label\":\"filter_by_q_value\",\"q3Label\":\"filter_by_q_value\"\
  },\"memUsage\":{\"mean\":0.87,\"min\":0.87,\"q1\":0.87,\"q2\":0.87,\"q3\":0.87,\"\
  max\":0.87,\"minLabel\":\"filter_by_q_value\",\"maxLabel\":\"filter_by_q_value\"\
  ,\"q1Label\":\"filter_by_q_value\",\"q2Label\":\"filter_by_q_value\",\"q3Label\"\
  :\"filter_by_q_value\"},\"timeUsage\":{\"mean\":0.04,\"min\":0.04,\"q1\":0.04,\"\
  q2\":0.04,\"q3\":0.04,\"max\":0.04,\"minLabel\":\"filter_by_q_value\",\"maxLabel\"\
  :\"filter_by_q_value\",\"q1Label\":\"filter_by_q_value\",\"q2Label\":\"filter_by_q_value\"\
  ,\"q3Label\":\"filter_by_q_value\"},\"vmem\":{\"mean\":152702976,\"min\":152702976,\"\
  q1\":152702976,\"q2\":152702976,\"q3\":152702976,\"max\":152702976,\"minLabel\"\
  :\"filter_by_q_value\",\"maxLabel\":\"filter_by_q_value\",\"q1Label\":\"filter_by_q_value\"\
  ,\"q2Label\":\"filter_by_q_value\",\"q3Label\":\"filter_by_q_value\"},\"reads\"\
  :{\"mean\":16348174,\"min\":16348174,\"q1\":16348174,\"q2\":16348174,\"q3\":16348174,\"\
  max\":16348174,\"minLabel\":\"filter_by_q_value\",\"maxLabel\":\"filter_by_q_value\"\
  ,\"q1Label\":\"filter_by_q_value\",\"q2Label\":\"filter_by_q_value\",\"q3Label\"\
  :\"filter_by_q_value\"},\"cpu\":{\"mean\":83.1,\"min\":83.1,\"q1\":83.1,\"q2\":83.1,\"\
  q3\":83.1,\"max\":83.1,\"minLabel\":\"filter_by_q_value\",\"maxLabel\":\"filter_by_q_value\"\
  ,\"q1Label\":\"filter_by_q_value\",\"q2Label\":\"filter_by_q_value\",\"q3Label\"\
  :\"filter_by_q_value\"},\"time\":{\"mean\":2734,\"min\":2734,\"q1\":2734,\"q2\"\
  :2734,\"q3\":2734,\"max\":2734,\"minLabel\":\"filter_by_q_value\",\"maxLabel\":\"\
  filter_by_q_value\",\"q1Label\":\"filter_by_q_value\",\"q2Label\":\"filter_by_q_value\"\
  ,\"q3Label\":\"filter_by_q_value\"},\"writes\":{\"mean\":978821,\"min\":978821,\"\
  q1\":978821,\"q2\":978821,\"q3\":978821,\"max\":978821,\"minLabel\":\"filter_by_q_value\"\
  ,\"maxLabel\":\"filter_by_q_value\",\"q1Label\":\"filter_by_q_value\",\"q2Label\"\
  :\"filter_by_q_value\",\"q3Label\":\"filter_by_q_value\"}},{\"cpuUsage\":{\"mean\"\
  :41.69,\"min\":36.5,\"q1\":41.3,\"q2\":42.97,\"q3\":43.36,\"max\":44.3,\"minLabel\"\
  :\"quantify_identifications_targeted (3)\",\"maxLabel\":\"quantify_identifications_targeted\
  \ (4)\",\"q1Label\":\"quantify_identifications_targeted (3)\",\"q2Label\":\"quantify_identifications_targeted\
  \ (2)\",\"q3Label\":\"quantify_identifications_targeted (1)\"},\"process\":\"quantify_identifications_targeted\"\
  ,\"mem\":{\"mean\":94819328,\"min\":92413952,\"q1\":93178880,\"q2\":93851648,\"\
  q3\":95492096,\"max\":99160064,\"minLabel\":\"quantify_identifications_targeted\
  \ (4)\",\"maxLabel\":\"quantify_identifications_targeted (3)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (4)\",\"q2Label\":\"quantify_identifications_targeted (2)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (1)\"},\"memUsage\":{\"mean\":1.47,\"min\":1.43,\"q1\":1.45,\"q2\":1.46,\"q3\"\
  :1.48,\"max\":1.54,\"minLabel\":\"quantify_identifications_targeted (4)\",\"maxLabel\"\
  :\"quantify_identifications_targeted (3)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (4)\",\"q2Label\":\"quantify_identifications_targeted (2)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (1)\"},\"timeUsage\":{\"mean\":0.01,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"q3\"\
  :0.01,\"max\":0.01,\"minLabel\":\"quantify_identifications_targeted (4)\",\"maxLabel\"\
  :\"quantify_identifications_targeted (3)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (4)\",\"q2Label\":\"quantify_identifications_targeted (1)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (2)\"},\"vmem\":{\"mean\":335713280,\"min\":280018944,\"q1\":280074240,\"q2\"\
  :280131584,\"q3\":335770624,\"max\":502571008,\"minLabel\":\"quantify_identifications_targeted\
  \ (2)\",\"maxLabel\":\"quantify_identifications_targeted (3)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (2)\",\"q2Label\":\"quantify_identifications_targeted (4)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (1)\"},\"reads\":{\"mean\":30085448.5,\"min\":29254518,\"q1\":29564432.25,\"q2\"\
  :30038882,\"q3\":30559898.25,\"max\":31009512,\"minLabel\":\"quantify_identifications_targeted\
  \ (2)\",\"maxLabel\":\"quantify_identifications_targeted (3)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (2)\",\"q2Label\":\"quantify_identifications_targeted (1)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (4)\"},\"cpu\":{\"mean\":83.38,\"min\":73,\"q1\":82.6,\"q2\":85.95,\"q3\":86.73,\"\
  max\":88.6,\"minLabel\":\"quantify_identifications_targeted (3)\",\"maxLabel\":\"\
  quantify_identifications_targeted (4)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (3)\",\"q2Label\":\"quantify_identifications_targeted (2)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (1)\"},\"time\":{\"mean\":6994.75,\"min\":5838,\"q1\":6641.25,\"q2\":7196,\"q3\"\
  :7549.5,\"max\":7749,\"minLabel\":\"quantify_identifications_targeted (4)\",\"maxLabel\"\
  :\"quantify_identifications_targeted (3)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (4)\",\"q2Label\":\"quantify_identifications_targeted (1)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (2)\"},\"writes\":{\"mean\":3725215.75,\"min\":3446178,\"q1\":3667314.75,\"q2\"\
  :3791318,\"q3\":3849219,\"max\":3872049,\"minLabel\":\"quantify_identifications_targeted\
  \ (4)\",\"maxLabel\":\"quantify_identifications_targeted (3)\",\"q1Label\":\"quantify_identifications_targeted\
  \ (4)\",\"q2Label\":\"quantify_identifications_targeted (1)\",\"q3Label\":\"quantify_identifications_targeted\
  \ (2)\"}},{\"cpuUsage\":{\"mean\":33.05,\"min\":33.05,\"q1\":33.05,\"q2\":33.05,\"\
  q3\":33.05,\"max\":33.05,\"minLabel\":\"link_extracted_features\",\"maxLabel\":\"\
  link_extracted_features\",\"q1Label\":\"link_extracted_features\",\"q2Label\":\"\
  link_extracted_features\",\"q3Label\":\"link_extracted_features\"},\"process\":\"\
  link_extracted_features\",\"mem\":{\"mean\":58241024,\"min\":58241024,\"q1\":58241024,\"\
  q2\":58241024,\"q3\":58241024,\"max\":58241024,\"minLabel\":\"link_extracted_features\"\
  ,\"maxLabel\":\"link_extracted_features\",\"q1Label\":\"link_extracted_features\"\
  ,\"q2Label\":\"link_extracted_features\",\"q3Label\":\"link_extracted_features\"\
  },\"memUsage\":{\"mean\":0.9,\"min\":0.9,\"q1\":0.9,\"q2\":0.9,\"q3\":0.9,\"max\"\
  :0.9,\"minLabel\":\"link_extracted_features\",\"maxLabel\":\"link_extracted_features\"\
  ,\"q1Label\":\"link_extracted_features\",\"q2Label\":\"link_extracted_features\"\
  ,\"q3Label\":\"link_extracted_features\"},\"timeUsage\":null,\"vmem\":{\"mean\"\
  :500064256,\"min\":500064256,\"q1\":500064256,\"q2\":500064256,\"q3\":500064256,\"\
  max\":500064256,\"minLabel\":\"link_extracted_features\",\"maxLabel\":\"link_extracted_features\"\
  ,\"q1Label\":\"link_extracted_features\",\"q2Label\":\"link_extracted_features\"\
  ,\"q3Label\":\"link_extracted_features\"},\"reads\":{\"mean\":18354040,\"min\":18354040,\"\
  q1\":18354040,\"q2\":18354040,\"q3\":18354040,\"max\":18354040,\"minLabel\":\"link_extracted_features\"\
  ,\"maxLabel\":\"link_extracted_features\",\"q1Label\":\"link_extracted_features\"\
  ,\"q2Label\":\"link_extracted_features\",\"q3Label\":\"link_extracted_features\"\
  },\"cpu\":{\"mean\":66.1,\"min\":66.1,\"q1\":66.1,\"q2\":66.1,\"q3\":66.1,\"max\"\
  :66.1,\"minLabel\":\"link_extracted_features\",\"maxLabel\":\"link_extracted_features\"\
  ,\"q1Label\":\"link_extracted_features\",\"q2Label\":\"link_extracted_features\"\
  ,\"q3Label\":\"link_extracted_features\"},\"time\":{\"mean\":4016,\"min\":4016,\"\
  q1\":4016,\"q2\":4016,\"q3\":4016,\"max\":4016,\"minLabel\":\"link_extracted_features\"\
  ,\"maxLabel\":\"link_extracted_features\",\"q1Label\":\"link_extracted_features\"\
  ,\"q2Label\":\"link_extracted_features\",\"q3Label\":\"link_extracted_features\"\
  },\"writes\":{\"mean\":4434948,\"min\":4434948,\"q1\":4434948,\"q2\":4434948,\"\
  q3\":4434948,\"max\":4434948,\"minLabel\":\"link_extracted_features\",\"maxLabel\"\
  :\"link_extracted_features\",\"q1Label\":\"link_extracted_features\",\"q2Label\"\
  :\"link_extracted_features\",\"q3Label\":\"link_extracted_features\"}},{\"cpuUsage\"\
  :{\"mean\":24.05,\"min\":24.05,\"q1\":24.05,\"q2\":24.05,\"q3\":24.05,\"max\":24.05,\"\
  minLabel\":\"resolve_conflicts\",\"maxLabel\":\"resolve_conflicts\",\"q1Label\"\
  :\"resolve_conflicts\",\"q2Label\":\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"\
  },\"process\":\"resolve_conflicts\",\"mem\":{\"mean\":44412928,\"min\":44412928,\"\
  q1\":44412928,\"q2\":44412928,\"q3\":44412928,\"max\":44412928,\"minLabel\":\"resolve_conflicts\"\
  ,\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\",\"q2Label\"\
  :\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"},\"memUsage\":{\"mean\"\
  :0.69,\"min\":0.69,\"q1\":0.69,\"q2\":0.69,\"q3\":0.69,\"max\":0.69,\"minLabel\"\
  :\"resolve_conflicts\",\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\"\
  ,\"q2Label\":\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"},\"timeUsage\"\
  :{\"mean\":0.04,\"min\":0.04,\"q1\":0.04,\"q2\":0.04,\"q3\":0.04,\"max\":0.04,\"\
  minLabel\":\"resolve_conflicts\",\"maxLabel\":\"resolve_conflicts\",\"q1Label\"\
  :\"resolve_conflicts\",\"q2Label\":\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"\
  },\"vmem\":{\"mean\":500015104,\"min\":500015104,\"q1\":500015104,\"q2\":500015104,\"\
  q3\":500015104,\"max\":500015104,\"minLabel\":\"resolve_conflicts\",\"maxLabel\"\
  :\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\",\"q2Label\":\"resolve_conflicts\"\
  ,\"q3Label\":\"resolve_conflicts\"},\"reads\":{\"mean\":7822376,\"min\":7822376,\"\
  q1\":7822376,\"q2\":7822376,\"q3\":7822376,\"max\":7822376,\"minLabel\":\"resolve_conflicts\"\
  ,\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\",\"q2Label\"\
  :\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"},\"cpu\":{\"mean\":48.1,\"\
  min\":48.1,\"q1\":48.1,\"q2\":48.1,\"q3\":48.1,\"max\":48.1,\"minLabel\":\"resolve_conflicts\"\
  ,\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\",\"q2Label\"\
  :\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"},\"time\":{\"mean\":2836,\"\
  min\":2836,\"q1\":2836,\"q2\":2836,\"q3\":2836,\"max\":2836,\"minLabel\":\"resolve_conflicts\"\
  ,\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\",\"q2Label\"\
  :\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"},\"writes\":{\"mean\":4588235,\"\
  min\":4588235,\"q1\":4588235,\"q2\":4588235,\"q3\":4588235,\"max\":4588235,\"minLabel\"\
  :\"resolve_conflicts\",\"maxLabel\":\"resolve_conflicts\",\"q1Label\":\"resolve_conflicts\"\
  ,\"q2Label\":\"resolve_conflicts\",\"q3Label\":\"resolve_conflicts\"}},{\"cpuUsage\"\
  :{\"mean\":22.2,\"min\":22.2,\"q1\":22.2,\"q2\":22.2,\"q3\":22.2,\"max\":22.2,\"\
  minLabel\":\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\":\"export_text\"\
  ,\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"},\"process\":\"export_text\"\
  ,\"mem\":{\"mean\":45842432,\"min\":45842432,\"q1\":45842432,\"q2\":45842432,\"\
  q3\":45842432,\"max\":45842432,\"minLabel\":\"export_text\",\"maxLabel\":\"export_text\"\
  ,\"q1Label\":\"export_text\",\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"\
  },\"memUsage\":{\"mean\":0.71,\"min\":0.71,\"q1\":0.71,\"q2\":0.71,\"q3\":0.71,\"\
  max\":0.71,\"minLabel\":\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\"\
  :\"export_text\",\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"},\"timeUsage\"\
  :{\"mean\":0.04,\"min\":0.04,\"q1\":0.04,\"q2\":0.04,\"q3\":0.04,\"max\":0.04,\"\
  minLabel\":\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\":\"export_text\"\
  ,\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"},\"vmem\":{\"mean\":500133888,\"\
  min\":500133888,\"q1\":500133888,\"q2\":500133888,\"q3\":500133888,\"max\":500133888,\"\
  minLabel\":\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\":\"export_text\"\
  ,\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"},\"reads\":{\"mean\":7976226,\"\
  min\":7976226,\"q1\":7976226,\"q2\":7976226,\"q3\":7976226,\"max\":7976226,\"minLabel\"\
  :\"export_text\",\"maxLabel\":\"export_text\",\"q1Label\":\"export_text\",\"q2Label\"\
  :\"export_text\",\"q3Label\":\"export_text\"},\"cpu\":{\"mean\":44.4,\"min\":44.4,\"\
  q1\":44.4,\"q2\":44.4,\"q3\":44.4,\"max\":44.4,\"minLabel\":\"export_text\",\"maxLabel\"\
  :\"export_text\",\"q1Label\":\"export_text\",\"q2Label\":\"export_text\",\"q3Label\"\
  :\"export_text\"},\"time\":{\"mean\":2981,\"min\":2981,\"q1\":2981,\"q2\":2981,\"\
  q3\":2981,\"max\":2981,\"minLabel\":\"export_text\",\"maxLabel\":\"export_text\"\
  ,\"q1Label\":\"export_text\",\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"\
  },\"writes\":{\"mean\":924673,\"min\":924673,\"q1\":924673,\"q2\":924673,\"q3\"\
  :924673,\"max\":924673,\"minLabel\":\"export_text\",\"maxLabel\":\"export_text\"\
  ,\"q1Label\":\"export_text\",\"q2Label\":\"export_text\",\"q3Label\":\"export_text\"\
  }},{\"cpuUsage\":{\"mean\":20.2,\"min\":20.2,\"q1\":20.2,\"q2\":20.2,\"q3\":20.2,\"\
  max\":20.2,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\",\"q1Label\"\
  :\"export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"},\"\
  process\":\"export_mztab\",\"mem\":{\"mean\":46600192,\"min\":46600192,\"q1\":46600192,\"\
  q2\":46600192,\"q3\":46600192,\"max\":46600192,\"minLabel\":\"export_mztab\",\"\
  maxLabel\":\"export_mztab\",\"q1Label\":\"export_mztab\",\"q2Label\":\"export_mztab\"\
  ,\"q3Label\":\"export_mztab\"},\"memUsage\":{\"mean\":0.72,\"min\":0.72,\"q1\":0.72,\"\
  q2\":0.72,\"q3\":0.72,\"max\":0.72,\"minLabel\":\"export_mztab\",\"maxLabel\":\"\
  export_mztab\",\"q1Label\":\"export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\"\
  :\"export_mztab\"},\"timeUsage\":{\"mean\":0.04,\"min\":0.04,\"q1\":0.04,\"q2\"\
  :0.04,\"q3\":0.04,\"max\":0.04,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\"\
  ,\"q1Label\":\"export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"\
  },\"vmem\":{\"mean\":500207616,\"min\":500207616,\"q1\":500207616,\"q2\":500207616,\"\
  q3\":500207616,\"max\":500207616,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\"\
  ,\"q1Label\":\"export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"\
  },\"reads\":{\"mean\":7976131,\"min\":7976131,\"q1\":7976131,\"q2\":7976131,\"q3\"\
  :7976131,\"max\":7976131,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\"\
  ,\"q1Label\":\"export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"\
  },\"cpu\":{\"mean\":40.4,\"min\":40.4,\"q1\":40.4,\"q2\":40.4,\"q3\":40.4,\"max\"\
  :40.4,\"minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\",\"q1Label\":\"\
  export_mztab\",\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"},\"time\"\
  :{\"mean\":3196,\"min\":3196,\"q1\":3196,\"q2\":3196,\"q3\":3196,\"max\":3196,\"\
  minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\",\"q1Label\":\"export_mztab\"\
  ,\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"},\"writes\":{\"mean\"\
  :141457,\"min\":141457,\"q1\":141457,\"q2\":141457,\"q3\":141457,\"max\":141457,\"\
  minLabel\":\"export_mztab\",\"maxLabel\":\"export_mztab\",\"q1Label\":\"export_mztab\"\
  ,\"q2Label\":\"export_mztab\",\"q3Label\":\"export_mztab\"}}]\nFeb-19 04:03:53.923\
  \ [main] DEBUG nextflow.trace.TimelineObserver - Workflow completed -- rendering\
  \ execution timeline\nFeb-19 04:03:54.298 [main] DEBUG nextflow.cache.CacheDB -\
  \ Closing CacheDB done\nFeb-19 04:03:54.329 [main] DEBUG nextflow.util.ThreadPoolManager\
  \ - Thread pool 'FileTransfer' shutdown completed (hard=false)\nFeb-19 04:03:54.330\
  \ [main] DEBUG nextflow.script.ScriptRunner - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2F3erd6u9e%2Flogs.tar.xz">0.083GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2F3erd6u9e%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.840GiB</td>
						<td>2647.9 seconds</td>
						<td>1973.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743144192">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.2.2">1.2.2</a>
						</td>
						<td>nextflow</td>
						<td>1483 days</td>
						<td>2023-02-19 06:30:03</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fst7socj1%2Flogs.tar.xz">0.078GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fst7socj1%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.801GiB</td>
						<td>2614.4 seconds</td>
						<td>1137.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743107824">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.0.5">2.0.5</a>
						</td>
						<td>nextflow</td>
						<td>1481 days</td>
						<td>2023-02-19 04:15:47</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 04:15:56.614
kind: NextflowMiscError
process: main
rest: Please specify either --singleEnd, --pairedEnd to execute the pipeline on FastQ
  files and --bam for previously processed BAM files!

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F3jpluqai%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F3jpluqai%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.579GiB</td>
						<td>21.9 seconds</td>
						<td>9.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743139632">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/1.3">1.3</a>
						</td>
						<td>nextflow</td>
						<td>1479 days</td>
						<td>2023-02-19 04:16:26</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2F3vy9c3u3%2Flogs.tar.xz">0.036GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2F3vy9c3u3%2Foutput.tar.xz">0.005GiB</a>
						</td>
						<td>1.620GiB</td>
						<td>631.1 seconds</td>
						<td>369.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743115984">
								<a href="https://nf-co.re/hlatyping">hlatyping</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hlatyping/tree/1.1.3">1.1.3</a>
						</td>
						<td>nextflow</td>
						<td>1476 days</td>
						<td>2023-02-19 06:00:26</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-19 07:00:26.365
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fdjt4qqd0%2Flogs.tar.xz">0.035GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fdjt4qqd0%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.532GiB</td>
						<td>437.6 seconds</td>
						<td>3604.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743143952">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.2.3">1.2.3</a>
						</td>
						<td>nextflow</td>
						<td>1476 days</td>
						<td>2023-02-19 03:59:23</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2F81qc5nxp%2Flogs.tar.xz">0.078GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2F81qc5nxp%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.792GiB</td>
						<td>2610.5 seconds</td>
						<td>1134.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743143712">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.2.4">1.2.4</a>
						</td>
						<td>nextflow</td>
						<td>1474 days</td>
						<td>2023-02-19 06:10:43</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fsayvc6st%2Flogs.tar.xz">0.077GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fsayvc6st%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.799GiB</td>
						<td>2619.5 seconds</td>
						<td>1469.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738324176">
								<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling">snakemake-workflows/dna-seq-gatk-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v1.0.1">v1.0.1</a>
						</td>
						<td>snakemake</td>
						<td>1472 days</td>
						<td>2023-02-19 06:00:19</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- rules
- common.smk
kind: FileNotFoundError
line_no: 18
rest: "[Errno 2] No such file or directory: 'data/ref/genome.chr21.fa.fai'\n  File\
  \ \"/home/azureuser/tmp/mmq9xrn6pf/code/Snakefile\", line 1, in <module>\n  File\
  \ \"/home/azureuser/tmp/mmq9xrn6pf/code/rules/common.smk\", line 18, in <module>\n\
  \  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 211, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 331, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1289, in read_table\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 605, in _read\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1442, in __init__\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1735, in _make_engine\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/common.py\"\
  , line 856, in get_handle"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2F6av8kx53%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2F6av8kx53%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>2.0 seconds</td>
						<td>2.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738324416">
								<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling">snakemake-workflows/dna-seq-gatk-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>1472 days</td>
						<td>2023-02-19 03:31:14</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- rules
- common.smk
kind: FileNotFoundError
line_no: 18
rest: "[Errno 2] No such file or directory: 'data/ref/genome.chr21.fa.fai'\n  File\
  \ \"/home/azureuser/tmp/75u31g35lr/code/Snakefile\", line 1, in <module>\n  File\
  \ \"/home/azureuser/tmp/75u31g35lr/code/rules/common.smk\", line 18, in <module>\n\
  \  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 211, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 331, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1289, in read_table\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 605, in _read\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1442, in __init__\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1735, in _make_engine\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/common.py\"\
  , line 856, in get_handle"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Fvsp33vqq%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Fvsp33vqq%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>1.8 seconds</td>
						<td>2.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743150432">
								<a href="https://nf-co.re/rnafusion">rnafusion</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnafusion/tree/1.0">1.0</a>
						</td>
						<td>nextflow</td>
						<td>1465 days</td>
						<td>2023-02-19 03:32:34</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 03:32:51.016
kind: NextflowMiscError
process: main
rest: 'Fasta file not found: tests/genome.fa'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2F7cshswfg%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2F7cshswfg%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.489GiB</td>
						<td>19.4 seconds</td>
						<td>16.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743143472">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.2.5">1.2.5</a>
						</td>
						<td>nextflow</td>
						<td>1448 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "CometAdapter  -in MM15_Melanom_2_B_1.mzML \\\n              -out\
  \ MM15_Melanom_2_B_1.idXML \\\n              -threads 2 \\\n              -database\
  \ test_decoy.fasta \\\n              -precursor_mass_tolerance 5 \\\n          \
  \    -fragment_bin_tolerance 0.02 \\\n              -fragment_bin_offset 0 \\\n\
  \              -num_hits 1 \\\n              -digest_mass_range 800:2500 \\\n  \
  \            -max_variable_mods_in_peptide 3 \\\n              -allowed_missed_cleavages\
  \ 0 \\\n              -precursor_charge 2:3 \\\n              -activation_method\
  \ ALL \\\n              -use_NL_ions true \\\n              -variable_modifications\
  \ 'Oxidation (M)' \\\n              -fixed_modifications  \\\n              -enzyme\
  \ 'unspecific cleavage' \\\n              -spectrum_batch_size 500"
error: "Comet is writing the default parameter file...\nExecuting: comet.exe\nExecuted\
  \ comet.exe successfully!\nProgress of 'loading spectra list':\n\n-- done [took\
  \ 0.75 s (CPU), 0.80 s (Wall)] -- \nProgress of 'loading chromatogram list':\n\n\
  -- done [took 0.00 s (CPU), 0.00 s (Wall)] -- \nFATAL: External invocation of comet.exe\
  \ failed. Standard output and error were:\n<Executing: comet.exe> occurred 2 times\n\
  \ Comet version \"2016.01 rev. 3\"\n\n Search start:  02/19/2023, 06:10:25 AM\n\
  \ - Input file: MM15_Melanom_2_B_1.mzML\n   - Load spectra: 502\n     - Search progress:\
  \   0%\b\b\b\b  0%\b\b\b\b  1%\b\b\b\b  1%\b\b\b\b  2%\b\b\b\b  2%\b\b\b\b  3%\b\
  \b\b\b  4%\b\b\b\b  4%\b\b\b\b  5%\b\b\b\b  5%\b\b\b\b  6%\b\b\b\b  6%\b\b\b\b \
  \ 7%\b\b\b\b  7%\b\b\b\b  8%\b\b\b\b  8%\b\b\b\b  9%\b\b\b\b  9%\b\b\b\b 10%\b\b\
  \b\b 10%\b\b\b\b 11%\b\b\b\b 11%\b\b\b\b 11%\b\b\b\b 12%\b\b\b\b 12%\b\b\b\b 13%\b\
  \b\b\b 13%\b\b\b\b 13%\b\b\b\b 14%\b\b\b\b 14%\b\b\b\b 15%\b\b\b\b 15%\b\b\b\b 15%\b\
  \b\b\b 16%\b\b\b\b 17%\b\b\b\b 17%\b\b\b\b 17%\b\b\b\b 18%\b\b\b\b 18%\b\b\b\b 19%\b\
  \b\b\b 19%\b\b\b\b 19%\b\b\b\b 20%\b\b\b\b 20%\b\b\b\b 21%\b\b\b\b 21%\b\b\b\b 22%\b\
  \b\b\b 22%\b\b\b\b 23%\b\b\b\b 24%\b\b\b\b 24%\b\b\b\b 25%\b\b\b\b 25%\b\b\b\b 26%\b\
  \b\b\b 26%\b\b\b\b 27%\b\b\b\b 27%\b\b\b\b 27%\b\b\b\b 28%\b\b\b\b 28%\b\b\b\b 29%\b\
  \b\b\b 29%\b\b\b\b 30%\b\b\b\b 30%\b\b\b\b 31%\b\b\b\b 31%\b\b\b\b 32%\b\b\b\b 32%\b\
  \b\b\b 33%\b\b\b\b 33%\b\b\b\b 33%\b\b\b\b 34%\b\b\b\b 34%\b\b\b\b 35%\b\b\b\b 35%\b\
  \b\b\b 36%\b\b\b\b 36%\b\b\b\b 36%\b\b\b\b 37%\b\b\b\b 37%\b\b\b\b 38%\b\b\b\b 38%\b\
  \b\b\b 38%\b\b\b\b 39%\b\b\b\b 40%\b\b\b\b 40%\b\b\b\b 41%\b\b\b\b 41%\b\b\b\b 42%\b\
  \b\b\b 42%\b\b\b\b 43%\b\b\b\b 43%\b\b\b\b 43%\b\b\b\b 44%\b\b\b\b 44%\b\b\b\b 45%\b\
  \b\b\b 45%\b\b\b\b 46%\b\b\b\b 46%\b\b\b\b 47%\b\b\b\b 47%\b\b\b\b 48%\b\b\b\b 48%\b\
  \b\b\b 49%\b\b\b\b 49%\b\b\b\b 50%\b\b\b\b 50%\b\b\b\b 51%\b\b\b\b 52%\b\b\b\b 52%\b\
  \b\b\b 53%\b\b\b\b 53%\b\b\b\b 54%\b\b\b\b 54%\b\b\b\b 55%\b\b\b\b 55%\b\b\b\b 56%\b\
  \b\b\b 56%\b\b\b\b 56%\b\b\b\b 57%\b\b\b\b 57%\b\b\b\b 57%\b\b\b\b 58%\b\b\b\b 58%\b\
  \b\b\b 59%\b\b\b\b 59%\b\b\b\b 60%\b\b\b\b 61%\b\b\b\b 61%\b\b\b\b 61%\b\b\b\b 62%\b\
  \b\b\b 63%\b\b\b\b 63%\b\b\b\b 64%\b\b\b\b 64%\b\b\b\b 65%\b\b\b\b 65%\b\b\b\b 66%\b\
  \b\b\b 66%\b\b\b\b 66%\b\b\b\b 67%\b\b\b\b 67%\b\b\b\b 68%\b\b\b\b 68%\b\b\b\b 69%\b\
  \b\b\b 69%\b\b\b\b 69%\b\b\b\b 70%\b\b\b\b 70%\b\b\b\b 71%\b\b\b\b 71%\b\b\b\b 72%\b\
  \b\b\b 72%\b\b\b\b 73%\b\b\b\b 73%\b\b\b\b 74%\b\b\b\b 74%\b\b\b\b 75%\b\b\b\b 75%\b\
  \b\b\b 76%\b\b\b\b 76%\b\b\b\b 76%\b\b\b\b 77%\b\b\b\b 77%\b\b\b\b 78%\b\b\b\b 78%\b\
  \b\b\b 79%\b\b\b\b 79%\b\b\b\b 80%\b\b\b\b 80%\b\b\b\b 81%\b\b\b\b 81%\b\b\b\b 82%\b\
  \b\b\b 83%\b\b\b\b 83%\b\b\b\b 83%\b\b\b\b 84%\b\b\b\b 84%\b\b\b\b 84%\b\b\b\b 85%\b\
  \b\b\b 86%\b\b\b\b 86%\b\b\b\b 86%\b\b\b\b 87%\b\b\b\b 87%\b\b\b\b 88%\b\b\b\b 88%\b\
  \b\b\b 89%\b\b\b\b 90%\b\b\b\b 90%\b\b\b\b 91%\b\b\b\b 91%\b\b\b\b 91%\b\b\b\b 91%\b\
  \b\b\b 92%\b\b\b\b 93%\b\b\b\b 93%\b\b\b\b 94%\b\b\b\b 94%\b\b\b\b 95%\b\b\b\b 95%\b\
  \b\b\b 96%\n     - Post analysis:  done\n   - Load spectra: 503\n     - Search progress:\
  \  96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 97%\n\
  \     - Post analysis:  done\n   - Load spectra: 502\n     - Search progress:  97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b\b\
  \ 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\
  \b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 97%\n     - Post analysis:  done\n   - Load spectra: 502\n     - Search progress:\
  \  97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 99%\n\
  \     - Post analysis:  done\n   - Load spectra: 158\n     - Search progress:  99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b100%\n     - Post\
  \ analysis:  done\n Search end:    02/19/2023, 06:16:43 AM\n\n\n\n6\nCometAdapter\
  \ took 06:19 m (wall), 0.94 s (CPU), 0.07 s (system), 0.87 s (user)."
exit_status: 9
kind: CalledProcessError
output: "Comet is writing the default parameter file...\nExecuting: comet.exe\nExecuted\
  \ comet.exe successfully!\nProgress of 'loading spectra list':\n\n-- done [took\
  \ 0.75 s (CPU), 0.80 s (Wall)] -- \nProgress of 'loading chromatogram list':\n\n\
  -- done [took 0.00 s (CPU), 0.00 s (Wall)] -- \nFATAL: External invocation of comet.exe\
  \ failed. Standard output and error were:\n<Executing: comet.exe> occurred 2 times\n\
  \ Comet version \"2016.01 rev. 3\"\n\n Search start:  02/19/2023, 06:10:25 AM\n\
  \ - Input file: MM15_Melanom_2_B_1.mzML\n   - Load spectra: 502\n     - Search progress:\
  \   0%\b\b\b\b  0%\b\b\b\b  1%\b\b\b\b  1%\b\b\b\b  2%\b\b\b\b  2%\b\b\b\b  3%\b\
  \b\b\b  4%\b\b\b\b  4%\b\b\b\b  5%\b\b\b\b  5%\b\b\b\b  6%\b\b\b\b  6%\b\b\b\b \
  \ 7%\b\b\b\b  7%\b\b\b\b  8%\b\b\b\b  8%\b\b\b\b  9%\b\b\b\b  9%\b\b\b\b 10%\b\b\
  \b\b 10%\b\b\b\b 11%\b\b\b\b 11%\b\b\b\b 11%\b\b\b\b 12%\b\b\b\b 12%\b\b\b\b 13%\b\
  \b\b\b 13%\b\b\b\b 13%\b\b\b\b 14%\b\b\b\b 14%\b\b\b\b 15%\b\b\b\b 15%\b\b\b\b 15%\b\
  \b\b\b 16%\b\b\b\b 17%\b\b\b\b 17%\b\b\b\b 17%\b\b\b\b 18%\b\b\b\b 18%\b\b\b\b 19%\b\
  \b\b\b 19%\b\b\b\b 19%\b\b\b\b 20%\b\b\b\b 20%\b\b\b\b 21%\b\b\b\b 21%\b\b\b\b 22%\b\
  \b\b\b 22%\b\b\b\b 23%\b\b\b\b 24%\b\b\b\b 24%\b\b\b\b 25%\b\b\b\b 25%\b\b\b\b 26%\b\
  \b\b\b 26%\b\b\b\b 27%\b\b\b\b 27%\b\b\b\b 27%\b\b\b\b 28%\b\b\b\b 28%\b\b\b\b 29%\b\
  \b\b\b 29%\b\b\b\b 30%\b\b\b\b 30%\b\b\b\b 31%\b\b\b\b 31%\b\b\b\b 32%\b\b\b\b 32%\b\
  \b\b\b 33%\b\b\b\b 33%\b\b\b\b 33%\b\b\b\b 34%\b\b\b\b 34%\b\b\b\b 35%\b\b\b\b 35%\b\
  \b\b\b 36%\b\b\b\b 36%\b\b\b\b 36%\b\b\b\b 37%\b\b\b\b 37%\b\b\b\b 38%\b\b\b\b 38%\b\
  \b\b\b 38%\b\b\b\b 39%\b\b\b\b 40%\b\b\b\b 40%\b\b\b\b 41%\b\b\b\b 41%\b\b\b\b 42%\b\
  \b\b\b 42%\b\b\b\b 43%\b\b\b\b 43%\b\b\b\b 43%\b\b\b\b 44%\b\b\b\b 44%\b\b\b\b 45%\b\
  \b\b\b 45%\b\b\b\b 46%\b\b\b\b 46%\b\b\b\b 47%\b\b\b\b 47%\b\b\b\b 48%\b\b\b\b 48%\b\
  \b\b\b 49%\b\b\b\b 49%\b\b\b\b 50%\b\b\b\b 50%\b\b\b\b 51%\b\b\b\b 52%\b\b\b\b 52%\b\
  \b\b\b 53%\b\b\b\b 53%\b\b\b\b 54%\b\b\b\b 54%\b\b\b\b 55%\b\b\b\b 55%\b\b\b\b 56%\b\
  \b\b\b 56%\b\b\b\b 56%\b\b\b\b 57%\b\b\b\b 57%\b\b\b\b 57%\b\b\b\b 58%\b\b\b\b 58%\b\
  \b\b\b 59%\b\b\b\b 59%\b\b\b\b 60%\b\b\b\b 61%\b\b\b\b 61%\b\b\b\b 61%\b\b\b\b 62%\b\
  \b\b\b 63%\b\b\b\b 63%\b\b\b\b 64%\b\b\b\b 64%\b\b\b\b 65%\b\b\b\b 65%\b\b\b\b 66%\b\
  \b\b\b 66%\b\b\b\b 66%\b\b\b\b 67%\b\b\b\b 67%\b\b\b\b 68%\b\b\b\b 68%\b\b\b\b 69%\b\
  \b\b\b 69%\b\b\b\b 69%\b\b\b\b 70%\b\b\b\b 70%\b\b\b\b 71%\b\b\b\b 71%\b\b\b\b 72%\b\
  \b\b\b 72%\b\b\b\b 73%\b\b\b\b 73%\b\b\b\b 74%\b\b\b\b 74%\b\b\b\b 75%\b\b\b\b 75%\b\
  \b\b\b 76%\b\b\b\b 76%\b\b\b\b 76%\b\b\b\b 77%\b\b\b\b 77%\b\b\b\b 78%\b\b\b\b 78%\b\
  \b\b\b 79%\b\b\b\b 79%\b\b\b\b 80%\b\b\b\b 80%\b\b\b\b 81%\b\b\b\b 81%\b\b\b\b 82%\b\
  \b\b\b 83%\b\b\b\b 83%\b\b\b\b 83%\b\b\b\b 84%\b\b\b\b 84%\b\b\b\b 84%\b\b\b\b 85%\b\
  \b\b\b 86%\b\b\b\b 86%\b\b\b\b 86%\b\b\b\b 87%\b\b\b\b 87%\b\b\b\b 88%\b\b\b\b 88%\b\
  \b\b\b 89%\b\b\b\b 90%\b\b\b\b 90%\b\b\b\b 91%\b\b\b\b 91%\b\b\b\b 91%\b\b\b\b 91%\b\
  \b\b\b 92%\b\b\b\b 93%\b\b\b\b 93%\b\b\b\b 94%\b\b\b\b 94%\b\b\b\b 95%\b\b\b\b 95%\b\
  \b\b\b 96%\n     - Post analysis:  done\n   - Load spectra: 503\n     - Search progress:\
  \  96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 97%\n\
  \     - Post analysis:  done\n   - Load spectra: 502\n     - Search progress:  97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b\b\
  \ 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\
  \b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 97%\n     - Post analysis:  done\n   - Load spectra: 502\n     - Search progress:\
  \  97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 99%\n\
  \     - Post analysis:  done\n   - Load spectra: 158\n     - Search progress:  99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b100%\n     - Post\
  \ analysis:  done\n Search end:    02/19/2023, 06:16:43 AM\n\n\n\n6\nCometAdapter\
  \ took 06:19 m (wall), 0.94 s (CPU), 0.07 s (system), 0.87 s (user)."
process: db_search_comet (4)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '11'
- ee1da19f530feb5de871daf91ee031

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fi7958rx9%2Flogs.tar.xz">0.074GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fi7958rx9%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.556GiB</td>
						<td>2558.6 seconds</td>
						<td>1350.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743143232">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.2.6">1.2.6</a>
						</td>
						<td>nextflow</td>
						<td>1446 days</td>
						<td>2023-02-19 04:17:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fsg22nhyq%2Flogs.tar.xz">0.107GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fsg22nhyq%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.769GiB</td>
						<td>2599.8 seconds</td>
						<td>1384.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743107584">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.0.6">2.0.6</a>
						</td>
						<td>nextflow</td>
						<td>1446 days</td>
						<td>2023-02-19 03:59:45</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 04:00:00.864
kind: NextflowMiscError
process: main
rest: Please specify either --singleEnd, --pairedEnd to execute the pipeline on FastQ
  files and --bam for previously processed BAM files!

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fl2bf1ilk%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fl2bf1ilk%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.554GiB</td>
						<td>17.7 seconds</td>
						<td>15.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743115744">
								<a href="https://nf-co.re/hlatyping">hlatyping</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hlatyping/tree/1.1.4">1.1.4</a>
						</td>
						<td>nextflow</td>
						<td>1445 days</td>
						<td>2023-02-19 06:22:05</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fvznf96d5%2Flogs.tar.xz">0.038GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fvznf96d5%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>1.504GiB</td>
						<td>451.1 seconds</td>
						<td>257.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738325616">
								<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2">snakemake-workflows/rna-seq-star-deseq2</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>1439 days</td>
						<td>2023-02-19 04:17:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- rules
- trim.smk
kind: MissingInputException
rest: "Missing input files for rule cutadapt_pe:\n    output: trimmed/A-lane1.1.fastq.gz,\
  \ trimmed/A-lane1.2.fastq.gz, trimmed/A-lane1.qc.txt\n    wildcards: sample=A, unit=lane1\n\
  \    affected files:\n        A.1.fq.gz\n        A.2.fq.gz"
rule: cutadapt_pe  in line 5

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-star-deseq2%2F2p12rkoh%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-star-deseq2%2F2p12rkoh%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.103GiB</td>
						<td>1.9 seconds</td>
						<td>2.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743187584">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/1.3">1.3</a>
						</td>
						<td>nextflow</td>
						<td>1425 days</td>
						<td>2023-02-19 06:53:03</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Ffwdzn0z0%2Flogs.tar.xz">0.024GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Ffwdzn0z0%2Foutput.tar.xz">0.005GiB</a>
						</td>
						<td>2.132GiB</td>
						<td>758.2 seconds</td>
						<td>381.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738290448">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>1419 days</td>
						<td>2023-02-19 06:15:07</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- rules
- common.smk
kind: WorkflowError
line_no: 10
rest: "Workflow defines configfile config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/ujpkna4li7/code/Snakefile\", line 1, in <module>\n\
  \  File \"/home/azureuser/tmp/ujpkna4li7/code/rules/common.smk\", line 10, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Ff1fhrrt6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Ff1fhrrt6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>2.0 seconds</td>
						<td>4.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738290208">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>1419 days</td>
						<td>2023-02-19 04:31:06</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- rules
- trim.smk
kind: MissingInputException
rest: "Missing input files for rule cutadapt_pe:\n    output: trimmed/A-lane1.1.fastq.gz,\
  \ trimmed/A-lane1.2.fastq.gz, trimmed/A-lane1.qc.txt\n    wildcards: sample=A, unit=lane1\n\
  \    affected files:\n        A.1.fq.gz\n        A.2.fq.gz"
rule: cutadapt_pe  in line 1

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fdaq3eh0u%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fdaq3eh0u%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.104GiB</td>
						<td>2.0 seconds</td>
						<td>2.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743150192">
								<a href="https://nf-co.re/rnafusion">rnafusion</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnafusion/tree/1.0.1">1.0.1</a>
						</td>
						<td>nextflow</td>
						<td>1414 days</td>
						<td>2023-02-19 06:17:53</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 06:18:19.621
kind: NextflowMiscError
process: main
rest: 'Fasta file not found: tests/genome.fa'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Fnkc6n55p%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Fnkc6n55p%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.437GiB</td>
						<td>20.8 seconds</td>
						<td>26.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745623072">
								<a href="https://nf-co.re/atacseq">atacseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/atacseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1412 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2F8951wp1v%2Flogs.tar.xz">0.353GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2F8951wp1v%2Foutput.tar.xz">0.062GiB</a>
						</td>
						<td>2.487GiB</td>
						<td>1159.0 seconds</td>
						<td>644.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743147312">
								<a href="https://nf-co.re/nascent">nascent</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/nascent/tree/1.0">1.0</a>
						</td>
						<td>nextflow</td>
						<td>1405 days</td>
						<td>2023-02-19 04:43:39</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: nextflow.exception.IllegalDirectiveException
msg: 'Unknown process directive: `validExitStatus`'
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.script.ProcessConfig.checkName(ProcessConfig.groovy:218)\n\tat nextflow.script.ProcessConfig.methodMissing(ProcessConfig.groovy:237)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaClassImpl.invokeMissingMethod(MetaClassImpl.java:953)\n\tat\
  \ groovy.lang.MetaClassImpl.invokePropertyOrMissing(MetaClassImpl.java:1347)\n\t\
  at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1270)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\
  \tat groovy.lang.MetaObjectProtocol$invokeMethod$2.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:157)\n\
  \tat nextflow.script.ProcessConfig.invokeMethod(ProcessConfig.groovy:233)\n\tat\
  \ org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:408)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:203)\n\
  \tat Script_c725d8c0$_runScript_closure22.doCall(Script_c725d8c0:295)\n\tat Script_c725d8c0$_runScript_closure22.doCall(Script_c725d8c0)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.ProcessFactory.createProcessor(ProcessFactory.groovy:95)\n\
  \tat nextflow.script.ProcessFactory$createProcessor.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.script.BaseScript.process(BaseScript.groovy:114)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat Script_c725d8c0.runScript(Script_c725d8c0:294)\n\tat nextflow.script.BaseScript.runDsl1(BaseScript.groovy:164)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\
  \tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\
  \tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nascent%2Fvxtonl6n%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nascent%2Fvxtonl6n%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.576GiB</td>
						<td>22.9 seconds</td>
						<td>13.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746032480">
								<a href="https://nf-co.re/airrflow">airrflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/airrflow/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1405 days</td>
						<td>2023-02-19 04:30:37</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: TIgGER-shazam.R blast_parse-select.tab imgtdb_base/human/vdj/imgt_human_IGHV.fasta
error: "Saving 7 x 7 in image\nError in h.ucv.default(unique(distances), 4) : \n \
  \ argument 'x' must be numeric and need at least 3 data points\nCalls: findThreshold\
  \ -> smoothValley -> <Anonymous> -> h.ucv.default\nExecution halted"
exit_status: 1
kind: CalledProcessError
output: (empty)
process: shazam (Sample4)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '81'
- 4187f9217868f475b6fa762fd500a9

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Fzsq2pkk8%2Flogs.tar.xz">0.035GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Fzsq2pkk8%2Foutput.tar.xz">0.020GiB</a>
						</td>
						<td>1.700GiB</td>
						<td>1261.0 seconds</td>
						<td>583.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742917552">
								<a href="https://nf-co.re/coproid">coproid</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/coproid/tree/1.0">1.0</a>
						</td>
						<td>nextflow</td>
						<td>1392 days</td>
						<td>2023-02-19 03:31:45</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "sourcepredict -di 2 \\\n              -k 5 \\\n              -l\
  \ test_labels.csv \\\n              -s test_sources.csv \\\n              -t 2 \\\
  \n              -o prediction.sourcepredict.csv \\\n              -e sourcepredict_embedding.csv\
  \ kraken_merged.csv"
error: "2456000 generating entries... \n 2457000 generating entries... \n 2458000\
  \ generating entries... \n 2459000 generating entries... \n 2460000 generating entries...\
  \ \n 2461000 generating entries... \n 2462000 generating entries... \n 2463000 generating\
  \ entries... \n 2464000 generating entries... \n 2465000 generating entries... \n\
  \ 2466000 generating entries... \n 2467000 generating entries... \n 2468000 generating\
  \ entries... \n 2469000 generating entries... \n 2470000 generating entries... \n\
  \ 2471000 generating entries... \n 2472000 generating entries... \n 2473000 generating\
  \ entries... \n 2474000 generating entries... \n 2475000 generating entries... \n\
  \ 2476000 generating entries... \n 2477000 generating entries... \n 2478000 generating\
  \ entries... \n 2479000 generating entries... \n 2480000 generating entries... \n\
  \ 2481000 generating entries... \n 2482000 generating entries... \n 2483000 generating\
  \ entries... \n 2484000 generating entries... \n 2485000 generating entries... \n\
  \ 2486000 generating entries... \n 2487000 generating entries... \n 2488000 generating\
  \ entries... \n 2489000 generating entries... \nUploading to /home/azureuser/.etetoolkit/taxa.sqlite\n\
  \nTraceback (most recent call last):\n  File \"/opt/conda/envs/nf-core-coproid-1.0/bin/sourcepredict\"\
  , line 139, in <module>\n    a.compute_distance(rank=tax_rank)\n  File \"/opt/conda/envs/nf-core-coproid-1.0/bin/sourcepredictlib/ml.py\"\
  , line 126, in compute_distance\n    ncbi = NCBITaxa()\n  File \"/opt/conda/envs/nf-core-coproid-1.0/lib/python3.6/site-packages/ete3/ncbi_taxonomy/ncbiquery.py\"\
  , line 110, in __init__\n    self.update_taxonomy_database(taxdump_file)\n  File\
  \ \"/opt/conda/envs/nf-core-coproid-1.0/lib/python3.6/site-packages/ete3/ncbi_taxonomy/ncbiquery.py\"\
  , line 129, in update_taxonomy_database\n    update_db(self.dbfile)\n  File \"/opt/conda/envs/nf-core-coproid-1.0/lib/python3.6/site-packages/ete3/ncbi_taxonomy/ncbiquery.py\"\
  , line 760, in update_db\n    upload_data(dbfile)\n  File \"/opt/conda/envs/nf-core-coproid-1.0/lib/python3.6/site-packages/ete3/ncbi_taxonomy/ncbiquery.py\"\
  , line 802, in upload_data\n    db.execute(\"INSERT INTO synonym (taxid, spname)\
  \ VALUES (?, ?);\", (taxid, spname))\nsqlite3.IntegrityError: UNIQUE constraint\
  \ failed: synonym.spname, synonym.taxid"
exit_status: 1
kind: CalledProcessError
output: "2442000 generating entries... \n 2443000 generating entries... \n 2444000\
  \ generating entries... \n 2445000 generating entries... \n 2446000 generating entries...\
  \ \n 2447000 generating entries... \n 2448000 generating entries... \n 2449000 generating\
  \ entries... \n 2450000 generating entries... \n 2451000 generating entries... \n\
  \ 2452000 generating entries... \n 2453000 generating entries... \n 2454000 generating\
  \ entries... \n 2455000 generating entries... \n 2456000 generating entries... \n\
  \ 2457000 generating entries... \n 2458000 generating entries... \n 2459000 generating\
  \ entries... \n 2460000 generating entries... \n 2461000 generating entries... \n\
  \ 2462000 generating entries... \n 2463000 generating entries... \n 2464000 generating\
  \ entries... \n 2465000 generating entries... \n 2466000 generating entries... \n\
  \ 2467000 generating entries... \n 2468000 generating entries... \n 2469000 generating\
  \ entries... \n 2470000 generating entries... \n 2471000 generating entries... \n\
  \ 2472000 generating entries... \n 2473000 generating entries... \n 2474000 generating\
  \ entries... \n 2475000 generating entries... \n 2476000 generating entries... \n\
  \ 2477000 generating entries... \n 2478000 generating entries... \n 2479000 generating\
  \ entries... \n 2480000 generating entries... \n 2481000 generating entries... \n\
  \ 2482000 generating entries... \n 2483000 generating entries... \n 2484000 generating\
  \ entries... \n 2485000 generating entries... \n 2486000 generating entries... \n\
  \ 2487000 generating entries... \n 2488000 generating entries... \n 2489000 generating\
  \ entries... \nUploading to /home/azureuser/.etetoolkit/taxa.sqlite"
process: sourcepredict
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- 0f
- ab8635bf5dd9a8f79311aaf8b60151

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/coproid%2Faxynftr5%2Flogs.tar.xz">0.325GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/coproid%2Faxynftr5%2Foutput.tar.xz">0.029GiB</a>
						</td>
						<td>2.942GiB</td>
						<td>1147.5 seconds</td>
						<td>753.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745943696">
								<a href="https://nf-co.re/bacass">bacass</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/bacass/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1385 days</td>
						<td>2023-02-19 06:33:12</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: prokka --cpus 2 --prefix "ERR044595" --outdir ERR044595_annotation  --fast
  /home/azureuser/tmp/o1yhopltrb/code/work/2d/a2e47f721441d2c3ef870d1c663b82/ERR044595_assembly.fasta
error: '[06:48:15] Fixed 2 duplicate /gene - hlb_1 hlb_2

  [06:48:15] Fixed 2 duplicate /gene - kipA_1 kipA_2

  [06:48:15] Fixed 2 duplicate /gene - mecA_1 mecA_2

  [06:48:15] Fixed 2 duplicate /gene - arcB_1 arcB_2

  [06:48:15] Fixed 2 duplicate /gene - ghrB_1 ghrB_2

  [06:48:15] Fixed 3 duplicate /gene - entA_1 entA_2 entA_3

  [06:48:15] Fixed 3 duplicate /gene - atl_1 atl_2 atl_3

  [06:48:15] Fixed 3 duplicate /gene - bceB_1 bceB_2 bceB_3

  [06:48:15] Fixed 2 duplicate /gene - yciC_1 yciC_2

  [06:48:15] Fixed 2 duplicate /gene - fhuD_1 fhuD_2

  [06:48:15] Fixed 4 duplicate /gene - map_1 map_2 map_3 map_4

  [06:48:15] Fixed 2 duplicate /gene - lacC_1 lacC_2

  [06:48:15] Fixed 2 duplicate /gene - pdhC_1 pdhC_2

  [06:48:15] Fixed 2 duplicate /gene - mnaA_1 mnaA_2

  [06:48:15] Fixed 2 duplicate /gene - emrB_1 emrB_2

  [06:48:15] Fixed 2 duplicate /gene - rluD_1 rluD_2

  [06:48:15] Fixed 2 duplicate /gene - mleN_1 mleN_2

  [06:48:15] Fixed 2 duplicate /gene - cspA_1 cspA_2

  [06:48:15] Fixed 2 duplicate /gene - aldC_1 aldC_2

  [06:48:15] Fixed 2 duplicate /gene - spsB_1 spsB_2

  [06:48:15] Fixed 2 duplicate /gene - alr1_1 alr1_2

  [06:48:15] Fixed 2 duplicate /gene - secA1_1 secA1_2

  [06:48:15] Fixed 2 duplicate /gene - lctP_1 lctP_2

  [06:48:15] Fixed 2 duplicate /gene - cap8A_1 cap8A_2

  [06:48:15] Fixed 2 duplicate /gene - garK_1 garK_2

  [06:48:15] Fixed 2 duplicate /gene - lpl2_1 lpl2_2

  [06:48:15] Fixed 3 duplicate /gene - scn_1 scn_2 scn_3

  [06:48:15] Fixed 2 duplicate /gene - phnE_1 phnE_2

  [06:48:15] Fixed 2 duplicate /gene - lipA_1 lipA_2

  [06:48:15] Fixed 3 duplicate /gene - opuD_1 opuD_2 opuD_3

  [06:48:15] Fixed 2 duplicate /gene - nhaK_1 nhaK_2

  [06:48:15] Fixed 2 duplicate /gene - ftsK_1 ftsK_2

  [06:48:15] Fixed 2 duplicate /gene - cls_1 cls_2

  [06:48:15] Fixed 2 duplicate /gene - bcr_1 bcr_2

  [06:48:15] Fixed 2 duplicate /gene - gatB_1 gatB_2

  [06:48:15] Fixed 2 duplicate /gene - metQ_1 metQ_2

  [06:48:15] Fixed 2 duplicate /gene - mnhD1_1 mnhD1_2

  [06:48:15] Fixed 2 duplicate /gene - tagH_1 tagH_2

  [06:48:15] Fixed 2 duplicate /gene - gpmA_1 gpmA_2

  [06:48:15] Fixed 2 duplicate /gene - argR_1 argR_2

  [06:48:15] Fixed 2 duplicate /gene - nikB_1 nikB_2

  [06:48:15] Fixed 2 duplicate /gene - oatA_1 oatA_2

  [06:48:15] Fixed 98 colliding /gene names.

  [06:48:15] Adding /locus_tag identifiers

  [06:48:15] Assigned 2585 locus_tags to CDS and RNA features.

  [06:48:15] Writing outputs to ERR044595_annotation/

  [06:48:17] Generating annotation statistics file

  [06:48:17] Generating Genbank and Sequin files

  [06:48:17] Running: tbl2asn -V b -a r10k -l paired-ends -M n -N 1 -y ''Annotated
  using prokka 1.13.3 from https://github.com/tseemann/prokka'' -Z ERR044595_annotation\/ERR044595\.err
  -i ERR044595_annotation\/ERR044595\.fsa 2> /dev/null

  [06:48:23] Could not run command: tbl2asn -V b -a r10k -l paired-ends -M n -N 1
  -y ''Annotated using prokka 1.13.3 from https://github.com/tseemann/prokka'' -Z
  ERR044595_annotation\/ERR044595\.err -i ERR044595_annotation\/ERR044595\.fsa 2>
  /dev/null'
exit_status: 2
kind: CalledProcessError
output: (empty)
process: prokka (ERR044595)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '70'
- 63f9e30eb1ec0538269019b279c42d

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/bacass%2Fuw8py88q%2Flogs.tar.xz">0.650GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/bacass%2Fuw8py88q%2Foutput.tar.xz">0.219GiB</a>
						</td>
						<td>1.848GiB</td>
						<td>2065.5 seconds</td>
						<td>927.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738269440">
								<a href="https://github.com/GooglingTheCancerGenome/sv-callers">GooglingTheCancerGenome/sv-callers</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>1384 days</td>
						<td>2023-02-19 04:34:04</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2F9aue4sj1%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2F9aue4sj1%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743114544">
								<a href="https://nf-co.re/hic">hic</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hic/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1384 days</td>
						<td>2023-02-19 03:53:23</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2Febi7e72r%2Flogs.tar.xz">0.208GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2Febi7e72r%2Foutput.tar.xz">0.038GiB</a>
						</td>
						<td>1.570GiB</td>
						<td>724.1 seconds</td>
						<td>341.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743149952">
								<a href="https://nf-co.re/rnafusion">rnafusion</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnafusion/tree/1.0.2">1.0.2</a>
						</td>
						<td>nextflow</td>
						<td>1375 days</td>
						<td>2023-02-19 06:00:41</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 06:01:02.044
kind: NextflowMiscError
process: Actor Thread 5
rest: 'Cannot find any reads matching: tests/reads_{1,2}.fq.gz'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Ffs409njz%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Ffs409njz%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.495GiB</td>
						<td>21.7 seconds</td>
						<td>21.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744558832">
								<a href="https://nf-co.re/chipseq">chipseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/chipseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1354 days</td>
						<td>2023-02-19 06:16:19</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2Fbcmva5qo%2Flogs.tar.xz">0.500GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2Fbcmva5qo%2Foutput.tar.xz">0.077GiB</a>
						</td>
						<td>2.576GiB</td>
						<td>3481.4 seconds</td>
						<td>1911.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743107344">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.0.7">2.0.7</a>
						</td>
						<td>nextflow</td>
						<td>1349 days</td>
						<td>2023-02-19 06:27:00</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 06:27:09.086
kind: NextflowMiscError
process: main
rest: Please specify either --singleEnd, --pairedEnd to execute the pipeline on FastQ
  files and --bam for previously processed BAM files!

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F6uv8emav%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F6uv8emav%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.567GiB</td>
						<td>23.1 seconds</td>
						<td>8.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743115504">
								<a href="https://nf-co.re/hlatyping">hlatyping</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hlatyping/tree/1.1.5">1.1.5</a>
						</td>
						<td>nextflow</td>
						<td>1333 days</td>
						<td>2023-02-19 07:01:04</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Frap7dd0y%2Flogs.tar.xz">0.038GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Frap7dd0y%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>1.600GiB</td>
						<td>535.7 seconds</td>
						<td>240.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744967760">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>1315 days</td>
						<td>2023-02-19 03:56:05</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: metadataCategoryPairwise.r Metadata.tsv
error: "Warning message:\nIn read.table(file = file, header = header, sep = sep, quote\
  \ = quote,  :\n  incomplete final line found by readTableHeader on 'Metadata.tsv'\n\
  Error in 1:ncol(data) : argument of length 0\nExecution halted"
exit_status: 1
kind: CalledProcessError
output: (empty)
process: metadata_category_pairwise (1)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- 9f
- d89e77232bce349c460cb03d38d5f1

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fvakdgfjk%2Flogs.tar.xz">0.010GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fvakdgfjk%2Foutput.tar.xz">0.008GiB</a>
						</td>
						<td>1.805GiB</td>
						<td>1144.4 seconds</td>
						<td>608.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743142992">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.3">1.3</a>
						</td>
						<td>nextflow</td>
						<td>1295 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fl6li4rbf%2Flogs.tar.xz">0.107GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fl6li4rbf%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.827GiB</td>
						<td>2576.5 seconds</td>
						<td>1182.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743194064">
								<a href="https://nf-co.re/smrnaseq">smrnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/smrnaseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1248 days</td>
						<td>2023-02-19 04:22:43</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: edgeR_miRBase.r sample_2.mature.stats sample_3.mature.stats sample_1.mature.stats
  sample_2.hairpin.stats sample_1.hairpin.stats sample_3.hairpin.stats
error: "Loading required package: limma\nLoading required package: edgeR\nLoading\
  \ required package: statmod\nLoading required package: data.table\nLoading required\
  \ package: gplots\n\nAttaching package: \u2018gplots\u2019\n\nThe following object\
  \ is masked from \u2018package:stats\u2019:\n\n    lowess\n\n$mature\n[1] \"sample_2.mature.stats\"\
  \ \"sample_3.mature.stats\" \"sample_1.mature.stats\"\n\n$hairpin\n[1] \"sample_2.hairpin.stats\"\
  \ \"sample_1.hairpin.stats\" \"sample_3.hairpin.stats\"\n\nError in heatmap.2(cpm(dataNorm),\
  \ col = redgreen(100), key = TRUE, scale = \"row\",  : \n  `x' must be a numeric\
  \ matrix\nIn addition: Warning message:\nIn DGEList(counts = data, genes = rownames(data))\
  \ :\n  library size of zero detected\nExecution halted"
exit_status: 1
kind: CalledProcessError
output: '$mature

  [1] "sample_2.mature.stats" "sample_3.mature.stats" "sample_1.mature.stats"


  $hairpin

  [1] "sample_2.hairpin.stats" "sample_1.hairpin.stats" "sample_3.hairpin.stats"'
process: edgeR_mirna
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- db
- aed2d2c6dd0c138a42818cf4dd8f99

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/smrnaseq%2Fijd11djx%2Flogs.tar.xz">0.094GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/smrnaseq%2Fijd11djx%2Foutput.tar.xz">0.008GiB</a>
						</td>
						<td>1.657GiB</td>
						<td>721.5 seconds</td>
						<td>378.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743191904">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/2.5">2.5</a>
						</td>
						<td>nextflow</td>
						<td>1230 days</td>
						<td>2023-02-19 16:49:25</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Ftnhs4n65%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Ftnhs4n65%2Foutput.tar.xz">0.015GiB</a>
						</td>
						<td>2.666GiB</td>
						<td>1107.6 seconds</td>
						<td>534.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738289968">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.0.0">v2.0.0</a>
						</td>
						<td>snakemake</td>
						<td>1223 days</td>
						<td>2023-02-19 06:00:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/7b9a4hoj2b/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/4a58f83f-cabe-43aa-bf5a-f54d682721d2\
  \ singularity  exec --home /home/azureuser/tmp/7b9a4hoj2b/code/.test  /home/azureuser/tmp/7b9a4hoj2b/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/7b9a4hoj2b/code/.test/.snakemake/conda/9c5ecf5e27e878c94180b37675021de6_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/7b9a4hoj2b/code/.test/.snakemake/conda/9c5ecf5e27e878c94180b37675021de6_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/8 [00:00<?, ?it/s]\nExamining pandas=0.25:   0%|          |\
  \ 0/8 [00:00<?, ?it/s] \nExamining @/linux-64::__glibc==2.31=0:  12%|\u2588\u258E\
  \        | 1/8 [00:05<00:40,  5.81s/it]\nExamining @/linux-64::__glibc==2.31=0:\
  \  25%|\u2588\u2588\u258C       | 2/8 [00:05<00:17,  2.90s/it]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  25%|\u2588\u2588\u258C       | 2/8 [00:05<00:17,  2.90s/it]\nExamining @/linux-64::__unix==0=0:\
  \  38%|\u2588\u2588\u2588\u258A      | 3/8 [00:05<00:14,  2.90s/it]      \nExamining\
  \ @/linux-64::__archspec==1=x86_64:  50%|\u2588\u2588\u2588\u2588\u2588     | 4/8\
  \ [00:05<00:11,  2.90s/it]\nExamining python=3.7:  62%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258E   | 5/8 [00:05<00:08,  2.90s/it]                      \nExamining goatools=0.9.7:\
  \  75%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258C  | 6/8 [00:10<00:05,  2.90s/it]\n\
  Examining goatools=0.9.7:  88%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \ | 7/8 [00:10<00:01,  1.41s/it]\nExamining matplotlib=3.1:  88%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [00:11<00:01,  1.41s/it]\nExamining matplotlib=3.1:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 8/8 [00:11<00:00,\
  \  1.30s/it]\n                                                                 \
  \      \n\nDetermining conflicts:   0%|          | 0/8 [00:00<?, ?it/s]\nExamining\
  \ conflict for pandas goatools:   0%|          | 0/8 [00:00<?, ?it/s]\nExamining\
  \ conflict for pandas matplotlib python goatools:  12%|\u2588\u258E        | 1/8\
  \ [00:04<00:28,  4.06s/it]\nExamining conflict for pandas matplotlib python goatools:\
  \  25%|\u2588\u2588\u258C       | 2/8 [00:04<00:12,  2.03s/it]\nExamining conflict\
  \ for pandas matplotlib goatools:  25%|\u2588\u2588\u258C       | 2/8 [00:14<00:12,\
  \  2.03s/it]       \nExamining conflict for pandas matplotlib goatools:  38%|\u2588\
  \u2588\u2588\u258A      | 3/8 [00:14<00:28,  5.62s/it]\nExamining conflict for pandas\
  \ matplotlib python:  38%|\u2588\u2588\u2588\u258A      | 3/8 [01:30<00:28,  5.62s/it]\
  \  \nExamining conflict for pandas matplotlib python:  50%|\u2588\u2588\u2588\u2588\
  \u2588     | 4/8 [01:30<02:07, 31.89s/it]\nExamining conflict for pandas matplotlib:\
  \  50%|\u2588\u2588\u2588\u2588\u2588     | 4/8 [01:43<02:07, 31.89s/it]       \n\
  Examining conflict for pandas matplotlib:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258E   | 5/8 [01:43<01:15, 25.16s/it]\nExamining conflict for pandas matplotlib\
  \ __glibc python:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\u258E   | 5/8 [01:45<01:15,\
  \ 25.16s/it]\nExamining conflict for pandas matplotlib __glibc python:  75%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258C  | 6/8 [01:45<00:35, 17.69s/it]\nExamining\
  \ conflict for python goatools:  75%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258C\
  \  | 6/8 [01:53<00:35, 17.69s/it]                 \nExamining conflict for python\
  \ goatools:  88%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [01:53<00:14,\
  \ 14.47s/it]\nExamining conflict for matplotlib goatools:  88%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [01:57<00:14, 14.47s/it]\nExamining conflict\
  \ for matplotlib goatools: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588| 8/8 [01:57<00:00, 11.18s/it]\n                                   \
  \                                                      \nfailed\nSolving environment:\
  \ ...working... \nFound conflicts! Looking for incompatible packages.\nThis can\
  \ take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|\
  \          | 0/8 [00:00<?, ?it/s]\nExamining pandas=0.25:   0%|          | 0/8 [00:00<?,\
  \ ?it/s] \nExamining @/linux-64::__glibc==2.31=0:  12%|\u2588\u258E        | 1/8\
  \ [00:06<00:47,  6.78s/it]\nExamining @/linux-64::__glibc==2.31=0:  25%|\u2588\u2588\
  \u258C       | 2/8 [00:06<00:20,  3.39s/it]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  25%|\u2588\u2588\u258C       | 2/8 [00:06<00:20,  3.39s/it]\nExamining @/linux-64::__unix==0=0:\
  \  38%|\u2588\u2588\u2588\u258A      | 3/8 [00:06<00:16,  3.39s/it]      \nExamining\
  \ @/linux-64::__archspec==1=x86_64:  50%|\u2588\u2588\u2588\u2588\u2588     | 4/8\
  \ [00:06<00:13,  3.39s/it]\nExamining python=3.7:  62%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258E   | 5/8 [00:06<00:10,  3.39s/it]                      \nExamining goatools=0.9.7:\
  \  75%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258C  | 6/8 [00:11<00:06,  3.39s/it]\n\
  Examining goatools=0.9.7:  88%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \ | 7/8 [00:11<00:01,  1.54s/it]\nExamining matplotlib=3.1:  88%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [00:12<00:01,  1.54s/it]\nExamining matplotlib=3.1:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 8/8 [00:12<00:00,\
  \  1.46s/it]\n                                                                 \
  \      \n\nDetermining conflicts:   0%|          | 0/8 [00:00<?, ?it/s]\nExamining\
  \ conflict for pandas goatools:   0%|          | 0/8 [00:00<?, ?it/s]\nExamining\
  \ conflict for pandas matplotlib python goatools:  12%|\u2588\u258E        | 1/8\
  \ [00:04<00:29,  4.15s/it]\nExamining conflict for pandas matplotlib python goatools:\
  \  25%|\u2588\u2588\u258C       | 2/8 [00:04<00:12,  2.08s/it]\nExamining conflict\
  \ for pandas matplotlib goatools:  25%|\u2588\u2588\u258C       | 2/8 [00:14<00:12,\
  \  2.08s/it]       \nExamining conflict for pandas matplotlib goatools:  38%|\u2588\
  \u2588\u2588\u258A      | 3/8 [00:14<00:27,  5.48s/it]\nExamining conflict for pandas\
  \ matplotlib python:  38%|\u2588\u2588\u2588\u258A      | 3/8 [01:33<00:27,  5.48s/it]\
  \  \nExamining conflict for pandas matplotlib python:  50%|\u2588\u2588\u2588\u2588\
  \u2588     | 4/8 [01:33<02:12, 33.09s/it]\nExamining conflict for pandas matplotlib:\
  \  50%|\u2588\u2588\u2588\u2588\u2588     | 4/8 [01:50<02:12, 33.09s/it]       \n\
  Examining conflict for pandas matplotlib:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258E   | 5/8 [01:50<01:22, 27.45s/it]\nExamining conflict for pandas matplotlib\
  \ __glibc python:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\u258E   | 5/8 [01:54<01:22,\
  \ 27.45s/it]\nExamining conflict for pandas matplotlib __glibc python:  75%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258C  | 6/8 [01:54<00:39, 19.63s/it]\nExamining\
  \ conflict for python goatools:  75%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258C\
  \  | 6/8 [02:04<00:39, 19.63s/it]                 \nExamining conflict for python\
  \ goatools:  88%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [02:04<00:16,\
  \ 16.36s/it]\nExamining conflict for matplotlib goatools:  88%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [02:08<00:16, 16.36s/it]\nExamining conflict\
  \ for matplotlib goatools: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588| 8/8 [02:08<00:00, 12.70s/it]\n                                   \
  \                                                      \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with each other:\n\n\
  Output in format: Requested package -> Available versions\n\nPackage ca-certificates\
  \ conflicts for:\npython=3.7 -> openssl[version='>=1.1.1l,<1.1.2a'] -> ca-certificates\n\
  goatools=0.9.7 -> python -> ca-certificates\n\nPackage python conflicts for:\ngoatools=0.9.7\
  \ -> python\nmatplotlib=3.1 -> pyqt -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.6|>=3.0,<=3.9|>=3.7|>=3.8|>=3']\n\
  goatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  matplotlib=3.1 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  python=3.7\n\nPackage pyparsing conflicts for:\nmatplotlib=3.1 -> pyparsing\ngoatools=0.9.7\
  \ -> pydot -> pyparsing[version='>=2.1.4|>=2.1.4,<3']\n\nPackage numpy-base conflicts\
  \ for:\nmatplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py36h2b20989_6|py37h2b20989_6|py37hdbf6ddf_6|py27hdbf6ddf_6|py27h2b20989_7|py36hdbf6ddf_7|py36h2b20989_7|py35hdbf6ddf_7|py37hdbf6ddf_7|py27h2b20989_7|py36h2b20989_7|py37hdbf6ddf_7|py36h2b20989_8|py37hdbf6ddf_8|py35h7cdd4dd_9|py37h3dfced4_9|py36h3dfced4_9|py35h3dfced4_9|py27h74e8950_9|py35h74e8950_9|py37h74e8950_9|py27h81de0dd_9|py27h74e8950_10|py27h81de0dd_10|py37h81de0dd_10|py36h81de0dd_10|py35h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py37h2f8d375_11|py36h2f8d375_11|py36hde5b4d6_11|py27hde5b4d6_11|py27h2f8d375_12|py37hde5b4d6_12|py27h0ea5e3f_1|py36h9be14a7_1|py35h9be14a7_1|py36h2b20989_0|py35hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_1|py36hdbf6ddf_1|py27h2b20989_1|py27h2b20989_2|py36h2b20989_2|py37hdbf6ddf_2|py36hdbf6ddf_2|py27h2b20989_4|py27hdbf6ddf_4|py37h2b20989_4|py37hdbf6ddf_4|py36h2b20989_4|py35h2b20989_4|py37hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_4|py35h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py35h81de0dd_4|py37h81de0dd_4|py36h81de0dd_4|py37h2f8d375_5|py37hde5b4d6_5|py36hde5b4d6_5|py27h7cdd4dd_0|py36h7cdd4dd_0|py36h3dfced4_0|py35h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py27h74e8950_0|py27h81de0dd_0|py35h81de0dd_0|py27h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py36h2f8d375_1|py27h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_1|py36h2f8d375_1|py37hde5b4d6_1|py27hde5b4d6_1|py27h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_1|py37h2f8d375_1|py37hde5b4d6_1|py27h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py39h76555f2_1|py38h41b4c56_3|py37hdc34a94_3|py37ha8aedfd_4|py39ha8aedfd_4|py37hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py37h4c65ebe_1|py39h4c65ebe_1|py38h4c65ebe_1|py37h21a3de8_1|py37h34387ca_0|py39h34387ca_0|py38he2ba247_0|py38hfae3a4d_0|py39he2ba247_0|py38h39b7dee_0|py37h39b7dee_0|py39h39b7dee_0|py39h79a1101_0|py37h79a1101_0|py39hf524024_1|py37hf524024_1|py310h9585f30_2|py310h375b286_3|py38ha15fc14_3|py310hcba007f_3|py39hf524024_0|py38hf524024_0|py39hb8be1f0_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_1|py38h31eccc5_1|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py39h31eccc5_0|py310h8e6c178_0|py38h31eccc5_0|py39h31eccc5_1|py310h8e6c178_1|py310h375b286_1|py38h1e6e340_1|py310hcba007f_0|py38hb8be1f0_0|py310hf2716ce_0|py310h9585f30_0|py39ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py37h1e6e340_3|py38hf524024_2|py37hf524024_2|py39hf524024_2|py38hb8be1f0_2|py37hb8be1f0_2|py39hb8be1f0_2|py310hf2716ce_2|py310h9585f30_1|py310hf2716ce_1|py38hf524024_1|py37hb8be1f0_1|py38hb8be1f0_1|py39hb8be1f0_1|py310h79a1101_0|py310h2b8c604_0|py38h2b8c604_0|py39h2b8c604_0|py37h2b8c604_0|py38h79a1101_0|py38h74d4b33_0|py39h74d4b33_0|py37h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py37hfae3a4d_0|py39h7d8b39e_0|py38h7d8b39e_0|py38h34387ca_0|py37h7d8b39e_0|py38h21a3de8_1|py39h21a3de8_1|py39h0f7b65f_0|py39h2ae0177_0|py38hfa32c7d_0|py37h75fe3a5_0|py38h75fe3a5_0|py37hfa32c7d_0|py38hfa32c7d_0|py38h75fe3a5_0|py38h73d599e_4|py39h73d599e_4|py37h73d599e_4|py38ha8aedfd_4|py39hdc34a94_3|py38hdc34a94_3|py36hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py39h41b4c56_3|py39hfb011de_1|py38hde5b4d6_0|py38h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py36h2f8d375_1|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py37h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py36h81de0dd_1|py27h2f8d375_1|py36h81de0dd_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py35h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py36h74e8950_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py27hde5b4d6_5|py27h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py38h2f8d375_4|py27h81de0dd_4|py37h2f8d375_0|py36h2f8d375_0|py35hdbf6ddf_4|py36hdbf6ddf_4|py27hdbf6ddf_3|py27h2b20989_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py37h2b20989_3|py36h2b20989_3|py27hdbf6ddf_2|py37h2b20989_2|py37hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py27hdbf6ddf_0|py36hdbf6ddf_0|py27h2b20989_0|py36h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35h2b20989_0|py27h2b20989_0|py27h9be14a7_1|py35h0ea5e3f_1|py36h0ea5e3f_1|py38h2f8d375_12|py38hde5b4d6_12|py36hde5b4d6_12|py27hde5b4d6_12|py36h2f8d375_12|py37h2f8d375_12|py37hde5b4d6_11|py27h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py36h74e8950_10|py35h74e8950_10|py37h74e8950_10|py35h81de0dd_9|py36h74e8950_9|py37h81de0dd_9|py36h81de0dd_9|py27h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py36h7cdd4dd_9|py35h2b20989_8|py35hdbf6ddf_8|py27hdbf6ddf_8|py27h2b20989_8|py37h2b20989_8|py36hdbf6ddf_8|py36hdbf6ddf_7|py27hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py35h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_6|py27h2b20989_6']\n\
  goatools=0.9.7 -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py37hde5b4d6_0|py36hde5b4d6_0|py37h2b20989_6|py27h2b20989_7|py36hdbf6ddf_7|py36h2b20989_7|py35hdbf6ddf_7|py37hdbf6ddf_7|py27h2b20989_7|py36h2b20989_7|py27hdbf6ddf_7|py37hdbf6ddf_7|py36hdbf6ddf_8|py36h2b20989_8|py37hdbf6ddf_8|py35h7cdd4dd_9|py37h7cdd4dd_9|py37h3dfced4_9|py36h3dfced4_9|py27h74e8950_9|py35h74e8950_9|py37h74e8950_9|py36h74e8950_9|py27h81de0dd_9|py27h81de0dd_10|py36h81de0dd_10|py35h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py35h2f8d375_10|py37h2f8d375_11|py27h2f8d375_11|py36h2f8d375_11|py36hde5b4d6_11|py27hde5b4d6_11|py37h2f8d375_12|py27h2f8d375_12|py38hde5b4d6_12|py38h2f8d375_12|py36h0ea5e3f_1|py27h0ea5e3f_1|py35h0ea5e3f_1|py36h9be14a7_1|py27h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py35hdbf6ddf_0|py36h2b20989_0|py27h2b20989_0|py35hdbf6ddf_0|py36h2b20989_1|py36hdbf6ddf_1|py27h2b20989_2|py36h2b20989_2|py27hdbf6ddf_2|py36hdbf6ddf_2|py27h2b20989_3|py27h2b20989_4|py37h2b20989_4|py37hdbf6ddf_4|py35hdbf6ddf_4|py35h2b20989_4|py37h2f8d375_4|py35h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py35h81de0dd_4|py37h81de0dd_4|py36h81de0dd_4|py37h2f8d375_5|py37hde5b4d6_5|py36hde5b4d6_5|py27h7cdd4dd_0|py36h7cdd4dd_0|py36h3dfced4_0|py35h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py27h74e8950_0|py27h81de0dd_0|py35h81de0dd_0|py27h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py36h2f8d375_1|py27h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_1|py36h2f8d375_1|py37hde5b4d6_1|py27hde5b4d6_1|py27h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_1|py37h2f8d375_1|py37hde5b4d6_1|py27h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py39h76555f2_1|py38h41b4c56_3|py37hdc34a94_3|py37ha8aedfd_4|py39ha8aedfd_4|py37hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py37h4c65ebe_1|py39h4c65ebe_1|py38h4c65ebe_1|py37h21a3de8_1|py37h34387ca_0|py39h34387ca_0|py38he2ba247_0|py38hfae3a4d_0|py39he2ba247_0|py38h39b7dee_0|py37h39b7dee_0|py39h39b7dee_0|py39h79a1101_0|py37h79a1101_0|py39hf524024_1|py37hf524024_1|py310h9585f30_2|py310h375b286_3|py38ha15fc14_3|py310hcba007f_3|py39hf524024_0|py38hf524024_0|py39hb8be1f0_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_1|py38h31eccc5_1|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py39h31eccc5_0|py310h8e6c178_0|py38h31eccc5_0|py39h31eccc5_1|py310h8e6c178_1|py310h375b286_1|py38h1e6e340_1|py310hcba007f_0|py38hb8be1f0_0|py310hf2716ce_0|py310h9585f30_0|py39ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py37h1e6e340_3|py38hf524024_2|py37hf524024_2|py39hf524024_2|py38hb8be1f0_2|py37hb8be1f0_2|py39hb8be1f0_2|py310hf2716ce_2|py310h9585f30_1|py310hf2716ce_1|py38hf524024_1|py37hb8be1f0_1|py38hb8be1f0_1|py39hb8be1f0_1|py310h79a1101_0|py310h2b8c604_0|py38h2b8c604_0|py39h2b8c604_0|py37h2b8c604_0|py38h79a1101_0|py38h74d4b33_0|py39h74d4b33_0|py37h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py37hfae3a4d_0|py39h7d8b39e_0|py38h7d8b39e_0|py38h34387ca_0|py37h7d8b39e_0|py38h21a3de8_1|py39h21a3de8_1|py39h0f7b65f_0|py39h2ae0177_0|py38hfa32c7d_0|py37h75fe3a5_0|py38h75fe3a5_0|py37hfa32c7d_0|py38hfa32c7d_0|py38h75fe3a5_0|py38h73d599e_4|py39h73d599e_4|py37h73d599e_4|py38ha8aedfd_4|py39hdc34a94_3|py38hdc34a94_3|py36hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py39h41b4c56_3|py39hfb011de_1|py38hde5b4d6_0|py38h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py36h2f8d375_1|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py37h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py36h81de0dd_1|py27h2f8d375_1|py36h81de0dd_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py35h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py36h74e8950_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py27hde5b4d6_5|py27h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py38h2f8d375_4|py27h81de0dd_4|py36hdbf6ddf_4|py36h2b20989_4|py27hdbf6ddf_4|py27hdbf6ddf_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py37h2b20989_3|py36h2b20989_3|py37hdbf6ddf_2|py37h2b20989_2|py27h2b20989_1|py37hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py27hdbf6ddf_0|py36hdbf6ddf_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35h2b20989_0|py36h2b20989_0|py36hde5b4d6_12|py37hde5b4d6_12|py27hde5b4d6_12|py36h2f8d375_12|py37hde5b4d6_11|py27h2f8d375_10|py37h81de0dd_10|py36h74e8950_10|py27h74e8950_10|py35h74e8950_10|py37h74e8950_10|py35h81de0dd_9|py37h81de0dd_9|py36h81de0dd_9|py27h3dfced4_9|py35h3dfced4_9|py27h7cdd4dd_9|py36h7cdd4dd_9|py35h2b20989_8|py35hdbf6ddf_8|py27hdbf6ddf_8|py27h2b20989_8|py37h2b20989_8|py36hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py35h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_6|py27hdbf6ddf_6|py37hdbf6ddf_6|py36h2b20989_6|py27h2b20989_6|py37h2f8d375_0|py36h2f8d375_0']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py27h0ea5e3f_1|py36h9be14a7_1|py35h9be14a7_1|py36h2b20989_0|py35hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_1|py36hdbf6ddf_1|py27h2b20989_1|py27h2b20989_2|py36h2b20989_2|py37hdbf6ddf_2|py36hdbf6ddf_2|py27h2b20989_4|py27hdbf6ddf_4|py37h2b20989_4|py37hdbf6ddf_4|py36h2b20989_4|py35h2b20989_4|py37hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_4|py35h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py35h81de0dd_4|py37h81de0dd_4|py36h81de0dd_4|py37h2f8d375_5|py37hde5b4d6_5|py36hde5b4d6_5|py27h7cdd4dd_0|py36h7cdd4dd_0|py36h3dfced4_0|py35h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py27h74e8950_0|py27h81de0dd_0|py35h81de0dd_0|py27h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py36h2f8d375_1|py27h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_1|py36h2f8d375_1|py37hde5b4d6_1|py27hde5b4d6_1|py27h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_1|py37h2f8d375_1|py37hde5b4d6_1|py27h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py39h76555f2_1|py38h41b4c56_3|py37hdc34a94_3|py37ha8aedfd_4|py39ha8aedfd_4|py37hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py37h4c65ebe_1|py39h4c65ebe_1|py38h4c65ebe_1|py37h21a3de8_1|py37h34387ca_0|py39h34387ca_0|py38he2ba247_0|py38hfae3a4d_0|py39he2ba247_0|py38h39b7dee_0|py37h39b7dee_0|py39h39b7dee_0|py39h79a1101_0|py37h79a1101_0|py39hf524024_1|py37hf524024_1|py310h9585f30_2|py310h375b286_3|py38ha15fc14_3|py310hcba007f_3|py39hf524024_0|py38hf524024_0|py39hb8be1f0_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_1|py38h31eccc5_1|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py39h31eccc5_0|py310h8e6c178_0|py38h31eccc5_0|py39h31eccc5_1|py310h8e6c178_1|py310h375b286_1|py38h1e6e340_1|py310hcba007f_0|py38hb8be1f0_0|py310hf2716ce_0|py310h9585f30_0|py39ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py37h1e6e340_3|py38hf524024_2|py37hf524024_2|py39hf524024_2|py38hb8be1f0_2|py37hb8be1f0_2|py39hb8be1f0_2|py310hf2716ce_2|py310h9585f30_1|py310hf2716ce_1|py38hf524024_1|py37hb8be1f0_1|py38hb8be1f0_1|py39hb8be1f0_1|py310h79a1101_0|py310h2b8c604_0|py38h2b8c604_0|py39h2b8c604_0|py37h2b8c604_0|py38h79a1101_0|py38h74d4b33_0|py39h74d4b33_0|py37h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py37hfae3a4d_0|py39h7d8b39e_0|py38h7d8b39e_0|py38h34387ca_0|py37h7d8b39e_0|py38h21a3de8_1|py39h21a3de8_1|py39h0f7b65f_0|py39h2ae0177_0|py38hfa32c7d_0|py37h75fe3a5_0|py38h75fe3a5_0|py37hfa32c7d_0|py38hfa32c7d_0|py38h75fe3a5_0|py38h73d599e_4|py39h73d599e_4|py37h73d599e_4|py38ha8aedfd_4|py39hdc34a94_3|py38hdc34a94_3|py36hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py39h41b4c56_3|py39hfb011de_1|py38hde5b4d6_0|py38h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py36h2f8d375_1|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py37h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py36h81de0dd_1|py27h2f8d375_1|py36h81de0dd_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py35h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py36h74e8950_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py27hde5b4d6_5|py27h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py38h2f8d375_4|py27h81de0dd_4|py37h2f8d375_0|py36h2f8d375_0|py35hdbf6ddf_4|py36hdbf6ddf_4|py27hdbf6ddf_3|py27h2b20989_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py37h2b20989_3|py36h2b20989_3|py27hdbf6ddf_2|py37h2b20989_2|py37hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py27hdbf6ddf_0|py36hdbf6ddf_0|py27h2b20989_0|py36h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35h2b20989_0|py27h2b20989_0|py27h9be14a7_1|py35h0ea5e3f_1|py36h0ea5e3f_1']\n\
  \nPackage _openmp_mutex conflicts for:\npandas=0.25 -> libgcc-ng[version='>=7.3.0']\
  \ -> _openmp_mutex[version='>=4.5']\npython=3.7 -> libgcc-ng[version='>=9.4.0']\
  \ -> _openmp_mutex[version='>=4.5']\nmatplotlib=3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> _openmp_mutex[version='>=4.5']\n\nPackage pandas conflicts for:\ngoatools=0.9.7\
  \ -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\npandas=0.25\n\
  \nPackage pytz conflicts for:\nmatplotlib=3.1 -> pytz\npandas=0.25 -> pytz[version='>=2017.2']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - matplotlib=3.1 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - python=3.7 -> libgcc-ng[version='>=9.4.0'] -> __glibc[version='>=2.17']\n\n\
  Your installed version is: 2.31\n\nNote that strict channel priority may have removed\
  \ packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fyhvk05em%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fyhvk05em%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.497GiB</td>
						<td>1694.1 seconds</td>
						<td>2686.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743187344">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/1.4">1.4</a>
						</td>
						<td>nextflow</td>
						<td>1223 days</td>
						<td>2023-02-19 06:33:14</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fmfv3i7sx%2Flogs.tar.xz">0.026GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fmfv3i7sx%2Foutput.tar.xz">0.007GiB</a>
						</td>
						<td>2.132GiB</td>
						<td>799.4 seconds</td>
						<td>422.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743114304">
								<a href="https://nf-co.re/hic">hic</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hic/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>1222 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2F449m3ffb%2Flogs.tar.xz">0.212GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2F449m3ffb%2Foutput.tar.xz">0.042GiB</a>
						</td>
						<td>1.543GiB</td>
						<td>709.4 seconds</td>
						<td>672.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743187104">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/1.4.1">1.4.1</a>
						</td>
						<td>nextflow</td>
						<td>1221 days</td>
						<td>2023-02-19 06:20:54</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fcbw6d270%2Flogs.tar.xz">0.026GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fcbw6d270%2Foutput.tar.xz">0.007GiB</a>
						</td>
						<td>2.132GiB</td>
						<td>789.6 seconds</td>
						<td>609.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743186864">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/1.4.2">1.4.2</a>
						</td>
						<td>nextflow</td>
						<td>1220 days</td>
						<td>2023-02-19 03:32:40</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Foil0u4om%2Flogs.tar.xz">0.026GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Foil0u4om%2Foutput.tar.xz">0.007GiB</a>
						</td>
						<td>2.134GiB</td>
						<td>836.7 seconds</td>
						<td>828.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743191664">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/2.5.1">2.5.1</a>
						</td>
						<td>nextflow</td>
						<td>1216 days</td>
						<td>2023-02-19 04:07:03</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fkucgdeob%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fkucgdeob%2Foutput.tar.xz">0.015GiB</a>
						</td>
						<td>1.885GiB</td>
						<td>1122.9 seconds</td>
						<td>565.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744507232">
								<a href="https://nf-co.re/chipseq">chipseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/chipseq/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>1202 days</td>
						<td>2023-02-19 09:27:11</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2Fz7n1w9mn%2Flogs.tar.xz">0.500GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2Fz7n1w9mn%2Foutput.tar.xz">0.077GiB</a>
						</td>
						<td>2.476GiB</td>
						<td>3297.5 seconds</td>
						<td>1418.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745538800">
								<a href="https://nf-co.re/atacseq">atacseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/atacseq/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>1202 days</td>
						<td>2023-02-19 03:42:52</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: nextflow.dag.MultipleInputChannelException
msg: Channel `design_multiple_samples` has been used as an input by more than a process
  or an operator
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:295)\n\
  \tat nextflow.dag.DAG.inbound(DAG.groovy:207)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat nextflow.dag.DAG.addVertex(DAG.groovy:145)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat nextflow.dag.DAG.addOperatorNode(DAG.groovy:116)\n\tat nextflow.dag.NodeMarker.addOperatorNode(NodeMarker.groovy:64)\n\
  \tat nextflow.extension.OpCall.addGraphNode(OpCall.groovy:178)\n\tat nextflow.extension.OpCall.invoke(OpCall.groovy:168)\n\
  \tat nextflow.extension.OpCall.call(OpCall.groovy:113)\n\tat nextflow.plugin.extension.PluginExtensionProvider.invokeExtensionMethod(PluginExtensionProvider.groovy:279)\n\
  \tat groovy.runtime.metaclass.NextflowDelegatingMetaClass.invokeMethod(NextflowDelegatingMetaClass.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.PojoMetaClassSite.call(PojoMetaClassSite.java:44)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_2f36603e.runScript(Script_2f36603e:354)\n\tat nextflow.script.BaseScript.runDsl1(BaseScript.groovy:164)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\
  \tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\
  \tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2F09ektrq5%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2F09ektrq5%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.736GiB</td>
						<td>566.0 seconds</td>
						<td>490.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010763149536">
								<a href="https://nf-co.re/airrflow">airrflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/airrflow/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>1199 days</td>
						<td>2023-02-19 06:33:53</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Foxqu2fup%2Flogs.tar.xz">0.050GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Foxqu2fup%2Foutput.tar.xz">0.019GiB</a>
						</td>
						<td>1.750GiB</td>
						<td>1545.0 seconds</td>
						<td>854.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743139392">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/1.4">1.4</a>
						</td>
						<td>nextflow</td>
						<td>1187 days</td>
						<td>2023-02-19 03:52:08</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2F27ajzof7%2Flogs.tar.xz">0.039GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2F27ajzof7%2Foutput.tar.xz">0.008GiB</a>
						</td>
						<td>1.634GiB</td>
						<td>716.2 seconds</td>
						<td>426.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743109984">
								<a href="https://nf-co.re/epitopeprediction">epitopeprediction</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/epitopeprediction/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1171 days</td>
						<td>2023-02-19 04:17:28</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epitopeprediction%2Fzfzf0mkh%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epitopeprediction%2Fzfzf0mkh%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.725GiB</td>
						<td>740.3 seconds</td>
						<td>466.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746997168">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/1.1.1">1.1.1</a>
						</td>
						<td>nextflow</td>
						<td>1168 days</td>
						<td>2023-02-19 04:24:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: metadataCategoryPairwise.r Metadata.tsv
error: "Warning message:\nIn read.table(file = file, header = header, sep = sep, quote\
  \ = quote,  :\n  incomplete final line found by readTableHeader on 'Metadata.tsv'\n\
  Error in 1:ncol(data) : argument of length 0\nExecution halted"
exit_status: 1
kind: CalledProcessError
output: (empty)
process: metadata_category_pairwise (1)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- 7e
- dba0da51adcc2e33ba1027ad074fc3

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fk0dglxqg%2Flogs.tar.xz">0.004GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fk0dglxqg%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>1.888GiB</td>
						<td>664.9 seconds</td>
						<td>369.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743192864">
								<a href="https://nf-co.re/scrnaseq">scrnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/scrnaseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1167 days</td>
						<td>2023-02-19 03:47:47</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/scrnaseq%2F7u17m76w%2Flogs.tar.xz">0.033GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/scrnaseq%2F7u17m76w%2Foutput.tar.xz">0.005GiB</a>
						</td>
						<td>1.650GiB</td>
						<td>590.9 seconds</td>
						<td>384.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743147552">
								<a href="https://nf-co.re/neutronstar">neutronstar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/neutronstar/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1165 days</td>
						<td>2023-02-19 04:39:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.cli.Launcher
date: Feb-19 04:39:23.153
kind: NextflowMiscError
process: main
rest: '@unknown'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/neutronstar%2Ff49ht084%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/neutronstar%2Ff49ht084%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.580GiB</td>
						<td>17.2 seconds</td>
						<td>6.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745859520">
								<a href="https://nf-co.re/bacass">bacass</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/bacass/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>1164 days</td>
						<td>2023-02-19 06:50:41</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: markdown_to_html.r output.md results_description.html
error: "/usr/bin/env: \u2018Rscript\u2019: No such file or directory"
exit_status: 127
kind: CalledProcessError
output: (empty)
process: output_documentation (1)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- 6b
- 586bf56ab3c1cc62f7ab56292f9635

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/bacass%2Fxu0d2udb%2Flogs.tar.xz">0.212GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/bacass%2Fxu0d2udb%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.747GiB</td>
						<td>32.1 seconds</td>
						<td>14.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743191424">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/2.5.2">2.5.2</a>
						</td>
						<td>nextflow</td>
						<td>1161 days</td>
						<td>2023-02-19 03:43:41</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fpsmgsiia%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fpsmgsiia%2Foutput.tar.xz">0.015GiB</a>
						</td>
						<td>1.883GiB</td>
						<td>1163.0 seconds</td>
						<td>599.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746961808">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/1.1.2">1.1.2</a>
						</td>
						<td>nextflow</td>
						<td>1157 days</td>
						<td>2023-02-19 04:31:51</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: metadataCategoryPairwise.r Metadata.tsv
error: "Warning message:\nIn read.table(file = file, header = header, sep = sep, quote\
  \ = quote,  :\n  incomplete final line found by readTableHeader on 'Metadata.tsv'\n\
  Error in 1:ncol(data) : argument of length 0\nExecution halted"
exit_status: 1
kind: CalledProcessError
output: (empty)
process: metadata_category_pairwise (1)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- 1e
- 069f2faaec15d64a13ffb85676dc44

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fl8iq1eeg%2Flogs.tar.xz">0.004GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fl8iq1eeg%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>1.916GiB</td>
						<td>650.5 seconds</td>
						<td>311.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743136992">
								<a href="https://nf-co.re/mag">mag</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mag/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1156 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fo0k2aafk%2Flogs.tar.xz">0.211GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fo0k2aafk%2Foutput.tar.xz">0.006GiB</a>
						</td>
						<td>1.813GiB</td>
						<td>1621.0 seconds</td>
						<td>960.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010762702656">
								<a href="https://nf-co.re/airrflow">airrflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/airrflow/tree/1.2.0">1.2.0</a>
						</td>
						<td>nextflow</td>
						<td>1132 days</td>
						<td>2023-02-19 06:05:32</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Frz337b9p%2Flogs.tar.xz">0.050GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Frz337b9p%2Foutput.tar.xz">0.019GiB</a>
						</td>
						<td>1.749GiB</td>
						<td>1523.5 seconds</td>
						<td>1048.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738269200">
								<a href="https://github.com/GooglingTheCancerGenome/sv-callers">GooglingTheCancerGenome/sv-callers</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.1.1">v1.1.1</a>
						</td>
						<td>snakemake</td>
						<td>1108 days</td>
						<td>2023-02-19 07:00:45</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2Fd9gfzjjy%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2Fd9gfzjjy%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743149712">
								<a href="https://nf-co.re/rnafusion">rnafusion</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnafusion/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>1105 days</td>
						<td>2023-02-19 03:31:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.nio.file.NoSuchFileException
msg: 'Config file does not exist: /home/azureuser/tmp/iwrc784pnz/code/conf/test.config'
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.config.ConfigBase.memoizedMethodPriv$readConfigFilePath(ConfigBase.groovy:136)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.config.ConfigBase$__clinit__closure3.doCall(ConfigBase.groovy)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:428)\n\tat groovy.lang.Closure$call.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat groovy.lang.Closure$call.call(Unknown Source)\n\tat nextflow.config.ConfigBase.readConfigFile(ConfigBase.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite$StaticMetaMethodSiteNoUnwrapNoCoerce.invoke(StaticMetaMethodSite.java:149)\n\
  \tat org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite.callStatic(StaticMetaMethodSite.java:100)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:55)\n\
  \tat org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite.callStatic(StaticMetaMethodSite.java:102)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:231)\n\
  \tat nextflow.config.ConfigBase.includeConfig(ConfigBase.groovy:91)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat _nf_config_c6a41225$_run_closure2$_closure12.doCall(_nf_config_c6a41225:123)\n\
  \tat _nf_config_c6a41225$_run_closure2$_closure12.doCall(_nf_config_c6a41225)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:38)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:53)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.config.ConfigParser$_parse_closure5.doCall(ConfigParser.groovy:394)\n\
  \tat jdk.internal.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)\n\tat\
  \ java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaMethod.invoke(ClosureMetaMethod.java:83)\n\
  \tat groovy.lang.ExpandoMetaClass.invokeMethod(ExpandoMetaClass.java:1118)\n\tat\
  \ groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)\n\
  \tat groovy.lang.DelegatingMetaClass.invokeMethod(DelegatingMetaClass.java:140)\n\
  \tat groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)\n\tat groovy.lang.Script.invokeMethod(Script.java:96)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:408)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:348)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat _nf_config_c6a41225$_run_closure2.doCall(_nf_config_c6a41225:123)\n\tat _nf_config_c6a41225$_run_closure2.doCall(_nf_config_c6a41225)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:38)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:53)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.config.ConfigParser$_parse_closure5.doCall(ConfigParser.groovy:370)\n\
  \tat jdk.internal.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)\n\tat\
  \ java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaMethod.invoke(ClosureMetaMethod.java:83)\n\
  \tat groovy.lang.ExpandoMetaClass.invokeMethod(ExpandoMetaClass.java:1118)\n\tat\
  \ groovy.lang.DelegatingMetaClass.invokeMethod(DelegatingMetaClass.java:226)\n\t\
  at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat _nf_config_c6a41225.run(_nf_config_c6a41225:108)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat org.codehaus.groovy.reflection.CachedMethod$invoke.call(Unknown Source)\n\t\
  at nextflow.config.ConfigParser$_parse_closure5.doCall(ConfigParser.groovy:449)\n\
  \tat jdk.internal.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)\n\tat\
  \ java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaMethod.invoke(ClosureMetaMethod.java:83)\n\
  \tat groovy.lang.ExpandoMetaClass.invokeMethod(ExpandoMetaClass.java:1118)\n\tat\
  \ groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.DelegatingMetaClass.invokeMethod(DelegatingMetaClass.java:147)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:38)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:53)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.config.ConfigParser.parse(ConfigParser.groovy:477)\n\tat nextflow.config.ConfigParser$parse$0.callCurrent(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat nextflow.config.ConfigParser.parse(ConfigParser.groovy:304)\n\tat nextflow.config.ConfigParser$parse$6.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat nextflow.config.ConfigBuilder.parse0(ConfigBuilder.groovy:436)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat nextflow.config.ConfigBuilder.merge0(ConfigBuilder.groovy:422)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:203)\n\
  \tat nextflow.config.ConfigBuilder.buildConfig0(ConfigBuilder.groovy:372)\n\tat\
  \ java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat nextflow.config.ConfigBuilder.buildGivenFiles(ConfigBuilder.groovy:317)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.config.ConfigBuilder.buildConfigObject(ConfigBuilder.groovy:780)\n\
  \tat nextflow.config.ConfigBuilder.build(ConfigBuilder.groovy:793)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:307)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Fzaavre4b%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Fzaavre4b%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.355GiB</td>
						<td>13.5 seconds</td>
						<td>5.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740285296">
								<a href="https://github.com/solida-core/dima">solida-core/dima</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/dima/tree/v1.0">v1.0</a>
						</td>
						<td>snakemake</td>
						<td>1088 days</td>
						<td>2023-02-19 04:30:55</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- Snakefile
kind: KeyError
line_no: 10
rest: "'samples'\n  File \"/home/azureuser/tmp/yh4t2wej0l/code/Snakefile\", line 10,\
  \ in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2Famqo7m0q%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2Famqo7m0q%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.8 seconds</td>
						<td>1.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743146592">
								<a href="https://nf-co.re/nanoseq">nanoseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/nanoseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1081 days</td>
						<td>2023-02-19 04:21:32</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: check_samplesheet.py samplesheet_bc_dx.csv samplesheet_reformat.csv
error: 'ERROR: Please check samplesheet header -> group,replicate,barcode,input_file,genome,transcriptome
  != sample,fastq,barcode,genome,transcriptome'
exit_status: 1
kind: CalledProcessError
output: 'ERROR: Please check samplesheet header -> group,replicate,barcode,input_file,genome,transcriptome
  != sample,fastq,barcode,genome,transcriptome'
process: CheckSampleSheet (samplesheet_bc_dx.csv)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '88'
- 59dab2bb25b854d35b4e0c561cd63b

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nanoseq%2F24cn5dsd%2Flogs.tar.xz">0.216GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nanoseq%2F24cn5dsd%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.610GiB</td>
						<td>459.6 seconds</td>
						<td>241.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743107104">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>1080 days</td>
						<td>2023-02-19 03:31:04</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 03:31:27.948
kind: NextflowMiscError
process: main
rest: Please specify either --single_end, --paired_end to execute the pipeline on
  FastQ files and --bam for previously processed BAM files!

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fbcmgleaz%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fbcmgleaz%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.619GiB</td>
						<td>30.7 seconds</td>
						<td>24.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743142752">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.4">1.4</a>
						</td>
						<td>nextflow</td>
						<td>1062 days</td>
						<td>2023-02-19 03:56:18</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fb69fq4a1%2Flogs.tar.xz">0.107GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fb69fq4a1%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.810GiB</td>
						<td>2708.7 seconds</td>
						<td>1624.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738218128">
								<a href="https://github.com/GooglingTheCancerGenome/sv-gen">GooglingTheCancerGenome/sv-gen</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/GooglingTheCancerGenome/sv-gen/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>1062 days</td>
						<td>2023-02-19 06:17:42</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-gen%2Furyigick%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-gen%2Furyigick%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743139152">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/1.5">1.5</a>
						</td>
						<td>nextflow</td>
						<td>1046 days</td>
						<td>2023-02-19 06:39:11</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fxlfmcznk%2Flogs.tar.xz">0.038GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fxlfmcznk%2Foutput.tar.xz">0.008GiB</a>
						</td>
						<td>1.455GiB</td>
						<td>597.4 seconds</td>
						<td>327.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743142512">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.5">1.5</a>
						</td>
						<td>nextflow</td>
						<td>1034 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2F7m621awo%2Flogs.tar.xz">0.110GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2F7m621awo%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.891GiB</td>
						<td>2898.1 seconds</td>
						<td>1844.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743142272">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.5.1">1.5.1</a>
						</td>
						<td>nextflow</td>
						<td>1031 days</td>
						<td>2023-02-19 03:31:02</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fu7w4alq3%2Flogs.tar.xz">0.110GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fu7w4alq3%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.893GiB</td>
						<td>2905.2 seconds</td>
						<td>1493.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742866096">
								<a href="https://nf-co.re/coproid">coproid</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/coproid/tree/1.1">1.1</a>
						</td>
						<td>nextflow</td>
						<td>1026 days</td>
						<td>2023-02-19 06:00:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "sourcepredict -di 2 \\\n              -kne 0 \\\n             \
  \ -me tsne \\\n              -n subsample \\\n              -l test_labels.csv \\\
  \n              -s test_sources.csv \\\n              -t 2 \\\n              -o\
  \ prediction.sourcepredict.csv \\\n              -e sourcepredict_embedding.csv\
  \ kraken_merged.csv"
error: "2458000 generating entries... \n 2459000 generating entries... \n 2460000\
  \ generating entries... \n 2461000 generating entries... \n 2462000 generating entries...\
  \ \n 2463000 generating entries... \n 2464000 generating entries... \n 2465000 generating\
  \ entries... \n 2466000 generating entries... \n 2467000 generating entries... \n\
  \ 2468000 generating entries... \n 2469000 generating entries... \n 2470000 generating\
  \ entries... \n 2471000 generating entries... \n 2472000 generating entries... \n\
  \ 2473000 generating entries... \n 2474000 generating entries... \n 2475000 generating\
  \ entries... \n 2476000 generating entries... \n 2477000 generating entries... \n\
  \ 2478000 generating entries... \n 2479000 generating entries... \n 2480000 generating\
  \ entries... \n 2481000 generating entries... \n 2482000 generating entries... \n\
  \ 2483000 generating entries... \n 2484000 generating entries... \n 2485000 generating\
  \ entries... \n 2486000 generating entries... \n 2487000 generating entries... \n\
  \ 2488000 generating entries... \n 2489000 generating entries... \nUploading to\
  \ /home/azureuser/.etetoolkit/taxa.sqlite\n\nTraceback (most recent call last):\n\
  \  File \"/opt/conda/envs/nf-core-coproid-1.1/bin/sourcepredict\", line 10, in <module>\n\
  \    sys.exit(main())\n  File \"/opt/conda/envs/nf-core-coproid-1.1/lib/python3.7/site-packages/sourcepredict/__main__.py\"\
  , line 172, in main\n    sm.compute_distance(distance_method=distance_method, rank=RANK)\n\
  \  File \"/opt/conda/envs/nf-core-coproid-1.1/lib/python3.7/site-packages/sourcepredict/sourcepredictlib/ml.py\"\
  , line 251, in compute_distance\n    ncbi = NCBITaxa()\n  File \"/opt/conda/envs/nf-core-coproid-1.1/lib/python3.7/site-packages/ete3/ncbi_taxonomy/ncbiquery.py\"\
  , line 110, in __init__\n    self.update_taxonomy_database(taxdump_file)\n  File\
  \ \"/opt/conda/envs/nf-core-coproid-1.1/lib/python3.7/site-packages/ete3/ncbi_taxonomy/ncbiquery.py\"\
  , line 129, in update_taxonomy_database\n    update_db(self.dbfile)\n  File \"/opt/conda/envs/nf-core-coproid-1.1/lib/python3.7/site-packages/ete3/ncbi_taxonomy/ncbiquery.py\"\
  , line 760, in update_db\n    upload_data(dbfile)\n  File \"/opt/conda/envs/nf-core-coproid-1.1/lib/python3.7/site-packages/ete3/ncbi_taxonomy/ncbiquery.py\"\
  , line 802, in upload_data\n    db.execute(\"INSERT INTO synonym (taxid, spname)\
  \ VALUES (?, ?);\", (taxid, spname))\nsqlite3.IntegrityError: UNIQUE constraint\
  \ failed: synonym.spname, synonym.taxid"
exit_status: 1
kind: CalledProcessError
output: "2442000 generating entries... \n 2443000 generating entries... \n 2444000\
  \ generating entries... \n 2445000 generating entries... \n 2446000 generating entries...\
  \ \n 2447000 generating entries... \n 2448000 generating entries... \n 2449000 generating\
  \ entries... \n 2450000 generating entries... \n 2451000 generating entries... \n\
  \ 2452000 generating entries... \n 2453000 generating entries... \n 2454000 generating\
  \ entries... \n 2455000 generating entries... \n 2456000 generating entries... \n\
  \ 2457000 generating entries... \n 2458000 generating entries... \n 2459000 generating\
  \ entries... \n 2460000 generating entries... \n 2461000 generating entries... \n\
  \ 2462000 generating entries... \n 2463000 generating entries... \n 2464000 generating\
  \ entries... \n 2465000 generating entries... \n 2466000 generating entries... \n\
  \ 2467000 generating entries... \n 2468000 generating entries... \n 2469000 generating\
  \ entries... \n 2470000 generating entries... \n 2471000 generating entries... \n\
  \ 2472000 generating entries... \n 2473000 generating entries... \n 2474000 generating\
  \ entries... \n 2475000 generating entries... \n 2476000 generating entries... \n\
  \ 2477000 generating entries... \n 2478000 generating entries... \n 2479000 generating\
  \ entries... \n 2480000 generating entries... \n 2481000 generating entries... \n\
  \ 2482000 generating entries... \n 2483000 generating entries... \n 2484000 generating\
  \ entries... \n 2485000 generating entries... \n 2486000 generating entries... \n\
  \ 2487000 generating entries... \n 2488000 generating entries... \n 2489000 generating\
  \ entries... \nUploading to /home/azureuser/.etetoolkit/taxa.sqlite"
process: sourcepredict
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '43'
- ab4ca0c49b467c00e925591335cfbb

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/coproid%2Fmb9ovfql%2Flogs.tar.xz">0.348GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/coproid%2Fmb9ovfql%2Foutput.tar.xz">0.029GiB</a>
						</td>
						<td>3.201GiB</td>
						<td>893.7 seconds</td>
						<td>745.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743193104">
								<a href="https://nf-co.re/slamseq">slamseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/slamseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>1011 days</td>
						<td>2023-02-19 14:54:47</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/slamseq%2Fh0us6opc%2Flogs.tar.xz">2.588GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/slamseq%2Fh0us6opc%2Foutput.tar.xz">0.240GiB</a>
						</td>
						<td>1.544GiB</td>
						<td>3205.7 seconds</td>
						<td>2304.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743191184">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/2.6">2.6</a>
						</td>
						<td>nextflow</td>
						<td>1001 days</td>
						<td>2023-02-19 04:32:27</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fxqdrpunv%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fxqdrpunv%2Foutput.tar.xz">0.016GiB</a>
						</td>
						<td>1.945GiB</td>
						<td>1262.0 seconds</td>
						<td>912.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743117184">
								<a href="https://nf-co.re/imcyto">imcyto</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/imcyto/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>997 days</td>
						<td>2023-02-20 05:13:38</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name quay.io-biocontainers-r-rmarkdown-0.9.5--r3.3.2_0.img.pulling.1676870034839\
  \ docker://quay.io/biocontainers/r-rmarkdown:0.9.5--r3.3.2_0 > /dev/null\n  status\
  \ : 255\n  message:\n    INFO:    Converting OCI blobs to SIF format\n    INFO:\
  \    Starting build...\n    Getting image source signatures\n    Copying blob sha256:d2ba336f2e4458a9223203bf17cc88d77e3006d9cbf4f0b24a1618d0a5b82053\n\
  \    Copying blob sha256:77c6c00e8b61bb628567c060b85690b0b0561bb37d8ad3f3792877bddcfe2500\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:00cf8b9f3d2a08745635830064530c931d16f549d031013a9b7c6535e7107b88\n\
  \    Copying blob sha256:3aaade50789a6510c60e536f5e75fe8b8fc84801620e575cb0435e2654ffd7f6\n\
  \    Copying blob sha256:7ff999a2256f84141f17d07d26539acea8a4d9c149fefbbcc9a8b4d15ea32de7\n\
  \    Copying blob sha256:dfda3e01f2b637b7b89adb401f2f763d592fcedd2937240e2eb3286fabce55f0\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:10c3bb32200bdb5006b484c59b5f0c71b4dbab611d33fca816cd44f9f5ce9e3c\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:c87a160bc3092ac61f772e33c5992b04c864435f55fb0fc8383da483235c5f51\n\
  \    Copying config sha256:af42e8bd9862484373a63ae67a97edb983ed11a9f135bfb72674ab9a3d786697\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.ccf8745d8ce852c56009b7a20326db5ed8aba2b6a65023a5dc70f475da3ca9f3\"\n\n\t\
  at nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\tat nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\
  \tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\tat nextflow.processor.TaskRun$isContainerEnabled$2.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name quay.io-biocontainers-r-rmarkdown-0.9.5--r3.3.2_0.img.pulling.1676870034839\
  \ docker://quay.io/biocontainers/r-rmarkdown:0.9.5--r3.3.2_0 > /dev/null\n  status\
  \ : 255\n  message:\n    INFO:    Converting OCI blobs to SIF format\n    INFO:\
  \    Starting build...\n    Getting image source signatures\n    Copying blob sha256:d2ba336f2e4458a9223203bf17cc88d77e3006d9cbf4f0b24a1618d0a5b82053\n\
  \    Copying blob sha256:77c6c00e8b61bb628567c060b85690b0b0561bb37d8ad3f3792877bddcfe2500\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:00cf8b9f3d2a08745635830064530c931d16f549d031013a9b7c6535e7107b88\n\
  \    Copying blob sha256:3aaade50789a6510c60e536f5e75fe8b8fc84801620e575cb0435e2654ffd7f6\n\
  \    Copying blob sha256:7ff999a2256f84141f17d07d26539acea8a4d9c149fefbbcc9a8b4d15ea32de7\n\
  \    Copying blob sha256:dfda3e01f2b637b7b89adb401f2f763d592fcedd2937240e2eb3286fabce55f0\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:10c3bb32200bdb5006b484c59b5f0c71b4dbab611d33fca816cd44f9f5ce9e3c\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:c87a160bc3092ac61f772e33c5992b04c864435f55fb0fc8383da483235c5f51\n\
  \    Copying config sha256:af42e8bd9862484373a63ae67a97edb983ed11a9f135bfb72674ab9a3d786697\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.ccf8745d8ce852c56009b7a20326db5ed8aba2b6a65023a5dc70f475da3ca9f3\"\n\n\t\
  at nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 05:14:09.485 [Actor Thread 1] DEBUG nextflow.Session\
  \ - Session aborted -- Cause: java.lang.IllegalStateException: Failed to pull singularity\
  \ image\n  command: singularity pull  --name quay.io-biocontainers-r-rmarkdown-0.9.5--r3.3.2_0.img.pulling.1676870034839\
  \ docker://quay.io/biocontainers/r-rmarkdown:0.9.5--r3.3.2_0 > /dev/null\n  status\
  \ : 255\n  message:\n    INFO:    Converting OCI blobs to SIF format\n    INFO:\
  \    Starting build...\n    Getting image source signatures\n    Copying blob sha256:d2ba336f2e4458a9223203bf17cc88d77e3006d9cbf4f0b24a1618d0a5b82053\n\
  \    Copying blob sha256:77c6c00e8b61bb628567c060b85690b0b0561bb37d8ad3f3792877bddcfe2500\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:00cf8b9f3d2a08745635830064530c931d16f549d031013a9b7c6535e7107b88\n\
  \    Copying blob sha256:3aaade50789a6510c60e536f5e75fe8b8fc84801620e575cb0435e2654ffd7f6\n\
  \    Copying blob sha256:7ff999a2256f84141f17d07d26539acea8a4d9c149fefbbcc9a8b4d15ea32de7\n\
  \    Copying blob sha256:dfda3e01f2b637b7b89adb401f2f763d592fcedd2937240e2eb3286fabce55f0\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:10c3bb32200bdb5006b484c59b5f0c71b4dbab611d33fca816cd44f9f5ce9e3c\n\
  \    Copying blob sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4\n\
  \    Copying blob sha256:c87a160bc3092ac61f772e33c5992b04c864435f55fb0fc8383da483235c5f51\n\
  \    Copying config sha256:af42e8bd9862484373a63ae67a97edb983ed11a9f135bfb72674ab9a3d786697\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.ccf8745d8ce852c56009b7a20326db5ed8aba2b6a65023a5dc70f475da3ca9f3\"\n\nFeb-20\
  \ 05:14:09.545 [Actor Thread 1] DEBUG nextflow.Session - The following nodes are\
  \ still active:\n[process] IMCTOOLS\n  status=ACTIVE\n  port 0: (queue) closed;\
  \ channel: -\n  port 1: (value) bound ; channel: metadata\n  port 2: (cntrl) - \
  \    ; channel: $\n\n[process] PREPROCESS_FULL_STACK\n  status=ACTIVE\n  port 0:\
  \ (queue) OPEN  ; channel: -\n  port 1: (value) bound ; channel: ctiff\n  port 2:\
  \ (value) bound ; channel: cppipe\n  port 3: (value) bound ; channel: plugin_dir\n\
  \  port 4: (cntrl) -     ; channel: $\n\n[process] PREPROCESS_ILASTIK_STACK\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound ; channel: ctiff\n\
  \  port 2: (value) bound ; channel: cppipe\n  port 3: (value) bound ; channel: plugin_dir\n\
  \  port 4: (cntrl) -     ; channel: $\n\n[process] ILASTIK\n  status=ACTIVE\n  port\
  \ 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound ; channel: ilastik_training_ilp\n\
  \  port 2: (cntrl) -     ; channel: $\n\n[process] SEGMENTATION\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound ; channel: cppipe\n\
  \  port 2: (value) bound ; channel: plugin_dir\n  port 3: (cntrl) -     ; channel:\
  \ $\n\n[process] output_documentation\n  status=ACTIVE\n  port 0: (value) bound\
  \ ; channel: output_docs\n  port 1: (value) bound ; channel: images\n  port 2: (cntrl)\
  \ -     ; channel: $\n\n[process] get_software_versions\n  status=ACTIVE\n  port\
  \ 0: (value) OPEN  ; channel: imctools\n  port 1: (value) OPEN  ; channel: cellprofiler\n\
  \  port 2: (value) OPEN  ; channel: ilastik\n  port 3: (cntrl) -     ; channel:\
  \ $\n\nFeb-20 05:14:09.566 [main] DEBUG nextflow.Session - Session await > all processes\
  \ finished\nFeb-20 05:14:09.566 [main] DEBUG nextflow.Session - Session await >\
  \ all barriers passed\nFeb-20 05:14:09.836 [main] INFO  nextflow.Nextflow - -\e\
  [0;35m[nf-core/imcyto]\e[0;31m Pipeline completed with errors\e[0m-\nFeb-20 05:14:09.846\
  \ [main] DEBUG nextflow.trace.WorkflowStatsObserver - Workflow completed > WorkflowStats[succeededCount=0;\
  \ failedCount=0; ignoredCount=0; cachedCount=0; pendingCount=0; submittedCount=0;\
  \ runningCount=0; retriesCount=0; abortedCount=0; succeedDuration=0ms; failedDuration=0ms;\
  \ cachedDuration=0ms;loadCpus=0; loadMemory=0; peakRunning=0; peakCpus=0; peakMemory=0;\
  \ ]\nFeb-20 05:14:09.847 [main] DEBUG nextflow.trace.TraceFileObserver - Workflow\
  \ completed -- saving trace file\nFeb-20 05:14:09.855 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Workflow completed -- rendering execution report\nFeb-20 05:14:09.864 [main]\
  \ DEBUG nextflow.trace.ReportObserver - Execution report summary data:\n  []\nFeb-20\
  \ 05:14:11.051 [main] DEBUG nextflow.trace.TimelineObserver - Workflow completed\
  \ -- rendering execution timeline\nFeb-20 05:14:11.371 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 05:14:11.399 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/imcyto%2F8rsl4nl6%2Flogs.tar.xz">0.017GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/imcyto%2F8rsl4nl6%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.518GiB</td>
						<td>34.3 seconds</td>
						<td>34.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743196464">
								<a href="https://nf-co.re/viralrecon">viralrecon</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/viralrecon/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>993 days</td>
						<td>2023-02-19 11:10:43</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Fx2ggt6yt%2Flogs.tar.xz">0.241GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Fx2ggt6yt%2Foutput.tar.xz">0.032GiB</a>
						</td>
						<td>2.064GiB</td>
						<td>3138.8 seconds</td>
						<td>1254.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738323936">
								<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling">snakemake-workflows/dna-seq-gatk-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.0.0">v2.0.0</a>
						</td>
						<td>snakemake</td>
						<td>992 days</td>
						<td>2023-02-19 06:46:23</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeInternalError
kind: SnakemakeInternalError
rest: "File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py\"\
  , line 747, in snakemake\n    success = workflow.execute(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py\"\
  , line 926, in execute\n    dag.create_conda_envs(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/dag.py\"\
  , line 338, in create_conda_envs\n    env.create(dryrun)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 397, in create\n    if self.pin_file:\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 109, in pin_file\n    f\".{self.conda.platform}.pin.txt\"\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 102, in conda\n    return Conda(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 667, in __init__\n    shell.check_output(self._get_cmd(f\"conda info --json\"\
  ), text=True)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/shell.py\"\
  , line 63, in check_output\n    return sp.check_output(cmd, shell=True, executable=executable,\
  \ **kwargs)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 421, in check_output\n    return run(*popenargs, stdout=PIPE, timeout=timeout,\
  \ check=True,\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 526, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError:\
  \ Command ' singularity --quiet --silent exec --home /home/azureuser/tmp/sispcsx8q2/code/.test\
  \  continuumio/miniconda3:4.8.2 sh -c 'conda info --json'' returned non-zero exit\
  \ status 255."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Fs82iprx0%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Fs82iprx0%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.105GiB</td>
						<td>2.0 seconds</td>
						<td>2.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743114064">
								<a href="https://nf-co.re/hic">hic</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hic/tree/1.2.0">1.2.0</a>
						</td>
						<td>nextflow</td>
						<td>975 days</td>
						<td>2023-02-19 06:33:06</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2Fs2hk02go%2Flogs.tar.xz">0.210GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2Fs2hk02go%2Foutput.tar.xz">0.041GiB</a>
						</td>
						<td>1.643GiB</td>
						<td>642.2 seconds</td>
						<td>407.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743190944">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/2.6.1">2.6.1</a>
						</td>
						<td>nextflow</td>
						<td>971 days</td>
						<td>2023-02-19 03:31:04</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fk2muqbh0%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fk2muqbh0%2Foutput.tar.xz">0.016GiB</a>
						</td>
						<td>1.895GiB</td>
						<td>1273.1 seconds</td>
						<td>1159.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740233504">
								<a href="https://github.com/solida-core/dima">solida-core/dima</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/dima/tree/v1.1">v1.1</a>
						</td>
						<td>snakemake</td>
						<td>971 days</td>
						<td>2023-02-19 04:44:21</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- Snakefile
kind: KeyError
line_no: 10
rest: "'samples'\n  File \"/home/azureuser/tmp/qfg6f1oqkt/code/Snakefile\", line 10,\
  \ in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2Fjzkzdnc6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2Fjzkzdnc6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.1 seconds</td>
						<td>1.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743196224">
								<a href="https://nf-co.re/viralrecon">viralrecon</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/viralrecon/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>971 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Fcabq334l%2Flogs.tar.xz">0.245GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Fcabq334l%2Foutput.tar.xz">0.034GiB</a>
						</td>
						<td>2.051GiB</td>
						<td>3389.1 seconds</td>
						<td>1786.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745487008">
								<a href="https://nf-co.re/atacseq">atacseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/atacseq/tree/1.2.0">1.2.0</a>
						</td>
						<td>nextflow</td>
						<td>962 days</td>
						<td>2023-02-19 06:00:26</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: nextflow.dag.MultipleInputChannelException
msg: Channel `design_multiple_samples` has been used as an input by more than a process
  or an operator
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:295)\n\
  \tat nextflow.dag.DAG.inbound(DAG.groovy:207)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat nextflow.dag.DAG.addVertex(DAG.groovy:145)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat nextflow.dag.DAG.addOperatorNode(DAG.groovy:116)\n\tat nextflow.dag.NodeMarker.addOperatorNode(NodeMarker.groovy:64)\n\
  \tat nextflow.extension.OpCall.addGraphNode(OpCall.groovy:178)\n\tat nextflow.extension.OpCall.invoke(OpCall.groovy:168)\n\
  \tat nextflow.extension.OpCall.call(OpCall.groovy:113)\n\tat nextflow.plugin.extension.PluginExtensionProvider.invokeExtensionMethod(PluginExtensionProvider.groovy:279)\n\
  \tat groovy.runtime.metaclass.NextflowDelegatingMetaClass.invokeMethod(NextflowDelegatingMetaClass.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.PojoMetaClassSite.call(PojoMetaClassSite.java:44)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_cf1875fb.runScript(Script_cf1875fb:377)\n\tat nextflow.script.BaseScript.runDsl1(BaseScript.groovy:164)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\
  \tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\
  \tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2Fw4dasst2%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2Fw4dasst2%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.656GiB</td>
						<td>570.8 seconds</td>
						<td>578.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744422672">
								<a href="https://nf-co.re/chipseq">chipseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/chipseq/tree/1.2.0">1.2.0</a>
						</td>
						<td>nextflow</td>
						<td>962 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2Fzzudome2%2Flogs.tar.xz">0.505GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2Fzzudome2%2Foutput.tar.xz">0.081GiB</a>
						</td>
						<td>2.499GiB</td>
						<td>2901.0 seconds</td>
						<td>2087.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743113824">
								<a href="https://nf-co.re/hic">hic</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hic/tree/1.2.1">1.2.1</a>
						</td>
						<td>nextflow</td>
						<td>955 days</td>
						<td>2023-02-19 04:18:01</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2Fzx173g70%2Flogs.tar.xz">0.211GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2Fzx173g70%2Foutput.tar.xz">0.041GiB</a>
						</td>
						<td>1.647GiB</td>
						<td>657.5 seconds</td>
						<td>410.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743149472">
								<a href="https://nf-co.re/rnafusion">rnafusion</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnafusion/tree/1.2.0">1.2.0</a>
						</td>
						<td>nextflow</td>
						<td>949 days</td>
						<td>2023-02-19 04:21:45</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Fjwzcfakl%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Fjwzcfakl%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.521GiB</td>
						<td>24.5 seconds</td>
						<td>11.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744354976">
								<a href="https://nf-co.re/chipseq">chipseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/chipseq/tree/1.2.1">1.2.1</a>
						</td>
						<td>nextflow</td>
						<td>935 days</td>
						<td>2023-02-19 14:52:32</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2Fhxuaqk6d%2Flogs.tar.xz">0.505GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2Fhxuaqk6d%2Foutput.tar.xz">0.081GiB</a>
						</td>
						<td>2.379GiB</td>
						<td>2986.9 seconds</td>
						<td>1843.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745419312">
								<a href="https://nf-co.re/atacseq">atacseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/atacseq/tree/1.2.1">1.2.1</a>
						</td>
						<td>nextflow</td>
						<td>935 days</td>
						<td>2023-02-19 04:04:00</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: nextflow.dag.MultipleInputChannelException
msg: Channel `design_multiple_samples` has been used as an input by more than a process
  or an operator
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:295)\n\
  \tat nextflow.dag.DAG.inbound(DAG.groovy:207)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat nextflow.dag.DAG.addVertex(DAG.groovy:145)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat nextflow.dag.DAG.addOperatorNode(DAG.groovy:116)\n\tat nextflow.dag.NodeMarker.addOperatorNode(NodeMarker.groovy:64)\n\
  \tat nextflow.extension.OpCall.addGraphNode(OpCall.groovy:178)\n\tat nextflow.extension.OpCall.invoke(OpCall.groovy:168)\n\
  \tat nextflow.extension.OpCall.call(OpCall.groovy:113)\n\tat nextflow.plugin.extension.PluginExtensionProvider.invokeExtensionMethod(PluginExtensionProvider.groovy:279)\n\
  \tat groovy.runtime.metaclass.NextflowDelegatingMetaClass.invokeMethod(NextflowDelegatingMetaClass.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.PojoMetaClassSite.call(PojoMetaClassSite.java:44)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_cf1875fb.runScript(Script_cf1875fb:377)\n\tat nextflow.script.BaseScript.runDsl1(BaseScript.groovy:164)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\
  \tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\
  \tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2Fxy3tsze7%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2Fxy3tsze7%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.614GiB</td>
						<td>575.6 seconds</td>
						<td>377.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743115264">
								<a href="https://nf-co.re/hlatyping">hlatyping</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hlatyping/tree/1.2.0">1.2.0</a>
						</td>
						<td>nextflow</td>
						<td>912 days</td>
						<td>2023-02-19 03:31:06</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fcue8b032%2Flogs.tar.xz">0.038GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fcue8b032%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>1.519GiB</td>
						<td>272.4 seconds</td>
						<td>226.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743113584">
								<a href="https://nf-co.re/hic">hic</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hic/tree/1.2.2">1.2.2</a>
						</td>
						<td>nextflow</td>
						<td>899 days</td>
						<td>2023-02-19 03:48:35</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2Futll5n0m%2Flogs.tar.xz">0.211GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2Futll5n0m%2Foutput.tar.xz">0.041GiB</a>
						</td>
						<td>1.629GiB</td>
						<td>639.1 seconds</td>
						<td>326.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738256480">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.0">1.0</a>
						</td>
						<td>snakemake</td>
						<td>886 days</td>
						<td>2023-02-19 03:32:30</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- Snakefile
kind: IndexError
line_no: 362
rest: "list index out of range\n  File \"/home/azureuser/tmp/9n28odktia/code/Snakefile\"\
  , line 362, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fqm91w0zs%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fqm91w0zs%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.063GiB</td>
						<td>1.3 seconds</td>
						<td>1.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743136752">
								<a href="https://nf-co.re/mag">mag</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mag/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>867 days</td>
						<td>2023-02-20 05:07:38</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name nfcore-mag-1.1.0.img.pulling.1676869668746\
  \ docker://nfcore/mag:1.1.0 > /dev/null\n  status : 255\n  message:\n    INFO: \
  \   Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:933e1ed248f5470a3939dfd6467d28365355726f793951fdf819528d180aff00\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:6b5e6e9d001075f1740c33c76ef98f510fb808dd7d6f8edf6aa7d7db6248d9a1\n\
  \    Copying blob sha256:e013801287a05245b86a6f03422be5ba4791b7db37de109c84767633fefa840d\n\
  \    Copying blob sha256:e627393bf2016c751efed5e0d1dc2d1b9d4086396dda7422bfe0a32d31b56b5f\n\
  \    Copying blob sha256:6d40bd2b2084d2d569c9b30ef0966f16bfacad3764d56903e623dea946e373a3\n\
  \    Copying blob sha256:6b06b45ef97476781692af9ae1985dcd1267860cc0f56dbdbffa2912f2b0ef3b\n\
  \    Copying config sha256:894b836d2c8e1f46975257c1946e4ba57dc7317a9b081525e0df0506dde9f00c\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.65dec89125219c03b99172e15eec146b691da6e29ad9487831840bda038b6535\"\n\n\t\
  at nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\t\
  at nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\
  \tat nextflow.processor.TaskRun$isContainerEnabled$2.call(Unknown Source)\n\tat\
  \ nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name nfcore-mag-1.1.0.img.pulling.1676869668746\
  \ docker://nfcore/mag:1.1.0 > /dev/null\n  status : 255\n  message:\n    INFO: \
  \   Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:933e1ed248f5470a3939dfd6467d28365355726f793951fdf819528d180aff00\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:6b5e6e9d001075f1740c33c76ef98f510fb808dd7d6f8edf6aa7d7db6248d9a1\n\
  \    Copying blob sha256:e013801287a05245b86a6f03422be5ba4791b7db37de109c84767633fefa840d\n\
  \    Copying blob sha256:e627393bf2016c751efed5e0d1dc2d1b9d4086396dda7422bfe0a32d31b56b5f\n\
  \    Copying blob sha256:6d40bd2b2084d2d569c9b30ef0966f16bfacad3764d56903e623dea946e373a3\n\
  \    Copying blob sha256:6b06b45ef97476781692af9ae1985dcd1267860cc0f56dbdbffa2912f2b0ef3b\n\
  \    Copying config sha256:894b836d2c8e1f46975257c1946e4ba57dc7317a9b081525e0df0506dde9f00c\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.65dec89125219c03b99172e15eec146b691da6e29ad9487831840bda038b6535\"\n\n\t\
  at nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 05:08:38.221 [Actor Thread 21] DEBUG nextflow.Session\
  \ - Session aborted -- Cause: java.lang.IllegalStateException: Failed to pull singularity\
  \ image\n  command: singularity pull  --name nfcore-mag-1.1.0.img.pulling.1676869668746\
  \ docker://nfcore/mag:1.1.0 > /dev/null\n  status : 255\n  message:\n    INFO: \
  \   Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:933e1ed248f5470a3939dfd6467d28365355726f793951fdf819528d180aff00\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:6b5e6e9d001075f1740c33c76ef98f510fb808dd7d6f8edf6aa7d7db6248d9a1\n\
  \    Copying blob sha256:e013801287a05245b86a6f03422be5ba4791b7db37de109c84767633fefa840d\n\
  \    Copying blob sha256:e627393bf2016c751efed5e0d1dc2d1b9d4086396dda7422bfe0a32d31b56b5f\n\
  \    Copying blob sha256:6d40bd2b2084d2d569c9b30ef0966f16bfacad3764d56903e623dea946e373a3\n\
  \    Copying blob sha256:6b06b45ef97476781692af9ae1985dcd1267860cc0f56dbdbffa2912f2b0ef3b\n\
  \    Copying config sha256:894b836d2c8e1f46975257c1946e4ba57dc7317a9b081525e0df0506dde9f00c\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.65dec89125219c03b99172e15eec146b691da6e29ad9487831840bda038b6535\"\n\nFeb-20\
  \ 05:08:38.281 [Actor Thread 21] DEBUG nextflow.Session - The following nodes are\
  \ still active:\n[process] get_busco_version\n  status=ACTIVE\n  port 0: (cntrl)\
  \ -     ; channel: $\n\n[process] get_software_versions\n  status=ACTIVE\n  port\
  \ 0: (value) OPEN  ; channel: busco_version\n  port 1: (cntrl) -     ; channel:\
  \ $\n\n[process] fastqc_raw\n  status=ACTIVE\n  port 0: (queue) closed; channel:\
  \ -\n  port 1: (cntrl) -     ; channel: $\n\n[process] remove_phix\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] fastqc_trimmed\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n\
  \  port 1: (cntrl) -     ; channel: $\n\n[process] filtlong\n  status=ACTIVE\n \
  \ port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process]\
  \ nanoplot\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl)\
  \ -     ; channel: $\n\n[process] centrifuge\n  status=ACTIVE\n  port 0: (queue)\
  \ OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] kraken2\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] krona\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel:\
  \ -\n  port 1: (cntrl) -     ; channel: $\n\n[process] megahit\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] spadeshybrid\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n\
  \  port 1: (cntrl) -     ; channel: $\n\n[process] spades\n  status=ACTIVE\n  port\
  \ 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process]\
  \ quast\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl)\
  \ -     ; channel: $\n\n[process] bowtie2\n  status=ACTIVE\n  port 0: (queue) OPEN\
  \  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] metabat\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound ; channel: min_size\n\
  \  port 2: (value) bound ; channel: max_unbinned\n  port 3: (value) bound ; channel:\
  \ min_length_unbinned\n  port 4: (cntrl) -     ; channel: $\n\n[process] busco\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] busco_plot\n  status=ACTIVE\n  port 0: (queue) OPEN\
  \  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] busco_summary\n\
  \  status=ACTIVE\n  port 0: (value) OPEN  ; channel: short_summary.*.txt\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] quast_bins\n  status=ACTIVE\n  port\
  \ 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process]\
  \ merge_quast_and_busco\n  status=ACTIVE\n  port 0: (value) OPEN  ; channel: quast_bin_sum\n\
  \  port 1: (value) OPEN  ; channel: busco_sum\n  port 2: (cntrl) -     ; channel:\
  \ $\n\n[process] cat\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] multiqc\n  status=ACTIVE\n  port 0:\
  \ (value) bound ; channel: multiqc_config\n  port 1: (value) bound ; channel: mqc_custom_config\n\
  \  port 2: (value) OPEN  ; channel: fastqc_raw\n  port 3: (value) OPEN  ; channel:\
  \ fastqc_trimmed\n  port 4: (value) bound ; channel: host_removal\n  port 5: (value)\
  \ OPEN  ; channel: quast*/*\n  port 6: (value) OPEN  ; channel: short_summary\n\
  \  port 7: (value) OPEN  ; channel: software_versions/*\n  port 8: (queue) OPEN\
  \  ; channel: workflow_summary\n  port 9: (cntrl) -     ; channel: $\n\nFeb-20 05:08:38.285\
  \ [main] DEBUG nextflow.Session - Session await > all processes finished\nFeb-20\
  \ 05:08:38.285 [main] DEBUG nextflow.Session - Session await > all barriers passed\n\
  Feb-20 05:08:38.550 [main] INFO  nextflow.Nextflow - -\e[0;35m[nf-core/mag]\e[0;31m\
  \ Pipeline completed with errors\e[0m-\nFeb-20 05:08:38.560 [main] DEBUG nextflow.trace.WorkflowStatsObserver\
  \ - Workflow completed > WorkflowStats[succeededCount=0; failedCount=0; ignoredCount=0;\
  \ cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0;\
  \ abortedCount=0; succeedDuration=0ms; failedDuration=0ms; cachedDuration=0ms;loadCpus=0;\
  \ loadMemory=0; peakRunning=0; peakCpus=0; peakMemory=0; ]\nFeb-20 05:08:38.560\
  \ [main] DEBUG nextflow.trace.TraceFileObserver - Workflow completed -- saving trace\
  \ file\nFeb-20 05:08:38.564 [main] DEBUG nextflow.trace.ReportObserver - Workflow\
  \ completed -- rendering execution report\nFeb-20 05:08:38.569 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Execution report summary data:\n  []\nFeb-20 05:08:39.221 [main] DEBUG nextflow.trace.TimelineObserver\
  \ - Workflow completed -- rendering execution timeline\nFeb-20 05:08:39.425 [main]\
  \ DEBUG nextflow.cache.CacheDB - Closing CacheDB done\nFeb-20 05:08:39.470 [main]\
  \ DEBUG nextflow.script.ScriptRunner - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fkbb55z58%2Flogs.tar.xz">0.016GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fkbb55z58%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.813GiB</td>
						<td>47.6 seconds</td>
						<td>61.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743142032">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/1.6.0">1.6.0</a>
						</td>
						<td>nextflow</td>
						<td>865 days</td>
						<td>2023-02-19 03:49:58</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "CometAdapter  -in MM15_Melanom_1_B_1.mzML \\\n              -out\
  \ MM16_Melanom_B_3.idXML \\\n              -threads 2 \\\n              -database\
  \ test_decoy.fasta \\\n              -precursor_mass_tolerance 5 \\\n          \
  \    -fragment_bin_tolerance 0.02 \\\n              -fragment_bin_offset 0 \\\n\
  \              -num_hits 1 \\\n              -digest_mass_range 800:2500 \\\n  \
  \            -max_variable_mods_in_peptide 3 \\\n              -allowed_missed_cleavages\
  \ 0 \\\n              -precursor_charge 2:3 \\\n              -activation_method\
  \ ALL \\\n              -variable_modifications 'Oxidation (M)' \\\n           \
  \   -fixed_modifications  \\\n              -enzyme 'unspecific cleavage' \\\n \
  \             -spectrum_batch_size 500 \\\n               \\\n               \\\n\
  \               \\\n               \\\n               \\\n               \\"
error: "Comet is writing the default parameter file...\nExecuting: /opt/conda/envs/nf-core-mhcquant-1.6.0/bin/comet.exe\n\
  Executed /opt/conda/envs/nf-core-mhcquant-1.6.0/bin/comet.exe successfully!\nProgress\
  \ of 'loading spectra list':\n\n  32.39 %               \n-- done [took 0.81 s (CPU),\
  \ 1.43 s (Wall)] -- \nProgress of 'loading chromatogram list':\n\n-- done [took\
  \ 0.00 s (CPU), 0.00 s (Wall)] -- \nFATAL ERROR: External invocation of /opt/conda/envs/nf-core-mhcquant-1.6.0/bin/comet.exe\
  \ failed. Standard output and error were:\n<Executing: /opt/conda/envs/nf-core-mhcquant-1.6.0/bin/comet.exe>\
  \ occurred 2 times\nStandard output:  Comet version \"2016.01 rev. 3\"\n\n Search\
  \ start:  02/19/2023, 04:01:08 AM\n - Input file: MM15_Melanom_1_B_1.mzML\n   -\
  \ Load spectra: 503\n     - Search progress:   0%\b\b\b\b  0%\b\b\b\b  1%\b\b\b\b\
  \  1%\b\b\b\b  2%\b\b\b\b  2%\b\b\b\b  3%\b\b\b\b  4%\b\b\b\b  4%\b\b\b\b  5%\b\b\
  \b\b  5%\b\b\b\b  6%\b\b\b\b  6%\b\b\b\b  7%\b\b\b\b  7%\b\b\b\b  8%\b\b\b\b  8%\b\
  \b\b\b  9%\b\b\b\b  9%\b\b\b\b 10%\b\b\b\b 10%\b\b\b\b 11%\b\b\b\b 11%\b\b\b\b 11%\b\
  \b\b\b 12%\b\b\b\b 12%\b\b\b\b 13%\b\b\b\b 13%\b\b\b\b 13%\b\b\b\b 14%\b\b\b\b 14%\b\
  \b\b\b 15%\b\b\b\b 15%\b\b\b\b 15%\b\b\b\b 16%\b\b\b\b 17%\b\b\b\b 17%\b\b\b\b 17%\b\
  \b\b\b 18%\b\b\b\b 18%\b\b\b\b 19%\b\b\b\b 19%\b\b\b\b 19%\b\b\b\b 20%\b\b\b\b 20%\b\
  \b\b\b 21%\b\b\b\b 21%\b\b\b\b 22%\b\b\b\b 22%\b\b\b\b 23%\b\b\b\b 24%\b\b\b\b 24%\b\
  \b\b\b 25%\b\b\b\b 25%\b\b\b\b 26%\b\b\b\b 26%\b\b\b\b 27%\b\b\b\b 27%\b\b\b\b 27%\b\
  \b\b\b 28%\b\b\b\b 28%\b\b\b\b 29%\b\b\b\b 29%\b\b\b\b 30%\b\b\b\b 30%\b\b\b\b 31%\b\
  \b\b\b 31%\b\b\b\b 32%\b\b\b\b 32%\b\b\b\b 33%\b\b\b\b 33%\b\b\b\b 33%\b\b\b\b 34%\b\
  \b\b\b 34%\b\b\b\b 35%\b\b\b\b 35%\b\b\b\b 36%\b\b\b\b 36%\b\b\b\b 36%\b\b\b\b 37%\b\
  \b\b\b 37%\b\b\b\b 38%\b\b\b\b 38%\b\b\b\b 38%\b\b\b\b 39%\b\b\b\b 40%\b\b\b\b 40%\b\
  \b\b\b 41%\b\b\b\b 41%\b\b\b\b 42%\b\b\b\b 42%\b\b\b\b 43%\b\b\b\b 43%\b\b\b\b 43%\b\
  \b\b\b 44%\b\b\b\b 44%\b\b\b\b 45%\b\b\b\b 45%\b\b\b\b 46%\b\b\b\b 46%\b\b\b\b 47%\b\
  \b\b\b 47%\b\b\b\b 48%\b\b\b\b 48%\b\b\b\b 49%\b\b\b\b 49%\b\b\b\b 50%\b\b\b\b 50%\b\
  \b\b\b 51%\b\b\b\b 52%\b\b\b\b 52%\b\b\b\b 53%\b\b\b\b 53%\b\b\b\b 54%\b\b\b\b 54%\b\
  \b\b\b 55%\b\b\b\b 55%\b\b\b\b 56%\b\b\b\b 56%\b\b\b\b 56%\b\b\b\b 57%\b\b\b\b 57%\b\
  \b\b\b 57%\b\b\b\b 58%\b\b\b\b 58%\b\b\b\b 59%\b\b\b\b 59%\b\b\b\b 60%\b\b\b\b 61%\b\
  \b\b\b 61%\b\b\b\b 61%\b\b\b\b 62%\b\b\b\b 63%\b\b\b\b 63%\b\b\b\b 64%\b\b\b\b 64%\b\
  \b\b\b 65%\b\b\b\b 65%\b\b\b\b 66%\b\b\b\b 66%\b\b\b\b 66%\b\b\b\b 67%\b\b\b\b 67%\b\
  \b\b\b 68%\b\b\b\b 68%\b\b\b\b 69%\b\b\b\b 69%\b\b\b\b 69%\b\b\b\b 70%\b\b\b\b 70%\b\
  \b\b\b 71%\b\b\b\b 71%\b\b\b\b 72%\b\b\b\b 72%\b\b\b\b 73%\b\b\b\b 73%\b\b\b\b 74%\b\
  \b\b\b 74%\b\b\b\b 75%\b\b\b\b 75%\b\b\b\b 76%\b\b\b\b 76%\b\b\b\b 76%\b\b\b\b 77%\b\
  \b\b\b 77%\b\b\b\b 78%\b\b\b\b 78%\b\b\b\b 79%\b\b\b\b 79%\b\b\b\b 80%\b\b\b\b 80%\b\
  \b\b\b 81%\b\b\b\b 81%\b\b\b\b 82%\b\b\b\b 83%\b\b\b\b 83%\b\b\b\b 83%\b\b\b\b 84%\b\
  \b\b\b 84%\b\b\b\b 84%\b\b\b\b 85%\b\b\b\b 86%\b\b\b\b 86%\b\b\b\b 86%\b\b\b\b 87%\b\
  \b\b\b 87%\b\b\b\b 88%\b\b\b\b 88%\b\b\b\b 89%\b\b\b\b 90%\b\b\b\b 90%\b\b\b\b 91%\b\
  \b\b\b 91%\b\b\b\b 91%\b\b\b\b 91%\b\b\b\b 92%\b\b\b\b 93%\b\b\b\b 93%\b\b\b\b 94%\b\
  \b\b\b 94%\b\b\b\b 95%\b\b\b\b 95%\b\b\b\b 96%\n     - Post analysis:  done\n  \
  \ - Load spectra: 504\n     - Search progress:  96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\
  \b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 97%\n     - Post analysis:  done\n  \
  \ - Load spectra: 502\n     - Search progress:  97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\
  \b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 9796%\b\b\b\b 96%\b\b\b\b\
  \ 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\
  \b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 97%\n     - Post analysis:  done\n   - Load spectra:\
  \ 502\n     - Search progress:  97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b\
  \ 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\
  \b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 98%\n     - Post analysis:  done\n   - Load spectra:\
  \ 391\n     - Search progress:  98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b\
  \ 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\
  \b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b100%\n     - Post analysis:  done\n Search end:  \
  \  02/19/2023, 04:10:29 AM\n\n\nStandard error: \nExit code: 6\nCometAdapter took\
  \ 09:22 m (wall), 0.97 s (CPU), 0.06 s (system), 0.91 s (user)."
exit_status: 9
kind: CalledProcessError
output: "Comet is writing the default parameter file...\nExecuting: /opt/conda/envs/nf-core-mhcquant-1.6.0/bin/comet.exe\n\
  Executed /opt/conda/envs/nf-core-mhcquant-1.6.0/bin/comet.exe successfully!\nProgress\
  \ of 'loading spectra list':\n\n  32.39 %               \n-- done [took 0.81 s (CPU),\
  \ 1.43 s (Wall)] -- \nProgress of 'loading chromatogram list':\n\n-- done [took\
  \ 0.00 s (CPU), 0.00 s (Wall)] -- \nFATAL ERROR: External invocation of /opt/conda/envs/nf-core-mhcquant-1.6.0/bin/comet.exe\
  \ failed. Standard output and error were:\n<Executing: /opt/conda/envs/nf-core-mhcquant-1.6.0/bin/comet.exe>\
  \ occurred 2 times\nStandard output:  Comet version \"2016.01 rev. 3\"\n\n Search\
  \ start:  02/19/2023, 04:01:08 AM\n - Input file: MM15_Melanom_1_B_1.mzML\n   -\
  \ Load spectra: 503\n     - Search progress:   0%\b\b\b\b  0%\b\b\b\b  1%\b\b\b\b\
  \  1%\b\b\b\b  2%\b\b\b\b  2%\b\b\b\b  3%\b\b\b\b  4%\b\b\b\b  4%\b\b\b\b  5%\b\b\
  \b\b  5%\b\b\b\b  6%\b\b\b\b  6%\b\b\b\b  7%\b\b\b\b  7%\b\b\b\b  8%\b\b\b\b  8%\b\
  \b\b\b  9%\b\b\b\b  9%\b\b\b\b 10%\b\b\b\b 10%\b\b\b\b 11%\b\b\b\b 11%\b\b\b\b 11%\b\
  \b\b\b 12%\b\b\b\b 12%\b\b\b\b 13%\b\b\b\b 13%\b\b\b\b 13%\b\b\b\b 14%\b\b\b\b 14%\b\
  \b\b\b 15%\b\b\b\b 15%\b\b\b\b 15%\b\b\b\b 16%\b\b\b\b 17%\b\b\b\b 17%\b\b\b\b 17%\b\
  \b\b\b 18%\b\b\b\b 18%\b\b\b\b 19%\b\b\b\b 19%\b\b\b\b 19%\b\b\b\b 20%\b\b\b\b 20%\b\
  \b\b\b 21%\b\b\b\b 21%\b\b\b\b 22%\b\b\b\b 22%\b\b\b\b 23%\b\b\b\b 24%\b\b\b\b 24%\b\
  \b\b\b 25%\b\b\b\b 25%\b\b\b\b 26%\b\b\b\b 26%\b\b\b\b 27%\b\b\b\b 27%\b\b\b\b 27%\b\
  \b\b\b 28%\b\b\b\b 28%\b\b\b\b 29%\b\b\b\b 29%\b\b\b\b 30%\b\b\b\b 30%\b\b\b\b 31%\b\
  \b\b\b 31%\b\b\b\b 32%\b\b\b\b 32%\b\b\b\b 33%\b\b\b\b 33%\b\b\b\b 33%\b\b\b\b 34%\b\
  \b\b\b 34%\b\b\b\b 35%\b\b\b\b 35%\b\b\b\b 36%\b\b\b\b 36%\b\b\b\b 36%\b\b\b\b 37%\b\
  \b\b\b 37%\b\b\b\b 38%\b\b\b\b 38%\b\b\b\b 38%\b\b\b\b 39%\b\b\b\b 40%\b\b\b\b 40%\b\
  \b\b\b 41%\b\b\b\b 41%\b\b\b\b 42%\b\b\b\b 42%\b\b\b\b 43%\b\b\b\b 43%\b\b\b\b 43%\b\
  \b\b\b 44%\b\b\b\b 44%\b\b\b\b 45%\b\b\b\b 45%\b\b\b\b 46%\b\b\b\b 46%\b\b\b\b 47%\b\
  \b\b\b 47%\b\b\b\b 48%\b\b\b\b 48%\b\b\b\b 49%\b\b\b\b 49%\b\b\b\b 50%\b\b\b\b 50%\b\
  \b\b\b 51%\b\b\b\b 52%\b\b\b\b 52%\b\b\b\b 53%\b\b\b\b 53%\b\b\b\b 54%\b\b\b\b 54%\b\
  \b\b\b 55%\b\b\b\b 55%\b\b\b\b 56%\b\b\b\b 56%\b\b\b\b 56%\b\b\b\b 57%\b\b\b\b 57%\b\
  \b\b\b 57%\b\b\b\b 58%\b\b\b\b 58%\b\b\b\b 59%\b\b\b\b 59%\b\b\b\b 60%\b\b\b\b 61%\b\
  \b\b\b 61%\b\b\b\b 61%\b\b\b\b 62%\b\b\b\b 63%\b\b\b\b 63%\b\b\b\b 64%\b\b\b\b 64%\b\
  \b\b\b 65%\b\b\b\b 65%\b\b\b\b 66%\b\b\b\b 66%\b\b\b\b 66%\b\b\b\b 67%\b\b\b\b 67%\b\
  \b\b\b 68%\b\b\b\b 68%\b\b\b\b 69%\b\b\b\b 69%\b\b\b\b 69%\b\b\b\b 70%\b\b\b\b 70%\b\
  \b\b\b 71%\b\b\b\b 71%\b\b\b\b 72%\b\b\b\b 72%\b\b\b\b 73%\b\b\b\b 73%\b\b\b\b 74%\b\
  \b\b\b 74%\b\b\b\b 75%\b\b\b\b 75%\b\b\b\b 76%\b\b\b\b 76%\b\b\b\b 76%\b\b\b\b 77%\b\
  \b\b\b 77%\b\b\b\b 78%\b\b\b\b 78%\b\b\b\b 79%\b\b\b\b 79%\b\b\b\b 80%\b\b\b\b 80%\b\
  \b\b\b 81%\b\b\b\b 81%\b\b\b\b 82%\b\b\b\b 83%\b\b\b\b 83%\b\b\b\b 83%\b\b\b\b 84%\b\
  \b\b\b 84%\b\b\b\b 84%\b\b\b\b 85%\b\b\b\b 86%\b\b\b\b 86%\b\b\b\b 86%\b\b\b\b 87%\b\
  \b\b\b 87%\b\b\b\b 88%\b\b\b\b 88%\b\b\b\b 89%\b\b\b\b 90%\b\b\b\b 90%\b\b\b\b 91%\b\
  \b\b\b 91%\b\b\b\b 91%\b\b\b\b 91%\b\b\b\b 92%\b\b\b\b 93%\b\b\b\b 93%\b\b\b\b 94%\b\
  \b\b\b 94%\b\b\b\b 95%\b\b\b\b 95%\b\b\b\b 96%\n     - Post analysis:  done\n  \
  \ - Load spectra: 504\n     - Search progress:  96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\
  \b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 97%\n     - Post analysis:  done\n  \
  \ - Load spectra: 502\n     - Search progress:  97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\
  \b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 9796%\b\b\b\b 96%\b\b\b\b\
  \ 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\
  \b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\b\b\b 96%\b\
  \b\b\b 96%\b\b\b\b 96%\b\b\b\b 97%\n     - Post analysis:  done\n   - Load spectra:\
  \ 502\n     - Search progress:  97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b\
  \ 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\
  \b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\b\b\b 97%\b\
  \b\b\b 97%\b\b\b\b 97%\b\b\b\b 98%\n     - Post analysis:  done\n   - Load spectra:\
  \ 391\n     - Search progress:  98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b\
  \ 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\
  \b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\b\b\b 98%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\b\b\b 99%\b\
  \b\b\b 99%\b\b\b\b 99%\b\b\b\b100%\n     - Post analysis:  done\n Search end:  \
  \  02/19/2023, 04:10:29 AM\n\n\nStandard error: \nExit code: 6\nCometAdapter took\
  \ 09:22 m (wall), 0.97 s (CPU), 0.06 s (system), 0.91 s (user)."
process: db_search_comet (MM16_Melanom)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- 0e
- f0b7c9490848b4cf08746afaaafd52

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fbo0ir1gi%2Flogs.tar.xz">0.076GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fbo0ir1gi%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.802GiB</td>
						<td>2710.5 seconds</td>
						<td>1780.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740181712">
								<a href="https://github.com/solida-core/dima">solida-core/dima</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/dima/tree/v1.2">v1.2</a>
						</td>
						<td>snakemake</td>
						<td>859 days</td>
						<td>2023-02-19 07:00:51</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- Snakefile
kind: KeyError
line_no: 10
rest: "'samples'\n  File \"/home/azureuser/tmp/6qzkdnl3ka/code/Snakefile\", line 10,\
  \ in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2Fia5dt17t%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2Fia5dt17t%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.9 seconds</td>
						<td>2.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744085424">
								<a href="https://nf-co.re/cageseq">cageseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/cageseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>856 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cageseq%2Fr9n7qegv%2Flogs.tar.xz">0.127GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cageseq%2Fr9n7qegv%2Foutput.tar.xz">0.005GiB</a>
						</td>
						<td>1.845GiB</td>
						<td>492.2 seconds</td>
						<td>443.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743148512">
								<a href="https://nf-co.re/proteomicslfq">proteomicslfq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/proteomicslfq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>853 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/proteomicslfq%2Fgeke7ybd%2Flogs.tar.xz">0.183GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/proteomicslfq%2Fgeke7ybd%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.698GiB</td>
						<td>1281.6 seconds</td>
						<td>938.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743109744">
								<a href="https://nf-co.re/epitopeprediction">epitopeprediction</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/epitopeprediction/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>852 days</td>
						<td>2023-02-19 06:15:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epitopeprediction%2F17ojtabe%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epitopeprediction%2F17ojtabe%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.718GiB</td>
						<td>533.8 seconds</td>
						<td>378.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743106864">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.2.0">2.2.0</a>
						</td>
						<td>nextflow</td>
						<td>851 days</td>
						<td>2023-02-19 04:47:36</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 04:47:54.639
kind: NextflowMiscError
process: main
rest: '[nf-core/eager] error: --input was not supplied! Please check ''--help'' or
  documentation under ''running the pipeline'' for details'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Ft2np5eyw%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Ft2np5eyw%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.758GiB</td>
						<td>33.1 seconds</td>
						<td>18.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738268960">
								<a href="https://github.com/GooglingTheCancerGenome/sv-callers">GooglingTheCancerGenome/sv-callers</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.1.2">v1.1.2</a>
						</td>
						<td>snakemake</td>
						<td>845 days</td>
						<td>2023-02-19 04:07:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2F63wptb62%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2F63wptb62%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743106624">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.2.1">2.2.1</a>
						</td>
						<td>nextflow</td>
						<td>843 days</td>
						<td>2023-02-19 06:46:32</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 06:46:50.219
kind: NextflowMiscError
process: main
rest: '[nf-core/eager] error: --input was not supplied! Please check ''--help'' or
  documentation under ''running the pipeline'' for details'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F8hfe80dn%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F8hfe80dn%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.821GiB</td>
						<td>31.2 seconds</td>
						<td>17.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746877680">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/1.1.3">1.1.3</a>
						</td>
						<td>nextflow</td>
						<td>839 days</td>
						<td>2023-02-19 04:06:45</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: metadataCategory.r Metadata.tsv
error: "Warning message:\nIn read.table(file = file, header = header, sep = sep, quote\
  \ = quote,  :\n  incomplete final line found by readTableHeader on 'Metadata.tsv'\n\
  Error in 2:ncol(data) : argument of length 0\nExecution halted"
exit_status: 1
kind: CalledProcessError
output: (empty)
process: metadata_category_all (1)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '67'
- 3e77b637b7ac6df24b41133d66191f

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2F4w5y3by0%2Flogs.tar.xz">0.006GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2F4w5y3by0%2Foutput.tar.xz">0.004GiB</a>
						</td>
						<td>1.769GiB</td>
						<td>665.5 seconds</td>
						<td>429.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745791584">
								<a href="https://nf-co.re/bacass">bacass</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/bacass/tree/1.1.1">1.1.1</a>
						</td>
						<td>nextflow</td>
						<td>837 days</td>
						<td>2023-02-20 04:32:19</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name quay.io-biocontainers-quast-5.0.2--py37pl526hb5aa323_2.img.pulling.1676868436918\
  \ docker://quay.io/biocontainers/quast:5.0.2--py37pl526hb5aa323_2 > /dev/null\n\
  \  status : 255\n  message:\n    INFO:    Converting OCI blobs to SIF format\n \
  \   INFO:    Starting build...\n    Getting image source signatures\n    Copying\
  \ blob sha256:12fe7046e2c5e38c781d69d3e77e4bc5ab4d95259ebfefb86e0801f5874af7d0\n\
  \    Copying blob sha256:4857ca6d930b60d4610559919f11d2ed48099ce5e0f4178892304dfa0df4f11a\n\
  \    Copying blob sha256:e6bcdfebf6a068565be0a4f6c4dbdc5f3c4b37bee8a51514663276f8f7514c78\n\
  \    Copying config sha256:e3bfea3cd3a2a5d32d60bb18b3a6070a5e52b6abb25eb37557623b63f0004a42\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.3a2a1dc6b53dcc86cec588d28685a7f14a903c6d449e088e75be7c9203c33e84\"\n\n\t\
  at nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\t\
  at nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\
  \tat nextflow.processor.TaskRun$isContainerEnabled$2.call(Unknown Source)\n\tat\
  \ nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name quay.io-biocontainers-quast-5.0.2--py37pl526hb5aa323_2.img.pulling.1676868436918\
  \ docker://quay.io/biocontainers/quast:5.0.2--py37pl526hb5aa323_2 > /dev/null\n\
  \  status : 255\n  message:\n    INFO:    Converting OCI blobs to SIF format\n \
  \   INFO:    Starting build...\n    Getting image source signatures\n    Copying\
  \ blob sha256:12fe7046e2c5e38c781d69d3e77e4bc5ab4d95259ebfefb86e0801f5874af7d0\n\
  \    Copying blob sha256:4857ca6d930b60d4610559919f11d2ed48099ce5e0f4178892304dfa0df4f11a\n\
  \    Copying blob sha256:e6bcdfebf6a068565be0a4f6c4dbdc5f3c4b37bee8a51514663276f8f7514c78\n\
  \    Copying config sha256:e3bfea3cd3a2a5d32d60bb18b3a6070a5e52b6abb25eb37557623b63f0004a42\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.3a2a1dc6b53dcc86cec588d28685a7f14a903c6d449e088e75be7c9203c33e84\"\n\n\t\
  at nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 04:48:05.038 [Actor Thread 1] DEBUG nextflow.Session\
  \ - Session aborted -- Cause: java.lang.IllegalStateException: Failed to pull singularity\
  \ image\n  command: singularity pull  --name quay.io-biocontainers-quast-5.0.2--py37pl526hb5aa323_2.img.pulling.1676868436918\
  \ docker://quay.io/biocontainers/quast:5.0.2--py37pl526hb5aa323_2 > /dev/null\n\
  \  status : 255\n  message:\n    INFO:    Converting OCI blobs to SIF format\n \
  \   INFO:    Starting build...\n    Getting image source signatures\n    Copying\
  \ blob sha256:12fe7046e2c5e38c781d69d3e77e4bc5ab4d95259ebfefb86e0801f5874af7d0\n\
  \    Copying blob sha256:4857ca6d930b60d4610559919f11d2ed48099ce5e0f4178892304dfa0df4f11a\n\
  \    Copying blob sha256:e6bcdfebf6a068565be0a4f6c4dbdc5f3c4b37bee8a51514663276f8f7514c78\n\
  \    Copying config sha256:e3bfea3cd3a2a5d32d60bb18b3a6070a5e52b6abb25eb37557623b63f0004a42\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.3a2a1dc6b53dcc86cec588d28685a7f14a903c6d449e088e75be7c9203c33e84\"\n\nFeb-20\
  \ 04:48:05.092 [Actor Thread 1] DEBUG nextflow.Session - The following nodes are\
  \ still active:\n[process] quast\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel:\
  \ -\n  port 1: (cntrl) -     ; channel: $\n\n[process] prokka\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] dfast\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1:\
  \ (value) bound ; channel: config\n  port 2: (cntrl) -     ; channel: $\n\n[process]\
  \ get_software_versions\n  status=ACTIVE\n  port 0: (value) OPEN  ; channel: quast_version\n\
  \  port 1: (value) bound ; channel: porechop_version\n  port 2: (value) bound ;\
  \ channel: pycoqc_version\n  port 3: (value) bound ; channel: unicycler_version\n\
  \  port 4: (value) bound ; channel: canu_version\n  port 5: (value) OPEN  ; channel:\
  \ prokka_version\n  port 6: (value) OPEN  ; channel: dfast_version\n  port 7: (value)\
  \ bound ; channel: nanopolish_version\n  port 8: (value) bound ; channel: samtools_version\n\
  \  port 9: (value) bound ; channel: nanoplot_version\n  port 10: (value) bound ;\
  \ channel: medaka_version\n  port 11: (cntrl) -     ; channel: $\n\n[process] multiqc\n\
  \  status=ACTIVE\n  port 0: (value) bound ; channel: multiqc_config\n  port 1: (value)\
  \ bound ; channel: mqc_custom_config\n  port 2: (value) OPEN  ; channel: quast_logs/*\n\
  \  port 3: (value) bound ; channel: fastqc/*\n  port 4: (value) OPEN  ; channel:\
  \ software_versions/*\n  port 5: (queue) OPEN  ; channel: workflow_summary\n  port\
  \ 6: (cntrl) -     ; channel: $\n\nFeb-20 04:48:05.094 [main] DEBUG nextflow.Session\
  \ - Session await > all processes finished\nFeb-20 04:48:05.094 [main] DEBUG nextflow.Session\
  \ - Session await > all barriers passed\nFeb-20 04:48:05.098 [main] WARN  nextflow.script.ScriptBinding\
  \ - Access to undefined parameter `email_on_fail` -- Initialise it to a default\
  \ value eg. `params.email_on_fail = some_value`\nFeb-20 04:48:05.197 [Task submitter]\
  \ DEBUG n.processor.TaskPollingMonitor - %% executor local > tasks in the submission\
  \ queue: 1 -- tasks to be submitted are shown below\n~> TaskHandler[id: 17; name:\
  \ get_software_versions; status: NEW; exit: -; error: -; workDir: /home/azureuser/tmp/x6zrqz3v4s/code/work/54/abaac5ae9a52d16965828f55d78c8c]\n\
  Feb-20 04:48:05.529 [main] INFO  nextflow.Nextflow - -\e[0;35m[nf-core/bacass]\e\
  [0;31m Pipeline completed with errors\e[0m-\nFeb-20 04:48:05.535 [main] WARN  n.processor.TaskPollingMonitor\
  \ - Killing running tasks (1)\nFeb-20 04:48:05.559 [main] DEBUG nextflow.trace.WorkflowStatsObserver\
  \ - Workflow completed > WorkflowStats[succeededCount=6; failedCount=0; ignoredCount=0;\
  \ cachedCount=0; pendingCount=1; submittedCount=0; runningCount=1; retriesCount=0;\
  \ abortedCount=0; succeedDuration=20m 14s; failedDuration=0ms; cachedDuration=0ms;loadCpus=2;\
  \ loadMemory=6 GB; peakRunning=2; peakCpus=4; peakMemory=12 GB; ]\nFeb-20 04:48:05.559\
  \ [main] DEBUG nextflow.trace.TraceFileObserver - Workflow completed -- saving trace\
  \ file\nFeb-20 04:48:05.561 [main] DEBUG nextflow.trace.ReportObserver - Workflow\
  \ completed -- rendering execution report\nFeb-20 04:48:05.654 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Execution report summary data:\n  [{\"cpuUsage\":{\"mean\":59.33,\"min\":59.05,\"\
  q1\":59.19,\"q2\":59.33,\"q3\":59.46,\"max\":59.6,\"minLabel\":\"trim_and_combine\
  \ (ERR044595)\",\"maxLabel\":\"trim_and_combine (ERR064912)\",\"q1Label\":\"trim_and_combine\
  \ (ERR044595)\",\"q2Label\":\"trim_and_combine (ERR044595)\",\"q3Label\":\"trim_and_combine\
  \ (ERR044595)\"},\"process\":\"trim_and_combine\",\"mem\":{\"mean\":16244736,\"\
  min\":16207872,\"q1\":16226304,\"q2\":16244736,\"q3\":16263168,\"max\":16281600,\"\
  minLabel\":\"trim_and_combine (ERR064912)\",\"maxLabel\":\"trim_and_combine (ERR044595)\"\
  ,\"q1Label\":\"trim_and_combine (ERR064912)\",\"q2Label\":\"trim_and_combine (ERR064912)\"\
  ,\"q3Label\":\"trim_and_combine (ERR064912)\"},\"memUsage\":{\"mean\":0.25,\"min\"\
  :0.25,\"q1\":0.25,\"q2\":0.25,\"q3\":0.25,\"max\":0.25,\"minLabel\":\"trim_and_combine\
  \ (ERR064912)\",\"maxLabel\":\"trim_and_combine (ERR044595)\",\"q1Label\":\"trim_and_combine\
  \ (ERR064912)\",\"q2Label\":\"trim_and_combine (ERR064912)\",\"q3Label\":\"trim_and_combine\
  \ (ERR064912)\"},\"timeUsage\":{\"mean\":0.15,\"min\":0.15,\"q1\":0.15,\"q2\":0.15,\"\
  q3\":0.15,\"max\":0.15,\"minLabel\":\"trim_and_combine (ERR064912)\",\"maxLabel\"\
  :\"trim_and_combine (ERR044595)\",\"q1Label\":\"trim_and_combine (ERR064912)\",\"\
  q2Label\":\"trim_and_combine (ERR064912)\",\"q3Label\":\"trim_and_combine (ERR064912)\"\
  },\"vmem\":{\"mean\":180457472,\"min\":180457472,\"q1\":180457472,\"q2\":180457472,\"\
  q3\":180457472,\"max\":180457472,\"minLabel\":\"trim_and_combine (ERR064912)\",\"\
  maxLabel\":\"trim_and_combine (ERR044595)\",\"q1Label\":\"trim_and_combine (ERR064912)\"\
  ,\"q2Label\":\"trim_and_combine (ERR064912)\",\"q3Label\":\"trim_and_combine (ERR064912)\"\
  },\"reads\":{\"mean\":1051688865,\"min\":1042766207,\"q1\":1047227536,\"q2\":1051688865,\"\
  q3\":1056150194,\"max\":1060611523,\"minLabel\":\"trim_and_combine (ERR064912)\"\
  ,\"maxLabel\":\"trim_and_combine (ERR044595)\",\"q1Label\":\"trim_and_combine (ERR064912)\"\
  ,\"q2Label\":\"trim_and_combine (ERR064912)\",\"q3Label\":\"trim_and_combine (ERR064912)\"\
  },\"cpu\":{\"mean\":118.65,\"min\":118.1,\"q1\":118.38,\"q2\":118.65,\"q3\":118.93,\"\
  max\":119.2,\"minLabel\":\"trim_and_combine (ERR044595)\",\"maxLabel\":\"trim_and_combine\
  \ (ERR064912)\",\"q1Label\":\"trim_and_combine (ERR044595)\",\"q2Label\":\"trim_and_combine\
  \ (ERR044595)\",\"q3Label\":\"trim_and_combine (ERR044595)\"},\"time\":{\"mean\"\
  :43293,\"min\":42186,\"q1\":42739.5,\"q2\":43293,\"q3\":43846.5,\"max\":44400,\"\
  minLabel\":\"trim_and_combine (ERR064912)\",\"maxLabel\":\"trim_and_combine (ERR044595)\"\
  ,\"q1Label\":\"trim_and_combine (ERR064912)\",\"q2Label\":\"trim_and_combine (ERR064912)\"\
  ,\"q3Label\":\"trim_and_combine (ERR064912)\"},\"writes\":{\"mean\":1053056433,\"\
  min\":1044141146,\"q1\":1048598789.5,\"q2\":1053056433,\"q3\":1057514076.5,\"max\"\
  :1061971720,\"minLabel\":\"trim_and_combine (ERR064912)\",\"maxLabel\":\"trim_and_combine\
  \ (ERR044595)\",\"q1Label\":\"trim_and_combine (ERR064912)\",\"q2Label\":\"trim_and_combine\
  \ (ERR064912)\",\"q3Label\":\"trim_and_combine (ERR064912)\"}},{\"cpuUsage\":{\"\
  mean\":33.2,\"min\":33.2,\"q1\":33.2,\"q2\":33.2,\"q3\":33.2,\"max\":33.2,\"minLabel\"\
  :\"output_documentation\",\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\"\
  ,\"q2Label\":\"output_documentation\",\"q3Label\":\"output_documentation\"},\"process\"\
  :\"output_documentation\",\"mem\":{\"mean\":17190912,\"min\":17190912,\"q1\":17190912,\"\
  q2\":17190912,\"q3\":17190912,\"max\":17190912,\"minLabel\":\"output_documentation\"\
  ,\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\"\
  :\"output_documentation\",\"q3Label\":\"output_documentation\"},\"memUsage\":{\"\
  mean\":0.27,\"min\":0.27,\"q1\":0.27,\"q2\":0.27,\"q3\":0.27,\"max\":0.27,\"minLabel\"\
  :\"output_documentation\",\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\"\
  ,\"q2Label\":\"output_documentation\",\"q3Label\":\"output_documentation\"},\"timeUsage\"\
  :{\"mean\":0.01,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"q3\":0.01,\"max\":0.01,\"\
  minLabel\":\"output_documentation\",\"maxLabel\":\"output_documentation\",\"q1Label\"\
  :\"output_documentation\",\"q2Label\":\"output_documentation\",\"q3Label\":\"output_documentation\"\
  },\"vmem\":{\"mean\":23207936,\"min\":23207936,\"q1\":23207936,\"q2\":23207936,\"\
  q3\":23207936,\"max\":23207936,\"minLabel\":\"output_documentation\",\"maxLabel\"\
  :\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\":\"output_documentation\"\
  ,\"q3Label\":\"output_documentation\"},\"reads\":{\"mean\":3950685,\"min\":3950685,\"\
  q1\":3950685,\"q2\":3950685,\"q3\":3950685,\"max\":3950685,\"minLabel\":\"output_documentation\"\
  ,\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\",\"q2Label\"\
  :\"output_documentation\",\"q3Label\":\"output_documentation\"},\"cpu\":{\"mean\"\
  :66.4,\"min\":66.4,\"q1\":66.4,\"q2\":66.4,\"q3\":66.4,\"max\":66.4,\"minLabel\"\
  :\"output_documentation\",\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\"\
  ,\"q2Label\":\"output_documentation\",\"q3Label\":\"output_documentation\"},\"time\"\
  :{\"mean\":535,\"min\":535,\"q1\":535,\"q2\":535,\"q3\":535,\"max\":535,\"minLabel\"\
  :\"output_documentation\",\"maxLabel\":\"output_documentation\",\"q1Label\":\"output_documentation\"\
  ,\"q2Label\":\"output_documentation\",\"q3Label\":\"output_documentation\"},\"writes\"\
  :{\"mean\":2140738,\"min\":2140738,\"q1\":2140738,\"q2\":2140738,\"q3\":2140738,\"\
  max\":2140738,\"minLabel\":\"output_documentation\",\"maxLabel\":\"output_documentation\"\
  ,\"q1Label\":\"output_documentation\",\"q2Label\":\"output_documentation\",\"q3Label\"\
  :\"output_documentation\"}},{\"cpuUsage\":{\"mean\":58.2,\"min\":56.5,\"q1\":57.35,\"\
  q2\":58.2,\"q3\":59.05,\"max\":59.9,\"minLabel\":\"fastqc (ERR064912)\",\"maxLabel\"\
  :\"fastqc (ERR044595)\",\"q1Label\":\"fastqc (ERR064912)\",\"q2Label\":\"fastqc\
  \ (ERR064912)\",\"q3Label\":\"fastqc (ERR064912)\"},\"process\":\"fastqc\",\"mem\"\
  :{\"mean\":473122816,\"min\":469143552,\"q1\":471133184,\"q2\":473122816,\"q3\"\
  :475112448,\"max\":477102080,\"minLabel\":\"fastqc (ERR064912)\",\"maxLabel\":\"\
  fastqc (ERR044595)\",\"q1Label\":\"fastqc (ERR064912)\",\"q2Label\":\"fastqc (ERR064912)\"\
  ,\"q3Label\":\"fastqc (ERR064912)\"},\"memUsage\":{\"mean\":44.06,\"min\":43.69,\"\
  q1\":43.88,\"q2\":44.06,\"q3\":44.25,\"max\":44.43,\"minLabel\":\"fastqc (ERR064912)\"\
  ,\"maxLabel\":\"fastqc (ERR044595)\",\"q1Label\":\"fastqc (ERR064912)\",\"q2Label\"\
  :\"fastqc (ERR064912)\",\"q3Label\":\"fastqc (ERR064912)\"},\"timeUsage\":{\"mean\"\
  :0.44,\"min\":0.44,\"q1\":0.44,\"q2\":0.44,\"q3\":0.44,\"max\":0.44,\"minLabel\"\
  :\"fastqc (ERR044595)\",\"maxLabel\":\"fastqc (ERR064912)\",\"q1Label\":\"fastqc\
  \ (ERR044595)\",\"q2Label\":\"fastqc (ERR044595)\",\"q3Label\":\"fastqc (ERR044595)\"\
  },\"vmem\":{\"mean\":3524079616,\"min\":3524079616,\"q1\":3524079616,\"q2\":3524079616,\"\
  q3\":3524079616,\"max\":3524079616,\"minLabel\":\"fastqc (ERR064912)\",\"maxLabel\"\
  :\"fastqc (ERR044595)\",\"q1Label\":\"fastqc (ERR064912)\",\"q2Label\":\"fastqc\
  \ (ERR064912)\",\"q3Label\":\"fastqc (ERR064912)\"},\"reads\":{\"mean\":131953958,\"\
  min\":128331158,\"q1\":130142558,\"q2\":131953958,\"q3\":133765358,\"max\":135576758,\"\
  minLabel\":\"fastqc (ERR064912)\",\"maxLabel\":\"fastqc (ERR044595)\",\"q1Label\"\
  :\"fastqc (ERR064912)\",\"q2Label\":\"fastqc (ERR064912)\",\"q3Label\":\"fastqc\
  \ (ERR064912)\"},\"cpu\":{\"mean\":116.4,\"min\":113,\"q1\":114.7,\"q2\":116.4,\"\
  q3\":118.1,\"max\":119.8,\"minLabel\":\"fastqc (ERR064912)\",\"maxLabel\":\"fastqc\
  \ (ERR044595)\",\"q1Label\":\"fastqc (ERR064912)\",\"q2Label\":\"fastqc (ERR064912)\"\
  ,\"q3Label\":\"fastqc (ERR064912)\"},\"time\":{\"mean\":15902,\"min\":15799,\"q1\"\
  :15850.5,\"q2\":15902,\"q3\":15953.5,\"max\":16005,\"minLabel\":\"fastqc (ERR044595)\"\
  ,\"maxLabel\":\"fastqc (ERR064912)\",\"q1Label\":\"fastqc (ERR044595)\",\"q2Label\"\
  :\"fastqc (ERR044595)\",\"q3Label\":\"fastqc (ERR044595)\"},\"writes\":{\"mean\"\
  :4463269,\"min\":4429299,\"q1\":4446284,\"q2\":4463269,\"q3\":4480254,\"max\":4497239,\"\
  minLabel\":\"fastqc (ERR064912)\",\"maxLabel\":\"fastqc (ERR044595)\",\"q1Label\"\
  :\"fastqc (ERR064912)\",\"q2Label\":\"fastqc (ERR064912)\",\"q3Label\":\"fastqc\
  \ (ERR064912)\"}},{\"cpuUsage\":{\"mean\":66.95,\"min\":66.95,\"q1\":66.95,\"q2\"\
  :66.95,\"q3\":66.95,\"max\":66.95,\"minLabel\":\"unicycler (ERR044595)\",\"maxLabel\"\
  :\"unicycler (ERR044595)\",\"q1Label\":\"unicycler (ERR044595)\",\"q2Label\":\"\
  unicycler (ERR044595)\",\"q3Label\":\"unicycler (ERR044595)\"},\"process\":\"unicycler\"\
  ,\"mem\":{\"mean\":1696669696,\"min\":1696669696,\"q1\":1696669696,\"q2\":1696669696,\"\
  q3\":1696669696,\"max\":1696669696,\"minLabel\":\"unicycler (ERR044595)\",\"maxLabel\"\
  :\"unicycler (ERR044595)\",\"q1Label\":\"unicycler (ERR044595)\",\"q2Label\":\"\
  unicycler (ERR044595)\",\"q3Label\":\"unicycler (ERR044595)\"},\"memUsage\":{\"\
  mean\":26.34,\"min\":26.34,\"q1\":26.34,\"q2\":26.34,\"q3\":26.34,\"max\":26.34,\"\
  minLabel\":\"unicycler (ERR044595)\",\"maxLabel\":\"unicycler (ERR044595)\",\"q1Label\"\
  :\"unicycler (ERR044595)\",\"q2Label\":\"unicycler (ERR044595)\",\"q3Label\":\"\
  unicycler (ERR044595)\"},\"timeUsage\":{\"mean\":0.28,\"min\":0.28,\"q1\":0.28,\"\
  q2\":0.28,\"q3\":0.28,\"max\":0.28,\"minLabel\":\"unicycler (ERR044595)\",\"maxLabel\"\
  :\"unicycler (ERR044595)\",\"q1Label\":\"unicycler (ERR044595)\",\"q2Label\":\"\
  unicycler (ERR044595)\",\"q3Label\":\"unicycler (ERR044595)\"},\"vmem\":{\"mean\"\
  :1885016064,\"min\":1885016064,\"q1\":1885016064,\"q2\":1885016064,\"q3\":1885016064,\"\
  max\":1885016064,\"minLabel\":\"unicycler (ERR044595)\",\"maxLabel\":\"unicycler\
  \ (ERR044595)\",\"q1Label\":\"unicycler (ERR044595)\",\"q2Label\":\"unicycler (ERR044595)\"\
  ,\"q3Label\":\"unicycler (ERR044595)\"},\"reads\":{\"mean\":3858385114,\"min\":3858385114,\"\
  q1\":3858385114,\"q2\":3858385114,\"q3\":3858385114,\"max\":3858385114,\"minLabel\"\
  :\"unicycler (ERR044595)\",\"maxLabel\":\"unicycler (ERR044595)\",\"q1Label\":\"\
  unicycler (ERR044595)\",\"q2Label\":\"unicycler (ERR044595)\",\"q3Label\":\"unicycler\
  \ (ERR044595)\"},\"cpu\":{\"mean\":133.9,\"min\":133.9,\"q1\":133.9,\"q2\":133.9,\"\
  q3\":133.9,\"max\":133.9,\"minLabel\":\"unicycler (ERR044595)\",\"maxLabel\":\"\
  unicycler (ERR044595)\",\"q1Label\":\"unicycler (ERR044595)\",\"q2Label\":\"unicycler\
  \ (ERR044595)\",\"q3Label\":\"unicycler (ERR044595)\"},\"time\":{\"mean\":488000,\"\
  min\":488000,\"q1\":488000,\"q2\":488000,\"q3\":488000,\"max\":488000,\"minLabel\"\
  :\"unicycler (ERR044595)\",\"maxLabel\":\"unicycler (ERR044595)\",\"q1Label\":\"\
  unicycler (ERR044595)\",\"q2Label\":\"unicycler (ERR044595)\",\"q3Label\":\"unicycler\
  \ (ERR044595)\"},\"writes\":{\"mean\":10180917371,\"min\":10180917371,\"q1\":10180917371,\"\
  q2\":10180917371,\"q3\":10180917371,\"max\":10180917371,\"minLabel\":\"unicycler\
  \ (ERR044595)\",\"maxLabel\":\"unicycler (ERR044595)\",\"q1Label\":\"unicycler (ERR044595)\"\
  ,\"q2Label\":\"unicycler (ERR044595)\",\"q3Label\":\"unicycler (ERR044595)\"}}]\n\
  Feb-20 04:48:06.560 [main] DEBUG nextflow.trace.TimelineObserver - Workflow completed\
  \ -- rendering execution timeline\nFeb-20 04:48:06.806 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 04:48:06.835 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/bacass%2Fw7z07cw6%2Flogs.tar.xz">0.744GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/bacass%2Fw7z07cw6%2Foutput.tar.xz">0.219GiB</a>
						</td>
						<td>1.516GiB</td>
						<td>1452.1 seconds</td>
						<td>948.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743146352">
								<a href="https://nf-co.re/nanoseq">nanoseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/nanoseq/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>835 days</td>
						<td>2023-02-19 03:31:11</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nanoseq%2Fuc4d4boq%2Flogs.tar.xz">0.224GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nanoseq%2Fuc4d4boq%2Foutput.tar.xz">0.006GiB</a>
						</td>
						<td>1.701GiB</td>
						<td>1326.3 seconds</td>
						<td>1341.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743136512">
								<a href="https://nf-co.re/mag">mag</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mag/tree/1.1.1">1.1.1</a>
						</td>
						<td>nextflow</td>
						<td>831 days</td>
						<td>2023-02-20 04:59:47</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name nfcore-magbusco-1.1.1.img.pulling.1676869200799\
  \ docker://nfcore/magbusco:1.1.1 > /dev/null\n  status : 255\n  message:\n    INFO:\
  \    Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:660636013d5405a0c93da103e2b136f042d429784bafcfe93e5ec537c62816b9\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:5019cfb0a680b922320d12e58ed76f322322992137372cf22d054279611b9696\n\
  \    Copying blob sha256:45a430dc1b0a097cc48e254a199fbd84bc34640923d64c47cffceb295fead8c4\n\
  \    Copying blob sha256:f24aefa408f08183b4868f2daf60b1eafc9536294461d973b41cdf931447f8a5\n\
  \    Copying blob sha256:e0cbfafe1448cfd9a602ced32812facd6f25b2db97263a5f4200cff68c8d6afc\n\
  \    Copying blob sha256:e826fc7540e7cc03ed61714c23fa70a52892be759ffff6ab5201a8c4680b1bcc\n\
  \    Copying blob sha256:ca6a4d707dd7e8b86a8aad8fd54c59b07c2e02128f8c58005cf4e94a634b0f57\n\
  \    Copying blob sha256:17a2b2fdc38c35d87d19689c1a96576e39880104549ea56c1518657a68187cc5\n\
  \    Copying config sha256:3a903d9b5a53c3cc671477d3deb9bf8275c7fb4f623fe1bc8dd194959ba93fe5\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.299e30469883b1da75b9ff0691d9eaeeb30439a73d3ddc4a5345f9a359e2a998\"\n\n\t\
  at nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\tat nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\
  \tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\tat nextflow.processor.TaskRun$isContainerEnabled$2.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name nfcore-magbusco-1.1.1.img.pulling.1676869200799\
  \ docker://nfcore/magbusco:1.1.1 > /dev/null\n  status : 255\n  message:\n    INFO:\
  \    Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:660636013d5405a0c93da103e2b136f042d429784bafcfe93e5ec537c62816b9\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:5019cfb0a680b922320d12e58ed76f322322992137372cf22d054279611b9696\n\
  \    Copying blob sha256:45a430dc1b0a097cc48e254a199fbd84bc34640923d64c47cffceb295fead8c4\n\
  \    Copying blob sha256:f24aefa408f08183b4868f2daf60b1eafc9536294461d973b41cdf931447f8a5\n\
  \    Copying blob sha256:e0cbfafe1448cfd9a602ced32812facd6f25b2db97263a5f4200cff68c8d6afc\n\
  \    Copying blob sha256:e826fc7540e7cc03ed61714c23fa70a52892be759ffff6ab5201a8c4680b1bcc\n\
  \    Copying blob sha256:ca6a4d707dd7e8b86a8aad8fd54c59b07c2e02128f8c58005cf4e94a634b0f57\n\
  \    Copying blob sha256:17a2b2fdc38c35d87d19689c1a96576e39880104549ea56c1518657a68187cc5\n\
  \    Copying config sha256:3a903d9b5a53c3cc671477d3deb9bf8275c7fb4f623fe1bc8dd194959ba93fe5\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.299e30469883b1da75b9ff0691d9eaeeb30439a73d3ddc4a5345f9a359e2a998\"\n\n\t\
  at nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 05:00:49.258 [Actor Thread 4] DEBUG nextflow.Session\
  \ - Session aborted -- Cause: java.lang.IllegalStateException: Failed to pull singularity\
  \ image\n  command: singularity pull  --name nfcore-magbusco-1.1.1.img.pulling.1676869200799\
  \ docker://nfcore/magbusco:1.1.1 > /dev/null\n  status : 255\n  message:\n    INFO:\
  \    Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:660636013d5405a0c93da103e2b136f042d429784bafcfe93e5ec537c62816b9\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:5019cfb0a680b922320d12e58ed76f322322992137372cf22d054279611b9696\n\
  \    Copying blob sha256:45a430dc1b0a097cc48e254a199fbd84bc34640923d64c47cffceb295fead8c4\n\
  \    Copying blob sha256:f24aefa408f08183b4868f2daf60b1eafc9536294461d973b41cdf931447f8a5\n\
  \    Copying blob sha256:e0cbfafe1448cfd9a602ced32812facd6f25b2db97263a5f4200cff68c8d6afc\n\
  \    Copying blob sha256:e826fc7540e7cc03ed61714c23fa70a52892be759ffff6ab5201a8c4680b1bcc\n\
  \    Copying blob sha256:ca6a4d707dd7e8b86a8aad8fd54c59b07c2e02128f8c58005cf4e94a634b0f57\n\
  \    Copying blob sha256:17a2b2fdc38c35d87d19689c1a96576e39880104549ea56c1518657a68187cc5\n\
  \    Copying config sha256:3a903d9b5a53c3cc671477d3deb9bf8275c7fb4f623fe1bc8dd194959ba93fe5\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.299e30469883b1da75b9ff0691d9eaeeb30439a73d3ddc4a5345f9a359e2a998\"\n\nFeb-20\
  \ 05:00:49.325 [Actor Thread 4] DEBUG nextflow.Session - The following nodes are\
  \ still active:\n[process] get_busco_version\n  status=ACTIVE\n  port 0: (cntrl)\
  \ -     ; channel: $\n\n[process] get_software_versions\n  status=ACTIVE\n  port\
  \ 0: (value) OPEN  ; channel: busco_version\n  port 1: (cntrl) -     ; channel:\
  \ $\n\n[process] fastqc_raw\n  status=ACTIVE\n  port 0: (queue) closed; channel:\
  \ -\n  port 1: (cntrl) -     ; channel: $\n\n[process] fastp\n  status=ACTIVE\n\
  \  port 0: (queue) closed; channel: -\n  port 1: (value) bound ; channel: adapter\n\
  \  port 2: (value) bound ; channel: adapter_reverse\n  port 3: (value) bound ; channel:\
  \ qual\n  port 4: (value) bound ; channel: trim_qual\n  port 5: (cntrl) -     ;\
  \ channel: $\n\n[process] phix_download_db\n  status=ACTIVE\n  port 0: (queue) closed;\
  \ channel: genome\n  port 1: (cntrl) -     ; channel: $\n\n[process] remove_phix\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] fastqc_trimmed\n  status=ACTIVE\n  port 0: (queue) OPEN\
  \  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] filtlong\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] nanoplot\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] centrifuge_db_preparation\n  status=ACTIVE\n\
  \  port 0: (queue) closed; channel: db\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] centrifuge\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] kraken2_db_preparation\n  status=ACTIVE\n\
  \  port 0: (queue) closed; channel: db\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] kraken2\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] krona\n  status=ACTIVE\n  port 0: (queue)\
  \ OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] megahit\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] spadeshybrid\n  status=ACTIVE\n  port 0: (queue) OPEN\
  \  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] spades\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] quast\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1:\
  \ (cntrl) -     ; channel: $\n\n[process] bowtie2\n  status=ACTIVE\n  port 0: (queue)\
  \ OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] metabat\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound\
  \ ; channel: min_size\n  port 2: (value) bound ; channel: max_unbinned\n  port 3:\
  \ (value) bound ; channel: min_length_unbinned\n  port 4: (cntrl) -     ; channel:\
  \ $\n\n[process] busco_db_preparation\n  status=ACTIVE\n  port 0: (queue) closed;\
  \ channel: database\n  port 1: (cntrl) -     ; channel: $\n\n[process] busco\n \
  \ status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ;\
  \ channel: $\n\n[process] busco_plot\n  status=ACTIVE\n  port 0: (queue) OPEN  ;\
  \ channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] busco_summary\n\
  \  status=ACTIVE\n  port 0: (value) OPEN  ; channel: short_summary.*.txt\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] quast_bins\n  status=ACTIVE\n  port\
  \ 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process]\
  \ merge_quast_and_busco\n  status=ACTIVE\n  port 0: (value) OPEN  ; channel: quast_bin_sum\n\
  \  port 1: (value) OPEN  ; channel: busco_sum\n  port 2: (cntrl) -     ; channel:\
  \ $\n\n[process] cat\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] multiqc\n  status=ACTIVE\n  port 0:\
  \ (value) bound ; channel: multiqc_config\n  port 1: (value) bound ; channel: mqc_custom_config\n\
  \  port 2: (value) OPEN  ; channel: fastqc_raw\n  port 3: (value) OPEN  ; channel:\
  \ fastqc_trimmed\n  port 4: (value) bound ; channel: host_removal\n  port 5: (value)\
  \ OPEN  ; channel: quast*/*\n  port 6: (value) OPEN  ; channel: short_summary\n\
  \  port 7: (value) OPEN  ; channel: software_versions/*\n  port 8: (queue) OPEN\
  \  ; channel: workflow_summary\n  port 9: (cntrl) -     ; channel: $\n\n[process]\
  \ output_documentation\n  status=ACTIVE\n  port 0: (value) bound ; channel: output_docs\n\
  \  port 1: (value) bound ; channel: images\n  port 2: (cntrl) -     ; channel: $\n\
  \nFeb-20 05:00:49.346 [main] DEBUG nextflow.Session - Session await > all processes\
  \ finished\nFeb-20 05:00:49.346 [main] DEBUG nextflow.Session - Session await >\
  \ all barriers passed\nFeb-20 05:00:49.613 [main] INFO  nextflow.Nextflow - -\e\
  [0;35m[nf-core/mag]\e[0;31m Pipeline completed with errors\e[0m-\nFeb-20 05:00:49.617\
  \ [main] DEBUG nextflow.trace.WorkflowStatsObserver - Workflow completed > WorkflowStats[succeededCount=0;\
  \ failedCount=0; ignoredCount=0; cachedCount=0; pendingCount=0; submittedCount=0;\
  \ runningCount=0; retriesCount=0; abortedCount=0; succeedDuration=0ms; failedDuration=0ms;\
  \ cachedDuration=0ms;loadCpus=0; loadMemory=0; peakRunning=0; peakCpus=0; peakMemory=0;\
  \ ]\nFeb-20 05:00:49.617 [main] DEBUG nextflow.trace.TraceFileObserver - Workflow\
  \ completed -- saving trace file\nFeb-20 05:00:49.623 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Workflow completed -- rendering execution report\nFeb-20 05:00:49.635 [main]\
  \ DEBUG nextflow.trace.ReportObserver - Execution report summary data:\n  []\nFeb-20\
  \ 05:00:50.308 [main] DEBUG nextflow.trace.TimelineObserver - Workflow completed\
  \ -- rendering execution timeline\nFeb-20 05:00:50.506 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 05:00:50.543 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2F4k9o44av%2Flogs.tar.xz">0.016GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2F4k9o44av%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.815GiB</td>
						<td>47.6 seconds</td>
						<td>63.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743186624">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/2.0">2.0</a>
						</td>
						<td>nextflow</td>
						<td>828 days</td>
						<td>2023-02-19 04:44:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: Include statement is not allowed within a workflow definition
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.script.BaseScript.include(BaseScript.groovy:150)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)\n\
  \tat groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:424)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat Script_63c2f779$_runScript_closure1$_closure2.doCall(Script_63c2f779:57)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:205)\n\
  \tat nextflow.script.WorkflowDef.run(WorkflowDef.groovy:189)\n\tat nextflow.script.BindableDef.invoke_a(BindableDef.groovy:52)\n\
  \tat nextflow.script.ChainableDef$invoke_a.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:208)\n\tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\
  \tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F2f76eha6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F2f76eha6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.481GiB</td>
						<td>20.5 seconds</td>
						<td>15.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743053088">
								<a href="https://nf-co.re/diaproteomics">diaproteomics</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/diaproteomics/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>828 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2Fb6q4qzza%2Flogs.tar.xz">0.123GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2Fb6q4qzza%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.824GiB</td>
						<td>1081.7 seconds</td>
						<td>969.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744033968">
								<a href="https://nf-co.re/cageseq">cageseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/cageseq/tree/1.0.1">1.0.1</a>
						</td>
						<td>nextflow</td>
						<td>818 days</td>
						<td>2023-02-19 06:52:11</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cageseq%2Fwtdww6ut%2Flogs.tar.xz">0.127GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cageseq%2Fwtdww6ut%2Foutput.tar.xz">0.005GiB</a>
						</td>
						<td>1.856GiB</td>
						<td>552.9 seconds</td>
						<td>564.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743119824">
								<a href="https://nf-co.re/mag">mag</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mag/tree/1.1.2">1.1.2</a>
						</td>
						<td>nextflow</td>
						<td>817 days</td>
						<td>2023-02-20 05:30:41</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name nfcore-magbusco-1.1.2.img.pulling.1676871052305\
  \ docker://nfcore/magbusco:1.1.2 > /dev/null\n  status : 255\n  message:\n    INFO:\
  \    Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:3bab47480a1cc14cd04a03c75c61ff04cc7583a7a62f5dbb885a6797045deaf8\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:4f6cc21320c8c8b5607cefa07c02048bf099d07ccc3748d1596e7a7ecc5569bf\n\
  \    Copying blob sha256:5858fd51e169294a90b8e2bd9ebab09150a8b367fbc112d796802f1da36d7ffa\n\
  \    Copying blob sha256:6df5a1f55177cc4d9d7c9d16a80b62c1668951ebf1ca567d0e4cd0c7efedca0c\n\
  \    Copying blob sha256:f6bd26cfda5e2c632ec708dea681651b459daab06231bd95a0a8f42ef619b718\n\
  \    Copying blob sha256:fe66bb2581d2151cfb6203ac73137c0147d37d9fbce581f164cd5a07686a6f7a\n\
  \    Copying blob sha256:dc56bba9bcf48fc03dcf808b7d0b10a266a094a2a024553a4a48b6af4bf9f4af\n\
  \    Copying blob sha256:aea059d8ecff4185a638756931c91bc42e04d057db0bc1d3d5a35ebc5b4df12a\n\
  \    Copying config sha256:0dff0ea6d30637103e8ac51b97067692875fdbb91f14d5f30fa069de98ef1639\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.20aec3ae49fe2aa9f2fff609532f9c5ab5dd927f3fe502958c0a77f397490472\"\n\n\t\
  at nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\tat nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\
  \tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\tat nextflow.processor.TaskRun$isContainerEnabled$2.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name nfcore-magbusco-1.1.2.img.pulling.1676871052305\
  \ docker://nfcore/magbusco:1.1.2 > /dev/null\n  status : 255\n  message:\n    INFO:\
  \    Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:3bab47480a1cc14cd04a03c75c61ff04cc7583a7a62f5dbb885a6797045deaf8\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:4f6cc21320c8c8b5607cefa07c02048bf099d07ccc3748d1596e7a7ecc5569bf\n\
  \    Copying blob sha256:5858fd51e169294a90b8e2bd9ebab09150a8b367fbc112d796802f1da36d7ffa\n\
  \    Copying blob sha256:6df5a1f55177cc4d9d7c9d16a80b62c1668951ebf1ca567d0e4cd0c7efedca0c\n\
  \    Copying blob sha256:f6bd26cfda5e2c632ec708dea681651b459daab06231bd95a0a8f42ef619b718\n\
  \    Copying blob sha256:fe66bb2581d2151cfb6203ac73137c0147d37d9fbce581f164cd5a07686a6f7a\n\
  \    Copying blob sha256:dc56bba9bcf48fc03dcf808b7d0b10a266a094a2a024553a4a48b6af4bf9f4af\n\
  \    Copying blob sha256:aea059d8ecff4185a638756931c91bc42e04d057db0bc1d3d5a35ebc5b4df12a\n\
  \    Copying config sha256:0dff0ea6d30637103e8ac51b97067692875fdbb91f14d5f30fa069de98ef1639\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.20aec3ae49fe2aa9f2fff609532f9c5ab5dd927f3fe502958c0a77f397490472\"\n\n\t\
  at nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 05:31:33.559 [Actor Thread 3] DEBUG nextflow.Session\
  \ - Session aborted -- Cause: java.lang.IllegalStateException: Failed to pull singularity\
  \ image\n  command: singularity pull  --name nfcore-magbusco-1.1.2.img.pulling.1676871052305\
  \ docker://nfcore/magbusco:1.1.2 > /dev/null\n  status : 255\n  message:\n    INFO:\
  \    Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:3bab47480a1cc14cd04a03c75c61ff04cc7583a7a62f5dbb885a6797045deaf8\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:4f6cc21320c8c8b5607cefa07c02048bf099d07ccc3748d1596e7a7ecc5569bf\n\
  \    Copying blob sha256:5858fd51e169294a90b8e2bd9ebab09150a8b367fbc112d796802f1da36d7ffa\n\
  \    Copying blob sha256:6df5a1f55177cc4d9d7c9d16a80b62c1668951ebf1ca567d0e4cd0c7efedca0c\n\
  \    Copying blob sha256:f6bd26cfda5e2c632ec708dea681651b459daab06231bd95a0a8f42ef619b718\n\
  \    Copying blob sha256:fe66bb2581d2151cfb6203ac73137c0147d37d9fbce581f164cd5a07686a6f7a\n\
  \    Copying blob sha256:dc56bba9bcf48fc03dcf808b7d0b10a266a094a2a024553a4a48b6af4bf9f4af\n\
  \    Copying blob sha256:aea059d8ecff4185a638756931c91bc42e04d057db0bc1d3d5a35ebc5b4df12a\n\
  \    Copying config sha256:0dff0ea6d30637103e8ac51b97067692875fdbb91f14d5f30fa069de98ef1639\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.20aec3ae49fe2aa9f2fff609532f9c5ab5dd927f3fe502958c0a77f397490472\"\n\nFeb-20\
  \ 05:31:33.630 [Actor Thread 3] DEBUG nextflow.Session - The following nodes are\
  \ still active:\n[process] get_busco_version\n  status=ACTIVE\n  port 0: (cntrl)\
  \ -     ; channel: $\n\n[process] get_software_versions\n  status=ACTIVE\n  port\
  \ 0: (value) OPEN  ; channel: busco_version\n  port 1: (cntrl) -     ; channel:\
  \ $\n\n[process] fastqc_raw\n  status=ACTIVE\n  port 0: (queue) closed; channel:\
  \ -\n  port 1: (cntrl) -     ; channel: $\n\n[process] fastp\n  status=ACTIVE\n\
  \  port 0: (queue) closed; channel: -\n  port 1: (value) bound ; channel: adapter\n\
  \  port 2: (value) bound ; channel: adapter_reverse\n  port 3: (value) bound ; channel:\
  \ qual\n  port 4: (value) bound ; channel: trim_qual\n  port 5: (cntrl) -     ;\
  \ channel: $\n\n[process] phix_download_db\n  status=ACTIVE\n  port 0: (queue) closed;\
  \ channel: genome\n  port 1: (cntrl) -     ; channel: $\n\n[process] remove_phix\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] fastqc_trimmed\n  status=ACTIVE\n  port 0: (queue) OPEN\
  \  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] filtlong\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] nanoplot\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] centrifuge_db_preparation\n  status=ACTIVE\n\
  \  port 0: (queue) closed; channel: db\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] centrifuge\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] kraken2_db_preparation\n  status=ACTIVE\n\
  \  port 0: (queue) closed; channel: db\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] kraken2\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] krona\n  status=ACTIVE\n  port 0: (queue)\
  \ OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] megahit\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] spadeshybrid\n  status=ACTIVE\n  port 0: (queue) OPEN\
  \  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] spades\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] quast\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1:\
  \ (cntrl) -     ; channel: $\n\n[process] bowtie2\n  status=ACTIVE\n  port 0: (queue)\
  \ OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] metabat\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound\
  \ ; channel: min_size\n  port 2: (value) bound ; channel: max_unbinned\n  port 3:\
  \ (value) bound ; channel: min_length_unbinned\n  port 4: (cntrl) -     ; channel:\
  \ $\n\n[process] busco_db_preparation\n  status=ACTIVE\n  port 0: (queue) closed;\
  \ channel: database\n  port 1: (cntrl) -     ; channel: $\n\n[process] busco\n \
  \ status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ;\
  \ channel: $\n\n[process] busco_plot\n  status=ACTIVE\n  port 0: (queue) OPEN  ;\
  \ channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] busco_summary\n\
  \  status=ACTIVE\n  port 0: (value) OPEN  ; channel: short_summary.*.txt\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] quast_bins\n  status=ACTIVE\n  port\
  \ 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process]\
  \ merge_quast_and_busco\n  status=ACTIVE\n  port 0: (value) OPEN  ; channel: quast_bin_sum\n\
  \  port 1: (value) OPEN  ; channel: busco_sum\n  port 2: (cntrl) -     ; channel:\
  \ $\n\n[process] cat\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] multiqc\n  status=ACTIVE\n  port 0:\
  \ (value) bound ; channel: multiqc_config\n  port 1: (value) bound ; channel: mqc_custom_config\n\
  \  port 2: (value) OPEN  ; channel: fastqc_raw\n  port 3: (value) OPEN  ; channel:\
  \ fastqc_trimmed\n  port 4: (value) bound ; channel: host_removal\n  port 5: (value)\
  \ OPEN  ; channel: quast*/*\n  port 6: (value) OPEN  ; channel: short_summary\n\
  \  port 7: (value) OPEN  ; channel: software_versions/*\n  port 8: (queue) OPEN\
  \  ; channel: workflow_summary\n  port 9: (cntrl) -     ; channel: $\n\n[process]\
  \ output_documentation\n  status=ACTIVE\n  port 0: (value) bound ; channel: output_docs\n\
  \  port 1: (value) bound ; channel: images\n  port 2: (cntrl) -     ; channel: $\n\
  \nFeb-20 05:31:33.636 [main] DEBUG nextflow.Session - Session await > all processes\
  \ finished\nFeb-20 05:31:33.636 [main] DEBUG nextflow.Session - Session await >\
  \ all barriers passed\nFeb-20 05:31:33.953 [main] INFO  nextflow.Nextflow - -\e\
  [0;35m[nf-core/mag]\e[0;31m Pipeline completed with errors\e[0m-\nFeb-20 05:31:33.965\
  \ [main] DEBUG nextflow.trace.WorkflowStatsObserver - Workflow completed > WorkflowStats[succeededCount=0;\
  \ failedCount=0; ignoredCount=0; cachedCount=0; pendingCount=0; submittedCount=0;\
  \ runningCount=0; retriesCount=0; abortedCount=0; succeedDuration=0ms; failedDuration=0ms;\
  \ cachedDuration=0ms;loadCpus=0; loadMemory=0; peakRunning=0; peakCpus=0; peakMemory=0;\
  \ ]\nFeb-20 05:31:33.965 [main] DEBUG nextflow.trace.TraceFileObserver - Workflow\
  \ completed -- saving trace file\nFeb-20 05:31:33.970 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Workflow completed -- rendering execution report\nFeb-20 05:31:33.978 [main]\
  \ DEBUG nextflow.trace.ReportObserver - Execution report summary data:\n  []\nFeb-20\
  \ 05:31:34.690 [main] DEBUG nextflow.trace.TimelineObserver - Workflow completed\
  \ -- rendering execution timeline\nFeb-20 05:31:34.918 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 05:31:34.960 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2F93xj59tm%2Flogs.tar.xz">0.016GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2F93xj59tm%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.815GiB</td>
						<td>46.9 seconds</td>
						<td>53.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743052848">
								<a href="https://nf-co.re/diaproteomics">diaproteomics</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/diaproteomics/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>808 days</td>
						<td>2023-02-19 16:54:55</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2F5dg5fhes%2Flogs.tar.xz">0.123GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2F5dg5fhes%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.848GiB</td>
						<td>1112.2 seconds</td>
						<td>612.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743106384">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.2.2">2.2.2</a>
						</td>
						<td>nextflow</td>
						<td>802 days</td>
						<td>2023-02-19 04:23:49</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 04:24:04.725
kind: NextflowMiscError
process: main
rest: '[nf-core/eager] error: --input was not supplied! Please check ''--help'' or
  documentation under ''running the pipeline'' for details'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F9nkkdum3%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F9nkkdum3%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.747GiB</td>
						<td>31.2 seconds</td>
						<td>14.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743186384">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.0">3.0</a>
						</td>
						<td>nextflow</td>
						<td>795 days</td>
						<td>2023-02-19 03:31:17</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: Include statement is not allowed within a workflow definition
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.script.BaseScript.include(BaseScript.groovy:150)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)\n\
  \tat groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:424)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat Script_abd36f0c$_runScript_closure1$_closure2.doCall(Script_abd36f0c:78)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:205)\n\
  \tat nextflow.script.WorkflowDef.run(WorkflowDef.groovy:189)\n\tat nextflow.script.BindableDef.invoke_a(BindableDef.groovy:52)\n\
  \tat nextflow.script.ChainableDef$invoke_a.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:208)\n\tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\
  \tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fu2ng9ga1%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fu2ng9ga1%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.466GiB</td>
						<td>22.5 seconds</td>
						<td>30.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743052608">
								<a href="https://nf-co.re/diaproteomics">diaproteomics</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/diaproteomics/tree/1.2.0">1.2.0</a>
						</td>
						<td>nextflow</td>
						<td>795 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2Fnpgb7bcm%2Flogs.tar.xz">0.123GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2Fnpgb7bcm%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.878GiB</td>
						<td>1130.0 seconds</td>
						<td>699.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743106144">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.3.0">2.3.0</a>
						</td>
						<td>nextflow</td>
						<td>769 days</td>
						<td>2023-02-19 03:42:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 03:42:36.552
kind: NextflowMiscError
process: main
rest: '[nf-core/eager] error: --input was not supplied! Please check ''--help'' or
  documentation under ''running the pipeline'' for details'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fhuhgul16%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fhuhgul16%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.769GiB</td>
						<td>33.2 seconds</td>
						<td>15.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743965792">
								<a href="https://nf-co.re/cageseq">cageseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/cageseq/tree/1.0.2">1.0.2</a>
						</td>
						<td>nextflow</td>
						<td>766 days</td>
						<td>2023-02-19 03:32:20</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cageseq%2Fjkt5x64z%2Flogs.tar.xz">0.127GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cageseq%2Fjkt5x64z%2Foutput.tar.xz">0.005GiB</a>
						</td>
						<td>1.855GiB</td>
						<td>514.2 seconds</td>
						<td>576.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743105904">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.3.1">2.3.1</a>
						</td>
						<td>nextflow</td>
						<td>766 days</td>
						<td>2023-02-19 04:07:19</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 04:07:42.480
kind: NextflowMiscError
process: main
rest: '[nf-core/eager] error: --input was not supplied! Please check ''--help'' or
  documentation under ''running the pipeline'' for details'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F365jodze%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F365jodze%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.607GiB</td>
						<td>34.8 seconds</td>
						<td>23.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738323696">
								<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling">snakemake-workflows/dna-seq-gatk-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.0.1">v2.0.1</a>
						</td>
						<td>snakemake</td>
						<td>766 days</td>
						<td>2023-02-19 04:15:39</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeInternalError
kind: SnakemakeInternalError
rest: "File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py\"\
  , line 747, in snakemake\n    success = workflow.execute(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py\"\
  , line 926, in execute\n    dag.create_conda_envs(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/dag.py\"\
  , line 338, in create_conda_envs\n    env.create(dryrun)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 397, in create\n    if self.pin_file:\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 109, in pin_file\n    f\".{self.conda.platform}.pin.txt\"\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 102, in conda\n    return Conda(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 667, in __init__\n    shell.check_output(self._get_cmd(f\"conda info --json\"\
  ), text=True)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/shell.py\"\
  , line 63, in check_output\n    return sp.check_output(cmd, shell=True, executable=executable,\
  \ **kwargs)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 421, in check_output\n    return run(*popenargs, stdout=PIPE, timeout=timeout,\
  \ check=True,\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 526, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError:\
  \ Command ' singularity --quiet --silent exec --home /home/azureuser/tmp/tcjm9hkevn/code/.test\
  \  continuumio/miniconda3:4.8.2 sh -c 'conda info --json'' returned non-zero exit\
  \ status 255."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2F8z204t9v%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2F8z204t9v%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.106GiB</td>
						<td>1.9 seconds</td>
						<td>2.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738289728">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.0.1">v2.0.1</a>
						</td>
						<td>snakemake</td>
						<td>766 days</td>
						<td>2023-02-19 04:03:05</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/aonnhkidz3/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/c7b2d82f-2bce-4752-b082-43464e2719b2\
  \ singularity  exec --home /home/azureuser/tmp/aonnhkidz3/code/.test  /home/azureuser/tmp/aonnhkidz3/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/aonnhkidz3/code/.test/.snakemake/conda/b2baafd8b6362bf0b798adc6e4788c46_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/aonnhkidz3/code/.test/.snakemake/conda/b2baafd8b6362bf0b798adc6e4788c46_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining matplotlib=3.1:   0%|       \
  \   | 0/9 [00:00<?, ?it/s]\nExamining pandas=0.25:  11%|\u2588         | 1/9 [00:02<00:20,\
  \  2.52s/it]\nExamining pandas=0.25:  22%|\u2588\u2588\u258F       | 2/9 [00:02<00:08,\
  \  1.26s/it]\nExamining python=3.7:  22%|\u2588\u2588\u258F       | 2/9 [00:04<00:08,\
  \  1.26s/it] \nExamining python=3.7:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:04<00:09,\
  \  1.66s/it]\nExamining pillow=6.2:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:06<00:09,\
  \  1.66s/it]\nExamining pillow=6.2:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9\
  \ [00:06<00:08,  1.73s/it]\nExamining @/linux-64::__glibc==2.31=0:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:07<00:08,  1.73s/it]\nExamining @/linux-64::__glibc==2.31=0:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:07<00:05,  1.34s/it]\n\
  Examining @/linux-64::__unix==0=0:  56%|\u2588\u2588\u2588\u2588\u2588\u258C   \
  \ | 5/9 [00:07<00:05,  1.34s/it]    \nExamining @/linux-64::__archspec==1=x86_64:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:07<00:04,  1.34s/it]\n\
  Examining @/linux-64::__linux==5.15.0=0:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:07<00:02,  1.34s/it]   \nExamining goatools=0.9.7:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:07<00:01,  1.34s/it]\
  \               \n                                                             \
  \          \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for matplotlib pandas python pillow goatools:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for matplotlib pandas goatools:  11%|\u2588       \
  \  | 1/9 [00:05<00:40,  5.10s/it]      \nExamining conflict for matplotlib pandas\
  \ goatools:  22%|\u2588\u2588\u258F       | 2/9 [00:05<00:17,  2.55s/it]\nExamining\
  \ conflict for matplotlib pandas python pillow:  22%|\u2588\u2588\u258F       |\
  \ 2/9 [00:44<00:17,  2.55s/it]\nExamining conflict for matplotlib pandas python\
  \ pillow:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:44<01:47, 17.87s/it]\nExamining\
  \ conflict for matplotlib pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:48<01:47,\
  \ 17.87s/it]              \nExamining conflict for matplotlib pandas:  44%|\u2588\
  \u2588\u2588\u2588\u258D     | 4/9 [00:48<01:04, 12.82s/it]\nExamining conflict\
  \ for matplotlib pandas python pillow __glibc:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [00:52<01:04, 12.82s/it]\nExamining conflict for matplotlib pandas python\
  \ pillow __glibc:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:52<00:38,\
  \  9.54s/it]\nExamining conflict for matplotlib goatools pandas pillow:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:55<00:38,  9.54s/it]      \nExamining\
  \ conflict for matplotlib goatools pandas pillow:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:55<00:22,  7.48s/it]\nExamining conflict for matplotlib\
  \ goatools:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [01:08<00:22,\
  \  7.48s/it]              \nExamining conflict for matplotlib goatools:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [01:08<00:18,  9.19s/it]\nExamining\
  \ conflict for matplotlib pillow:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [01:12<00:18,  9.19s/it]  \nExamining conflict for matplotlib pillow:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [01:12<00:07,\
  \  7.75s/it]\nExamining conflict for matplotlib pandas python:  89%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [01:15<00:07,  7.75s/it]\nExamining\
  \ conflict for matplotlib pandas python: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588| 9/9 [01:15<00:00,  6.26s/it]\nExamining conflict for pandas\
  \ goatools: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9\
  \ [01:16<00:00,  6.26s/it]         \nExamining conflict for pandas goatools: : 10it\
  \ [01:16,  4.43s/it]                    \nExamining conflict for python pillow:\
  \ : 10it [01:17,  4.43s/it]  \nExamining conflict for python pillow: : 11it [01:17,\
  \  3.38s/it]\nExamining conflict for goatools python pillow: : 11it [01:17,  3.38s/it]\n\
  Examining conflict for goatools python pillow: : 12it [01:17,  2.52s/it]\n     \
  \                                                                   \nfailed\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining matplotlib=3.1:   0%|       \
  \   | 0/9 [00:00<?, ?it/s]\nExamining pandas=0.25:  11%|\u2588         | 1/9 [00:01<00:12,\
  \  1.58s/it]\nExamining pandas=0.25:  22%|\u2588\u2588\u258F       | 2/9 [00:01<00:05,\
  \  1.26it/s]\nExamining python=3.7:  22%|\u2588\u2588\u258F       | 2/9 [00:02<00:05,\
  \  1.26it/s] \nExamining python=3.7:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:02<00:06,\
  \  1.04s/it]\nExamining pillow=6.2:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:04<00:06,\
  \  1.04s/it]\nExamining pillow=6.2:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9\
  \ [00:04<00:05,  1.08s/it]\nExamining @/linux-64::__glibc==2.31=0:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:04<00:05,  1.08s/it]\nExamining @/linux-64::__glibc==2.31=0:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:04<00:03,  1.19it/s]\n\
  Examining @/linux-64::__unix==0=0:  56%|\u2588\u2588\u2588\u2588\u2588\u258C   \
  \ | 5/9 [00:04<00:03,  1.19it/s]    \nExamining @/linux-64::__archspec==1=x86_64:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:04<00:02,  1.19it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:04<00:01,  1.19it/s]   \nExamining goatools=0.9.7:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:04<00:00,  1.19it/s]\
  \               \n                                                             \
  \          \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for matplotlib pandas python pillow goatools:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for matplotlib pandas goatools:  11%|\u2588       \
  \  | 1/9 [00:03<00:26,  3.30s/it]      \nExamining conflict for matplotlib pandas\
  \ goatools:  22%|\u2588\u2588\u258F       | 2/9 [00:03<00:11,  1.65s/it]\nExamining\
  \ conflict for matplotlib pandas python pillow:  22%|\u2588\u2588\u258F       |\
  \ 2/9 [00:32<00:11,  1.65s/it]\nExamining conflict for matplotlib pandas python\
  \ pillow:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:32<01:17, 12.97s/it]\nExamining\
  \ conflict for matplotlib pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:36<01:17,\
  \ 12.97s/it]              \nExamining conflict for matplotlib pandas:  44%|\u2588\
  \u2588\u2588\u2588\u258D     | 4/9 [00:36<00:48,  9.71s/it]\nExamining conflict\
  \ for matplotlib pandas python pillow __glibc:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [00:38<00:48,  9.71s/it]\nExamining conflict for matplotlib pandas python\
  \ pillow __glibc:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:38<00:28,\
  \  7.14s/it]\nExamining conflict for matplotlib goatools pandas pillow:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:41<00:28,  7.14s/it]      \nExamining\
  \ conflict for matplotlib goatools pandas pillow:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:41<00:16,  5.59s/it]\nExamining conflict for matplotlib\
  \ goatools:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:49<00:16,\
  \  5.59s/it]              \nExamining conflict for matplotlib goatools:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:49<00:12,  6.39s/it]\nExamining\
  \ conflict for matplotlib pillow:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:51<00:12,  6.39s/it]  \nExamining conflict for matplotlib pillow:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:51<00:05,\
  \  5.14s/it]\nExamining conflict for matplotlib pandas python:  89%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:53<00:05,  5.14s/it]\nExamining\
  \ conflict for matplotlib pandas python: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588| 9/9 [00:53<00:00,  4.04s/it]\nExamining conflict for pandas\
  \ goatools: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9\
  \ [00:53<00:00,  4.04s/it]         \nExamining conflict for pandas goatools: : 10it\
  \ [00:53,  2.85s/it]                    \nExamining conflict for python pillow:\
  \ : 10it [00:54,  2.85s/it]  \nExamining conflict for python pillow: : 11it [00:54,\
  \  2.24s/it]\nExamining conflict for goatools python pillow: : 11it [00:54,  2.24s/it]\n\
  Examining conflict for goatools python pillow: : 12it [00:54,  1.71s/it]\n     \
  \                                                                   \nfailed\n\n\
  UnsatisfiableError: The following specifications were found to be incompatible with\
  \ each other:\n\nOutput in format: Requested package -> Available versions\n\nPackage\
  \ python conflicts for:\ngoatools=0.9.7 -> python\npandas=0.25 -> python[version='>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']\n\
  python=3.7\nmatplotlib=3.1 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  matplotlib=3.1 -> pyqt -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.6|>=3.0,<=3.9|>=3.7|>=3.8|>=3']\n\
  goatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|>=3.6|>=3|3.4.*']\n\
  pillow=6.2 -> olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  pillow=6.2 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  \nPackage _libgcc_mutex conflicts for:\npandas=0.25 -> libgcc-ng[version='>=7.3.0']\
  \ -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\nmatplotlib=3.1\
  \ -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  python=3.7 -> libgcc-ng[version='>=9.4.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main|main']\n\
  pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  \nPackage libgcc conflicts for:\nmatplotlib=3.1 -> pyqt -> libgcc\ngoatools=0.9.7\
  \ -> scipy -> libgcc\n\nPackage pypy3.8 conflicts for:\ngoatools=0.9.7 -> numpy\
  \ -> pypy3.8[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\npillow=6.2\
  \ -> python[version='>=3.8,<3.9.0a0'] -> pypy3.8[version='7.3.11.*|7.3.9.*|7.3.8.*']\n\
  matplotlib=3.1 -> python[version='>=3.8,<3.9.0a0'] -> pypy3.8[version='7.3.11.*|7.3.9.*|7.3.8.*|>=7.3.9|>=7.3.8|>=7.3.11']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> pypy3.8[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\n\
  \nPackage numpy-base conflicts for:\ngoatools=0.9.7 -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py27h2b20989_6|py36h2b20989_6|py27h2b20989_7|py36hdbf6ddf_7|py36h2b20989_7|py35hdbf6ddf_7|py37h2b20989_7|py27h2b20989_7|py36hdbf6ddf_7|py37hdbf6ddf_7|py36hdbf6ddf_8|py36h2b20989_8|py37hdbf6ddf_8|py27h2b20989_8|py37h7cdd4dd_9|py36h3dfced4_9|py35h3dfced4_9|py36h81de0dd_9|py27h74e8950_9|py37h74e8950_9|py36h74e8950_9|py27h81de0dd_9|py35h81de0dd_9|py35h74e8950_10|py27h74e8950_10|py27h81de0dd_10|py36h2f8d375_10|py27h2f8d375_10|py35h2f8d375_10|py37h2f8d375_11|py27h2f8d375_11|py36h2f8d375_11|py36hde5b4d6_11|py27hde5b4d6_11|py36h2f8d375_12|py37hde5b4d6_12|py38hde5b4d6_12|py36h9be14a7_1|py27h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py35h2b20989_0|py36h2b20989_0|py35hdbf6ddf_0|py36hdbf6ddf_1|py37hdbf6ddf_1|py27h2b20989_2|py36h2b20989_2|py37h2b20989_2|py37hdbf6ddf_2|py36hdbf6ddf_2|py36h2b20989_3|py36hdbf6ddf_3|py27h2b20989_3|py37hdbf6ddf_4|py36h2b20989_4|py35h2b20989_4|py35h2f8d375_4|py35h81de0dd_4|py38hde5b4d6_4|py36h2f8d375_5|py37h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py36h7cdd4dd_0|py36h3dfced4_0|py27h74e8950_0|py36h74e8950_0|py37h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py36h2f8d375_1|py27h2f8d375_1|py36h81de0dd_1|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_1|py27hde5b4d6_1|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py39h76555f2_1|py39h41b4c56_3|py37h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py39hdc34a94_3|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py37hfa32c7d_0|py38h75fe3a5_0|py38hfa32c7d_0|py36hfa32c7d_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py39h2ae0177_0|py39h0f7b65f_0|py37h4c65ebe_1|py39h4c65ebe_1|py39h21a3de8_1|py38h21a3de8_1|py37h34387ca_0|py37h7d8b39e_0|py39h34387ca_0|py38h7d8b39e_0|py39he2ba247_0|py37h74d4b33_0|py38h39b7dee_0|py39h79a1101_0|py37h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py38hb8be1f0_1|py37hb8be1f0_1|py310h9585f30_1|py310hf2716ce_2|py39hb8be1f0_2|py310h9585f30_2|py37hf524024_2|py38h1e6e340_3|py39hf524024_0|py310h9585f30_0|py310hf2716ce_0|py38hb8be1f0_0|py39h1e6e340_0|py38h1e6e340_0|py39ha15fc14_0|py38ha15fc14_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py310h375b286_1|py38h31eccc5_1|py310h8e6c178_1|py39h31eccc5_1|py310h8e6c178_0|py38h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_1|py38h1e6e340_1|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py310h375b286_0|py39hb8be1f0_0|py38hf524024_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py37h1e6e340_3|py38hf524024_2|py39hf524024_2|py38hb8be1f0_2|py37hb8be1f0_2|py310hf2716ce_1|py37hf524024_1|py38hf524024_1|py39hf524024_1|py39hb8be1f0_1|py310h79a1101_0|py38h2b8c604_0|py39h2b8c604_0|py38h79a1101_0|py39h39b7dee_0|py37h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py37hfae3a4d_0|py38hfae3a4d_0|py38he2ba247_0|py39h7d8b39e_0|py38h34387ca_0|py37h21a3de8_1|py38h4c65ebe_1|py36h75fe3a5_0|py36h75fe3a5_0|py38h73d599e_4|py39ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py38hdc34a94_3|py38h41b4c56_3|py39hfb011de_1|py27hde5b4d6_0|py27h2f8d375_0|py38hde5b4d6_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py37hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py27hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py37h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_1|py37h81de0dd_1|py27h81de0dd_1|py36h81de0dd_0|py35h81de0dd_0|py27h81de0dd_0|py27h2f8d375_0|py35h81de0dd_0|py27h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py35h74e8950_0|py37h3dfced4_0|py35h3dfced4_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py27h2f8d375_4|py36h2f8d375_4|py27h81de0dd_4|py37h2f8d375_4|py35hdbf6ddf_4|py36hdbf6ddf_4|py37h2b20989_4|py27hdbf6ddf_4|py27h2b20989_4|py27hdbf6ddf_3|py37hdbf6ddf_3|py37h2b20989_3|py27hdbf6ddf_2|py27h2b20989_1|py27hdbf6ddf_1|py37h2b20989_1|py36h2b20989_1|py27hdbf6ddf_0|py36hdbf6ddf_0|py27h2b20989_0|py35hdbf6ddf_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35h9be14a7_1|py35h0ea5e3f_1|py27h0ea5e3f_1|py36h0ea5e3f_1|py38h2f8d375_12|py36hde5b4d6_12|py27hde5b4d6_12|py27h2f8d375_12|py37h2f8d375_12|py37hde5b4d6_11|py37h2f8d375_10|py35h81de0dd_10|py36h81de0dd_10|py37h81de0dd_10|py36h74e8950_10|py37h74e8950_10|py35h74e8950_9|py37h81de0dd_9|py27h3dfced4_9|py37h3dfced4_9|py27h7cdd4dd_9|py35h7cdd4dd_9|py36h7cdd4dd_9|py35h2b20989_8|py35hdbf6ddf_8|py27hdbf6ddf_8|py37h2b20989_8|py27hdbf6ddf_7|py36h2b20989_7|py37h2b20989_7|py37hdbf6ddf_7|py35h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_6|py27hdbf6ddf_6|py37hdbf6ddf_6|py37h2b20989_6|py37hde5b4d6_0']\n\
  matplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py27h2b20989_6|py36h2b20989_6|py37hdbf6ddf_6|py36hdbf6ddf_6|py27h2b20989_7|py36hdbf6ddf_7|py35hdbf6ddf_7|py37hdbf6ddf_7|py27h2b20989_7|py36h2b20989_7|py36hdbf6ddf_8|py36h2b20989_8|py37hdbf6ddf_8|py27h2b20989_8|py35h2b20989_8|py35h7cdd4dd_9|py37h7cdd4dd_9|py35h3dfced4_9|py36h81de0dd_9|py37h81de0dd_9|py36h74e8950_9|py27h81de0dd_9|py35h74e8950_10|py27h74e8950_10|py37h81de0dd_10|py36h2f8d375_10|py27h2f8d375_11|py36h2f8d375_11|py36hde5b4d6_11|py27hde5b4d6_11|py27h2f8d375_12|py38h2f8d375_12|py36h9be14a7_1|py36h2b20989_0|py35h2b20989_0|py36h2b20989_0|py27h2b20989_0|py36h2b20989_1|py36hdbf6ddf_1|py37hdbf6ddf_1|py36h2b20989_2|py37h2b20989_2|py27hdbf6ddf_2|py37hdbf6ddf_2|py36hdbf6ddf_2|py36h2b20989_3|py27h2b20989_3|py27h2b20989_4|py37hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py36hde5b4d6_0|py35h2f8d375_4|py35h81de0dd_4|py38hde5b4d6_4|py36h2f8d375_5|py37h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py36h7cdd4dd_0|py36h3dfced4_0|py27h74e8950_0|py36h74e8950_0|py37h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py36h2f8d375_1|py27h2f8d375_1|py36h81de0dd_1|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_1|py27hde5b4d6_1|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py39h76555f2_1|py39h41b4c56_3|py37h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py39hdc34a94_3|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py37hfa32c7d_0|py38h75fe3a5_0|py38hfa32c7d_0|py36hfa32c7d_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py39h2ae0177_0|py39h0f7b65f_0|py37h4c65ebe_1|py39h4c65ebe_1|py39h21a3de8_1|py38h21a3de8_1|py37h34387ca_0|py37h7d8b39e_0|py39h34387ca_0|py38h7d8b39e_0|py39he2ba247_0|py37h74d4b33_0|py38h39b7dee_0|py39h79a1101_0|py37h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py38hb8be1f0_1|py37hb8be1f0_1|py310h9585f30_1|py310hf2716ce_2|py39hb8be1f0_2|py310h9585f30_2|py37hf524024_2|py38h1e6e340_3|py39hf524024_0|py310h9585f30_0|py310hf2716ce_0|py38hb8be1f0_0|py39h1e6e340_0|py38h1e6e340_0|py39ha15fc14_0|py38ha15fc14_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py310h375b286_1|py38h31eccc5_1|py310h8e6c178_1|py39h31eccc5_1|py310h8e6c178_0|py38h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_1|py38h1e6e340_1|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py310h375b286_0|py39hb8be1f0_0|py38hf524024_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py37h1e6e340_3|py38hf524024_2|py39hf524024_2|py38hb8be1f0_2|py37hb8be1f0_2|py310hf2716ce_1|py37hf524024_1|py38hf524024_1|py39hf524024_1|py39hb8be1f0_1|py310h79a1101_0|py38h2b8c604_0|py39h2b8c604_0|py38h79a1101_0|py39h39b7dee_0|py37h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py37hfae3a4d_0|py38hfae3a4d_0|py38he2ba247_0|py39h7d8b39e_0|py38h34387ca_0|py37h21a3de8_1|py38h4c65ebe_1|py36h75fe3a5_0|py36h75fe3a5_0|py38h73d599e_4|py39ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py38hdc34a94_3|py38h41b4c56_3|py39hfb011de_1|py27hde5b4d6_0|py27h2f8d375_0|py38hde5b4d6_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py37hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py27hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py37h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_1|py37h81de0dd_1|py27h81de0dd_1|py36h81de0dd_0|py35h81de0dd_0|py27h81de0dd_0|py27h2f8d375_0|py35h81de0dd_0|py27h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py35h74e8950_0|py37h3dfced4_0|py35h3dfced4_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py27h2f8d375_4|py36h2f8d375_4|py27h81de0dd_4|py37h2f8d375_4|py37hde5b4d6_0|py37h2f8d375_0|py35hdbf6ddf_4|py36hdbf6ddf_4|py36h2b20989_4|py37h2b20989_4|py27hdbf6ddf_4|py27hdbf6ddf_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py37h2b20989_3|py27h2b20989_2|py27h2b20989_1|py27hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_0|py35h9be14a7_1|py27h9be14a7_1|py35h0ea5e3f_1|py27h0ea5e3f_1|py36h0ea5e3f_1|py38hde5b4d6_12|py36hde5b4d6_12|py37hde5b4d6_12|py27hde5b4d6_12|py36h2f8d375_12|py37h2f8d375_12|py37hde5b4d6_11|py37h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py37h2f8d375_10|py35h81de0dd_10|py36h81de0dd_10|py27h81de0dd_10|py36h74e8950_10|py37h74e8950_10|py35h81de0dd_9|py37h74e8950_9|py35h74e8950_9|py27h74e8950_9|py27h3dfced4_9|py36h3dfced4_9|py37h3dfced4_9|py27h7cdd4dd_9|py36h7cdd4dd_9|py35hdbf6ddf_8|py27hdbf6ddf_8|py37h2b20989_8|py37hdbf6ddf_7|py36hdbf6ddf_7|py27hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py35h2b20989_7|py36h2b20989_7|py27hdbf6ddf_7|py27hdbf6ddf_6|py37h2b20989_6']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py36h9be14a7_1|py36h2b20989_0|py35h2b20989_0|py36h2b20989_0|py27h2b20989_0|py36h2b20989_1|py36hdbf6ddf_1|py37hdbf6ddf_1|py36h2b20989_2|py37h2b20989_2|py27hdbf6ddf_2|py37hdbf6ddf_2|py36hdbf6ddf_2|py36h2b20989_3|py27h2b20989_3|py27h2b20989_4|py37hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py36hde5b4d6_0|py35h2f8d375_4|py35h81de0dd_4|py38hde5b4d6_4|py36h2f8d375_5|py37h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py36h7cdd4dd_0|py36h3dfced4_0|py27h74e8950_0|py36h74e8950_0|py37h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py36h2f8d375_1|py27h2f8d375_1|py36h81de0dd_1|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_1|py27hde5b4d6_1|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py39h76555f2_1|py39h41b4c56_3|py37h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py39hdc34a94_3|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py37hfa32c7d_0|py38h75fe3a5_0|py38hfa32c7d_0|py36hfa32c7d_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py39h2ae0177_0|py39h0f7b65f_0|py37h4c65ebe_1|py39h4c65ebe_1|py39h21a3de8_1|py38h21a3de8_1|py37h34387ca_0|py37h7d8b39e_0|py39h34387ca_0|py38h7d8b39e_0|py39he2ba247_0|py37h74d4b33_0|py38h39b7dee_0|py39h79a1101_0|py37h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py38hb8be1f0_1|py37hb8be1f0_1|py310h9585f30_1|py310hf2716ce_2|py39hb8be1f0_2|py310h9585f30_2|py37hf524024_2|py38h1e6e340_3|py39hf524024_0|py310h9585f30_0|py310hf2716ce_0|py38hb8be1f0_0|py39h1e6e340_0|py38h1e6e340_0|py39ha15fc14_0|py38ha15fc14_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py310h375b286_1|py38h31eccc5_1|py310h8e6c178_1|py39h31eccc5_1|py310h8e6c178_0|py38h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_1|py38h1e6e340_1|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py310h375b286_0|py39hb8be1f0_0|py38hf524024_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py37h1e6e340_3|py38hf524024_2|py39hf524024_2|py38hb8be1f0_2|py37hb8be1f0_2|py310hf2716ce_1|py37hf524024_1|py38hf524024_1|py39hf524024_1|py39hb8be1f0_1|py310h79a1101_0|py38h2b8c604_0|py39h2b8c604_0|py38h79a1101_0|py39h39b7dee_0|py37h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py37hfae3a4d_0|py38hfae3a4d_0|py38he2ba247_0|py39h7d8b39e_0|py38h34387ca_0|py37h21a3de8_1|py38h4c65ebe_1|py36h75fe3a5_0|py36h75fe3a5_0|py38h73d599e_4|py39ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py38hdc34a94_3|py38h41b4c56_3|py39hfb011de_1|py27hde5b4d6_0|py27h2f8d375_0|py38hde5b4d6_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py37hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py27hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py37h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_1|py37h81de0dd_1|py27h81de0dd_1|py36h81de0dd_0|py35h81de0dd_0|py27h81de0dd_0|py27h2f8d375_0|py35h81de0dd_0|py27h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py35h74e8950_0|py37h3dfced4_0|py35h3dfced4_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py27h2f8d375_4|py36h2f8d375_4|py27h81de0dd_4|py37h2f8d375_4|py37hde5b4d6_0|py37h2f8d375_0|py35hdbf6ddf_4|py36hdbf6ddf_4|py36h2b20989_4|py37h2b20989_4|py27hdbf6ddf_4|py27hdbf6ddf_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py37h2b20989_3|py27h2b20989_2|py27h2b20989_1|py27hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_0|py35h9be14a7_1|py27h9be14a7_1|py35h0ea5e3f_1|py27h0ea5e3f_1|py36h0ea5e3f_1']\n\
  \nPackage ca-certificates conflicts for:\ngoatools=0.9.7 -> python -> ca-certificates\n\
  python=3.7 -> openssl[version='>=1.1.1l,<1.1.2a'] -> ca-certificates\npillow=6.2\
  \ -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n\nPackage six conflicts\
  \ for:\npandas=0.25 -> python-dateutil[version='>=2.6.1'] -> six[version='>=1.5']\n\
  matplotlib=3.1 -> cycler[version='>=0.10'] -> six[version='>=1.5']\n\nPackage libpng\
  \ conflicts for:\nmatplotlib=3.1 -> libpng[version='>=1.6.37,<1.7.0a0']\nmatplotlib=3.1\
  \ -> matplotlib-base==3.1.1=py36he7580a8_1 -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  \nPackage freetype conflicts for:\nmatplotlib=3.1 -> freetype[version='>=2.9.1,<3.0a0']\n\
  pillow=6.2 -> freetype[version='>=2.9.1,<3.0a0']\n\nPackage setuptools conflicts\
  \ for:\npython=3.7 -> pip -> setuptools\npandas=0.25 -> setuptools[version='<60.0.0']\n\
  matplotlib=3.1 -> setuptools\n\nPackage numpy conflicts for:\nmatplotlib=3.1 ->\
  \ numpy[version='>=1.14.6,<2.0a0']\nmatplotlib=3.1 -> matplotlib-base==3.1.0=py37h5f35d83_0\
  \ -> numpy[version='>=1.11.3,<2.0a0']\n\nPackage pandas conflicts for:\npandas=0.25\n\
  goatools=0.9.7 -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\n\
  \nPackage gdbm conflicts for:\npillow=6.2 -> pypy3.6[version='>=7.3.1'] -> gdbm[version='>=1.18,<1.19.0a0']\n\
  python=3.7 -> pypy3.7=7.3.7 -> gdbm[version='>=1.18,<1.19.0a0']The following specifications\
  \ were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - matplotlib=3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> __glibc[version='>=2.17']\n  - pandas=0.25 -> libgcc-ng[version='>=7.3.0']\
  \ -> __glibc[version='>=2.17']\n  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] ->\
  \ __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2F346yfs3s%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2F346yfs3s%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.228GiB</td>
						<td>1005.5 seconds</td>
						<td>1071.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738206128">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.3.4">1.3.4</a>
						</td>
						<td>snakemake</td>
						<td>755 days</td>
						<td>2023-02-19 12:50:47</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fhz3f2dna%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fhz3f2dna%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.966GiB</td>
						<td>125.0 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743190704">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/2.7">2.7</a>
						</td>
						<td>nextflow</td>
						<td>754 days</td>
						<td>2023-02-19 04:10:58</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2F9h8vz5zc%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2F9h8vz5zc%2Foutput.tar.xz">0.016GiB</a>
						</td>
						<td>1.899GiB</td>
						<td>1236.7 seconds</td>
						<td>619.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746842512">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/1.2.0">1.2.0</a>
						</td>
						<td>nextflow</td>
						<td>745 days</td>
						<td>2023-02-19 04:08:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: metadataCategoryPairwise.r Metadata.tsv
error: "Warning message:\nIn read.table(file = file, header = header, sep = sep, quote\
  \ = quote,  :\n  incomplete final line found by readTableHeader on 'Metadata.tsv'\n\
  Error in 1:ncol(data) : argument of length 0\nExecution halted"
exit_status: 1
kind: CalledProcessError
output: (empty)
process: metadata_category_pairwise (1)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- 2a
- 6be1324c1ecefc3af0ab73e4abcfea

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fv8g4h1s7%2Flogs.tar.xz">0.006GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fv8g4h1s7%2Foutput.tar.xz">0.005GiB</a>
						</td>
						<td>1.905GiB</td>
						<td>677.7 seconds</td>
						<td>569.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743119584">
								<a href="https://nf-co.re/mag">mag</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mag/tree/1.2.0">1.2.0</a>
						</td>
						<td>nextflow</td>
						<td>740 days</td>
						<td>2023-02-20 04:31:46</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name nfcore-magbusco-1.2.0.img.pulling.1676867537523\
  \ docker://nfcore/magbusco:1.2.0 > /dev/null\n  status : 255\n  message:\n    INFO:\
  \    Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:59a5002f4594201d780ba58b6d8e3a848ed4a77bcaf3132086752c7562cb0928\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:3fbba84366ddeb8b6dece9fb1c547db6afa6e727c4bfe599a5ca2cfdff4c964e\n\
  \    Copying blob sha256:2ac1746f5739a40bb38a1153169721ea57ef15bf4ba5b74d6bd13769fdb0f5cd\n\
  \    Copying blob sha256:af56095fd632392ba1a6fb65fa6fae8fd1dfbe1477c2674ea661512c44b063e6\n\
  \    Copying blob sha256:53df098841a0891de73c90f76c1fc19543754b542ad1c27a76da237170c51891\n\
  \    Copying blob sha256:d1cd1ad1be47a7239afba8db55cc22b5aae34a1be1c2c3f38485ebd726a49a2e\n\
  \    Copying blob sha256:fa3c4f8591b605acc5591534fcd35b52ea325ce3c7b61d810f76367267f86668\n\
  \    Copying blob sha256:badf730e88f35f53dfbe2a56f21e8f78046a8af319dcce25c52ebed5c704f669\n\
  \    Copying config sha256:5b65db54e7c339290592fa87471d9d7fda987a135694a8c78500bd0c3b6abfed\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.3aa1ecf171b5a82584e2a2cafc99ed0296125fc2aee4b372797b47b4bdfa8d0a\"\n\n\t\
  at nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\tat nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\
  \tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\tat nextflow.processor.TaskRun$isContainerEnabled$2.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name nfcore-magbusco-1.2.0.img.pulling.1676867537523\
  \ docker://nfcore/magbusco:1.2.0 > /dev/null\n  status : 255\n  message:\n    INFO:\
  \    Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:59a5002f4594201d780ba58b6d8e3a848ed4a77bcaf3132086752c7562cb0928\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:3fbba84366ddeb8b6dece9fb1c547db6afa6e727c4bfe599a5ca2cfdff4c964e\n\
  \    Copying blob sha256:2ac1746f5739a40bb38a1153169721ea57ef15bf4ba5b74d6bd13769fdb0f5cd\n\
  \    Copying blob sha256:af56095fd632392ba1a6fb65fa6fae8fd1dfbe1477c2674ea661512c44b063e6\n\
  \    Copying blob sha256:53df098841a0891de73c90f76c1fc19543754b542ad1c27a76da237170c51891\n\
  \    Copying blob sha256:d1cd1ad1be47a7239afba8db55cc22b5aae34a1be1c2c3f38485ebd726a49a2e\n\
  \    Copying blob sha256:fa3c4f8591b605acc5591534fcd35b52ea325ce3c7b61d810f76367267f86668\n\
  \    Copying blob sha256:badf730e88f35f53dfbe2a56f21e8f78046a8af319dcce25c52ebed5c704f669\n\
  \    Copying config sha256:5b65db54e7c339290592fa87471d9d7fda987a135694a8c78500bd0c3b6abfed\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.3aa1ecf171b5a82584e2a2cafc99ed0296125fc2aee4b372797b47b4bdfa8d0a\"\n\n\t\
  at nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 04:34:08.742 [Actor Thread 4] DEBUG nextflow.Session\
  \ - Session aborted -- Cause: java.lang.IllegalStateException: Failed to pull singularity\
  \ image\n  command: singularity pull  --name nfcore-magbusco-1.2.0.img.pulling.1676867537523\
  \ docker://nfcore/magbusco:1.2.0 > /dev/null\n  status : 255\n  message:\n    INFO:\
  \    Converting OCI blobs to SIF format\n    INFO:    Starting build...\n    Getting\
  \ image source signatures\n    Copying blob sha256:59a5002f4594201d780ba58b6d8e3a848ed4a77bcaf3132086752c7562cb0928\n\
  \    Copying blob sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b\n\
  \    Copying blob sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3\n\
  \    Copying blob sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b\n\
  \    Copying blob sha256:3fbba84366ddeb8b6dece9fb1c547db6afa6e727c4bfe599a5ca2cfdff4c964e\n\
  \    Copying blob sha256:2ac1746f5739a40bb38a1153169721ea57ef15bf4ba5b74d6bd13769fdb0f5cd\n\
  \    Copying blob sha256:af56095fd632392ba1a6fb65fa6fae8fd1dfbe1477c2674ea661512c44b063e6\n\
  \    Copying blob sha256:53df098841a0891de73c90f76c1fc19543754b542ad1c27a76da237170c51891\n\
  \    Copying blob sha256:d1cd1ad1be47a7239afba8db55cc22b5aae34a1be1c2c3f38485ebd726a49a2e\n\
  \    Copying blob sha256:fa3c4f8591b605acc5591534fcd35b52ea325ce3c7b61d810f76367267f86668\n\
  \    Copying blob sha256:badf730e88f35f53dfbe2a56f21e8f78046a8af319dcce25c52ebed5c704f669\n\
  \    Copying config sha256:5b65db54e7c339290592fa87471d9d7fda987a135694a8c78500bd0c3b6abfed\n\
  \    Writing manifest to image destination\n    Storing signatures\n    FATAL: \
  \  While making image from oci registry: error fetching image to cache: while building\
  \ SIF from layers: conveyor failed to get: no descriptor found for reference \"\
  sha256.3aa1ecf171b5a82584e2a2cafc99ed0296125fc2aee4b372797b47b4bdfa8d0a\"\n\nFeb-20\
  \ 04:34:08.819 [Actor Thread 4] DEBUG nextflow.Session - The following nodes are\
  \ still active:\n[process] get_busco_version\n  status=ACTIVE\n  port 0: (cntrl)\
  \ -     ; channel: $\n\n[process] get_software_versions\n  status=ACTIVE\n  port\
  \ 0: (value) OPEN  ; channel: busco_version\n  port 1: (cntrl) -     ; channel:\
  \ $\n\n[process] rename_short_read_fastqs\n  status=ACTIVE\n  port 0: (queue) closed;\
  \ channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] fastqc_raw\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] fastp\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1:\
  \ (value) bound ; channel: adapter\n  port 2: (value) bound ; channel: adapter_reverse\n\
  \  port 3: (value) bound ; channel: qual\n  port 4: (value) bound ; channel: trim_qual\n\
  \  port 5: (cntrl) -     ; channel: $\n\n[process] phix_download_db\n  status=ACTIVE\n\
  \  port 0: (queue) closed; channel: genome\n  port 1: (cntrl) -     ; channel: $\n\
  \n[process] remove_phix\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n\
  \  port 1: (cntrl) -     ; channel: $\n\n[process] fastqc_trimmed\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] filtlong\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] nanoplot\n  status=ACTIVE\n  port 0:\
  \ (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process]\
  \ centrifuge_db_preparation\n  status=ACTIVE\n  port 0: (queue) closed; channel:\
  \ db\n  port 1: (cntrl) -     ; channel: $\n\n[process] centrifuge\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] kraken2_db_preparation\n  status=ACTIVE\n  port 0: (queue) closed; channel:\
  \ db\n  port 1: (cntrl) -     ; channel: $\n\n[process] kraken2\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] krona\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1:\
  \ (cntrl) -     ; channel: $\n\n[process] megahit\n  status=ACTIVE\n  port 0: (queue)\
  \ OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] spadeshybrid\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] spades\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel:\
  \ -\n  port 1: (cntrl) -     ; channel: $\n\n[process] quast\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] bowtie2\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port\
  \ 1: (cntrl) -     ; channel: $\n\n[process] metabat\n  status=ACTIVE\n  port 0:\
  \ (queue) OPEN  ; channel: -\n  port 1: (value) bound ; channel: min_size\n  port\
  \ 2: (value) bound ; channel: max_unbinned\n  port 3: (value) bound ; channel: min_length_unbinned\n\
  \  port 4: (cntrl) -     ; channel: $\n\n[process] busco_db_preparation\n  status=ACTIVE\n\
  \  port 0: (queue) closed; channel: database\n  port 1: (cntrl) -     ; channel:\
  \ $\n\n[process] busco\n  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n\
  \  port 1: (cntrl) -     ; channel: $\n\n[process] busco_plot\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] busco_summary\n  status=ACTIVE\n  port 0: (value) OPEN  ; channel: short_summary.*.txt\n\
  \  port 1: (cntrl) -     ; channel: $\n\n[process] quast_bins\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] merge_quast_and_busco\n  status=ACTIVE\n  port 0: (value) OPEN  ; channel:\
  \ quast_bin_sum\n  port 1: (value) OPEN  ; channel: busco_sum\n  port 2: (cntrl)\
  \ -     ; channel: $\n\n[process] cat\n  status=ACTIVE\n  port 0: (queue) OPEN \
  \ ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process] multiqc\n  status=ACTIVE\n\
  \  port 0: (value) bound ; channel: multiqc_config\n  port 1: (value) bound ; channel:\
  \ mqc_custom_config\n  port 2: (value) OPEN  ; channel: fastqc_raw\n  port 3: (value)\
  \ OPEN  ; channel: fastqc_trimmed\n  port 4: (value) bound ; channel: host_removal\n\
  \  port 5: (value) OPEN  ; channel: quast*/*\n  port 6: (value) OPEN  ; channel:\
  \ bowtie2log/*\n  port 7: (value) OPEN  ; channel: short_summary\n  port 8: (value)\
  \ OPEN  ; channel: software_versions/*\n  port 9: (queue) OPEN  ; channel: workflow_summary\n\
  \  port 10: (cntrl) -     ; channel: $\n\n[process] output_documentation\n  status=ACTIVE\n\
  \  port 0: (value) bound ; channel: output_docs\n  port 1: (value) bound ; channel:\
  \ images\n  port 2: (cntrl) -     ; channel: $\n\nFeb-20 04:34:08.823 [main] DEBUG\
  \ nextflow.Session - Session await > all processes finished\nFeb-20 04:34:08.823\
  \ [main] DEBUG nextflow.Session - Session await > all barriers passed\nFeb-20 04:34:09.221\
  \ [main] INFO  nextflow.Nextflow - -\e[0;35m[nf-core/mag]\e[0;31m Pipeline completed\
  \ with errors\e[0m-\nFeb-20 04:34:09.234 [main] DEBUG nextflow.trace.WorkflowStatsObserver\
  \ - Workflow completed > WorkflowStats[succeededCount=0; failedCount=0; ignoredCount=0;\
  \ cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0;\
  \ abortedCount=0; succeedDuration=0ms; failedDuration=0ms; cachedDuration=0ms;loadCpus=0;\
  \ loadMemory=0; peakRunning=0; peakCpus=0; peakMemory=0; ]\nFeb-20 04:34:09.234\
  \ [main] DEBUG nextflow.trace.TraceFileObserver - Workflow completed -- saving trace\
  \ file\nFeb-20 04:34:09.242 [main] DEBUG nextflow.trace.ReportObserver - Workflow\
  \ completed -- rendering execution report\nFeb-20 04:34:09.247 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Execution report summary data:\n  []\nFeb-20 04:34:10.785 [main] DEBUG nextflow.trace.TimelineObserver\
  \ - Workflow completed -- rendering execution timeline\nFeb-20 04:34:11.151 [main]\
  \ DEBUG nextflow.cache.CacheDB - Closing CacheDB done\nFeb-20 04:34:11.200 [main]\
  \ DEBUG nextflow.script.ScriptRunner - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fuevjdjye%2Flogs.tar.xz">0.016GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fuevjdjye%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.731GiB</td>
						<td>51.5 seconds</td>
						<td>145.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743053808">
								<a href="https://nf-co.re/dualrnaseq">dualrnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/dualrnaseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>734 days</td>
						<td>2023-02-19 04:31:02</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dualrnaseq%2F9x46t3wg%2Flogs.tar.xz">0.961GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dualrnaseq%2F9x46t3wg%2Foutput.tar.xz">0.007GiB</a>
						</td>
						<td>1.687GiB</td>
						<td>638.7 seconds</td>
						<td>491.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743052368">
								<a href="https://nf-co.re/diaproteomics">diaproteomics</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/diaproteomics/tree/1.2.1">1.2.1</a>
						</td>
						<td>nextflow</td>
						<td>731 days</td>
						<td>2023-02-19 06:52:30</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2F8yu5x4q2%2Flogs.tar.xz">0.123GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2F8yu5x4q2%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.902GiB</td>
						<td>1101.5 seconds</td>
						<td>633.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743052128">
								<a href="https://nf-co.re/diaproteomics">diaproteomics</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/diaproteomics/tree/1.2.2">1.2.2</a>
						</td>
						<td>nextflow</td>
						<td>724 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2Fu4y600yz%2Flogs.tar.xz">0.123GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2Fu4y600yz%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.928GiB</td>
						<td>1095.1 seconds</td>
						<td>926.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743105664">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.3.2">2.3.2</a>
						</td>
						<td>nextflow</td>
						<td>705 days</td>
						<td>2023-02-19 06:20:46</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 06:20:58.459
kind: NextflowMiscError
process: main
rest: 'ERROR: Validation of pipeline parameters failed!'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fdsn1z2sc%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fdsn1z2sc%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.886GiB</td>
						<td>31.5 seconds</td>
						<td>11.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743138912">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/1.6">1.6</a>
						</td>
						<td>nextflow</td>
						<td>695 days</td>
						<td>2023-02-19 04:43:03</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: trim_galore --fastqc --gzip SRR389222_sub2.fastq.gz                  --cores
  1
error: 'Proceeding with single-core trimming (user-defined)

  Path to Cutadapt set as: ''cutadapt'' (default)

  Cutadapt seems to be working fine (tested command ''cutadapt --version'')

  Cutadapt version: 3.3

  single-core operation.

  No quality encoding type selected. Assuming that the data provided uses Sanger encoded
  Phred scores (default)


  gzip: SRR389222_sub2.fastq.gz: No such file or directory

  Input file ''SRR389222_sub2.fastq.gz'' seems to be completely empty. Consider respecifying!'
exit_status: 255
kind: CalledProcessError
output: (empty)
process: trim_galore (SRR389222_sub2)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- 9c
- eacc560a0db6ed43a25c1a5f50e61e

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fk4mwr7li%2Flogs.tar.xz">0.011GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fk4mwr7li%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>1.517GiB</td>
						<td>475.6 seconds</td>
						<td>286.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743192624">
								<a href="https://nf-co.re/scrnaseq">scrnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/scrnaseq/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>692 days</td>
						<td>2023-02-19 06:18:45</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: gffread -F gencode.vM19.annotation.chr19.gtf -w "GRCm38.p6.genome.chr19.fa.transcriptome.fa"
  -g GRCm38.p6.genome.chr19.fa
error: 'FASTA index file GRCm38.p6.genome.chr19.fa.fai created.

  Error: cannot open input file gencode.vM19.annotation.chr19.gtf!'
exit_status: 1
kind: CalledProcessError
output: (empty)
process: extract_transcriptome (GRCm38.p6.genome.chr19.fa)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '81'
- e544f7f0ad42859bd158e5c433d503

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/scrnaseq%2Feoc9nytg%2Flogs.tar.xz">0.015GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/scrnaseq%2Feoc9nytg%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>1.498GiB</td>
						<td>405.1 seconds</td>
						<td>322.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738321536">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>689 days</td>
						<td>2023-02-19 04:09:22</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/7v8b6hnjhj/code/workflow/rules/../envs/rbt.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/70576fd7-49d7-48e5-963d-252bc7236206\
  \ singularity  exec --home /home/azureuser/tmp/7v8b6hnjhj/code/.test  /home/azureuser/tmp/7v8b6hnjhj/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/7v8b6hnjhj/code/.test/.snakemake/conda/2c2e901b7c4485f6ae6d503eee64fb23_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/7v8b6hnjhj/code/.test/.snakemake/conda/2c2e901b7c4485f6ae6d503eee64fb23_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|\
  \          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:00, 6186.29it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 6892.86it/s]\nExamining\
  \ rust-bio-tools=0.19:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 7695.97it/s]          \nExamining @/linux-64::__archspec==1=x86_64:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 4/6 [00:00<00:00, 60.43it/s]\nExamining\
  \ bcftools=1.10:  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6\
  \ [00:00<00:00, 75.30it/s]                   \n                                \
  \                                      \n\nDetermining conflicts:   0%|        \
  \  | 0/6 [00:00<?, ?it/s]\nExamining conflict for rust-bio-tools __glibc bcftools:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining conflict for rust-bio-tools bcftools:\
  \  17%|\u2588\u258B        | 1/6 [00:00<00:00, 63.25it/s]\n                    \
  \                                                                         \nfailed\n\
  Solving environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|\
  \          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:00, 5924.16it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 6307.22it/s]\nExamining\
  \ rust-bio-tools=0.19:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 6577.58it/s]          \nExamining @/linux-64::__archspec==1=x86_64:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 4/6 [00:00<00:00, 45.31it/s]\nExamining\
  \ bcftools=1.10:  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6\
  \ [00:00<00:00, 56.51it/s]                   \n                                \
  \                                      \n\nDetermining conflicts:   0%|        \
  \  | 0/6 [00:00<?, ?it/s]\nExamining conflict for rust-bio-tools __glibc bcftools:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining conflict for rust-bio-tools bcftools:\
  \  17%|\u2588\u258B        | 1/6 [00:00<00:00, 57.60it/s]\n                    \
  \                                                                         \nfailed\n\
  \nUnsatisfiableError: The following specifications were found to be incompatible\
  \ with each other:\n\nOutput in format: Requested package -> Available versions\n\
  \nPackage libopenblas conflicts for:\nrust-bio-tools=0.19 -> gsl[version='>=2.6,<2.7.0a0']\
  \ -> libopenblas[version='>=0.3.3,<1.0a0']\nrust-bio-tools=0.19 -> libopenblas[version='>=0.3.13,<1.0a0']\n\
  \nPackage libzlib conflicts for:\nrust-bio-tools=0.19 -> zlib[version='>=1.2.11,<1.3.0a0']\
  \ -> libzlib[version='1.2.11|1.2.11|1.2.11|1.2.12|1.2.12|1.2.12|1.2.12|1.2.12|1.2.13',build='h36c2ea0_1012|h36c2ea0_1013|h166bdaf_1014|h166bdaf_0|h166bdaf_2|h166bdaf_3|h166bdaf_4|h166bdaf_1']\n\
  bcftools=1.10 -> zlib[version='>=1.2.11,<1.3.0a0'] -> libzlib[version='1.2.11|1.2.11|1.2.11|1.2.12|1.2.12|1.2.12|1.2.12|1.2.12|1.2.13',build='h36c2ea0_1012|h36c2ea0_1013|h166bdaf_1014|h166bdaf_0|h166bdaf_2|h166bdaf_3|h166bdaf_4|h166bdaf_1']\n\
  \nPackage libgcc-ng conflicts for:\nrust-bio-tools=0.19 -> libgcc-ng[version='>=7.5.0']\n\
  rust-bio-tools=0.19 -> gsl[version='>=2.6,<2.7.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=7.3.0|>=9.3.0|>=9.4.0|>=11.2.0|>=4.9|>=7.2.0']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - bcftools=1.10 -> libgcc-ng[version='>=7.5.0']\
  \ -> __glibc[version='>=2.17']\n  - rust-bio-tools=0.19 -> libgcc-ng[version='>=7.5.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fdegy5cdg%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fdegy5cdg%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.321GiB</td>
						<td>518.4 seconds</td>
						<td>850.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743105424">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.3.3">2.3.3</a>
						</td>
						<td>nextflow</td>
						<td>682 days</td>
						<td>2023-02-19 06:34:07</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 06:34:29.220
kind: NextflowMiscError
process: main
rest: '[nf-core/eager] error: --input was not supplied! Please check ''--help'' or
  documentation under ''running the pipeline'' for details'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fpflqls9f%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fpflqls9f%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.704GiB</td>
						<td>35.9 seconds</td>
						<td>22.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738289488">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.0.2">v2.0.2</a>
						</td>
						<td>snakemake</td>
						<td>682 days</td>
						<td>2023-02-19 06:00:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/h0lw9e5onk/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/147047c4-e93e-443b-994a-b9dec4520486\
  \ singularity  exec --home /home/azureuser/tmp/h0lw9e5onk/code/.test  /home/azureuser/tmp/h0lw9e5onk/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/h0lw9e5onk/code/.test/.snakemake/conda/c277073b691ea2fc60ead0455685ec5b_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/h0lw9e5onk/code/.test/.snakemake/conda/c277073b691ea2fc60ead0455685ec5b_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining pandas=0.25:  11%|\u2588    \
  \     | 1/9 [00:00<00:00, 6668.21it/s]        \nExamining pillow=6.2:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:02<00:09,  1.36s/it]   \nExamining pillow=6.2:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:02<00:05,  1.10it/s]\nExamining @/linux-64::__unix==0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:03<00:05,  1.10it/s]\nExamining @/linux-64::__unix==0=0:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:03<00:04,  1.15it/s]\nExamining\
  \ goatools=0.9.7:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:03<00:04,  1.15it/s]\
  \         \nExamining @/linux-64::__glibc==2.31=0:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:03<00:03,  1.15it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:03<00:01,  1.78it/s]\n\
  Examining python=3.7:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:03<00:01,\
  \  1.78it/s]                 \nExamining matplotlib=3.1:  78%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258A  | 7/9 [00:06<00:01,  1.78it/s]\nExamining matplotlib=3.1:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:06<00:00,\
  \  1.27it/s]\nExamining @/linux-64::__archspec==1=x86_64:  89%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:09<00:00,  1.27it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:09<00:00,\
  \  1.33s/it]\n                                                                 \
  \                        \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for pandas goatools:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for pandas matplotlib goatools:  11%|\u2588       \
  \  | 1/9 [00:01<00:12,  1.56s/it]\nExamining conflict for pandas matplotlib goatools:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:01<00:05,  1.28it/s]\nExamining conflict\
  \ for pandas pillow goatools python matplotlib:  22%|\u2588\u2588\u258F       |\
  \ 2/9 [00:26<00:05,  1.28it/s]\nExamining conflict for pandas pillow goatools python\
  \ matplotlib:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:26<01:05, 10.93s/it]\n\
  Examining conflict for pandas matplotlib:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [00:29<01:05, 10.93s/it]                       \nExamining conflict for pandas\
  \ matplotlib:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:29<00:39,  7.92s/it]\n\
  Examining conflict for python pandas pillow matplotlib:  44%|\u2588\u2588\u2588\u2588\
  \u258D     | 4/9 [00:30<00:39,  7.92s/it]\nExamining conflict for python pandas\
  \ pillow matplotlib:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:30<00:21,\
  \  5.49s/it]\nExamining conflict for pandas pillow goatools matplotlib:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:34<00:21,  5.49s/it]\nExamining conflict\
  \ for pandas pillow goatools matplotlib:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 6/9 [00:34<00:14,  4.96s/it]\nExamining conflict for pandas pillow __glibc\
  \ python matplotlib:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:39<00:14,\
  \  4.96s/it]\nExamining conflict for pandas pillow __glibc python matplotlib:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:39<00:10,  5.11s/it]\nExamining\
  \ conflict for python pandas matplotlib:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:41<00:10,  5.11s/it]               \nExamining conflict\
  \ for python pandas matplotlib:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:41<00:04,  4.08s/it]\nExamining conflict for matplotlib pillow:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:41<00:04,\
  \  4.08s/it]       \nExamining conflict for matplotlib pillow: 100%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:41<00:00,  2.86s/it]\n\
  Examining conflict for python pillow: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588| 9/9 [00:43<00:00,  2.86s/it]    \nExamining conflict for python\
  \ pillow: : 10it [00:43,  2.45s/it]                    \nExamining conflict for\
  \ python pillow goatools: : 10it [00:43,  2.45s/it]\nExamining conflict for python\
  \ pillow goatools: : 11it [00:43,  1.85s/it]\nExamining conflict for matplotlib\
  \ goatools: : 11it [00:44,  1.85s/it]   \nExamining conflict for matplotlib goatools:\
  \ : 12it [00:44,  1.63s/it]\n                                                  \
  \                   \nfailed\nSolving environment: ...working... \nFound conflicts!\
  \ Looking for incompatible packages.\nThis can take several minutes.  Press CTRL-C\
  \ to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ @/linux-64::__linux==5.15.0=0:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ pandas=0.25:  11%|\u2588         | 1/9 [00:00<00:00, 7332.70it/s]        \nExamining\
  \ pillow=6.2:  22%|\u2588\u2588\u258F       | 2/9 [00:02<00:07,  1.14s/it]   \n\
  Examining pillow=6.2:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:02<00:04,  1.32it/s]\n\
  Examining @/linux-64::__unix==0=0:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:02<00:04,\
  \  1.32it/s]\nExamining @/linux-64::__unix==0=0:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [00:02<00:03,  1.39it/s]\nExamining goatools=0.9.7:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:02<00:03,  1.39it/s]         \nExamining @/linux-64::__glibc==2.31=0:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:02,  1.39it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 6/9 [00:03<00:01,  2.20it/s]\nExamining python=3.7:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:03<00:01,  2.20it/s]                \
  \ \nExamining matplotlib=3.1:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [00:05<00:00,  2.20it/s]\nExamining matplotlib=3.1:  89%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:05<00:00,  1.54it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:07<00:00,\
  \  1.54it/s]\nExamining @/linux-64::__archspec==1=x86_64: 100%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:07<00:00,  1.08s/it]\n     \
  \                                                                              \
  \      \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for pandas goatools:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for pandas matplotlib goatools:  11%|\u2588         | 1/9 [00:01<00:09,\
  \  1.25s/it]\nExamining conflict for pandas matplotlib goatools:  22%|\u2588\u2588\
  \u258F       | 2/9 [00:01<00:04,  1.60it/s]\nExamining conflict for pandas pillow\
  \ goatools python matplotlib:  22%|\u2588\u2588\u258F       | 2/9 [00:23<00:04,\
  \  1.60it/s]\nExamining conflict for pandas pillow goatools python matplotlib: \
  \ 33%|\u2588\u2588\u2588\u258E      | 3/9 [00:23<00:57,  9.57s/it]\nExamining conflict\
  \ for pandas matplotlib:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:27<00:57,\
  \  9.57s/it]                       \nExamining conflict for pandas matplotlib: \
  \ 44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:27<00:37,  7.43s/it]\nExamining\
  \ conflict for python pandas pillow matplotlib:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [00:28<00:37,  7.43s/it]\nExamining conflict for python pandas pillow\
  \ matplotlib:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:28<00:20, \
  \ 5.23s/it]\nExamining conflict for pandas pillow goatools matplotlib:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:32<00:20,  5.23s/it]\nExamining conflict\
  \ for pandas pillow goatools matplotlib:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 6/9 [00:32<00:14,  4.88s/it]\nExamining conflict for pandas pillow __glibc\
  \ python matplotlib:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:40<00:14,\
  \  4.88s/it]\nExamining conflict for pandas pillow __glibc python matplotlib:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:40<00:11,  5.96s/it]\nExamining\
  \ conflict for python pandas matplotlib:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:43<00:11,  5.96s/it]               \nExamining conflict\
  \ for python pandas matplotlib:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:43<00:04,  4.97s/it]\nExamining conflict for matplotlib pillow:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:43<00:04,\
  \  4.97s/it]       \nExamining conflict for matplotlib pillow: 100%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:43<00:00,  3.50s/it]\n\
  Examining conflict for python pillow: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588| 9/9 [00:46<00:00,  3.50s/it]    \nExamining conflict for python\
  \ pillow: : 10it [00:46,  3.14s/it]                    \nExamining conflict for\
  \ python pillow goatools: : 10it [00:46,  3.14s/it]\nExamining conflict for python\
  \ pillow goatools: : 11it [00:46,  2.41s/it]\nExamining conflict for matplotlib\
  \ goatools: : 11it [00:48,  2.41s/it]   \nExamining conflict for matplotlib goatools:\
  \ : 12it [00:48,  2.20s/it]\n                                                  \
  \                   \nfailed\n\nUnsatisfiableError: The following specifications\
  \ were found to be incompatible with each other:\n\nOutput in format: Requested\
  \ package -> Available versions\n\nPackage python conflicts for:\npillow=6.2 ->\
  \ olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  goatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  goatools=0.9.7 -> python\npillow=6.2 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  matplotlib=3.1 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  python=3.7\nmatplotlib=3.1 -> pyqt -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.6|>=3.0,<=3.9|>=3.7|>=3.8|>=3']\n\
  \nPackage libpng conflicts for:\nmatplotlib=3.1 -> libpng[version='>=1.6.37,<1.7.0a0']\n\
  matplotlib=3.1 -> matplotlib-base==3.1.1=py36he7580a8_1 -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  \nPackage pypy3.8 conflicts for:\npillow=6.2 -> python[version='>=3.8,<3.9.0a0']\
  \ -> pypy3.8[version='7.3.11.*|7.3.9.*|7.3.8.*']\ngoatools=0.9.7 -> numpy -> pypy3.8[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> pypy3.8[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\n\
  matplotlib=3.1 -> python[version='>=3.8,<3.9.0a0'] -> pypy3.8[version='7.3.11.*|7.3.9.*|7.3.8.*|>=7.3.9|>=7.3.8|>=7.3.11']\n\
  \nPackage ca-certificates conflicts for:\ngoatools=0.9.7 -> python -> ca-certificates\n\
  pillow=6.2 -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\npython=3.7 ->\
  \ openssl[version='>=1.1.1l,<1.1.2a'] -> ca-certificates\n\nPackage _libgcc_mutex\
  \ conflicts for:\npython=3.7 -> libgcc-ng[version='>=9.4.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main|main']\n\
  pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  pandas=0.25 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  matplotlib=3.1 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  \nPackage freetype conflicts for:\nmatplotlib=3.1 -> freetype[version='>=2.9.1,<3.0a0']\n\
  pillow=6.2 -> freetype[version='>=2.9.1,<3.0a0']\n\nPackage pandas conflicts for:\n\
  pandas=0.25\ngoatools=0.9.7 -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\n\
  \nPackage setuptools conflicts for:\npandas=0.25 -> setuptools[version='<60.0.0']\n\
  matplotlib=3.1 -> setuptools\npython=3.7 -> pip -> setuptools\n\nPackage numpy-base\
  \ conflicts for:\nmatplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py27h2b20989_6|py37hdbf6ddf_6|py27hdbf6ddf_6|py36hdbf6ddf_6|py27hdbf6ddf_7|py27h2b20989_7|py36hdbf6ddf_7|py35hdbf6ddf_7|py37hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py36hdbf6ddf_7|py37hdbf6ddf_7|py36hdbf6ddf_8|py37hdbf6ddf_8|py37h2b20989_8|py27hdbf6ddf_8|py35h2b20989_8|py27h7cdd4dd_9|py37h7cdd4dd_9|py36h3dfced4_9|py37h74e8950_9|py27h81de0dd_9|py35h81de0dd_9|py27h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py27h2f8d375_10|py35h2f8d375_10|py27h2f8d375_11|py37hde5b4d6_11|py27hde5b4d6_11|py27hde5b4d6_12|py27h0ea5e3f_1|py35h0ea5e3f_1|py35h9be14a7_1|py27hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_0|py27h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35hdbf6ddf_0|py37h2b20989_1|py27hdbf6ddf_1|py36hdbf6ddf_1|py27h2b20989_1|py36h2b20989_2|py37h2b20989_2|py36h2b20989_3|py37hdbf6ddf_3|py36hdbf6ddf_3|py27h2b20989_4|py37h2b20989_4|py36h2f8d375_0|py37h2f8d375_0|py37h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py35h81de0dd_4|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py36h7cdd4dd_0|py36h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py27h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py36h2f8d375_0|py36h81de0dd_0|py36h2f8d375_1|py36h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py36h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py36h81de0dd_0|py27h81de0dd_0|py27hde5b4d6_0|py37hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_1|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_1|py36h2f8d375_1|py36hde5b4d6_1|py27hde5b4d6_1|py37h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py38hde5b4d6_0|py27hde5b4d6_0|py39hfb011de_1|py39h76555f2_1|py38h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py38hdc34a94_3|py37hdc34a94_3|py37ha8aedfd_4|py39ha8aedfd_4|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py38h73d599e_4|py38h75fe3a5_0|py38hfa32c7d_0|py38h75fe3a5_0|py38hfa32c7d_0|py39h0f7b65f_0|py37h4c65ebe_1|py39h4c65ebe_1|py38h4c65ebe_1|py39h21a3de8_1|py37h34387ca_0|py37h7d8b39e_0|py38h34387ca_0|py38h7d8b39e_0|py39h7d8b39e_0|py38he2ba247_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py37h74d4b33_0|py38h74d4b33_0|py39h39b7dee_0|py39h79a1101_0|py37h2b8c604_0|py39h2b8c604_0|py38h2b8c604_0|py310h79a1101_0|py39hb8be1f0_1|py37hf524024_1|py39hb8be1f0_2|py38hb8be1f0_2|py39hf524024_2|py38hf524024_2|py37h1e6e340_3|py310h375b286_3|py37ha15fc14_3|py39hf524024_0|py38hf524024_0|py39hb8be1f0_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py310hcba007f_0|py38h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_1|py38h31eccc5_1|py310h8e6c178_1|py39h31eccc5_1|py310h8e6c178_0|py38h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py39h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py310h375b286_1|py38h1e6e340_1|py39h31eccc5_0|py38hb8be1f0_0|py310hf2716ce_0|py310h9585f30_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py37hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py310hf2716ce_2|py310h9585f30_1|py310hf2716ce_1|py38hf524024_1|py37hb8be1f0_1|py38hb8be1f0_1|py39hf524024_1|py310h2b8c604_0|py37h79a1101_0|py38h79a1101_0|py37h39b7dee_0|py38h39b7dee_0|py39h74d4b33_0|py39hfae3a4d_0|py38hfae3a4d_0|py39h34387ca_0|py38h21a3de8_1|py37h21a3de8_1|py39h2ae0177_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py37h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py37hfa32c7d_0|py39hdc34a94_3|py37h41b4c56_3|py39h41b4c56_3|py27h2f8d375_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_1|py27h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_1|py37hde5b4d6_1|py27h2f8d375_1|py37h2f8d375_1|py36hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py27h81de0dd_1|py27h2f8d375_1|py35h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py36h74e8950_0|py35h3dfced4_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py27h81de0dd_4|py35h2f8d375_4|py36hde5b4d6_0|py37hde5b4d6_0|py35h2b20989_4|py35hdbf6ddf_4|py36hdbf6ddf_4|py36h2b20989_4|py37hdbf6ddf_4|py27hdbf6ddf_4|py27hdbf6ddf_3|py27h2b20989_3|py37h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27hdbf6ddf_2|py27h2b20989_2|py37hdbf6ddf_1|py36h2b20989_1|py36hdbf6ddf_0|py35h2b20989_0|py36h2b20989_0|py27h2b20989_0|py27h9be14a7_1|py36h9be14a7_1|py36h0ea5e3f_1|py38h2f8d375_12|py38hde5b4d6_12|py36hde5b4d6_12|py37hde5b4d6_12|py36h2f8d375_12|py27h2f8d375_12|py37h2f8d375_12|py36hde5b4d6_11|py36h2f8d375_11|py37h2f8d375_11|py35h81de0dd_10|py36h81de0dd_10|py37h81de0dd_10|py36h74e8950_10|py27h74e8950_10|py35h74e8950_10|py37h74e8950_10|py36h74e8950_9|py35h74e8950_9|py27h74e8950_9|py37h81de0dd_9|py36h81de0dd_9|py27h3dfced4_9|py35h3dfced4_9|py37h3dfced4_9|py35h7cdd4dd_9|py36h7cdd4dd_9|py35hdbf6ddf_8|py27h2b20989_8|py36h2b20989_8|py27hdbf6ddf_7|py36h2b20989_7|py27h2b20989_7|py35h2b20989_7|py36h2b20989_7|py37h2b20989_6|py36h2b20989_6']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py27h0ea5e3f_1|py35h0ea5e3f_1|py35h9be14a7_1|py27hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_0|py27h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35hdbf6ddf_0|py37h2b20989_1|py27hdbf6ddf_1|py36hdbf6ddf_1|py27h2b20989_1|py36h2b20989_2|py37h2b20989_2|py36h2b20989_3|py37hdbf6ddf_3|py36hdbf6ddf_3|py27h2b20989_4|py37h2b20989_4|py36h2f8d375_0|py37h2f8d375_0|py37h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py35h81de0dd_4|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py36h7cdd4dd_0|py36h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py27h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py36h2f8d375_0|py36h81de0dd_0|py36h2f8d375_1|py36h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py36h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py36h81de0dd_0|py27h81de0dd_0|py27hde5b4d6_0|py37hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_1|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_1|py36h2f8d375_1|py36hde5b4d6_1|py27hde5b4d6_1|py37h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py38hde5b4d6_0|py27hde5b4d6_0|py39hfb011de_1|py39h76555f2_1|py38h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py38hdc34a94_3|py37hdc34a94_3|py37ha8aedfd_4|py39ha8aedfd_4|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py38h73d599e_4|py38h75fe3a5_0|py38hfa32c7d_0|py38h75fe3a5_0|py38hfa32c7d_0|py39h0f7b65f_0|py37h4c65ebe_1|py39h4c65ebe_1|py38h4c65ebe_1|py39h21a3de8_1|py37h34387ca_0|py37h7d8b39e_0|py38h34387ca_0|py38h7d8b39e_0|py39h7d8b39e_0|py38he2ba247_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py37h74d4b33_0|py38h74d4b33_0|py39h39b7dee_0|py39h79a1101_0|py37h2b8c604_0|py39h2b8c604_0|py38h2b8c604_0|py310h79a1101_0|py39hb8be1f0_1|py37hf524024_1|py39hb8be1f0_2|py38hb8be1f0_2|py39hf524024_2|py38hf524024_2|py37h1e6e340_3|py310h375b286_3|py37ha15fc14_3|py39hf524024_0|py38hf524024_0|py39hb8be1f0_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py310hcba007f_0|py38h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_1|py38h31eccc5_1|py310h8e6c178_1|py39h31eccc5_1|py310h8e6c178_0|py38h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py39h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py310h375b286_1|py38h1e6e340_1|py39h31eccc5_0|py38hb8be1f0_0|py310hf2716ce_0|py310h9585f30_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py37hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py310hf2716ce_2|py310h9585f30_1|py310hf2716ce_1|py38hf524024_1|py37hb8be1f0_1|py38hb8be1f0_1|py39hf524024_1|py310h2b8c604_0|py37h79a1101_0|py38h79a1101_0|py37h39b7dee_0|py38h39b7dee_0|py39h74d4b33_0|py39hfae3a4d_0|py38hfae3a4d_0|py39h34387ca_0|py38h21a3de8_1|py37h21a3de8_1|py39h2ae0177_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py37h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py37hfa32c7d_0|py39hdc34a94_3|py37h41b4c56_3|py39h41b4c56_3|py27h2f8d375_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_1|py27h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_1|py37hde5b4d6_1|py27h2f8d375_1|py37h2f8d375_1|py36hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py27h81de0dd_1|py27h2f8d375_1|py35h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py36h74e8950_0|py35h3dfced4_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py27h81de0dd_4|py35h2f8d375_4|py36hde5b4d6_0|py37hde5b4d6_0|py35h2b20989_4|py35hdbf6ddf_4|py36hdbf6ddf_4|py36h2b20989_4|py37hdbf6ddf_4|py27hdbf6ddf_4|py27hdbf6ddf_3|py27h2b20989_3|py37h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27hdbf6ddf_2|py27h2b20989_2|py37hdbf6ddf_1|py36h2b20989_1|py36hdbf6ddf_0|py35h2b20989_0|py36h2b20989_0|py27h2b20989_0|py27h9be14a7_1|py36h9be14a7_1|py36h0ea5e3f_1']\n\
  goatools=0.9.7 -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py37h2f8d375_0|py37hde5b4d6_0|py36h2b20989_6|py37h2b20989_6|py37hdbf6ddf_6|py27hdbf6ddf_6|py27hdbf6ddf_7|py36hdbf6ddf_7|py35hdbf6ddf_7|py35h2b20989_7|py37hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_7|py37hdbf6ddf_7|py36hdbf6ddf_8|py37hdbf6ddf_8|py37h2b20989_8|py27h2b20989_8|py27hdbf6ddf_8|py35h2b20989_8|py35h7cdd4dd_9|py27h7cdd4dd_9|py37h7cdd4dd_9|py36h3dfced4_9|py27h74e8950_9|py35h74e8950_9|py37h74e8950_9|py36h74e8950_9|py35h81de0dd_9|py36h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py27h2f8d375_10|py35h2f8d375_10|py27h2f8d375_11|py36h2f8d375_11|py37hde5b4d6_11|py27hde5b4d6_11|py37h2f8d375_12|py27h2f8d375_12|py27hde5b4d6_12|py36hde5b4d6_12|py38h2f8d375_12|py36h0ea5e3f_1|py27h0ea5e3f_1|py27h9be14a7_1|py35h9be14a7_1|py36h2b20989_0|py35h2b20989_0|py36hdbf6ddf_0|py36h2b20989_0|py27h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35hdbf6ddf_0|py27hdbf6ddf_1|py36hdbf6ddf_1|py36h2b20989_2|py37h2b20989_2|py27hdbf6ddf_2|py36h2b20989_3|py37hdbf6ddf_3|py36hdbf6ddf_3|py27hdbf6ddf_3|py27h2b20989_4|py37h2b20989_4|py36h2b20989_4|py37h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py35h81de0dd_4|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py36h7cdd4dd_0|py36h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py27h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py36h2f8d375_0|py36h81de0dd_0|py36h2f8d375_1|py36h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py36h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py36h81de0dd_0|py27h81de0dd_0|py27hde5b4d6_0|py37hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_1|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_1|py36h2f8d375_1|py36hde5b4d6_1|py27hde5b4d6_1|py37h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py38hde5b4d6_0|py27hde5b4d6_0|py39hfb011de_1|py39h76555f2_1|py38h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py38hdc34a94_3|py37hdc34a94_3|py37ha8aedfd_4|py39ha8aedfd_4|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py38h73d599e_4|py38h75fe3a5_0|py38hfa32c7d_0|py38h75fe3a5_0|py38hfa32c7d_0|py39h0f7b65f_0|py37h4c65ebe_1|py39h4c65ebe_1|py38h4c65ebe_1|py39h21a3de8_1|py37h34387ca_0|py37h7d8b39e_0|py38h34387ca_0|py38h7d8b39e_0|py39h7d8b39e_0|py38he2ba247_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py37h74d4b33_0|py38h74d4b33_0|py39h39b7dee_0|py39h79a1101_0|py37h2b8c604_0|py39h2b8c604_0|py38h2b8c604_0|py310h79a1101_0|py39hb8be1f0_1|py37hf524024_1|py39hb8be1f0_2|py38hb8be1f0_2|py39hf524024_2|py38hf524024_2|py37h1e6e340_3|py310h375b286_3|py37ha15fc14_3|py39hf524024_0|py38hf524024_0|py39hb8be1f0_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py310hcba007f_0|py38h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_1|py38h31eccc5_1|py310h8e6c178_1|py39h31eccc5_1|py310h8e6c178_0|py38h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py39h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py310h375b286_1|py38h1e6e340_1|py39h31eccc5_0|py38hb8be1f0_0|py310hf2716ce_0|py310h9585f30_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py37hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py310hf2716ce_2|py310h9585f30_1|py310hf2716ce_1|py38hf524024_1|py37hb8be1f0_1|py38hb8be1f0_1|py39hf524024_1|py310h2b8c604_0|py37h79a1101_0|py38h79a1101_0|py37h39b7dee_0|py38h39b7dee_0|py39h74d4b33_0|py39hfae3a4d_0|py38hfae3a4d_0|py39h34387ca_0|py38h21a3de8_1|py37h21a3de8_1|py39h2ae0177_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py37h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py37hfa32c7d_0|py39hdc34a94_3|py37h41b4c56_3|py39h41b4c56_3|py27h2f8d375_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_1|py27h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_1|py37hde5b4d6_1|py27h2f8d375_1|py37h2f8d375_1|py36hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py27h81de0dd_1|py27h2f8d375_1|py35h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py36h74e8950_0|py35h3dfced4_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py27h81de0dd_4|py35h2f8d375_4|py35h2b20989_4|py35hdbf6ddf_4|py36hdbf6ddf_4|py37hdbf6ddf_4|py27hdbf6ddf_4|py27h2b20989_3|py37h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27h2b20989_2|py27h2b20989_1|py37hdbf6ddf_1|py37h2b20989_1|py36h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_0|py36h9be14a7_1|py35h0ea5e3f_1|py38hde5b4d6_12|py37hde5b4d6_12|py36h2f8d375_12|py36hde5b4d6_11|py37h2f8d375_11|py35h81de0dd_10|py37h81de0dd_10|py27h81de0dd_10|py36h74e8950_10|py27h74e8950_10|py35h74e8950_10|py37h74e8950_10|py27h81de0dd_9|py37h81de0dd_9|py36h81de0dd_9|py27h3dfced4_9|py35h3dfced4_9|py37h3dfced4_9|py36h7cdd4dd_9|py35hdbf6ddf_8|py36h2b20989_8|py36h2b20989_7|py27h2b20989_7|py36h2b20989_7|py27h2b20989_7|py36hdbf6ddf_6|py27h2b20989_6|py36hde5b4d6_0|py36h2f8d375_0']\n\
  \nPackage libgcc conflicts for:\nmatplotlib=3.1 -> pyqt -> libgcc\ngoatools=0.9.7\
  \ -> scipy -> libgcc\n\nPackage pytz conflicts for:\npandas=0.25 -> pytz[version='>=2017.2']\n\
  matplotlib=3.1 -> pytz\n\nPackage expat conflicts for:\npillow=6.2 -> pypy3.6[version='>=7.3.1']\
  \ -> expat[version='>=2.2.9,<3.0.0a0']\npython=3.7 -> pypy3.7=7.3.7 -> expat[version='>=2.2.9,<3.0.0a0|>=2.3.0,<3.0a0|>=2.4.1,<3.0a0']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - matplotlib=3.1 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n \
  \ - python=3.7 -> libgcc-ng[version='>=9.4.0'] -> __glibc[version='>=2.17']\n\n\
  Your installed version is: 2.31\n\nNote that strict channel priority may have removed\
  \ packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fz1dyz98v%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fz1dyz98v%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.288GiB</td>
						<td>1328.4 seconds</td>
						<td>1882.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738220528">
								<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow">SchlossLab/mikropml-snakemake-workflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>681 days</td>
						<td>2023-02-19 06:58:05</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- Snakefile
kind: CalledProcessError
line_no: 31
rest: "Command 'set -euo pipefail;  Rscript --vanilla /home/azureuser/tmp/8p6uk30bag/code/.snakemake/scripts/tmpg13k7t7c.preproc.R'\
  \ returned non-zero exit status 127.\n  File \"/home/azureuser/tmp/8p6uk30bag/code/Snakefile\"\
  , line 31, in __rule_preprocess_data\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-19T065806.385014.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/SchlossLab%252Fmikropml-snakemake-workflow%2Fuxfjgemc%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/SchlossLab%252Fmikropml-snakemake-workflow%2Fuxfjgemc%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.065GiB</td>
						<td>3.1 seconds</td>
						<td>3.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738323456">
								<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling">snakemake-workflows/dna-seq-gatk-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.0.2">v2.0.2</a>
						</td>
						<td>snakemake</td>
						<td>680 days</td>
						<td>2023-02-19 06:00:35</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeInternalError
kind: SnakemakeInternalError
rest: "File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py\"\
  , line 747, in snakemake\n    success = workflow.execute(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py\"\
  , line 926, in execute\n    dag.create_conda_envs(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/dag.py\"\
  , line 338, in create_conda_envs\n    env.create(dryrun)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 397, in create\n    if self.pin_file:\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 109, in pin_file\n    f\".{self.conda.platform}.pin.txt\"\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 102, in conda\n    return Conda(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 667, in __init__\n    shell.check_output(self._get_cmd(f\"conda info --json\"\
  ), text=True)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/shell.py\"\
  , line 63, in check_output\n    return sp.check_output(cmd, shell=True, executable=executable,\
  \ **kwargs)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 421, in check_output\n    return run(*popenargs, stdout=PIPE, timeout=timeout,\
  \ check=True,\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 526, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError:\
  \ Command ' singularity --quiet --silent exec --home /home/azureuser/tmp/ttzyc9ep1l/code/.test\
  \  continuumio/miniconda3:4.8.2 sh -c 'conda info --json'' returned non-zero exit\
  \ status 255."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Fv0spi40l%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Fv0spi40l%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.105GiB</td>
						<td>1.9 seconds</td>
						<td>2.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738323216">
								<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling">snakemake-workflows/dna-seq-gatk-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.0.3">v2.0.3</a>
						</td>
						<td>snakemake</td>
						<td>680 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeInternalError
kind: SnakemakeInternalError
rest: "File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py\"\
  , line 747, in snakemake\n    success = workflow.execute(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py\"\
  , line 926, in execute\n    dag.create_conda_envs(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/dag.py\"\
  , line 338, in create_conda_envs\n    env.create(dryrun)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 397, in create\n    if self.pin_file:\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 109, in pin_file\n    f\".{self.conda.platform}.pin.txt\"\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 102, in conda\n    return Conda(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 667, in __init__\n    shell.check_output(self._get_cmd(f\"conda info --json\"\
  ), text=True)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/shell.py\"\
  , line 63, in check_output\n    return sp.check_output(cmd, shell=True, executable=executable,\
  \ **kwargs)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 421, in check_output\n    return run(*popenargs, stdout=PIPE, timeout=timeout,\
  \ check=True,\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 526, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError:\
  \ Command ' singularity --quiet --silent exec --home /home/azureuser/tmp/cjc4hdcfp3/code/.test\
  \  continuumio/miniconda3:4.8.2 sh -c 'conda info --json'' returned non-zero exit\
  \ status 255."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Fohl98sy4%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Fohl98sy4%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.105GiB</td>
						<td>2.3 seconds</td>
						<td>3.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738322256">
								<a href="https://github.com/franciscozorrilla/metaGEM">franciscozorrilla/metaGEM</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/franciscozorrilla/metaGEM/tree/v1.0.1">v1.0.1</a>
						</td>
						<td>snakemake</td>
						<td>669 days</td>
						<td>2023-02-19 04:29:50</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/franciscozorrilla%252FmetaGEM%2Fnoo4tvw4%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/franciscozorrilla%252FmetaGEM%2Fnoo4tvw4%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.8 seconds</td>
						<td>1.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738322496">
								<a href="https://github.com/franciscozorrilla/metaGEM">franciscozorrilla/metaGEM</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/franciscozorrilla/metaGEM/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>669 days</td>
						<td>2023-02-19 04:27:41</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/franciscozorrilla%252FmetaGEM%2Fzmoi8vuh%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/franciscozorrilla%252FmetaGEM%2Fzmoi8vuh%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>2.8 seconds</td>
						<td>5.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744270512">
								<a href="https://nf-co.re/chipseq">chipseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/chipseq/tree/1.2.2">1.2.2</a>
						</td>
						<td>nextflow</td>
						<td>668 days</td>
						<td>2023-02-19 12:09:29</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2Fm5rjv39r%2Flogs.tar.xz">0.506GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2Fm5rjv39r%2Foutput.tar.xz">0.081GiB</a>
						</td>
						<td>2.536GiB</td>
						<td>3049.3 seconds</td>
						<td>1452.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743051888">
								<a href="https://nf-co.re/diaproteomics">diaproteomics</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/diaproteomics/tree/1.2.3">1.2.3</a>
						</td>
						<td>nextflow</td>
						<td>664 days</td>
						<td>2023-02-19 03:40:08</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2Fwzlipb9u%2Flogs.tar.xz">0.123GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2Fwzlipb9u%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.899GiB</td>
						<td>1199.5 seconds</td>
						<td>1219.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743147792">
								<a href="https://nf-co.re/pgdb">pgdb</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/pgdb/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>663 days</td>
						<td>2023-02-19 03:41:05</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/pgdb%2F00xvawl1%2Flogs.tar.xz">0.326GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/pgdb%2F00xvawl1%2Foutput.tar.xz">0.010GiB</a>
						</td>
						<td>3.074GiB</td>
						<td>310.4 seconds</td>
						<td>305.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744795040">
								<a href="https://nf-co.re/clipseq">clipseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/clipseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>663 days</td>
						<td>2023-02-19 04:17:17</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/clipseq%2Flbcrmf5e%2Flogs.tar.xz">0.572GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/clipseq%2Flbcrmf5e%2Foutput.tar.xz">0.011GiB</a>
						</td>
						<td>2.028GiB</td>
						<td>678.6 seconds</td>
						<td>556.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738321296">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.0.1">v1.0.1</a>
						</td>
						<td>snakemake</td>
						<td>663 days</td>
						<td>2023-02-19 06:27:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/k04w5jl23a/singularity-tmp/tmpjwprveqw.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/62589f3b-5d61-474a-85bb-0862d4d8751a\
  \ singularity  exec --home /home/azureuser/tmp/k04w5jl23a/code/.test  /home/azureuser/tmp/k04w5jl23a/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/k04w5jl23a/code/.test/.snakemake/conda/586838a62130a5b0bd300402f50c35ec_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/k04w5jl23a/code/.test/.snakemake/conda/586838a62130a5b0bd300402f50c35ec_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining htslib[version='>=1.10,<1.11.0a0']:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining libgcc-ng[version='>=7.3.0']:\
  \  11%|\u2588         | 1/9 [00:00<00:00, 14.17it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 17.39it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 26.05it/s]\nExamining perl:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 26.05it/s]            \
  \                \nExamining zlib[version='>=1.2.11,<1.3.0a0']:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:00<00:00, 26.05it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 26.05it/s]  \
  \    \nExamining @/linux-64::__glibc==2.31=0:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:00<00:00, 19.54it/s]\nExamining gsl[version='>=2.5,<2.6.0a0']:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 19.54it/s]\n\
  Examining @/linux-64::__unix==0=0:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:00<00:00, 19.54it/s]      \nExamining @/linux-64::__linux==5.15.0=0:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00,\
  \ 19.54it/s]\n                                                                 \
  \                     \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for htslib libgcc-ng perl zlib gsl:   0%|         \
  \ | 0/9 [00:00<?, ?it/s]\nExamining conflict for htslib zlib:  11%|\u2588      \
  \   | 1/9 [00:00<00:01,  7.92it/s]           \nExamining conflict for htslib zlib:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 15.82it/s]\nExamining conflict\
  \ for htslib libgcc-ng perl zlib __glibc gsl:  22%|\u2588\u2588\u258F       | 2/9\
  \ [00:00<00:00, 15.82it/s]\n                                                   \
  \                                                         \nfailed\nSolving environment:\
  \ ...working... \nFound conflicts! Looking for incompatible packages.\nThis can\
  \ take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining htslib[version='>=1.10,<1.11.0a0']:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining libgcc-ng[version='>=7.3.0']:\
  \  11%|\u2588         | 1/9 [00:00<00:00, 23.61it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 29.25it/s]\nExamining perl:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 43.78it/s]            \
  \                \nExamining zlib[version='>=1.2.11,<1.3.0a0']:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:00<00:00, 33.96it/s]\nExamining zlib[version='>=1.2.11,<1.3.0a0']:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 42.41it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:00<00:00, 42.41it/s]      \nExamining gsl[version='>=2.5,<2.6.0a0']:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 42.41it/s]\n\
  Examining @/linux-64::__unix==0=0:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:00<00:00, 42.41it/s]      \nExamining @/linux-64::__linux==5.15.0=0:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00,\
  \ 42.41it/s]\n                                                                 \
  \                     \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for htslib libgcc-ng perl zlib gsl:   0%|         \
  \ | 0/9 [00:00<?, ?it/s]\nExamining conflict for htslib zlib:  11%|\u2588      \
  \   | 1/9 [00:00<00:00, 12.92it/s]           \nExamining conflict for htslib libgcc-ng\
  \ perl zlib __glibc gsl:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 17.15it/s]\n\
  Examining conflict for htslib libgcc-ng perl zlib __glibc gsl:  33%|\u2588\u2588\
  \u2588\u258E      | 3/9 [00:00<00:00, 25.70it/s]\n                             \
  \                                                                              \
  \ \nfailed\n\nUnsatisfiableError: The following specifications were found to be\
  \ incompatible with each other:\n\nOutput in format: Requested package -> Available\
  \ versions\n\nPackage libgcc-ng conflicts for:\ngsl[version='>=2.5,<2.6.0a0'] ->\
  \ libblas[version='>=3.8.0,<4.0a0'] -> libgcc-ng[version='>=7.5.0|>=9.3.0']\nhtslib[version='>=1.10,<1.11.0a0']\
  \ -> libgcc-ng[version='>=7.3.0|>=7.5.0']\nhtslib[version='>=1.10,<1.11.0a0'] ->\
  \ bzip2[version='>=1.0.8,<2.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=9.3.0|>=9.4.0|>=11.2.0|>=4.9|>=7.2.0']\n\
  gsl[version='>=2.5,<2.6.0a0'] -> libgcc-ng[version='>=7.3.0']\nperl -> libgcc-ng[version='>=11.2.0|>=4.9|>=7.3.0|>=7.5.0|>=9.3.0|>=9.4.0|>=7.2.0']\n\
  zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=7.5.0|>=7.3.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  libgcc-ng[version='>=7.3.0']\n\nPackage zlib conflicts for:\nhtslib[version='>=1.10,<1.11.0a0']\
  \ -> zlib[version='>=1.2.11,<1.3.0a0']\nhtslib[version='>=1.10,<1.11.0a0'] -> libcurl[version='>=7.71.1,<8.0a0']\
  \ -> zlib[version='>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\nzlib[version='>=1.2.11,<1.3.0a0']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - gsl[version='>=2.5,<2.6.0a0'] -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - htslib[version='>=1.10,<1.11.0a0'] -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \  - perl -> libgcc-ng[version='>=9.4.0'] -> __glibc[version='>=2.17']\n  - zlib[version='>=1.2.11,<1.3.0a0']\
  \ -> libgcc-ng[version='>=10.3.0'] -> __glibc[version='>=2.17']\n\nYour installed\
  \ version is: 2.31\n\nNote that strict channel priority may have removed packages\
  \ required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fabj1tsx5%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fabj1tsx5%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.180GiB</td>
						<td>426.8 seconds</td>
						<td>395.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743051648">
								<a href="https://nf-co.re/diaproteomics">diaproteomics</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/diaproteomics/tree/1.2.4">1.2.4</a>
						</td>
						<td>nextflow</td>
						<td>661 days</td>
						<td>2023-02-19 04:37:44</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2F4sboytnx%2Flogs.tar.xz">0.123GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/diaproteomics%2F4sboytnx%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.888GiB</td>
						<td>1207.3 seconds</td>
						<td>808.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738322976">
								<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling">snakemake-workflows/dna-seq-gatk-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.1.0">v2.1.0</a>
						</td>
						<td>snakemake</td>
						<td>659 days</td>
						<td>2023-02-19 03:35:11</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeInternalError
kind: SnakemakeInternalError
rest: "File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py\"\
  , line 747, in snakemake\n    success = workflow.execute(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py\"\
  , line 926, in execute\n    dag.create_conda_envs(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/dag.py\"\
  , line 338, in create_conda_envs\n    env.create(dryrun)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 397, in create\n    if self.pin_file:\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 109, in pin_file\n    f\".{self.conda.platform}.pin.txt\"\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 102, in conda\n    return Conda(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 667, in __init__\n    shell.check_output(self._get_cmd(f\"conda info --json\"\
  ), text=True)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/shell.py\"\
  , line 63, in check_output\n    return sp.check_output(cmd, shell=True, executable=executable,\
  \ **kwargs)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 421, in check_output\n    return run(*popenargs, stdout=PIPE, timeout=timeout,\
  \ check=True,\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 526, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError:\
  \ Command ' singularity --quiet --silent exec --home /home/azureuser/tmp/cmmlwnrp8i/code/.test\
  \  continuumio/miniconda3:4.8.2 sh -c 'conda info --json'' returned non-zero exit\
  \ status 255."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Flydw0atz%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Flydw0atz%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.105GiB</td>
						<td>1.4 seconds</td>
						<td>1.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738322736">
								<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling">snakemake-workflows/dna-seq-gatk-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling/tree/v2.1.1">v2.1.1</a>
						</td>
						<td>snakemake</td>
						<td>658 days</td>
						<td>2023-02-19 04:44:11</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeInternalError
kind: SnakemakeInternalError
rest: "File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/__init__.py\"\
  , line 747, in snakemake\n    success = workflow.execute(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/workflow.py\"\
  , line 926, in execute\n    dag.create_conda_envs(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/dag.py\"\
  , line 338, in create_conda_envs\n    env.create(dryrun)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 397, in create\n    if self.pin_file:\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 109, in pin_file\n    f\".{self.conda.platform}.pin.txt\"\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/common/__init__.py\"\
  , line 213, in __get__\n    value = self.method(instance)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 102, in conda\n    return Conda(\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/deployment/conda.py\"\
  , line 667, in __init__\n    shell.check_output(self._get_cmd(f\"conda info --json\"\
  ), text=True)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/snakemake/shell.py\"\
  , line 63, in check_output\n    return sp.check_output(cmd, shell=True, executable=executable,\
  \ **kwargs)\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 421, in check_output\n    return run(*popenargs, stdout=PIPE, timeout=timeout,\
  \ check=True,\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/subprocess.py\"\
  , line 526, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError:\
  \ Command ' singularity --quiet --silent exec --home /home/azureuser/tmp/o9jpa10ssj/code/.test\
  \  continuumio/miniconda3:4.8.2 sh -c 'conda info --json'' returned non-zero exit\
  \ status 255."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Fbdnod6y3%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-gatk-variant-calling%2Fbdnod6y3%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.105GiB</td>
						<td>2.4 seconds</td>
						<td>5.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738164176">
								<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling">nikostr/dna-seq-deepvariant-glnexus-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling/tree/v0.1.0-alpha">v0.1.0-alpha</a>
						</td>
						<td>snakemake</td>
						<td>657 days</td>
						<td>2023-02-19 04:13:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/6gozzo5ud7/singularity-tmp/tmpv7hpd447.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/6gozzo5ud7/code/.tests/integration/.snakemake/conda/1fa4a3e530aaad0601ef29312c2de5bc_.yaml"
  --prefix "/home/azureuser/tmp/6gozzo5ud7/code/.tests/integration/.snakemake/conda/1fa4a3e530aaad0601ef29312c2de5bc_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working...'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nikostr%252Fdna-seq-deepvariant-glnexus-variant-calling%2F0gkf10x2%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nikostr%252Fdna-seq-deepvariant-glnexus-variant-calling%2F0gkf10x2%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>12.523GiB</td>
						<td>931.1 seconds</td>
						<td>954.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738325376">
								<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2">snakemake-workflows/rna-seq-star-deseq2</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>656 days</td>
						<td>2023-02-19 03:33:54</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/3irn1sc9yv/code/workflow/rules/../envs/deseq2.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/f67daefc-3492-48c6-9cef-05a876f81567\
  \ singularity  exec --home /home/azureuser/tmp/3irn1sc9yv/code/.test  /home/azureuser/tmp/3irn1sc9yv/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/3irn1sc9yv/code/.test/.snakemake/conda/fabc879adf663262b20dbab80952b6e5_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/3irn1sc9yv/code/.test/.snakemake/conda/fabc879adf663262b20dbab80952b6e5_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|\
  \          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64: \
  \ 17%|\u2588\u258B        | 1/6 [00:00<00:00, 4928.68it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 6223.00it/s]     \nExamining\
  \ @/linux-64::__linux==5.15.0=0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 7157.52it/s]\nExamining bioconductor-apeglm=1.12.0:  67%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258B   | 4/6 [00:00<00:00, 7865.55it/s]   \nExamining bioconductor-deseq2=1.30.0:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 63.05it/s]  \n                                                               \
  \                    \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining conflict for bioconductor-apeglm __glibc bioconductor-deseq2:   0%|  \
  \        | 0/6 [00:00<?, ?it/s]\nExamining conflict for bioconductor-apeglm bioconductor-deseq2:\
  \  17%|\u2588\u258B        | 1/6 [00:00<00:04,  1.10it/s]\nExamining conflict for\
  \ bioconductor-apeglm bioconductor-deseq2:  33%|\u2588\u2588\u2588\u258E      |\
  \ 2/6 [00:00<00:01,  2.20it/s]\n                                               \
  \                                                              \nfailed\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|\
  \          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64: \
  \ 17%|\u2588\u258B        | 1/6 [00:00<00:00, 6132.02it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 6775.94it/s]     \nExamining\
  \ @/linux-64::__linux==5.15.0=0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 7285.99it/s]\nExamining bioconductor-apeglm=1.12.0:  67%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258B   | 4/6 [00:00<00:00, 7713.66it/s]   \nExamining bioconductor-deseq2=1.30.0:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 51.42it/s]  \n                                                               \
  \                    \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining conflict for bioconductor-apeglm __glibc bioconductor-deseq2:   0%|  \
  \        | 0/6 [00:00<?, ?it/s]\nExamining conflict for bioconductor-apeglm bioconductor-deseq2:\
  \  17%|\u2588\u258B        | 1/6 [00:00<00:04,  1.03it/s]\nExamining conflict for\
  \ bioconductor-apeglm bioconductor-deseq2:  33%|\u2588\u2588\u2588\u258E      |\
  \ 2/6 [00:00<00:01,  2.05it/s]\n                                               \
  \                                                              \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with each other:\n\n\
  Output in format: Requested package -> Available versions\n\nPackage libgcc-ng conflicts\
  \ for:\nbioconductor-apeglm=1.12.0 -> libblas[version='>=3.8.0,<4.0a0'] -> libgcc-ng[version='7.2.0.*|>=10.3.0|>=12|>=7.3.0|>=9.4.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  bioconductor-apeglm=1.12.0 -> libgcc-ng[version='>=7.5.0|>=9.3.0']\n\nPackage bioconductor-summarizedexperiment\
  \ conflicts for:\nbioconductor-apeglm=1.12.0 -> bioconductor-summarizedexperiment[version='>=1.20.0,<1.21.0']\n\
  bioconductor-deseq2=1.30.0 -> bioconductor-summarizedexperiment[version='>=1.20.0,<1.21.0']\n\
  \nPackage r-rcpp conflicts for:\nbioconductor-apeglm=1.12.0 -> r-rcpp\nbioconductor-apeglm=1.12.0\
  \ -> r-rcppeigen -> r-rcpp[version='>=0.11.0']\n\nPackage libblas conflicts for:\n\
  bioconductor-apeglm=1.12.0 -> libblas[version='>=3.8.0,<4.0a0']\nbioconductor-apeglm=1.12.0\
  \ -> liblapack[version='>=3.8.0,<4.0a0'] -> libblas[version='3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0.*|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0.*|>=3.9.0,<4.0a0',build='5_openblas|6_openblas|6_mkl|7_openblas|7_mkl|9_mkl|10_openblas|10_mkl|11_openblas|12_openblas|13_mkl|14_openblas|14_mkl|16_openblas|16_mkl|18_mkl|20_mkl|0_openblas|3_openblas|4_openblas|5_openblas|6_mkl|7_openblas|8_openblas|9_mkl|10_openblas|10_mkl|11_linux64_openblas|11_linux64_mkl|12_linux64_openblas|12_linux64_mkl|14_linux64_openblas|16_linux64_openblas|16_linux64_mkl|15_linux64_mkl|15_linux64_openblas|14_linux64_mkl|13_linux64_mkl|13_linux64_openblas|9_openblas|8_mkl|7_mkl|6_openblas|5_mkl|4_mkl|2_openblas|1_openblas|21_mkl|19_mkl|17_openblas|15_openblas|15_mkl|13_openblas|12_mkl|11_mkl|9_openblas|8_mkl|8_openblas|5_mkl|4_mkl|4_openblas']\n\
  \nPackage r-base conflicts for:\nbioconductor-apeglm=1.12.0 -> r-emdbook -> r-base[version='3.2.0.*|3.2.1.*|3.2.2.*|3.3.1.*|3.3.2.*|>=4.2,<4.3.0a0|>=4.1,<4.2.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|>=3.5.1,<3.5.2.0a0|>=3.4.1,<3.4.2.0a0|3.4.1.*|>=3.5.0,<3.5.1.0a0|>=3.4.3,<3.4.4.0a0|>=3.4.2,<3.4.3a0']\n\
  bioconductor-apeglm=1.12.0 -> r-base[version='>=4.0,<4.1.0a0']\n\nPackage liblapack\
  \ conflicts for:\nbioconductor-apeglm=1.12.0 -> liblapack[version='>=3.8.0,<4.0a0']\n\
  bioconductor-apeglm=1.12.0 -> bioconductor-genomicranges[version='>=1.42.0,<1.43.0']\
  \ -> liblapack[version='>=3.8.0,<3.9.0a0|>=3.9.0,<4.0a0|>=3.8.0,<4.0.0a0']\n\nPackage\
  \ libstdcxx-ng conflicts for:\nbioconductor-apeglm=1.12.0 -> r-base[version='>=4.0,<4.1.0a0']\
  \ -> libstdcxx-ng[version='7.2.0.*|>=10.3.0|>=12|>=9.4.0|>=7.3.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  bioconductor-apeglm=1.12.0 -> libstdcxx-ng[version='>=7.5.0|>=9.3.0']The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - bioconductor-apeglm=1.12.0 -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17|>=2.17,<3.0.a0']\n\
  \  - bioconductor-deseq2=1.30.0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17|>=2.17,<3.0.a0']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-star-deseq2%2Fu06rpwo9%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-star-deseq2%2Fu06rpwo9%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.148GiB</td>
						<td>377.6 seconds</td>
						<td>584.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743105184">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.3.4">2.3.4</a>
						</td>
						<td>nextflow</td>
						<td>655 days</td>
						<td>2023-02-19 04:10:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 04:10:33.882
kind: NextflowMiscError
process: main
rest: '[nf-core/eager] error: --input was not supplied! Please check ''--help'' or
  documentation under ''running the pipeline'' for details'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F9t9hwjto%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F9t9hwjto%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.646GiB</td>
						<td>32.5 seconds</td>
						<td>15.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738325136">
								<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2">snakemake-workflows/rna-seq-star-deseq2</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.1.1">v1.1.1</a>
						</td>
						<td>snakemake</td>
						<td>654 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/69z64krphr/code/workflow/rules/../envs/deseq2.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/23dbfe68-6802-4d91-a7a9-938687c47627\
  \ singularity  exec --home /home/azureuser/tmp/69z64krphr/code/.test  /home/azureuser/tmp/69z64krphr/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/69z64krphr/code/.test/.snakemake/conda/d7e4cabd3fd24b979da0323035682280_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/69z64krphr/code/.test/.snakemake/conda/d7e4cabd3fd24b979da0323035682280_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:00, 7530.17it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 8525.01it/s]\nExamining\
  \ bioconductor-apeglm=1.12.0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 8811.56it/s] \nExamining bioconductor-deseq2=1.30.0:  67%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258B   | 4/6 [00:00<00:00, 54.33it/s]  \nExamining @/linux-64::__archspec==1=x86_64:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 39.59it/s]\nExamining @/linux-64::__archspec==1=x86_64: 100%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588| 6/6 [00:00<00:00, 47.41it/s]\n     \
  \                                                                              \
  \      \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ conflict for __glibc bioconductor-deseq2 bioconductor-apeglm:   0%|          |\
  \ 0/6 [00:00<?, ?it/s]\nExamining conflict for bioconductor-apeglm bioconductor-deseq2:\
  \  17%|\u2588\u258B        | 1/6 [00:00<00:03,  1.60it/s]\nExamining conflict for\
  \ bioconductor-apeglm bioconductor-deseq2:  33%|\u2588\u2588\u2588\u258E      |\
  \ 2/6 [00:00<00:01,  3.21it/s]\n                                               \
  \                                                              \nfailed\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:00, 6186.29it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 6949.97it/s]\nExamining\
  \ bioconductor-apeglm=1.12.0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 7476.48it/s] \nExamining bioconductor-deseq2=1.30.0:  67%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258B   | 4/6 [00:00<00:00, 50.42it/s]  \nExamining @/linux-64::__archspec==1=x86_64:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 31.26it/s]\nExamining @/linux-64::__archspec==1=x86_64: 100%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588| 6/6 [00:00<00:00, 34.23it/s]\n     \
  \                                                                              \
  \      \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ conflict for __glibc bioconductor-deseq2 bioconductor-apeglm:   0%|          |\
  \ 0/6 [00:00<?, ?it/s]\nExamining conflict for bioconductor-apeglm bioconductor-deseq2:\
  \  17%|\u2588\u258B        | 1/6 [00:00<00:03,  1.35it/s]\nExamining conflict for\
  \ bioconductor-apeglm bioconductor-deseq2:  33%|\u2588\u2588\u2588\u258E      |\
  \ 2/6 [00:00<00:01,  2.69it/s]\n                                               \
  \                                                              \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with each other:\n\n\
  Output in format: Requested package -> Available versions\n\nPackage libblas conflicts\
  \ for:\nbioconductor-apeglm=1.12.0 -> libblas[version='>=3.8.0,<4.0a0']\nbioconductor-apeglm=1.12.0\
  \ -> liblapack[version='>=3.8.0,<4.0a0'] -> libblas[version='3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0.*|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0.*|>=3.9.0,<4.0a0',build='4_mkl|5_mkl|6_openblas|6_mkl|7_openblas|7_mkl|8_openblas|8_mkl|10_openblas|10_mkl|13_openblas|14_openblas|15_mkl|15_openblas|19_mkl|20_mkl|21_mkl|0_openblas|2_openblas|4_openblas|4_mkl|5_mkl|7_openblas|8_openblas|10_openblas|10_mkl|11_linux64_openblas|13_linux64_mkl|14_linux64_openblas|14_linux64_mkl|15_linux64_openblas|16_linux64_openblas|16_linux64_mkl|15_linux64_mkl|13_linux64_openblas|12_linux64_mkl|12_linux64_openblas|11_linux64_mkl|9_mkl|9_openblas|8_mkl|7_mkl|6_mkl|6_openblas|5_openblas|3_openblas|1_openblas|18_mkl|17_openblas|16_mkl|16_openblas|14_mkl|13_mkl|12_mkl|12_openblas|11_mkl|11_openblas|9_mkl|9_openblas|5_openblas|4_openblas']\n\
  \nPackage libstdcxx-ng conflicts for:\nbioconductor-apeglm=1.12.0 -> r-base[version='>=4.0,<4.1.0a0']\
  \ -> libstdcxx-ng[version='7.2.0.*|>=10.3.0|>=12|>=9.4.0|>=7.3.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  bioconductor-apeglm=1.12.0 -> libstdcxx-ng[version='>=7.5.0|>=9.3.0']\n\nPackage\
  \ liblapack conflicts for:\nbioconductor-apeglm=1.12.0 -> bioconductor-genomicranges[version='>=1.42.0,<1.43.0']\
  \ -> liblapack[version='>=3.8.0,<3.9.0a0|>=3.9.0,<4.0a0|>=3.8.0,<4.0.0a0']\nbioconductor-apeglm=1.12.0\
  \ -> liblapack[version='>=3.8.0,<4.0a0']\n\nPackage _libgcc_mutex conflicts for:\n\
  bioconductor-deseq2=1.30.0 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='main|conda_forge|main']\n\
  bioconductor-apeglm=1.12.0 -> libgcc-ng[version='>=9.3.0'] -> _libgcc_mutex[version='*|0.1',build='main|conda_forge|main']\n\
  \nPackage r-rcpp conflicts for:\nbioconductor-apeglm=1.12.0 -> r-rcppeigen -> r-rcpp[version='>=0.11.0']\n\
  bioconductor-apeglm=1.12.0 -> r-rcpp\n\nPackage r-base conflicts for:\nbioconductor-apeglm=1.12.0\
  \ -> r-base[version='>=4.0,<4.1.0a0']\nbioconductor-apeglm=1.12.0 -> r-emdbook ->\
  \ r-base[version='3.2.0.*|3.2.1.*|3.2.2.*|3.3.1.*|3.3.2.*|>=4.2,<4.3.0a0|>=4.1,<4.2.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|>=3.5.1,<3.5.2.0a0|>=3.4.1,<3.4.2.0a0|3.4.1.*|>=3.5.0,<3.5.1.0a0|>=3.4.3,<3.4.4.0a0|>=3.4.2,<3.4.3a0']\n\
  \nPackage libgcc-ng conflicts for:\nbioconductor-apeglm=1.12.0 -> libblas[version='>=3.8.0,<4.0a0']\
  \ -> libgcc-ng[version='7.2.0.*|>=10.3.0|>=12|>=7.3.0|>=9.4.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  bioconductor-apeglm=1.12.0 -> libgcc-ng[version='>=7.5.0|>=9.3.0']The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - bioconductor-apeglm=1.12.0 -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17|>=2.17,<3.0.a0']\n\
  \  - bioconductor-deseq2=1.30.0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17|>=2.17,<3.0.a0']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-star-deseq2%2Fy4e8my0f%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-star-deseq2%2Fy4e8my0f%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.156GiB</td>
						<td>292.7 seconds</td>
						<td>292.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738324896">
								<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2">snakemake-workflows/rna-seq-star-deseq2</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.1.2">v1.1.2</a>
						</td>
						<td>snakemake</td>
						<td>653 days</td>
						<td>2023-02-19 06:00:27</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/c9avnuogdj/code/workflow/rules/../envs/deseq2.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/5abf40ca-a0c4-44f7-9199-60c5430e6952\
  \ singularity  exec --home /home/azureuser/tmp/c9avnuogdj/code/.test  /home/azureuser/tmp/c9avnuogdj/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/c9avnuogdj/code/.test/.snakemake/conda/585b61222ef0fc9b9cecba387335ab77_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/c9avnuogdj/code/.test/.snakemake/conda/585b61222ef0fc9b9cecba387335ab77_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:00, 6177.18it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 6996.34it/s]\nExamining\
  \ bioconductor-deseq2=1.30.0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 7616.77it/s]   \nExamining @/linux-64::__archspec==1=x86_64:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 4/6 [00:00<00:00, 77.82it/s]\nExamining bioconductor-apeglm=1.12.0:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 96.98it/s]      \n                                                           \
  \                        \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for __glibc bioconductor-deseq2 bioconductor-apeglm:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining conflict for bioconductor-deseq2\
  \ bioconductor-apeglm:  17%|\u2588\u258B        | 1/6 [00:00<00:02,  2.44it/s]\n\
  Examining conflict for bioconductor-deseq2 bioconductor-apeglm:  33%|\u2588\u2588\
  \u2588\u258E      | 2/6 [00:00<00:00,  4.89it/s]\n                             \
  \                                                                              \
  \  \nfailed\nSolving environment: ...working... \nFound conflicts! Looking for incompatible\
  \ packages.\nThis can take several minutes.  Press CTRL-C to abort.\n\nBuilding\
  \ graph of deps:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:00, 6944.21it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 7557.30it/s]\nExamining\
  \ bioconductor-deseq2=1.30.0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 8181.35it/s]   \nExamining @/linux-64::__archspec==1=x86_64:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 4/6 [00:00<00:00, 75.59it/s]\nExamining bioconductor-apeglm=1.12.0:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 94.22it/s]      \n                                                           \
  \                        \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for __glibc bioconductor-deseq2 bioconductor-apeglm:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining conflict for bioconductor-deseq2\
  \ bioconductor-apeglm:  17%|\u2588\u258B        | 1/6 [00:00<00:02,  1.99it/s]\n\
  Examining conflict for bioconductor-deseq2 bioconductor-apeglm:  33%|\u2588\u2588\
  \u2588\u258E      | 2/6 [00:00<00:01,  3.98it/s]\n                             \
  \                                                                              \
  \  \nfailed\n\nUnsatisfiableError: The following specifications were found to be\
  \ incompatible with each other:\n\nOutput in format: Requested package -> Available\
  \ versions\n\nPackage libstdcxx-ng conflicts for:\nbioconductor-deseq2=1.30.0 ->\
  \ bioconductor-biocparallel[version='>=1.24.0,<1.25.0'] -> libstdcxx-ng[version='7.2.0.*|>=10.3.0|>=12|>=9.3.0|>=9.4.0|>=7.3.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  bioconductor-deseq2=1.30.0 -> libstdcxx-ng[version='>=7.5.0']\n\nPackage r-base\
  \ conflicts for:\nbioconductor-deseq2=1.30.0 -> r-ggplot2 -> r-base[version='3.2.0.*|3.2.1.*|3.2.2.*|3.3.1.*|3.3.2.*|>=4.2,<4.3.0a0|>=4.1,<4.2.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|>=3.5.1,<3.5.2.0a0|>=3.4.1,<3.4.2.0a0|3.4.1.*|>=3.5.0,<3.5.1.0a0|>=3.4.3,<3.4.4.0a0|>=3.4.2,<3.4.3a0']\n\
  bioconductor-deseq2=1.30.0 -> r-base[version='>=4.0,<4.1.0a0']\n\nPackage libblas\
  \ conflicts for:\nbioconductor-deseq2=1.30.0 -> liblapack[version='>=3.8.0,<4.0a0']\
  \ -> libblas[version='3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0.*|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0.*|>=3.9.0,<4.0a0',build='4_openblas|4_mkl|6_mkl|7_mkl|8_openblas|8_mkl|11_openblas|12_openblas|16_openblas|17_openblas|21_mkl|4_mkl|6_openblas|6_mkl|7_mkl|8_openblas|8_mkl|11_linux64_openblas|13_linux64_openblas|13_linux64_mkl|14_linux64_mkl|15_linux64_openblas|16_linux64_openblas|16_linux64_mkl|15_linux64_mkl|14_linux64_openblas|12_linux64_mkl|12_linux64_openblas|11_linux64_mkl|10_mkl|10_openblas|9_mkl|9_openblas|7_openblas|5_mkl|5_openblas|4_openblas|3_openblas|2_openblas|1_openblas|0_openblas|20_mkl|19_mkl|18_mkl|16_mkl|15_openblas|15_mkl|14_mkl|14_openblas|13_openblas|13_mkl|12_mkl|11_mkl|10_mkl|10_openblas|9_mkl|9_openblas|7_openblas|6_openblas|5_mkl|5_openblas']\n\
  bioconductor-deseq2=1.30.0 -> libblas[version='>=3.8.0,<4.0a0']\n\nPackage r-rcpp\
  \ conflicts for:\nbioconductor-deseq2=1.30.0 -> r-rcpparmadillo -> r-rcpp\nbioconductor-deseq2=1.30.0\
  \ -> r-rcpp[version='>=0.11.0']\n\nPackage libgfortran5 conflicts for:\nbioconductor-apeglm=1.12.0\
  \ -> libblas[version='>=3.8.0,<4.0a0'] -> libgfortran5[version='>=10.3.0|>=10.4.0|>=9.3.0|>=9.4.0']\n\
  bioconductor-deseq2=1.30.0 -> bioconductor-genefilter[version='>=1.72.0,<1.73.0']\
  \ -> libgfortran5[version='>=10.3.0|>=10.4.0|>=9.3.0|>=9.4.0']\n\nPackage libgcc-ng\
  \ conflicts for:\nbioconductor-deseq2=1.30.0 -> bioconductor-biobase[version='>=2.50.0,<2.51.0']\
  \ -> libgcc-ng[version='7.2.0.*|>=10.3.0|>=12|>=7.3.0|>=9.3.0|>=9.4.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  bioconductor-deseq2=1.30.0 -> libgcc-ng[version='>=7.5.0']\n\nPackage liblapack\
  \ conflicts for:\nbioconductor-deseq2=1.30.0 -> liblapack[version='>=3.8.0,<4.0a0']\n\
  bioconductor-deseq2=1.30.0 -> bioconductor-biobase[version='>=2.50.0,<2.51.0'] ->\
  \ liblapack[version='>=3.8.0,<3.9.0a0|>=3.9.0,<4.0a0|>=3.8.0,<4.0.0a0']The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - bioconductor-apeglm=1.12.0 -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17|>=2.17,<3.0.a0']\n\
  \  - bioconductor-deseq2=1.30.0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17|>=2.17,<3.0.a0']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-star-deseq2%2Fd9vtufyg%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-star-deseq2%2Fd9vtufyg%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.090GiB</td>
						<td>249.8 seconds</td>
						<td>246.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738321056">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>653 days</td>
						<td>2023-02-19 04:38:41</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/3sjyifschy/singularity-tmp/tmp81vsz0pi.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/d8011bb3-ec1e-41f1-aac3-a9d8ddb97b84\
  \ singularity  exec --home /home/azureuser/tmp/3sjyifschy/code/.test  /home/azureuser/tmp/3sjyifschy/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/3sjyifschy/code/.test/.snakemake/conda/8735ed571bf54496a9ca15a54b88ccc1_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/3sjyifschy/code/.test/.snakemake/conda/8735ed571bf54496a9ca15a54b88ccc1_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining samtools==1.10:  20%|\u2588\u2588\
  \        | 1/5 [00:00<00:00, 6875.91it/s]        \nExamining @/linux-64::__linux==5.15.0=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 51.62it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00, 77.07it/s]  \n\
  Examining @/linux-64::__unix==0=0:  80%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588  | 4/5 [00:00<00:00, 102.47it/s]   \n                                   \
  \                                              \n\nDetermining conflicts:   0%|\
  \          | 0/5 [00:00<?, ?it/s]\nExamining conflict for samtools __glibc:   0%|\
  \          | 0/5 [00:00<?, ?it/s]\n                                            \
  \                                  \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/5 [00:00<?,\
  \ ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:   0%|          | 0/5 [00:00<?,\
  \ ?it/s]\nExamining samtools==1.10:  20%|\u2588\u2588        | 1/5 [00:00<00:00,\
  \ 6105.25it/s]        \nExamining @/linux-64::__linux==5.15.0=0:  40%|\u2588\u2588\
  \u2588\u2588      | 2/5 [00:00<00:00, 31.84it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00, 47.60it/s]  \n\
  Examining @/linux-64::__unix==0=0:  80%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588  | 4/5 [00:00<00:00, 63.36it/s]    \n                                   \
  \                                             \n\nDetermining conflicts:   0%| \
  \         | 0/5 [00:00<?, ?it/s]\nExamining conflict for samtools __glibc:   0%|\
  \          | 0/5 [00:00<?, ?it/s]\n                                            \
  \                                  \nfailed\n\nUnsatisfiableError: The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - samtools==1.10 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fd2d9jhsd%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fd2d9jhsd%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.143GiB</td>
						<td>121.6 seconds</td>
						<td>234.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743137472">
								<a href="https://nf-co.re/metaboigniter">metaboigniter</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/metaboigniter/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>651 days</td>
						<td>2023-02-19 03:31:44</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/metaboigniter%2Fpgchlzm2%2Flogs.tar.xz">0.388GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/metaboigniter%2Fpgchlzm2%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.969GiB</td>
						<td>2336.8 seconds</td>
						<td>1861.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743138672">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/1.6.1">1.6.1</a>
						</td>
						<td>nextflow</td>
						<td>650 days</td>
						<td>2023-02-19 04:25:07</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: trim_galore --fastqc --gzip SRR389222_sub3.fastq.gz                  --cores
  1
error: 'Proceeding with single-core trimming (user-defined)

  Path to Cutadapt set as: ''cutadapt'' (default)

  Cutadapt seems to be working fine (tested command ''cutadapt --version'')

  Cutadapt version: 3.4

  single-core operation.

  No quality encoding type selected. Assuming that the data provided uses Sanger encoded
  Phred scores (default)


  gzip: SRR389222_sub3.fastq.gz: No such file or directory

  Input file ''SRR389222_sub3.fastq.gz'' seems to be completely empty. Consider respecifying!'
exit_status: 255
kind: CalledProcessError
output: (empty)
process: trim_galore (SRR389222_sub3)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '71'
- 6e48f4bdfeb64dfe69f78904591361

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fdgqz2y8x%2Flogs.tar.xz">0.016GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fdgqz2y8x%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>1.490GiB</td>
						<td>494.6 seconds</td>
						<td>293.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738289248">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.0.3">v2.0.3</a>
						</td>
						<td>snakemake</td>
						<td>649 days</td>
						<td>2023-02-19 06:11:44</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/xhgdr2leo2/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/cc960e4c-afa1-4a87-9b69-5a4780be7b0f\
  \ singularity  exec --home /home/azureuser/tmp/xhgdr2leo2/code/.test  /home/azureuser/tmp/xhgdr2leo2/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/xhgdr2leo2/code/.test/.snakemake/conda/b4d9c233054d7955a514152f2efeb46f_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/xhgdr2leo2/code/.test/.snakemake/conda/b4d9c233054d7955a514152f2efeb46f_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining pillow=6.2:   0%|          |\
  \ 0/9 [00:00<?, ?it/s]  \nExamining matplotlib=3.1:  11%|\u2588         | 1/9 [00:00<00:05,\
  \  1.58it/s]\nExamining matplotlib=3.1:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:02,\
  \  3.15it/s]\nExamining @/linux-64::__archspec==1=x86_64:  22%|\u2588\u2588\u258F\
  \       | 2/9 [00:02<00:02,  3.15it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:02<00:06,  1.06s/it]\nExamining python=3.7:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:02<00:06,  1.06s/it]            \
  \          \nExamining @/linux-64::__glibc==2.31=0:  44%|\u2588\u2588\u2588\u2588\
  \u258D     | 4/9 [00:03<00:05,  1.06s/it]\nExamining @/linux-64::__glibc==2.31=0:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:03,  1.23it/s]\n\
  Examining pandas=0.25:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:03,\
  \  1.23it/s]                \nExamining goatools=0.9.7:  67%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258B   | 6/9 [00:05<00:02,  1.23it/s]\nExamining goatools=0.9.7: \
  \ 78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:05<00:01,  1.29it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:05<00:01,  1.29it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:05<00:00,\
  \  1.55it/s]\nExamining @/linux-64::__unix==0=0:  89%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2589 | 8/9 [00:05<00:00,  1.55it/s]      \n                \
  \                                                                \n\nDetermining\
  \ conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for pillow\
  \ matplotlib python pandas goatools:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for pillow matplotlib:  11%|\u2588         | 1/9 [00:03<00:26,  3.31s/it]\
  \               \nExamining conflict for pillow matplotlib:  22%|\u2588\u2588\u258F\
  \       | 2/9 [00:03<00:11,  1.65s/it]\nExamining conflict for pillow matplotlib\
  \ goatools pandas:  22%|\u2588\u2588\u258F       | 2/9 [00:03<00:11,  1.65s/it]\n\
  Examining conflict for pillow matplotlib goatools pandas:  33%|\u2588\u2588\u2588\
  \u258E      | 3/9 [00:03<00:06,  1.03s/it]\nExamining conflict for pillow matplotlib\
  \ python __glibc pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:07<00:06,\
  \  1.03s/it]\nExamining conflict for pillow matplotlib python __glibc pandas:  44%|\u2588\
  \u2588\u2588\u2588\u258D     | 4/9 [00:07<00:10,  2.03s/it]\nExamining conflict\
  \ for pillow matplotlib python pandas:  44%|\u2588\u2588\u2588\u2588\u258D     |\
  \ 4/9 [00:09<00:10,  2.03s/it]        \nExamining conflict for pillow matplotlib\
  \ python pandas:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:09<00:08,\
  \  2.19s/it]\nExamining conflict for pillow python:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:12<00:08,  2.19s/it]                  \nExamining conflict\
  \ for pillow python:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:12<00:07,\
  \  2.40s/it]\nExamining conflict for pillow goatools python:  67%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258B   | 6/9 [00:12<00:07,  2.40s/it]\nExamining conflict for\
  \ pillow goatools python:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [00:12<00:03,  1.78s/it]\nExamining conflict for pandas matplotlib goatools:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:14<00:03,  1.78s/it]\n\
  Examining conflict for pandas matplotlib goatools:  89%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2589 | 8/9 [00:14<00:01,  1.58s/it]\nExamining conflict\
  \ for pandas matplotlib:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589\
  \ | 8/9 [00:28<00:01,  1.58s/it]         \nExamining conflict for pandas matplotlib:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:28<00:00,\
  \  5.70s/it]\nExamining conflict for matplotlib goatools: 100%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:31<00:00,  5.70s/it]\nExamining\
  \ conflict for matplotlib goatools: : 10it [00:31,  4.65s/it]                  \
  \  \nExamining conflict for pandas matplotlib python: : 10it [00:33,  4.65s/it]\n\
  Examining conflict for pandas matplotlib python: : 11it [00:33,  3.95s/it]\nExamining\
  \ conflict for pandas goatools: : 11it [00:33,  3.95s/it]         \nExamining conflict\
  \ for pandas goatools: : 12it [00:33,  2.80s/it]\n                             \
  \                                    \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining pillow=6.2:   0%|          | 0/9 [00:00<?, ?it/s]  \nExamining\
  \ matplotlib=3.1:  11%|\u2588         | 1/9 [00:00<00:03,  2.48it/s]\nExamining\
  \ matplotlib=3.1:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:01,  4.95it/s]\n\
  Examining @/linux-64::__archspec==1=x86_64:  22%|\u2588\u2588\u258F       | 2/9\
  \ [00:02<00:01,  4.95it/s]\nExamining @/linux-64::__archspec==1=x86_64:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:02<00:04,  1.23it/s]\nExamining python=3.7:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:02<00:04,  1.23it/s]                      \nExamining\
  \ @/linux-64::__glibc==2.31=0:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:03<00:04,\
  \  1.23it/s]\nExamining @/linux-64::__glibc==2.31=0:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:03<00:02,  1.42it/s]\nExamining pandas=0.25:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:02,  1.42it/s]               \
  \ \nExamining goatools=0.9.7:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B  \
  \ | 6/9 [00:04<00:02,  1.42it/s]\nExamining goatools=0.9.7:  78%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258A  | 7/9 [00:04<00:01,  1.42it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:06<00:01,  1.42it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2589 | 8/9 [00:06<00:00,  1.18it/s]\nExamining @/linux-64::__unix==0=0:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:06<00:00,\
  \  1.18it/s]      \n                                                           \
  \                     \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for pillow matplotlib python pandas goatools:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining conflict for pillow matplotlib:  11%|\u2588\
  \         | 1/9 [00:03<00:26,  3.35s/it]               \nExamining conflict for\
  \ pillow matplotlib:  22%|\u2588\u2588\u258F       | 2/9 [00:03<00:11,  1.68s/it]\n\
  Examining conflict for pillow matplotlib goatools pandas:  22%|\u2588\u2588\u258F\
  \       | 2/9 [00:03<00:11,  1.68s/it]\nExamining conflict for pillow matplotlib\
  \ goatools pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:03<00:06,  1.04s/it]\n\
  Examining conflict for pillow matplotlib python __glibc pandas:  33%|\u2588\u2588\
  \u2588\u258E      | 3/9 [00:07<00:06,  1.04s/it]\nExamining conflict for pillow\
  \ matplotlib python __glibc pandas:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9\
  \ [00:07<00:10,  2.10s/it]\nExamining conflict for pillow matplotlib python pandas:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:09<00:10,  2.10s/it]       \
  \ \nExamining conflict for pillow matplotlib python pandas:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:09<00:08,  2.25s/it]\nExamining conflict for pillow\
  \ python:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:12<00:08,  2.25s/it]\
  \                  \nExamining conflict for pillow python:  67%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258B   | 6/9 [00:12<00:07,  2.47s/it]\nExamining conflict for\
  \ pillow goatools python:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9\
  \ [00:13<00:07,  2.47s/it]\nExamining conflict for pillow goatools python:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:13<00:03,  1.83s/it]\nExamining\
  \ conflict for pandas matplotlib goatools:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:14<00:03,  1.83s/it]\nExamining conflict for pandas matplotlib\
  \ goatools:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:14<00:01,\
  \  1.61s/it]\nExamining conflict for pandas matplotlib:  89%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2589 | 8/9 [00:29<00:01,  1.61s/it]         \nExamining\
  \ conflict for pandas matplotlib: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588| 9/9 [00:29<00:00,  5.91s/it]\nExamining conflict for matplotlib\
  \ goatools: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9\
  \ [00:32<00:00,  5.91s/it]\nExamining conflict for matplotlib goatools: : 10it [00:32,\
  \  4.83s/it]                    \nExamining conflict for pandas matplotlib python:\
  \ : 10it [00:34,  4.83s/it]\nExamining conflict for pandas matplotlib python: :\
  \ 11it [00:34,  4.10s/it]\nExamining conflict for pandas goatools: : 11it [00:34,\
  \  4.10s/it]         \nExamining conflict for pandas goatools: : 12it [00:34,  2.90s/it]\n\
  \                                                                 \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with each other:\n\n\
  Output in format: Requested package -> Available versions\n\nPackage pypy3.6 conflicts\
  \ for:\npillow=6.2 -> python[version='>=3.6,<3.7.0a0'] -> pypy3.6[version='7.3.*|7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*']\n\
  pillow=6.2 -> pypy3.6[version='>=7.3.1']\n\nPackage _openmp_mutex conflicts for:\n\
  matplotlib=3.1 -> libgcc-ng[version='>=7.3.0'] -> _openmp_mutex[version='>=4.5']\n\
  python=3.7 -> libgcc-ng[version='>=9.4.0'] -> _openmp_mutex[version='>=4.5']\npandas=0.25\
  \ -> libgcc-ng[version='>=7.3.0'] -> _openmp_mutex[version='>=4.5']\npillow=6.2\
  \ -> libgcc-ng[version='>=7.5.0'] -> _openmp_mutex[version='>=4.5']\n\nPackage pyparsing\
  \ conflicts for:\ngoatools=0.9.7 -> pydot -> pyparsing[version='>=2.1.4|>=2.1.4,<3']\n\
  matplotlib=3.1 -> pyparsing\n\nPackage python conflicts for:\ngoatools=0.9.7 ->\
  \ numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  pandas=0.25 -> python[version='>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']\n\
  pillow=6.2 -> olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  goatools=0.9.7 -> python\nmatplotlib=3.1 -> pyqt -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.6|>=3.0,<=3.9|>=3.7|>=3.8|>=3']\n\
  pillow=6.2 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  matplotlib=3.1 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|>=3.6|>=3|3.4.*']\n\
  python=3.7\n\nPackage ca-certificates conflicts for:\ngoatools=0.9.7 -> python ->\
  \ ca-certificates\npython=3.7 -> openssl[version='>=1.1.1l,<1.1.2a'] -> ca-certificates\n\
  pillow=6.2 -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n\nPackage numpy-base\
  \ conflicts for:\nmatplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py37h2b20989_6|py37hdbf6ddf_6|py27hdbf6ddf_6|py27hdbf6ddf_7|py36hdbf6ddf_7|py37hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_7|py37hdbf6ddf_7|py36hdbf6ddf_8|py37h2b20989_8|py35hdbf6ddf_8|py35h2b20989_8|py35h7cdd4dd_9|py37h3dfced4_9|py36h3dfced4_9|py37h81de0dd_9|py35h74e8950_9|py37h74e8950_9|py35h81de0dd_9|py35h74e8950_10|py27h74e8950_10|py27h81de0dd_10|py36h81de0dd_10|py35h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py35h2f8d375_10|py27h2f8d375_11|py36h2f8d375_11|py37hde5b4d6_11|py27hde5b4d6_11|py37hde5b4d6_12|py36hde5b4d6_12|py38hde5b4d6_12|py38h2f8d375_12|py27h0ea5e3f_1|py35h0ea5e3f_1|py36h9be14a7_1|py27h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py36hdbf6ddf_0|py36h2b20989_0|py36hdbf6ddf_0|py36h2b20989_1|py27h2b20989_1|py37h2b20989_2|py36hdbf6ddf_2|py37h2b20989_3|py27hdbf6ddf_3|py27hdbf6ddf_4|py37hdbf6ddf_4|py36h2b20989_4|py36hdbf6ddf_4|py36h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_4|py35h81de0dd_4|py37h81de0dd_4|py36h81de0dd_4|py38h2f8d375_4|py37hde5b4d6_5|py27h7cdd4dd_0|py37h7cdd4dd_0|py35h7cdd4dd_0|py35h3dfced4_0|py37h3dfced4_0|py27h74e8950_0|py36h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py36h2f8d375_0|py36h81de0dd_0|py36h2f8d375_1|py27h2f8d375_1|py27h81de0dd_1|py36h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py27h2f8d375_0|py36h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_1|py27h2f8d375_1|py36h2f8d375_1|py27hde5b4d6_1|py36hde5b4d6_1|py27h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_1|py37h2f8d375_1|py36h2f8d375_1|py36hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py38h2f8d375_0|py27hde5b4d6_0|py39hfb011de_1|py37hdc34a94_3|py39h73d599e_4|py38h73d599e_4|py37hfa32c7d_0|py38h75fe3a5_0|py38hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py39h2ae0177_0|py38h4c65ebe_1|py39h21a3de8_1|py37h7d8b39e_0|py39h34387ca_0|py38h34387ca_0|py39h7d8b39e_0|py38he2ba247_0|py39he2ba247_0|py37h74d4b33_0|py37h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py37h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py310h79a1101_0|py38hb8be1f0_1|py37hf524024_1|py310hf2716ce_1|py310h9585f30_1|py39hb8be1f0_2|py38hb8be1f0_2|py310h9585f30_2|py39hf524024_2|py38hf524024_2|py310h375b286_3|py310hcba007f_3|py39hb8be1f0_0|py310hf2716ce_0|py38h1e6e340_0|py38ha15fc14_0|py38h31eccc5_0|py38h1e6e340_0|py310h375b286_1|py39h1e6e340_1|py310h8e6c178_1|py39h31eccc5_1|py310h8e6c178_0|py38h1e6e340_0|py38h31eccc5_0|py310h8e6c178_0|py39h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py38h31eccc5_1|py38h1e6e340_1|py310h375b286_0|py39h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py310hcba007f_0|py39ha15fc14_0|py310h375b286_0|py39h1e6e340_0|py38hb8be1f0_0|py38hf524024_0|py310h9585f30_0|py39hf524024_0|py39ha15fc14_3|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py37h1e6e340_3|py37hf524024_2|py37hb8be1f0_2|py310hf2716ce_2|py38hf524024_1|py37hb8be1f0_1|py39hf524024_1|py39hb8be1f0_1|py38h2b8c604_0|py39h2b8c604_0|py39h39b7dee_0|py38h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py37hfae3a4d_0|py38hfae3a4d_0|py38h7d8b39e_0|py37h34387ca_0|py38h21a3de8_1|py37h21a3de8_1|py39h4c65ebe_1|py37h4c65ebe_1|py39h0f7b65f_0|py38hfa32c7d_0|py36hfa32c7d_0|py36hfa32c7d_0|py37h73d599e_4|py38ha8aedfd_4|py39ha8aedfd_4|py37ha8aedfd_4|py39hdc34a94_3|py38hdc34a94_3|py36hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py38h41b4c56_3|py39h41b4c56_3|py39h76555f2_1|py27h2f8d375_0|py38hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_1|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_1|py36h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py37h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py35h74e8950_0|py36h3dfced4_0|py27h3dfced4_0|py36h7cdd4dd_0|py36hde5b4d6_5|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py27h2f8d375_4|py36h2f8d375_4|py27h81de0dd_4|py35h2f8d375_4|py37h2f8d375_0|py35h2b20989_4|py35hdbf6ddf_4|py37h2b20989_4|py27h2b20989_4|py27h2b20989_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py36h2b20989_3|py37hdbf6ddf_2|py27hdbf6ddf_2|py36h2b20989_2|py27h2b20989_2|py37hdbf6ddf_1|py36hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_0|py35hdbf6ddf_0|py27hdbf6ddf_0|py35h2b20989_0|py36h0ea5e3f_1|py27hde5b4d6_12|py36h2f8d375_12|py27h2f8d375_12|py37h2f8d375_12|py36hde5b4d6_11|py37h2f8d375_11|py27h2f8d375_10|py37h81de0dd_10|py36h74e8950_10|py37h74e8950_10|py27h81de0dd_9|py36h74e8950_9|py27h74e8950_9|py36h81de0dd_9|py27h3dfced4_9|py35h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py36h7cdd4dd_9|py27hdbf6ddf_8|py27h2b20989_8|py37hdbf6ddf_8|py36h2b20989_8|py36h2b20989_7|py27h2b20989_7|py35h2b20989_7|py35hdbf6ddf_7|py36h2b20989_7|py27h2b20989_7|py36hdbf6ddf_6|py36h2b20989_6|py27h2b20989_6']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py27h0ea5e3f_1|py35h0ea5e3f_1|py36h9be14a7_1|py27h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py36hdbf6ddf_0|py36h2b20989_0|py36hdbf6ddf_0|py36h2b20989_1|py27h2b20989_1|py37h2b20989_2|py36hdbf6ddf_2|py37h2b20989_3|py27hdbf6ddf_3|py27hdbf6ddf_4|py37hdbf6ddf_4|py36h2b20989_4|py36hdbf6ddf_4|py36h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_4|py35h81de0dd_4|py37h81de0dd_4|py36h81de0dd_4|py38h2f8d375_4|py37hde5b4d6_5|py27h7cdd4dd_0|py37h7cdd4dd_0|py35h7cdd4dd_0|py35h3dfced4_0|py37h3dfced4_0|py27h74e8950_0|py36h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py36h2f8d375_0|py36h81de0dd_0|py36h2f8d375_1|py27h2f8d375_1|py27h81de0dd_1|py36h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py27h2f8d375_0|py36h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_1|py27h2f8d375_1|py36h2f8d375_1|py27hde5b4d6_1|py36hde5b4d6_1|py27h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_1|py37h2f8d375_1|py36h2f8d375_1|py36hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py38h2f8d375_0|py27hde5b4d6_0|py39hfb011de_1|py37hdc34a94_3|py39h73d599e_4|py38h73d599e_4|py37hfa32c7d_0|py38h75fe3a5_0|py38hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py39h2ae0177_0|py38h4c65ebe_1|py39h21a3de8_1|py37h7d8b39e_0|py39h34387ca_0|py38h34387ca_0|py39h7d8b39e_0|py38he2ba247_0|py39he2ba247_0|py37h74d4b33_0|py37h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py37h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py310h79a1101_0|py38hb8be1f0_1|py37hf524024_1|py310hf2716ce_1|py310h9585f30_1|py39hb8be1f0_2|py38hb8be1f0_2|py310h9585f30_2|py39hf524024_2|py38hf524024_2|py310h375b286_3|py310hcba007f_3|py39hb8be1f0_0|py310hf2716ce_0|py38h1e6e340_0|py38ha15fc14_0|py38h31eccc5_0|py38h1e6e340_0|py310h375b286_1|py39h1e6e340_1|py310h8e6c178_1|py39h31eccc5_1|py310h8e6c178_0|py38h1e6e340_0|py38h31eccc5_0|py310h8e6c178_0|py39h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py38h31eccc5_1|py38h1e6e340_1|py310h375b286_0|py39h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py310hcba007f_0|py39ha15fc14_0|py310h375b286_0|py39h1e6e340_0|py38hb8be1f0_0|py38hf524024_0|py310h9585f30_0|py39hf524024_0|py39ha15fc14_3|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py37h1e6e340_3|py37hf524024_2|py37hb8be1f0_2|py310hf2716ce_2|py38hf524024_1|py37hb8be1f0_1|py39hf524024_1|py39hb8be1f0_1|py38h2b8c604_0|py39h2b8c604_0|py39h39b7dee_0|py38h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py37hfae3a4d_0|py38hfae3a4d_0|py38h7d8b39e_0|py37h34387ca_0|py38h21a3de8_1|py37h21a3de8_1|py39h4c65ebe_1|py37h4c65ebe_1|py39h0f7b65f_0|py38hfa32c7d_0|py36hfa32c7d_0|py36hfa32c7d_0|py37h73d599e_4|py38ha8aedfd_4|py39ha8aedfd_4|py37ha8aedfd_4|py39hdc34a94_3|py38hdc34a94_3|py36hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py38h41b4c56_3|py39h41b4c56_3|py39h76555f2_1|py27h2f8d375_0|py38hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_1|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_1|py36h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py37h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py35h74e8950_0|py36h3dfced4_0|py27h3dfced4_0|py36h7cdd4dd_0|py36hde5b4d6_5|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py27h2f8d375_4|py36h2f8d375_4|py27h81de0dd_4|py35h2f8d375_4|py37h2f8d375_0|py35h2b20989_4|py35hdbf6ddf_4|py37h2b20989_4|py27h2b20989_4|py27h2b20989_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py36h2b20989_3|py37hdbf6ddf_2|py27hdbf6ddf_2|py36h2b20989_2|py27h2b20989_2|py37hdbf6ddf_1|py36hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_0|py35hdbf6ddf_0|py27hdbf6ddf_0|py35h2b20989_0|py36h0ea5e3f_1']\n\
  goatools=0.9.7 -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py36h2f8d375_0|py36hde5b4d6_0|py27h2b20989_6|py37h2b20989_6|py37hdbf6ddf_6|py27hdbf6ddf_6|py36hdbf6ddf_6|py27hdbf6ddf_7|py27h2b20989_7|py37hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py27h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_7|py37hdbf6ddf_7|py36hdbf6ddf_8|py37hdbf6ddf_8|py37h2b20989_8|py27h2b20989_8|py35hdbf6ddf_8|py35h2b20989_8|py36h7cdd4dd_9|py36h3dfced4_9|py35h3dfced4_9|py27h3dfced4_9|py36h81de0dd_9|py37h81de0dd_9|py27h74e8950_9|py35h74e8950_9|py36h74e8950_9|py35h74e8950_10|py27h74e8950_10|py36h81de0dd_10|py35h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py35h2f8d375_10|py37h2f8d375_11|py27h2f8d375_11|py36h2f8d375_11|py37hde5b4d6_11|py27hde5b4d6_11|py27h2f8d375_12|py36hde5b4d6_12|py38hde5b4d6_12|py38h2f8d375_12|py27h0ea5e3f_1|py35h0ea5e3f_1|py36h9be14a7_1|py35h9be14a7_1|py36h2b20989_0|py36h2b20989_0|py27h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py37h2b20989_1|py27h2b20989_2|py37h2b20989_2|py27hdbf6ddf_2|py36hdbf6ddf_2|py37h2b20989_3|py36hdbf6ddf_3|py27hdbf6ddf_3|py27hdbf6ddf_4|py37hdbf6ddf_4|py36h2b20989_4|py37h2f8d375_4|py35h81de0dd_4|py37h81de0dd_4|py36h81de0dd_4|py38h2f8d375_4|py37hde5b4d6_5|py27h7cdd4dd_0|py37h7cdd4dd_0|py35h7cdd4dd_0|py35h3dfced4_0|py37h3dfced4_0|py27h74e8950_0|py36h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py36h2f8d375_0|py36h81de0dd_0|py36h2f8d375_1|py27h2f8d375_1|py27h81de0dd_1|py36h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py27h2f8d375_0|py36h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_1|py27h2f8d375_1|py36h2f8d375_1|py27hde5b4d6_1|py36hde5b4d6_1|py27h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_1|py37h2f8d375_1|py36h2f8d375_1|py36hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py38h2f8d375_0|py27hde5b4d6_0|py39hfb011de_1|py37hdc34a94_3|py39h73d599e_4|py38h73d599e_4|py37hfa32c7d_0|py38h75fe3a5_0|py38hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py39h2ae0177_0|py38h4c65ebe_1|py39h21a3de8_1|py37h7d8b39e_0|py39h34387ca_0|py38h34387ca_0|py39h7d8b39e_0|py38he2ba247_0|py39he2ba247_0|py37h74d4b33_0|py37h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py37h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py310h79a1101_0|py38hb8be1f0_1|py37hf524024_1|py310hf2716ce_1|py310h9585f30_1|py39hb8be1f0_2|py38hb8be1f0_2|py310h9585f30_2|py39hf524024_2|py38hf524024_2|py310h375b286_3|py310hcba007f_3|py39hb8be1f0_0|py310hf2716ce_0|py38h1e6e340_0|py38ha15fc14_0|py38h31eccc5_0|py38h1e6e340_0|py310h375b286_1|py39h1e6e340_1|py310h8e6c178_1|py39h31eccc5_1|py310h8e6c178_0|py38h1e6e340_0|py38h31eccc5_0|py310h8e6c178_0|py39h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py38h31eccc5_1|py38h1e6e340_1|py310h375b286_0|py39h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py310hcba007f_0|py39ha15fc14_0|py310h375b286_0|py39h1e6e340_0|py38hb8be1f0_0|py38hf524024_0|py310h9585f30_0|py39hf524024_0|py39ha15fc14_3|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py37h1e6e340_3|py37hf524024_2|py37hb8be1f0_2|py310hf2716ce_2|py38hf524024_1|py37hb8be1f0_1|py39hf524024_1|py39hb8be1f0_1|py38h2b8c604_0|py39h2b8c604_0|py39h39b7dee_0|py38h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py37hfae3a4d_0|py38hfae3a4d_0|py38h7d8b39e_0|py37h34387ca_0|py38h21a3de8_1|py37h21a3de8_1|py39h4c65ebe_1|py37h4c65ebe_1|py39h0f7b65f_0|py38hfa32c7d_0|py36hfa32c7d_0|py36hfa32c7d_0|py37h73d599e_4|py38ha8aedfd_4|py39ha8aedfd_4|py37ha8aedfd_4|py39hdc34a94_3|py38hdc34a94_3|py36hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py38h41b4c56_3|py39h41b4c56_3|py39h76555f2_1|py27h2f8d375_0|py38hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_1|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_1|py36h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py37h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py35h74e8950_0|py36h3dfced4_0|py27h3dfced4_0|py36h7cdd4dd_0|py36hde5b4d6_5|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py27h2f8d375_4|py36h2f8d375_4|py27h81de0dd_4|py35h2f8d375_4|py35h2b20989_4|py35hdbf6ddf_4|py36hdbf6ddf_4|py37h2b20989_4|py27h2b20989_4|py27h2b20989_3|py37hdbf6ddf_3|py36h2b20989_3|py37hdbf6ddf_2|py36h2b20989_2|py27h2b20989_1|py37hdbf6ddf_1|py36hdbf6ddf_1|py27hdbf6ddf_1|py36h2b20989_1|py35hdbf6ddf_0|py35hdbf6ddf_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35h2b20989_0|py27h2b20989_0|py27h9be14a7_1|py36h0ea5e3f_1|py37hde5b4d6_12|py27hde5b4d6_12|py36h2f8d375_12|py37h2f8d375_12|py36hde5b4d6_11|py27h2f8d375_10|py37h81de0dd_10|py27h81de0dd_10|py36h74e8950_10|py37h74e8950_10|py35h81de0dd_9|py27h81de0dd_9|py37h74e8950_9|py37h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py35h7cdd4dd_9|py27hdbf6ddf_8|py36h2b20989_8|py36h2b20989_7|py35h2b20989_7|py35hdbf6ddf_7|py36h2b20989_7|py36hdbf6ddf_7|py36h2b20989_6|py37hde5b4d6_0|py37h2f8d375_0']\n\
  \nPackage freetype conflicts for:\npillow=6.2 -> freetype[version='>=2.9.1,<3.0a0']\n\
  matplotlib=3.1 -> freetype[version='>=2.9.1,<3.0a0']\n\nPackage setuptools conflicts\
  \ for:\npandas=0.25 -> setuptools[version='<60.0.0']\nmatplotlib=3.1 -> setuptools\n\
  python=3.7 -> pip -> setuptools\n\nPackage gdbm conflicts for:\npython=3.7 -> pypy3.7=7.3.7\
  \ -> gdbm[version='>=1.18,<1.19.0a0']\npillow=6.2 -> pypy3.6[version='>=7.3.1']\
  \ -> gdbm[version='>=1.18,<1.19.0a0']\n\nPackage six conflicts for:\npandas=0.25\
  \ -> python-dateutil[version='>=2.6.1'] -> six[version='>=1.5']\nmatplotlib=3.1\
  \ -> cycler[version='>=0.10'] -> six[version='>=1.5']\n\nPackage libuuid conflicts\
  \ for:\ngoatools=0.9.7 -> python -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0|>=2.32.1,<3.0a0']\n\
  pillow=6.2 -> python[version='>=3.8,<3.9.0a0'] -> libuuid[version='>=2.32.1,<3.0a0']\n\
  pandas=0.25 -> python[version='>=3.8,<3.9.0a0'] -> libuuid[version='>=2.32.1,<3.0a0']\n\
  matplotlib=3.1 -> python[version='>=3.8,<3.9.0a0'] -> libuuid[version='>=2.32.1,<3.0a0']\n\
  \nPackage pandas conflicts for:\ngoatools=0.9.7 -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\n\
  pandas=0.25The following specifications were found to be incompatible with your\
  \ system:\n\n  - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - matplotlib=3.1 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - pandas=0.25 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\n\
  Your installed version is: 2.31\n\nNote that strict channel priority may have removed\
  \ packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Frf5ofbpc%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Frf5ofbpc%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.126GiB</td>
						<td>614.0 seconds</td>
						<td>628.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738289008">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.0.4">v2.0.4</a>
						</td>
						<td>snakemake</td>
						<td>649 days</td>
						<td>2023-02-19 06:22:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/r6ritosj95/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/2d2a110f-7873-45a4-8ec4-252573e3be62\
  \ singularity  exec --home /home/azureuser/tmp/r6ritosj95/code/.test  /home/azureuser/tmp/r6ritosj95/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/r6ritosj95/code/.test/.snakemake/conda/40a15d99dffe76b9fe013684b80e0b9f_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/r6ritosj95/code/.test/.snakemake/conda/40a15d99dffe76b9fe013684b80e0b9f_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining pandas=0.25:   0%|          |\
  \ 0/9 [00:00<?, ?it/s] \nExamining matplotlib=3.1:  11%|\u2588         | 1/9 [00:01<00:11,\
  \  1.43s/it]\nExamining matplotlib=3.1:  22%|\u2588\u2588\u258F       | 2/9 [00:01<00:04,\
  \  1.40it/s]\nExamining python=3.7:  22%|\u2588\u2588\u258F       | 2/9 [00:03<00:04,\
  \  1.40it/s]    \nExamining python=3.7:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [00:03<00:06,  1.09s/it]\nExamining @/linux-64::__glibc==2.31=0:  33%|\u2588\u2588\
  \u2588\u258E      | 3/9 [00:04<00:06,  1.09s/it]\nExamining @/linux-64::__glibc==2.31=0:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:04<00:05,  1.11s/it]\nExamining\
  \ @/linux-64::__unix==0=0:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:04<00:05,\
  \  1.11s/it]    \nExamining @/linux-64::__archspec==1=x86_64:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:04<00:04,  1.11s/it]\nExamining pillow=6.2:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:04<00:03,  1.11s/it]          \
  \            \nExamining goatools=0.9.7:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:04<00:02,  1.11s/it]\nExamining goatools=0.9.7:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:04<00:00,  2.37it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2589 | 8/9 [00:04<00:00,  2.37it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:04<00:00,\
  \  2.63it/s]\n                                                                 \
  \                     \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for pandas goatools:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for pandas matplotlib python pillow goatools:  11%|\u2588\
  \         | 1/9 [00:00<00:06,  1.23it/s]\nExamining conflict for pandas matplotlib\
  \ python pillow goatools:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:02,  2.47it/s]\n\
  Examining conflict for matplotlib pandas goatools:  22%|\u2588\u2588\u258F     \
  \  | 2/9 [00:03<00:02,  2.47it/s]              \nExamining conflict for matplotlib\
  \ pandas goatools:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:03<00:07,  1.28s/it]\n\
  Examining conflict for matplotlib pandas pillow python:  33%|\u2588\u2588\u2588\u258E\
  \      | 3/9 [00:30<00:07,  1.28s/it]\nExamining conflict for matplotlib pandas\
  \ pillow python:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:30<00:55, 11.08s/it]\n\
  Examining conflict for matplotlib pandas pillow goatools:  44%|\u2588\u2588\u2588\
  \u2588\u258D     | 4/9 [00:34<00:55, 11.08s/it]\nExamining conflict for matplotlib\
  \ pandas pillow goatools:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:34<00:34,\
  \  8.63s/it]\nExamining conflict for matplotlib pandas:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:42<00:34,  8.63s/it]                \nExamining conflict\
  \ for matplotlib pandas:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9\
  \ [00:42<00:24,  8.33s/it]\nExamining conflict for pandas matplotlib python __glibc\
  \ pillow:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:44<00:24,\
  \  8.33s/it]\nExamining conflict for pandas matplotlib python __glibc pillow:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:44<00:12,  6.15s/it]\nExamining\
  \ conflict for matplotlib pandas python:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:46<00:12,  6.15s/it]               \nExamining conflict\
  \ for matplotlib pandas python:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:46<00:04,  4.79s/it]\nExamining conflict for matplotlib pillow:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:46<00:04,\
  \  4.79s/it]       \nExamining conflict for matplotlib pillow: 100%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:46<00:00,  3.36s/it]\n\
  Examining conflict for matplotlib goatools: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588| 9/9 [00:47<00:00,  3.36s/it]\nExamining conflict for matplotlib\
  \ goatools: : 10it [00:47,  2.79s/it]                    \nExamining conflict for\
  \ pillow python: : 10it [00:50,  2.79s/it]      \nExamining conflict for pillow\
  \ python: : 11it [00:50,  2.65s/it]\nExamining conflict for goatools pillow python:\
  \ : 11it [00:50,  2.65s/it]\nExamining conflict for goatools pillow python: : 12it\
  \ [00:50,  1.98s/it]\n                                                         \
  \               \nfailed\nSolving environment: ...working... \nFound conflicts!\
  \ Looking for incompatible packages.\nThis can take several minutes.  Press CTRL-C\
  \ to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ pandas=0.25:   0%|          | 0/9 [00:00<?, ?it/s] \nExamining matplotlib=3.1:\
  \  11%|\u2588         | 1/9 [00:01<00:14,  1.83s/it]\nExamining matplotlib=3.1:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:01<00:06,  1.09it/s]\nExamining python=3.7:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:03<00:06,  1.09it/s]    \nExamining python=3.7:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:03<00:08,  1.38s/it]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:05<00:08,  1.38s/it]\nExamining @/linux-64::__glibc==2.31=0:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:05<00:07,  1.43s/it]\nExamining\
  \ @/linux-64::__unix==0=0:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:05<00:07,\
  \  1.43s/it]    \nExamining @/linux-64::__archspec==1=x86_64:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:05<00:05,  1.43s/it]\nExamining pillow=6.2:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:05<00:04,  1.43s/it]          \
  \            \nExamining goatools=0.9.7:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:05<00:02,  1.43s/it]\nExamining goatools=0.9.7:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:05<00:00,  1.83it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2589 | 8/9 [00:06<00:00,  1.83it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:06<00:00,\
  \  2.04it/s]\n                                                                 \
  \                     \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for pandas goatools:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for pandas matplotlib python pillow goatools:  11%|\u2588\
  \         | 1/9 [00:00<00:07,  1.02it/s]\nExamining conflict for pandas matplotlib\
  \ python pillow goatools:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:03,  2.05it/s]\n\
  Examining conflict for matplotlib pandas goatools:  22%|\u2588\u2588\u258F     \
  \  | 2/9 [00:04<00:03,  2.05it/s]              \nExamining conflict for matplotlib\
  \ pandas goatools:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:04<00:09,  1.58s/it]\n\
  Examining conflict for matplotlib pandas pillow python:  33%|\u2588\u2588\u2588\u258E\
  \      | 3/9 [00:36<00:09,  1.58s/it]\nExamining conflict for matplotlib pandas\
  \ pillow python:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:36<01:04, 12.91s/it]\n\
  Examining conflict for matplotlib pandas pillow goatools:  44%|\u2588\u2588\u2588\
  \u2588\u258D     | 4/9 [00:40<01:04, 12.91s/it]\nExamining conflict for matplotlib\
  \ pandas pillow goatools:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:40<00:39,\
  \  9.89s/it]\nExamining conflict for matplotlib pandas:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:48<00:39,  9.89s/it]                \nExamining conflict\
  \ for matplotlib pandas:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9\
  \ [00:48<00:27,  9.27s/it]\nExamining conflict for pandas matplotlib python __glibc\
  \ pillow:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:49<00:27,\
  \  9.27s/it]\nExamining conflict for pandas matplotlib python __glibc pillow:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:49<00:13,  6.79s/it]\nExamining\
  \ conflict for matplotlib pandas python:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:52<00:13,  6.79s/it]               \nExamining conflict\
  \ for matplotlib pandas python:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:52<00:05,  5.56s/it]\nExamining conflict for matplotlib pillow:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:52<00:05,\
  \  5.56s/it]       \nExamining conflict for matplotlib pillow: 100%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:52<00:00,  3.92s/it]\n\
  Examining conflict for matplotlib goatools: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588| 9/9 [00:56<00:00,  3.92s/it]\nExamining conflict for matplotlib\
  \ goatools: : 10it [00:56,  3.66s/it]                    \nExamining conflict for\
  \ pillow python: : 10it [01:00,  3.66s/it]      \nExamining conflict for pillow\
  \ python: : 11it [01:00,  3.99s/it]\nExamining conflict for goatools pillow python:\
  \ : 11it [01:01,  3.99s/it]\nExamining conflict for goatools pillow python: : 12it\
  \ [01:01,  3.05s/it]\n                                                         \
  \               \nfailed\n\nUnsatisfiableError: The following specifications were\
  \ found to be incompatible with each other:\n\nOutput in format: Requested package\
  \ -> Available versions\n\nPackage libuuid conflicts for:\npandas=0.25 -> python[version='>=3.8,<3.9.0a0']\
  \ -> libuuid[version='>=2.32.1,<3.0a0']\ngoatools=0.9.7 -> python -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0|>=2.32.1,<3.0a0']\n\
  matplotlib=3.1 -> python[version='>=3.8,<3.9.0a0'] -> libuuid[version='>=2.32.1,<3.0a0']\n\
  pillow=6.2 -> python[version='>=3.8,<3.9.0a0'] -> libuuid[version='>=2.32.1,<3.0a0']\n\
  \nPackage pyparsing conflicts for:\nmatplotlib=3.1 -> pyparsing\ngoatools=0.9.7\
  \ -> pydot -> pyparsing[version='>=2.1.4|>=2.1.4,<3']\n\nPackage numpy-base conflicts\
  \ for:\nmatplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py36h2b20989_6|py37h2b20989_6|py37hdbf6ddf_6|py27h2b20989_7|py36hdbf6ddf_7|py35h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_7|py36hdbf6ddf_8|py36h2b20989_8|py35hdbf6ddf_8|py36h7cdd4dd_9|py37h7cdd4dd_9|py36h3dfced4_9|py35h74e8950_9|py37h74e8950_9|py27h81de0dd_9|py35h81de0dd_9|py35h74e8950_10|py36h74e8950_10|py36h81de0dd_10|py36h2f8d375_10|py27h2f8d375_11|py36h2f8d375_11|py27hde5b4d6_11|py27h2f8d375_12|py36h2f8d375_12|py37hde5b4d6_12|py36h0ea5e3f_1|py27h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py36h2b20989_0|py36hdbf6ddf_0|py36h2b20989_1|py37hdbf6ddf_1|py36h2b20989_2|py37h2b20989_2|py37hdbf6ddf_2|py37h2b20989_3|py27hdbf6ddf_3|py27h2b20989_4|py37hdbf6ddf_4|py36hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_4|py27h2f8d375_4|py36h81de0dd_4|py38h2f8d375_4|py38hde5b4d6_4|py37hde5b4d6_5|py36hde5b4d6_5|py27h7cdd4dd_0|py36h7cdd4dd_0|py37h7cdd4dd_0|py35h7cdd4dd_0|py27h3dfced4_0|py35h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py36h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py27h2f8d375_1|py37h81de0dd_1|py27h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py36h81de0dd_0|py27h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_1|py36h2f8d375_1|py37hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_1|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_0|py38h2f8d375_0|py38hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py39h41b4c56_3|py38h41b4c56_3|py36hdc34a94_3|py37hdc34a94_3|py37ha8aedfd_4|py39ha8aedfd_4|py37hfa32c7d_0|py38hfa32c7d_0|py36hfa32c7d_0|py37hfa32c7d_0|py37h75fe3a5_0|py37h4c65ebe_1|py38h4c65ebe_1|py37h21a3de8_1|py39h21a3de8_1|py38h21a3de8_1|py37h7d8b39e_0|py39h34387ca_0|py38h34387ca_0|py39h7d8b39e_0|py39he2ba247_0|py37hfae3a4d_0|py37h74d4b33_0|py39h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py37h2b8c604_0|py38h2b8c604_0|py310h2b8c604_0|py39hb8be1f0_1|py39hf524024_1|py38hf524024_1|py310h9585f30_1|py310hf2716ce_2|py37hb8be1f0_2|py310h9585f30_2|py38h1e6e340_3|py310h375b286_3|py38ha15fc14_3|py39ha15fc14_3|py39hf524024_0|py310h9585f30_0|py38hf524024_0|py39hb8be1f0_0|py310hf2716ce_0|py39h1e6e340_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py39h1e6e340_0|py39h1e6e340_1|py310h8e6c178_1|py39h31eccc5_1|py38h31eccc5_0|py39h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h8e6c178_0|py38h31eccc5_1|py310h375b286_1|py38h1e6e340_1|py310h375b286_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py38h1e6e340_0|py38hb8be1f0_0|py310hcba007f_3|py37ha15fc14_3|py39h1e6e340_3|py37h1e6e340_3|py38hf524024_2|py37hf524024_2|py39hf524024_2|py38hb8be1f0_2|py39hb8be1f0_2|py310hf2716ce_1|py37hf524024_1|py37hb8be1f0_1|py38hb8be1f0_1|py310h79a1101_0|py37h79a1101_0|py39h2b8c604_0|py37h39b7dee_0|py38h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py38hfae3a4d_0|py38he2ba247_0|py38h7d8b39e_0|py37h34387ca_0|py39h4c65ebe_1|py39h0f7b65f_0|py39h2ae0177_0|py38hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py38h75fe3a5_0|py38h73d599e_4|py39h73d599e_4|py37h73d599e_4|py38ha8aedfd_4|py39hdc34a94_3|py38hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_1|py27hde5b4d6_1|py37h2f8d375_1|py37hde5b4d6_0|py37hde5b4d6_0|py37h81de0dd_0|py37h2f8d375_0|py27h81de0dd_0|py37h2f8d375_0|py37h2f8d375_1|py36h81de0dd_1|py27h81de0dd_1|py36h2f8d375_1|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py27h74e8950_0|py36h3dfced4_0|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py37h81de0dd_4|py35h81de0dd_4|py36h2f8d375_4|py27h81de0dd_4|py35h2f8d375_4|py37hde5b4d6_0|py37h2f8d375_0|py35hdbf6ddf_4|py36h2b20989_4|py37h2b20989_4|py27hdbf6ddf_4|py27h2b20989_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py36h2b20989_3|py36hdbf6ddf_2|py27hdbf6ddf_2|py27h2b20989_2|py27h2b20989_1|py36hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_0|py35hdbf6ddf_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35h2b20989_0|py36h9be14a7_1|py35h0ea5e3f_1|py27h0ea5e3f_1|py38h2f8d375_12|py38hde5b4d6_12|py36hde5b4d6_12|py27hde5b4d6_12|py37h2f8d375_12|py37hde5b4d6_11|py36hde5b4d6_11|py37h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py37h2f8d375_10|py35h81de0dd_10|py37h81de0dd_10|py27h81de0dd_10|py27h74e8950_10|py37h74e8950_10|py36h74e8950_9|py27h74e8950_9|py37h81de0dd_9|py36h81de0dd_9|py27h3dfced4_9|py35h3dfced4_9|py37h3dfced4_9|py27h7cdd4dd_9|py35h7cdd4dd_9|py35h2b20989_8|py27hdbf6ddf_8|py27h2b20989_8|py37h2b20989_8|py37hdbf6ddf_8|py37hdbf6ddf_7|py36h2b20989_7|py27h2b20989_7|py37h2b20989_7|py37h2b20989_7|py37hdbf6ddf_7|py35hdbf6ddf_7|py36h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_6|py27hdbf6ddf_6|py27h2b20989_6']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py36h0ea5e3f_1|py27h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py36h2b20989_0|py36hdbf6ddf_0|py36h2b20989_1|py37hdbf6ddf_1|py36h2b20989_2|py37h2b20989_2|py37hdbf6ddf_2|py37h2b20989_3|py27hdbf6ddf_3|py27h2b20989_4|py37hdbf6ddf_4|py36hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_4|py27h2f8d375_4|py36h81de0dd_4|py38h2f8d375_4|py38hde5b4d6_4|py37hde5b4d6_5|py36hde5b4d6_5|py27h7cdd4dd_0|py36h7cdd4dd_0|py37h7cdd4dd_0|py35h7cdd4dd_0|py27h3dfced4_0|py35h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py36h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py27h2f8d375_1|py37h81de0dd_1|py27h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py36h81de0dd_0|py27h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_1|py36h2f8d375_1|py37hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_1|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_0|py38h2f8d375_0|py38hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py39h41b4c56_3|py38h41b4c56_3|py36hdc34a94_3|py37hdc34a94_3|py37ha8aedfd_4|py39ha8aedfd_4|py37hfa32c7d_0|py38hfa32c7d_0|py36hfa32c7d_0|py37hfa32c7d_0|py37h75fe3a5_0|py37h4c65ebe_1|py38h4c65ebe_1|py37h21a3de8_1|py39h21a3de8_1|py38h21a3de8_1|py37h7d8b39e_0|py39h34387ca_0|py38h34387ca_0|py39h7d8b39e_0|py39he2ba247_0|py37hfae3a4d_0|py37h74d4b33_0|py39h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py37h2b8c604_0|py38h2b8c604_0|py310h2b8c604_0|py39hb8be1f0_1|py39hf524024_1|py38hf524024_1|py310h9585f30_1|py310hf2716ce_2|py37hb8be1f0_2|py310h9585f30_2|py38h1e6e340_3|py310h375b286_3|py38ha15fc14_3|py39ha15fc14_3|py39hf524024_0|py310h9585f30_0|py38hf524024_0|py39hb8be1f0_0|py310hf2716ce_0|py39h1e6e340_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py39h1e6e340_0|py39h1e6e340_1|py310h8e6c178_1|py39h31eccc5_1|py38h31eccc5_0|py39h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h8e6c178_0|py38h31eccc5_1|py310h375b286_1|py38h1e6e340_1|py310h375b286_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py38h1e6e340_0|py38hb8be1f0_0|py310hcba007f_3|py37ha15fc14_3|py39h1e6e340_3|py37h1e6e340_3|py38hf524024_2|py37hf524024_2|py39hf524024_2|py38hb8be1f0_2|py39hb8be1f0_2|py310hf2716ce_1|py37hf524024_1|py37hb8be1f0_1|py38hb8be1f0_1|py310h79a1101_0|py37h79a1101_0|py39h2b8c604_0|py37h39b7dee_0|py38h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py38hfae3a4d_0|py38he2ba247_0|py38h7d8b39e_0|py37h34387ca_0|py39h4c65ebe_1|py39h0f7b65f_0|py39h2ae0177_0|py38hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py38h75fe3a5_0|py38h73d599e_4|py39h73d599e_4|py37h73d599e_4|py38ha8aedfd_4|py39hdc34a94_3|py38hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_1|py27hde5b4d6_1|py37h2f8d375_1|py37hde5b4d6_0|py37hde5b4d6_0|py37h81de0dd_0|py37h2f8d375_0|py27h81de0dd_0|py37h2f8d375_0|py37h2f8d375_1|py36h81de0dd_1|py27h81de0dd_1|py36h2f8d375_1|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py27h74e8950_0|py36h3dfced4_0|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py37h81de0dd_4|py35h81de0dd_4|py36h2f8d375_4|py27h81de0dd_4|py35h2f8d375_4|py37hde5b4d6_0|py37h2f8d375_0|py35hdbf6ddf_4|py36h2b20989_4|py37h2b20989_4|py27hdbf6ddf_4|py27h2b20989_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py36h2b20989_3|py36hdbf6ddf_2|py27hdbf6ddf_2|py27h2b20989_2|py27h2b20989_1|py36hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_0|py35hdbf6ddf_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35h2b20989_0|py36h9be14a7_1|py35h0ea5e3f_1|py27h0ea5e3f_1']\n\
  goatools=0.9.7 -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py36h2f8d375_0|py36hde5b4d6_0|py36h2b20989_6|py37hdbf6ddf_6|py27hdbf6ddf_7|py27h2b20989_7|py27h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_7|py27h2b20989_8|py35hdbf6ddf_8|py36h7cdd4dd_9|py37h7cdd4dd_9|py36h3dfced4_9|py35h74e8950_9|py37h74e8950_9|py27h81de0dd_9|py35h81de0dd_9|py37h74e8950_10|py35h74e8950_10|py36h74e8950_10|py35h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py27h2f8d375_10|py27h2f8d375_11|py36h2f8d375_11|py27hde5b4d6_11|py27h2f8d375_12|py36h2f8d375_12|py38h2f8d375_12|py36h0ea5e3f_1|py35h0ea5e3f_1|py35h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py36hdbf6ddf_0|py36h2b20989_0|py36hdbf6ddf_0|py36h2b20989_1|py27hdbf6ddf_1|py36hdbf6ddf_1|py37hdbf6ddf_1|py37h2b20989_2|py37h2b20989_3|py36hdbf6ddf_3|py27hdbf6ddf_3|py27h2b20989_4|py27hdbf6ddf_4|py37h2b20989_4|py37hdbf6ddf_4|py36hdbf6ddf_4|py35h2b20989_4|py37h2f8d375_4|py27h2f8d375_4|py36h81de0dd_4|py38h2f8d375_4|py38hde5b4d6_4|py37hde5b4d6_5|py36hde5b4d6_5|py27h7cdd4dd_0|py36h7cdd4dd_0|py37h7cdd4dd_0|py35h7cdd4dd_0|py27h3dfced4_0|py35h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py36h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py27h2f8d375_1|py37h81de0dd_1|py27h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py36h81de0dd_0|py27h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_1|py36h2f8d375_1|py37hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_1|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_0|py38h2f8d375_0|py38hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py39h41b4c56_3|py38h41b4c56_3|py36hdc34a94_3|py37hdc34a94_3|py37ha8aedfd_4|py39ha8aedfd_4|py37hfa32c7d_0|py38hfa32c7d_0|py36hfa32c7d_0|py37hfa32c7d_0|py37h75fe3a5_0|py37h4c65ebe_1|py38h4c65ebe_1|py37h21a3de8_1|py39h21a3de8_1|py38h21a3de8_1|py37h7d8b39e_0|py39h34387ca_0|py38h34387ca_0|py39h7d8b39e_0|py39he2ba247_0|py37hfae3a4d_0|py37h74d4b33_0|py39h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py37h2b8c604_0|py38h2b8c604_0|py310h2b8c604_0|py39hb8be1f0_1|py39hf524024_1|py38hf524024_1|py310h9585f30_1|py310hf2716ce_2|py37hb8be1f0_2|py310h9585f30_2|py38h1e6e340_3|py310h375b286_3|py38ha15fc14_3|py39ha15fc14_3|py39hf524024_0|py310h9585f30_0|py38hf524024_0|py39hb8be1f0_0|py310hf2716ce_0|py39h1e6e340_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py39h1e6e340_0|py39h1e6e340_1|py310h8e6c178_1|py39h31eccc5_1|py38h31eccc5_0|py39h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h8e6c178_0|py38h31eccc5_1|py310h375b286_1|py38h1e6e340_1|py310h375b286_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py38h1e6e340_0|py38hb8be1f0_0|py310hcba007f_3|py37ha15fc14_3|py39h1e6e340_3|py37h1e6e340_3|py38hf524024_2|py37hf524024_2|py39hf524024_2|py38hb8be1f0_2|py39hb8be1f0_2|py310hf2716ce_1|py37hf524024_1|py37hb8be1f0_1|py38hb8be1f0_1|py310h79a1101_0|py37h79a1101_0|py39h2b8c604_0|py37h39b7dee_0|py38h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py39hfae3a4d_0|py37he2ba247_0|py38hfae3a4d_0|py38he2ba247_0|py38h7d8b39e_0|py37h34387ca_0|py39h4c65ebe_1|py39h0f7b65f_0|py39h2ae0177_0|py38hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py38h75fe3a5_0|py38h73d599e_4|py39h73d599e_4|py37h73d599e_4|py38ha8aedfd_4|py39hdc34a94_3|py38hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_1|py27hde5b4d6_1|py37h2f8d375_1|py37hde5b4d6_0|py37hde5b4d6_0|py37h81de0dd_0|py37h2f8d375_0|py27h81de0dd_0|py37h2f8d375_0|py37h2f8d375_1|py36h81de0dd_1|py27h81de0dd_1|py36h2f8d375_1|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py27h74e8950_0|py36h3dfced4_0|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py37h81de0dd_4|py35h81de0dd_4|py36h2f8d375_4|py27h81de0dd_4|py35h2f8d375_4|py35hdbf6ddf_4|py36h2b20989_4|py27h2b20989_3|py37hdbf6ddf_3|py36h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27hdbf6ddf_2|py36h2b20989_2|py27h2b20989_2|py27h2b20989_1|py37h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_0|py35hdbf6ddf_0|py27hdbf6ddf_0|py35h2b20989_0|py27h9be14a7_1|py36h9be14a7_1|py27h0ea5e3f_1|py38hde5b4d6_12|py36hde5b4d6_12|py37hde5b4d6_12|py27hde5b4d6_12|py37h2f8d375_12|py37hde5b4d6_11|py36hde5b4d6_11|py37h2f8d375_11|py35h2f8d375_10|py36h81de0dd_10|py37h81de0dd_10|py27h81de0dd_10|py27h74e8950_10|py36h74e8950_9|py27h74e8950_9|py37h81de0dd_9|py36h81de0dd_9|py27h3dfced4_9|py35h3dfced4_9|py37h3dfced4_9|py27h7cdd4dd_9|py35h7cdd4dd_9|py35h2b20989_8|py27hdbf6ddf_8|py37h2b20989_8|py37hdbf6ddf_8|py36h2b20989_8|py36hdbf6ddf_8|py37hdbf6ddf_7|py36h2b20989_7|py37h2b20989_7|py37h2b20989_7|py37hdbf6ddf_7|py35h2b20989_7|py35hdbf6ddf_7|py36h2b20989_7|py36hdbf6ddf_7|py36hdbf6ddf_6|py27hdbf6ddf_6|py37h2b20989_6|py27h2b20989_6|py37hde5b4d6_0|py37h2f8d375_0']\n\
  \nPackage _libgcc_mutex conflicts for:\nmatplotlib=3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> _libgcc_mutex[version='*|0.1|0.1',build='main|conda_forge|main']\npython=3.7\
  \ -> libgcc-ng[version='>=9.4.0'] -> _libgcc_mutex[version='*|0.1',build='main|conda_forge|main']\n\
  pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='main|conda_forge|main']\n\
  pandas=0.25 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='main|conda_forge|main']\n\
  \nPackage freetype conflicts for:\nmatplotlib=3.1 -> freetype[version='>=2.9.1,<3.0a0']\n\
  pillow=6.2 -> freetype[version='>=2.9.1,<3.0a0']\n\nPackage pytz conflicts for:\n\
  matplotlib=3.1 -> pytz\npandas=0.25 -> pytz[version='>=2017.2']\n\nPackage python\
  \ conflicts for:\npillow=6.2 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  matplotlib=3.1 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  python=3.7\npillow=6.2 -> olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  goatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  goatools=0.9.7 -> python\nmatplotlib=3.1 -> pyqt -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.6|>=3.0,<=3.9|>=3.7|>=3.8|>=3']\n\
  \nPackage setuptools conflicts for:\nmatplotlib=3.1 -> setuptools\npython=3.7 ->\
  \ pip -> setuptools\npandas=0.25 -> setuptools[version='<60.0.0']\n\nPackage gdbm\
  \ conflicts for:\npython=3.7 -> pypy3.7=7.3.7 -> gdbm[version='>=1.18,<1.19.0a0']\n\
  pillow=6.2 -> pypy3.6[version='>=7.3.1'] -> gdbm[version='>=1.18,<1.19.0a0']\n\n\
  Package libpng conflicts for:\nmatplotlib=3.1 -> libpng[version='>=1.6.37,<1.7.0a0']\n\
  matplotlib=3.1 -> matplotlib-base==3.1.1=py36he7580a8_1 -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  \nPackage numpy conflicts for:\nmatplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0']\n\
  matplotlib=3.1 -> matplotlib-base==3.1.0=py37h5f35d83_0 -> numpy[version='>=1.11.3,<2.0a0']\n\
  \nPackage pandas conflicts for:\npandas=0.25\ngoatools=0.9.7 -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\n\
  \nPackage ca-certificates conflicts for:\ngoatools=0.9.7 -> python -> ca-certificates\n\
  pillow=6.2 -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\npython=3.7 ->\
  \ openssl[version='>=1.1.1l,<1.1.2a'] -> ca-certificatesThe following specifications\
  \ were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - matplotlib=3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> __glibc[version='>=2.17']\n  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] ->\
  \ __glibc[version='>=2.17']\n  - python=3.7 -> libgcc-ng[version='>=9.4.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fovef7wmt%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fovef7wmt%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.198GiB</td>
						<td>764.4 seconds</td>
						<td>780.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743186144">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.1">3.1</a>
						</td>
						<td>nextflow</td>
						<td>647 days</td>
						<td>2023-02-19 04:11:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: Include statement is not allowed within a workflow definition
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.script.BaseScript.include(BaseScript.groovy:150)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)\n\
  \tat groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:424)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat Script_a26fe0b9$_runScript_closure1$_closure3.doCall(Script_a26fe0b9:56)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:205)\n\
  \tat nextflow.script.WorkflowDef.run(WorkflowDef.groovy:189)\n\tat nextflow.script.BindableDef.invoke_a(BindableDef.groovy:52)\n\
  \tat nextflow.script.ComponentDef.invoke_o(ComponentDef.groovy:41)\n\tat nextflow.script.WorkflowBinding.invokeMethod(WorkflowBinding.groovy:94)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:408)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat Script_a26fe0b9$_runScript_closure2$_closure4.doCall(Script_a26fe0b9:72)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:205)\n\
  \tat nextflow.script.WorkflowDef.run(WorkflowDef.groovy:189)\n\tat nextflow.script.BindableDef.invoke_a(BindableDef.groovy:52)\n\
  \tat nextflow.script.IterableDef$invoke_a.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:208)\n\tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\
  \tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fwwuhc6pp%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fwwuhc6pp%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.598GiB</td>
						<td>22.2 seconds</td>
						<td>15.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743195984">
								<a href="https://nf-co.re/viralrecon">viralrecon</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/viralrecon/tree/2.0">2.0</a>
						</td>
						<td>nextflow</td>
						<td>647 days</td>
						<td>2023-02-19 06:15:27</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: Include statement is not allowed within a workflow definition
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.script.BaseScript.include(BaseScript.groovy:150)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)\n\
  \tat groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:424)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat Script_dd365653$_runScript_closure1$_closure3.doCall(Script_dd365653:63)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:205)\n\
  \tat nextflow.script.WorkflowDef.run(WorkflowDef.groovy:189)\n\tat nextflow.script.BindableDef.invoke_a(BindableDef.groovy:52)\n\
  \tat nextflow.script.ComponentDef.invoke_o(ComponentDef.groovy:41)\n\tat nextflow.script.WorkflowBinding.invokeMethod(WorkflowBinding.groovy:94)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:408)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat Script_dd365653$_runScript_closure2$_closure4.doCall(Script_dd365653:86)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:205)\n\
  \tat nextflow.script.WorkflowDef.run(WorkflowDef.groovy:189)\n\tat nextflow.script.BindableDef.invoke_a(BindableDef.groovy:52)\n\
  \tat nextflow.script.ChainableDef$invoke_a.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:208)\n\tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\
  \tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2F3ufcy2cn%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2F3ufcy2cn%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.546GiB</td>
						<td>23.6 seconds</td>
						<td>11.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738288768">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.1.0">v2.1.0</a>
						</td>
						<td>snakemake</td>
						<td>646 days</td>
						<td>2023-02-19 06:00:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/c63dz2kcnw/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/0440ab90-323f-4b43-bdaf-c4644f79c055\
  \ singularity  exec --home /home/azureuser/tmp/c63dz2kcnw/code/.test  /home/azureuser/tmp/c63dz2kcnw/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/c63dz2kcnw/code/.test/.snakemake/conda/559225bfa41aafb4de0c204832fccacf_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/c63dz2kcnw/code/.test/.snakemake/conda/559225bfa41aafb4de0c204832fccacf_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining goatools=0.9.7:  11%|\u2588 \
  \        | 1/9 [00:00<00:00, 3782.06it/s]        \nExamining matplotlib=3.1:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:00<00:01,  3.79it/s]  \nExamining matplotlib=3.1: \
  \ 33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:01,  5.68it/s]\nExamining pillow=6.2:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:03<00:01,  5.68it/s]    \nExamining\
  \ pillow=6.2:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:03<00:05,  1.11s/it]\n\
  Examining python=3.7:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:04<00:05,\
  \  1.11s/it]\nExamining python=3.7:  56%|\u2588\u2588\u2588\u2588\u2588\u258C  \
  \  | 5/9 [00:04<00:04,  1.04s/it]\nExamining @/linux-64::__glibc==2.31=0:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:06<00:04,  1.04s/it]\nExamining @/linux-64::__glibc==2.31=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:06<00:04,  1.35s/it]\n\
  Examining @/linux-64::__linux==5.15.0=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 6/9 [00:06<00:04,  1.35s/it]\nExamining pandas=0.25:  78%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:06<00:02,  1.35s/it]           \
  \       \nExamining @/linux-64::__unix==0=0:  89%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2589 | 8/9 [00:09<00:01,  1.35s/it]\nExamining @/linux-64::__unix==0=0:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:09<00:00,\
  \  1.10s/it]\n                                                                 \
  \               \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\n\
  Examining conflict for pandas goatools matplotlib:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for goatools matplotlib pillow python pandas:  11%|\u2588\
  \         | 1/9 [00:24<03:12, 24.00s/it]\nExamining conflict for goatools matplotlib\
  \ pillow python pandas:  22%|\u2588\u2588\u258F       | 2/9 [00:24<01:24, 12.00s/it]\n\
  Examining conflict for pillow pandas goatools matplotlib:  22%|\u2588\u2588\u258F\
  \       | 2/9 [00:28<01:24, 12.00s/it]       \nExamining conflict for pillow pandas\
  \ goatools matplotlib:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:28<00:52,  8.69s/it]\n\
  Examining conflict for goatools matplotlib:  33%|\u2588\u2588\u2588\u258E      |\
  \ 3/9 [00:34<00:52,  8.69s/it]              \nExamining conflict for goatools matplotlib:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:34<00:39,  7.90s/it]\nExamining\
  \ conflict for pandas goatools:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:39<00:39,\
  \  7.90s/it]    \nExamining conflict for pandas goatools:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:39<00:27,  6.95s/it]\nExamining conflict for pillow\
  \ python goatools:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:41<00:27,\
  \  6.95s/it]\nExamining conflict for pillow python goatools:  67%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258B   | 6/9 [00:41<00:15,  5.10s/it]\nExamining conflict for\
  \ pillow matplotlib:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:43<00:15,\
  \  5.10s/it]     \nExamining conflict for pillow matplotlib:  78%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258A  | 7/9 [00:43<00:08,  4.09s/it]\nExamining conflict\
  \ for pandas matplotlib:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A \
  \ | 7/9 [00:46<00:08,  4.09s/it]\nExamining conflict for pandas matplotlib:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:46<00:03,  3.76s/it]\n\
  Examining conflict for matplotlib pillow python __glibc pandas:  89%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:50<00:03,  3.76s/it]\nExamining\
  \ conflict for matplotlib pillow python __glibc pandas: 100%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:50<00:00,  3.80s/it]\nExamining conflict\
  \ for pillow python pandas matplotlib: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588| 9/9 [00:53<00:00,  3.80s/it]        \nExamining conflict\
  \ for pillow python pandas matplotlib: : 10it [00:53,  3.57s/it]               \
  \     \nExamining conflict for python pandas matplotlib: : 10it [00:57,  3.57s/it]\
  \       \nExamining conflict for python pandas matplotlib: : 11it [00:57,  3.97s/it]\n\
  Examining conflict for pillow python: : 11it [00:58,  3.97s/it]           \nExamining\
  \ conflict for pillow python: : 12it [00:58,  2.86s/it]\n                      \
  \                                         \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining goatools=0.9.7:  11%|\u2588         | 1/9 [00:00<00:00, 7653.84it/s]\
  \        \nExamining matplotlib=3.1:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \  9.55it/s]  \nExamining matplotlib=3.1:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [00:00<00:00, 14.32it/s]\nExamining pillow=6.2:  33%|\u2588\u2588\u2588\u258E\
  \      | 3/9 [00:01<00:00, 14.32it/s]    \nExamining python=3.7:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:02<00:00, 14.32it/s]\nExamining python=3.7:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:02<00:02,  1.90it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:02,  1.90it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 6/9 [00:03<00:02,  1.43it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:03<00:02,  1.43it/s]\n\
  Examining pandas=0.25:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  |\
  \ 7/9 [00:03<00:01,  1.43it/s]                  \nExamining @/linux-64::__unix==0=0:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:04<00:00,\
  \  1.43it/s]\nExamining @/linux-64::__unix==0=0: 100%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588| 9/9 [00:04<00:00,  1.73it/s]\n                 \
  \                                                               \n\nDetermining\
  \ conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for pandas\
  \ goatools matplotlib:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for goatools matplotlib pillow python pandas:  11%|\u2588         | 1/9 [00:16<02:08,\
  \ 16.09s/it]\nExamining conflict for goatools matplotlib pillow python pandas: \
  \ 22%|\u2588\u2588\u258F       | 2/9 [00:16<00:56,  8.04s/it]\nExamining conflict\
  \ for pillow pandas goatools matplotlib:  22%|\u2588\u2588\u258F       | 2/9 [00:20<00:56,\
  \  8.04s/it]       \nExamining conflict for pillow pandas goatools matplotlib: \
  \ 33%|\u2588\u2588\u2588\u258E      | 3/9 [00:20<00:38,  6.38s/it]\nExamining conflict\
  \ for goatools matplotlib:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:25<00:38,\
  \  6.38s/it]              \nExamining conflict for goatools matplotlib:  44%|\u2588\
  \u2588\u2588\u2588\u258D     | 4/9 [00:25<00:30,  6.18s/it]\nExamining conflict\
  \ for pandas goatools:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:29<00:30,\
  \  6.18s/it]    \nExamining conflict for pandas goatools:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:29<00:20,  5.18s/it]\nExamining conflict for pillow\
  \ python goatools:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:30<00:20,\
  \  5.18s/it]\nExamining conflict for pillow python goatools:  67%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258B   | 6/9 [00:30<00:11,  3.77s/it]\nExamining conflict for\
  \ pillow matplotlib:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:31<00:11,\
  \  3.77s/it]     \nExamining conflict for pillow matplotlib:  78%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258A  | 7/9 [00:31<00:05,  2.99s/it]\nExamining conflict\
  \ for pandas matplotlib:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A \
  \ | 7/9 [00:33<00:05,  2.99s/it]\nExamining conflict for pandas matplotlib:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:33<00:02,  2.80s/it]\n\
  Examining conflict for matplotlib pillow python __glibc pandas:  89%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:36<00:02,  2.80s/it]\nExamining\
  \ conflict for matplotlib pillow python __glibc pandas: 100%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:36<00:00,  2.66s/it]\nExamining conflict\
  \ for pillow python pandas matplotlib: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588| 9/9 [00:38<00:00,  2.66s/it]        \nExamining conflict\
  \ for pillow python pandas matplotlib: : 10it [00:38,  2.41s/it]               \
  \     \nExamining conflict for python pandas matplotlib: : 10it [00:40,  2.41s/it]\
  \       \nExamining conflict for python pandas matplotlib: : 11it [00:40,  2.50s/it]\n\
  Examining conflict for pillow python: : 11it [00:41,  2.50s/it]           \nExamining\
  \ conflict for pillow python: : 12it [00:41,  1.79s/it]\n                      \
  \                                         \nfailed\n\nUnsatisfiableError: The following\
  \ specifications were found to be incompatible with each other:\n\nOutput in format:\
  \ Requested package -> Available versions\n\nPackage numpy-base conflicts for:\n\
  matplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py27h2b20989_6|py36h2b20989_6|py37hdbf6ddf_6|py27hdbf6ddf_6|py36hdbf6ddf_6|py27h2b20989_7|py36hdbf6ddf_7|py36h2b20989_7|py37hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py36h2b20989_7|py36hdbf6ddf_7|py36hdbf6ddf_8|py37hdbf6ddf_8|py27h2b20989_8|py27hdbf6ddf_8|py35h2b20989_8|py35h7cdd4dd_9|py27h7cdd4dd_9|py36h3dfced4_9|py35h3dfced4_9|py37h81de0dd_9|py35h81de0dd_9|py35h74e8950_10|py36h74e8950_10|py27h2f8d375_11|py36h2f8d375_11|py27hde5b4d6_11|py36h2f8d375_12|py37hde5b4d6_12|py38h2f8d375_12|py27h0ea5e3f_1|py27h9be14a7_1|py36h2b20989_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_0|py36h2b20989_1|py27hdbf6ddf_1|py36hdbf6ddf_1|py27h2b20989_1|py27h2b20989_2|py27hdbf6ddf_2|py37hdbf6ddf_2|py36h2b20989_3|py27h2b20989_3|py37hdbf6ddf_4|py36h2b20989_4|py36hdbf6ddf_4|py35hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py37h2f8d375_4|py27h81de0dd_4|py27h2f8d375_4|py35h81de0dd_4|py36h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py37hde5b4d6_5|py36hde5b4d6_5|py27h7cdd4dd_0|py37h7cdd4dd_0|py37h3dfced4_0|py27h74e8950_0|py36h74e8950_0|py36h81de0dd_0|py37h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py37h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py27h2f8d375_1|py37h81de0dd_1|py37h2f8d375_1|py36h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py27h81de0dd_0|py27hde5b4d6_0|py37h2f8d375_1|py37hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_1|py37h2f8d375_1|py36h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py39hfb011de_1|py39h41b4c56_3|py38h41b4c56_3|py37h41b4c56_3|py36h41b4c56_3|py37ha8aedfd_4|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py37hfa32c7d_0|py37h75fe3a5_0|py37hfa32c7d_0|py37h75fe3a5_0|py36h75fe3a5_0|py39h0f7b65f_0|py37h4c65ebe_1|py38h4c65ebe_1|py39h21a3de8_1|py38h21a3de8_1|py37h34387ca_0|py39h34387ca_0|py39h7d8b39e_0|py38he2ba247_0|py38hfae3a4d_0|py39he2ba247_0|py37hfae3a4d_0|py39hfae3a4d_0|py39h74d4b33_0|py38h74d4b33_0|py38h39b7dee_0|py39h39b7dee_0|py39h79a1101_0|py37h2b8c604_0|py39h2b8c604_0|py37h79a1101_0|py39hf524024_1|py37hb8be1f0_1|py38hf524024_1|py310hf2716ce_1|py310h9585f30_1|py310hf2716ce_2|py39hb8be1f0_2|py310h9585f30_2|py39hf524024_2|py37h1e6e340_3|py38h1e6e340_3|py37ha15fc14_3|py39hf524024_0|py39hb8be1f0_0|py38hb8be1f0_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py310hcba007f_0|py38h31eccc5_0|py310h8e6c178_0|py310h375b286_0|py39h1e6e340_1|py38h31eccc5_1|py38h31eccc5_0|py39h31eccc5_0|py310h375b286_0|py310h375b286_0|py39h31eccc5_0|py310h8e6c178_0|py38h31eccc5_0|py39h1e6e340_0|py38h1e6e340_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_1|py310h8e6c178_1|py310h375b286_1|py38h1e6e340_1|py39h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py38h1e6e340_0|py39h1e6e340_0|py310hf2716ce_0|py38hf524024_0|py310h9585f30_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py38hf524024_2|py37hf524024_2|py38hb8be1f0_2|py37hb8be1f0_2|py37hf524024_1|py38hb8be1f0_1|py39hb8be1f0_1|py310h79a1101_0|py310h2b8c604_0|py38h2b8c604_0|py38h79a1101_0|py37h39b7dee_0|py37h74d4b33_0|py37he2ba247_0|py38h7d8b39e_0|py38h34387ca_0|py37h7d8b39e_0|py37h21a3de8_1|py39h4c65ebe_1|py39h2ae0177_0|py38hfa32c7d_0|py36hfa32c7d_0|py38h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py38h75fe3a5_0|py38h73d599e_4|py39ha8aedfd_4|py37hdc34a94_3|py39hdc34a94_3|py38hdc34a94_3|py36hdc34a94_3|py39h76555f2_1|py38hde5b4d6_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py36h81de0dd_1|py27h81de0dd_1|py36h2f8d375_1|py27h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py27h81de0dd_0|py37h74e8950_0|py35h74e8950_0|py35h3dfced4_0|py36h3dfced4_0|py27h3dfced4_0|py35h7cdd4dd_0|py36h7cdd4dd_0|py37h2f8d375_5|py38hde5b4d6_4|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py36h2f8d375_4|py35h2f8d375_4|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py37h2b20989_4|py27hdbf6ddf_4|py27h2b20989_4|py27hdbf6ddf_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py37h2b20989_3|py36hdbf6ddf_2|py37h2b20989_2|py36h2b20989_2|py37hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py27h2b20989_0|py27hdbf6ddf_0|py35h2b20989_0|py27h2b20989_0|py35h9be14a7_1|py36h9be14a7_1|py35h0ea5e3f_1|py36h0ea5e3f_1|py38hde5b4d6_12|py36hde5b4d6_12|py27hde5b4d6_12|py27h2f8d375_12|py37h2f8d375_12|py37hde5b4d6_11|py36hde5b4d6_11|py37h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py36h2f8d375_10|py37h2f8d375_10|py35h81de0dd_10|py36h81de0dd_10|py37h81de0dd_10|py27h81de0dd_10|py27h74e8950_10|py37h74e8950_10|py27h81de0dd_9|py36h74e8950_9|py37h74e8950_9|py35h74e8950_9|py27h74e8950_9|py36h81de0dd_9|py27h3dfced4_9|py37h3dfced4_9|py37h7cdd4dd_9|py36h7cdd4dd_9|py35hdbf6ddf_8|py37h2b20989_8|py36h2b20989_8|py37hdbf6ddf_7|py27hdbf6ddf_7|py27h2b20989_7|py35h2b20989_7|py35hdbf6ddf_7|py27hdbf6ddf_7|py37h2b20989_6']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py27h0ea5e3f_1|py27h9be14a7_1|py36h2b20989_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_0|py36h2b20989_1|py27hdbf6ddf_1|py36hdbf6ddf_1|py27h2b20989_1|py27h2b20989_2|py27hdbf6ddf_2|py37hdbf6ddf_2|py36h2b20989_3|py27h2b20989_3|py37hdbf6ddf_4|py36h2b20989_4|py36hdbf6ddf_4|py35hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py37h2f8d375_4|py27h81de0dd_4|py27h2f8d375_4|py35h81de0dd_4|py36h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py37hde5b4d6_5|py36hde5b4d6_5|py27h7cdd4dd_0|py37h7cdd4dd_0|py37h3dfced4_0|py27h74e8950_0|py36h74e8950_0|py36h81de0dd_0|py37h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py37h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py27h2f8d375_1|py37h81de0dd_1|py37h2f8d375_1|py36h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py27h81de0dd_0|py27hde5b4d6_0|py37h2f8d375_1|py37hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_1|py37h2f8d375_1|py36h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py39hfb011de_1|py39h41b4c56_3|py38h41b4c56_3|py37h41b4c56_3|py36h41b4c56_3|py37ha8aedfd_4|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py37hfa32c7d_0|py37h75fe3a5_0|py37hfa32c7d_0|py37h75fe3a5_0|py36h75fe3a5_0|py39h0f7b65f_0|py37h4c65ebe_1|py38h4c65ebe_1|py39h21a3de8_1|py38h21a3de8_1|py37h34387ca_0|py39h34387ca_0|py39h7d8b39e_0|py38he2ba247_0|py38hfae3a4d_0|py39he2ba247_0|py37hfae3a4d_0|py39hfae3a4d_0|py39h74d4b33_0|py38h74d4b33_0|py38h39b7dee_0|py39h39b7dee_0|py39h79a1101_0|py37h2b8c604_0|py39h2b8c604_0|py37h79a1101_0|py39hf524024_1|py37hb8be1f0_1|py38hf524024_1|py310hf2716ce_1|py310h9585f30_1|py310hf2716ce_2|py39hb8be1f0_2|py310h9585f30_2|py39hf524024_2|py37h1e6e340_3|py38h1e6e340_3|py37ha15fc14_3|py39hf524024_0|py39hb8be1f0_0|py38hb8be1f0_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py310hcba007f_0|py38h31eccc5_0|py310h8e6c178_0|py310h375b286_0|py39h1e6e340_1|py38h31eccc5_1|py38h31eccc5_0|py39h31eccc5_0|py310h375b286_0|py310h375b286_0|py39h31eccc5_0|py310h8e6c178_0|py38h31eccc5_0|py39h1e6e340_0|py38h1e6e340_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_1|py310h8e6c178_1|py310h375b286_1|py38h1e6e340_1|py39h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py38h1e6e340_0|py39h1e6e340_0|py310hf2716ce_0|py38hf524024_0|py310h9585f30_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py38hf524024_2|py37hf524024_2|py38hb8be1f0_2|py37hb8be1f0_2|py37hf524024_1|py38hb8be1f0_1|py39hb8be1f0_1|py310h79a1101_0|py310h2b8c604_0|py38h2b8c604_0|py38h79a1101_0|py37h39b7dee_0|py37h74d4b33_0|py37he2ba247_0|py38h7d8b39e_0|py38h34387ca_0|py37h7d8b39e_0|py37h21a3de8_1|py39h4c65ebe_1|py39h2ae0177_0|py38hfa32c7d_0|py36hfa32c7d_0|py38h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py38h75fe3a5_0|py38h73d599e_4|py39ha8aedfd_4|py37hdc34a94_3|py39hdc34a94_3|py38hdc34a94_3|py36hdc34a94_3|py39h76555f2_1|py38hde5b4d6_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py36h81de0dd_1|py27h81de0dd_1|py36h2f8d375_1|py27h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py27h81de0dd_0|py37h74e8950_0|py35h74e8950_0|py35h3dfced4_0|py36h3dfced4_0|py27h3dfced4_0|py35h7cdd4dd_0|py36h7cdd4dd_0|py37h2f8d375_5|py38hde5b4d6_4|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py36h2f8d375_4|py35h2f8d375_4|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py37h2b20989_4|py27hdbf6ddf_4|py27h2b20989_4|py27hdbf6ddf_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py37h2b20989_3|py36hdbf6ddf_2|py37h2b20989_2|py36h2b20989_2|py37hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py27h2b20989_0|py27hdbf6ddf_0|py35h2b20989_0|py27h2b20989_0|py35h9be14a7_1|py36h9be14a7_1|py35h0ea5e3f_1|py36h0ea5e3f_1']\n\
  goatools=0.9.7 -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py36h2f8d375_0|py27h2b20989_6|py36h2b20989_6|py37hdbf6ddf_6|py27hdbf6ddf_6|py36hdbf6ddf_6|py27h2b20989_7|py36hdbf6ddf_7|py36h2b20989_7|py35hdbf6ddf_7|py35h2b20989_7|py37hdbf6ddf_7|py37h2b20989_7|py27h2b20989_7|py36h2b20989_7|py36hdbf6ddf_7|py37hdbf6ddf_7|py36hdbf6ddf_8|py37hdbf6ddf_8|py27h2b20989_8|py27hdbf6ddf_8|py35h7cdd4dd_9|py27h7cdd4dd_9|py36h3dfced4_9|py35h3dfced4_9|py36h81de0dd_9|py37h81de0dd_9|py37h74e8950_9|py36h74e8950_9|py35h81de0dd_9|py37h74e8950_10|py35h74e8950_10|py36h74e8950_10|py27h81de0dd_10|py37h2f8d375_10|py27h2f8d375_11|py36h2f8d375_11|py27hde5b4d6_12|py37hde5b4d6_12|py36hde5b4d6_12|py38h2f8d375_12|py27h0ea5e3f_1|py35h0ea5e3f_1|py27h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_1|py36hdbf6ddf_1|py27h2b20989_1|py27h2b20989_2|py27hdbf6ddf_2|py37hdbf6ddf_2|py27h2b20989_3|py37hdbf6ddf_4|py36h2b20989_4|py36hdbf6ddf_4|py35hdbf6ddf_4|py35h2b20989_4|py37h2f8d375_4|py27h81de0dd_4|py27h2f8d375_4|py35h81de0dd_4|py36h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py37hde5b4d6_5|py36hde5b4d6_5|py27h7cdd4dd_0|py37h7cdd4dd_0|py37h3dfced4_0|py27h74e8950_0|py36h74e8950_0|py36h81de0dd_0|py37h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py37h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py27h2f8d375_1|py37h81de0dd_1|py37h2f8d375_1|py36h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py27h81de0dd_0|py27hde5b4d6_0|py37h2f8d375_1|py37hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_1|py37h2f8d375_1|py36h2f8d375_0|py27hde5b4d6_0|py37h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py39hfb011de_1|py39h41b4c56_3|py38h41b4c56_3|py37h41b4c56_3|py36h41b4c56_3|py37ha8aedfd_4|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py37hfa32c7d_0|py37h75fe3a5_0|py37hfa32c7d_0|py37h75fe3a5_0|py36h75fe3a5_0|py39h0f7b65f_0|py37h4c65ebe_1|py38h4c65ebe_1|py39h21a3de8_1|py38h21a3de8_1|py37h34387ca_0|py39h34387ca_0|py39h7d8b39e_0|py38he2ba247_0|py38hfae3a4d_0|py39he2ba247_0|py37hfae3a4d_0|py39hfae3a4d_0|py39h74d4b33_0|py38h74d4b33_0|py38h39b7dee_0|py39h39b7dee_0|py39h79a1101_0|py37h2b8c604_0|py39h2b8c604_0|py37h79a1101_0|py39hf524024_1|py37hb8be1f0_1|py38hf524024_1|py310hf2716ce_1|py310h9585f30_1|py310hf2716ce_2|py39hb8be1f0_2|py310h9585f30_2|py39hf524024_2|py37h1e6e340_3|py38h1e6e340_3|py37ha15fc14_3|py39hf524024_0|py39hb8be1f0_0|py38hb8be1f0_0|py310h375b286_0|py39ha15fc14_0|py38ha15fc14_0|py310hcba007f_0|py38h31eccc5_0|py310h8e6c178_0|py310h375b286_0|py39h1e6e340_1|py38h31eccc5_1|py38h31eccc5_0|py39h31eccc5_0|py310h375b286_0|py310h375b286_0|py39h31eccc5_0|py310h8e6c178_0|py38h31eccc5_0|py39h1e6e340_0|py38h1e6e340_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_1|py310h8e6c178_1|py310h375b286_1|py38h1e6e340_1|py39h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py38h1e6e340_0|py39h1e6e340_0|py310hf2716ce_0|py38hf524024_0|py310h9585f30_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py38hf524024_2|py37hf524024_2|py38hb8be1f0_2|py37hb8be1f0_2|py37hf524024_1|py38hb8be1f0_1|py39hb8be1f0_1|py310h79a1101_0|py310h2b8c604_0|py38h2b8c604_0|py38h79a1101_0|py37h39b7dee_0|py37h74d4b33_0|py37he2ba247_0|py38h7d8b39e_0|py38h34387ca_0|py37h7d8b39e_0|py37h21a3de8_1|py39h4c65ebe_1|py39h2ae0177_0|py38hfa32c7d_0|py36hfa32c7d_0|py38h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py38h75fe3a5_0|py38h73d599e_4|py39ha8aedfd_4|py37hdc34a94_3|py39hdc34a94_3|py38hdc34a94_3|py36hdc34a94_3|py39h76555f2_1|py38hde5b4d6_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py36h81de0dd_1|py27h81de0dd_1|py36h2f8d375_1|py27h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py27h81de0dd_0|py37h74e8950_0|py35h74e8950_0|py35h3dfced4_0|py36h3dfced4_0|py27h3dfced4_0|py35h7cdd4dd_0|py36h7cdd4dd_0|py37h2f8d375_5|py38hde5b4d6_4|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py36h2f8d375_4|py35h2f8d375_4|py37h2b20989_4|py27hdbf6ddf_4|py27h2b20989_4|py27hdbf6ddf_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py37h2b20989_3|py36h2b20989_3|py36hdbf6ddf_2|py37h2b20989_2|py36h2b20989_2|py37hdbf6ddf_1|py37h2b20989_1|py36h2b20989_1|py35hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_0|py27hdbf6ddf_0|py35h2b20989_0|py35h9be14a7_1|py36h9be14a7_1|py36h0ea5e3f_1|py38hde5b4d6_12|py36h2f8d375_12|py27h2f8d375_12|py37h2f8d375_12|py27hde5b4d6_11|py37hde5b4d6_11|py36hde5b4d6_11|py37h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py36h2f8d375_10|py35h81de0dd_10|py36h81de0dd_10|py37h81de0dd_10|py27h74e8950_10|py27h81de0dd_9|py35h74e8950_9|py27h74e8950_9|py27h3dfced4_9|py37h3dfced4_9|py37h7cdd4dd_9|py36h7cdd4dd_9|py35h2b20989_8|py35hdbf6ddf_8|py37h2b20989_8|py36h2b20989_8|py27hdbf6ddf_7|py37h2b20989_7|py27hdbf6ddf_7|py37h2b20989_6|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0']\n\
  \nPackage python conflicts for:\ngoatools=0.9.7 -> python\npython=3.7\nmatplotlib=3.1\
  \ -> pyqt -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.6|>=3.0,<=3.9|>=3.7|>=3.8|>=3']\n\
  goatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  matplotlib=3.1 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  pillow=6.2 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  pillow=6.2 -> olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  \nPackage six conflicts for:\npandas=0.25 -> python-dateutil[version='>=2.6.1']\
  \ -> six[version='>=1.5']\nmatplotlib=3.1 -> cycler[version='>=0.10'] -> six[version='>=1.5']\n\
  \nPackage libgcc conflicts for:\ngoatools=0.9.7 -> scipy -> libgcc\nmatplotlib=3.1\
  \ -> pyqt -> libgcc\n\nPackage ca-certificates conflicts for:\npython=3.7 -> openssl[version='>=1.1.1l,<1.1.2a']\
  \ -> ca-certificates\npillow=6.2 -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n\
  goatools=0.9.7 -> python -> ca-certificates\n\nPackage pypy3.8 conflicts for:\n\
  goatools=0.9.7 -> numpy -> pypy3.8[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> pypy3.8[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\n\
  matplotlib=3.1 -> python[version='>=3.8,<3.9.0a0'] -> pypy3.8[version='7.3.11.*|7.3.9.*|7.3.8.*|>=7.3.9|>=7.3.8|>=7.3.11']\n\
  pillow=6.2 -> python[version='>=3.8,<3.9.0a0'] -> pypy3.8[version='7.3.11.*|7.3.9.*|7.3.8.*']\n\
  \nPackage pandas conflicts for:\ngoatools=0.9.7 -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\n\
  pandas=0.25\n\nPackage gdbm conflicts for:\npython=3.7 -> pypy3.7=7.3.7 -> gdbm[version='>=1.18,<1.19.0a0']\n\
  pillow=6.2 -> pypy3.6[version='>=7.3.1'] -> gdbm[version='>=1.18,<1.19.0a0']\n\n\
  Package libpng conflicts for:\nmatplotlib=3.1 -> matplotlib-base==3.1.1=py36he7580a8_1\
  \ -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.39,<1.7.0a0']\nmatplotlib=3.1\
  \ -> libpng[version='>=1.6.37,<1.7.0a0']\npillow=6.2 -> freetype[version='>=2.9.1,<3.0a0']\
  \ -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.37,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  \nPackage _libgcc_mutex conflicts for:\npython=3.7 -> libgcc-ng[version='>=9.4.0']\
  \ -> _libgcc_mutex[version='*|0.1',build='conda_forge|main|main']\npandas=0.25 ->\
  \ libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  matplotlib=3.1 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  \nPackage setuptools conflicts for:\npython=3.7 -> pip -> setuptools\nmatplotlib=3.1\
  \ -> setuptools\npandas=0.25 -> setuptools[version='<60.0.0']\n\nPackage pypy3.6\
  \ conflicts for:\npillow=6.2 -> pypy3.6[version='>=7.3.1']\npillow=6.2 -> python[version='>=3.6,<3.7.0a0']\
  \ -> pypy3.6[version='7.3.*|7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*']The following specifications\
  \ were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - matplotlib=3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> __glibc[version='>=2.17']\n  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] ->\
  \ __glibc[version='>=2.17']\n  - python=3.7 -> libgcc-ng[version='>=9.4.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fnuzr9cka%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fnuzr9cka%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.294GiB</td>
						<td>2249.5 seconds</td>
						<td>2764.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738288528">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.1.1">v2.1.1</a>
						</td>
						<td>snakemake</td>
						<td>643 days</td>
						<td>2023-02-19 04:08:23</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/llw50rspql/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/99c29b0c-db9f-494d-8c71-eccf8e898491\
  \ singularity  exec --home /home/azureuser/tmp/llw50rspql/code/.test  /home/azureuser/tmp/llw50rspql/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/llw50rspql/code/.test/.snakemake/conda/48acf01366b94735a12fe766056b063a_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/llw50rspql/code/.test/.snakemake/conda/48acf01366b94735a12fe766056b063a_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining pandas=0.25:   0%|          |\
  \ 0/9 [00:00<?, ?it/s] \nExamining @/linux-64::__glibc==2.31=0:  11%|\u2588    \
  \     | 1/9 [00:02<00:21,  2.74s/it]\nExamining @/linux-64::__glibc==2.31=0:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:02<00:09,  1.37s/it]\nExamining @/linux-64::__unix==0=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:02<00:09,  1.37s/it]    \nExamining goatools=0.9.7:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:02<00:08,  1.37s/it]         \nExamining\
  \ @/linux-64::__archspec==1=x86_64:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9\
  \ [00:03<00:06,  1.37s/it]\nExamining @/linux-64::__archspec==1=x86_64:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:02,  1.84it/s]\nExamining pillow=6.2:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:02,  1.84it/s]  \
  \                    \nExamining python=3.7:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:04<00:01,  1.84it/s]\nExamining python=3.7:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:04<00:01,  1.94it/s]\nExamining\
  \ matplotlib=3.1:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:07<00:01,\
  \  1.94it/s]\nExamining matplotlib=3.1:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2589 | 8/9 [00:07<00:01,  1.24s/it]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:11<00:01,\
  \  1.24s/it]\nExamining @/linux-64::__linux==5.15.0=0: 100%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:11<00:00,  1.83s/it]\n           \
  \                                                                           \n\n\
  Determining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for pandas goatools:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for pandas goatools pillow python matplotlib:  11%|\u2588         | 1/9 [00:01<00:13,\
  \  1.64s/it]\nExamining conflict for pandas goatools pillow python matplotlib: \
  \ 22%|\u2588\u2588\u258F       | 2/9 [00:01<00:05,  1.22it/s]\nExamining conflict\
  \ for pandas goatools matplotlib:  22%|\u2588\u2588\u258F       | 2/9 [00:06<00:05,\
  \  1.22it/s]              \nExamining conflict for pandas goatools matplotlib: \
  \ 33%|\u2588\u2588\u2588\u258E      | 3/9 [00:06<00:14,  2.45s/it]\nExamining conflict\
  \ for pandas pillow python matplotlib:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [00:39<00:14,  2.45s/it]\nExamining conflict for pandas pillow python matplotlib:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:39<01:08, 13.71s/it]\nExamining\
  \ conflict for pandas pillow goatools matplotlib:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [00:44<01:08, 13.71s/it]\nExamining conflict for pandas pillow goatools\
  \ matplotlib:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:44<00:44, 11.00s/it]\n\
  Examining conflict for pandas matplotlib:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:54<00:44, 11.00s/it]                \nExamining conflict for pandas\
  \ matplotlib:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:54<00:31,\
  \ 10.61s/it]\nExamining conflict for pandas __glibc pillow python matplotlib:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:56<00:31, 10.61s/it]\nExamining\
  \ conflict for pandas __glibc pillow python matplotlib:  78%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258A  | 7/9 [00:56<00:15,  7.65s/it]\nExamining conflict for\
  \ pandas python matplotlib:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [00:59<00:15,  7.65s/it]               \nExamining conflict for pandas\
  \ python matplotlib:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589\
  \ | 8/9 [00:59<00:06,  6.23s/it]\nExamining conflict for goatools matplotlib:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:59<00:06,  6.23s/it]\
  \     \nExamining conflict for goatools matplotlib: 100%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:59<00:00,  4.40s/it]\nExamining conflict\
  \ for pillow python goatools: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588| 9/9 [01:02<00:00,  4.40s/it]\nExamining conflict for pillow python\
  \ goatools: : 10it [01:02,  4.00s/it]                    \nExamining conflict for\
  \ pillow python: : 10it [01:04,  4.00s/it]         \nExamining conflict for pillow\
  \ python: : 11it [01:04,  3.41s/it]\nExamining conflict for pillow matplotlib: :\
  \ 11it [01:05,  3.41s/it]\nExamining conflict for pillow matplotlib: : 12it [01:05,\
  \  2.65s/it]\n                                                                 \
  \  \nfailed\nSolving environment: ...working... \nFound conflicts! Looking for incompatible\
  \ packages.\nThis can take several minutes.  Press CTRL-C to abort.\n\nBuilding\
  \ graph of deps:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining pandas=0.25: \
  \  0%|          | 0/9 [00:00<?, ?it/s] \nExamining @/linux-64::__glibc==2.31=0:\
  \  11%|\u2588         | 1/9 [00:02<00:18,  2.28s/it]\nExamining @/linux-64::__glibc==2.31=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:02<00:07,  1.14s/it]\nExamining @/linux-64::__unix==0=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:02<00:07,  1.14s/it]    \nExamining goatools=0.9.7:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:02<00:06,  1.14s/it]         \nExamining\
  \ @/linux-64::__archspec==1=x86_64:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9\
  \ [00:02<00:05,  1.14s/it]\nExamining @/linux-64::__archspec==1=x86_64:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:02<00:01,  2.28it/s]\nExamining pillow=6.2:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:02<00:01,  2.28it/s]  \
  \                    \nExamining python=3.7:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:03<00:01,  2.28it/s]\nExamining python=3.7:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:03<00:00,  2.56it/s]\nExamining\
  \ matplotlib=3.1:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:05<00:00,\
  \  2.56it/s]\nExamining matplotlib=3.1:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2589 | 8/9 [00:05<00:00,  1.40it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:07<00:00,\
  \  1.40it/s]\nExamining @/linux-64::__linux==5.15.0=0: 100%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:07<00:00,  1.15s/it]\n           \
  \                                                                           \n\n\
  Determining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for pandas goatools:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for pandas goatools pillow python matplotlib:  11%|\u2588         | 1/9 [00:01<00:10,\
  \  1.32s/it]\nExamining conflict for pandas goatools pillow python matplotlib: \
  \ 22%|\u2588\u2588\u258F       | 2/9 [00:01<00:04,  1.52it/s]\nExamining conflict\
  \ for pandas goatools matplotlib:  22%|\u2588\u2588\u258F       | 2/9 [00:05<00:04,\
  \  1.52it/s]              \nExamining conflict for pandas goatools matplotlib: \
  \ 33%|\u2588\u2588\u2588\u258E      | 3/9 [00:05<00:12,  2.06s/it]\nExamining conflict\
  \ for pandas pillow python matplotlib:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [00:28<00:12,  2.06s/it]\nExamining conflict for pandas pillow python matplotlib:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:28<00:49,  9.98s/it]\nExamining\
  \ conflict for pandas pillow goatools matplotlib:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [00:34<00:49,  9.98s/it]\nExamining conflict for pandas pillow goatools\
  \ matplotlib:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:34<00:33, \
  \ 8.42s/it]\nExamining conflict for pandas matplotlib:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:43<00:33,  8.42s/it]                \nExamining conflict\
  \ for pandas matplotlib:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9\
  \ [00:43<00:25,  8.66s/it]\nExamining conflict for pandas __glibc pillow python\
  \ matplotlib:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:44<00:25,\
  \  8.66s/it]\nExamining conflict for pandas __glibc pillow python matplotlib:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:44<00:12,  6.26s/it]\nExamining\
  \ conflict for pandas python matplotlib:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:47<00:12,  6.26s/it]               \nExamining conflict\
  \ for pandas python matplotlib:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:47<00:05,  5.27s/it]\nExamining conflict for goatools matplotlib:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:47<00:05,\
  \  5.27s/it]     \nExamining conflict for goatools matplotlib: 100%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:47<00:00,  3.72s/it]\n\
  Examining conflict for pillow python goatools: 100%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588| 9/9 [00:50<00:00,  3.72s/it]\nExamining conflict\
  \ for pillow python goatools: : 10it [00:50,  3.45s/it]                    \nExamining\
  \ conflict for pillow python: : 10it [00:52,  3.45s/it]         \nExamining conflict\
  \ for pillow python: : 11it [00:52,  2.99s/it]\nExamining conflict for pillow matplotlib:\
  \ : 11it [00:53,  2.99s/it]\nExamining conflict for pillow matplotlib: : 12it [00:53,\
  \  2.32s/it]\n                                                                 \
  \  \nfailed\n\nUnsatisfiableError: The following specifications were found to be\
  \ incompatible with each other:\n\nOutput in format: Requested package -> Available\
  \ versions\n\nPackage gdbm conflicts for:\npython=3.7 -> pypy3.7=7.3.7 -> gdbm[version='>=1.18,<1.19.0a0']\n\
  pillow=6.2 -> pypy3.6[version='>=7.3.1'] -> gdbm[version='>=1.18,<1.19.0a0']\n\n\
  Package pypy3.8 conflicts for:\ngoatools=0.9.7 -> numpy -> pypy3.8[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> pypy3.8[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\n\
  pillow=6.2 -> python[version='>=3.8,<3.9.0a0'] -> pypy3.8[version='7.3.11.*|7.3.9.*|7.3.8.*']\n\
  matplotlib=3.1 -> python[version='>=3.8,<3.9.0a0'] -> pypy3.8[version='7.3.11.*|7.3.9.*|7.3.8.*|>=7.3.9|>=7.3.8|>=7.3.11']\n\
  \nPackage _openmp_mutex conflicts for:\npillow=6.2 -> libgcc-ng[version='>=7.5.0']\
  \ -> _openmp_mutex[version='>=4.5']\npandas=0.25 -> libgcc-ng[version='>=7.3.0']\
  \ -> _openmp_mutex[version='>=4.5']\nmatplotlib=3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> _openmp_mutex[version='>=4.5']\npython=3.7 -> libgcc-ng[version='>=9.4.0']\
  \ -> _openmp_mutex[version='>=4.5']\n\nPackage numpy-base conflicts for:\ngoatools=0.9.7\
  \ -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py37h2f8d375_0|py36hde5b4d6_0|py37hdbf6ddf_6|py27hdbf6ddf_6|py36hdbf6ddf_6|py36hdbf6ddf_7|py37hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py36h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_7|py37hdbf6ddf_7|py37h2b20989_8|py35h2b20989_8|py36h3dfced4_9|py27h3dfced4_9|py36h81de0dd_9|py35h74e8950_9|py37h74e8950_9|py36h74e8950_9|py27h81de0dd_9|py35h81de0dd_9|py35h74e8950_10|py27h74e8950_10|py37h81de0dd_10|py35h81de0dd_10|py36h2f8d375_10|py27h2f8d375_10|py35h2f8d375_10|py27h2f8d375_11|py37h2f8d375_12|py27h2f8d375_12|py27hde5b4d6_12|py37hde5b4d6_12|py36hde5b4d6_12|py35h0ea5e3f_1|py27h9be14a7_1|py36h2b20989_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_0|py27h2b20989_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py37h2b20989_1|py27hdbf6ddf_1|py37hdbf6ddf_1|py27h2b20989_1|py36h2b20989_2|py37h2b20989_2|py27hdbf6ddf_2|py36hdbf6ddf_2|py36h2b20989_3|py37h2b20989_3|py27hdbf6ddf_3|py27h2b20989_4|py27hdbf6ddf_4|py37hdbf6ddf_4|py36hdbf6ddf_4|py35h2b20989_4|py37h2f8d375_4|py27h81de0dd_4|py36h2f8d375_4|py35h81de0dd_4|py36h81de0dd_4|py38h2f8d375_4|py37hde5b4d6_5|py36hde5b4d6_5|py37h7cdd4dd_0|py35h7cdd4dd_0|py27h3dfced4_0|py36h3dfced4_0|py37h3dfced4_0|py27h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py27h81de0dd_0|py37h81de0dd_0|py35h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py35h81de0dd_0|py36h81de0dd_0|py27h2f8d375_1|py36h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py27h2f8d375_0|py27h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_1|py36h2f8d375_1|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py38hde5b4d6_0|py27h2f8d375_0|py39h41b4c56_3|py38h41b4c56_3|py36hdc34a94_3|py39hdc34a94_3|py39ha8aedfd_4|py37h73d599e_4|py38h73d599e_4|py38h75fe3a5_0|py36h75fe3a5_0|py37h75fe3a5_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py39h4c65ebe_1|py37h21a3de8_1|py38h21a3de8_1|py38h7d8b39e_0|py39h7d8b39e_0|py37he2ba247_0|py37h74d4b33_0|py39h74d4b33_0|py38h74d4b33_0|py38h39b7dee_0|py39h39b7dee_0|py39h79a1101_0|py39h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py39hb8be1f0_1|py38hb8be1f0_1|py37hb8be1f0_1|py38hf524024_1|py310hf2716ce_1|py310h9585f30_1|py310hf2716ce_2|py39hb8be1f0_2|py39hf524024_2|py38hf524024_2|py38h1e6e340_3|py39h1e6e340_3|py37ha15fc14_3|py38ha15fc14_3|py39hf524024_0|py310h9585f30_0|py39hb8be1f0_0|py310hf2716ce_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py38ha15fc14_0|py38h31eccc5_0|py310h8e6c178_0|py310h375b286_0|py310h8e6c178_1|py38h31eccc5_0|py310h8e6c178_0|py39h1e6e340_0|py310h375b286_0|py310h375b286_0|py38h31eccc5_0|py310h8e6c178_0|py39h31eccc5_0|py39h1e6e340_0|py38h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py39h31eccc5_1|py38h31eccc5_1|py39h1e6e340_1|py310h375b286_1|py38h1e6e340_1|py39h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py310hcba007f_0|py39ha15fc14_0|py38hb8be1f0_0|py38hf524024_0|py39ha15fc14_3|py310hcba007f_3|py310h375b286_3|py37h1e6e340_3|py37hf524024_2|py310h9585f30_2|py38hb8be1f0_2|py37hb8be1f0_2|py37hf524024_1|py39hf524024_1|py310h79a1101_0|py38h2b8c604_0|py37h2b8c604_0|py38h79a1101_0|py37h39b7dee_0|py39hfae3a4d_0|py37hfae3a4d_0|py39he2ba247_0|py38hfae3a4d_0|py38he2ba247_0|py38h34387ca_0|py39h34387ca_0|py37h7d8b39e_0|py37h34387ca_0|py39h21a3de8_1|py38h4c65ebe_1|py37h4c65ebe_1|py39h0f7b65f_0|py39h2ae0177_0|py38hfa32c7d_0|py36hfa32c7d_0|py37hfa32c7d_0|py36hfa32c7d_0|py38hfa32c7d_0|py37hfa32c7d_0|py39h73d599e_4|py38ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py38hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py27hde5b4d6_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py36h2f8d375_0|py27h81de0dd_1|py36h2f8d375_1|py36h2f8d375_0|py35h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h2f8d375_0|py36h74e8950_0|py35h74e8950_0|py35h3dfced4_0|py36h7cdd4dd_0|py27h7cdd4dd_0|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py37h81de0dd_4|py27h2f8d375_4|py35h2f8d375_4|py35hdbf6ddf_4|py36h2b20989_4|py37h2b20989_4|py27h2b20989_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py37hdbf6ddf_2|py27h2b20989_2|py36hdbf6ddf_1|py36h2b20989_1|py27hdbf6ddf_0|py27hdbf6ddf_0|py35h2b20989_0|py27h2b20989_0|py35h9be14a7_1|py36h9be14a7_1|py27h0ea5e3f_1|py36h0ea5e3f_1|py38h2f8d375_12|py38hde5b4d6_12|py36h2f8d375_12|py27hde5b4d6_11|py37hde5b4d6_11|py36hde5b4d6_11|py36h2f8d375_11|py37h2f8d375_11|py37h2f8d375_10|py36h81de0dd_10|py27h81de0dd_10|py36h74e8950_10|py37h74e8950_10|py27h74e8950_9|py37h81de0dd_9|py35h3dfced4_9|py37h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py35h7cdd4dd_9|py36h7cdd4dd_9|py35hdbf6ddf_8|py27hdbf6ddf_8|py27h2b20989_8|py37hdbf6ddf_8|py36h2b20989_8|py36hdbf6ddf_8|py27h2b20989_7|py35h2b20989_7|py35hdbf6ddf_7|py36h2b20989_7|py27h2b20989_7|py27hdbf6ddf_7|py37h2b20989_6|py36h2b20989_6|py27h2b20989_6|py37hde5b4d6_0|py36h2f8d375_0']\n\
  matplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py27hdbf6ddf_6|py36hdbf6ddf_7|py36h2b20989_7|py37hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py36h2b20989_7|py36hdbf6ddf_7|py37hdbf6ddf_7|py35h2b20989_8|py36h3dfced4_9|py35h3dfced4_9|py35h74e8950_9|py37h74e8950_9|py36h74e8950_9|py37h74e8950_10|py35h74e8950_10|py27h74e8950_10|py36h2f8d375_10|py35h2f8d375_10|py27h2f8d375_11|py36h2f8d375_11|py36hde5b4d6_11|py27hde5b4d6_11|py37h2f8d375_12|py27h2f8d375_12|py36h2f8d375_12|py36hde5b4d6_12|py27h0ea5e3f_1|py36h9be14a7_1|py27h9be14a7_1|py35h9be14a7_1|py36h2b20989_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py27hdbf6ddf_1|py36hdbf6ddf_1|py37hdbf6ddf_1|py27h2b20989_1|py37h2b20989_2|py27hdbf6ddf_2|py37hdbf6ddf_2|py36hdbf6ddf_2|py36h2b20989_3|py37hdbf6ddf_3|py36hdbf6ddf_3|py27hdbf6ddf_3|py27h2b20989_4|py27hdbf6ddf_4|py37hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py37h2f8d375_0|py37h2f8d375_4|py27h81de0dd_4|py36h2f8d375_4|py35h81de0dd_4|py36h81de0dd_4|py38h2f8d375_4|py37hde5b4d6_5|py36hde5b4d6_5|py37h7cdd4dd_0|py35h7cdd4dd_0|py27h3dfced4_0|py36h3dfced4_0|py37h3dfced4_0|py27h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py27h81de0dd_0|py37h81de0dd_0|py35h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py35h81de0dd_0|py36h81de0dd_0|py27h2f8d375_1|py36h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py27h2f8d375_0|py27h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_1|py36h2f8d375_1|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py38hde5b4d6_0|py27h2f8d375_0|py39h41b4c56_3|py38h41b4c56_3|py36hdc34a94_3|py39hdc34a94_3|py39ha8aedfd_4|py37h73d599e_4|py38h73d599e_4|py38h75fe3a5_0|py36h75fe3a5_0|py37h75fe3a5_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py39h4c65ebe_1|py37h21a3de8_1|py38h21a3de8_1|py38h7d8b39e_0|py39h7d8b39e_0|py37he2ba247_0|py37h74d4b33_0|py39h74d4b33_0|py38h74d4b33_0|py38h39b7dee_0|py39h39b7dee_0|py39h79a1101_0|py39h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py39hb8be1f0_1|py38hb8be1f0_1|py37hb8be1f0_1|py38hf524024_1|py310hf2716ce_1|py310h9585f30_1|py310hf2716ce_2|py39hb8be1f0_2|py39hf524024_2|py38hf524024_2|py38h1e6e340_3|py39h1e6e340_3|py37ha15fc14_3|py38ha15fc14_3|py39hf524024_0|py310h9585f30_0|py39hb8be1f0_0|py310hf2716ce_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py38ha15fc14_0|py38h31eccc5_0|py310h8e6c178_0|py310h375b286_0|py310h8e6c178_1|py38h31eccc5_0|py310h8e6c178_0|py39h1e6e340_0|py310h375b286_0|py310h375b286_0|py38h31eccc5_0|py310h8e6c178_0|py39h31eccc5_0|py39h1e6e340_0|py38h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py39h31eccc5_1|py38h31eccc5_1|py39h1e6e340_1|py310h375b286_1|py38h1e6e340_1|py39h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py310hcba007f_0|py39ha15fc14_0|py38hb8be1f0_0|py38hf524024_0|py39ha15fc14_3|py310hcba007f_3|py310h375b286_3|py37h1e6e340_3|py37hf524024_2|py310h9585f30_2|py38hb8be1f0_2|py37hb8be1f0_2|py37hf524024_1|py39hf524024_1|py310h79a1101_0|py38h2b8c604_0|py37h2b8c604_0|py38h79a1101_0|py37h39b7dee_0|py39hfae3a4d_0|py37hfae3a4d_0|py39he2ba247_0|py38hfae3a4d_0|py38he2ba247_0|py38h34387ca_0|py39h34387ca_0|py37h7d8b39e_0|py37h34387ca_0|py39h21a3de8_1|py38h4c65ebe_1|py37h4c65ebe_1|py39h0f7b65f_0|py39h2ae0177_0|py38hfa32c7d_0|py36hfa32c7d_0|py37hfa32c7d_0|py36hfa32c7d_0|py38hfa32c7d_0|py37hfa32c7d_0|py39h73d599e_4|py38ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py38hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py27hde5b4d6_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py36h2f8d375_0|py27h81de0dd_1|py36h2f8d375_1|py36h2f8d375_0|py35h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h2f8d375_0|py36h74e8950_0|py35h74e8950_0|py35h3dfced4_0|py36h7cdd4dd_0|py27h7cdd4dd_0|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py37h81de0dd_4|py27h2f8d375_4|py35h2f8d375_4|py36hde5b4d6_0|py37hde5b4d6_0|py35hdbf6ddf_4|py36hdbf6ddf_4|py36h2b20989_4|py37h2b20989_4|py27h2b20989_3|py37h2b20989_3|py36h2b20989_2|py27h2b20989_2|py37h2b20989_1|py36h2b20989_1|py27hdbf6ddf_0|py27h2b20989_0|py35h2b20989_0|py27h2b20989_0|py35h0ea5e3f_1|py36h0ea5e3f_1|py38h2f8d375_12|py38hde5b4d6_12|py37hde5b4d6_12|py27hde5b4d6_12|py37hde5b4d6_11|py37h2f8d375_11|py27h2f8d375_10|py37h2f8d375_10|py35h81de0dd_10|py36h81de0dd_10|py37h81de0dd_10|py27h81de0dd_10|py36h74e8950_10|py35h81de0dd_9|py27h81de0dd_9|py27h74e8950_9|py37h81de0dd_9|py36h81de0dd_9|py27h3dfced4_9|py37h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py35h7cdd4dd_9|py36h7cdd4dd_9|py35hdbf6ddf_8|py27hdbf6ddf_8|py27h2b20989_8|py37h2b20989_8|py37hdbf6ddf_8|py36h2b20989_8|py36hdbf6ddf_8|py27hdbf6ddf_7|py27h2b20989_7|py35h2b20989_7|py35hdbf6ddf_7|py27h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_6|py37hdbf6ddf_6|py37h2b20989_6|py36h2b20989_6|py27h2b20989_6']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py27h0ea5e3f_1|py36h9be14a7_1|py27h9be14a7_1|py35h9be14a7_1|py36h2b20989_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py27hdbf6ddf_1|py36hdbf6ddf_1|py37hdbf6ddf_1|py27h2b20989_1|py37h2b20989_2|py27hdbf6ddf_2|py37hdbf6ddf_2|py36hdbf6ddf_2|py36h2b20989_3|py37hdbf6ddf_3|py36hdbf6ddf_3|py27hdbf6ddf_3|py27h2b20989_4|py27hdbf6ddf_4|py37hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py37h2f8d375_0|py37h2f8d375_4|py27h81de0dd_4|py36h2f8d375_4|py35h81de0dd_4|py36h81de0dd_4|py38h2f8d375_4|py37hde5b4d6_5|py36hde5b4d6_5|py37h7cdd4dd_0|py35h7cdd4dd_0|py27h3dfced4_0|py36h3dfced4_0|py37h3dfced4_0|py27h74e8950_0|py37h74e8950_0|py36h81de0dd_0|py27h81de0dd_0|py37h81de0dd_0|py35h81de0dd_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py35h81de0dd_0|py36h81de0dd_0|py27h2f8d375_1|py36h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py27h2f8d375_0|py27h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_1|py36h2f8d375_1|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py38hde5b4d6_0|py27h2f8d375_0|py39h41b4c56_3|py38h41b4c56_3|py36hdc34a94_3|py39hdc34a94_3|py39ha8aedfd_4|py37h73d599e_4|py38h73d599e_4|py38h75fe3a5_0|py36h75fe3a5_0|py37h75fe3a5_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py39h4c65ebe_1|py37h21a3de8_1|py38h21a3de8_1|py38h7d8b39e_0|py39h7d8b39e_0|py37he2ba247_0|py37h74d4b33_0|py39h74d4b33_0|py38h74d4b33_0|py38h39b7dee_0|py39h39b7dee_0|py39h79a1101_0|py39h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py39hb8be1f0_1|py38hb8be1f0_1|py37hb8be1f0_1|py38hf524024_1|py310hf2716ce_1|py310h9585f30_1|py310hf2716ce_2|py39hb8be1f0_2|py39hf524024_2|py38hf524024_2|py38h1e6e340_3|py39h1e6e340_3|py37ha15fc14_3|py38ha15fc14_3|py39hf524024_0|py310h9585f30_0|py39hb8be1f0_0|py310hf2716ce_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py38ha15fc14_0|py38h31eccc5_0|py310h8e6c178_0|py310h375b286_0|py310h8e6c178_1|py38h31eccc5_0|py310h8e6c178_0|py39h1e6e340_0|py310h375b286_0|py310h375b286_0|py38h31eccc5_0|py310h8e6c178_0|py39h31eccc5_0|py39h1e6e340_0|py38h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py39h31eccc5_1|py38h31eccc5_1|py39h1e6e340_1|py310h375b286_1|py38h1e6e340_1|py39h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py310hcba007f_0|py39ha15fc14_0|py38hb8be1f0_0|py38hf524024_0|py39ha15fc14_3|py310hcba007f_3|py310h375b286_3|py37h1e6e340_3|py37hf524024_2|py310h9585f30_2|py38hb8be1f0_2|py37hb8be1f0_2|py37hf524024_1|py39hf524024_1|py310h79a1101_0|py38h2b8c604_0|py37h2b8c604_0|py38h79a1101_0|py37h39b7dee_0|py39hfae3a4d_0|py37hfae3a4d_0|py39he2ba247_0|py38hfae3a4d_0|py38he2ba247_0|py38h34387ca_0|py39h34387ca_0|py37h7d8b39e_0|py37h34387ca_0|py39h21a3de8_1|py38h4c65ebe_1|py37h4c65ebe_1|py39h0f7b65f_0|py39h2ae0177_0|py38hfa32c7d_0|py36hfa32c7d_0|py37hfa32c7d_0|py36hfa32c7d_0|py38hfa32c7d_0|py37hfa32c7d_0|py39h73d599e_4|py38ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py38hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py27hde5b4d6_0|py38h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py37hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py36h81de0dd_0|py37h81de0dd_0|py36h2f8d375_0|py36h2f8d375_0|py27h81de0dd_1|py36h2f8d375_1|py36h2f8d375_0|py35h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h2f8d375_0|py36h74e8950_0|py35h74e8950_0|py35h3dfced4_0|py36h7cdd4dd_0|py27h7cdd4dd_0|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py37h81de0dd_4|py27h2f8d375_4|py35h2f8d375_4|py36hde5b4d6_0|py37hde5b4d6_0|py35hdbf6ddf_4|py36hdbf6ddf_4|py36h2b20989_4|py37h2b20989_4|py27h2b20989_3|py37h2b20989_3|py36h2b20989_2|py27h2b20989_2|py37h2b20989_1|py36h2b20989_1|py27hdbf6ddf_0|py27h2b20989_0|py35h2b20989_0|py27h2b20989_0|py35h0ea5e3f_1|py36h0ea5e3f_1']\n\
  \nPackage setuptools conflicts for:\npython=3.7 -> pip -> setuptools\npandas=0.25\
  \ -> setuptools[version='<60.0.0']\nmatplotlib=3.1 -> setuptools\n\nPackage ca-certificates\
  \ conflicts for:\ngoatools=0.9.7 -> python -> ca-certificates\npillow=6.2 -> python[version='>=2.7,<2.8.0a0']\
  \ -> ca-certificates\npython=3.7 -> openssl[version='>=1.1.1l,<1.1.2a'] -> ca-certificates\n\
  \nPackage pandas conflicts for:\npandas=0.25\ngoatools=0.9.7 -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\n\
  \nPackage pyparsing conflicts for:\nmatplotlib=3.1 -> pyparsing\ngoatools=0.9.7\
  \ -> pydot -> pyparsing[version='>=2.1.4|>=2.1.4,<3']\n\nPackage python-dateutil\
  \ conflicts for:\nmatplotlib=3.1 -> python-dateutil\npandas=0.25 -> python-dateutil[version='>=2.6.1']\n\
  \nPackage python conflicts for:\npillow=6.2 -> olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  pillow=6.2 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  python=3.7\nmatplotlib=3.1 -> pyqt -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.6|>=3.0,<=3.9|>=3.7|>=3.8|>=3']\n\
  goatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  matplotlib=3.1 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  goatools=0.9.7 -> python\n\nPackage libpng conflicts for:\npillow=6.2 -> freetype[version='>=2.9.1,<3.0a0']\
  \ -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.37,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  matplotlib=3.1 -> matplotlib-base==3.1.1=py36he7580a8_1 -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  matplotlib=3.1 -> libpng[version='>=1.6.37,<1.7.0a0']The following specifications\
  \ were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - matplotlib=3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> __glibc[version='>=2.17']\n  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] ->\
  \ __glibc[version='>=2.17']\n  - python=3.7 -> libgcc-ng[version='>=9.4.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2F59iim0dr%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2F59iim0dr%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.260GiB</td>
						<td>1322.6 seconds</td>
						<td>1514.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743137232">
								<a href="https://nf-co.re/metaboigniter">metaboigniter</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/metaboigniter/tree/1.0.1">1.0.1</a>
						</td>
						<td>nextflow</td>
						<td>642 days</td>
						<td>2023-02-19 06:20:47</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/metaboigniter%2F0xmtwtkf%2Flogs.tar.xz">0.388GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/metaboigniter%2F0xmtwtkf%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.963GiB</td>
						<td>2393.7 seconds</td>
						<td>1895.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738320816">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.2.0">v1.2.0</a>
						</td>
						<td>snakemake</td>
						<td>641 days</td>
						<td>2023-02-19 04:10:34</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/dfmdvth4x3/singularity-tmp/tmpjjosq7a6.yaml:

  Command:

  SINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/9648df2a-71d2-4ef7-a84e-a936db030a50 singularity  exec
  --home /home/azureuser/tmp/dfmdvth4x3/code/.test  /home/azureuser/tmp/dfmdvth4x3/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg
  sh -c ''conda config --set channel_priority strict && conda env create --quiet --file
  "/home/azureuser/tmp/dfmdvth4x3/code/.test/.snakemake/conda/238f7d62f919beabb764bbd63faa52c3_.yaml"
  --prefix "/home/azureuser/tmp/dfmdvth4x3/code/.test/.snakemake/conda/238f7d62f919beabb764bbd63faa52c3_"''

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working... Killed'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fp6ontat8%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fp6ontat8%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>11.185GiB</td>
						<td>889.0 seconds</td>
						<td>1133.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743113344">
								<a href="https://nf-co.re/hic">hic</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hic/tree/1.3.0">1.3.0</a>
						</td>
						<td>nextflow</td>
						<td>635 days</td>
						<td>2023-02-19 04:00:01</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2F8b3uv9hj%2Flogs.tar.xz">0.211GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2F8b3uv9hj%2Foutput.tar.xz">0.036GiB</a>
						</td>
						<td>1.706GiB</td>
						<td>769.3 seconds</td>
						<td>377.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738320576">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.3.0">v1.3.0</a>
						</td>
						<td>snakemake</td>
						<td>632 days</td>
						<td>2023-02-19 03:31:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/hnrin0okn5/singularity-tmp/tmpbvm3if5m.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/a57d80f1-4613-41c7-8408-77af3930229f\
  \ singularity  exec --home /home/azureuser/tmp/hnrin0okn5/code/.test  /home/azureuser/tmp/hnrin0okn5/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/hnrin0okn5/code/.test/.snakemake/conda/a7cbd2b33577065999d9fa5797d0f70f_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/hnrin0okn5/code/.test/.snakemake/conda/a7cbd2b33577065999d9fa5797d0f70f_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:  11%|\u2588\
  \         | 1/9 [00:00<00:00, 6384.02it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 7619.08it/s]  \nExamining freebayes=1.3.1:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 8548.17it/s]          \
  \  \nExamining sed=4.7:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,\
  \ 30.53it/s]          \nExamining sed=4.7:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:00<00:00, 35.91it/s]\nExamining parallel=20190522:  56%|\u2588\u2588\
  \u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 35.91it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 35.91it/s]\n\
  Examining bedtools[version='>=2.29']:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:00<00:00, 35.91it/s]      \nExamining bcftools=1.11:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00, 35.91it/s]\
  \             \nExamining bcftools=1.11: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588| 9/9 [00:00<00:00, 31.58it/s]\n                       \
  \                                               \n\nDetermining conflicts:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining conflict for __glibc freebayes sed\
  \ bedtools bcftools:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for\
  \ freebayes sed parallel bedtools bcftools:  11%|\u2588         | 1/9 [00:00<00:01,\
  \  4.48it/s]\nExamining conflict for freebayes sed parallel bedtools bcftools: \
  \ 22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,  8.96it/s]\nExamining conflict\
  \ for freebayes sed bcftools bedtools:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \  8.96it/s]         \nExamining conflict for freebayes sed bcftools bedtools: \
  \ 33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00,  6.49it/s]\nExamining conflict\
  \ for freebayes bcftools bedtools:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00,\
  \  6.49it/s]    \nExamining conflict for freebayes bcftools bedtools:  44%|\u2588\
  \u2588\u2588\u2588\u258D     | 4/9 [00:00<00:01,  4.02it/s]\nExamining conflict\
  \ for freebayes bedtools:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:02<00:01,\
  \  4.02it/s]         \nExamining conflict for freebayes bedtools:  56%|\u2588\u2588\
  \u2588\u2588\u2588\u258C    | 5/9 [00:02<00:02,  1.48it/s]\nExamining conflict for\
  \ freebayes bcftools:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:02<00:02,\
  \  1.48it/s]\nExamining conflict for freebayes bcftools:  67%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258B   | 6/9 [00:02<00:01,  1.85it/s]\nExamining conflict for\
  \ freebayes parallel bcftools:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B \
  \  | 6/9 [00:02<00:01,  1.85it/s]\nExamining conflict for freebayes parallel:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:02<00:01,  1.85it/s]     \
  \    \nExamining conflict for freebayes parallel:  89%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2589 | 8/9 [00:02<00:00,  2.72it/s]\nExamining conflict for\
  \ parallel bcftools:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589\
  \ | 8/9 [00:02<00:00,  2.72it/s] \n                                            \
  \                                           \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|          | 0/9 [00:00<?, ?it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  11%|\u2588         | 1/9 [00:00<00:00,\
  \ 7598.38it/s]\nExamining @/linux-64::__glibc==2.31=0:  22%|\u2588\u2588\u258F \
  \      | 2/9 [00:00<00:00, 8152.19it/s]  \nExamining freebayes=1.3.1:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:00<00:00, 8750.29it/s]            \nExamining\
  \ sed=4.7:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00, 53.68it/s]\
  \          \nExamining parallel=20190522:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:00<00:00, 63.17it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 75.20it/s]\n\
  Examining bedtools[version='>=2.29']:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:00<00:00, 87.57it/s]      \nExamining bcftools=1.11:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00, 55.47it/s]\
  \             \nExamining bcftools=1.11: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588| 9/9 [00:00<00:00, 62.33it/s]\n                       \
  \                                               \n\nDetermining conflicts:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining conflict for __glibc freebayes sed\
  \ bedtools bcftools:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for\
  \ freebayes sed parallel bedtools bcftools:  11%|\u2588         | 1/9 [00:00<00:01,\
  \  5.28it/s]\nExamining conflict for freebayes sed parallel bedtools bcftools: \
  \ 22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 10.55it/s]\nExamining conflict\
  \ for freebayes sed bcftools bedtools:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \ 10.55it/s]         \nExamining conflict for freebayes bcftools bedtools:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:00<00:00, 10.55it/s]    \nExamining conflict for\
  \ freebayes bcftools bedtools:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:01,\
  \  4.73it/s]\nExamining conflict for freebayes bedtools:  44%|\u2588\u2588\u2588\
  \u2588\u258D     | 4/9 [00:01<00:01,  4.73it/s]         \nExamining conflict for\
  \ freebayes bedtools:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:01<00:01,\
  \  3.06it/s]\nExamining conflict for freebayes bcftools:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:01<00:01,  3.06it/s]\nExamining conflict for freebayes\
  \ bcftools:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:01<00:00,\
  \  3.61it/s]\nExamining conflict for freebayes parallel bcftools:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:01<00:00,  3.61it/s]\nExamining conflict\
  \ for freebayes parallel:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [00:01<00:00,  3.61it/s]         \nExamining conflict for freebayes parallel:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:01<00:00,\
  \  5.29it/s]\nExamining conflict for parallel bcftools:  89%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2589 | 8/9 [00:01<00:00,  5.29it/s] \n               \
  \                                                                        \nfailed\n\
  \nUnsatisfiableError: The following specifications were found to be incompatible\
  \ with each other:\n\nOutput in format: Requested package -> Available versions\n\
  \nPackage libcurl conflicts for:\nfreebayes=1.3.1 -> htslib -> libcurl[version='>=7.64.1,<8.0a0|>=7.71.1,<8.0a0|>=7.75.0,<8.0a0|>=7.77.0,<8.0a0|>=7.79.1,<8.0a0|>=7.81.0,<8.0a0|>=7.82.0,<8.0a0|>=7.83.1,<8.0a0']\n\
  bcftools=1.11 -> htslib[version='>=1.11,<1.12.0a0'] -> libcurl[version='>=7.71.1,<8.0a0']\n\
  bedtools[version='>=2.29'] -> curl[version='>=7.64.1,<8.0a0'] -> libcurl[version='7.64.1|7.64.1|7.64.1|7.65.2|7.65.3|7.68.0|7.68.0|7.69.1|7.71.0|7.71.1|7.71.1|7.71.1|7.71.1|7.71.1|7.71.1|7.71.1|7.71.1|7.75.0|7.76.0|7.76.1|7.76.1|7.76.1|7.77.0|7.78.0|7.79.0|7.79.1|7.79.1|7.79.1|7.80.0|7.80.0|7.80.0|7.81.0|7.81.0|7.82.0|7.82.0|7.83.0|7.83.0|7.83.1|7.83.1|7.85.0|7.85.0|7.86.0|7.86.0|7.86.0|7.86.0|7.86.0|7.86.0|7.87.0|7.87.0|7.88.0|7.87.0|7.86.0|7.85.0|7.84.0|7.83.0|7.82.0|7.80.0|7.78.0|7.71.1|7.71.1|7.71.0|7.69.1|7.68.0|7.67.0|7.65.3|7.65.2',build='h20c2e04_1|h303737a_2|h0b77cf5_0|h0b77cf5_0|h0b77cf5_0|h0b77cf5_0|h91b91d3_0|h91b91d3_0|h91b91d3_0|hcdd3856_1|hcdd3856_4|hcdd3856_5|h2574ce0_0|h2574ce0_0|h2574ce0_0|h2574ce0_0|h2574ce0_1|h2574ce0_0|h2574ce0_0|h7bff187_0|h7bff187_0|h2283fc2_0|h7bff187_0|h2283fc2_0|h7bff187_0|h7bff187_0|hdc1c0ab_2|hdc1c0ab_0|h6312ad2_0|hdc1c0ab_0|h6312ad2_2|h2283fc2_1|h7bff187_1|h2283fc2_0|h2283fc2_0|h2283fc2_0|h494985f_0|h494985f_1|h494985f_0|h494985f_1|h2574ce0_2|hc4aaa36_1|hc4aaa36_0|hc4aaa36_0|hc4aaa36_0|hcdd3856_8|hcdd3856_6|hcdd3856_3|hcdd3856_2|hcdd3856_0|hcdd3856_0|hf7181ac_0|hf7181ac_1|hda55be3_0|hda55be3_0|hda55be3_0|hf7181ac_1|hda55be3_0|h91b91d3_0|h20c2e04_0|h20c2e04_0|h20c2e04_0|h20c2e04_0|h20c2e04_0|h20c2e04_0|h20c2e04_0']\n\
  \nPackage _libgcc_mutex conflicts for:\nbedtools[version='>=2.29'] -> libgcc-ng[version='>=10.3.0']\
  \ -> _libgcc_mutex[version='*|0.1',build='main|conda_forge|main']\nsed=4.7 -> libgcc-ng[version='>=7.3.0']\
  \ -> _libgcc_mutex[version='*|0.1|0.1',build='main|conda_forge|main']\nbcftools=1.11\
  \ -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='main|conda_forge|main']\n\
  freebayes=1.3.1 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='main|conda_forge|main']\n\
  \nPackage curl conflicts for:\nfreebayes=1.3.1 -> htslib -> curl[version='>=7.59.0,<8.0a0|>=7.64.0,<8.0a0|>=7.64.1,<8.0a0|>=7.81.0,<8.0a0|>=7.75.0,<8.0a0|>=7.71.1,<8.0a0|>=7.78.0,<8.0a0']\n\
  bedtools[version='>=2.29'] -> curl[version='>=7.64.1,<8.0a0']\n\nPackage zlib conflicts\
  \ for:\nfreebayes=1.3.1 -> htslib -> zlib[version='1.2.*|1.2.11|1.2.11.*|>=1.2.12,<1.3.0a0|1.2.8|>=1.2.13,<1.3.0a0']\n\
  freebayes=1.3.1 -> zlib[version='>=1.2.11,<1.3.0a0']\n\nPackage perl conflicts for:\n\
  bcftools=1.11 -> perl\nfreebayes=1.3.1 -> parallel -> perl[version='5.20.3.*|5.20.3.1.*|5.22.0.*|>5.0.0|5.26.0.*|5.22.2.1.*|5.22.0.1.*']\n\
  parallel=20190522 -> perl\n\nPackage parallel conflicts for:\nfreebayes=1.3.1 ->\
  \ parallel\nparallel=20190522\n\nPackage htslib conflicts for:\nbcftools=1.11 ->\
  \ htslib[version='>=1.11,<1.12.0a0']\nfreebayes=1.3.1 -> htslib\n\nPackage libgcc-ng\
  \ conflicts for:\nbcftools=1.11 -> gsl[version='>=2.6,<2.7.0a0'] -> libgcc-ng[version='>=10.3.0|>=11.2.0|>=4.9|>=7.3.0|>=9.3.0|>=9.4.0|>=7.2.0|>=12']\n\
  freebayes=1.3.1 -> libgcc-ng[version='>=7.3.0']\nbcftools=1.11 -> libgcc-ng[version='>=7.5.0']\n\
  sed=4.7 -> libgcc-ng[version='>=7.3.0']\nbedtools[version='>=2.29'] -> libgcc-ng[version='>=10.3.0|>=9.4.0|>=9.3.0|>=7.5.0|>=7.3.0']\n\
  parallel=20190522 -> perl -> libgcc-ng[version='>=11.2.0|>=4.9|>=7.3.0|>=7.5.0|>=9.3.0|>=9.4.0|>=7.2.0']\n\
  freebayes=1.3.1 -> bzip2[version='>=1.0.6,<2.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=4.9|>=7.5.0|>=9.3.0|>=7.2.0|>=9.4.0|>=11.2.0']\n\
  bedtools[version='>=2.29'] -> libzlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=11.2.0|>=12|>=4.9|>=7.2.0']\n\
  \nPackage gdbm conflicts for:\nparallel=20190522 -> perl -> gdbm[version='>=1.18|>=1.18,<1.19.0a0']\n\
  bcftools=1.11 -> perl -> gdbm[version='>=1.18|>=1.18,<1.19.0a0']\n\nPackage libstdcxx-ng\
  \ conflicts for:\nfreebayes=1.3.1 -> libstdcxx-ng[version='>=7.3.0']\nfreebayes=1.3.1\
  \ -> python[version='>=2.7,<2.8.0a0'] -> libstdcxx-ng[version='>=4.9|>=7.2.0|>=9.4.0|>=9.3.0|>=7.5.0']\n\
  \nPackage xz conflicts for:\nfreebayes=1.3.1 -> htslib -> xz[version='5.2.*|>=5.2.3,<5.3.0a0|>=5.2.5,<5.3.0a0|>=5.2.6,<5.3.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0|>=5.2.10,<6.0a0|>=5.2.6,<6.0a0']\n\
  freebayes=1.3.1 -> xz[version='>=5.2.4,<5.3.0a0']\n\nPackage bzip2 conflicts for:\n\
  freebayes=1.3.1 -> bzip2[version='>=1.0.6,<2.0a0']\nfreebayes=1.3.1 -> htslib ->\
  \ bzip2[version='1.0.*|>=1.0.8,<2.0a0']The following specifications were found to\
  \ be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - bcftools=1.11 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \  - bedtools[version='>=2.29'] -> libgcc-ng[version='>=10.3.0'] -> __glibc[version='>=2.17']\n\
  \  - freebayes=1.3.1 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - sed=4.7 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\nYour\
  \ installed version is: 2.31\n\nNote that strict channel priority may have removed\
  \ packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fcb3sui9k%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fcb3sui9k%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.289GiB</td>
						<td>949.0 seconds</td>
						<td>1036.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738320336">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.3.1">v1.3.1</a>
						</td>
						<td>snakemake</td>
						<td>632 days</td>
						<td>2023-02-19 03:33:55</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/gscjfd6m5r/singularity-tmp/tmp0_5g4lp5.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/52226efc-b550-4aaa-8bb2-cf97ac13e0fb\
  \ singularity  exec --home /home/azureuser/tmp/gscjfd6m5r/code/.test  /home/azureuser/tmp/gscjfd6m5r/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/gscjfd6m5r/code/.test/.snakemake/conda/12d469a4662c017048330b5bfb2dee37_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/gscjfd6m5r/code/.test/.snakemake/conda/12d469a4662c017048330b5bfb2dee37_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining bcftools=1.10:   0%|        \
  \  | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:  17%|\u2588\u258B\
  \        | 1/6 [00:00<00:00, 26.09it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 51.96it/s]  \nExamining\
  \ curl:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00, 77.74it/s] \
  \                      \nExamining @/linux-64::__archspec==1=x86_64:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 4/6 [00:00<00:00,  6.92it/s]\nExamining\
  \ @/linux-64::__archspec==1=x86_64:  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258E | 5/6 [00:00<00:00,  8.65it/s]\nExamining @/linux-64::__unix==0=0:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \  8.65it/s]         \n                                                        \
  \                        \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for bcftools curl:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining conflict for bcftools __glibc curl:  17%|\u2588\u258B        | 1/6 [00:00<00:00,\
  \ 136.34it/s]\n                                                                \
  \                            \nfailed\nSolving environment: ...working... \nFound\
  \ conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining bcftools=1.10:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ @/linux-64::__linux==5.15.0=0:  17%|\u2588\u258B        | 1/6 [00:00<00:00, 26.37it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  33%|\u2588\u2588\u2588\u258E      | 2/6\
  \ [00:00<00:00, 52.53it/s]  \nExamining curl:  50%|\u2588\u2588\u2588\u2588\u2588\
  \     | 3/6 [00:00<00:00, 78.57it/s]                       \nExamining @/linux-64::__archspec==1=x86_64:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 4/6 [00:00<00:00,  7.08it/s]\n\
  Examining @/linux-64::__archspec==1=x86_64:  83%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258E | 5/6 [00:00<00:00,  8.84it/s]\nExamining @/linux-64::__unix==0=0:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \  8.84it/s]         \n                                                        \
  \                        \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for bcftools curl:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining conflict for bcftools __glibc curl:  17%|\u2588\u258B        | 1/6 [00:00<00:00,\
  \ 136.98it/s]\n                                                                \
  \                            \nfailed\n\nUnsatisfiableError: The following specifications\
  \ were found to be incompatible with each other:\n\nOutput in format: Requested\
  \ package -> Available versions\n\nPackage curl conflicts for:\ncurl\nbcftools=1.10\
  \ -> htslib[version='>=1.10,<1.11.0a0'] -> curl[version='>=7.64.1,<8.0a0']The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - bcftools=1.10 -> libgcc-ng[version='>=7.5.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fzpeytgsw%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fzpeytgsw%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.371GiB</td>
						<td>985.9 seconds</td>
						<td>1252.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738320096">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.3.2">v1.3.2</a>
						</td>
						<td>snakemake</td>
						<td>632 days</td>
						<td>2023-02-19 03:31:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/9z5qy0f7k8/singularity-tmp/tmpi6_qgm6h.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/cdb42d4b-b3fd-4eab-9826-f4d3e76c4d18\
  \ singularity  exec --home /home/azureuser/tmp/9z5qy0f7k8/code/.test  /home/azureuser/tmp/9z5qy0f7k8/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/9z5qy0f7k8/code/.test/.snakemake/conda/5d7c5fa5ac3e09bc6e23d3346c598c9f_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/9z5qy0f7k8/code/.test/.snakemake/conda/5d7c5fa5ac3e09bc6e23d3346c598c9f_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining htslib==1.10:   0%|         \
  \ | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:  20%|\u2588\u2588 \
  \       | 1/5 [00:00<00:00, 29.19it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 58.13it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00, 86.96it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  80%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588  | 4/5 [00:00<00:00, 115.65it/s]    \n                            \
  \                                                         \n\nDetermining conflicts:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict for htslib __glibc:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\n                                      \
  \                                      \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/5 [00:00<?,\
  \ ?it/s]\nExamining htslib==1.10:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining\
  \ @/linux-64::__unix==0=0:  20%|\u2588\u2588        | 1/5 [00:00<00:00, 18.37it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  40%|\u2588\u2588\u2588\u2588      | 2/5\
  \ [00:00<00:00, 31.85it/s]\nExamining @/linux-64::__archspec==1=x86_64:  60%|\u2588\
  \u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00, 47.64it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  80%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 63.35it/s]\
  \     \n                                                                       \
  \             \n\nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\n\
  Examining conflict for htslib __glibc:   0%|          | 0/5 [00:00<?, ?it/s]\n \
  \                                                                           \nfailed\n\
  \nUnsatisfiableError: The following specifications were found to be incompatible\
  \ with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n  - htslib==1.10\
  \ -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\nYour installed\
  \ version is: 2.31\n\nNote that strict channel priority may have removed packages\
  \ required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fkwslwrcp%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fkwslwrcp%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.173GiB</td>
						<td>333.0 seconds</td>
						<td>587.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743119344">
								<a href="https://nf-co.re/mag">mag</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mag/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>628 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: Include statement is not allowed within a workflow definition
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.script.BaseScript.include(BaseScript.groovy:150)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)\n\
  \tat groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:424)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat Script_6712bf69$_runScript_closure1$_closure2.doCall(Script_6712bf69:55)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:205)\n\
  \tat nextflow.script.WorkflowDef.run(WorkflowDef.groovy:189)\n\tat nextflow.script.BindableDef.invoke_a(BindableDef.groovy:52)\n\
  \tat nextflow.script.IterableDef$invoke_a.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:208)\n\tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\
  \tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fkwmc3c35%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fkwmc3c35%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.544GiB</td>
						<td>21.4 seconds</td>
						<td>20.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738163936">
								<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling">nikostr/dna-seq-deepvariant-glnexus-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling/tree/v0.2.0">v0.2.0</a>
						</td>
						<td>snakemake</td>
						<td>628 days</td>
						<td>2023-02-19 03:32:03</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/arlod1i3cb/singularity-tmp/tmpgk5qa7i3.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/arlod1i3cb/code/.snakemake/conda/0e53a080460aae12dfcb7878f17aaa48_.yaml"
  --prefix "/home/azureuser/tmp/arlod1i3cb/code/.snakemake/conda/0e53a080460aae12dfcb7878f17aaa48_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working...'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nikostr%252Fdna-seq-deepvariant-glnexus-variant-calling%2F9kbko822%2Flogs.tar.xz">0.155GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nikostr%252Fdna-seq-deepvariant-glnexus-variant-calling%2F9kbko822%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>9.834GiB</td>
						<td>1059.4 seconds</td>
						<td>1711.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738163696">
								<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling">nikostr/dna-seq-deepvariant-glnexus-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling/tree/v0.3.0">v0.3.0</a>
						</td>
						<td>snakemake</td>
						<td>625 days</td>
						<td>2023-02-19 11:59:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/3yyjb2b1l9/singularity-tmp/tmpx2phuk4u.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/3yyjb2b1l9/code/.snakemake/conda/200f6ea900008dbe45569151829e5c68_.yaml"
  --prefix "/home/azureuser/tmp/3yyjb2b1l9/code/.snakemake/conda/200f6ea900008dbe45569151829e5c68_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working...'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nikostr%252Fdna-seq-deepvariant-glnexus-variant-calling%2Fiwivz54b%2Flogs.tar.xz">2.460GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nikostr%252Fdna-seq-deepvariant-glnexus-variant-calling%2Fiwivz54b%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>13.164GiB</td>
						<td>1603.0 seconds</td>
						<td>2239.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743104944">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.3.5">2.3.5</a>
						</td>
						<td>nextflow</td>
						<td>625 days</td>
						<td>2023-02-19 04:16:26</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.Nextflow
date: Feb-19 04:16:44.501
kind: NextflowMiscError
process: main
rest: '[nf-core/eager] error: --input was not supplied! Please check ''--help'' or
  documentation under ''running the pipeline'' for details'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fszmewi4h%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fszmewi4h%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.639GiB</td>
						<td>34.0 seconds</td>
						<td>18.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738319856">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.4.0">v1.4.0</a>
						</td>
						<td>snakemake</td>
						<td>621 days</td>
						<td>2023-02-19 06:00:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/nm0fonhe0q/singularity-tmp/tmp6qhxiy0a.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/c004e9cc-be7a-4cfe-a015-dda866bca6eb\
  \ singularity  exec --home /home/azureuser/tmp/nm0fonhe0q/code/.test  /home/azureuser/tmp/nm0fonhe0q/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/nm0fonhe0q/code/.test/.snakemake/conda/7e1d690137f5acd9316537cdeeae70bd_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/nm0fonhe0q/code/.test/.snakemake/conda/7e1d690137f5acd9316537cdeeae70bd_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  11%|\u2588         | 1/9 [00:00<00:00, 4132.32it/s]\nExamining libgcc-ng[version='>=7.3.0']:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 5038.20it/s]\nExamining htslib[version='>=1.10,<1.11.0a0']:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 53.00it/s]\nExamining zlib[version='>=1.2.11,<1.3.0a0']:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00, 27.23it/s] \nExamining\
  \ zlib[version='>=1.2.11,<1.3.0a0']:  56%|\u2588\u2588\u2588\u2588\u2588\u258C \
  \   | 5/9 [00:00<00:00, 33.99it/s]\nExamining perl:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:00<00:00, 33.99it/s]                             \nExamining\
  \ @/linux-64::__linux==5.15.0=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B\
  \   | 6/9 [00:00<00:00, 33.99it/s]\nExamining @/linux-64::__unix==0=0:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:00<00:00, 33.99it/s]     \
  \ \nExamining gsl[version='>=2.5,<2.6.0a0']:  89%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00, 33.99it/s]\nExamining gsl[version='>=2.5,<2.6.0a0']:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:00<00:00,\
  \ 22.77it/s]\n                                                                 \
  \                     \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for __glibc libgcc-ng htslib zlib perl gsl:   0%| \
  \         | 0/9 [00:00<?, ?it/s]\nExamining conflict for libgcc-ng htslib zlib perl\
  \ gsl:  11%|\u2588         | 1/9 [00:00<00:01,  6.91it/s]\nExamining conflict for\
  \ libgcc-ng htslib zlib perl gsl:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \ 13.81it/s]\nExamining conflict for htslib zlib:  22%|\u2588\u2588\u258F      \
  \ | 2/9 [00:00<00:00, 13.81it/s]                   \n                          \
  \                                                       \nfailed\nSolving environment:\
  \ ...working... \nFound conflicts! Looking for incompatible packages.\nThis can\
  \ take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64: \
  \  0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0: \
  \ 11%|\u2588         | 1/9 [00:00<00:00, 5769.33it/s]\nExamining libgcc-ng[version='>=7.3.0']:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 6482.70it/s]\nExamining htslib[version='>=1.10,<1.11.0a0']:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 66.19it/s]\nExamining zlib[version='>=1.2.11,<1.3.0a0']:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00, 33.78it/s] \nExamining\
  \ zlib[version='>=1.2.11,<1.3.0a0']:  56%|\u2588\u2588\u2588\u2588\u2588\u258C \
  \   | 5/9 [00:00<00:00, 42.17it/s]\nExamining perl:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:00<00:00, 42.17it/s]                             \nExamining\
  \ @/linux-64::__linux==5.15.0=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B\
  \   | 6/9 [00:00<00:00, 42.17it/s]\nExamining @/linux-64::__unix==0=0:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:00<00:00, 42.17it/s]     \
  \ \nExamining gsl[version='>=2.5,<2.6.0a0']:  89%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00, 42.17it/s]\n                      \
  \                                                                \n\nDetermining\
  \ conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for __glibc\
  \ libgcc-ng htslib zlib perl gsl:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for libgcc-ng htslib zlib perl gsl:  11%|\u2588         | 1/9 [00:00<00:01,\
  \  7.55it/s]\nExamining conflict for libgcc-ng htslib zlib perl gsl:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:00<00:00, 15.09it/s]\nExamining conflict for htslib\
  \ zlib:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 15.09it/s]           \
  \        \n                                                                    \
  \             \nfailed\n\nUnsatisfiableError: The following specifications were\
  \ found to be incompatible with each other:\n\nOutput in format: Requested package\
  \ -> Available versions\n\nPackage libgcc-ng conflicts for:\nzlib[version='>=1.2.11,<1.3.0a0']\
  \ -> libgcc-ng[version='>=10.3.0|>=12|>=7.5.0|>=7.3.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> bzip2[version='>=1.0.8,<2.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=9.3.0|>=9.4.0|>=11.2.0|>=4.9|>=7.2.0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> libgcc-ng[version='>=7.3.0|>=7.5.0']\ngsl[version='>=2.5,<2.6.0a0']\
  \ -> libgcc-ng[version='>=7.3.0']\nlibgcc-ng[version='>=7.3.0']\ngsl[version='>=2.5,<2.6.0a0']\
  \ -> libblas[version='>=3.8.0,<4.0a0'] -> libgcc-ng[version='>=7.5.0|>=9.3.0']\n\
  perl -> libgcc-ng[version='>=11.2.0|>=4.9|>=7.3.0|>=7.5.0|>=9.3.0|>=9.4.0|>=7.2.0']\n\
  \nPackage zlib conflicts for:\nzlib[version='>=1.2.11,<1.3.0a0']\nhtslib[version='>=1.10,<1.11.0a0']\
  \ -> libcurl[version='>=7.71.1,<8.0a0'] -> zlib[version='>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0']The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - gsl[version='>=2.5,<2.6.0a0'] -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - htslib[version='>=1.10,<1.11.0a0'] -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \  - libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n  - perl -> libgcc-ng[version='>=9.4.0']\
  \ -> __glibc[version='>=2.17']\n  - zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=10.3.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fc7ha9u72%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fc7ha9u72%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.227GiB</td>
						<td>857.8 seconds</td>
						<td>942.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743112384">
								<a href="https://nf-co.re/fetchngs">fetchngs</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/fetchngs/tree/1.0">1.0</a>
						</td>
						<td>nextflow</td>
						<td>621 days</td>
						<td>2023-02-19 06:45:30</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: Include statement is not allowed within a workflow definition
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.script.BaseScript.include(BaseScript.groovy:150)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)\n\
  \tat groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:424)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat Script_50940136$_runScript_closure1$_closure3.doCall(Script_50940136:33)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:205)\n\
  \tat nextflow.script.WorkflowDef.run(WorkflowDef.groovy:189)\n\tat nextflow.script.BindableDef.invoke_a(BindableDef.groovy:52)\n\
  \tat nextflow.script.ComponentDef.invoke_o(ComponentDef.groovy:41)\n\tat nextflow.script.WorkflowBinding.invokeMethod(WorkflowBinding.groovy:94)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:408)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat Script_50940136$_runScript_closure2$_closure4.doCall(Script_50940136:48)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:205)\n\
  \tat nextflow.script.WorkflowDef.run(WorkflowDef.groovy:189)\n\tat nextflow.script.BindableDef.invoke_a(BindableDef.groovy:52)\n\
  \tat nextflow.script.ChainableDef$invoke_a.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:208)\n\tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\
  \tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fmxuy3dps%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fmxuy3dps%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.404GiB</td>
						<td>22.1 seconds</td>
						<td>20.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743190464">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/2.7.1">2.7.1</a>
						</td>
						<td>nextflow</td>
						<td>615 days</td>
						<td>2023-02-19 03:31:05</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: nextflow.exception.AbortOperationException
msg: 'DAG file already exists: /home/azureuser/tmp/1x4e7lmh39/out/pipeline_info/pipeline_dag_2023-02-19_03-31-19.svg
  -- enable `dag.overwrite` in your config file to overwrite existing DAG files'
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.trace.GraphObserver.onFlowCreate(GraphObserver.groovy:77)\n\tat nextflow.Session$_notifyFlowCreate_closure22.doCall(Session.groovy:1055)\n\
  \tat nextflow.Session$_notifyFlowCreate_closure22.call(Session.groovy)\n\tat org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2359)\n\
  \tat org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2344)\n\
  \tat org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2385)\n\
  \tat nextflow.Session.notifyFlowCreate(Session.groovy:1055)\n\tat nextflow.Session.start(Session.groovy:517)\n\
  \tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:126)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fordjjam4%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fordjjam4%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.404GiB</td>
						<td>17.3 seconds</td>
						<td>15.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743195744">
								<a href="https://nf-co.re/viralrecon">viralrecon</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/viralrecon/tree/2.1">2.1</a>
						</td>
						<td>nextflow</td>
						<td>615 days</td>
						<td>2023-02-20 05:31:53</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 08:31:53.846
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Fr72lohbo%2Flogs.tar.xz">0.095GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Fr72lohbo%2Foutput.tar.xz">0.018GiB</a>
						</td>
						<td>1.283GiB</td>
						<td>504.6 seconds</td>
						<td>10802.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743193824">
								<a href="https://nf-co.re/smrnaseq">smrnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/smrnaseq/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>614 days</td>
						<td>2023-02-19 04:31:30</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: edgeR_miRBase.r sample_3.mature.stats sample_2.mature.stats sample_2.hairpin.stats
  sample_3.hairpin.stats sample_1.mature.stats sample_1.hairpin.stats
error: "Loading required package: limma\nLoading required package: edgeR\nLoading\
  \ required package: statmod\nLoading required package: data.table\nLoading required\
  \ package: gplots\n\nAttaching package: \u2018gplots\u2019\n\nThe following object\
  \ is masked from \u2018package:stats\u2019:\n\n    lowess\n\n$mature\n[1] \"sample_3.mature.stats\"\
  \ \"sample_2.mature.stats\" \"sample_1.mature.stats\"\n\n$hairpin\n[1] \"sample_2.hairpin.stats\"\
  \ \"sample_3.hairpin.stats\" \"sample_1.hairpin.stats\"\n\nError in heatmap.2(cpm(dataNorm),\
  \ col = redgreen(100), key = TRUE, scale = \"row\",  : \n  `x' must be a numeric\
  \ matrix\nIn addition: Warning message:\nIn DGEList(counts = data, genes = rownames(data))\
  \ :\n  library size of zero detected\nExecution halted"
exit_status: 1
kind: CalledProcessError
output: '$mature

  [1] "sample_3.mature.stats" "sample_2.mature.stats" "sample_1.mature.stats"


  $hairpin

  [1] "sample_2.hairpin.stats" "sample_3.hairpin.stats" "sample_1.hairpin.stats"'
process: edgeR_mirna
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- 8e
- e0d6defa72c6280e955c6863364760

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/smrnaseq%2Fi57psnpl%2Flogs.tar.xz">0.095GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/smrnaseq%2Fi57psnpl%2Foutput.tar.xz">0.009GiB</a>
						</td>
						<td>1.635GiB</td>
						<td>740.1 seconds</td>
						<td>433.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743914288">
								<a href="https://nf-co.re/bactmap">bactmap</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/bactmap/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>611 days</td>
						<td>2023-02-19 04:10:28</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.NullPointerException
msg: Cannot get property 'args' on null object
rest: "at org.codehaus.groovy.runtime.NullObject.getProperty(NullObject.java:60)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:194)\n\
  \tat org.codehaus.groovy.runtime.callsite.NullCallSite.getProperty(NullCallSite.java:46)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:329)\n\
  \tat Script_c99ee09b.runScript(Script_c99ee09b:38)\n\tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:170)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:216)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:206)\n\
  \tat nextflow.script.IncludeDef.memoizedMethodPriv$loadModule0PathMapSession(IncludeDef.groovy:152)\n\
  \tat nextflow.script.IncludeDef.access$0(IncludeDef.groovy)\n\tat nextflow.script.IncludeDef$__clinit__closure2.doCall(IncludeDef.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat nextflow.script.IncludeDef.loadModule0(IncludeDef.groovy)\n\tat nextflow.script.IncludeDef.load0(IncludeDef.groovy:124)\n\
  \tat nextflow.script.IncludeDef$load0$1.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_14934cd2.runScript(Script_14934cd2:28)\n\tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:170)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\
  \tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\
  \tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/bactmap%2Fmcp3t1v9%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/bactmap%2Fmcp3t1v9%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.522GiB</td>
						<td>20.4 seconds</td>
						<td>9.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743185904">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.2">3.2</a>
						</td>
						<td>nextflow</td>
						<td>611 days</td>
						<td>2023-02-19 03:31:32</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.NullPointerException
msg: Cannot get property 'args' on null object
rest: "at org.codehaus.groovy.runtime.NullObject.getProperty(NullObject.java:60)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:194)\n\
  \tat org.codehaus.groovy.runtime.callsite.NullCallSite.getProperty(NullCallSite.java:46)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:329)\n\
  \tat Script_5d239e76.runScript(Script_5d239e76:80)\n\tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:170)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:216)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:206)\n\
  \tat nextflow.script.IncludeDef.memoizedMethodPriv$loadModule0PathMapSession(IncludeDef.groovy:152)\n\
  \tat nextflow.script.IncludeDef.access$0(IncludeDef.groovy)\n\tat nextflow.script.IncludeDef$__clinit__closure2.doCall(IncludeDef.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat nextflow.script.IncludeDef.loadModule0(IncludeDef.groovy)\n\tat nextflow.script.IncludeDef.load0(IncludeDef.groovy:124)\n\
  \tat nextflow.script.IncludeDef$load0$1.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_4c643811.runScript(Script_4c643811:43)\n\tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:170)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\
  \tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\
  \tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F7by3cuj7%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F7by3cuj7%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.599GiB</td>
						<td>29.2 seconds</td>
						<td>23.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743112144">
								<a href="https://nf-co.re/fetchngs">fetchngs</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/fetchngs/tree/1.1">1.1</a>
						</td>
						<td>nextflow</td>
						<td>608 days</td>
						<td>2023-02-20 04:31:44</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867535702\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\t\
  at nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\
  \tat nextflow.processor.TaskRun$isContainerEnabled$2.call(Unknown Source)\n\tat\
  \ nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867535702\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 07:31:45.006 [Actor Thread 12] DEBUG nextflow.file.SortFileCollector\
  \ - FileCollector temp dir not removed: null\nFeb-20 07:31:45.069 [main] INFO  nextflow.Nextflow\
  \ - -\e[0;35m[nf-core/fetchngs]\e[0;31m Pipeline completed with errors\e[0m-\nFeb-20\
  \ 07:31:45.070 [main] WARN  nextflow.Nextflow - =============================================================================\n\
  \  Please double-check the samplesheet that has been auto-created by the pipeline.\n\
  \n  Public databases don't reliably hold information such as strandedness\n  information,\
  \ controls etc\n\n  All of the sample metadata obtained from the ENA has been appended\n\
  \  as additional columns to help you manually curate the samplesheet before\n  running\
  \ nf-core/other pipelines.\n===================================================================================\n\
  Feb-20 07:31:45.074 [main] DEBUG nextflow.trace.WorkflowStatsObserver - Workflow\
  \ completed > WorkflowStats[succeededCount=0; failedCount=0; ignoredCount=0; cachedCount=0;\
  \ pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0; abortedCount=0;\
  \ succeedDuration=0ms; failedDuration=0ms; cachedDuration=0ms;loadCpus=0; loadMemory=0;\
  \ peakRunning=0; peakCpus=0; peakMemory=0; ]\nFeb-20 07:31:45.074 [main] DEBUG nextflow.trace.TraceFileObserver\
  \ - Workflow completed -- saving trace file\nFeb-20 07:31:45.076 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Workflow completed -- rendering execution report\nFeb-20 07:31:45.076 [main]\
  \ DEBUG nextflow.trace.ReportObserver - Execution report summary data:\n  []\nFeb-20\
  \ 07:31:45.493 [main] DEBUG nextflow.trace.TimelineObserver - Workflow completed\
  \ -- rendering execution timeline\nFeb-20 07:31:45.696 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 07:31:45.709 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fvwpvbe36%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fvwpvbe36%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.654GiB</td>
						<td>44.3 seconds</td>
						<td>10800.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746724944">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>600 days</td>
						<td>2023-02-19 03:56:06</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: Include statement is not allowed within a workflow definition
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.script.BaseScript.include(BaseScript.groovy:150)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)\n\
  \tat groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:424)\n\
  \tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:350)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat Script_09faa843$_runScript_closure1$_closure3.doCall(Script_09faa843:64)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:205)\n\
  \tat nextflow.script.WorkflowDef.run(WorkflowDef.groovy:189)\n\tat nextflow.script.BindableDef.invoke_a(BindableDef.groovy:52)\n\
  \tat nextflow.script.ChainableDef$invoke_a.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:208)\n\tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\
  \tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2F9tz09hum%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2F9tz09hum%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.467GiB</td>
						<td>21.6 seconds</td>
						<td>12.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738163456">
								<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling">nikostr/dna-seq-deepvariant-glnexus-variant-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nikostr/dna-seq-deepvariant-glnexus-variant-calling/tree/v0.3.1">v0.3.1</a>
						</td>
						<td>snakemake</td>
						<td>599 days</td>
						<td>2023-02-19 03:51:27</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/o6ptp0h33c/singularity-tmp/tmp34ppeosb.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/o6ptp0h33c/code/.snakemake/conda/0298a9492ba5aec693bd08123ae8f65e_.yaml"
  --prefix "/home/azureuser/tmp/o6ptp0h33c/code/.snakemake/conda/0298a9492ba5aec693bd08123ae8f65e_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working...'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nikostr%252Fdna-seq-deepvariant-glnexus-variant-calling%2F8n74kr9a%2Flogs.tar.xz">0.117GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nikostr%252Fdna-seq-deepvariant-glnexus-variant-calling%2F8n74kr9a%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>11.118GiB</td>
						<td>806.2 seconds</td>
						<td>1062.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738319616">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v1.5.0">v1.5.0</a>
						</td>
						<td>snakemake</td>
						<td>597 days</td>
						<td>2023-02-19 06:51:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/4zpujiljp9/code/workflow/rules/../envs/rbt.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/341cadb9-348f-45e5-b193-7a2354a30526\
  \ singularity  exec --home /home/azureuser/tmp/4zpujiljp9/code/.test  /home/azureuser/tmp/4zpujiljp9/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/4zpujiljp9/code/.test/.snakemake/conda/52712530b43ff18127176fd9d8dbce82_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/4zpujiljp9/code/.test/.snakemake/conda/52712530b43ff18127176fd9d8dbce82_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining bcftools=1.12:   0%|        \
  \  | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:00, 54.07it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 107.17it/s]    \nExamining\
  \ @/linux-64::__linux==5.15.0=0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 159.82it/s]\nExamining @/linux-64::__unix==0=0:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 4/6 [00:00<00:00, 211.94it/s]      \nExamining rust-bio-tools=0.21:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 263.64it/s]    \n                                                            \
  \                 \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining conflict for bcftools rust-bio-tools:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining conflict for bcftools rust-bio-tools __glibc:  17%|\u2588\u258B      \
  \  | 1/6 [00:00<00:01,  2.64it/s]\nExamining conflict for bcftools rust-bio-tools\
  \ __glibc:  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00,  5.28it/s]\n \
  \                                                                              \
  \                      \nfailed\nSolving environment: ...working... \nFound conflicts!\
  \ Looking for incompatible packages.\nThis can take several minutes.  Press CTRL-C\
  \ to abort.\n\nBuilding graph of deps:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ bcftools=1.12:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  17%|\u2588\u258B        | 1/6 [00:00<00:00, 35.48it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 70.52it/s]     \nExamining\
  \ @/linux-64::__linux==5.15.0=0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 105.33it/s]\nExamining @/linux-64::__unix==0=0:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 4/6 [00:00<00:00, 139.88it/s]      \nExamining rust-bio-tools=0.21:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 174.23it/s]    \n                                                            \
  \                 \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining conflict for bcftools rust-bio-tools:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining conflict for bcftools rust-bio-tools __glibc:  17%|\u2588\u258B      \
  \  | 1/6 [00:00<00:03,  1.34it/s]\nExamining conflict for bcftools rust-bio-tools\
  \ __glibc:  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:01,  2.67it/s]\n \
  \                                                                              \
  \                      \nfailed\n\nUnsatisfiableError: The following specifications\
  \ were found to be incompatible with each other:\n\nOutput in format: Requested\
  \ package -> Available versions\n\nPackage libblas conflicts for:\nbcftools=1.12\
  \ -> gsl[version='>=2.6,<2.7.0a0'] -> libblas[version='3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0.*|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0.*|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|>=3.8.0,<4.0a0',build='4_openblas|5_blis|6_blis|7_blis|8_blis|8_openblas|9_openblas|9_blis|9_mkl|10_blis|11_blis|11_openblas|11_mkl|12_mkl|13_mkl|13_blis|14_openblas|14_blis|15_blis|16_blis|18_mkl|19_mkl|20_mkl|0_openblas|1_openblas|2_openblas|3_blis|3_openblas|4_blis|4_openblas|4_mkl|5_openblas|6_blis|6_openblas|7_blis|8_blis|8_openblas|9_blis|9_openblas|9_mkl|10_mkl|11_linux64_blis|12_linux64_openblas|12_linux64_blis|13_linux64_openblas|13_linux64_mkl|14_linux64_openblas|15_linux64_blis|15_linux64_openblas|15_linux64_mkl|16_linux64_blis|16_linux64_openblas|16_linux64_mkl|14_linux64_mkl|14_linux64_blis|13_linux64_blis|12_linux64_mkl|11_linux64_mkl|11_linux64_openblas|10_openblas|10_blis|8_mkl|7_mkl|7_openblas|6_mkl|5_mkl|5_blis|2_blis|1_blis|0_blis|21_mkl|17_openblas|16_mkl|16_openblas|15_openblas|15_mkl|14_mkl|13_openblas|12_openblas|12_blis|10_mkl|10_openblas|8_mkl|7_mkl|7_openblas|6_mkl|6_openblas|5_mkl|5_openblas|4_mkl|4_blis']\n\
  rust-bio-tools=0.21 -> gsl[version='>=2.6,<2.7.0a0'] -> libblas[version='3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0.*|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0.*|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|>=3.8.0,<4.0a0',build='4_openblas|5_blis|6_blis|7_blis|8_blis|8_openblas|9_openblas|9_blis|9_mkl|10_blis|11_blis|11_openblas|11_mkl|12_mkl|13_mkl|13_blis|14_openblas|14_blis|15_blis|16_blis|18_mkl|19_mkl|20_mkl|0_openblas|1_openblas|2_openblas|3_blis|3_openblas|4_blis|4_openblas|4_mkl|5_openblas|6_blis|6_openblas|7_blis|8_blis|8_openblas|9_blis|9_openblas|9_mkl|10_mkl|11_linux64_blis|12_linux64_openblas|12_linux64_blis|13_linux64_openblas|13_linux64_mkl|14_linux64_openblas|15_linux64_blis|15_linux64_openblas|15_linux64_mkl|16_linux64_blis|16_linux64_openblas|16_linux64_mkl|14_linux64_mkl|14_linux64_blis|13_linux64_blis|12_linux64_mkl|11_linux64_mkl|11_linux64_openblas|10_openblas|10_blis|8_mkl|7_mkl|7_openblas|6_mkl|5_mkl|5_blis|2_blis|1_blis|0_blis|21_mkl|17_openblas|16_mkl|16_openblas|15_openblas|15_mkl|14_mkl|13_openblas|12_openblas|12_blis|10_mkl|10_openblas|8_mkl|7_mkl|7_openblas|6_mkl|6_openblas|5_mkl|5_openblas|4_mkl|4_blis']\n\
  \nPackage libgcc-ng conflicts for:\nbcftools=1.12 -> libgcc-ng[version='>=7.5.0|>=9.3.0']\n\
  bcftools=1.12 -> gsl[version='>=2.6,<2.7.0a0'] -> libgcc-ng[version='>=10.3.0|>=11.2.0|>=4.9|>=7.3.0|>=9.4.0|>=7.2.0|>=12']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - rust-bio-tools=0.21 -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fm7rpou26%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fm7rpou26%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.239GiB</td>
						<td>252.6 seconds</td>
						<td>307.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738302928">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.0">v2.0.0</a>
						</td>
						<td>snakemake</td>
						<td>589 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/40awvswq0r/singularity-tmp/tmphcol7p1j.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/921a20aa-0a06-4b5a-9507-c1d01a23c520\
  \ singularity  exec --home /home/azureuser/tmp/40awvswq0r/code/.test  /home/azureuser/tmp/40awvswq0r/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/40awvswq0r/code/.test/.snakemake/conda/f18217c71861a2eae1a5da62e55c09db_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/40awvswq0r/code/.test/.snakemake/conda/f18217c71861a2eae1a5da62e55c09db_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining gsl[version='>=2.5,<2.6.0a0']:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining zlib[version='>=1.2.11,<1.3.0a0']:\
  \  11%|\u2588         | 1/9 [00:00<00:00, 16.58it/s]\nExamining perl:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:00<00:00, 10.35it/s]                             \n\
  Examining perl:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 15.52it/s]\n\
  Examining htslib[version='>=1.10,<1.11.0a0']:  33%|\u2588\u2588\u2588\u258E    \
  \  | 3/9 [00:00<00:00, 15.52it/s]\nExamining @/linux-64::__linux==5.15.0=0:  44%|\u2588\
  \u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00, 15.52it/s]     \nExamining @/linux-64::__linux==5.15.0=0:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 11.40it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:00<00:00, 11.40it/s]  \nExamining @/linux-64::__unix==0=0:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 11.40it/s]    \nExamining\
  \ libgcc-ng[version='>=7.3.0']:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [00:00<00:00, 11.40it/s]\nExamining @/linux-64::__archspec==1=x86_64: \
  \ 89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00,\
  \ 11.40it/s]\nExamining @/linux-64::__archspec==1=x86_64: 100%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:00<00:00, 18.43it/s]\n     \
  \                                                                              \
  \      \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for gsl zlib perl htslib __glibc libgcc-ng:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for gsl zlib perl htslib libgcc-ng:  11%|\u2588   \
  \      | 1/9 [00:00<00:02,  3.83it/s]\nExamining conflict for gsl zlib perl htslib\
  \ libgcc-ng:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,  7.66it/s]\nExamining\
  \ conflict for htslib zlib:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,  7.66it/s]\
  \                   \nExamining conflict for htslib zlib:  33%|\u2588\u2588\u2588\
  \u258E      | 3/9 [00:00<00:01,  5.75it/s]\n                                   \
  \                                              \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining gsl[version='>=2.5,<2.6.0a0']:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining zlib[version='>=1.2.11,<1.3.0a0']:  11%|\u2588         | 1/9\
  \ [00:00<00:00, 29.91it/s]\nExamining perl:  22%|\u2588\u2588\u258F       | 2/9\
  \ [00:00<00:00,  9.42it/s]                             \nExamining perl:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:00<00:00, 14.13it/s]\nExamining htslib[version='>=1.10,<1.11.0a0']:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 14.13it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00, 14.13it/s]     \n\
  Examining @/linux-64::__linux==5.15.0=0:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:00<00:00, 12.48it/s]\nExamining @/linux-64::__glibc==2.31=0:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 12.48it/s]  \nExamining @/linux-64::__unix==0=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 12.48it/s]\
  \    \nExamining libgcc-ng[version='>=7.3.0']:  78%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258A  | 7/9 [00:00<00:00, 12.48it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00,\
  \ 12.48it/s]\n                                                                 \
  \                        \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for gsl zlib perl htslib __glibc libgcc-ng:   0%| \
  \         | 0/9 [00:00<?, ?it/s]\nExamining conflict for gsl zlib perl htslib libgcc-ng:\
  \  11%|\u2588         | 1/9 [00:00<00:01,  6.42it/s]\nExamining conflict for gsl\
  \ zlib perl htslib libgcc-ng:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \ 12.83it/s]\nExamining conflict for htslib zlib:  22%|\u2588\u2588\u258F      \
  \ | 2/9 [00:00<00:00, 12.83it/s]                   \n                          \
  \                                                       \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with each other:\n\n\
  Output in format: Requested package -> Available versions\n\nPackage libgcc-ng conflicts\
  \ for:\nhtslib[version='>=1.10,<1.11.0a0'] -> bzip2[version='>=1.0.8,<2.0a0'] ->\
  \ libgcc-ng[version='>=10.3.0|>=12|>=9.3.0|>=9.4.0|>=11.2.0|>=4.9|>=7.2.0']\nlibgcc-ng[version='>=7.3.0']\n\
  zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=7.5.0|>=7.3.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  gsl[version='>=2.5,<2.6.0a0'] -> libgcc-ng[version='>=7.3.0']\nperl -> libgcc-ng[version='>=11.2.0|>=4.9|>=7.3.0|>=7.5.0|>=9.3.0|>=9.4.0|>=7.2.0']\n\
  gsl[version='>=2.5,<2.6.0a0'] -> libblas[version='>=3.8.0,<4.0a0'] -> libgcc-ng[version='>=7.5.0|>=9.3.0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> libgcc-ng[version='>=7.3.0|>=7.5.0']\n\nPackage\
  \ zlib conflicts for:\nzlib[version='>=1.2.11,<1.3.0a0']\nhtslib[version='>=1.10,<1.11.0a0']\
  \ -> libcurl[version='>=7.71.1,<8.0a0'] -> zlib[version='>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0']The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - gsl[version='>=2.5,<2.6.0a0'] -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - htslib[version='>=1.10,<1.11.0a0'] -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \  - libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n  - perl -> libgcc-ng[version='>=9.4.0']\
  \ -> __glibc[version='>=2.17']\n  - zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=10.3.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Ff7cunwcg%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Ff7cunwcg%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.263GiB</td>
						<td>806.2 seconds</td>
						<td>1019.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738302688">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.1">v2.0.1</a>
						</td>
						<td>snakemake</td>
						<td>585 days</td>
						<td>2023-02-19 06:40:06</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/w4yqvled48/singularity-tmp/tmpbfs7dlcq.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/c2986714-d6bd-4fd4-ae44-7d9670956197\
  \ singularity  exec --home /home/azureuser/tmp/w4yqvled48/code/.test  /home/azureuser/tmp/w4yqvled48/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/w4yqvled48/code/.test/.snakemake/conda/ee2051a8a316d00b92064e456a2cb804_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/w4yqvled48/code/.test/.snakemake/conda/ee2051a8a316d00b92064e456a2cb804_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining perl:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]        \nExamining @/linux-64::__glibc==2.31=0:  11%|\u2588         | 1/9\
  \ [00:00<00:00, 10.21it/s]\nExamining zlib[version='>=1.2.11,<1.3.0a0']:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:00<00:00, 20.38it/s]\nExamining libgcc-ng[version='>=7.3.0']:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 14.65it/s]     \nExamining\
  \ libgcc-ng[version='>=7.3.0']:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,\
  \ 19.51it/s]\nExamining @/linux-64::__archspec==1=x86_64:  44%|\u2588\u2588\u2588\
  \u2588\u258D     | 4/9 [00:00<00:00, 19.51it/s]\nExamining gsl[version='>=2.5,<2.6.0a0']:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 19.51it/s]  \
  \ \nExamining @/linux-64::__linux==5.15.0=0:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:00<00:00, 19.51it/s]\nExamining htslib[version='>=1.10,<1.11.0a0']:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:00<00:00, 19.51it/s]\n\
  Examining @/linux-64::__unix==0=0:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2589 | 8/9 [00:00<00:00, 19.51it/s]           \nExamining @/linux-64::__unix==0=0:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:00<00:00,\
  \ 27.35it/s]\n                                                                 \
  \               \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\n\
  Examining conflict for perl zlib libgcc-ng gsl htslib:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for perl __glibc zlib libgcc-ng gsl htslib:  11%|\u2588\
  \         | 1/9 [00:00<00:01,  7.83it/s]\nExamining conflict for perl __glibc zlib\
  \ libgcc-ng gsl htslib:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 15.64it/s]\n\
  Examining conflict for zlib htslib:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \ 15.64it/s]                           \n                                      \
  \                                           \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining perl:   0%|          | 0/9 [00:00<?, ?it/s]        \nExamining\
  \ @/linux-64::__glibc==2.31=0:  11%|\u2588         | 1/9 [00:00<00:00, 15.16it/s]\n\
  Examining zlib[version='>=1.2.11,<1.3.0a0']:  22%|\u2588\u2588\u258F       | 2/9\
  \ [00:00<00:00, 30.22it/s]\nExamining libgcc-ng[version='>=7.3.0']:  33%|\u2588\u2588\
  \u2588\u258E      | 3/9 [00:00<00:00, 17.89it/s]     \nExamining libgcc-ng[version='>=7.3.0']:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00, 23.84it/s]\nExamining\
  \ @/linux-64::__archspec==1=x86_64:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9\
  \ [00:00<00:00, 23.84it/s]\nExamining gsl[version='>=2.5,<2.6.0a0']:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 23.84it/s]   \nExamining @/linux-64::__linux==5.15.0=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 23.84it/s]\n\
  Examining htslib[version='>=1.10,<1.11.0a0']:  78%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258A  | 7/9 [00:00<00:00, 23.84it/s]\nExamining @/linux-64::__unix==0=0:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00,\
  \ 23.84it/s]           \nExamining @/linux-64::__unix==0=0: 100%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:00<00:00, 35.79it/s]\n     \
  \                                                                           \n\n\
  Determining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for perl zlib libgcc-ng gsl htslib:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for perl __glibc zlib libgcc-ng gsl htslib:  11%|\u2588         | 1/9\
  \ [00:00<00:00, 11.62it/s]\nExamining conflict for zlib htslib:  22%|\u2588\u2588\
  \u258F       | 2/9 [00:00<00:00,  9.74it/s]                           \nExamining\
  \ conflict for zlib htslib:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00,\
  \ 14.60it/s]\n                                                                 \
  \                \nfailed\n\nUnsatisfiableError: The following specifications were\
  \ found to be incompatible with each other:\n\nOutput in format: Requested package\
  \ -> Available versions\n\nPackage libgcc-ng conflicts for:\nlibgcc-ng[version='>=7.3.0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> bzip2[version='>=1.0.8,<2.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=9.3.0|>=9.4.0|>=11.2.0|>=4.9|>=7.2.0']\n\
  perl -> libgcc-ng[version='>=11.2.0|>=4.9|>=7.3.0|>=7.5.0|>=9.3.0|>=9.4.0|>=7.2.0']\n\
  gsl[version='>=2.5,<2.6.0a0'] -> libblas[version='>=3.8.0,<4.0a0'] -> libgcc-ng[version='>=7.5.0|>=9.3.0']\n\
  zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=7.5.0|>=7.3.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> libgcc-ng[version='>=7.3.0|>=7.5.0']\ngsl[version='>=2.5,<2.6.0a0']\
  \ -> libgcc-ng[version='>=7.3.0']\n\nPackage zlib conflicts for:\nzlib[version='>=1.2.11,<1.3.0a0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> libcurl[version='>=7.71.1,<8.0a0'] -> zlib[version='>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0']The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - gsl[version='>=2.5,<2.6.0a0'] ->\
  \ libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n  - htslib[version='>=1.10,<1.11.0a0']\
  \ -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n  - perl -> libgcc-ng[version='>=9.4.0']\
  \ -> __glibc[version='>=2.17']\n  - zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=10.3.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Ffmxvdge2%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Ffmxvdge2%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.250GiB</td>
						<td>608.4 seconds</td>
						<td>724.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738302208">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.3">v2.0.3</a>
						</td>
						<td>snakemake</td>
						<td>585 days</td>
						<td>2023-02-19 06:00:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/5yu44nwb9u/code/workflow/rules/../envs/rbt.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/1e7c813a-c3ab-486b-9377-aa39b9329f69\
  \ singularity  exec --home /home/azureuser/tmp/5yu44nwb9u/code/.test  /home/azureuser/tmp/5yu44nwb9u/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/5yu44nwb9u/code/.test/.snakemake/conda/3ddb80ea2a1c971d4bdce55b96cd68ca_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/5yu44nwb9u/code/.test/.snakemake/conda/3ddb80ea2a1c971d4bdce55b96cd68ca_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining bcftools=1.12:   0%|        \
  \  | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:00, 44.72it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 88.68it/s]     \nExamining\
  \ @/linux-64::__unix==0=0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 132.22it/s]   \nExamining rust-bio-tools=0.21:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 4/6 [00:00<00:00, 175.38it/s]    \nExamining @/linux-64::__linux==5.15.0=0:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 120.72it/s]\n                                                                \
  \                       \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for bcftools rust-bio-tools:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for __glibc bcftools rust-bio-tools:  17%|\u2588\u258B\
  \        | 1/6 [00:01<00:07,  1.47s/it]\nExamining conflict for __glibc bcftools\
  \ rust-bio-tools:  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:01<00:02,  1.36it/s]\n\
  \                                                                              \
  \                       \nfailed\nSolving environment: ...working... \nFound conflicts!\
  \ Looking for incompatible packages.\nThis can take several minutes.  Press CTRL-C\
  \ to abort.\n\nBuilding graph of deps:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ bcftools=1.12:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  17%|\u2588\u258B        | 1/6 [00:00<00:00, 26.63it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 53.01it/s]     \nExamining\
  \ @/linux-64::__unix==0=0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 79.21it/s]    \nExamining rust-bio-tools=0.21:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 4/6 [00:00<00:00, 105.22it/s]   \nExamining @/linux-64::__linux==5.15.0=0:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 60.72it/s]\n                                                                 \
  \                     \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for bcftools rust-bio-tools:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for __glibc bcftools rust-bio-tools:  17%|\u2588\u258B\
  \        | 1/6 [00:00<00:03,  1.38it/s]\nExamining conflict for __glibc bcftools\
  \ rust-bio-tools:  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:01,  2.76it/s]\n\
  \                                                                              \
  \                       \nfailed\n\nUnsatisfiableError: The following specifications\
  \ were found to be incompatible with each other:\n\nOutput in format: Requested\
  \ package -> Available versions\n\nPackage libgcc-ng conflicts for:\nbcftools=1.12\
  \ -> libgcc-ng[version='>=7.5.0|>=9.3.0']\nbcftools=1.12 -> gsl[version='>=2.6,<2.7.0a0']\
  \ -> libgcc-ng[version='>=10.3.0|>=11.2.0|>=4.9|>=7.3.0|>=9.4.0|>=7.2.0|>=12']\n\
  \nPackage libblas conflicts for:\nbcftools=1.12 -> gsl[version='>=2.6,<2.7.0a0']\
  \ -> libblas[version='3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0.*|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0.*|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|>=3.8.0,<4.0a0',build='5_blis|5_mkl|6_mkl|7_openblas|7_mkl|8_blis|8_mkl|9_openblas|9_blis|11_blis|11_openblas|12_blis|12_openblas|13_blis|14_blis|14_mkl|15_blis|15_openblas|16_openblas|16_blis|21_mkl|0_openblas|1_blis|2_openblas|2_blis|4_mkl|5_blis|5_mkl|6_blis|7_blis|7_openblas|8_mkl|9_blis|9_openblas|11_linux64_openblas|11_linux64_mkl|16_linux64_blis|16_linux64_mkl|16_linux64_openblas|15_linux64_mkl|15_linux64_openblas|15_linux64_blis|14_linux64_mkl|14_linux64_openblas|14_linux64_blis|13_linux64_mkl|13_linux64_openblas|13_linux64_blis|12_linux64_mkl|12_linux64_blis|12_linux64_openblas|11_linux64_blis|10_mkl|10_openblas|10_blis|9_mkl|8_openblas|8_blis|7_mkl|6_mkl|6_openblas|5_openblas|4_openblas|4_blis|3_openblas|3_blis|1_openblas|0_blis|20_mkl|19_mkl|18_mkl|17_openblas|16_mkl|15_mkl|14_openblas|13_openblas|13_mkl|12_mkl|11_mkl|10_mkl|10_openblas|10_blis|9_mkl|8_openblas|7_blis|6_openblas|6_blis|5_openblas|4_mkl|4_openblas|4_blis']\n\
  rust-bio-tools=0.21 -> gsl[version='>=2.6,<2.7.0a0'] -> libblas[version='3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0.*|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0.*|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|>=3.8.0,<4.0a0',build='5_blis|5_mkl|6_mkl|7_openblas|7_mkl|8_blis|8_mkl|9_openblas|9_blis|11_blis|11_openblas|12_blis|12_openblas|13_blis|14_blis|14_mkl|15_blis|15_openblas|16_openblas|16_blis|21_mkl|0_openblas|1_blis|2_openblas|2_blis|4_mkl|5_blis|5_mkl|6_blis|7_blis|7_openblas|8_mkl|9_blis|9_openblas|11_linux64_openblas|11_linux64_mkl|16_linux64_blis|16_linux64_mkl|16_linux64_openblas|15_linux64_mkl|15_linux64_openblas|15_linux64_blis|14_linux64_mkl|14_linux64_openblas|14_linux64_blis|13_linux64_mkl|13_linux64_openblas|13_linux64_blis|12_linux64_mkl|12_linux64_blis|12_linux64_openblas|11_linux64_blis|10_mkl|10_openblas|10_blis|9_mkl|8_openblas|8_blis|7_mkl|6_mkl|6_openblas|5_openblas|4_openblas|4_blis|3_openblas|3_blis|1_openblas|0_blis|20_mkl|19_mkl|18_mkl|17_openblas|16_mkl|15_mkl|14_openblas|13_openblas|13_mkl|12_mkl|11_mkl|10_mkl|10_openblas|10_blis|9_mkl|8_openblas|7_blis|6_openblas|6_blis|5_openblas|4_mkl|4_openblas|4_blis']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - rust-bio-tools=0.21 -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Feaxozlm9%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Feaxozlm9%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.250GiB</td>
						<td>693.3 seconds</td>
						<td>908.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738302448">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.2">v2.0.2</a>
						</td>
						<td>snakemake</td>
						<td>585 days</td>
						<td>2023-02-19 03:33:47</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/cvp4cryeuu/singularity-tmp/tmpon7znrkw.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/d3fba509-a317-47fe-9c24-95b02254a7c7\
  \ singularity  exec --home /home/azureuser/tmp/cvp4cryeuu/code/.test  /home/azureuser/tmp/cvp4cryeuu/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/cvp4cryeuu/code/.test/.snakemake/conda/eabbbee84c4e004a9eb49db777d2d355_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/cvp4cryeuu/code/.test/.snakemake/conda/eabbbee84c4e004a9eb49db777d2d355_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:  20%|\u2588\
  \u2588        | 1/5 [00:00<00:00, 3830.41it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 5292.50it/s]\nExamining\
  \ @/linux-64::__archspec==1=x86_64:  60%|\u2588\u2588\u2588\u2588\u2588\u2588  \
  \  | 3/5 [00:00<00:00, 6323.07it/s]\nExamining htslib==1.10:  80%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 7105.98it/s]               \
  \     \n                                                                       \n\
  \nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict\
  \ for htslib __glibc:   0%|          | 0/5 [00:00<?, ?it/s]\n                  \
  \                                                          \nfailed\nSolving environment:\
  \ ...working... \nFound conflicts! Looking for incompatible packages.\nThis can\
  \ take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|\
  \          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:   0%|\
  \          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:  20%|\u2588\
  \u2588        | 1/5 [00:00<00:00, 6754.11it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 6684.15it/s]\nExamining\
  \ @/linux-64::__archspec==1=x86_64:  60%|\u2588\u2588\u2588\u2588\u2588\u2588  \
  \  | 3/5 [00:00<00:00, 6689.48it/s]\nExamining htslib==1.10:  80%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 7231.56it/s]               \
  \     \n                                                                       \n\
  \nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict\
  \ for htslib __glibc:   0%|          | 0/5 [00:00<?, ?it/s]\n                  \
  \                                                          \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with your system:\n\
  \n  - feature:/linux-64::__glibc==2.31=0\n  - htslib==1.10 -> libgcc-ng[version='>=7.3.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fcxdux0ng%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fcxdux0ng%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.209GiB</td>
						<td>393.8 seconds</td>
						<td>482.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738301968">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.4">v2.0.4</a>
						</td>
						<td>snakemake</td>
						<td>584 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/4itjkbx2za/singularity-tmp/tmpzqomwdht.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/98673ac1-6d91-4959-b569-997ad77869a3\
  \ singularity  exec --home /home/azureuser/tmp/4itjkbx2za/code/.test  /home/azureuser/tmp/4itjkbx2za/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/4itjkbx2za/code/.test/.snakemake/conda/d32524df403f84ccbf906113853d4bc6_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/4itjkbx2za/code/.test/.snakemake/conda/d32524df403f84ccbf906113853d4bc6_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining libgcc-ng[version='>=7.3.0']:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  11%|\u2588         | 1/9 [00:00<00:00, 17.10it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 34.10it/s]   \nExamining @/linux-64::__unix==0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 51.05it/s]      \nExamining\
  \ htslib[version='>=1.10,<1.11.0a0']:  44%|\u2588\u2588\u2588\u2588\u258D     |\
  \ 4/9 [00:00<00:00, 67.95it/s]\nExamining @/linux-64::__glibc==2.31=0:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 34.12it/s]       \nExamining\
  \ @/linux-64::__glibc==2.31=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B \
  \  | 6/9 [00:00<00:00, 40.91it/s]\nExamining perl:  67%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258B   | 6/9 [00:00<00:00, 40.91it/s]                       \nExamining\
  \ zlib[version='>=1.2.11,<1.3.0a0']:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:00<00:00, 40.91it/s]\nExamining gsl[version='>=2.5,<2.6.0a0']:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00,\
  \ 40.91it/s]    \n                                                             \
  \                         \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for libgcc-ng htslib perl zlib gsl:   0%|         \
  \ | 0/9 [00:00<?, ?it/s]\nExamining conflict for libgcc-ng htslib __glibc perl zlib\
  \ gsl:  11%|\u2588         | 1/9 [00:00<00:01,  7.35it/s]\nExamining conflict for\
  \ libgcc-ng htslib __glibc perl zlib gsl:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \ 14.69it/s]\nExamining conflict for htslib zlib:  22%|\u2588\u2588\u258F      \
  \ | 2/9 [00:00<00:00, 14.69it/s]                           \n                  \
  \                                                               \nfailed\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining libgcc-ng[version='>=7.3.0']:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  11%|\u2588         | 1/9 [00:00<00:00,  9.94it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 19.86it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 19.86it/s]   \nExamining @/linux-64::__unix==0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 19.86it/s]      \nExamining\
  \ htslib[version='>=1.10,<1.11.0a0']:  44%|\u2588\u2588\u2588\u2588\u258D     |\
  \ 4/9 [00:00<00:00, 19.86it/s]\nExamining @/linux-64::__glibc==2.31=0:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 19.86it/s]       \nExamining\
  \ @/linux-64::__glibc==2.31=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B \
  \  | 6/9 [00:00<00:00, 24.76it/s]\nExamining perl:  67%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258B   | 6/9 [00:00<00:00, 24.76it/s]                       \nExamining\
  \ zlib[version='>=1.2.11,<1.3.0a0']:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:00<00:00, 24.76it/s]\nExamining gsl[version='>=2.5,<2.6.0a0']:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00,\
  \ 24.76it/s]    \nExamining gsl[version='>=2.5,<2.6.0a0']: 100%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:00<00:00, 12.87it/s]\n     \
  \                                                                              \
  \   \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for libgcc-ng htslib perl zlib gsl:   0%|          | 0/9 [00:00<?, ?it/s]\n\
  Examining conflict for libgcc-ng htslib __glibc perl zlib gsl:  11%|\u2588     \
  \    | 1/9 [00:00<00:02,  3.84it/s]\nExamining conflict for libgcc-ng htslib __glibc\
  \ perl zlib gsl:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,  7.68it/s]\n\
  Examining conflict for htslib zlib:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \  7.68it/s]                           \nExamining conflict for htslib zlib:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:00<00:01,  5.45it/s]\n                       \
  \                                                          \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with each other:\n\n\
  Output in format: Requested package -> Available versions\n\nPackage zlib conflicts\
  \ for:\nhtslib[version='>=1.10,<1.11.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0']\n\
  zlib[version='>=1.2.11,<1.3.0a0']\nhtslib[version='>=1.10,<1.11.0a0'] -> libcurl[version='>=7.71.1,<8.0a0']\
  \ -> zlib[version='>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n\nPackage libgcc-ng conflicts\
  \ for:\nzlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=7.5.0|>=7.3.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  perl -> libgcc-ng[version='>=11.2.0|>=4.9|>=7.3.0|>=7.5.0|>=9.3.0|>=9.4.0|>=7.2.0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> libgcc-ng[version='>=7.3.0|>=7.5.0']\nhtslib[version='>=1.10,<1.11.0a0']\
  \ -> bzip2[version='>=1.0.8,<2.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=9.3.0|>=9.4.0|>=11.2.0|>=4.9|>=7.2.0']\n\
  gsl[version='>=2.5,<2.6.0a0'] -> libgcc-ng[version='>=7.3.0']\nlibgcc-ng[version='>=7.3.0']\n\
  gsl[version='>=2.5,<2.6.0a0'] -> libblas[version='>=3.8.0,<4.0a0'] -> libgcc-ng[version='>=7.5.0|>=9.3.0']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - gsl[version='>=2.5,<2.6.0a0'] -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - htslib[version='>=1.10,<1.11.0a0'] -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \  - perl -> libgcc-ng[version='>=9.4.0'] -> __glibc[version='>=2.17']\n  - zlib[version='>=1.2.11,<1.3.0a0']\
  \ -> libgcc-ng[version='>=10.3.0'] -> __glibc[version='>=2.17']\n\nYour installed\
  \ version is: 2.31\n\nNote that strict channel priority may have removed packages\
  \ required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Ffe9t5ckh%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Ffe9t5ckh%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.356GiB</td>
						<td>1330.6 seconds</td>
						<td>1874.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010737817712">
								<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake">IMS-Bio2Core-Facility/GTExSnake</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>584 days</td>
						<td>2023-02-19 03:49:46</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/0r0vhfsxeq/code/workflow/envs/biomart.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/9302794a-5162-46ce-9b92-98d416638da3\
  \ singularity  exec --home /home/azureuser/tmp/0r0vhfsxeq/code  /home/azureuser/tmp/0r0vhfsxeq/code/.snakemake/singularity/9f56b38ad905baa9ffe4c187a4f76e32.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/0r0vhfsxeq/code/.snakemake/conda/f0d03f553d349981c22b12a9309e15bd_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/0r0vhfsxeq/code/.snakemake/conda/f0d03f553d349981c22b12a9309e15bd_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nBuilding graph of deps:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining python==3.9.5:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ @/linux-64::__linux==5.15.0=0:  11%|\u2588         | 1/9 [00:00<00:01,  5.86it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \ 11.71it/s]\nExamining bioservices==1.7.11:  22%|\u2588\u2588\u258F       | 2/9\
  \ [00:00<00:00, 11.71it/s]          \nExamining @/linux-64::__archspec==1=x86_64:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 11.71it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,  6.66it/s]\nExamining\
  \ @/linux-64::__unix==0=0:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,\
  \  6.66it/s]         \nExamining pip==21.1.3:  56%|\u2588\u2588\u2588\u2588\u2588\
  \u258C    | 5/9 [00:00<00:00,  6.66it/s]            \nExamining @/linux-64::__glibc==2.31=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:01<00:00,  6.66it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:01<00:00,  5.37it/s]\nExamining pandas==1.2.4:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:01<00:00,  5.37it/s]     \
  \         \nExamining lxml==4.6.3:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2589 | 8/9 [00:03<00:00,  5.37it/s]  \nExamining lxml==4.6.3: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:03<00:00,  2.75it/s]\n\
  \                                                                    \n\nDetermining\
  \ conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for python\
  \ bioservices pip pandas lxml:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for python lxml pandas:  11%|\u2588         | 1/9 [00:02<00:18,  2.27s/it]\
  \        \nExamining conflict for python lxml pandas:  22%|\u2588\u2588\u258F  \
  \     | 2/9 [00:02<00:07,  1.14s/it]\nExamining conflict for python pip bioservices\
  \ pandas:  22%|\u2588\u2588\u258F       | 2/9 [00:03<00:07,  1.14s/it]\nExamining\
  \ conflict for python pip bioservices pandas:  33%|\u2588\u2588\u2588\u258E    \
  \  | 3/9 [00:03<00:07,  1.24s/it]\nExamining conflict for python bioservices:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:04<00:07,  1.24s/it]           \nExamining conflict\
  \ for python bioservices:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:04<00:05,\
  \  1.05s/it]\nExamining conflict for python pip:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [00:04<00:05,  1.05s/it]        \nExamining conflict for python pip:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:04<00:03,  1.12it/s]\n\
  Examining conflict for python __glibc lxml pandas:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:04<00:03,  1.12it/s]\nExamining conflict for bioservices\
  \ pandas:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:05<00:02,\
  \  1.12it/s]        \nExamining conflict for bioservices pandas:  78%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:05<00:01,  1.33it/s]\nExamining\
  \ conflict for bioservices lxml:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:05<00:01,  1.33it/s]  \nExamining conflict for pip bioservices\
  \ lxml pandas:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9\
  \ [00:05<00:00,  1.33it/s]\nExamining conflict for pip bioservices pandas: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:09<00:00,  1.33it/s]\
  \     \nExamining conflict for pip bioservices pandas: : 10it [00:09,  1.10it/s]\
  \                    \nExamining conflict for lxml pandas: : 10it [00:11,  1.10it/s]\
  \           \nExamining conflict for lxml pandas: : 11it [00:11,  1.29s/it]\n  \
  \                                                           \nFound conflicts! Looking\
  \ for incompatible packages.\nThis can take several minutes.  Press CTRL-C to abort.\n\
  failed\nSolving environment: ...working... \nBuilding graph of deps:   0%|     \
  \     | 0/9 [00:00<?, ?it/s]\nExamining python==3.9.5:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:  11%|\u2588         | 1/9 [00:00<00:00,\
  \ 11.51it/s]\nExamining bioservices==1.7.11:  22%|\u2588\u2588\u258F       | 2/9\
  \ [00:00<00:00, 22.96it/s]          \nExamining @/linux-64::__archspec==1=x86_64:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:01,  5.20it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,  6.93it/s]\nExamining\
  \ @/linux-64::__unix==0=0:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,\
  \  6.93it/s]         \nExamining pip==21.1.3:  56%|\u2588\u2588\u2588\u2588\u2588\
  \u258C    | 5/9 [00:00<00:00,  6.93it/s]            \nExamining @/linux-64::__glibc==2.31=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:01<00:00,  6.93it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:01<00:00,  6.80it/s]\nExamining pandas==1.2.4:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:01<00:00,  6.80it/s]     \
  \         \nExamining lxml==4.6.3:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2589 | 8/9 [00:01<00:00,  6.80it/s]  \nExamining lxml==4.6.3: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:01<00:00,  4.09it/s]\n\
  \                                                                    \n\nDetermining\
  \ conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for python\
  \ bioservices pip pandas lxml:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for python lxml pandas:  11%|\u2588         | 1/9 [00:01<00:11,  1.43s/it]\
  \        \nExamining conflict for python lxml pandas:  22%|\u2588\u2588\u258F  \
  \     | 2/9 [00:01<00:05,  1.40it/s]\nExamining conflict for python pip bioservices\
  \ pandas:  22%|\u2588\u2588\u258F       | 2/9 [00:02<00:05,  1.40it/s]\nExamining\
  \ conflict for python pip bioservices pandas:  33%|\u2588\u2588\u2588\u258E    \
  \  | 3/9 [00:02<00:04,  1.34it/s]\nExamining conflict for python bioservices:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:02<00:04,  1.34it/s]           \nExamining conflict\
  \ for python bioservices:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:02<00:03,\
  \  1.53it/s]\nExamining conflict for python pip:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [00:03<00:03,  1.53it/s]        \nExamining conflict for python pip:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:02,  1.70it/s]\n\
  Examining conflict for python __glibc lxml pandas:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:03<00:02,  1.70it/s]\nExamining conflict for bioservices\
  \ pandas:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:03<00:01,\
  \  1.70it/s]        \nExamining conflict for bioservices pandas:  78%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:03<00:01,  1.96it/s]\nExamining\
  \ conflict for bioservices lxml:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:03<00:01,  1.96it/s]  \nExamining conflict for pip bioservices\
  \ lxml pandas:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9\
  \ [00:03<00:00,  1.96it/s]\nExamining conflict for pip bioservices pandas: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:06<00:00,  1.96it/s]\
  \     \nExamining conflict for pip bioservices pandas: : 10it [00:06,  1.53it/s]\
  \                    \nExamining conflict for lxml pandas: : 10it [00:07,  1.53it/s]\
  \           \nExamining conflict for lxml pandas: : 11it [00:07,  1.24it/s]\n  \
  \                                                           \nFound conflicts! Looking\
  \ for incompatible packages.\nThis can take several minutes.  Press CTRL-C to abort.\n\
  failed\n\nUnsatisfiableError: The following specifications were found to be incompatible\
  \ with each other:\n\nOutput in format: Requested package -> Available versions\n\
  \nPackage setuptools conflicts for:\npandas==1.2.4 -> setuptools[version='<60.0.0']\n\
  pip==21.1.3 -> setuptools\nbioservices==1.7.11 -> easydev[version='>=0.9.36'] ->\
  \ setuptools[version='<60.0.0']\npython==3.9.5 -> pip -> setuptools\n\nPackage pypy3.6\
  \ conflicts for:\nlxml==4.6.3 -> pypy3.6[version='>=7.3.3']\npip==21.1.3 -> python[version='>=3.6']\
  \ -> pypy3.6[version='7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*|>=7.3.3|>=7.3.2|>=7.3.1']\n\
  lxml==4.6.3 -> python[version='>=3.6,<3.7.0a0'] -> pypy3.6[version='7.3.*|7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*']\n\
  pandas==1.2.4 -> numpy[version='>=1.19.5,<2.0a0'] -> pypy3.6[version='>=7.3.1|>=7.3.2|>=7.3.3']\n\
  bioservices==1.7.11 -> beautifulsoup4 -> pypy3.6[version='7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*|>=7.3.1|>=7.3.2|>=7.3.3']\n\
  \nPackage ca-certificates conflicts for:\nbioservices==1.7.11 -> python -> ca-certificates\n\
  python==3.9.5 -> openssl[version='>=1.1.1k,<1.1.2a'] -> ca-certificates\n\nPackage\
  \ pandas conflicts for:\nbioservices==1.7.11 -> pandas\npandas==1.2.4\n\nPackage\
  \ python conflicts for:\nbioservices==1.7.11 -> appdirs -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.10,<3.11.0a0|>=3.9,<3.10.0a0|>=3.7|>=3.5|>=3.11,<3.12.0a0|>=3.7.1,<3.8.0a0|>=3.7,<4.0|>=3.6,<4.0']\n\
  pip==21.1.3 -> python[version='>=3.6|>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0']\n\
  lxml==4.6.3 -> python_abi=3.6[build=*_cp36m] -> python[version='3.6.*|3.9.*|3.7.*|3.8.*']\n\
  lxml==4.6.3 -> python[version='>=3.10,<3.11.0a0|>=3.6,<3.7.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  pandas==1.2.4 -> python[version='>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0']\n\
  pip==21.1.3 -> setuptools -> python[version='!=3.0,!=3.1,!=3.2,!=3.3,!=3.4|2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.7|>=3.8|>=3.11,<3.12.0a0|>=3.5,<3.6.0a0|3.4.*|2.7.*|>=3.6']\n\
  bioservices==1.7.11 -> python[version='>=3.6']\npandas==1.2.4 -> numpy[version='>=1.19.5,<2.0a0']\
  \ -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.6,<3.7.0a0|>=3.6|>=3.5,<3.6.0a0|>=3|3.4.*|3.7.*|3.8.*|3.9.*']\n\
  \nPackage certifi conflicts for:\npandas==1.2.4 -> setuptools[version='<60.0.0']\
  \ -> certifi[version='>=2016.09|>=2016.9.26']\npip==21.1.3 -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']\n\
  bioservices==1.7.11 -> requests -> certifi[version='>=2017.4.17']\n\nPackage _libgcc_mutex\
  \ conflicts for:\npandas==1.2.4 -> libgcc-ng[version='>=9.3.0'] -> _libgcc_mutex[version='*|0.1',build='main|conda_forge|main']\n\
  lxml==4.6.3 -> libgcc-ng[version='>=9.3.0'] -> _libgcc_mutex[version='*|0.1',build='main|conda_forge|main']\n\
  python==3.9.5 -> libgcc-ng[version='>=9.3.0'] -> _libgcc_mutex[version='*|0.1',build='main|conda_forge|main']\n\
  \nPackage gdbm conflicts for:\nlxml==4.6.3 -> pypy3.6[version='>=7.3.3'] -> gdbm[version='>=1.18,<1.19.0a0']\n\
  pandas==1.2.4 -> pypy3.7[version='>=7.3.3'] -> gdbm[version='>=1.18,<1.19.0a0']\n\
  \nPackage lxml conflicts for:\nlxml==4.6.3\nbioservices==1.7.11 -> lxml\n\nPackage\
  \ wheel conflicts for:\npip==21.1.3 -> wheel\npython==3.9.5 -> pip -> wheelThe following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - lxml==4.6.3 -> libgcc-ng[version='>=9.3.0']\
  \ -> __glibc[version='>=2.17']\n  - pandas==1.2.4 -> libgcc-ng[version='>=9.3.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252FGTExSnake%2Fbsmqn7mh%2Flogs.tar.xz">0.127GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252FGTExSnake%2Fbsmqn7mh%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>2.608GiB</td>
						<td>685.7 seconds</td>
						<td>835.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010737749536">
								<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake">IMS-Bio2Core-Facility/GTExSnake</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>584 days</td>
						<td>2023-02-19 04:00:21</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/8hnweg66ve/code/workflow/rules/../envs/request.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/79107f3b-244e-49e6-a97e-df67ae6e6a51\
  \ singularity  exec --home /home/azureuser/tmp/8hnweg66ve/code  /home/azureuser/tmp/8hnweg66ve/code/.snakemake/singularity/9f56b38ad905baa9ffe4c187a4f76e32.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/8hnweg66ve/code/.snakemake/conda/88a16d7b8f540b2b7d538c9552da5ec0_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/8hnweg66ve/code/.snakemake/conda/88a16d7b8f540b2b7d538c9552da5ec0_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nBuilding graph of deps:   0%|          | 0/8 [00:00<?,\
  \ ?it/s]\nExamining pip==21.1.3:   0%|          | 0/8 [00:00<?, ?it/s] \nExamining\
  \ python==3.9.5:  12%|\u2588\u258E        | 1/8 [00:00<00:01,  3.74it/s]\nExamining\
  \ python==3.9.5:  25%|\u2588\u2588\u258C       | 2/8 [00:00<00:00,  7.47it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  25%|\u2588\u2588\u258C       | 2/8 [00:00<00:00,\
  \  7.47it/s]\nExamining @/linux-64::__archspec==1=x86_64:  38%|\u2588\u2588\u2588\
  \u258A      | 3/8 [00:00<00:00,  7.47it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  50%|\u2588\u2588\u2588\u2588\u2588     | 4/8 [00:00<00:00,  7.47it/s]   \nExamining\
  \ pandas==1.2.4:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\u258E   | 5/8 [00:00<00:00,\
  \  7.47it/s]                \nExamining requests==2.25.1:  75%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258C  | 6/8 [00:00<00:00,  7.47it/s]\nExamining requests==2.25.1:\
  \  88%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [00:00<00:00,\
  \  7.67it/s]\nExamining @/linux-64::__unix==0=0:  88%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258A | 7/8 [00:01<00:00,  7.67it/s]\nExamining @/linux-64::__unix==0=0:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 8/8 [00:01<00:00,\
  \  6.99it/s]\n                                                                 \
  \               \n\nDetermining conflicts:   0%|          | 0/8 [00:00<?, ?it/s]\n\
  Examining conflict for python requests pandas pip:   0%|          | 0/8 [00:00<?,\
  \ ?it/s]\nExamining conflict for requests pandas pip:  12%|\u2588\u258E        |\
  \ 1/8 [00:00<00:03,  1.97it/s]\nExamining conflict for requests pandas pip:  25%|\u2588\
  \u2588\u258C       | 2/8 [00:00<00:01,  3.94it/s]\nExamining conflict for python\
  \ pip:  25%|\u2588\u2588\u258C       | 2/8 [00:01<00:01,  3.94it/s]         \nExamining\
  \ conflict for python pip:  38%|\u2588\u2588\u2588\u258A      | 3/8 [00:01<00:02,\
  \  2.12it/s]\nExamining conflict for python pandas pip:  38%|\u2588\u2588\u2588\u258A\
  \      | 3/8 [00:01<00:02,  2.12it/s]\nExamining conflict for python pandas:  50%|\u2588\
  \u2588\u2588\u2588\u2588     | 4/8 [00:01<00:01,  2.12it/s]    \nExamining conflict\
  \ for python __glibc pandas:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\u258E   |\
  \ 5/8 [00:01<00:01,  2.12it/s]\nExamining conflict for python __glibc pandas:  75%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258C  | 6/8 [00:01<00:00,  2.71it/s]\nExamining\
  \ conflict for python requests:  75%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258C\
  \  | 6/8 [00:02<00:00,  2.71it/s]      \nExamining conflict for python requests:\
  \  88%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [00:02<00:00,\
  \  2.80it/s]\n                                                                 \
  \                    \nFound conflicts! Looking for incompatible packages.\nThis\
  \ can take several minutes.  Press CTRL-C to abort.\nfailed\nSolving environment:\
  \ ...working... \nBuilding graph of deps:   0%|          | 0/8 [00:00<?, ?it/s]\n\
  Examining pip==21.1.3:   0%|          | 0/8 [00:00<?, ?it/s] \nExamining python==3.9.5:\
  \  12%|\u2588\u258E        | 1/8 [00:00<00:01,  3.58it/s]\nExamining python==3.9.5:\
  \  25%|\u2588\u2588\u258C       | 2/8 [00:00<00:00,  7.16it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  25%|\u2588\u2588\u258C       | 2/8 [00:00<00:00,  7.16it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  38%|\u2588\u2588\u2588\u258A      | 3/8 [00:00<00:00,  7.16it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  50%|\u2588\u2588\u2588\u2588\u2588     | 4/8 [00:00<00:00,  7.16it/s]   \nExamining\
  \ pandas==1.2.4:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\u258E   | 5/8 [00:00<00:00,\
  \  7.16it/s]                \nExamining requests==2.25.1:  75%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258C  | 6/8 [00:00<00:00,  7.16it/s]\nExamining requests==2.25.1:\
  \  88%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [00:00<00:00,\
  \  7.42it/s]\nExamining @/linux-64::__unix==0=0:  88%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258A | 7/8 [00:01<00:00,  7.42it/s]\nExamining @/linux-64::__unix==0=0:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 8/8 [00:01<00:00,\
  \  6.76it/s]\n                                                                 \
  \               \n\nDetermining conflicts:   0%|          | 0/8 [00:00<?, ?it/s]\n\
  Examining conflict for python requests pandas pip:   0%|          | 0/8 [00:00<?,\
  \ ?it/s]\nExamining conflict for requests pandas pip:  12%|\u2588\u258E        |\
  \ 1/8 [00:00<00:03,  1.85it/s]\nExamining conflict for requests pandas pip:  25%|\u2588\
  \u2588\u258C       | 2/8 [00:00<00:01,  3.71it/s]\nExamining conflict for python\
  \ pip:  25%|\u2588\u2588\u258C       | 2/8 [00:01<00:01,  3.71it/s]         \nExamining\
  \ conflict for python pip:  38%|\u2588\u2588\u2588\u258A      | 3/8 [00:01<00:02,\
  \  1.99it/s]\nExamining conflict for python pandas pip:  38%|\u2588\u2588\u2588\u258A\
  \      | 3/8 [00:01<00:02,  1.99it/s]\nExamining conflict for python pandas:  50%|\u2588\
  \u2588\u2588\u2588\u2588     | 4/8 [00:01<00:02,  1.99it/s]    \nExamining conflict\
  \ for python __glibc pandas:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\u258E   |\
  \ 5/8 [00:02<00:01,  1.99it/s]\nExamining conflict for python __glibc pandas:  75%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258C  | 6/8 [00:02<00:00,  2.54it/s]\nExamining\
  \ conflict for python requests:  75%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258C\
  \  | 6/8 [00:02<00:00,  2.54it/s]      \nExamining conflict for python requests:\
  \  88%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [00:02<00:00,\
  \  2.62it/s]\n                                                                 \
  \                    \nFound conflicts! Looking for incompatible packages.\nThis\
  \ can take several minutes.  Press CTRL-C to abort.\nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with each other:\n\n\
  Output in format: Requested package -> Available versions\n\nPackage python conflicts\
  \ for:\npandas==1.2.4 -> numpy[version='>=1.19.5,<2.0a0'] -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.6,<3.7.0a0|>=3.6|>=3.5,<3.6.0a0|>=3|3.4.*|3.7.*|3.8.*|3.9.*']\n\
  requests==2.25.1 -> certifi[version='>=2017.4.17'] -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.7,<3.8.0a0|>=3.7|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|>=3.11,<3.12.0a0|<4.0|>=3.6,<4.0']\n\
  pip==21.1.3 -> setuptools -> python[version='!=3.0,!=3.1,!=3.2,!=3.3,!=3.4|2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.7|>=3.8|>=3.11,<3.12.0a0|>=3.5,<3.6.0a0|3.4.*|2.7.*|>=3.6']\n\
  pandas==1.2.4 -> python[version='>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0']\n\
  pip==21.1.3 -> python[version='>=3.6|>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0']\n\
  requests==2.25.1 -> python\n\nPackage ca-certificates conflicts for:\npython==3.9.5\
  \ -> openssl[version='>=1.1.1k,<1.1.2a'] -> ca-certificates\nrequests==2.25.1 ->\
  \ python -> ca-certificates\n\nPackage _libgcc_mutex conflicts for:\npython==3.9.5\
  \ -> libgcc-ng[version='>=9.3.0'] -> _libgcc_mutex[version='*|0.1',build='main|main|conda_forge']\n\
  pandas==1.2.4 -> libgcc-ng[version='>=9.3.0'] -> _libgcc_mutex[version='*|0.1',build='main|main|conda_forge']\n\
  \nPackage setuptools conflicts for:\npython==3.9.5 -> pip -> setuptools\npandas==1.2.4\
  \ -> setuptools[version='<60.0.0']\npip==21.1.3 -> setuptools\n\nPackage certifi\
  \ conflicts for:\nrequests==2.25.1 -> urllib3[version='>=1.21.1,<1.27'] -> certifi\n\
  requests==2.25.1 -> certifi[version='>=2017.4.17']\n\nPackage graalpy conflicts\
  \ for:\nrequests==2.25.1 -> python -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\
  pandas==1.2.4 -> python[version='>=3.8,<3.9.0a0'] -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\
  pip==21.1.3 -> python[version='>=3.6'] -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\
  \nPackage wheel conflicts for:\npython==3.9.5 -> pip -> wheel\npip==21.1.3 -> wheelThe\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - pandas==1.2.4 -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252FGTExSnake%2F0n391717%2Flogs.tar.xz">0.127GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252FGTExSnake%2F0n391717%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>2.550GiB</td>
						<td>523.4 seconds</td>
						<td>605.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010762896384">
								<a href="https://nf-co.re/airrflow">airrflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/airrflow/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>581 days</td>
						<td>2023-02-19 23:32:34</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 02:32:34.214
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2F2arkv21g%2Flogs.tar.xz">0.021GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2F2arkv21g%2Foutput.tar.xz">0.013GiB</a>
						</td>
						<td>0.902GiB</td>
						<td>189.9 seconds</td>
						<td>10801.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738301728">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.5">v2.0.5</a>
						</td>
						<td>snakemake</td>
						<td>580 days</td>
						<td>2023-02-19 04:32:12</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/73ftsyyx39/singularity-tmp/tmptegsom3r.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/9f71a7c8-cf94-404a-b7fd-f64b80956d96\
  \ singularity  exec --home /home/azureuser/tmp/73ftsyyx39/code/.test  /home/azureuser/tmp/73ftsyyx39/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/73ftsyyx39/code/.test/.snakemake/conda/f54f0356d651bd6065d3078d906255c0_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/73ftsyyx39/code/.test/.snakemake/conda/f54f0356d651bd6065d3078d906255c0_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining samtools==1.10:   0%|       \
  \   | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:  20%|\u2588\u2588\
  \        | 1/5 [00:00<00:00, 62.94it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 124.73it/s]\nExamining\
  \ @/linux-64::__archspec==1=x86_64:  60%|\u2588\u2588\u2588\u2588\u2588\u2588  \
  \  | 3/5 [00:00<00:00, 186.05it/s]\nExamining @/linux-64::__unix==0=0:  80%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 246.79it/s]    \
  \     \n                                                                       \
  \          \n\nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining\
  \ conflict for samtools __glibc:   0%|          | 0/5 [00:00<?, ?it/s]\n       \
  \                                                                       \nfailed\n\
  Solving environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining samtools==1.10:   0%|       \
  \   | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:  20%|\u2588\u2588\
  \        | 1/5 [00:00<00:00, 41.15it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 81.67it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00, 121.83it/s]\n\
  Examining @/linux-64::__unix==0=0:  80%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588  | 4/5 [00:00<00:00, 161.66it/s]         \n                             \
  \                                                    \n\nDetermining conflicts:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict for samtools __glibc:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\n                                      \
  \                                        \nfailed\n\nUnsatisfiableError: The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - samtools==1.10 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fnn1wtehv%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fnn1wtehv%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.269GiB</td>
						<td>458.2 seconds</td>
						<td>561.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739661120">
								<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake">IMS-Bio2Core-Facility/GTExSnake</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake/tree/v1.1.2">v1.1.2</a>
						</td>
						<td>snakemake</td>
						<td>579 days</td>
						<td>2023-02-20 04:31:42</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- api_queries.smk
kind: CalledProcessError
line_no: 17
rest: "Command ' singularity  exec --home /home/azureuser/tmp/viyysrzeyh/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/viyysrzeyh/code/.snakemake/singularity/7854903ba970aa1cae6503ecd6762e3d.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/home/azureuser/tmp/viyysrzeyh/code/.snakemake/conda/5ec3f6774ac5aa7a0e44c10640cb516c_'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/viyysrzeyh/code/.snakemake/scripts/tmpz60w_t2g.request.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/viyysrzeyh/code/workflow/rules/api_queries.smk\"\
  , line 17, in __rule_request\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T043143.256187.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252FGTExSnake%2Fzcbx5mna%2Flogs.tar.xz">0.319GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252FGTExSnake%2Fzcbx5mna%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>2.603GiB</td>
						<td>402.3 seconds</td>
						<td>517.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010737664976">
								<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake">IMS-Bio2Core-Facility/GTExSnake</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake/tree/v1.1.1">v1.1.1</a>
						</td>
						<td>snakemake</td>
						<td>579 days</td>
						<td>2023-02-19 04:21:58</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/nj4ua52p3y/code/workflow/rules/../envs/biomart.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/93cf35c7-0ba4-4f12-91c7-35e161d4b7db\
  \ singularity  exec --home /home/azureuser/tmp/nj4ua52p3y/code  /home/azureuser/tmp/nj4ua52p3y/code/.snakemake/singularity/9f56b38ad905baa9ffe4c187a4f76e32.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/nj4ua52p3y/code/.snakemake/conda/da1e6a0f10b2c621c9d122e8134c1ddd_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/nj4ua52p3y/code/.snakemake/conda/da1e6a0f10b2c621c9d122e8134c1ddd_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nBuilding graph of deps:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining lxml==4.6.3:   0%|          | 0/9 [00:00<?, ?it/s] \nExamining\
  \ @/linux-64::__linux==5.15.0=0:  11%|\u2588         | 1/9 [00:00<00:01,  5.76it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \ 11.52it/s]\nExamining @/linux-64::__archspec==1=x86_64:  22%|\u2588\u2588\u258F\
  \       | 2/9 [00:00<00:00, 11.52it/s]\nExamining pip==21.1.3:  33%|\u2588\u2588\
  \u2588\u258E      | 3/9 [00:00<00:00, 11.52it/s]                     \nExamining\
  \ bioservices==1.7.11:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,\
  \ 11.52it/s]\nExamining bioservices==1.7.11:  56%|\u2588\u2588\u2588\u2588\u2588\
  \u258C    | 5/9 [00:00<00:00, 11.50it/s]\nExamining @/linux-64::__unix==0=0:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 11.50it/s]\nExamining @/linux-64::__unix==0=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00,  7.27it/s]\n\
  Examining python==3.9.5:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9\
  \ [00:00<00:00,  7.27it/s]          \nExamining pandas==1.2.4:  78%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:00<00:00,  7.27it/s]\nExamining\
  \ @/linux-64::__glibc==2.31=0:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:01<00:00,  7.27it/s]\nExamining @/linux-64::__glibc==2.31=0: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:01<00:00,  6.47it/s]\n\
  \                                                                              \
  \      \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for lxml bioservices:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ conflict for lxml pip bioservices pandas:  11%|\u2588         | 1/9 [00:00<00:00,\
  \ 1422.28it/s]\nExamining conflict for lxml pip bioservices python pandas:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:01<00:06,  1.14it/s]\nExamining conflict for lxml pip\
  \ bioservices python pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:01<00:03,\
  \  1.71it/s]\nExamining conflict for lxml pandas:  33%|\u2588\u2588\u2588\u258E\
  \      | 3/9 [00:02<00:03,  1.71it/s]                       \nExamining conflict\
  \ for lxml pandas:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:02<00:02, \
  \ 1.73it/s]\nExamining conflict for python lxml pandas:  44%|\u2588\u2588\u2588\u2588\
  \u258D     | 4/9 [00:02<00:02,  1.73it/s]\nExamining conflict for python lxml pandas:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:02<00:02,  1.74it/s]\n\
  Examining conflict for python lxml __glibc pandas:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:03<00:02,  1.74it/s]\nExamining conflict for python lxml\
  \ __glibc pandas:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:03<00:01,\
  \  1.84it/s]\nExamining conflict for pip bioservices pandas:  67%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258B   | 6/9 [00:03<00:01,  1.84it/s]    \nExamining conflict\
  \ for pip bioservices pandas:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [00:03<00:00,  2.07it/s]\nExamining conflict for python pip bioservices\
  \ pandas:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:04<00:00,\
  \  2.07it/s]\nExamining conflict for python pip bioservices pandas:  89%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:04<00:00,  1.86it/s]\nExamining\
  \ conflict for python pip:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:04<00:00,  1.86it/s]                   \nExamining conflict for\
  \ python pip: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  | 9/9 [00:04<00:00,  2.22it/s]\nExamining conflict for bioservices pandas: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:04<00:00,  2.22it/s]\n\
  Examining conflict for python bioservices: : 10it [00:04,  2.22it/s]           \
  \         \n                                                                   \
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\nfailed\nSolving environment: ...working... \nBuilding\
  \ graph of deps:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining lxml==4.6.3: \
  \  0%|          | 0/9 [00:00<?, ?it/s] \nExamining @/linux-64::__linux==5.15.0=0:\
  \  11%|\u2588         | 1/9 [00:00<00:01,  5.59it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 11.18it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 11.18it/s]\nExamining pip==21.1.3:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 11.18it/s]            \
  \         \nExamining bioservices==1.7.11:  44%|\u2588\u2588\u2588\u2588\u258D \
  \    | 4/9 [00:00<00:00, 11.18it/s]\nExamining bioservices==1.7.11:  56%|\u2588\u2588\
  \u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 11.13it/s]\nExamining @/linux-64::__unix==0=0:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 11.13it/s]\n\
  Examining @/linux-64::__unix==0=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B\
  \   | 6/9 [00:00<00:00,  7.05it/s]\nExamining python==3.9.5:  67%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00,  7.05it/s]          \nExamining pandas==1.2.4:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:00<00:00,  7.05it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2589 | 8/9 [00:01<00:00,  7.05it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \ 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:01<00:00,\
  \  6.24it/s]\n                                                                 \
  \                   \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\n\
  Examining conflict for lxml bioservices:   0%|          | 0/9 [00:00<?, ?it/s]\n\
  Examining conflict for lxml pip bioservices pandas:  11%|\u2588         | 1/9 [00:00<00:00,\
  \ 1415.08it/s]\nExamining conflict for lxml pip bioservices python pandas:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:01<00:06,  1.08it/s]\nExamining conflict for lxml pip\
  \ bioservices python pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:01<00:03,\
  \  1.62it/s]\nExamining conflict for lxml pandas:  33%|\u2588\u2588\u2588\u258E\
  \      | 3/9 [00:02<00:03,  1.62it/s]                       \nExamining conflict\
  \ for lxml pandas:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:02<00:03, \
  \ 1.64it/s]\nExamining conflict for python lxml pandas:  44%|\u2588\u2588\u2588\u2588\
  \u258D     | 4/9 [00:03<00:03,  1.64it/s]\nExamining conflict for python lxml pandas:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:02,  1.65it/s]\n\
  Examining conflict for python lxml __glibc pandas:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [00:03<00:02,  1.65it/s]\nExamining conflict for python lxml\
  \ __glibc pandas:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:03<00:01,\
  \  1.74it/s]\nExamining conflict for pip bioservices pandas:  67%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258B   | 6/9 [00:03<00:01,  1.74it/s]    \nExamining conflict\
  \ for pip bioservices pandas:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [00:03<00:01,  1.96it/s]\nExamining conflict for python pip bioservices\
  \ pandas:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:04<00:01,\
  \  1.96it/s]\nExamining conflict for python pip bioservices pandas:  89%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:04<00:00,  1.75it/s]\nExamining\
  \ conflict for python pip:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:04<00:00,  1.75it/s]                   \nExamining conflict for\
  \ python pip: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  | 9/9 [00:04<00:00,  2.10it/s]\nExamining conflict for bioservices pandas: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:04<00:00,  2.10it/s]\n\
  Examining conflict for python bioservices: : 10it [00:04,  2.10it/s]           \
  \         \n                                                                   \
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\nfailed\n\nUnsatisfiableError: The following specifications\
  \ were found to be incompatible with each other:\n\nOutput in format: Requested\
  \ package -> Available versions\n\nPackage pypy3.6 conflicts for:\nlxml==4.6.3 ->\
  \ pypy3.6[version='>=7.3.3']\nlxml==4.6.3 -> python[version='>=3.6,<3.7.0a0'] ->\
  \ pypy3.6[version='7.3.*|7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*']\n\nPackage expat conflicts\
  \ for:\nlxml==4.6.3 -> pypy3.6[version='>=7.3.3'] -> expat[version='>=2.2.9,<3.0.0a0|>=2.4.1,<3.0a0|>=2.3.0,<3.0a0']\n\
  pandas==1.2.4 -> pypy3.7[version='>=7.3.3'] -> expat[version='>=2.2.9,<3.0.0a0|>=2.3.0,<3.0a0|>=2.4.1,<3.0a0']\n\
  \nPackage pypy3.7 conflicts for:\nlxml==4.6.3 -> pypy3.7[version='>=7.3.3']\nlxml==4.6.3\
  \ -> python[version='>=3.7,<3.8.0a0'] -> pypy3.7[version='7.3.*|7.3.3.*|7.3.4.*|7.3.5.*|7.3.7.*']\n\
  \nPackage ca-certificates conflicts for:\nbioservices==1.7.11 -> python -> ca-certificates\n\
  python==3.9.5 -> openssl[version='>=1.1.1k,<1.1.2a'] -> ca-certificates\n\nPackage\
  \ pandas conflicts for:\npandas==1.2.4\nbioservices==1.7.11 -> pandas\n\nPackage\
  \ certifi conflicts for:\nbioservices==1.7.11 -> requests -> certifi[version='>=2017.4.17']\n\
  pip==21.1.3 -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']\npandas==1.2.4\
  \ -> setuptools[version='<60.0.0'] -> certifi[version='>=2016.09|>=2016.9.26']\n\
  \nPackage pip conflicts for:\npip==21.1.3\nbioservices==1.7.11 -> python -> pip\n\
  pandas==1.2.4 -> python[version='>=3.7,<3.8.0a0'] -> pip\npython==3.9.5 -> pip\n\
  \nPackage libuuid conflicts for:\npip==21.1.3 -> python[version='>=3.6'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0|>=2.32.1,<3.0a0']\n\
  pandas==1.2.4 -> python[version='>=3.8,<3.9.0a0'] -> libuuid[version='>=2.32.1,<3.0a0']\n\
  lxml==4.6.3 -> python[version='>=3.9,<3.10.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0|>=2.32.1,<3.0a0']\n\
  bioservices==1.7.11 -> python -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0|>=2.32.1,<3.0a0']\n\
  \nPackage lxml conflicts for:\nlxml==4.6.3\nbioservices==1.7.11 -> lxml\n\nPackage\
  \ setuptools conflicts for:\nbioservices==1.7.11 -> easydev[version='>=0.9.36']\
  \ -> setuptools[version='<60.0.0']\npython==3.9.5 -> pip -> setuptools\npip==21.1.3\
  \ -> setuptools\npandas==1.2.4 -> setuptools[version='<60.0.0']\n\nPackage python_abi\
  \ conflicts for:\nlxml==4.6.3 -> python[version='>=3.9,<3.10.0a0'] -> python_abi[version='3.8.*|3.8|3.9',build='*_pypy39_pp73|*_pypy38_pp73|*_graalpy223_38_native']\n\
  lxml==4.6.3 -> python_abi[version='3.6|3.6.*|3.9.*|3.7.*|3.8.*|3.7',build='*_pypy37_pp73|*_cp39|*_cp36m|*_cp37m|*_cp38|*_pypy36_pp73']\n\
  \nPackage _libgcc_mutex conflicts for:\nlxml==4.6.3 -> libgcc-ng[version='>=9.3.0']\
  \ -> _libgcc_mutex[version='*|0.1',build='main|conda_forge|main']\npython==3.9.5\
  \ -> libgcc-ng[version='>=9.3.0'] -> _libgcc_mutex[version='*|0.1',build='main|conda_forge|main']\n\
  pandas==1.2.4 -> libgcc-ng[version='>=9.3.0'] -> _libgcc_mutex[version='*|0.1',build='main|conda_forge|main']\n\
  \nPackage wheel conflicts for:\npip==21.1.3 -> wheel\npython==3.9.5 -> pip -> wheelThe\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - pandas==1.2.4 -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17']\n\
  \  - python==3.9.5 -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252FGTExSnake%2Fx1i82sd3%2Flogs.tar.xz">0.127GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252FGTExSnake%2Fx1i82sd3%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>2.531GiB</td>
						<td>434.0 seconds</td>
						<td>390.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738205888">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.3.5">1.3.5</a>
						</td>
						<td>snakemake</td>
						<td>578 days</td>
						<td>2023-02-19 06:33:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- dtbjnxj7wj
- code
- rules
- blast.smk
kind: CalledProcessError
line_no: 148
rest: "Command 'set -euo pipefail;  /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/python3.10\
  \ /home/azureuser/tmp/dtbjnxj7wj/code/tests/.snakemake/scripts/tmpwi8m0sd4.blast_to_lca.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/dtbjnxj7wj/code/rules/blast.smk\"\
  , line 148, in __rule_blast2lca\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Sun Feb 19 06:41:23 2023]\nFinished job 45.\n26 of 62 steps\
  \ (42%) done\nRemoving temporary output NTC/trimmed/NTC_R2.fastq.\nRemoving temporary\
  \ output NTC/trimmed/NTC_R1.fastq.\nRemoving temporary output NTC/denoising/NTC_R1_filtered.fasta.\n\
  Removing temporary output NTC/denoising/NTC_R2_filtered.fasta.\nShutting down, this\
  \ might take some time.\nExiting because a job execution failed. Look above for\
  \ error message\nComplete log: .snakemake/log/2023-02-19T063319.717473.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fd15e67n0%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fd15e67n0%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.007GiB</td>
						<td>320.8 seconds</td>
						<td>487.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738301488">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v2.0.6">v2.0.6</a>
						</td>
						<td>snakemake</td>
						<td>578 days</td>
						<td>2023-02-19 03:31:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/erm8khcnuq/singularity-tmp/tmpw3l1ax_l.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/e7e961c5-6134-44c1-8e3b-49690b8c63f1\
  \ singularity  exec --home /home/azureuser/tmp/erm8khcnuq/code/.test  /home/azureuser/tmp/erm8khcnuq/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/erm8khcnuq/code/.test/.snakemake/conda/2e9923dffae43ad0855e6f82ee268672_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/erm8khcnuq/code/.test/.snakemake/conda/2e9923dffae43ad0855e6f82ee268672_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining samtools==1.10:  20%|\u2588\u2588\
  \        | 1/5 [00:00<00:00, 7884.03it/s]     \nExamining @/linux-64::__archspec==1=x86_64:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 124.03it/s]\nExamining\
  \ @/linux-64::__unix==0=0:  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00,\
  \ 184.66it/s]         \nExamining @/linux-64::__glibc==2.31=0:  80%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 245.01it/s]\n        \
  \                                                                             \n\
  \nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict\
  \ for __glibc samtools:   0%|          | 0/5 [00:00<?, ?it/s]\n                \
  \                                                              \nfailed\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining samtools==1.10:  20%|\u2588\u2588\
  \        | 1/5 [00:00<00:00, 6026.30it/s]     \nExamining @/linux-64::__archspec==1=x86_64:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 70.70it/s]\nExamining @/linux-64::__unix==0=0:\
  \  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00, 105.34it/s] \
  \       \nExamining @/linux-64::__glibc==2.31=0:  80%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588  | 4/5 [00:00<00:00, 139.74it/s]\n                          \
  \                                                           \n\nDetermining conflicts:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict for __glibc samtools:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\n                                      \
  \                                        \nfailed\n\nUnsatisfiableError: The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - samtools==1.10 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fn7t2b141%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fn7t2b141%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.213GiB</td>
						<td>439.0 seconds</td>
						<td>432.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743111904">
								<a href="https://nf-co.re/fetchngs">fetchngs</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/fetchngs/tree/1.2">1.2</a>
						</td>
						<td>nextflow</td>
						<td>572 days</td>
						<td>2023-02-19 20:29:32</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-19 22:29:32.144
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fybbs5heq%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fybbs5heq%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.570GiB</td>
						<td>41.2 seconds</td>
						<td>7200.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743119104">
								<a href="https://nf-co.re/mag">mag</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mag/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>571 days</td>
						<td>2023-02-20 04:31:45</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 07:31:45.923
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fdq3s0f19%2Flogs.tar.xz">0.108GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fdq3s0f19%2Foutput.tar.xz">0.008GiB</a>
						</td>
						<td>1.475GiB</td>
						<td>776.6 seconds</td>
						<td>10800.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743195504">
								<a href="https://nf-co.re/viralrecon">viralrecon</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/viralrecon/tree/2.2">2.2</a>
						</td>
						<td>nextflow</td>
						<td>571 days</td>
						<td>2023-02-20 05:20:31</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 08:20:31.034
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Fpgyjqzvp%2Flogs.tar.xz">0.096GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Fpgyjqzvp%2Foutput.tar.xz">0.018GiB</a>
						</td>
						<td>1.205GiB</td>
						<td>592.7 seconds</td>
						<td>10802.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743185664">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.3">3.3</a>
						</td>
						<td>nextflow</td>
						<td>570 days</td>
						<td>2023-02-19 04:30:57</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.NullPointerException
msg: Cannot get property 'args' on null object
rest: "at org.codehaus.groovy.runtime.NullObject.getProperty(NullObject.java:60)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:194)\n\
  \tat org.codehaus.groovy.runtime.callsite.NullCallSite.getProperty(NullCallSite.java:46)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:329)\n\
  \tat Script_0e771576.runScript(Script_0e771576:80)\n\tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:170)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:216)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:206)\n\
  \tat nextflow.script.IncludeDef.memoizedMethodPriv$loadModule0PathMapSession(IncludeDef.groovy:152)\n\
  \tat nextflow.script.IncludeDef.access$0(IncludeDef.groovy)\n\tat nextflow.script.IncludeDef$__clinit__closure2.doCall(IncludeDef.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat nextflow.script.IncludeDef.loadModule0(IncludeDef.groovy)\n\tat nextflow.script.IncludeDef.load0(IncludeDef.groovy:124)\n\
  \tat nextflow.script.IncludeDef$load0$1.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_4c643811.runScript(Script_4c643811:43)\n\tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:170)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\
  \tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\
  \tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fwgbz74du%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fwgbz74du%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.808GiB</td>
						<td>23.3 seconds</td>
						<td>16.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738267760">
								<a href="https://github.com/mrvollger/StainedGlass">mrvollger/StainedGlass</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/StainedGlass/tree/v0.1">v0.1</a>
						</td>
						<td>snakemake</td>
						<td>545 days</td>
						<td>2023-02-19 06:00:33</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/suyy0z3il0/code/workflow/envs/env.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/suyy0z3il0/code/.snakemake/conda/76796b704a0bb8e96b920ea5d3ddcfec_.yaml"
  --prefix "/home/azureuser/tmp/suyy0z3il0/code/.snakemake/conda/76796b704a0bb8e96b920ea5d3ddcfec_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working...'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FStainedGlass%2Fi1elljid%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FStainedGlass%2Fi1elljid%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>9.230GiB</td>
						<td>938.1 seconds</td>
						<td>1089.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738324656">
								<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2">snakemake-workflows/rna-seq-star-deseq2</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.2.0">v1.2.0</a>
						</td>
						<td>snakemake</td>
						<td>544 days</td>
						<td>2023-02-19 04:28:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/wg1bxme9mx/code/workflow/rules/../envs/deseq2.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/22240503-3428-4e2b-aa9d-0dc03e60fa16\
  \ singularity  exec --home /home/azureuser/tmp/wg1bxme9mx/code/.test  /home/azureuser/tmp/wg1bxme9mx/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/wg1bxme9mx/code/.test/.snakemake/conda/2e89e211a3c06350ccd826f2f72f1dea_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/wg1bxme9mx/code/.test/.snakemake/conda/2e89e211a3c06350ccd826f2f72f1dea_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|\
  \          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64: \
  \ 17%|\u2588\u258B        | 1/6 [00:00<00:00, 6009.03it/s]\nExamining r-ashr=2.2_47:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 7598.38it/s]          \
  \         \nExamining @/linux-64::__linux==5.15.0=0:  50%|\u2588\u2588\u2588\u2588\
  \u2588     | 3/6 [00:00<00:00, 25.27it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 4/6 [00:00<00:00, 33.66it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 4/6 [00:00<00:00, 33.66it/s]  \nExamining bioconductor-deseq2=1.30.0:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 33.66it/s] \n                                                                \
  \                   \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining conflict for r-ashr bioconductor-deseq2:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for r-ashr __glibc bioconductor-deseq2:  17%|\u2588\
  \u258B        | 1/6 [00:11<00:59, 11.81s/it]\nExamining conflict for r-ashr __glibc\
  \ bioconductor-deseq2:  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:11<00:23,  5.91s/it]\n\
  \                                                                              \
  \                          \nfailed\nSolving environment: ...working... \nFound\
  \ conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining @/linux-64::__archspec==1=x86_64:  17%|\u2588\u258B        | 1/6 [00:00<00:00,\
  \ 8224.13it/s]\nExamining r-ashr=2.2_47:  33%|\u2588\u2588\u2588\u258E      | 2/6\
  \ [00:00<00:00, 9404.27it/s]                   \nExamining @/linux-64::__linux==5.15.0=0:\
  \  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00, 23.55it/s]\nExamining\
  \ @/linux-64::__linux==5.15.0=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B\
  \   | 4/6 [00:00<00:00, 31.36it/s]\nExamining @/linux-64::__glibc==2.31=0:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 4/6 [00:00<00:00, 31.36it/s]  \nExamining\
  \ bioconductor-deseq2=1.30.0:  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258E | 5/6 [00:00<00:00, 31.36it/s] \n                                       \
  \                                            \n\nDetermining conflicts:   0%|  \
  \        | 0/6 [00:00<?, ?it/s]\nExamining conflict for r-ashr bioconductor-deseq2:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining conflict for r-ashr __glibc bioconductor-deseq2:\
  \  17%|\u2588\u258B        | 1/6 [00:11<00:58, 11.68s/it]\nExamining conflict for\
  \ r-ashr __glibc bioconductor-deseq2:  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:11<00:23,\
  \  5.84s/it]\n                                                                 \
  \                                       \nfailed\n\nUnsatisfiableError: The following\
  \ specifications were found to be incompatible with each other:\n\nOutput in format:\
  \ Requested package -> Available versions\n\nPackage libstdcxx-ng conflicts for:\n\
  r-ashr=2.2_47 -> libstdcxx-ng[version='>=7.3.0|>=9.3.0']\nr-ashr=2.2_47 -> r-base[version='>=4.0,<4.1.0a0']\
  \ -> libstdcxx-ng[version='7.2.0.*|>=10.3.0|>=12|>=9.4.0|>=7.5.0|>=11.2.0|>=4.9|>=7.2.0']\n\
  \nPackage r-base conflicts for:\nr-ashr=2.2_47 -> r-base[version='>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=4.0,<4.1.0a0|>=4.1,<4.2.0a0']\n\
  r-ashr=2.2_47 -> r-etrunct -> r-base[version='3.1.3.*|3.2.0.*|3.2.1.*|3.2.2.*|3.3.1.*|3.3.2.*|3.4.1.*|>=3.5.1,<3.5.2.0a0|>=4.2,<4.3.0a0|>=3.4.1,<3.4.2.0a0|>=3.5.0,<3.5.1.0a0|>=3.4.3,<3.4.4.0a0|>=3.4.2,<3.4.3a0']\n\
  \nPackage libgcc-ng conflicts for:\nr-ashr=2.2_47 -> libgcc-ng[version='>=7.3.0|>=9.3.0']\n\
  r-ashr=2.2_47 -> r-base[version='>=4.0,<4.1.0a0'] -> libgcc-ng[version='7.2.0.*|>=10.3.0|>=12|>=9.4.0|>=7.5.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  \nPackage r-rcpp conflicts for:\nr-ashr=2.2_47 -> r-mixsqp -> r-rcpp[version='>=0.12.15']\n\
  r-ashr=2.2_47 -> r-rcpp[version='>=0.10.5']\n\nPackage gcc_linux-64 conflicts for:\n\
  r-ashr=2.2_47 -> r-base[version='>=4.0,<4.1.0a0'] -> gcc_linux-64[version='7.*|>=10|>=9|>=7|7.3.0.*']\n\
  bioconductor-deseq2=1.30.0 -> r-base[version='>=4.0,<4.1.0a0'] -> gcc_linux-64[version='>=10|>=9|>=7']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - bioconductor-deseq2=1.30.0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17|>=2.17,<3.0.a0']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-star-deseq2%2Fx5ry9h5m%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-star-deseq2%2Fx5ry9h5m%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.140GiB</td>
						<td>180.1 seconds</td>
						<td>198.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745674624">
								<a href="https://nf-co.re/bacass">bacass</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/bacass/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>541 days</td>
						<td>2023-02-19 06:15:40</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/bacass%2Fjaw6wm77%2Flogs.tar.xz">0.636GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/bacass%2Fjaw6wm77%2Foutput.tar.xz">0.219GiB</a>
						</td>
						<td>1.517GiB</td>
						<td>2839.8 seconds</td>
						<td>1800.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738288288">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.2.0">v2.2.0</a>
						</td>
						<td>snakemake</td>
						<td>538 days</td>
						<td>2023-02-19 06:00:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/on6kjixqmo/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/1b2b218c-eb64-445c-b7aa-cfe63e993550\
  \ singularity  exec --home /home/azureuser/tmp/on6kjixqmo/code/.test  /home/azureuser/tmp/on6kjixqmo/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/on6kjixqmo/code/.test/.snakemake/conda/0d91dad51f085d88ee3c0ca9c35e252a_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/on6kjixqmo/code/.test/.snakemake/conda/0d91dad51f085d88ee3c0ca9c35e252a_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining goatools=0.9.7:   0%|       \
  \   | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:  11%|\u2588     \
  \    | 1/9 [00:00<00:03,  2.63it/s]\nExamining @/linux-64::__unix==0=0:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:00<00:01,  5.26it/s]\nExamining pillow=6.2:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:00<00:01,  5.26it/s]             \nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:01<00:01,  5.26it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:01<00:01,  3.77it/s]\nExamining\
  \ matplotlib=3.1:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:01<00:01,  3.77it/s]\
  \             \nExamining python=3.7:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:03<00:01,  3.77it/s]    \nExamining python=3.7:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:03<00:02,  1.32it/s]\nExamining pandas=0.25:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:06<00:02,  1.32it/s]\n\
  Examining pandas=0.25:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  |\
  \ 7/9 [00:06<00:02,  1.21s/it]\nExamining @/linux-64::__archspec==1=x86_64:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:09<00:02,  1.21s/it]\nExamining\
  \ @/linux-64::__archspec==1=x86_64:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2589 | 8/9 [00:09<00:01,  1.68s/it]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:09<00:01,\
  \  1.68s/it]   \n                                                              \
  \                        \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for matplotlib goatools pandas:   0%|          | 0/9\
  \ [00:00<?, ?it/s]\nExamining conflict for goatools pillow matplotlib python pandas:\
  \  11%|\u2588         | 1/9 [00:43<05:45, 43.16s/it]\nExamining conflict for goatools\
  \ pillow matplotlib python pandas:  22%|\u2588\u2588\u258F       | 2/9 [00:43<02:31,\
  \ 21.58s/it]\nExamining conflict for matplotlib goatools:  22%|\u2588\u2588\u258F\
  \       | 2/9 [00:46<02:31, 21.58s/it]                     \nExamining conflict\
  \ for matplotlib goatools:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:46<01:22,\
  \ 13.78s/it]\nExamining conflict for goatools pandas:  33%|\u2588\u2588\u2588\u258E\
  \      | 3/9 [00:50<01:22, 13.78s/it]    \nExamining conflict for goatools pandas:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:50<00:50, 10.17s/it]\nExamining\
  \ conflict for matplotlib pillow goatools pandas:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [00:51<00:50, 10.17s/it]\nExamining conflict for matplotlib pillow goatools\
  \ pandas:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:51<00:28,  7.19s/it]\n\
  Examining conflict for python pillow goatools:  56%|\u2588\u2588\u2588\u2588\u2588\
  \u258C    | 5/9 [01:05<00:28,  7.19s/it]           \nExamining conflict for python\
  \ pillow goatools:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [01:05<00:28,\
  \  9.39s/it]\nExamining conflict for python matplotlib pillow pandas:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [01:07<00:28,  9.39s/it]\nExamining\
  \ conflict for python matplotlib pillow pandas:  78%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258A  | 7/9 [01:07<00:13,  6.98s/it]\nExamining conflict for matplotlib\
  \ pillow:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [01:12<00:13,\
  \  6.98s/it]              \nExamining conflict for matplotlib pillow:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [01:12<00:06,  6.28s/it]\n\
  Examining conflict for pillow __glibc matplotlib python pandas:  89%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [01:14<00:06,  6.28s/it]\nExamining\
  \ conflict for pillow __glibc matplotlib python pandas: 100%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [01:14<00:00,  5.19s/it]\nExamining conflict\
  \ for python pillow: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588| 9/9 [01:16<00:00,  5.19s/it]                          \nExamining conflict\
  \ for python pillow: : 10it [01:16,  4.14s/it]                    \nExamining conflict\
  \ for matplotlib pandas: : 10it [01:17,  4.14s/it]\nExamining conflict for matplotlib\
  \ pandas: : 11it [01:17,  3.09s/it]\nExamining conflict for matplotlib python pandas:\
  \ : 11it [01:19,  3.09s/it]\nExamining conflict for matplotlib python pandas: :\
  \ 12it [01:19,  2.90s/it]\n                                                    \
  \                      \nfailed\nSolving environment: ...working... \nFound conflicts!\
  \ Looking for incompatible packages.\nThis can take several minutes.  Press CTRL-C\
  \ to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ goatools=0.9.7:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:\
  \  11%|\u2588         | 1/9 [00:00<00:03,  2.29it/s]\nExamining @/linux-64::__unix==0=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:01,  4.58it/s]\nExamining pillow=6.2:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:01,  4.58it/s]             \nExamining\
  \ @/linux-64::__glibc==2.31=0:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:01<00:01,\
  \  4.58it/s]\nExamining @/linux-64::__glibc==2.31=0:  44%|\u2588\u2588\u2588\u2588\
  \u258D     | 4/9 [00:01<00:01,  3.38it/s]\nExamining matplotlib=3.1:  44%|\u2588\
  \u2588\u2588\u2588\u258D     | 4/9 [00:01<00:01,  3.38it/s]             \nExamining\
  \ python=3.7:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:01, \
  \ 3.38it/s]    \nExamining python=3.7:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 6/9 [00:03<00:02,  1.26it/s]\nExamining pandas=0.25:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:05<00:02,  1.26it/s]\nExamining pandas=0.25:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:05<00:02,  1.03s/it]\n\
  Examining @/linux-64::__archspec==1=x86_64:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:08<00:02,  1.03s/it]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:08<00:01,\
  \  1.40s/it]\nExamining @/linux-64::__linux==5.15.0=0:  89%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2589 | 8/9 [00:08<00:01,  1.40s/it]   \n             \
  \                                                                         \n\nDetermining\
  \ conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for matplotlib\
  \ goatools pandas:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for\
  \ goatools pillow matplotlib python pandas:  11%|\u2588         | 1/9 [00:49<06:34,\
  \ 49.28s/it]\nExamining conflict for goatools pillow matplotlib python pandas: \
  \ 22%|\u2588\u2588\u258F       | 2/9 [00:49<02:52, 24.64s/it]\nExamining conflict\
  \ for matplotlib goatools:  22%|\u2588\u2588\u258F       | 2/9 [00:53<02:52, 24.64s/it]\
  \                     \nExamining conflict for matplotlib goatools:  33%|\u2588\u2588\
  \u2588\u258E      | 3/9 [00:53<01:35, 15.93s/it]\nExamining conflict for goatools\
  \ pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:57<01:35, 15.93s/it]    \n\
  Examining conflict for goatools pandas:  44%|\u2588\u2588\u2588\u2588\u258D    \
  \ | 4/9 [00:57<00:57, 11.56s/it]\nExamining conflict for matplotlib pillow goatools\
  \ pandas:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:58<00:57, 11.56s/it]\n\
  Examining conflict for matplotlib pillow goatools pandas:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:58<00:32,  8.07s/it]\nExamining conflict for python\
  \ pillow goatools:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [01:13<00:32,\
  \  8.07s/it]           \nExamining conflict for python pillow goatools:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [01:13<00:30, 10.33s/it]\nExamining\
  \ conflict for python matplotlib pillow pandas:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [01:15<00:30, 10.33s/it]\nExamining conflict for python matplotlib\
  \ pillow pandas:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [01:15<00:15,\
  \  7.62s/it]\nExamining conflict for matplotlib pillow:  78%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258A  | 7/9 [01:20<00:15,  7.62s/it]              \nExamining\
  \ conflict for matplotlib pillow:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2589 | 8/9 [01:20<00:06,  6.72s/it]\nExamining conflict for pillow __glibc\
  \ matplotlib python pandas:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [01:23<00:06,  6.72s/it]\nExamining conflict for pillow __glibc matplotlib\
  \ python pandas: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  | 9/9 [01:23<00:00,  5.48s/it]\nExamining conflict for python pillow: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [01:24<00:00,  5.48s/it]\
  \                          \nExamining conflict for python pillow: : 10it [01:24,\
  \  4.32s/it]                    \nExamining conflict for matplotlib pandas: : 10it\
  \ [01:25,  4.32s/it]\nExamining conflict for matplotlib pandas: : 11it [01:25, \
  \ 3.24s/it]\nExamining conflict for matplotlib python pandas: : 11it [01:28,  3.24s/it]\n\
  Examining conflict for matplotlib python pandas: : 12it [01:28,  3.02s/it]\n   \
  \                                                                       \nfailed\n\
  \nUnsatisfiableError: The following specifications were found to be incompatible\
  \ with each other:\n\nOutput in format: Requested package -> Available versions\n\
  \nPackage _libgcc_mutex conflicts for:\npandas=0.25 -> libgcc-ng[version='>=7.3.0']\
  \ -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\nmatplotlib=3.1\
  \ -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  python=3.7 -> libgcc-ng[version='>=9.4.0'] -> _libgcc_mutex[version='*|0.1',build='main|conda_forge|main']\n\
  pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  \nPackage python conflicts for:\npillow=6.2 -> olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  pandas=0.25 -> python[version='>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']\n\
  python=3.7\ngoatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  goatools=0.9.7 -> python\npandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|>=3.6|>=3|3.4.*']\n\
  pillow=6.2 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  \nPackage gdbm conflicts for:\npillow=6.2 -> pypy3.6[version='>=7.3.1'] -> gdbm[version='>=1.18,<1.19.0a0']\n\
  python=3.7 -> pypy3.7=7.3.7 -> gdbm[version='>=1.18,<1.19.0a0']\n\nPackage freetype\
  \ conflicts for:\nmatplotlib=3.1 -> freetype[version='>=2.9.1,<3.0a0']\npillow=6.2\
  \ -> freetype[version='>=2.9.1,<3.0a0']\n\nPackage pandas conflicts for:\ngoatools=0.9.7\
  \ -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\npandas=0.25\n\
  \nPackage pypy3.8 conflicts for:\ngoatools=0.9.7 -> numpy -> pypy3.8[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\n\
  matplotlib=3.1 -> python[version='>=3.8,<3.9.0a0'] -> pypy3.8[version='7.3.11.*|7.3.9.*|7.3.8.*|>=7.3.9|>=7.3.8|>=7.3.11']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> pypy3.8[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\n\
  pillow=6.2 -> python[version='>=3.8,<3.9.0a0'] -> pypy3.8[version='7.3.11.*|7.3.9.*|7.3.8.*']\n\
  \nPackage numpy conflicts for:\nmatplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0']\n\
  matplotlib=3.1 -> matplotlib-base==3.1.0=py37h5f35d83_0 -> numpy[version='>=1.11.3,<2.0a0']\n\
  \nPackage libpng conflicts for:\nmatplotlib=3.1 -> libpng[version='>=1.6.37,<1.7.0a0']\n\
  matplotlib=3.1 -> matplotlib-base==3.1.1=py36he7580a8_1 -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  \nPackage libgcc conflicts for:\nmatplotlib=3.1 -> pyqt -> libgcc\ngoatools=0.9.7\
  \ -> scipy -> libgcc\n\nPackage pytz conflicts for:\npandas=0.25 -> pytz[version='>=2017.2']\n\
  matplotlib=3.1 -> pytz\n\nPackage ca-certificates conflicts for:\ngoatools=0.9.7\
  \ -> python -> ca-certificates\npillow=6.2 -> python[version='>=2.7,<2.8.0a0'] ->\
  \ ca-certificates\npython=3.7 -> openssl[version='>=1.1.1l,<1.1.2a'] -> ca-certificates\n\
  \nPackage setuptools conflicts for:\nmatplotlib=3.1 -> setuptools\npandas=0.25 ->\
  \ setuptools[version='<60.0.0']\npython=3.7 -> pip -> setuptools\n\nPackage numpy-base\
  \ conflicts for:\ngoatools=0.9.7 -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py36h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h2b20989_6|py37h2b20989_6|py27hdbf6ddf_6|py27hdbf6ddf_7|py27h2b20989_7|py36hdbf6ddf_7|py36h2b20989_7|py37hdbf6ddf_7|py37h2b20989_7|py27h2b20989_7|py36hdbf6ddf_7|py36hdbf6ddf_8|py37h2b20989_8|py27hdbf6ddf_8|py35h2b20989_8|py36h7cdd4dd_9|py35h7cdd4dd_9|py27h7cdd4dd_9|py37h3dfced4_9|py27h3dfced4_9|py37h81de0dd_9|py27h74e8950_9|py35h74e8950_9|py36h74e8950_9|py27h74e8950_10|py36h74e8950_10|py27h81de0dd_10|py36h81de0dd_10|py36h2f8d375_10|py27h2f8d375_10|py35h2f8d375_10|py36h2f8d375_11|py37h2f8d375_12|py27h2f8d375_12|py27hde5b4d6_12|py38h2f8d375_12|py35h0ea5e3f_1|py27h9be14a7_1|py35h9be14a7_1|py36h2b20989_0|py35h2b20989_0|py36hdbf6ddf_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_1|py37hdbf6ddf_2|py36hdbf6ddf_3|py27h2b20989_3|py37h2b20989_4|py37hdbf6ddf_4|py35hdbf6ddf_4|py35h2b20989_4|py37h2f8d375_4|py36h2f8d375_4|py38hde5b4d6_4|py27h2f8d375_5|py37hde5b4d6_5|py27h7cdd4dd_0|py37h7cdd4dd_0|py27h3dfced4_0|py36h3dfced4_0|py35h3dfced4_0|py35h74e8950_0|py27h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py27h81de0dd_1|py37h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py27h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_1|py37hde5b4d6_1|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_1|py36h2f8d375_1|py37hde5b4d6_1|py37h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py36hdc34a94_3|py38h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py38hfa32c7d_0|py39h0f7b65f_0|py38h4c65ebe_1|py37h21a3de8_1|py38h21a3de8_1|py39h34387ca_0|py38h34387ca_0|py38hfae3a4d_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py37h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py39h2b8c604_0|py38h2b8c604_0|py310h2b8c604_0|py39hb8be1f0_1|py38hb8be1f0_1|py37hb8be1f0_1|py310hf2716ce_1|py39hb8be1f0_2|py37hb8be1f0_2|py38hb8be1f0_2|py310h9585f30_2|py37hf524024_2|py38hf524024_2|py37h1e6e340_3|py310h375b286_3|py38ha15fc14_3|py310h9585f30_0|py310hf2716ce_0|py38h1e6e340_0|py310h8e6c178_0|py39h1e6e340_0|py38h1e6e340_1|py39h1e6e340_1|py39h31eccc5_1|py38h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py310h375b286_0|py39h31eccc5_0|py310h8e6c178_1|py38h31eccc5_1|py310h375b286_1|py310h375b286_0|py38h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py38ha15fc14_0|py39ha15fc14_0|py310h375b286_0|py39h1e6e340_0|py38hb8be1f0_0|py39hb8be1f0_0|py38hf524024_0|py39hf524024_0|py39ha15fc14_3|py310hcba007f_3|py37ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py39hf524024_2|py310hf2716ce_2|py310h9585f30_1|py37hf524024_1|py38hf524024_1|py39hf524024_1|py310h79a1101_0|py37h79a1101_0|py37h2b8c604_0|py39h39b7dee_0|py38h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py37h74d4b33_0|py39hfae3a4d_0|py38he2ba247_0|py39h7d8b39e_0|py38h7d8b39e_0|py37h7d8b39e_0|py37h34387ca_0|py39h21a3de8_1|py39h4c65ebe_1|py37h4c65ebe_1|py39h2ae0177_0|py36hfa32c7d_0|py36h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py37hfa32c7d_0|py38h73d599e_4|py39h73d599e_4|py37h73d599e_4|py38ha8aedfd_4|py39ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py39hdc34a94_3|py38hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py38h41b4c56_3|py39h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py27h2f8d375_0|py38hde5b4d6_0|py38h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py37h2f8d375_1|py37hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_0|py37hde5b4d6_0|py36h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py37h81de0dd_1|py36h81de0dd_1|py27h2f8d375_1|py36h2f8d375_1|py36h81de0dd_0|py35h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py36h74e8950_0|py27h74e8950_0|py37h3dfced4_0|py35h7cdd4dd_0|py36h7cdd4dd_0|py36hde5b4d6_5|py27hde5b4d6_5|py37h2f8d375_5|py36h2f8d375_5|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py35h81de0dd_4|py27h2f8d375_4|py27h81de0dd_4|py35h2f8d375_4|py36hdbf6ddf_4|py36h2b20989_4|py27hdbf6ddf_4|py27h2b20989_4|py27hdbf6ddf_3|py37hdbf6ddf_3|py37h2b20989_3|py36h2b20989_3|py36hdbf6ddf_2|py27hdbf6ddf_2|py37h2b20989_2|py36h2b20989_2|py27h2b20989_2|py37hdbf6ddf_1|py36hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py36h2b20989_1|py35hdbf6ddf_0|py27h2b20989_0|py36h2b20989_0|py35hdbf6ddf_0|py27hdbf6ddf_0|py27h2b20989_0|py36h9be14a7_1|py27h0ea5e3f_1|py36h0ea5e3f_1|py38hde5b4d6_12|py36hde5b4d6_12|py37hde5b4d6_12|py36h2f8d375_12|py27hde5b4d6_11|py37hde5b4d6_11|py36hde5b4d6_11|py27h2f8d375_11|py37h2f8d375_11|py37h2f8d375_10|py35h81de0dd_10|py37h81de0dd_10|py35h74e8950_10|py37h74e8950_10|py35h81de0dd_9|py27h81de0dd_9|py37h74e8950_9|py36h81de0dd_9|py35h3dfced4_9|py36h3dfced4_9|py37h7cdd4dd_9|py35hdbf6ddf_8|py27h2b20989_8|py37hdbf6ddf_8|py36h2b20989_8|py37hdbf6ddf_7|py27hdbf6ddf_7|py36h2b20989_7|py37h2b20989_7|py35h2b20989_7|py35hdbf6ddf_7|py36hdbf6ddf_6|py37hdbf6ddf_6|py27h2b20989_6|py37h2f8d375_0']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py35h9be14a7_1|py36h2b20989_0|py35h2b20989_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py27h2b20989_1|py37h2b20989_2|py37h2b20989_3|py37hdbf6ddf_3|py27h2b20989_3|py27hdbf6ddf_3|py37h2b20989_4|py37hdbf6ddf_4|py35hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_4|py36h2f8d375_4|py38hde5b4d6_4|py27h2f8d375_5|py37hde5b4d6_5|py27h7cdd4dd_0|py37h7cdd4dd_0|py27h3dfced4_0|py36h3dfced4_0|py35h3dfced4_0|py35h74e8950_0|py27h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py27h81de0dd_1|py37h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py27h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_1|py37hde5b4d6_1|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_1|py36h2f8d375_1|py37hde5b4d6_1|py37h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py36hdc34a94_3|py38h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py38hfa32c7d_0|py39h0f7b65f_0|py38h4c65ebe_1|py37h21a3de8_1|py38h21a3de8_1|py39h34387ca_0|py38h34387ca_0|py38hfae3a4d_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py37h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py39h2b8c604_0|py38h2b8c604_0|py310h2b8c604_0|py39hb8be1f0_1|py38hb8be1f0_1|py37hb8be1f0_1|py310hf2716ce_1|py39hb8be1f0_2|py37hb8be1f0_2|py38hb8be1f0_2|py310h9585f30_2|py37hf524024_2|py38hf524024_2|py37h1e6e340_3|py310h375b286_3|py38ha15fc14_3|py310h9585f30_0|py310hf2716ce_0|py38h1e6e340_0|py310h8e6c178_0|py39h1e6e340_0|py38h1e6e340_1|py39h1e6e340_1|py39h31eccc5_1|py38h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py310h375b286_0|py39h31eccc5_0|py310h8e6c178_1|py38h31eccc5_1|py310h375b286_1|py310h375b286_0|py38h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py38ha15fc14_0|py39ha15fc14_0|py310h375b286_0|py39h1e6e340_0|py38hb8be1f0_0|py39hb8be1f0_0|py38hf524024_0|py39hf524024_0|py39ha15fc14_3|py310hcba007f_3|py37ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py39hf524024_2|py310hf2716ce_2|py310h9585f30_1|py37hf524024_1|py38hf524024_1|py39hf524024_1|py310h79a1101_0|py37h79a1101_0|py37h2b8c604_0|py39h39b7dee_0|py38h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py37h74d4b33_0|py39hfae3a4d_0|py38he2ba247_0|py39h7d8b39e_0|py38h7d8b39e_0|py37h7d8b39e_0|py37h34387ca_0|py39h21a3de8_1|py39h4c65ebe_1|py37h4c65ebe_1|py39h2ae0177_0|py36hfa32c7d_0|py36h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py37hfa32c7d_0|py38h73d599e_4|py39h73d599e_4|py37h73d599e_4|py38ha8aedfd_4|py39ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py39hdc34a94_3|py38hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py38h41b4c56_3|py39h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py27h2f8d375_0|py38hde5b4d6_0|py38h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py37h2f8d375_1|py37hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_0|py37hde5b4d6_0|py36h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py37h81de0dd_1|py36h81de0dd_1|py27h2f8d375_1|py36h2f8d375_1|py36h81de0dd_0|py35h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py36h74e8950_0|py27h74e8950_0|py37h3dfced4_0|py35h7cdd4dd_0|py36h7cdd4dd_0|py36hde5b4d6_5|py27hde5b4d6_5|py37h2f8d375_5|py36h2f8d375_5|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py35h81de0dd_4|py27h2f8d375_4|py27h81de0dd_4|py35h2f8d375_4|py37hde5b4d6_0|py36hdbf6ddf_4|py36h2b20989_4|py27hdbf6ddf_4|py27h2b20989_4|py36hdbf6ddf_3|py36h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27hdbf6ddf_2|py36h2b20989_2|py27h2b20989_2|py37hdbf6ddf_1|py36hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py36h2b20989_1|py27hdbf6ddf_0|py27h2b20989_0|py27h2b20989_0|py27h9be14a7_1|py36h9be14a7_1|py35h0ea5e3f_1|py27h0ea5e3f_1|py36h0ea5e3f_1']\n\
  matplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py36h2b20989_6|py37h2b20989_6|py27hdbf6ddf_6|py36hdbf6ddf_6|py27h2b20989_7|py36hdbf6ddf_7|py37h2b20989_7|py37h2b20989_7|py27h2b20989_7|py36h2b20989_7|py36hdbf6ddf_7|py36hdbf6ddf_8|py37hdbf6ddf_8|py37h2b20989_8|py27h2b20989_8|py35hdbf6ddf_8|py36h7cdd4dd_9|py35h7cdd4dd_9|py37h3dfced4_9|py35h3dfced4_9|py27h3dfced4_9|py36h81de0dd_9|py35h74e8950_9|py37h74e8950_9|py27h74e8950_10|py27h81de0dd_10|py36h81de0dd_10|py35h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py27h2f8d375_10|py35h2f8d375_10|py37h2f8d375_12|py27h2f8d375_12|py27hde5b4d6_12|py36hde5b4d6_12|py38h2f8d375_12|py35h9be14a7_1|py36h2b20989_0|py35h2b20989_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py27h2b20989_1|py37h2b20989_2|py37h2b20989_3|py37hdbf6ddf_3|py27h2b20989_3|py27hdbf6ddf_3|py37h2b20989_4|py37hdbf6ddf_4|py35hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_4|py36h2f8d375_4|py38hde5b4d6_4|py27h2f8d375_5|py37hde5b4d6_5|py27h7cdd4dd_0|py37h7cdd4dd_0|py27h3dfced4_0|py36h3dfced4_0|py35h3dfced4_0|py35h74e8950_0|py27h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py27h81de0dd_1|py37h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py27h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_1|py37hde5b4d6_1|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_1|py36h2f8d375_1|py37hde5b4d6_1|py37h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py36hdc34a94_3|py38h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py38hfa32c7d_0|py39h0f7b65f_0|py38h4c65ebe_1|py37h21a3de8_1|py38h21a3de8_1|py39h34387ca_0|py38h34387ca_0|py38hfae3a4d_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py37h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py39h2b8c604_0|py38h2b8c604_0|py310h2b8c604_0|py39hb8be1f0_1|py38hb8be1f0_1|py37hb8be1f0_1|py310hf2716ce_1|py39hb8be1f0_2|py37hb8be1f0_2|py38hb8be1f0_2|py310h9585f30_2|py37hf524024_2|py38hf524024_2|py37h1e6e340_3|py310h375b286_3|py38ha15fc14_3|py310h9585f30_0|py310hf2716ce_0|py38h1e6e340_0|py310h8e6c178_0|py39h1e6e340_0|py38h1e6e340_1|py39h1e6e340_1|py39h31eccc5_1|py38h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py310h375b286_0|py39h31eccc5_0|py310h8e6c178_1|py38h31eccc5_1|py310h375b286_1|py310h375b286_0|py38h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py38ha15fc14_0|py39ha15fc14_0|py310h375b286_0|py39h1e6e340_0|py38hb8be1f0_0|py39hb8be1f0_0|py38hf524024_0|py39hf524024_0|py39ha15fc14_3|py310hcba007f_3|py37ha15fc14_3|py39h1e6e340_3|py38h1e6e340_3|py39hf524024_2|py310hf2716ce_2|py310h9585f30_1|py37hf524024_1|py38hf524024_1|py39hf524024_1|py310h79a1101_0|py37h79a1101_0|py37h2b8c604_0|py39h39b7dee_0|py38h39b7dee_0|py38h74d4b33_0|py39h74d4b33_0|py37h74d4b33_0|py39hfae3a4d_0|py38he2ba247_0|py39h7d8b39e_0|py38h7d8b39e_0|py37h7d8b39e_0|py37h34387ca_0|py39h21a3de8_1|py39h4c65ebe_1|py37h4c65ebe_1|py39h2ae0177_0|py36hfa32c7d_0|py36h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py37hfa32c7d_0|py38h73d599e_4|py39h73d599e_4|py37h73d599e_4|py38ha8aedfd_4|py39ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py39hdc34a94_3|py38hdc34a94_3|py36h41b4c56_3|py37h41b4c56_3|py38h41b4c56_3|py39h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py27h2f8d375_0|py38hde5b4d6_0|py38h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_1|py36hde5b4d6_1|py37h2f8d375_1|py37hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_0|py37hde5b4d6_0|py36h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py37h81de0dd_1|py36h81de0dd_1|py27h2f8d375_1|py36h2f8d375_1|py36h81de0dd_0|py35h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py36h74e8950_0|py27h74e8950_0|py37h3dfced4_0|py35h7cdd4dd_0|py36h7cdd4dd_0|py36hde5b4d6_5|py27hde5b4d6_5|py37h2f8d375_5|py36h2f8d375_5|py38h2f8d375_4|py36h81de0dd_4|py37h81de0dd_4|py35h81de0dd_4|py27h2f8d375_4|py27h81de0dd_4|py35h2f8d375_4|py37hde5b4d6_0|py36hdbf6ddf_4|py36h2b20989_4|py27hdbf6ddf_4|py27h2b20989_4|py36hdbf6ddf_3|py36h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27hdbf6ddf_2|py36h2b20989_2|py27h2b20989_2|py37hdbf6ddf_1|py36hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py36h2b20989_1|py27hdbf6ddf_0|py27h2b20989_0|py27h2b20989_0|py27h9be14a7_1|py36h9be14a7_1|py35h0ea5e3f_1|py27h0ea5e3f_1|py36h0ea5e3f_1|py38hde5b4d6_12|py37hde5b4d6_12|py36h2f8d375_12|py27hde5b4d6_11|py37hde5b4d6_11|py36hde5b4d6_11|py36h2f8d375_11|py27h2f8d375_11|py37h2f8d375_11|py37h81de0dd_10|py36h74e8950_10|py35h74e8950_10|py37h74e8950_10|py35h81de0dd_9|py27h81de0dd_9|py36h74e8950_9|py27h74e8950_9|py37h81de0dd_9|py36h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py35h2b20989_8|py27hdbf6ddf_8|py36h2b20989_8|py37hdbf6ddf_7|py27hdbf6ddf_7|py37hdbf6ddf_7|py35h2b20989_7|py35hdbf6ddf_7|py36h2b20989_7|py27hdbf6ddf_7|py37hdbf6ddf_6|py27h2b20989_6']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - pandas=0.25 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n \
  \ - python=3.7 -> libgcc-ng[version='>=9.4.0'] -> __glibc[version='>=2.17']\n\n\
  Your installed version is: 2.31\n\nNote that strict channel priority may have removed\
  \ packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Ff9hpd0fl%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Ff9hpd0fl%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.187GiB</td>
						<td>1348.5 seconds</td>
						<td>1714.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738288048">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.2.1">v2.2.1</a>
						</td>
						<td>snakemake</td>
						<td>537 days</td>
						<td>2023-02-19 06:13:32</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/626zn3w6jd/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/e851377f-ec3b-41a4-a801-502292d3de9f\
  \ singularity  exec --home /home/azureuser/tmp/626zn3w6jd/code/.test  /home/azureuser/tmp/626zn3w6jd/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/626zn3w6jd/code/.test/.snakemake/conda/21e85eebdd757eb07fad6719618891f9_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/626zn3w6jd/code/.test/.snakemake/conda/21e85eebdd757eb07fad6719618891f9_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining pillow=6.2:   0%|          |\
  \ 0/9 [00:00<?, ?it/s]  \nExamining goatools=0.9.7:  11%|\u2588         | 1/9 [00:01<00:09,\
  \  1.22s/it]\nExamining goatools=0.9.7:  22%|\u2588\u2588\u258F       | 2/9 [00:01<00:04,\
  \  1.63it/s]\nExamining python=3.7:  22%|\u2588\u2588\u258F       | 2/9 [00:01<00:04,\
  \  1.63it/s]    \nExamining python=3.7:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [00:01<00:03,  1.58it/s]\nExamining @/linux-64::__linux==5.15.0=0:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:04<00:03,  1.58it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:04<00:07,  1.53s/it]\nExamining\
  \ matplotlib=3.1:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:04<00:07,  1.53s/it]\
  \               \nExamining pandas=0.25:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:09<00:06,  1.53s/it]   \nExamining pandas=0.25:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:09<00:05,  1.91s/it]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:12<00:05,  1.91s/it]\n\
  Examining @/linux-64::__archspec==1=x86_64:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:12<00:04,  2.28s/it]\nExamining @/linux-64::__unix==0=0:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:12<00:04,  2.28s/it]\
  \         \nExamining @/linux-64::__glibc==2.31=0:  89%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2589 | 8/9 [00:12<00:02,  2.28s/it]\n                \
  \                                                                    \n\nDetermining\
  \ conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for pandas\
  \ pillow matplotlib python:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for pandas pillow matplotlib goatools:  11%|\u2588         | 1/9 [00:07<00:56,\
  \  7.02s/it]\nExamining conflict for pandas pillow matplotlib goatools:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:07<00:24,  3.51s/it]\nExamining conflict for pillow\
  \ goatools python pandas matplotlib:  22%|\u2588\u2588\u258F       | 2/9 [00:21<00:24,\
  \  3.51s/it]\nExamining conflict for pillow goatools python pandas matplotlib: \
  \ 33%|\u2588\u2588\u2588\u258E      | 3/9 [00:21<00:47,  7.92s/it]\nExamining conflict\
  \ for pillow matplotlib:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:27<00:47,\
  \  7.92s/it]                       \nExamining conflict for pillow matplotlib: \
  \ 44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:27<00:36,  7.22s/it]\nExamining\
  \ conflict for pillow python pandas matplotlib __glibc:  44%|\u2588\u2588\u2588\u2588\
  \u258D     | 4/9 [00:31<00:36,  7.22s/it]\nExamining conflict for pillow python\
  \ pandas matplotlib __glibc:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9\
  \ [00:31<00:24,  6.10s/it]\nExamining conflict for pillow python:  56%|\u2588\u2588\
  \u2588\u2588\u2588\u258C    | 5/9 [00:36<00:24,  6.10s/it]                     \
  \     \nExamining conflict for pillow python:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:36<00:17,  5.82s/it]\nExamining conflict for pillow goatools\
  \ python:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:37<00:17,\
  \  5.82s/it]\nExamining conflict for pillow goatools python:  78%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258A  | 7/9 [00:37<00:08,  4.34s/it]\nExamining conflict\
  \ for pandas goatools matplotlib:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:40<00:08,  4.34s/it]\nExamining conflict for pandas goatools matplotlib:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:40<00:03,\
  \  3.84s/it]\nExamining conflict for goatools matplotlib:  89%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [01:16<00:03,  3.84s/it]       \nExamining\
  \ conflict for goatools matplotlib: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588| 9/9 [01:16<00:00, 13.80s/it]\nExamining conflict for goatools\
  \ pandas: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9\
  \ [01:24<00:00, 13.80s/it]    \nExamining conflict for goatools pandas: : 10it [01:24,\
  \ 12.09s/it]                    \nExamining conflict for pandas matplotlib python:\
  \ : 10it [01:26, 12.09s/it]\nExamining conflict for pandas matplotlib python: :\
  \ 11it [01:26,  9.15s/it]\nExamining conflict for pandas matplotlib: : 11it [01:27,\
  \  9.15s/it]       \nExamining conflict for pandas matplotlib: : 12it [01:27,  6.53s/it]\n\
  \                                                                   \nfailed\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining pillow=6.2:   0%|          |\
  \ 0/9 [00:00<?, ?it/s]  \nExamining goatools=0.9.7:  11%|\u2588         | 1/9 [00:00<00:05,\
  \  1.54it/s]\nExamining goatools=0.9.7:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:02,\
  \  3.09it/s]\nExamining python=3.7:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:02,\
  \  3.09it/s]    \nExamining python=3.7:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [00:00<00:01,  3.04it/s]\nExamining @/linux-64::__linux==5.15.0=0:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:02<00:01,  3.04it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:02<00:04,  1.08it/s]\nExamining\
  \ matplotlib=3.1:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:02<00:04,  1.08it/s]\
  \               \nExamining pandas=0.25:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:05<00:03,  1.08it/s]   \nExamining pandas=0.25:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:05<00:03,  1.11s/it]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:07<00:03,  1.11s/it]\n\
  Examining @/linux-64::__archspec==1=x86_64:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:07<00:02,  1.43s/it]\nExamining @/linux-64::__unix==0=0:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:07<00:02,  1.43s/it]\
  \         \nExamining @/linux-64::__glibc==2.31=0:  89%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2589 | 8/9 [00:07<00:01,  1.43s/it]\n                \
  \                                                                    \n\nDetermining\
  \ conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for pandas\
  \ pillow matplotlib python:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for pandas pillow matplotlib goatools:  11%|\u2588         | 1/9 [00:04<00:39,\
  \  4.88s/it]\nExamining conflict for pandas pillow matplotlib goatools:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:04<00:17,  2.44s/it]\nExamining conflict for pillow\
  \ goatools python pandas matplotlib:  22%|\u2588\u2588\u258F       | 2/9 [00:11<00:17,\
  \  2.44s/it]\nExamining conflict for pillow goatools python pandas matplotlib: \
  \ 33%|\u2588\u2588\u2588\u258E      | 3/9 [00:11<00:25,  4.27s/it]\nExamining conflict\
  \ for pillow matplotlib:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:14<00:25,\
  \  4.27s/it]                       \nExamining conflict for pillow matplotlib: \
  \ 44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:14<00:18,  3.68s/it]\nExamining\
  \ conflict for pillow python pandas matplotlib __glibc:  44%|\u2588\u2588\u2588\u2588\
  \u258D     | 4/9 [00:16<00:18,  3.68s/it]\nExamining conflict for pillow python\
  \ pandas matplotlib __glibc:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9\
  \ [00:16<00:12,  3.03s/it]\nExamining conflict for pillow python:  56%|\u2588\u2588\
  \u2588\u2588\u2588\u258C    | 5/9 [00:18<00:12,  3.03s/it]                     \
  \     \nExamining conflict for pillow python:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:18<00:08,  2.88s/it]\nExamining conflict for pillow goatools\
  \ python:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:19<00:08,\
  \  2.88s/it]\nExamining conflict for pillow goatools python:  78%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258A  | 7/9 [00:19<00:04,  2.22s/it]\nExamining conflict\
  \ for pandas goatools matplotlib:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258A  | 7/9 [00:21<00:04,  2.22s/it]\nExamining conflict for pandas goatools matplotlib:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:21<00:02,\
  \  2.12s/it]\nExamining conflict for goatools matplotlib:  89%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:44<00:02,  2.12s/it]       \nExamining\
  \ conflict for goatools matplotlib: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588| 9/9 [00:44<00:00,  8.74s/it]\nExamining conflict for goatools\
  \ pandas: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9\
  \ [00:50<00:00,  8.74s/it]    \nExamining conflict for goatools pandas: : 10it [00:50,\
  \  7.81s/it]                    \nExamining conflict for pandas matplotlib python:\
  \ : 10it [00:52,  7.81s/it]\nExamining conflict for pandas matplotlib python: :\
  \ 11it [00:52,  5.91s/it]\nExamining conflict for pandas matplotlib: : 11it [00:52,\
  \  5.91s/it]       \nExamining conflict for pandas matplotlib: : 12it [00:52,  4.21s/it]\n\
  \                                                                   \nfailed\n\n\
  UnsatisfiableError: The following specifications were found to be incompatible with\
  \ each other:\n\nOutput in format: Requested package -> Available versions\n\nPackage\
  \ python conflicts for:\npillow=6.2 -> olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  matplotlib=3.1 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  pillow=6.2 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  goatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  matplotlib=3.1 -> pyqt -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.6|>=3.0,<=3.9|>=3.7|>=3.8|>=3']\n\
  goatools=0.9.7 -> python\npython=3.7\n\nPackage _libgcc_mutex conflicts for:\npandas=0.25\
  \ -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='main|main|conda_forge']\n\
  matplotlib=3.1 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='main|main|conda_forge']\n\
  pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='main|main|conda_forge']\n\
  python=3.7 -> libgcc-ng[version='>=9.4.0'] -> _libgcc_mutex[version='*|0.1',build='main|main|conda_forge']\n\
  \nPackage ca-certificates conflicts for:\npython=3.7 -> openssl[version='>=1.1.1l,<1.1.2a']\
  \ -> ca-certificates\npillow=6.2 -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n\
  goatools=0.9.7 -> python -> ca-certificates\n\nPackage pandas conflicts for:\ngoatools=0.9.7\
  \ -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\npandas=0.25\n\
  \nPackage libpng conflicts for:\npillow=6.2 -> freetype[version='>=2.9.1,<3.0a0']\
  \ -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.37,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  matplotlib=3.1 -> matplotlib-base==3.1.1=py36he7580a8_1 -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  matplotlib=3.1 -> libpng[version='>=1.6.37,<1.7.0a0']\n\nPackage numpy-base conflicts\
  \ for:\nmatplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py27h2b20989_6|py37h2b20989_6|py37hdbf6ddf_6|py36hdbf6ddf_6|py27h2b20989_7|py36h2b20989_7|py35hdbf6ddf_7|py35h2b20989_7|py37h2b20989_7|py37h2b20989_7|py27h2b20989_7|py36h2b20989_7|py36hdbf6ddf_8|py36h2b20989_8|py37hdbf6ddf_8|py27hdbf6ddf_8|py35h2b20989_8|py27h7cdd4dd_9|py37h3dfced4_9|py36h3dfced4_9|py27h74e8950_9|py35h74e8950_9|py37h74e8950_9|py27h81de0dd_9|py27h74e8950_10|py37h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py37h2f8d375_11|py37h2f8d375_12|py36hde5b4d6_12|py38hde5b4d6_12|py38h2f8d375_12|py36h0ea5e3f_1|py35h0ea5e3f_1|py27h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py35h2b20989_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py27h2b20989_0|py35hdbf6ddf_0|py36h2b20989_1|py37hdbf6ddf_1|py27h2b20989_2|py36h2b20989_2|py36h2b20989_3|py37h2b20989_3|py36hdbf6ddf_3|py27hdbf6ddf_4|py36h2b20989_4|py36hdbf6ddf_4|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py35h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py37h81de0dd_4|py36h81de0dd_4|py38hde5b4d6_4|py37h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py36h3dfced4_0|py35h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py27h74e8950_0|py27h81de0dd_0|py37h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_1|py27h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_1|py37hde5b4d6_1|py36hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_1|py37hde5b4d6_1|py36hde5b4d6_1|py27hde5b4d6_1|py27h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py38h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py39h76555f2_1|py38h41b4c56_3|py37h41b4c56_3|py36hdc34a94_3|py39hdc34a94_3|py37hdc34a94_3|py37ha8aedfd_4|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py37hfa32c7d_0|py38hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py39h2ae0177_0|py39h4c65ebe_1|py37h21a3de8_1|py39h34387ca_0|py38h34387ca_0|py39h7d8b39e_0|py38hfae3a4d_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py39hfae3a4d_0|py39h74d4b33_0|py37h39b7dee_0|py39h39b7dee_0|py37h2b8c604_0|py39h2b8c604_0|py38hf524024_1|py37hf524024_1|py310hf2716ce_1|py310hf2716ce_2|py39hb8be1f0_2|py38hb8be1f0_2|py39h1e6e340_3|py310hcba007f_3|py39ha15fc14_3|py39hf524024_0|py310h9585f30_0|py38hf524024_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py310hcba007f_0|py38h31eccc5_0|py38h1e6e340_1|py310h375b286_1|py39h1e6e340_1|py38h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py310h375b286_0|py310h375b286_0|py38h31eccc5_0|py310h8e6c178_0|py39h31eccc5_0|py39h1e6e340_0|py38h1e6e340_0|py39h1e6e340_0|py39h31eccc5_0|py39h31eccc5_1|py310h8e6c178_1|py38h31eccc5_1|py310h375b286_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38ha15fc14_0|py39ha15fc14_0|py38hb8be1f0_0|py310hf2716ce_0|py39hb8be1f0_0|py38ha15fc14_3|py37ha15fc14_3|py310h375b286_3|py38h1e6e340_3|py37h1e6e340_3|py38hf524024_2|py37hf524024_2|py39hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py310h9585f30_1|py37hb8be1f0_1|py38hb8be1f0_1|py39hf524024_1|py39hb8be1f0_1|py310h79a1101_0|py310h2b8c604_0|py38h2b8c604_0|py37h79a1101_0|py39h79a1101_0|py38h79a1101_0|py38h39b7dee_0|py38h74d4b33_0|py37h74d4b33_0|py38he2ba247_0|py38h7d8b39e_0|py37h7d8b39e_0|py37h34387ca_0|py38h21a3de8_1|py39h21a3de8_1|py38h4c65ebe_1|py37h4c65ebe_1|py39h0f7b65f_0|py38h75fe3a5_0|py38h73d599e_4|py39ha8aedfd_4|py38hdc34a94_3|py36h41b4c56_3|py39h41b4c56_3|py39hfb011de_1|py38hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37h2f8d375_1|py27h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py36h81de0dd_0|py36h2f8d375_0|py36h81de0dd_0|py36h2f8d375_0|py36h81de0dd_1|py27h2f8d375_1|py36h81de0dd_0|py36h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py37h74e8950_0|py36h74e8950_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py36h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py36h2f8d375_5|py38h2f8d375_4|py35h81de0dd_4|py27h81de0dd_4|py37h2f8d375_4|py37hde5b4d6_0|py35h2b20989_4|py35hdbf6ddf_4|py37hdbf6ddf_4|py37h2b20989_4|py27h2b20989_4|py27hdbf6ddf_3|py27h2b20989_3|py37hdbf6ddf_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27hdbf6ddf_2|py37h2b20989_2|py27h2b20989_1|py36hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py27hdbf6ddf_0|py36hdbf6ddf_0|py36h2b20989_0|py36h2b20989_0|py36h9be14a7_1|py27h0ea5e3f_1|py37hde5b4d6_12|py27hde5b4d6_12|py36h2f8d375_12|py27h2f8d375_12|py27hde5b4d6_11|py37hde5b4d6_11|py36hde5b4d6_11|py36h2f8d375_11|py27h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py35h81de0dd_10|py36h81de0dd_10|py27h81de0dd_10|py36h74e8950_10|py35h74e8950_10|py37h74e8950_10|py35h81de0dd_9|py36h74e8950_9|py37h81de0dd_9|py36h81de0dd_9|py27h3dfced4_9|py35h3dfced4_9|py37h7cdd4dd_9|py35h7cdd4dd_9|py36h7cdd4dd_9|py35hdbf6ddf_8|py27h2b20989_8|py37h2b20989_8|py37hdbf6ddf_7|py36hdbf6ddf_7|py27hdbf6ddf_7|py37hdbf6ddf_7|py36hdbf6ddf_7|py27hdbf6ddf_7|py27hdbf6ddf_6|py36h2b20989_6']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py36h0ea5e3f_1|py35h0ea5e3f_1|py27h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py35h2b20989_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py27h2b20989_0|py35hdbf6ddf_0|py36h2b20989_1|py37hdbf6ddf_1|py27h2b20989_2|py36h2b20989_2|py36h2b20989_3|py37h2b20989_3|py36hdbf6ddf_3|py27hdbf6ddf_4|py36h2b20989_4|py36hdbf6ddf_4|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py35h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py37h81de0dd_4|py36h81de0dd_4|py38hde5b4d6_4|py37h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py36h3dfced4_0|py35h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py27h74e8950_0|py27h81de0dd_0|py37h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_1|py27h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_1|py37hde5b4d6_1|py36hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_1|py37hde5b4d6_1|py36hde5b4d6_1|py27hde5b4d6_1|py27h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py38h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py39h76555f2_1|py38h41b4c56_3|py37h41b4c56_3|py36hdc34a94_3|py39hdc34a94_3|py37hdc34a94_3|py37ha8aedfd_4|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py37hfa32c7d_0|py38hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py39h2ae0177_0|py39h4c65ebe_1|py37h21a3de8_1|py39h34387ca_0|py38h34387ca_0|py39h7d8b39e_0|py38hfae3a4d_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py39hfae3a4d_0|py39h74d4b33_0|py37h39b7dee_0|py39h39b7dee_0|py37h2b8c604_0|py39h2b8c604_0|py38hf524024_1|py37hf524024_1|py310hf2716ce_1|py310hf2716ce_2|py39hb8be1f0_2|py38hb8be1f0_2|py39h1e6e340_3|py310hcba007f_3|py39ha15fc14_3|py39hf524024_0|py310h9585f30_0|py38hf524024_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py310hcba007f_0|py38h31eccc5_0|py38h1e6e340_1|py310h375b286_1|py39h1e6e340_1|py38h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py310h375b286_0|py310h375b286_0|py38h31eccc5_0|py310h8e6c178_0|py39h31eccc5_0|py39h1e6e340_0|py38h1e6e340_0|py39h1e6e340_0|py39h31eccc5_0|py39h31eccc5_1|py310h8e6c178_1|py38h31eccc5_1|py310h375b286_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38ha15fc14_0|py39ha15fc14_0|py38hb8be1f0_0|py310hf2716ce_0|py39hb8be1f0_0|py38ha15fc14_3|py37ha15fc14_3|py310h375b286_3|py38h1e6e340_3|py37h1e6e340_3|py38hf524024_2|py37hf524024_2|py39hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py310h9585f30_1|py37hb8be1f0_1|py38hb8be1f0_1|py39hf524024_1|py39hb8be1f0_1|py310h79a1101_0|py310h2b8c604_0|py38h2b8c604_0|py37h79a1101_0|py39h79a1101_0|py38h79a1101_0|py38h39b7dee_0|py38h74d4b33_0|py37h74d4b33_0|py38he2ba247_0|py38h7d8b39e_0|py37h7d8b39e_0|py37h34387ca_0|py38h21a3de8_1|py39h21a3de8_1|py38h4c65ebe_1|py37h4c65ebe_1|py39h0f7b65f_0|py38h75fe3a5_0|py38h73d599e_4|py39ha8aedfd_4|py38hdc34a94_3|py36h41b4c56_3|py39h41b4c56_3|py39hfb011de_1|py38hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37h2f8d375_1|py27h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py36h81de0dd_0|py36h2f8d375_0|py36h81de0dd_0|py36h2f8d375_0|py36h81de0dd_1|py27h2f8d375_1|py36h81de0dd_0|py36h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py37h74e8950_0|py36h74e8950_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py36h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py36h2f8d375_5|py38h2f8d375_4|py35h81de0dd_4|py27h81de0dd_4|py37h2f8d375_4|py37hde5b4d6_0|py35h2b20989_4|py35hdbf6ddf_4|py37hdbf6ddf_4|py37h2b20989_4|py27h2b20989_4|py27hdbf6ddf_3|py27h2b20989_3|py37hdbf6ddf_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27hdbf6ddf_2|py37h2b20989_2|py27h2b20989_1|py36hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py27hdbf6ddf_0|py36hdbf6ddf_0|py36h2b20989_0|py36h2b20989_0|py36h9be14a7_1|py27h0ea5e3f_1']\n\
  goatools=0.9.7 -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py37h2f8d375_0|py37hde5b4d6_0|py27h2b20989_6|py37h2b20989_6|py37hdbf6ddf_6|py36hdbf6ddf_6|py27h2b20989_7|py35hdbf6ddf_7|py35h2b20989_7|py37h2b20989_7|py37h2b20989_7|py27h2b20989_7|py36hdbf6ddf_8|py36h2b20989_8|py37hdbf6ddf_8|py27h2b20989_8|py27hdbf6ddf_8|py35hdbf6ddf_8|py37h3dfced4_9|py36h3dfced4_9|py27h3dfced4_9|py27h74e8950_9|py35h74e8950_9|py36h74e8950_9|py27h81de0dd_9|py35h81de0dd_9|py37h74e8950_10|py27h74e8950_10|py36h74e8950_10|py35h81de0dd_10|py36h2f8d375_10|py27h2f8d375_10|py37hde5b4d6_11|py37h2f8d375_12|py36hde5b4d6_12|py38hde5b4d6_12|py36h0ea5e3f_1|py27h0ea5e3f_1|py36h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py35h2b20989_0|py27hdbf6ddf_0|py35hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_1|py27hdbf6ddf_1|py36hdbf6ddf_1|py37hdbf6ddf_1|py27h2b20989_1|py27h2b20989_2|py37h2b20989_2|py37h2b20989_3|py37hdbf6ddf_3|py36hdbf6ddf_3|py27h2b20989_3|py27hdbf6ddf_4|py36h2b20989_4|py36hdbf6ddf_4|py35h2b20989_4|py35h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py37h81de0dd_4|py36h81de0dd_4|py38hde5b4d6_4|py37h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py36h3dfced4_0|py35h3dfced4_0|py37h3dfced4_0|py35h74e8950_0|py27h74e8950_0|py27h81de0dd_0|py37h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_1|py27h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py37h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_1|py37hde5b4d6_1|py36hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_1|py37hde5b4d6_1|py36hde5b4d6_1|py27hde5b4d6_1|py27h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py27hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py38h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py39h76555f2_1|py38h41b4c56_3|py37h41b4c56_3|py36hdc34a94_3|py39hdc34a94_3|py37hdc34a94_3|py37ha8aedfd_4|py38ha8aedfd_4|py37h73d599e_4|py39h73d599e_4|py37hfa32c7d_0|py38hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py37hfa32c7d_0|py38h75fe3a5_0|py37h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py38hfa32c7d_0|py39h2ae0177_0|py39h4c65ebe_1|py37h21a3de8_1|py39h34387ca_0|py38h34387ca_0|py39h7d8b39e_0|py38hfae3a4d_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py39hfae3a4d_0|py39h74d4b33_0|py37h39b7dee_0|py39h39b7dee_0|py37h2b8c604_0|py39h2b8c604_0|py38hf524024_1|py37hf524024_1|py310hf2716ce_1|py310hf2716ce_2|py39hb8be1f0_2|py38hb8be1f0_2|py39h1e6e340_3|py310hcba007f_3|py39ha15fc14_3|py39hf524024_0|py310h9585f30_0|py38hf524024_0|py39h1e6e340_0|py38h1e6e340_0|py310h375b286_0|py310hcba007f_0|py38h31eccc5_0|py38h1e6e340_1|py310h375b286_1|py39h1e6e340_1|py38h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py310h375b286_0|py310h375b286_0|py38h31eccc5_0|py310h8e6c178_0|py39h31eccc5_0|py39h1e6e340_0|py38h1e6e340_0|py39h1e6e340_0|py39h31eccc5_0|py39h31eccc5_1|py310h8e6c178_1|py38h31eccc5_1|py310h375b286_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38ha15fc14_0|py39ha15fc14_0|py38hb8be1f0_0|py310hf2716ce_0|py39hb8be1f0_0|py38ha15fc14_3|py37ha15fc14_3|py310h375b286_3|py38h1e6e340_3|py37h1e6e340_3|py38hf524024_2|py37hf524024_2|py39hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py310h9585f30_1|py37hb8be1f0_1|py38hb8be1f0_1|py39hf524024_1|py39hb8be1f0_1|py310h79a1101_0|py310h2b8c604_0|py38h2b8c604_0|py37h79a1101_0|py39h79a1101_0|py38h79a1101_0|py38h39b7dee_0|py38h74d4b33_0|py37h74d4b33_0|py38he2ba247_0|py38h7d8b39e_0|py37h7d8b39e_0|py37h34387ca_0|py38h21a3de8_1|py39h21a3de8_1|py38h4c65ebe_1|py37h4c65ebe_1|py39h0f7b65f_0|py38h75fe3a5_0|py38h73d599e_4|py39ha8aedfd_4|py38hdc34a94_3|py36h41b4c56_3|py39h41b4c56_3|py39hfb011de_1|py38hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37h2f8d375_1|py27h2f8d375_1|py37hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py36hde5b4d6_0|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py36h81de0dd_0|py36h2f8d375_0|py36h81de0dd_0|py36h2f8d375_0|py36h81de0dd_1|py27h2f8d375_1|py36h81de0dd_0|py36h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py37h74e8950_0|py36h74e8950_0|py27h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py36h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py36h2f8d375_5|py38h2f8d375_4|py35h81de0dd_4|py27h81de0dd_4|py37h2f8d375_4|py35hdbf6ddf_4|py37hdbf6ddf_4|py37h2b20989_4|py27h2b20989_4|py27hdbf6ddf_3|py36h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27hdbf6ddf_2|py36h2b20989_2|py37h2b20989_1|py27hdbf6ddf_0|py36hdbf6ddf_0|py27h2b20989_0|py36h2b20989_0|py36hdbf6ddf_0|py36h2b20989_0|py27h9be14a7_1|py35h0ea5e3f_1|py38h2f8d375_12|py37hde5b4d6_12|py27hde5b4d6_12|py36h2f8d375_12|py27h2f8d375_12|py27hde5b4d6_11|py36hde5b4d6_11|py36h2f8d375_11|py27h2f8d375_11|py37h2f8d375_11|py35h2f8d375_10|py37h2f8d375_10|py36h81de0dd_10|py37h81de0dd_10|py27h81de0dd_10|py35h74e8950_10|py37h74e8950_9|py37h81de0dd_9|py36h81de0dd_9|py35h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py35h7cdd4dd_9|py36h7cdd4dd_9|py35h2b20989_8|py37h2b20989_8|py37hdbf6ddf_7|py36hdbf6ddf_7|py27hdbf6ddf_7|py36h2b20989_7|py37hdbf6ddf_7|py36h2b20989_7|py36hdbf6ddf_7|py27hdbf6ddf_7|py27hdbf6ddf_6|py36h2b20989_6|py36hde5b4d6_0|py36h2f8d375_0']\n\
  \nPackage setuptools conflicts for:\nmatplotlib=3.1 -> setuptools\npandas=0.25 ->\
  \ setuptools[version='<60.0.0']\npython=3.7 -> pip -> setuptools\n\nPackage graalpy\
  \ conflicts for:\npillow=6.2 -> python[version='>=3.8,<3.9.0a0'] -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\
  matplotlib=3.1 -> python[version='>=3.8,<3.9.0a0'] -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\
  pandas=0.25 -> python[version='>=3.8,<3.9.0a0'] -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\
  goatools=0.9.7 -> python -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\nPackage expat\
  \ conflicts for:\npython=3.7 -> pypy3.7=7.3.7 -> expat[version='>=2.2.9,<3.0.0a0|>=2.3.0,<3.0a0|>=2.4.1,<3.0a0']\n\
  pillow=6.2 -> pypy3.6[version='>=7.3.1'] -> expat[version='>=2.2.9,<3.0.0a0']\n\n\
  Package libgcc conflicts for:\ngoatools=0.9.7 -> scipy -> libgcc\nmatplotlib=3.1\
  \ -> pyqt -> libgcc\n\nPackage python-dateutil conflicts for:\nmatplotlib=3.1 ->\
  \ python-dateutil\npandas=0.25 -> python-dateutil[version='>=2.6.1']The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - matplotlib=3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> __glibc[version='>=2.17']\n  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] ->\
  \ __glibc[version='>=2.17']\n  - python=3.7 -> libgcc-ng[version='>=9.4.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fz5j87huo%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fz5j87huo%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.222GiB</td>
						<td>1343.7 seconds</td>
						<td>2141.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738264880">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>536 days</td>
						<td>2023-02-19 06:44:31</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/otf981ylet/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fpia9l2tb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fpia9l2tb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>1.9 seconds</td>
						<td>3.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738264640">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.1">v0.1.1</a>
						</td>
						<td>snakemake</td>
						<td>536 days</td>
						<td>2023-02-19 06:45:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/xbvf7kn17s/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F3b2j6e3k%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F3b2j6e3k%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>1.9 seconds</td>
						<td>2.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743141792">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>534 days</td>
						<td>2023-02-19 04:31:33</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Flwz82ulv%2Flogs.tar.xz">0.110GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Flwz82ulv%2Foutput.tar.xz">0.004GiB</a>
						</td>
						<td>1.002GiB</td>
						<td>2045.2 seconds</td>
						<td>851.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738301248">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.0.0">v3.0.0</a>
						</td>
						<td>snakemake</td>
						<td>534 days</td>
						<td>2023-02-19 06:23:11</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/l03aacjtzj/singularity-tmp/tmpcux8cx41.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/8cba9c87-efef-4d7f-bfbc-d63199027f63\
  \ singularity  exec --home /home/azureuser/tmp/l03aacjtzj/code/.test  /home/azureuser/tmp/l03aacjtzj/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/l03aacjtzj/code/.test/.snakemake/conda/1f77d6ba3caf35d7cfacbd6a7703fda3_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/l03aacjtzj/code/.test/.snakemake/conda/1f77d6ba3caf35d7cfacbd6a7703fda3_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  11%|\u2588         | 1/9 [00:00<00:00, 4905.62it/s]\nExamining bedtools[version='>=2.29']:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 6004.73it/s]   \nExamining bcftools=1.11:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 17.06it/s]            \
  \   \nExamining bcftools=1.11:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,\
  \ 22.73it/s]\nExamining freebayes=1.3.1:  44%|\u2588\u2588\u2588\u2588\u258D   \
  \  | 4/9 [00:00<00:00, 22.73it/s]\nExamining @/linux-64::__unix==0=0:  56%|\u2588\
  \u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 22.73it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 22.73it/s]\n\
  Examining @/linux-64::__archspec==1=x86_64:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:00<00:00, 24.59it/s]\nExamining parallel=20190522:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:00<00:00, 24.59it/s]     \
  \          \nExamining sed=4.7:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:00<00:00, 24.59it/s]          \n                              \
  \                                  \n\nDetermining conflicts:   0%|          | 0/9\
  \ [00:00<?, ?it/s]\nExamining conflict for __glibc bedtools freebayes bcftools sed:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for freebayes bedtools\
  \ bcftools:  11%|\u2588         | 1/9 [00:00<00:02,  3.74it/s]    \nExamining conflict\
  \ for freebayes bedtools bcftools:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \  7.48it/s]\nExamining conflict for freebayes sed bedtools bcftools:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:01<00:00,  7.48it/s]\nExamining conflict for freebayes\
  \ sed bedtools bcftools:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:01<00:04,\
  \  1.30it/s]\nExamining conflict for bedtools freebayes bcftools parallel sed: \
  \ 33%|\u2588\u2588\u2588\u258E      | 3/9 [00:02<00:04,  1.30it/s]\nExamining conflict\
  \ for bedtools freebayes bcftools parallel sed:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [00:02<00:03,  1.47it/s]\nExamining conflict for bedtools freebayes:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:02<00:03,  1.47it/s]       \
  \               \nExamining conflict for bedtools freebayes:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:02<00:02,  1.85it/s]\nExamining conflict for parallel\
  \ bcftools:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:02,  1.85it/s]\
  \ \nExamining conflict for parallel bcftools:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:03<00:01,  2.16it/s]\nExamining conflict for parallel freebayes\
  \ bcftools:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:03<00:01,\
  \  2.16it/s]\nExamining conflict for freebayes bcftools:  78%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258A  | 7/9 [00:03<00:00,  2.16it/s]         \nExamining\
  \ conflict for freebayes bcftools:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2589 | 8/9 [00:03<00:00,  3.44it/s]\nExamining conflict for parallel freebayes:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:03<00:00,\
  \  3.44it/s]\n                                                                 \
  \                       \nfailed\nSolving environment: ...working... \nFound conflicts!\
  \ Looking for incompatible packages.\nThis can take several minutes.  Press CTRL-C\
  \ to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ @/linux-64::__glibc==2.31=0:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ @/linux-64::__linux==5.15.0=0:  11%|\u2588         | 1/9 [00:00<00:00, 82.27it/s]\n\
  Examining bedtools[version='>=2.29']:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \ 162.38it/s]  \nExamining bcftools=1.11:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [00:00<00:00, 16.78it/s]              \nExamining bcftools=1.11:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:00<00:00, 22.35it/s]\nExamining freebayes=1.3.1:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00, 22.35it/s]\nExamining\
  \ @/linux-64::__unix==0=0:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00,\
  \ 22.35it/s]\nExamining @/linux-64::__archspec==1=x86_64:  67%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 22.35it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:00<00:00, 14.98it/s]\n\
  Examining parallel=20190522:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [00:00<00:00, 14.98it/s]               \nExamining sed=4.7:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00, 14.98it/s]\
  \          \n                                                                \n\n\
  Determining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for __glibc bedtools freebayes bcftools sed:   0%|          | 0/9 [00:00<?, ?it/s]\n\
  Examining conflict for freebayes bedtools bcftools:  11%|\u2588         | 1/9 [00:00<00:03,\
  \  2.65it/s]    \nExamining conflict for freebayes bedtools bcftools:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:00<00:01,  5.29it/s]\nExamining conflict for freebayes\
  \ sed bedtools bcftools:  22%|\u2588\u2588\u258F       | 2/9 [00:02<00:01,  5.29it/s]\n\
  Examining conflict for freebayes sed bedtools bcftools:  33%|\u2588\u2588\u2588\u258E\
  \      | 3/9 [00:02<00:06,  1.13s/it]\nExamining conflict for bedtools freebayes\
  \ bcftools parallel sed:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:03<00:06,\
  \  1.13s/it]\nExamining conflict for bedtools freebayes bcftools parallel sed: \
  \ 44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:03<00:04,  1.10it/s]\nExamining\
  \ conflict for bedtools freebayes:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9\
  \ [00:03<00:04,  1.10it/s]                      \nExamining conflict for bedtools\
  \ freebayes:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:03<00:02,  1.43it/s]\n\
  Examining conflict for parallel bcftools:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:03<00:02,  1.43it/s] \nExamining conflict for parallel bcftools:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:03<00:01,  1.77it/s]\n\
  Examining conflict for parallel freebayes bcftools:  67%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258B   | 6/9 [00:03<00:01,  1.77it/s]\nExamining conflict for freebayes\
  \ bcftools:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:04<00:01,\
  \  1.77it/s]         \nExamining conflict for freebayes bcftools:  89%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:04<00:00,  2.68it/s]\nExamining\
  \ conflict for parallel freebayes:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2589 | 8/9 [00:04<00:00,  2.68it/s]\n                                  \
  \                                                      \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with each other:\n\n\
  Output in format: Requested package -> Available versions\n\nPackage xz conflicts\
  \ for:\nfreebayes=1.3.1 -> xz[version='>=5.2.4,<5.3.0a0']\nfreebayes=1.3.1 -> htslib\
  \ -> xz[version='5.2.*|>=5.2.3,<5.3.0a0|>=5.2.5,<5.3.0a0|>=5.2.6,<5.3.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0|>=5.2.10,<6.0a0|>=5.2.6,<6.0a0']\n\
  \nPackage libgcc-ng conflicts for:\nfreebayes=1.3.1 -> libgcc-ng[version='>=7.3.0']\n\
  bedtools[version='>=2.29'] -> libgcc-ng[version='>=10.3.0|>=9.4.0|>=9.3.0|>=7.5.0|>=7.3.0']\n\
  parallel=20190522 -> perl -> libgcc-ng[version='>=11.2.0|>=4.9|>=7.3.0|>=7.5.0|>=9.3.0|>=9.4.0|>=7.2.0']\n\
  bedtools[version='>=2.29'] -> libzlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=11.2.0|>=12|>=4.9|>=7.2.0']\n\
  freebayes=1.3.1 -> bzip2[version='>=1.0.6,<2.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=4.9|>=7.5.0|>=9.3.0|>=7.2.0|>=9.4.0|>=11.2.0']\n\
  sed=4.7 -> libgcc-ng[version='>=7.3.0']\nbcftools=1.11 -> libgcc-ng[version='>=7.5.0']\n\
  bcftools=1.11 -> gsl[version='>=2.6,<2.7.0a0'] -> libgcc-ng[version='>=10.3.0|>=11.2.0|>=4.9|>=7.3.0|>=9.3.0|>=9.4.0|>=7.2.0|>=12']\n\
  \nPackage libnsl conflicts for:\nparallel=20190522 -> perl -> libnsl[version='>=2.0.0,<2.1.0a0']\n\
  bcftools=1.11 -> perl -> libnsl[version='>=2.0.0,<2.1.0a0']\nfreebayes=1.3.1 ->\
  \ python[version='>=3.6,<3.7.0a0'] -> libnsl[version='>=2.0.0,<2.1.0a0']\n\nPackage\
  \ gdbm conflicts for:\nbcftools=1.11 -> perl -> gdbm[version='>=1.18|>=1.18,<1.19.0a0']\n\
  parallel=20190522 -> perl -> gdbm[version='>=1.18|>=1.18,<1.19.0a0']\n\nPackage\
  \ parallel conflicts for:\nparallel=20190522\nfreebayes=1.3.1 -> parallel\n\nPackage\
  \ bzip2 conflicts for:\nfreebayes=1.3.1 -> htslib -> bzip2[version='1.0.*|>=1.0.8,<2.0a0']\n\
  freebayes=1.3.1 -> bzip2[version='>=1.0.6,<2.0a0']\n\nPackage curl conflicts for:\n\
  freebayes=1.3.1 -> htslib -> curl[version='>=7.59.0,<8.0a0|>=7.64.0,<8.0a0|>=7.64.1,<8.0a0|>=7.81.0,<8.0a0|>=7.75.0,<8.0a0|>=7.71.1,<8.0a0|>=7.78.0,<8.0a0']\n\
  bedtools[version='>=2.29'] -> curl[version='>=7.64.1,<8.0a0']\n\nPackage libcurl\
  \ conflicts for:\nfreebayes=1.3.1 -> htslib -> libcurl[version='>=7.64.1,<8.0a0|>=7.71.1,<8.0a0|>=7.75.0,<8.0a0|>=7.77.0,<8.0a0|>=7.79.1,<8.0a0|>=7.81.0,<8.0a0|>=7.82.0,<8.0a0|>=7.83.1,<8.0a0']\n\
  bcftools=1.11 -> htslib[version='>=1.11,<1.12.0a0'] -> libcurl[version='>=7.71.1,<8.0a0']\n\
  bedtools[version='>=2.29'] -> curl[version='>=7.64.1,<8.0a0'] -> libcurl[version='7.64.1|7.64.1|7.64.1|7.65.2|7.65.3|7.68.0|7.68.0|7.69.1|7.71.0|7.71.1|7.71.1|7.71.1|7.71.1|7.71.1|7.71.1|7.71.1|7.71.1|7.75.0|7.76.0|7.76.1|7.76.1|7.76.1|7.77.0|7.78.0|7.79.0|7.79.1|7.79.1|7.79.1|7.80.0|7.80.0|7.80.0|7.81.0|7.81.0|7.82.0|7.82.0|7.83.0|7.83.0|7.83.1|7.83.1|7.85.0|7.85.0|7.86.0|7.86.0|7.86.0|7.86.0|7.86.0|7.86.0|7.87.0|7.87.0|7.88.0|7.87.0|7.86.0|7.85.0|7.84.0|7.83.0|7.82.0|7.80.0|7.78.0|7.71.1|7.71.1|7.71.0|7.69.1|7.68.0|7.67.0|7.65.3|7.65.2',build='h20c2e04_0|h20c2e04_0|h20c2e04_0|h20c2e04_0|h20c2e04_0|h91b91d3_0|hf7181ac_0|hcdd3856_1|hc4aaa36_1|h2574ce0_0|h2574ce0_0|h2574ce0_0|h2574ce0_0|h2574ce0_1|h494985f_0|h2574ce0_0|h494985f_1|h494985f_0|h2574ce0_0|h7bff187_0|h7bff187_1|h6312ad2_0|hdc1c0ab_0|hdc1c0ab_0|hdc1c0ab_2|h6312ad2_2|h2283fc2_1|h7bff187_0|h2283fc2_0|h2283fc2_0|h7bff187_0|h2283fc2_0|h7bff187_0|h2283fc2_0|h2283fc2_0|h7bff187_0|h494985f_1|h2574ce0_2|hc4aaa36_0|hc4aaa36_0|hc4aaa36_0|hcdd3856_8|hcdd3856_6|hcdd3856_5|hcdd3856_4|hcdd3856_3|hcdd3856_2|hcdd3856_0|hcdd3856_0|hf7181ac_1|hda55be3_0|hda55be3_0|hda55be3_0|hf7181ac_1|hda55be3_0|h91b91d3_0|h91b91d3_0|h91b91d3_0|h0b77cf5_0|h0b77cf5_0|h0b77cf5_0|h0b77cf5_0|h303737a_2|h20c2e04_1|h20c2e04_0|h20c2e04_0']\n\
  \nPackage _openmp_mutex conflicts for:\nsed=4.7 -> libgcc-ng[version='>=7.3.0']\
  \ -> _openmp_mutex[version='>=4.5']\nbedtools[version='>=2.29'] -> libgcc-ng[version='>=10.3.0']\
  \ -> _openmp_mutex[version='>=4.5']\nfreebayes=1.3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> _openmp_mutex[version='>=4.5']\nbcftools=1.11 -> libgcc-ng[version='>=7.5.0']\
  \ -> _openmp_mutex[version='>=4.5']\n\nPackage htslib conflicts for:\nbcftools=1.11\
  \ -> htslib[version='>=1.11,<1.12.0a0']\nfreebayes=1.3.1 -> htslib\n\nPackage zlib\
  \ conflicts for:\nfreebayes=1.3.1 -> htslib -> zlib[version='1.2.*|1.2.11|1.2.11.*|>=1.2.12,<1.3.0a0|1.2.8|>=1.2.13,<1.3.0a0']\n\
  freebayes=1.3.1 -> zlib[version='>=1.2.11,<1.3.0a0']The following specifications\
  \ were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - bcftools=1.11 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \  - bedtools[version='>=2.29'] -> libgcc-ng[version='>=10.3.0'] -> __glibc[version='>=2.17']\n\
  \  - freebayes=1.3.1 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - sed=4.7 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\nYour\
  \ installed version is: 2.31\n\nNote that strict channel priority may have removed\
  \ packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Flf5ekptw%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Flf5ekptw%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.288GiB</td>
						<td>760.9 seconds</td>
						<td>1175.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738211408">
								<a href="https://github.com/snakemake-workflows/species-quantification">snakemake-workflows/species-quantification</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/species-quantification/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>531 days</td>
						<td>2023-02-19 07:00:28</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- abundances.smk
kind: WorkflowError
line_no: 20
rest: "Rule is marked for between workflow caching but has multiple output files.\
  \ This is only allowed if multiext() is used to declare them (see docs on between\
  \ workflow caching).\n  File \"/home/azureuser/tmp/hmjwmucqpl/code/workflow/Snakefile\"\
  , line 17, in <module>\n  File \"/home/azureuser/tmp/hmjwmucqpl/code/workflow/rules/abundances.smk\"\
  , line 20, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fspecies-quantification%2F0llallii%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fspecies-quantification%2F0llallii%2Foutput.tar.xz">0.030GiB</a>
						</td>
						<td>0.099GiB</td>
						<td>1.8 seconds</td>
						<td>2.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738155728">
								<a href="https://github.com/huzuner/Tumor-microbiome-calling">huzuner/Tumor-microbiome-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/huzuner/Tumor-microbiome-calling/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>531 days</td>
						<td>2023-02-19 06:45:55</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- abundances.smk
kind: WorkflowError
line_no: 20
rest: "Rule is marked for between workflow caching but has multiple output files.\
  \ This is only allowed if multiext() is used to declare them (see docs on between\
  \ workflow caching).\n  File \"/home/azureuser/tmp/tsnjjwroa4/code/workflow/Snakefile\"\
  , line 17, in <module>\n  File \"/home/azureuser/tmp/tsnjjwroa4/code/workflow/rules/abundances.smk\"\
  , line 20, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/huzuner%252FTumor-microbiome-calling%2Fg9o3d6de%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/huzuner%252FTumor-microbiome-calling%2Fg9o3d6de%2Foutput.tar.xz">0.030GiB</a>
						</td>
						<td>0.099GiB</td>
						<td>1.9 seconds</td>
						<td>2.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738264400">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.2">v0.1.2</a>
						</td>
						<td>snakemake</td>
						<td>530 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/o8xo0i2ma0/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fqwst25xm%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fqwst25xm%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>2.0 seconds</td>
						<td>3.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739033728">
								<a href="https://github.com/mrvollger/Rhodonite">mrvollger/Rhodonite</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/Rhodonite/tree/v0.1-alpha">v0.1-alpha</a>
						</td>
						<td>snakemake</td>
						<td>525 days</td>
						<td>2023-02-19 03:31:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: SystemExit
line_no: 38
rest: "1\n  File \"/home/azureuser/tmp/jcv4fj3xzx/code/workflow/Snakefile\", line\
  \ 38, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Ffbwcig02%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Ffbwcig02%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>2.0 seconds</td>
						<td>2.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746658192">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>524 days</td>
						<td>2023-02-20 05:09:00</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.NullPointerException
msg: Cannot invoke method getAt() on null object
rest: "at org.codehaus.groovy.runtime.NullObject.invokeMethod(NullObject.java:91)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:44)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.NullCallSite.call(NullCallSite.java:34)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_893dbdf9$_runScript_closure4$_closure6$_closure20.doCall(Script_893dbdf9:314)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:38)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:53)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat nextflow.extension.DataflowHelper$_subscribeImpl_closure2.doCall(DataflowHelper.groovy:285)\n\
  \tat jdk.internal.reflect.GeneratedMethodAccessor179.invoke(Unknown Source)\n\t\
  at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.onMessage(DataflowOperatorActor.java:108)\n\
  \tat groovyx.gpars.actor.impl.SDAClosure$1.call(SDAClosure.java:43)\n\tat groovyx.gpars.actor.AbstractLoopingActor.runEnhancedWithoutRepliesOnMessages(AbstractLoopingActor.java:293)\n\
  \tat groovyx.gpars.actor.AbstractLoopingActor.access$400(AbstractLoopingActor.java:30)\n\
  \tat groovyx.gpars.actor.AbstractLoopingActor$1.handleMessage(AbstractLoopingActor.java:93)\n\
  \tat groovyx.gpars.util.AsyncMessagingCore.run(AsyncMessagingCore.java:132)\n\t\
  at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-20 08:09:00.788 [main]\
  \ DEBUG nextflow.trace.ReportObserver - Execution report summary data:\n  [{\"cpuUsage\"\
  :{\"mean\":62.75,\"min\":62.75,\"q1\":62.75,\"q2\":62.75,\"q3\":62.75,\"max\":62.75,\"\
  minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"maxLabel\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"},\"process\"\
  :\"METADATA_ALL\",\"mem\":{\"mean\":4722688,\"min\":4722688,\"q1\":4722688,\"q2\"\
  :4722688,\"q3\":4722688,\"max\":4722688,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\"},\"memUsage\":{\"mean\":0.07,\"min\":0.07,\"q1\":0.07,\"q2\"\
  :0.07,\"q3\":0.07,\"max\":0.07,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\"},\"timeUsage\":null,\"vmem\":{\"mean\":58093568,\"min\":58093568,\"\
  q1\":58093568,\"q2\":58093568,\"q3\":58093568,\"max\":58093568,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\"},\"reads\":{\"mean\":6002297,\"min\":6002297,\"q1\":6002297,\"\
  q2\":6002297,\"q3\":6002297,\"max\":6002297,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\"},\"cpu\":{\"mean\":125.5,\"min\":125.5,\"q1\":125.5,\"q2\":125.5,\"\
  q3\":125.5,\"max\":125.5,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q1Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  },\"time\":null,\"writes\":{\"mean\":513,\"min\":513,\"q1\":513,\"q2\":513,\"q3\"\
  :513,\"max\":513,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q1Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  }},{\"cpuUsage\":{\"mean\":56.55,\"min\":56.55,\"q1\":56.55,\"q2\":56.55,\"q3\"\
  :56.55,\"max\":56.55,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q1Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  },\"process\":\"METADATA_PAIRWISE\",\"mem\":{\"mean\":4952064,\"min\":4952064,\"\
  q1\":4952064,\"q2\":4952064,\"q3\":4952064,\"max\":4952064,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"memUsage\":{\"mean\":0.08,\"min\":0.08,\"q1\":0.08,\"q2\"\
  :0.08,\"q3\":0.08,\"max\":0.08,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"timeUsage\":null,\"vmem\":{\"mean\":86671360,\"min\":86671360,\"\
  q1\":86671360,\"q2\":86671360,\"q3\":86671360,\"max\":86671360,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"reads\":{\"mean\":6002435,\"min\":6002435,\"q1\":6002435,\"\
  q2\":6002435,\"q3\":6002435,\"max\":6002435,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"cpu\":{\"mean\":113.1,\"min\":113.1,\"q1\":113.1,\"q2\":113.1,\"\
  q3\":113.1,\"max\":113.1,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"time\":null,\"writes\":{\"mean\":514,\"min\":514,\"q1\":514,\"\
  q2\":514,\"q3\":514,\"max\":514,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"}}]\nFeb-20 08:09:01.155 [main] DEBUG nextflow.trace.TimelineObserver\
  \ - Workflow completed -- rendering execution timeline\nFeb-20 08:09:01.318 [main]\
  \ DEBUG nextflow.cache.CacheDB - Closing CacheDB done\nFeb-20 08:09:01.337 [main]\
  \ DEBUG nextflow.script.ScriptRunner - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2F24iwu01o%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2F24iwu01o%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.177GiB</td>
						<td>70.5 seconds</td>
						<td>10800.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738264160">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.3">v0.1.3</a>
						</td>
						<td>snakemake</td>
						<td>524 days</td>
						<td>2023-02-19 04:36:19</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/bt6s5b2p45/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F392xk9pc%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F392xk9pc%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>2.0 seconds</td>
						<td>4.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743104704">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.4.0">2.4.0</a>
						</td>
						<td>nextflow</td>
						<td>522 days</td>
						<td>2023-02-19 03:31:08</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fhv3f6ubs%2Flogs.tar.xz">0.015GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fhv3f6ubs%2Foutput.tar.xz">0.012GiB</a>
						</td>
						<td>1.690GiB</td>
						<td>700.1 seconds</td>
						<td>453.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738263920">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.4">v0.1.4</a>
						</td>
						<td>snakemake</td>
						<td>522 days</td>
						<td>2023-02-19 06:50:36</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/u386z68jcz/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Ftvytce41%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Ftvytce41%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>1.4 seconds</td>
						<td>2.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738263680">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.5">v0.1.5</a>
						</td>
						<td>snakemake</td>
						<td>522 days</td>
						<td>2023-02-19 04:36:42</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/yt1622tx34/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Ffsybgn0z%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Ffsybgn0z%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>1.8 seconds</td>
						<td>1.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743111664">
								<a href="https://nf-co.re/fetchngs">fetchngs</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/fetchngs/tree/1.3">1.3</a>
						</td>
						<td>nextflow</td>
						<td>521 days</td>
						<td>2023-02-19 03:42:58</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "echo SRR11140744 > id.txt\nsra_ids_to_runinfo.py \\\n    id.txt\
  \ \\\n    SRR11140744.runinfo.tsv \\"
error: '[ERROR] The server couldn''t fulfill the request.

  [ERROR] Status: 400 Bad Request. Both list of IDs and query_key are empty'
exit_status: 1
kind: CalledProcessError
output: (empty)
process: NFCORE_FETCHNGS:FETCHNGS:SRA_IDS_TO_RUNINFO (SRR11140744)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '96'
- f747a7db9682f001cb3ef6e14f7db0

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2F0rx1u4fv%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2F0rx1u4fv%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.649GiB</td>
						<td>35.5 seconds</td>
						<td>22.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738932688">
								<a href="https://github.com/mrvollger/Rhodonite">mrvollger/Rhodonite</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/Rhodonite/tree/v0.2-alpha">v0.2-alpha</a>
						</td>
						<td>snakemake</td>
						<td>521 days</td>
						<td>2023-02-19 03:31:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: SystemExit
line_no: 38
rest: "1\n  File \"/home/azureuser/tmp/sm2m55pirb/code/workflow/Snakefile\", line\
  \ 38, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2F0q1hn1x2%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2F0q1hn1x2%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.8 seconds</td>
						<td>2.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738263440">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.6">v0.1.6</a>
						</td>
						<td>snakemake</td>
						<td>521 days</td>
						<td>2023-02-19 04:25:55</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/fmty43mdz4/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F3w1aglxf%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F3w1aglxf%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>1.9 seconds</td>
						<td>2.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738263200">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.7">v0.1.7</a>
						</td>
						<td>snakemake</td>
						<td>517 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 6cp7af5r2q
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 7
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/6cp7af5r2q/code/workflow/Snakefile\", line 7, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fdc93stm6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fdc93stm6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.6 seconds</td>
						<td>3.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738206368">
								<a href="https://github.com/mrvollger/fastCN-smk">mrvollger/fastCN-smk</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/fastCN-smk/tree/v0.1-alpha">v0.1-alpha</a>
						</td>
						<td>snakemake</td>
						<td>517 days</td>
						<td>2023-02-19 06:20:40</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: KeyError
line_no: 15
rest: "'manifest'\n  File \"/home/azureuser/tmp/1t1x0ihhd4/code/workflow/Snakefile\"\
  , line 15, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FfastCN-smk%2Fxw4zs14m%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FfastCN-smk%2Fxw4zs14m%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.1 seconds</td>
						<td>1.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738262960">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.1.8">v0.1.8</a>
						</td>
						<td>snakemake</td>
						<td>516 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/a99juw45a5/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fn52ypsys%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fn52ypsys%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>2.1 seconds</td>
						<td>4.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738262720">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.0">v0.2.0</a>
						</td>
						<td>snakemake</td>
						<td>515 days</td>
						<td>2023-02-19 06:33:07</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/dqvel9gyvf/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fbxdp5tyu%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fbxdp5tyu%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>1.9 seconds</td>
						<td>4.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738262480">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.1">v0.2.1</a>
						</td>
						<td>snakemake</td>
						<td>514 days</td>
						<td>2023-02-19 04:10:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/ec0uagag0c/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fll5d1bln%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fll5d1bln%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>1.9 seconds</td>
						<td>2.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738262240">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.2">v0.2.2</a>
						</td>
						<td>snakemake</td>
						<td>514 days</td>
						<td>2023-02-19 04:22:04</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/at11u755yk/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F3tofkyts%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F3tofkyts%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>1.9 seconds</td>
						<td>2.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738262000">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.3">v0.2.3</a>
						</td>
						<td>snakemake</td>
						<td>514 days</td>
						<td>2023-02-19 06:40:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/apnxi490h6/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fa0oh493w%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fa0oh493w%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>1.8 seconds</td>
						<td>1.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738261520">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.5">v0.2.5</a>
						</td>
						<td>snakemake</td>
						<td>514 days</td>
						<td>2023-02-19 06:32:00</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/5deotq8hmu/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fx801845q%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fx801845q%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>1.9 seconds</td>
						<td>2.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738261760">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.2.4">v0.2.4</a>
						</td>
						<td>snakemake</td>
						<td>514 days</td>
						<td>2023-02-19 04:16:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 24
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/7vdvtgh7cu/code/workflow/Snakefile\", line 24, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fsdf9yyzn%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fsdf9yyzn%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>1.7 seconds</td>
						<td>2.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738864560">
								<a href="https://github.com/mrvollger/Rhodonite">mrvollger/Rhodonite</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/Rhodonite/tree/v0.3-alpha">v0.3-alpha</a>
						</td>
						<td>snakemake</td>
						<td>505 days</td>
						<td>2023-02-19 03:38:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: SystemExit
line_no: 38
rest: "1\n  File \"/home/azureuser/tmp/k1xywwdn5m/code/workflow/Snakefile\", line\
  \ 38, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2F72evoju8%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2F72evoju8%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.098GiB</td>
						<td>1.5 seconds</td>
						<td>1.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743152592">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.4">3.4</a>
						</td>
						<td>nextflow</td>
						<td>502 days</td>
						<td>2023-02-19 03:32:04</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.NullPointerException
msg: Cannot get property 'args' on null object
rest: "at org.codehaus.groovy.runtime.NullObject.getProperty(NullObject.java:60)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:194)\n\
  \tat org.codehaus.groovy.runtime.callsite.NullCallSite.getProperty(NullCallSite.java:46)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:329)\n\
  \tat Script_53bf4788.runScript(Script_53bf4788:89)\n\tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:170)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:216)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:206)\n\
  \tat nextflow.script.IncludeDef.memoizedMethodPriv$loadModule0PathMapSession(IncludeDef.groovy:152)\n\
  \tat nextflow.script.IncludeDef.access$0(IncludeDef.groovy)\n\tat nextflow.script.IncludeDef$__clinit__closure2.doCall(IncludeDef.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat nextflow.script.IncludeDef.loadModule0(IncludeDef.groovy)\n\tat nextflow.script.IncludeDef.load0(IncludeDef.groovy:124)\n\
  \tat nextflow.script.IncludeDef$load0$1.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_20d7ff23.runScript(Script_20d7ff23:44)\n\tat nextflow.script.BaseScript.runDsl2(BaseScript.groovy:170)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\
  \tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\
  \tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fdnqcopm9%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fdnqcopm9%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.642GiB</td>
						<td>26.2 seconds</td>
						<td>28.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738287808">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.3.0">v2.3.0</a>
						</td>
						<td>snakemake</td>
						<td>499 days</td>
						<td>2023-02-19 06:20:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/vu2pdd4fjb/code/workflow/rules/../envs/isoform-switch-analyzer.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/7229a714-4532-4ebd-a055-84723fc4fcd4\
  \ singularity  exec --home /home/azureuser/tmp/vu2pdd4fjb/code/.test  /home/azureuser/tmp/vu2pdd4fjb/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/vu2pdd4fjb/code/.test/.snakemake/conda/6fdbd6a8a2dfd9c2ac7e68d8be3b9fc9_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/vu2pdd4fjb/code/.test/.snakemake/conda/6fdbd6a8a2dfd9c2ac7e68d8be3b9fc9_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/8 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|\
  \          | 0/8 [00:00<?, ?it/s]\nExamining bioconductor-isoformswitchanalyzer=1.14:\
  \  12%|\u2588\u258E        | 1/8 [00:00<00:00, 6594.82it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  25%|\u2588\u2588\u258C       | 2/8 [00:00<00:00, 20.23it/s]            \nExamining\
  \ bioconductor-rhdf5=2.36:  38%|\u2588\u2588\u2588\u258A      | 3/8 [00:00<00:00,\
  \ 30.28it/s]      \nExamining r-base=4.1:  50%|\u2588\u2588\u2588\u2588\u2588  \
  \   | 4/8 [00:00<00:00, 33.76it/s]             \nExamining r-base=4.1:  62%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258E   | 5/8 [00:00<00:00, 42.16it/s]\nExamining\
  \ r-tidyverse=1.3:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\u258E   | 5/8 [00:00<00:00,\
  \ 42.16it/s]\nExamining @/linux-64::__archspec==1=x86_64:  75%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258C  | 6/8 [00:02<00:00, 42.16it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  88%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [00:02<00:00,\
  \ 42.16it/s]     \n                                                            \
  \                        \n\nDetermining conflicts:   0%|          | 0/8 [00:00<?,\
  \ ?it/s]\nExamining conflict for r-tidyverse bioconductor-isoformswitchanalyzer:\
  \   0%|          | 0/8 [00:00<?, ?it/s]\nExamining conflict for bioconductor-isoformswitchanalyzer\
  \ bioconductor-rhdf5 r-base:  12%|\u2588\u258E        | 1/8 [01:57<13:44, 117.77s/it]\n\
  Examining conflict for bioconductor-isoformswitchanalyzer bioconductor-rhdf5 r-base:\
  \  25%|\u2588\u2588\u258C       | 2/8 [01:57<05:53, 58.89s/it] \nExamining conflict\
  \ for r-tidyverse bioconductor-isoformswitchanalyzer bioconductor-rhdf5 r-base:\
  \  25%|\u2588\u2588\u258C       | 2/8 [02:04<05:53, 58.89s/it]\nExamining conflict\
  \ for r-tidyverse bioconductor-isoformswitchanalyzer bioconductor-rhdf5 r-base:\
  \  38%|\u2588\u2588\u2588\u258A      | 3/8 [02:04<03:06, 37.35s/it]\nExamining conflict\
  \ for bioconductor-isoformswitchanalyzer bioconductor-rhdf5 r-base __glibc:  38%|\u2588\
  \u2588\u2588\u258A      | 3/8 [02:05<03:06, 37.35s/it]    \nExamining conflict for\
  \ bioconductor-isoformswitchanalyzer bioconductor-rhdf5 r-base __glibc:  50%|\u2588\
  \u2588\u2588\u2588\u2588     | 4/8 [02:05<01:34, 23.74s/it]\nExamining conflict\
  \ for bioconductor-rhdf5 r-base:  50%|\u2588\u2588\u2588\u2588\u2588     | 4/8 [02:10<01:34,\
  \ 23.74s/it]                                           \nExamining conflict for\
  \ bioconductor-rhdf5 r-base:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\u258E   |\
  \ 5/8 [02:10<00:51, 17.08s/it]\nExamining conflict for r-tidyverse r-base:  62%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258E   | 5/8 [02:14<00:51, 17.08s/it]       \nExamining\
  \ conflict for r-tidyverse r-base:  75%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258C  | 6/8 [02:14<00:25, 12.61s/it]\n                                       \
  \                                                 \nfailed\nSolving environment:\
  \ ...working... \nFound conflicts! Looking for incompatible packages.\nThis can\
  \ take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|\
  \          | 0/8 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|    \
  \      | 0/8 [00:00<?, ?it/s]\nExamining bioconductor-isoformswitchanalyzer=1.14:\
  \  12%|\u2588\u258E        | 1/8 [00:00<00:00, 6605.20it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  25%|\u2588\u2588\u258C       | 2/8 [00:00<00:00, 20.17it/s]            \nExamining\
  \ bioconductor-rhdf5=2.36:  38%|\u2588\u2588\u2588\u258A      | 3/8 [00:00<00:00,\
  \ 30.18it/s]      \nExamining r-base=4.1:  50%|\u2588\u2588\u2588\u2588\u2588  \
  \   | 4/8 [00:00<00:00, 33.25it/s]             \nExamining r-base=4.1:  62%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258E   | 5/8 [00:00<00:00, 41.52it/s]\nExamining\
  \ r-tidyverse=1.3:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\u258E   | 5/8 [00:00<00:00,\
  \ 41.52it/s]\nExamining @/linux-64::__archspec==1=x86_64:  75%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258C  | 6/8 [00:02<00:00, 41.52it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  88%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A | 7/8 [00:02<00:00,\
  \ 41.52it/s]     \n                                                            \
  \                        \n\nDetermining conflicts:   0%|          | 0/8 [00:00<?,\
  \ ?it/s]\nExamining conflict for r-tidyverse bioconductor-isoformswitchanalyzer:\
  \   0%|          | 0/8 [00:00<?, ?it/s]\nExamining conflict for bioconductor-isoformswitchanalyzer\
  \ bioconductor-rhdf5 r-base:  12%|\u2588\u258E        | 1/8 [02:02<14:16, 122.39s/it]\n\
  Examining conflict for bioconductor-isoformswitchanalyzer bioconductor-rhdf5 r-base:\
  \  25%|\u2588\u2588\u258C       | 2/8 [02:02<06:07, 61.20s/it] \nExamining conflict\
  \ for r-tidyverse bioconductor-isoformswitchanalyzer bioconductor-rhdf5 r-base:\
  \  25%|\u2588\u2588\u258C       | 2/8 [02:09<06:07, 61.20s/it]\nExamining conflict\
  \ for r-tidyverse bioconductor-isoformswitchanalyzer bioconductor-rhdf5 r-base:\
  \  38%|\u2588\u2588\u2588\u258A      | 3/8 [02:09<03:12, 38.59s/it]\nExamining conflict\
  \ for bioconductor-isoformswitchanalyzer bioconductor-rhdf5 r-base __glibc:  38%|\u2588\
  \u2588\u2588\u258A      | 3/8 [02:10<03:12, 38.59s/it]    \nExamining conflict for\
  \ bioconductor-isoformswitchanalyzer bioconductor-rhdf5 r-base __glibc:  50%|\u2588\
  \u2588\u2588\u2588\u2588     | 4/8 [02:10<01:38, 24.50s/it]\nExamining conflict\
  \ for bioconductor-rhdf5 r-base:  50%|\u2588\u2588\u2588\u2588\u2588     | 4/8 [02:14<01:38,\
  \ 24.50s/it]                                           \nExamining conflict for\
  \ bioconductor-rhdf5 r-base:  62%|\u2588\u2588\u2588\u2588\u2588\u2588\u258E   |\
  \ 5/8 [02:14<00:52, 17.54s/it]\nExamining conflict for r-tidyverse r-base:  62%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258E   | 5/8 [02:18<00:52, 17.54s/it]       \nExamining\
  \ conflict for r-tidyverse r-base:  75%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258C  | 6/8 [02:18<00:25, 12.90s/it]\n                                       \
  \                                                 \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with each other:\n\n\
  Output in format: Requested package -> Available versions\n\nPackage libiconv conflicts\
  \ for:\nr-tidyverse=1.3 -> r-haven[version='>=2.3.1'] -> libiconv[version='>=1.15,<2.0.0a0|>=1.16,<2.0.0a0|>=1.17,<2.0a0|>=1.15,<2.0a0']\n\
  r-base=4.1 -> libglib[version='>=2.74.1,<3.0a0'] -> libiconv[version='>=1.16,<2.0.0a0|>=1.17,<2.0.0a0|>=1.17,<2.0a0']\n\
  \nPackage r-ggplot2 conflicts for:\nr-tidyverse=1.3 -> r-ggplot2[version='>=2.2.1|>=3.2.1|>=3.3.3']\n\
  r-tidyverse=1.3 -> r-broom[version='>=0.7.6'] -> r-ggplot2\n\nPackage r-rlang conflicts\
  \ for:\nr-tidyverse=1.3 -> r-rlang[version='>=0.1.4|>=0.4.1|>=0.4.10']\nr-tidyverse=1.3\
  \ -> r-broom[version='>=0.7.6'] -> r-rlang[version='>=0.1|>=0.1.0|>=0.1.6|>=0.2.0|>=0.4.7|>=1.0.2|>=1.0.0|>=0.3.0|>=0.3.1|>=0.4.9|>=0.4.11|>=0.4.0|>=0.4.3|>=1.0.1|>=1.0.4|>=0.4.6|>=0.1.2|>=0.2.1|>=0.3.0.1']\n\
  \nPackage openblas conflicts for:\nbioconductor-rhdf5=2.36 -> libblas[version='>=3.8.0,<4.0a0']\
  \ -> openblas[version='0.3.5.*|>=0.3.6,<0.3.7.0a0']\nr-base=4.1 -> libblas[version='>=3.8.0,<4.0a0']\
  \ -> openblas[version='0.3.5.*|>=0.3.6,<0.3.7.0a0']\nbioconductor-isoformswitchanalyzer=1.14\
  \ -> libblas[version='>=3.8.0,<4.0a0'] -> openblas[version='0.3.5.*|>=0.3.6,<0.3.7.0a0']\n\
  \nPackage r-base conflicts for:\nr-base=4.1\nbioconductor-isoformswitchanalyzer=1.14\
  \ -> r-dbi -> r-base[version='3.1.3.*|3.2.0.*|3.2.1.*|3.2.2.*|3.3.1.*|3.3.2.*|3.4.1.*|>=3.4.1,<3.4.2.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=4.0,<4.1.0a0|>=4.2,<4.3.0a0|>=3.5.1,<3.5.2.0a0|>=3.5.0,<3.5.1.0a0|>=3.4.3,<3.4.4.0a0|>=3.4.2,<3.4.3a0']\n\
  bioconductor-isoformswitchanalyzer=1.14 -> r-base[version='>=4.1,<4.2.0a0']\nbioconductor-rhdf5=2.36\
  \ -> r-base[version='>=4.1,<4.2.0a0']\n\nPackage r-hms conflicts for:\nr-tidyverse=1.3\
  \ -> r-haven[version='>=2.3.1'] -> r-hms[version='>=0.4.1']\nr-tidyverse=1.3 ->\
  \ r-hms[version='>=0.3|>=0.5.2|>=1.0.0']\n\nPackage r-magrittr conflicts for:\n\
  r-tidyverse=1.3 -> r-magrittr[version='>=1.5|>=2.0.1']\nr-tidyverse=1.3 -> r-dbplyr[version='>=2.1.1']\
  \ -> r-magrittr\n\nPackage r-cli conflicts for:\nr-tidyverse=1.3 -> r-cli[version='>=1.0.0|>=1.1.0|>=2.4.0']\n\
  r-tidyverse=1.3 -> r-forcats[version='>=0.5.1'] -> r-cli[version='>=1.0.1|>=2.3.0|>=3.0.0|>=3.4|>=3.2.0|>=2.3.1|>=3.4.1']\n\
  \nPackage r-digest conflicts for:\nr-tidyverse=1.3 -> r-ggplot2[version='>=3.3.3']\
  \ -> r-digest\nbioconductor-isoformswitchanalyzer=1.14 -> r-ggplot2 -> r-digest\n\
  \nPackage r-crayon conflicts for:\nr-tidyverse=1.3 -> r-crayon[version='>=1.3.4|>=1.4.1']\n\
  r-tidyverse=1.3 -> r-dtplyr[version='>=1.1.0'] -> r-crayon\n\nPackage r-tibble conflicts\
  \ for:\nr-tidyverse=1.3 -> r-tibble[version='>=1.3.4|>=2.1.3|>=3.1.0']\nr-tidyverse=1.3\
  \ -> r-broom[version='>=0.7.6'] -> r-tibble[version='>=1.1|>=1.3.0.9007|>=1.3.1|>=1.4.2|>=3.0.0|>=2.0.0|>=2.1.1|>=2.0.1|>=1.4.1']\n\
  \nPackage openssl conflicts for:\nbioconductor-rhdf5=2.36 -> openssl[version='>=1.1.1l,<1.1.2a']\n\
  r-base=4.1 -> curl -> openssl[version='1.0.*|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=3.0.8,<4.0a0|>=3.0.7,<4.0a0|>=3.0.5,<4.0a0|>=3.0.3,<4.0a0|>=3.0.2,<4.0a0|>=3.0.0,<4.0a0|>=1.0.2m,<1.0.3a|>=1.1.1k,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1d,<1.1.2a']\n\
  \nPackage r-jsonlite conflicts for:\nr-tidyverse=1.3 -> r-jsonlite[version='>=1.5|>=1.6|>=1.7.2']\n\
  r-tidyverse=1.3 -> r-googledrive[version='>=1.0.1'] -> r-jsonlite\n\nPackage r-stringr\
  \ conflicts for:\nr-tidyverse=1.3 -> r-stringr[version='>=1.2.0|>=1.4.0']\nr-tidyverse=1.3\
  \ -> r-broom[version='>=0.7.6'] -> r-stringr[version='>=1.5.0']\n\nPackage r-dplyr\
  \ conflicts for:\nr-tidyverse=1.3 -> r-dplyr[version='>=0.7.4|>=0.8.3|>=1.0.5']\n\
  r-tidyverse=1.3 -> r-broom[version='>=0.7.6'] -> r-dplyr[version='>=0.5.0.9004|>=0.7.5|>=0.8.0|>=1.0.0|>=1.0.4|>=1.0.3|>=0.8.1|>=1.0.10|>=0.8.2|>=0.7.0']\n\
  \nPackage r-readr conflicts for:\nr-tidyverse=1.3 -> r-haven[version='>=2.3.1']\
  \ -> r-readr[version='>=0.1.0']\nr-tidyverse=1.3 -> r-readr[version='>=1.1.1|>=1.3.1|>=1.4.0']\n\
  \nPackage r-pillar conflicts for:\nr-tidyverse=1.3 -> r-pillar[version='>=1.4.2|>=1.6.0']\n\
  r-tidyverse=1.3 -> r-dplyr[version='>=1.0.5'] -> r-pillar[version='>=1.1.0|>=1.3.1|>=1.4.3|>=1.5.0|>=1.5.1|>=1.7.0']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - bioconductor-isoformswitchanalyzer=1.14 -> libgcc-ng[version='>=9.3.0'] ->\
  \ __glibc[version='>=2.17']\n  - bioconductor-rhdf5=2.36 -> libgcc-ng[version='>=9.4.0']\
  \ -> __glibc[version='>=2.17']\n  - r-base=4.1 -> libgfortran-ng -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Feod8yfup%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Feod8yfup%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.289GiB</td>
						<td>1804.7 seconds</td>
						<td>1965.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739592944">
								<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake">IMS-Bio2Core-Facility/GTExSnake</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/GTExSnake/tree/v1.2.0">v1.2.0</a>
						</td>
						<td>snakemake</td>
						<td>481 days</td>
						<td>2023-02-20 04:31:42</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- api_queries.smk
kind: CalledProcessError
line_no: 17
rest: "Command ' singularity  exec --home /home/azureuser/tmp/8rmo5x0ogv/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/8rmo5x0ogv/code/.snakemake/singularity/7854903ba970aa1cae6503ecd6762e3d.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/home/azureuser/tmp/8rmo5x0ogv/code/.snakemake/conda/0c16560bc5be7f6942e553c06b7fa5c8_'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/8rmo5x0ogv/code/.snakemake/scripts/tmpcd1a7pkg.request.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/8rmo5x0ogv/code/workflow/rules/api_queries.smk\"\
  , line 17, in __rule_request\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T043143.750823.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252FGTExSnake%2Fw9wlyxrj%2Flogs.tar.xz">0.319GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252FGTExSnake%2Fw9wlyxrj%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>2.614GiB</td>
						<td>405.9 seconds</td>
						<td>584.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738054720">
								<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing">IMS-Bio2Core-Facility/single_snake_sequencing</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>480 days</td>
						<td>2023-02-19 06:37:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- resources.smk
kind: InputFunctionException
line_no: 56
rest: "Error:\n  KeyError: '1_your'\nWildcards:\n  lane=1_your\n  sample=sample\n\
  \  read=R1\nTraceback:\n  File \"/home/azureuser/tmp/063wk1ddjo/code/workflow/rules/common.smk\"\
  , line 26, in get_fastqs"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fsingle-snake-sequencing%2F1banmun1%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fsingle-snake-sequencing%2F1banmun1%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.5 seconds</td>
						<td>1.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746555904">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/2.1.1">2.1.1</a>
						</td>
						<td>nextflow</td>
						<td>480 days</td>
						<td>2023-02-20 04:31:43</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 07:31:43.336
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fizzrtttu%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fizzrtttu%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.151GiB</td>
						<td>83.0 seconds</td>
						<td>10800.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010737937392">
								<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing">IMS-Bio2Core-Facility/single_snake_sequencing</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing/tree/v0.1.1">v0.1.1</a>
						</td>
						<td>snakemake</td>
						<td>474 days</td>
						<td>2023-02-19 04:18:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- resources.smk
kind: MissingInputException
rest: "Missing input files for rule clean_names:\n    output: data/sample1_S1_L001_R1_001.fastq.gz\n\
  \    wildcards: lane=1, sample=sample1, read=R1\n    affected files:\n        path/to/your/data/r1.fq.gz"
rule: clean_names  in line 56

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fsingle-snake-sequencing%2Fint92yqr%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fsingle-snake-sequencing%2Fint92yqr%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.8 seconds</td>
						<td>1.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738301008">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.1.0">v3.1.0</a>
						</td>
						<td>snakemake</td>
						<td>473 days</td>
						<td>2023-02-19 06:08:08</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/l4jr1fpnbe/singularity-tmp/tmpov_r0beo.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/1b632f80-c285-47d8-a23a-71a4c32253e3\
  \ singularity  exec --home /home/azureuser/tmp/l4jr1fpnbe/code/.test  /home/azureuser/tmp/l4jr1fpnbe/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/l4jr1fpnbe/code/.test/.snakemake/conda/aa3d806ef91fe8a302ee93d3f2efac46_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/l4jr1fpnbe/code/.test/.snakemake/conda/aa3d806ef91fe8a302ee93d3f2efac46_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining bcftools=1.10:   0%|        \
  \  | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:  17%|\u2588\u258B\
  \        | 1/6 [00:00<00:00, 19.60it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 39.08it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00, 58.47it/s]     \n\
  Examining @/linux-64::__unix==0=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B\
  \   | 4/6 [00:00<00:00, 77.77it/s]    \nExamining curl:  83%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00, 96.99it/s]                  \
  \ \n                                                             \n\nDetermining\
  \ conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining conflict for bcftools\
  \ curl:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining conflict for bcftools __glibc\
  \ curl:  17%|\u2588\u258B        | 1/6 [00:00<00:00, 109.66it/s]\n             \
  \                                                                              \
  \ \nfailed\nSolving environment: ...working... \nFound conflicts! Looking for incompatible\
  \ packages.\nThis can take several minutes.  Press CTRL-C to abort.\n\nBuilding\
  \ graph of deps:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining bcftools=1.10:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  17%|\u2588\u258B        | 1/6 [00:00<00:00, 25.15it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 50.10it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00, 74.92it/s]     \n\
  Examining @/linux-64::__unix==0=0:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B\
  \   | 4/6 [00:00<00:00, 99.62it/s]    \nExamining curl:  83%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00, 124.21it/s]                 \
  \ \n                                                              \n\nDetermining\
  \ conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining conflict for bcftools\
  \ curl:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining conflict for bcftools __glibc\
  \ curl:  17%|\u2588\u258B        | 1/6 [00:00<00:00, 207.24it/s]\n             \
  \                                                                              \
  \ \nfailed\n\nUnsatisfiableError: The following specifications were found to be\
  \ incompatible with each other:\n\nOutput in format: Requested package -> Available\
  \ versions\n\nPackage curl conflicts for:\nbcftools=1.10 -> htslib[version='>=1.10,<1.11.0a0']\
  \ -> curl[version='>=7.64.1,<8.0a0']\ncurlThe following specifications were found\
  \ to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - bcftools=1.10 -> libgcc-ng[version='>=7.5.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fccjxtxlr%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fccjxtxlr%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.196GiB</td>
						<td>240.0 seconds</td>
						<td>210.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743050688">
								<a href="https://nf-co.re/cutandrun">cutandrun</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/cutandrun/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>471 days</td>
						<td>2023-02-19 16:38:15</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-19 17:38:15.972
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cutandrun%2Fnlss7i7k%2Flogs.tar.xz">0.109GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cutandrun%2Fnlss7i7k%2Foutput.tar.xz">0.004GiB</a>
						</td>
						<td>1.273GiB</td>
						<td>163.0 seconds</td>
						<td>3601.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738266080">
								<a href="https://github.com/sneuensc/mapache">sneuensc/mapache</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/sneuensc/mapache/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>471 days</td>
						<td>2023-02-19 06:41:41</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>unknown error</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sneuensc%252Fmapache%2Futwf970n%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sneuensc%252Fmapache%2Futwf970n%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.7 seconds</td>
						<td>2.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738261280">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.3.0">v0.3.0</a>
						</td>
						<td>snakemake</td>
						<td>471 days</td>
						<td>2023-02-19 03:31:36</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 29
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/81esy1j188/code/workflow/Snakefile\", line 29, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fur8jvjq6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fur8jvjq6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.106GiB</td>
						<td>1.9 seconds</td>
						<td>1.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743111424">
								<a href="https://nf-co.re/fetchngs">fetchngs</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/fetchngs/tree/1.4">1.4</a>
						</td>
						<td>nextflow</td>
						<td>467 days</td>
						<td>2023-02-19 04:30:06</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "echo SRR9984183 > id.txt\nsra_ids_to_runinfo.py \\\n    id.txt\
  \ \\\n    SRR9984183.runinfo.tsv \\\n\n\ncat <<-END_VERSIONS > versions.yml\nSRA_IDS_TO_RUNINFO:\n\
  \    python: $(python --version | sed 's/Python //g')\nEND_VERSIONS"
error: '[ERROR] The server couldn''t fulfill the request.

  [ERROR] Status: 400 Bad Request. Both list of IDs and query_key are empty'
exit_status: 1
kind: CalledProcessError
output: (empty)
process: NFCORE_FETCHNGS:SRA:SRA_IDS_TO_RUNINFO (SRR9984183)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- c9
- 12b9a06aea7d10e03d69f3a6b8382a

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fgotasdl4%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fgotasdl4%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.625GiB</td>
						<td>47.9 seconds</td>
						<td>48.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738300768">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.2.0">v3.2.0</a>
						</td>
						<td>snakemake</td>
						<td>465 days</td>
						<td>2023-02-19 03:31:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/2e55tn4qum/singularity-tmp/tmpsnc916tu.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/dbc30e86-8956-44dd-905b-dccfc8de01d9\
  \ singularity  exec --home /home/azureuser/tmp/2e55tn4qum/code/.test  /home/azureuser/tmp/2e55tn4qum/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/2e55tn4qum/code/.test/.snakemake/conda/09bfd0a6183df6fa79e0354be6f24218_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/2e55tn4qum/code/.test/.snakemake/conda/09bfd0a6183df6fa79e0354be6f24218_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:  20%|\u2588\
  \u2588        | 1/5 [00:00<00:00, 7307.15it/s]\nExamining samtools==1.10:  40%|\u2588\
  \u2588\u2588\u2588      | 2/5 [00:00<00:00, 8330.30it/s]         \nExamining @/linux-64::__archspec==1=x86_64:\
  \  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00, 74.97it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  80%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588  | 4/5 [00:00<00:00, 99.57it/s]     \n                            \
  \                                                        \n\nDetermining conflicts:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict for __glibc samtools:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\n                                      \
  \                                        \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/5 [00:00<?,\
  \ ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:   0%|          | 0/5 [00:00<?,\
  \ ?it/s]\nExamining @/linux-64::__unix==0=0:  20%|\u2588\u2588        | 1/5 [00:00<00:00,\
  \ 6423.13it/s]\nExamining samtools==1.10:  40%|\u2588\u2588\u2588\u2588      | 2/5\
  \ [00:00<00:00, 7175.88it/s]         \nExamining @/linux-64::__archspec==1=x86_64:\
  \  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00, 69.11it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  80%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588  | 4/5 [00:00<00:00, 71.50it/s]     \n                            \
  \                                                        \n\nDetermining conflicts:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict for __glibc samtools:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\n                                      \
  \                                        \nfailed\n\nUnsatisfiableError: The following\
  \ specifications were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - samtools==1.10 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F4yrurf1l%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F4yrurf1l%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.182GiB</td>
						<td>309.0 seconds</td>
						<td>374.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738205648">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.0">1.4.0</a>
						</td>
						<td>snakemake</td>
						<td>458 days</td>
						<td>2023-02-19 04:48:23</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 8s3hf4cn6o
- code
- rules
- blast.smk
kind: CalledProcessError
line_no: 70
rest: "Command 'set -euo pipefail;  /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/python3.10\
  \ /home/azureuser/tmp/8s3hf4cn6o/code/tests/.snakemake/scripts/tmpsabsl5xf.make_blast_mask.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/8s3hf4cn6o/code/rules/blast.smk\"\
  , line 70, in __rule_create_blast_mask\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Sun Feb 19 04:54:17 2023]\nFinished job 47.\n24 of 64 steps\
  \ (38%) done\nRemoving temporary output NTC/trimmed/NTC_R1.fastq.\nRemoving temporary\
  \ output NTC/trimmed/NTC_R2.fastq.\nRemoving temporary output NTC/denoising/NTC_R1_filtered.fasta.\n\
  Removing temporary output NTC/denoising/NTC_R2_filtered.fasta.\nShutting down, this\
  \ might take some time.\nExiting because a job execution failed. Look above for\
  \ error message\nComplete log: .snakemake/log/2023-02-19T044824.099910.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fw5iddmxp%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fw5iddmxp%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.937GiB</td>
						<td>344.7 seconds</td>
						<td>355.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738812816">
								<a href="https://github.com/mrvollger/Rhodonite">mrvollger/Rhodonite</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/Rhodonite/tree/v0.4-alpha">v0.4-alpha</a>
						</td>
						<td>snakemake</td>
						<td>457 days</td>
						<td>2023-02-19 06:01:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: SystemExit
line_no: 38
rest: "1\n  File \"/home/azureuser/tmp/gduvwe1lq0/code/workflow/Snakefile\", line\
  \ 38, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Fkdqfhgb7%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Fkdqfhgb7%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.9 seconds</td>
						<td>2.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738300528">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.3.0">v3.3.0</a>
						</td>
						<td>snakemake</td>
						<td>457 days</td>
						<td>2023-02-19 04:02:48</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/1htjjmibhn/singularity-tmp/tmpbg5se_jz.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/015211bc-6942-45fc-9536-1e8a63f7f6f4\
  \ singularity  exec --home /home/azureuser/tmp/1htjjmibhn/code/.test  /home/azureuser/tmp/1htjjmibhn/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/1htjjmibhn/code/.test/.snakemake/conda/e10448b37a2300c25a497c51c551b422_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/1htjjmibhn/code/.test/.snakemake/conda/e10448b37a2300c25a497c51c551b422_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining samtools==1.10:  20%|\u2588\u2588\
  \        | 1/5 [00:00<00:00, 7681.88it/s]        \nExamining @/linux-64::__glibc==2.31=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 129.17it/s]\nExamining\
  \ @/linux-64::__linux==5.15.0=0:  60%|\u2588\u2588\u2588\u2588\u2588\u2588    |\
  \ 3/5 [00:00<00:00, 192.46it/s]\nExamining @/linux-64::__unix==0=0:  80%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 255.07it/s]      \n  \
  \                                                                              \
  \ \n\nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict\
  \ for samtools __glibc:   0%|          | 0/5 [00:00<?, ?it/s]\n                \
  \                                                              \nfailed\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining samtools==1.10:  20%|\u2588\u2588\
  \        | 1/5 [00:00<00:00, 8371.86it/s]        \nExamining @/linux-64::__glibc==2.31=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 127.47it/s]\nExamining\
  \ @/linux-64::__linux==5.15.0=0:  60%|\u2588\u2588\u2588\u2588\u2588\u2588    |\
  \ 3/5 [00:00<00:00, 189.69it/s]\nExamining @/linux-64::__unix==0=0:  80%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 251.40it/s]      \n  \
  \                                                                              \
  \ \n\nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict\
  \ for samtools __glibc:   0%|          | 0/5 [00:00<?, ?it/s]\n                \
  \                                                              \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with your system:\n\
  \n  - feature:/linux-64::__glibc==2.31=0\n  - samtools==1.10 -> libgcc-ng[version='>=7.5.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fpxowkc8q%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fpxowkc8q%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.152GiB</td>
						<td>376.7 seconds</td>
						<td>380.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738695424">
								<a href="https://github.com/mrvollger/Rhodonite">mrvollger/Rhodonite</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/Rhodonite/tree/v0.5-alpha">v0.5-alpha</a>
						</td>
						<td>snakemake</td>
						<td>457 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: SystemExit
line_no: 38
rest: "1\n  File \"/home/azureuser/tmp/awmaux96nb/code/workflow/Snakefile\", line\
  \ 38, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Fw4i1q55n%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Fw4i1q55n%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.2 seconds</td>
						<td>1.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740708016">
								<a href="https://github.com/mrvollger/Rhodonite">mrvollger/Rhodonite</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/Rhodonite/tree/v0.6-alpha">v0.6-alpha</a>
						</td>
						<td>snakemake</td>
						<td>457 days</td>
						<td>2023-02-19 03:31:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: SystemExit
line_no: 38
rest: "1\n  File \"/home/azureuser/tmp/jh2gnn6s6l/code/workflow/Snakefile\", line\
  \ 38, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Fljkv5lau%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Fljkv5lau%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.098GiB</td>
						<td>1.9 seconds</td>
						<td>3.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738287568">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.3.1">v2.3.1</a>
						</td>
						<td>snakemake</td>
						<td>455 days</td>
						<td>2023-02-19 12:41:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/vekhvfhx0u/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/309e114e-0dc4-4927-9546-0cddd37fe199\
  \ singularity  exec --home /home/azureuser/tmp/vekhvfhx0u/code/.test  /home/azureuser/tmp/vekhvfhx0u/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/vekhvfhx0u/code/.test/.snakemake/conda/8c7908aa3d111a647c2d99586930fb49_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/vekhvfhx0u/code/.test/.snakemake/conda/8c7908aa3d111a647c2d99586930fb49_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining matplotlib=3.1:   0%|       \
  \   | 0/9 [00:00<?, ?it/s]\nExamining python=3.7:  11%|\u2588         | 1/9 [00:07<00:58,\
  \  7.35s/it]\nExamining python=3.7:  22%|\u2588\u2588\u258F       | 2/9 [00:07<00:25,\
  \  3.68s/it]\nExamining @/linux-64::__unix==0=0:  22%|\u2588\u2588\u258F       |\
  \ 2/9 [00:11<00:25,  3.68s/it]\nExamining @/linux-64::__unix==0=0:  33%|\u2588\u2588\
  \u2588\u258E      | 3/9 [00:11<00:22,  3.68s/it]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:11<00:22,  3.68s/it]\nExamining pillow=6.2:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:11<00:18,  3.68s/it]       \
  \            \nExamining goatools=0.9.7:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:12<00:14,  3.68s/it]\nExamining goatools=0.9.7:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:12<00:04,  1.58s/it]\nExamining pandas=0.25:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:12<00:04,  1.58s/it]\
  \   \nExamining pandas=0.25:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [00:12<00:02,  1.33s/it]\nExamining @/linux-64::__glibc==2.31=0:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:16<00:02,  1.33s/it]\nExamining\
  \ @/linux-64::__glibc==2.31=0:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:16<00:01,  1.96s/it]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:16<00:01,\
  \  1.96s/it]\n                                                                 \
  \                        \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for matplotlib python pillow goatools pandas:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining conflict for matplotlib pandas:  11%|\u2588\
  \         | 1/9 [00:09<01:14,  9.37s/it]               \nExamining conflict for\
  \ matplotlib pandas:  22%|\u2588\u2588\u258F       | 2/9 [00:09<00:32,  4.69s/it]\n\
  Examining conflict for matplotlib goatools pandas:  22%|\u2588\u2588\u258F     \
  \  | 2/9 [00:16<00:32,  4.69s/it]\nExamining conflict for matplotlib goatools pandas:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:16<00:34,  5.72s/it]\nExamining conflict\
  \ for matplotlib pillow goatools pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [01:21<00:34,  5.72s/it]\nExamining conflict for matplotlib pillow goatools pandas:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [01:21<02:19, 27.87s/it]\nExamining\
  \ conflict for pandas matplotlib python pillow:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [01:44<02:19, 27.87s/it]  \nExamining conflict for pandas matplotlib\
  \ python pillow:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [01:44<01:45,\
  \ 26.30s/it]\nExamining conflict for matplotlib pillow:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [01:55<01:45, 26.30s/it]              \nExamining conflict\
  \ for matplotlib pillow:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9\
  \ [01:55<01:02, 20.91s/it]\nExamining conflict for matplotlib python pandas:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [01:59<01:02, 20.91s/it]\nExamining\
  \ conflict for matplotlib python pandas:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [01:59<00:31, 15.51s/it]\nExamining conflict for matplotlib\
  \ python pillow __glibc pandas:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [01:59<00:31, 15.51s/it]\nExamining conflict for matplotlib python pillow\
  \ __glibc pandas:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 |\
  \ 8/9 [01:59<00:10, 10.76s/it]\nExamining conflict for matplotlib goatools:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [02:06<00:10, 10.76s/it]\
  \                    \nExamining conflict for matplotlib goatools: 100%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [02:06<00:00,  9.61s/it]\n\
  Examining conflict for python pillow: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588| 9/9 [02:12<00:00,  9.61s/it]      \nExamining conflict for python\
  \ pillow: : 10it [02:12,  8.57s/it]                    \nExamining conflict for\
  \ python goatools pillow: : 10it [02:14,  8.57s/it]\nExamining conflict for python\
  \ goatools pillow: : 11it [02:14,  6.33s/it]\nExamining conflict for goatools pandas:\
  \ : 11it [02:16,  6.33s/it]       \nExamining conflict for goatools pandas: : 12it\
  \ [02:16,  5.29s/it]\n                                                         \
  \        \nfailed\nSolving environment: ...working... \nFound conflicts! Looking\
  \ for incompatible packages.\nThis can take several minutes.  Press CTRL-C to abort.\n\
  \nBuilding graph of deps:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining matplotlib=3.1:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining python=3.7:  11%|\u2588     \
  \    | 1/9 [00:02<00:20,  2.56s/it]\nExamining python=3.7:  22%|\u2588\u2588\u258F\
  \       | 2/9 [00:02<00:08,  1.28s/it]\nExamining @/linux-64::__unix==0=0:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:04<00:08,  1.28s/it]\nExamining @/linux-64::__unix==0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:04<00:09,  1.54s/it]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:04<00:09,  1.54s/it]\nExamining pillow=6.2:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:04<00:07,  1.54s/it]       \
  \            \nExamining goatools=0.9.7:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:05<00:06,  1.54s/it]\nExamining goatools=0.9.7:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:05<00:02,  1.45it/s]\nExamining pandas=0.25:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:05<00:02,  1.45it/s]\
  \   \nExamining pandas=0.25:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [00:05<00:01,  1.66it/s]\nExamining @/linux-64::__glibc==2.31=0:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:07<00:01,  1.66it/s]\nExamining\
  \ @/linux-64::__glibc==2.31=0:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:07<00:01,  1.01s/it]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:07<00:01,\
  \  1.01s/it]\n                                                                 \
  \                        \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for matplotlib python pillow goatools pandas:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining conflict for matplotlib pandas:  11%|\u2588\
  \         | 1/9 [00:04<00:38,  4.86s/it]               \nExamining conflict for\
  \ matplotlib pandas:  22%|\u2588\u2588\u258F       | 2/9 [00:04<00:17,  2.43s/it]\n\
  Examining conflict for matplotlib goatools pandas:  22%|\u2588\u2588\u258F     \
  \  | 2/9 [00:08<00:17,  2.43s/it]\nExamining conflict for matplotlib goatools pandas:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:08<00:16,  2.82s/it]\nExamining conflict\
  \ for matplotlib pillow goatools pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [00:50<00:16,  2.82s/it]\nExamining conflict for matplotlib pillow goatools pandas:\
  \  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:50<01:26, 17.40s/it]\nExamining\
  \ conflict for pandas matplotlib python pillow:  44%|\u2588\u2588\u2588\u2588\u258D\
  \     | 4/9 [01:01<01:26, 17.40s/it]  \nExamining conflict for pandas matplotlib\
  \ python pillow:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [01:01<01:01,\
  \ 15.43s/it]\nExamining conflict for matplotlib pillow:  56%|\u2588\u2588\u2588\u2588\
  \u2588\u258C    | 5/9 [01:06<01:01, 15.43s/it]              \nExamining conflict\
  \ for matplotlib pillow:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9\
  \ [01:06<00:35, 11.95s/it]\nExamining conflict for matplotlib python pandas:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [01:09<00:35, 11.95s/it]\nExamining\
  \ conflict for matplotlib python pandas:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [01:09<00:17,  8.86s/it]\nExamining conflict for matplotlib\
  \ python pillow __glibc pandas:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A\
  \  | 7/9 [01:09<00:17,  8.86s/it]\nExamining conflict for matplotlib python pillow\
  \ __glibc pandas:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 |\
  \ 8/9 [01:09<00:06,  6.15s/it]\nExamining conflict for matplotlib goatools:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [01:13<00:06,  6.15s/it]\
  \                    \nExamining conflict for matplotlib goatools: 100%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [01:13<00:00,  5.37s/it]\n\
  Examining conflict for python pillow: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588| 9/9 [01:16<00:00,  5.37s/it]      \nExamining conflict for python\
  \ pillow: : 10it [01:16,  4.81s/it]                    \nExamining conflict for\
  \ python goatools pillow: : 10it [01:17,  4.81s/it]\nExamining conflict for python\
  \ goatools pillow: : 11it [01:17,  3.57s/it]\nExamining conflict for goatools pandas:\
  \ : 11it [01:18,  3.57s/it]       \nExamining conflict for goatools pandas: : 12it\
  \ [01:18,  3.00s/it]\n                                                         \
  \        \nfailed\n\nUnsatisfiableError: The following specifications were found\
  \ to be incompatible with each other:\n\nOutput in format: Requested package ->\
  \ Available versions\n\nPackage six conflicts for:\nmatplotlib=3.1 -> cycler[version='>=0.10']\
  \ -> six[version='>=1.5']\npandas=0.25 -> python-dateutil[version='>=2.6.1'] ->\
  \ six[version='>=1.5']\n\nPackage setuptools conflicts for:\npython=3.7 -> pip ->\
  \ setuptools\nmatplotlib=3.1 -> setuptools\npandas=0.25 -> setuptools[version='<60.0.0']\n\
  \nPackage numpy conflicts for:\nmatplotlib=3.1 -> matplotlib-base==3.1.0=py37h5f35d83_0\
  \ -> numpy[version='>=1.11.3,<2.0a0']\nmatplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0']\n\
  \nPackage python conflicts for:\ngoatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  pandas=0.25 -> python[version='>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']\n\
  python=3.7\nmatplotlib=3.1 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  pillow=6.2 -> olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|>=3.6|>=3|3.4.*']\n\
  matplotlib=3.1 -> pyqt -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.6|>=3.0,<=3.9|>=3.7|>=3.8|>=3']\n\
  pillow=6.2 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  goatools=0.9.7 -> python\n\nPackage _libgcc_mutex conflicts for:\npandas=0.25 ->\
  \ libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  python=3.7 -> libgcc-ng[version='>=9.4.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main|main']\n\
  matplotlib=3.1 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  \nPackage expat conflicts for:\npillow=6.2 -> pypy3.6[version='>=7.3.1'] -> expat[version='>=2.2.9,<3.0.0a0']\n\
  python=3.7 -> pypy3.7=7.3.7 -> expat[version='>=2.2.9,<3.0.0a0|>=2.3.0,<3.0a0|>=2.4.1,<3.0a0']\n\
  \nPackage pypy3.6 conflicts for:\ngoatools=0.9.7 -> numpy -> pypy3.6[version='7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*|>=7.3.1|>=7.3.2|>=7.3.3']\n\
  matplotlib=3.1 -> python[version='>=3.6,<3.7.0a0'] -> pypy3.6[version='7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*|>=7.3.3|>=7.3.2|>=7.3.1']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> pypy3.6[version='7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*|>=7.3.1|>=7.3.2|>=7.3.3']\n\
  pillow=6.2 -> python[version='>=3.6,<3.7.0a0'] -> pypy3.6[version='7.3.*|7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*']\n\
  pillow=6.2 -> pypy3.6[version='>=7.3.1']\n\nPackage ca-certificates conflicts for:\n\
  goatools=0.9.7 -> python -> ca-certificates\npython=3.7 -> openssl[version='>=1.1.1l,<1.1.2a']\
  \ -> ca-certificates\npillow=6.2 -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n\
  \nPackage freetype conflicts for:\npillow=6.2 -> freetype[version='>=2.9.1,<3.0a0']\n\
  matplotlib=3.1 -> freetype[version='>=2.9.1,<3.0a0']\n\nPackage numpy-base conflicts\
  \ for:\npandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py36h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py36h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py36h2b20989_1|py37h2b20989_1|py27hdbf6ddf_1|py37hdbf6ddf_1|py27h2b20989_1|py27h2b20989_2|py27hdbf6ddf_2|py36h2b20989_3|py37h2b20989_3|py37hdbf6ddf_3|py37h2b20989_4|py36h2b20989_4|py36hdbf6ddf_4|py35hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py37h81de0dd_4|py38h2f8d375_4|py36h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py27h3dfced4_0|py36h74e8950_0|py36h81de0dd_0|py27h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py37h81de0dd_1|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_1|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py38h2f8d375_0|py39h76555f2_1|py39h41b4c56_3|py38h41b4c56_3|py37h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py38hdc34a94_3|py39hdc34a94_3|py39ha8aedfd_4|py39h73d599e_4|py38h73d599e_4|py37hfa32c7d_0|py38hfa32c7d_0|py36hfa32c7d_0|py36hfa32c7d_0|py38hfa32c7d_0|py39h2ae0177_0|py39h0f7b65f_0|py37h4c65ebe_1|py38h4c65ebe_1|py39h21a3de8_1|py37h34387ca_0|py37h7d8b39e_0|py39h34387ca_0|py38h34387ca_0|py38h7d8b39e_0|py38he2ba247_0|py38hfae3a4d_0|py37hfae3a4d_0|py37h74d4b33_0|py39h74d4b33_0|py38h74d4b33_0|py38h39b7dee_0|py37h39b7dee_0|py39h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py37h2b8c604_0|py39h2b8c604_0|py37h79a1101_0|py38h2b8c604_0|py310h79a1101_0|py39hb8be1f0_1|py39hf524024_1|py38hb8be1f0_1|py37hb8be1f0_1|py38hf524024_1|py37hf524024_1|py310hf2716ce_1|py310h9585f30_1|py38hb8be1f0_2|py39hf524024_2|py37h1e6e340_3|py38h1e6e340_3|py39hf524024_0|py38hf524024_0|py310hf2716ce_0|py38hb8be1f0_0|py38h1e6e340_0|py310h8e6c178_0|py310h8e6c178_1|py39h31eccc5_1|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py38h31eccc5_0|py38h31eccc5_1|py39h1e6e340_1|py310h375b286_1|py38h1e6e340_1|py310h375b286_0|py39h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py38ha15fc14_0|py39ha15fc14_0|py310h375b286_0|py39h1e6e340_0|py39hb8be1f0_0|py310h9585f30_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py38hf524024_2|py37hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py39hb8be1f0_2|py310hf2716ce_2|py310h2b8c604_0|py39hfae3a4d_0|py37he2ba247_0|py39he2ba247_0|py39h7d8b39e_0|py38h21a3de8_1|py37h21a3de8_1|py39h4c65ebe_1|py36h75fe3a5_0|py37h75fe3a5_0|py38h75fe3a5_0|py37hfa32c7d_0|py37h75fe3a5_0|py36h75fe3a5_0|py38h75fe3a5_0|py37h73d599e_4|py38ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py39hfb011de_1|py27hde5b4d6_0|py27h2f8d375_0|py38hde5b4d6_0|py37hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py37h2f8d375_1|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py27h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py36h81de0dd_1|py27h81de0dd_1|py27h2f8d375_1|py36h2f8d375_1|py36h81de0dd_0|py35h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py27h81de0dd_0|py37h74e8950_0|py27h74e8950_0|py35h74e8950_0|py37h3dfced4_0|py35h3dfced4_0|py36h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py36h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py37h2f8d375_5|py38hde5b4d6_4|py36h81de0dd_4|py35h81de0dd_4|py27h81de0dd_4|py35h2f8d375_4|py37hde5b4d6_0|py37hdbf6ddf_4|py27hdbf6ddf_4|py27h2b20989_4|py27hdbf6ddf_3|py27h2b20989_3|py36hdbf6ddf_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py37h2b20989_2|py36h2b20989_2|py36hdbf6ddf_1|py35hdbf6ddf_0|py27h2b20989_0|py35hdbf6ddf_0|py35h2b20989_0|py35h9be14a7_1|py27h9be14a7_1|py35h0ea5e3f_1|py27h0ea5e3f_1|py36h0ea5e3f_1']\n\
  matplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py27h2b20989_6|py36hdbf6ddf_6|py27h2b20989_7|py36h2b20989_7|py35h2b20989_7|py36h2b20989_7|py36hdbf6ddf_8|py36h2b20989_8|py27h2b20989_8|py36h7cdd4dd_9|py35h7cdd4dd_9|py36h3dfced4_9|py35h3dfced4_9|py27h3dfced4_9|py36h81de0dd_9|py37h74e8950_9|py35h81de0dd_9|py37h74e8950_10|py35h74e8950_10|py27h74e8950_10|py27h81de0dd_10|py36h81de0dd_10|py35h81de0dd_10|py37h2f8d375_10|py27h2f8d375_10|py27h2f8d375_11|py27hde5b4d6_11|py27h2f8d375_12|py36h2f8d375_12|py36hde5b4d6_12|py36h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py27hdbf6ddf_0|py36hdbf6ddf_0|py36h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py36h2b20989_1|py37h2b20989_1|py27hdbf6ddf_1|py37hdbf6ddf_1|py27h2b20989_1|py27h2b20989_2|py27hdbf6ddf_2|py36h2b20989_3|py37h2b20989_3|py37hdbf6ddf_3|py37h2b20989_4|py36h2b20989_4|py36hdbf6ddf_4|py35hdbf6ddf_4|py35h2b20989_4|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py37h81de0dd_4|py38h2f8d375_4|py36h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py27h3dfced4_0|py36h74e8950_0|py36h81de0dd_0|py27h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py37h81de0dd_1|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_1|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py38h2f8d375_0|py39h76555f2_1|py39h41b4c56_3|py38h41b4c56_3|py37h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py38hdc34a94_3|py39hdc34a94_3|py39ha8aedfd_4|py39h73d599e_4|py38h73d599e_4|py37hfa32c7d_0|py38hfa32c7d_0|py36hfa32c7d_0|py36hfa32c7d_0|py38hfa32c7d_0|py39h2ae0177_0|py39h0f7b65f_0|py37h4c65ebe_1|py38h4c65ebe_1|py39h21a3de8_1|py37h34387ca_0|py37h7d8b39e_0|py39h34387ca_0|py38h34387ca_0|py38h7d8b39e_0|py38he2ba247_0|py38hfae3a4d_0|py37hfae3a4d_0|py37h74d4b33_0|py39h74d4b33_0|py38h74d4b33_0|py38h39b7dee_0|py37h39b7dee_0|py39h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py37h2b8c604_0|py39h2b8c604_0|py37h79a1101_0|py38h2b8c604_0|py310h79a1101_0|py39hb8be1f0_1|py39hf524024_1|py38hb8be1f0_1|py37hb8be1f0_1|py38hf524024_1|py37hf524024_1|py310hf2716ce_1|py310h9585f30_1|py38hb8be1f0_2|py39hf524024_2|py37h1e6e340_3|py38h1e6e340_3|py39hf524024_0|py38hf524024_0|py310hf2716ce_0|py38hb8be1f0_0|py38h1e6e340_0|py310h8e6c178_0|py310h8e6c178_1|py39h31eccc5_1|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py38h31eccc5_0|py38h31eccc5_1|py39h1e6e340_1|py310h375b286_1|py38h1e6e340_1|py310h375b286_0|py39h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py38ha15fc14_0|py39ha15fc14_0|py310h375b286_0|py39h1e6e340_0|py39hb8be1f0_0|py310h9585f30_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py38hf524024_2|py37hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py39hb8be1f0_2|py310hf2716ce_2|py310h2b8c604_0|py39hfae3a4d_0|py37he2ba247_0|py39he2ba247_0|py39h7d8b39e_0|py38h21a3de8_1|py37h21a3de8_1|py39h4c65ebe_1|py36h75fe3a5_0|py37h75fe3a5_0|py38h75fe3a5_0|py37hfa32c7d_0|py37h75fe3a5_0|py36h75fe3a5_0|py38h75fe3a5_0|py37h73d599e_4|py38ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py39hfb011de_1|py27hde5b4d6_0|py27h2f8d375_0|py38hde5b4d6_0|py37hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py37h2f8d375_1|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py27h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py36h81de0dd_1|py27h81de0dd_1|py27h2f8d375_1|py36h2f8d375_1|py36h81de0dd_0|py35h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py27h81de0dd_0|py37h74e8950_0|py27h74e8950_0|py35h74e8950_0|py37h3dfced4_0|py35h3dfced4_0|py36h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py36h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py37h2f8d375_5|py38hde5b4d6_4|py36h81de0dd_4|py35h81de0dd_4|py27h81de0dd_4|py35h2f8d375_4|py37hde5b4d6_0|py37hdbf6ddf_4|py27hdbf6ddf_4|py27h2b20989_4|py27hdbf6ddf_3|py27h2b20989_3|py36hdbf6ddf_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py37h2b20989_2|py36h2b20989_2|py36hdbf6ddf_1|py35hdbf6ddf_0|py27h2b20989_0|py35hdbf6ddf_0|py35h2b20989_0|py35h9be14a7_1|py27h9be14a7_1|py35h0ea5e3f_1|py27h0ea5e3f_1|py36h0ea5e3f_1|py38h2f8d375_12|py38hde5b4d6_12|py37hde5b4d6_12|py27hde5b4d6_12|py37h2f8d375_12|py37hde5b4d6_11|py36hde5b4d6_11|py36h2f8d375_11|py37h2f8d375_11|py35h2f8d375_10|py36h2f8d375_10|py37h81de0dd_10|py36h74e8950_10|py27h81de0dd_9|py36h74e8950_9|py35h74e8950_9|py27h74e8950_9|py37h81de0dd_9|py37h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py35h2b20989_8|py35hdbf6ddf_8|py27hdbf6ddf_8|py37h2b20989_8|py37hdbf6ddf_8|py37hdbf6ddf_7|py36hdbf6ddf_7|py27hdbf6ddf_7|py27h2b20989_7|py37h2b20989_7|py37h2b20989_7|py37hdbf6ddf_7|py35hdbf6ddf_7|py36hdbf6ddf_7|py27hdbf6ddf_7|py27hdbf6ddf_6|py37hdbf6ddf_6|py37h2b20989_6|py36h2b20989_6']\n\
  goatools=0.9.7 -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py27hdbf6ddf_6|py36hdbf6ddf_6|py27hdbf6ddf_7|py27h2b20989_7|py36h2b20989_7|py35hdbf6ddf_7|py35h2b20989_7|py36h2b20989_7|py36hdbf6ddf_8|py36h2b20989_8|py36h7cdd4dd_9|py36h3dfced4_9|py35h3dfced4_9|py27h3dfced4_9|py36h81de0dd_9|py37h81de0dd_9|py27h81de0dd_9|py35h81de0dd_9|py37h74e8950_10|py35h74e8950_10|py37h81de0dd_10|py36h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py27h2f8d375_11|py36h2f8d375_11|py27hde5b4d6_11|py37h2f8d375_12|py27h2f8d375_12|py36h2f8d375_12|py27hde5b4d6_12|py37hde5b4d6_12|py36hde5b4d6_12|py27h0ea5e3f_1|py36h9be14a7_1|py27h9be14a7_1|py36h2b20989_0|py27hdbf6ddf_0|py36h2b20989_0|py27h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py36h2b20989_1|py27hdbf6ddf_1|py37hdbf6ddf_1|py27h2b20989_1|py27hdbf6ddf_2|py37h2b20989_3|py37hdbf6ddf_3|py27h2b20989_4|py37h2b20989_4|py37hdbf6ddf_4|py36h2b20989_4|py35h2b20989_4|py37h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py37h81de0dd_4|py38h2f8d375_4|py36h2f8d375_5|py27h2f8d375_5|py27hde5b4d6_5|py36hde5b4d6_5|py27h3dfced4_0|py36h74e8950_0|py36h81de0dd_0|py27h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py37h81de0dd_1|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py36h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_1|py36h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py38h2f8d375_0|py39h76555f2_1|py39h41b4c56_3|py38h41b4c56_3|py37h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py38hdc34a94_3|py39hdc34a94_3|py39ha8aedfd_4|py39h73d599e_4|py38h73d599e_4|py37hfa32c7d_0|py38hfa32c7d_0|py36hfa32c7d_0|py36hfa32c7d_0|py38hfa32c7d_0|py39h2ae0177_0|py39h0f7b65f_0|py37h4c65ebe_1|py38h4c65ebe_1|py39h21a3de8_1|py37h34387ca_0|py37h7d8b39e_0|py39h34387ca_0|py38h34387ca_0|py38h7d8b39e_0|py38he2ba247_0|py38hfae3a4d_0|py37hfae3a4d_0|py37h74d4b33_0|py39h74d4b33_0|py38h74d4b33_0|py38h39b7dee_0|py37h39b7dee_0|py39h39b7dee_0|py38h79a1101_0|py39h79a1101_0|py37h2b8c604_0|py39h2b8c604_0|py37h79a1101_0|py38h2b8c604_0|py310h79a1101_0|py39hb8be1f0_1|py39hf524024_1|py38hb8be1f0_1|py37hb8be1f0_1|py38hf524024_1|py37hf524024_1|py310hf2716ce_1|py310h9585f30_1|py38hb8be1f0_2|py39hf524024_2|py37h1e6e340_3|py38h1e6e340_3|py39hf524024_0|py38hf524024_0|py310hf2716ce_0|py38hb8be1f0_0|py38h1e6e340_0|py310h8e6c178_0|py310h8e6c178_1|py39h31eccc5_1|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_0|py310h375b286_0|py39h1e6e340_0|py38h31eccc5_0|py38h31eccc5_1|py39h1e6e340_1|py310h375b286_1|py38h1e6e340_1|py310h375b286_0|py39h1e6e340_0|py38h1e6e340_0|py39h31eccc5_0|py38h31eccc5_0|py310hcba007f_0|py38ha15fc14_0|py39ha15fc14_0|py310h375b286_0|py39h1e6e340_0|py39hb8be1f0_0|py310h9585f30_0|py39ha15fc14_3|py310hcba007f_3|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py38hf524024_2|py37hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py39hb8be1f0_2|py310hf2716ce_2|py310h2b8c604_0|py39hfae3a4d_0|py37he2ba247_0|py39he2ba247_0|py39h7d8b39e_0|py38h21a3de8_1|py37h21a3de8_1|py39h4c65ebe_1|py36h75fe3a5_0|py37h75fe3a5_0|py38h75fe3a5_0|py37hfa32c7d_0|py37h75fe3a5_0|py36h75fe3a5_0|py38h75fe3a5_0|py37h73d599e_4|py38ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py39hfb011de_1|py27hde5b4d6_0|py27h2f8d375_0|py38hde5b4d6_0|py37hde5b4d6_0|py37hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py37h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py37hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py37h2f8d375_1|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py27h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py36h81de0dd_1|py27h81de0dd_1|py27h2f8d375_1|py36h2f8d375_1|py36h81de0dd_0|py35h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py35h2f8d375_0|py35h81de0dd_0|py37h81de0dd_0|py27h81de0dd_0|py37h74e8950_0|py27h74e8950_0|py35h74e8950_0|py37h3dfced4_0|py35h3dfced4_0|py36h3dfced4_0|py35h7cdd4dd_0|py37h7cdd4dd_0|py36h7cdd4dd_0|py27h7cdd4dd_0|py37hde5b4d6_5|py37h2f8d375_5|py38hde5b4d6_4|py36h81de0dd_4|py35h81de0dd_4|py27h81de0dd_4|py35h2f8d375_4|py35hdbf6ddf_4|py36hdbf6ddf_4|py27hdbf6ddf_4|py27hdbf6ddf_3|py27h2b20989_3|py36hdbf6ddf_3|py36h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py37h2b20989_2|py36h2b20989_2|py27h2b20989_2|py36hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py35hdbf6ddf_0|py36hdbf6ddf_0|py35h2b20989_0|py27h2b20989_0|py35h9be14a7_1|py35h0ea5e3f_1|py36h0ea5e3f_1|py38h2f8d375_12|py38hde5b4d6_12|py37hde5b4d6_11|py36hde5b4d6_11|py37h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py35h81de0dd_10|py27h81de0dd_10|py36h74e8950_10|py27h74e8950_10|py36h74e8950_9|py37h74e8950_9|py35h74e8950_9|py27h74e8950_9|py37h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py35h7cdd4dd_9|py35h2b20989_8|py35hdbf6ddf_8|py27hdbf6ddf_8|py27h2b20989_8|py37h2b20989_8|py37hdbf6ddf_8|py37hdbf6ddf_7|py36hdbf6ddf_7|py27hdbf6ddf_7|py27h2b20989_7|py37h2b20989_7|py37h2b20989_7|py37hdbf6ddf_7|py36hdbf6ddf_7|py37hdbf6ddf_6|py37h2b20989_6|py36h2b20989_6|py27h2b20989_6|py37hde5b4d6_0']\n\
  \nPackage libpng conflicts for:\nmatplotlib=3.1 -> libpng[version='>=1.6.37,<1.7.0a0']\n\
  matplotlib=3.1 -> matplotlib-base==3.1.1=py36he7580a8_1 -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  \nPackage pyparsing conflicts for:\nmatplotlib=3.1 -> pyparsing\ngoatools=0.9.7\
  \ -> pydot -> pyparsing[version='>=2.1.4|>=2.1.4,<3']\n\nPackage pandas conflicts\
  \ for:\ngoatools=0.9.7 -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\n\
  pandas=0.25The following specifications were found to be incompatible with your\
  \ system:\n\n  - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - matplotlib=3.1 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - pandas=0.25 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\n\
  Your installed version is: 2.31\n\nNote that strict channel priority may have removed\
  \ packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fw90izivv%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fw90izivv%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.223GiB</td>
						<td>2312.6 seconds</td>
						<td>2946.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739508384">
								<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover">IMS-Bio2Core-Facility/polya_liftover</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v0.2.0">v0.2.0</a>
						</td>
						<td>snakemake</td>
						<td>454 days</td>
						<td>2023-02-19 06:27:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- resources.smk
kind: MissingInputException
rest: "Missing input files for rule clean_names:\n    output: data/sample1_S1_L001_R1_001.fastq.gz\n\
  \    wildcards: lane=1, sample=sample1, read=R1\n    affected files:\n        path/to/your/data/r1.fq.gz"
rule: clean_names  in line 56

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2F4t7kdqh6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2F4t7kdqh6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.6 seconds</td>
						<td>1.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738267520">
								<a href="https://github.com/mrvollger/StainedGlass">mrvollger/StainedGlass</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/StainedGlass/tree/v0.2">v0.2</a>
						</td>
						<td>snakemake</td>
						<td>453 days</td>
						<td>2023-02-19 13:30:58</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/lu7p2kpy92/code/workflow/envs/env.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/lu7p2kpy92/code/.snakemake/conda/a28a88bb4cf34ee22e35486c422e822c_.yaml"
  --prefix "/home/azureuser/tmp/lu7p2kpy92/code/.snakemake/conda/a28a88bb4cf34ee22e35486c422e822c_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working...'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FStainedGlass%2Fqy6p3ww1%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FStainedGlass%2Fqy6p3ww1%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>10.542GiB</td>
						<td>1233.0 seconds</td>
						<td>1255.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738267280">
								<a href="https://github.com/mrvollger/StainedGlass">mrvollger/StainedGlass</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/StainedGlass/tree/v0.3">v0.3</a>
						</td>
						<td>snakemake</td>
						<td>451 days</td>
						<td>2023-02-19 03:31:04</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/aztp3auhq0/code/workflow/envs/env.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/aztp3auhq0/code/.snakemake/conda/0c9419fb4ec9ff90b94ace2d98654d71_.yaml"
  --prefix "/home/azureuser/tmp/aztp3auhq0/code/.snakemake/conda/0c9419fb4ec9ff90b94ace2d98654d71_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working...'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FStainedGlass%2Fvwalw1dg%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FStainedGlass%2Fvwalw1dg%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>8.724GiB</td>
						<td>1107.3 seconds</td>
						<td>1493.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743118864">
								<a href="https://nf-co.re/mag">mag</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mag/tree/2.1.1">2.1.1</a>
						</td>
						<td>nextflow</td>
						<td>450 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-19 07:00:17.350
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2F9lm6ph9o%2Flogs.tar.xz">0.121GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2F9lm6ph9o%2Foutput.tar.xz">0.018GiB</a>
						</td>
						<td>1.030GiB</td>
						<td>1400.4 seconds</td>
						<td>3601.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743146112">
								<a href="https://nf-co.re/nanoseq">nanoseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/nanoseq/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>449 days</td>
						<td>2023-02-19 03:35:21</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-19 04:35:21.554
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nanoseq%2Fs1s730vh%2Flogs.tar.xz">0.217GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nanoseq%2Fs1s730vh%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>1.046GiB</td>
						<td>704.3 seconds</td>
						<td>3602.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743145872">
								<a href="https://nf-co.re/nanoseq">nanoseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/nanoseq/tree/2.0.1">2.0.1</a>
						</td>
						<td>nextflow</td>
						<td>447 days</td>
						<td>2023-02-20 04:31:43</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867577368\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\tat nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\
  \tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\tat nextflow.processor.TaskRun$isContainerEnabled$3.call(Unknown\
  \ Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)\n\
  \tat nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867577368\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 07:31:43.211 [SIGTERM handler] DEBUG nextflow.Session\
  \ - Session aborted -- Cause: SIGTERM\nFeb-20 07:31:43.212 [Actor Thread 2] DEBUG\
  \ nextflow.Session - Session aborted -- Cause: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867577368\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \nFeb-20 07:31:43.214 [Actor Thread 2] DEBUG nextflow.Session - The following nodes\
  \ are still active:\n[process] NFCORE_NANOSEQ:NANOSEQ:GET_NANOLYSE_FASTA\n  status=ACTIVE\n\
  \  port 0: (cntrl) -     ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:NANOLYSE\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) OPEN \
  \ ; channel: fasta\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:QCFASTQ_NANOPLOT_FASTQC:NANOPLOT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:QCFASTQ_NANOPLOT_FASTQC:FASTQC\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:PREPARE_GENOME:GET_CHROM_SIZES\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:PREPARE_GENOME:GTF2BED\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:MINIMAP2_INDEX\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:MINIMAP2_ALIGN\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:SAMTOOLS_VIEW_BAM\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_STATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_FLAGSTAT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_IDXSTATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:BEDTOOLS_UCSC_BIGWIG:BEDTOOLS_GENOMECOV\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:BEDTOOLS_UCSC_BIGWIG:UCSC_BEDGRAPHTOBIGWIG\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:BEDTOOLS_UCSC_BIGBED:BEDTOOLS_BAMBED\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:BEDTOOLS_UCSC_BIGBED:UCSC_BED12TOBIGBED\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:CUSTOM_DUMPSOFTWAREVERSIONS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: versions\n  port 1: (cntrl)\
  \ -     ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:MULTIQC\n  status=ACTIVE\n\
  \  port 0: (value) bound ; channel: ch_multiqc_config\n  port 1: (value) bound ;\
  \ channel: ch_multiqc_custom_config\n  port 2: (value) bound ; channel: ch_pycoqc_multiqc\n\
  \  port 3: (queue) OPEN  ; channel: ch_fastqc_multiqc\n  port 4: (value) OPEN  ;\
  \ channel: ch_sortbam_stats_multiqc\n  port 5: (queue) OPEN  ; channel: ch_featurecounts_gene_multiqc\n\
  \  port 6: (queue) OPEN  ; channel: ch_featurecounts_transcript_multiqc\n  port\
  \ 7: (value) OPEN  ; channel: software_versions_yaml\n  port 8: (queue) OPEN  ;\
  \ channel: workflow_summary\n  port 9: (cntrl) -     ; channel: $\n\nFeb-20 07:31:43.217\
  \ [SIGTERM handler] DEBUG nextflow.Session - The following nodes are still active:\n\
  [process] NFCORE_NANOSEQ:NANOSEQ:GET_NANOLYSE_FASTA\n  status=ACTIVE\n  port 0:\
  \ (cntrl) -     ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:NANOLYSE\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) OPEN  ; channel: fasta\n\
  \  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:QCFASTQ_NANOPLOT_FASTQC:NANOPLOT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:QCFASTQ_NANOPLOT_FASTQC:FASTQC\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:PREPARE_GENOME:GET_CHROM_SIZES\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:PREPARE_GENOME:GTF2BED\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:MINIMAP2_INDEX\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:MINIMAP2_ALIGN\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:SAMTOOLS_VIEW_BAM\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_STATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_FLAGSTAT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:ALIGN_MINIMAP2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_IDXSTATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:BEDTOOLS_UCSC_BIGWIG:BEDTOOLS_GENOMECOV\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:BEDTOOLS_UCSC_BIGWIG:UCSC_BEDGRAPHTOBIGWIG\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:BEDTOOLS_UCSC_BIGBED:BEDTOOLS_BAMBED\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:BEDTOOLS_UCSC_BIGBED:UCSC_BED12TOBIGBED\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:CUSTOM_DUMPSOFTWAREVERSIONS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: versions\n  port 1: (cntrl)\
  \ -     ; channel: $\n\n[process] NFCORE_NANOSEQ:NANOSEQ:MULTIQC\n  status=ACTIVE\n\
  \  port 0: (value) bound ; channel: ch_multiqc_config\n  port 1: (value) bound ;\
  \ channel: ch_multiqc_custom_config\n  port 2: (value) bound ; channel: ch_pycoqc_multiqc\n\
  \  port 3: (queue) OPEN  ; channel: ch_fastqc_multiqc\n  port 4: (value) OPEN  ;\
  \ channel: ch_sortbam_stats_multiqc\n  port 5: (queue) OPEN  ; channel: ch_featurecounts_gene_multiqc\n\
  \  port 6: (queue) OPEN  ; channel: ch_featurecounts_transcript_multiqc\n  port\
  \ 7: (value) OPEN  ; channel: software_versions_yaml\n  port 8: (queue) OPEN  ;\
  \ channel: workflow_summary\n  port 9: (cntrl) -     ; channel: $\n\nFeb-20 07:31:43.219\
  \ [main] DEBUG nextflow.Session - Session await > all processes finished\nFeb-20\
  \ 07:31:43.219 [main] DEBUG nextflow.Session - Session await > all barriers passed\n\
  Feb-20 07:31:43.230 [main] INFO  nextflow.Nextflow - -\e[0;35m[nf-core/nanoseq]\e\
  [1;91m Pipeline completed with errors\e[0m-\nFeb-20 07:31:43.235 [main] DEBUG nextflow.trace.WorkflowStatsObserver\
  \ - Workflow completed > WorkflowStats[succeededCount=5; failedCount=0; ignoredCount=0;\
  \ cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0;\
  \ abortedCount=0; succeedDuration=13m 23s; failedDuration=0ms; cachedDuration=0ms;loadCpus=0;\
  \ loadMemory=0; peakRunning=2; peakCpus=4; peakMemory=12 GB; ]\nFeb-20 07:31:43.235\
  \ [main] DEBUG nextflow.trace.TraceFileObserver - Workflow completed -- saving trace\
  \ file\nFeb-20 07:31:43.238 [main] DEBUG nextflow.trace.ReportObserver - Workflow\
  \ completed -- rendering execution report\nFeb-20 07:31:43.294 [Actor Thread 52]\
  \ DEBUG nextflow.sort.BigSort - Sort completed -- entries: 2; slices: 1; internal\
  \ sort time: 0.013 s; external sort time: 0.0 s; total time: 0.013 s\nFeb-20 07:31:43.306\
  \ [Actor Thread 52] DEBUG nextflow.file.FileCollector - Saved collect-files list\
  \ to: /home/azureuser/tmp/mmaeeh0a2r/singularity-tmp/c0d365a557d16d3bd84f25efd8cf0280.collect-file\n\
  Feb-20 07:31:43.319 [main] DEBUG nextflow.trace.ReportObserver - Execution report\
  \ summary data:\n  [{\"cpuUsage\":{\"mean\":44.45,\"min\":44.45,\"q1\":44.45,\"\
  q2\":44.45,\"q3\":44.45,\"max\":44.45,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\"},\"process\":\"SAMPLESHEET_CHECK\",\"mem\":{\"mean\"\
  :7663616,\"min\":7663616,\"q1\":7663616,\"q2\":7663616,\"q3\":7663616,\"max\":7663616,\"\
  minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_bc_dx.csv)\"\
  ,\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_bc_dx.csv)\"\
  ,\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_bc_dx.csv)\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_bc_dx.csv)\"\
  ,\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_bc_dx.csv)\"\
  },\"memUsage\":{\"mean\":0.12,\"min\":0.12,\"q1\":0.12,\"q2\":0.12,\"q3\":0.12,\"\
  max\":0.12,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_bc_dx.csv)\"\
  ,\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_bc_dx.csv)\"\
  ,\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_bc_dx.csv)\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_bc_dx.csv)\"\
  ,\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_bc_dx.csv)\"\
  },\"timeUsage\":null,\"vmem\":{\"mean\":31895552,\"min\":31895552,\"q1\":31895552,\"\
  q2\":31895552,\"q3\":31895552,\"max\":31895552,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\"},\"reads\":{\"mean\":1475887,\"min\":1475887,\"q1\"\
  :1475887,\"q2\":1475887,\"q3\":1475887,\"max\":1475887,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\"},\"cpu\":{\"mean\":88.9,\"min\":88.9,\"q1\":88.9,\"\
  q2\":88.9,\"q3\":88.9,\"max\":88.9,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\"},\"time\":null,\"writes\":{\"mean\":419,\"min\":419,\"\
  q1\":419,\"q2\":419,\"q3\":419,\"max\":419,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_bc_dx.csv)\"}},{\"cpuUsage\":{\"mean\":14.7,\"min\":14.7,\"q1\":14.7,\"\
  q2\":14.7,\"q3\":14.7,\"max\":14.7,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  },\"process\":\"GET_TEST_DATA\",\"mem\":{\"mean\":108552192,\"min\":108552192,\"\
  q1\":108552192,\"q2\":108552192,\"q3\":108552192,\"max\":108552192,\"minLabel\"\
  :\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"},\"memUsage\":{\"mean\":1.68,\"\
  min\":1.68,\"q1\":1.68,\"q2\":1.68,\"q3\":1.68,\"max\":1.68,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  },\"timeUsage\":{\"mean\":0.11,\"min\":0.11,\"q1\":0.11,\"q2\":0.11,\"q3\":0.11,\"\
  max\":0.11,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"maxLabel\":\"\
  NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  },\"vmem\":{\"mean\":431677440,\"min\":431677440,\"q1\":431677440,\"q2\":431677440,\"\
  q3\":431677440,\"max\":431677440,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  },\"reads\":{\"mean\":288631785,\"min\":288631785,\"q1\":288631785,\"q2\":288631785,\"\
  q3\":288631785,\"max\":288631785,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  },\"cpu\":{\"mean\":29.4,\"min\":29.4,\"q1\":29.4,\"q2\":29.4,\"q3\":29.4,\"max\"\
  :29.4,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"},\"time\":{\"mean\":15511,\"\
  min\":15511,\"q1\":15511,\"q2\":15511,\"q3\":15511,\"max\":15511,\"minLabel\":\"\
  NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"},\"writes\":{\"mean\":632205373,\"\
  min\":632205373,\"q1\":632205373,\"q2\":632205373,\"q3\":632205373,\"max\":632205373,\"\
  minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"\
  ,\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GET_TEST_DATA\"}},{\"cpuUsage\":{\"mean\"\
  :50,\"min\":50,\"q1\":50,\"q2\":50,\"q3\":50,\"max\":50,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\"},\"process\":\"GUPPY\",\"mem\":{\"mean\":221347840,\"min\":221347840,\"\
  q1\":221347840,\"q2\":221347840,\"q3\":221347840,\"max\":221347840,\"minLabel\"\
  :\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\"},\"memUsage\":{\"mean\"\
  :3.44,\"min\":3.44,\"q1\":3.44,\"q2\":3.44,\"q3\":3.44,\"max\":3.44,\"minLabel\"\
  :\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\"},\"timeUsage\":{\"mean\"\
  :1.21,\"min\":1.21,\"q1\":1.21,\"q2\":1.21,\"q3\":1.21,\"max\":1.21,\"minLabel\"\
  :\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\"},\"vmem\":{\"mean\":1825701888,\"\
  min\":1825701888,\"q1\":1825701888,\"q2\":1825701888,\"q3\":1825701888,\"max\":1825701888,\"\
  minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\"},\"reads\":{\"mean\":253098613,\"\
  min\":253098613,\"q1\":253098613,\"q2\":253098613,\"q3\":253098613,\"max\":253098613,\"\
  minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\"},\"cpu\":{\"mean\":100,\"\
  min\":100,\"q1\":100,\"q2\":100,\"q3\":100,\"max\":100,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\"},\"time\":{\"mean\":347907,\"min\":347907,\"q1\":347907,\"q2\":347907,\"\
  q3\":347907,\"max\":347907,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"\
  maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\"},\"writes\":{\"mean\":656887,\"min\":656887,\"q1\":656887,\"q2\":656887,\"\
  q3\":656887,\"max\":656887,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"\
  maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY (1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:GUPPY\
  \ (1)\"}},{\"cpuUsage\":{\"mean\":52.3,\"min\":52.3,\"q1\":52.3,\"q2\":52.3,\"q3\"\
  :52.3,\"max\":52.3,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\"},\"process\":\"PYCOQC\",\"mem\":{\"mean\":170938368,\"\
  min\":170938368,\"q1\":170938368,\"q2\":170938368,\"q3\":170938368,\"max\":170938368,\"\
  minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC (sequencing_summary.txt)\"\
  ,\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC (sequencing_summary.txt)\"\
  ,\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC (sequencing_summary.txt)\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC (sequencing_summary.txt)\"\
  ,\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC (sequencing_summary.txt)\"\
  },\"memUsage\":{\"mean\":2.65,\"min\":2.65,\"q1\":2.65,\"q2\":2.65,\"q3\":2.65,\"\
  max\":2.65,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\"},\"timeUsage\":{\"mean\":0.03,\"min\":0.03,\"q1\":0.03,\"\
  q2\":0.03,\"q3\":0.03,\"max\":0.03,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\"},\"vmem\":{\"mean\":677154816,\"min\":677154816,\"\
  q1\":677154816,\"q2\":677154816,\"q3\":677154816,\"max\":677154816,\"minLabel\"\
  :\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC (sequencing_summary.txt)\"\
  ,\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC (sequencing_summary.txt)\"\
  ,\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC (sequencing_summary.txt)\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC (sequencing_summary.txt)\"\
  ,\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC (sequencing_summary.txt)\"\
  },\"reads\":{\"mean\":123614109,\"min\":123614109,\"q1\":123614109,\"q2\":123614109,\"\
  q3\":123614109,\"max\":123614109,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\"},\"cpu\":{\"mean\":104.6,\"min\":104.6,\"q1\":104.6,\"\
  q2\":104.6,\"q3\":104.6,\"max\":104.6,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\"},\"time\":{\"mean\":10000,\"min\":10000,\"q1\":10000,\"\
  q2\":10000,\"q3\":10000,\"max\":10000,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\"},\"writes\":{\"mean\":2917925,\"min\":2917925,\"q1\"\
  :2917925,\"q2\":2917925,\"q3\":2917925,\"max\":2917925,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:PYCOQC\
  \ (sequencing_summary.txt)\"}},{\"cpuUsage\":{\"mean\":13.3,\"min\":13.3,\"q1\"\
  :13.3,\"q2\":13.3,\"q3\":13.3,\"max\":13.3,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\"},\"process\":\"NANOPLOT\",\"mem\":{\"mean\":2381045760,\"\
  min\":2381045760,\"q1\":2381045760,\"q2\":2381045760,\"q3\":2381045760,\"max\":2381045760,\"\
  minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT (CALIBRATION_RUN_R1)\"\
  ,\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT (CALIBRATION_RUN_R1)\"\
  ,\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT (CALIBRATION_RUN_R1)\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT (CALIBRATION_RUN_R1)\"\
  ,\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT (CALIBRATION_RUN_R1)\"\
  },\"memUsage\":{\"mean\":36.96,\"min\":36.96,\"q1\":36.96,\"q2\":36.96,\"q3\":36.96,\"\
  max\":36.96,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\"},\"timeUsage\":{\"mean\":0.19,\"min\":0.19,\"q1\":0.19,\"\
  q2\":0.19,\"q3\":0.19,\"max\":0.19,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\"},\"vmem\":{\"mean\":71028002816,\"min\":71028002816,\"\
  q1\":71028002816,\"q2\":71028002816,\"q3\":71028002816,\"max\":71028002816,\"minLabel\"\
  :\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT (CALIBRATION_RUN_R1)\"\
  ,\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT (CALIBRATION_RUN_R1)\"\
  ,\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT (CALIBRATION_RUN_R1)\"\
  ,\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT (CALIBRATION_RUN_R1)\"\
  ,\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT (CALIBRATION_RUN_R1)\"\
  },\"reads\":{\"mean\":101423402,\"min\":101423402,\"q1\":101423402,\"q2\":101423402,\"\
  q3\":101423402,\"max\":101423402,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\"},\"cpu\":{\"mean\":26.6,\"min\":26.6,\"q1\":26.6,\"q2\"\
  :26.6,\"q3\":26.6,\"max\":26.6,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\"},\"time\":{\"mean\":28000,\"min\":28000,\"q1\":28000,\"\
  q2\":28000,\"q3\":28000,\"max\":28000,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\"},\"writes\":{\"mean\":1168439,\"min\":1168439,\"q1\":1168439,\"\
  q2\":1168439,\"q3\":1168439,\"max\":1168439,\"minLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"maxLabel\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q1Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q2Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\",\"q3Label\":\"NFCORE_NANOSEQ:NANOSEQ:QCBASECALL_PYCOQC_NANOPLOT:NANOPLOT\
  \ (CALIBRATION_RUN_R1)\"}}]\nFeb-20 07:31:43.333 [Actor Thread 52] DEBUG nextflow.file.FileCollector\
  \ - Deleting file collector temp dir: /home/azureuser/tmp/mmaeeh0a2r/singularity-tmp/nxf-6564123014163695124\n\
  Feb-20 07:31:43.960 [main] DEBUG nextflow.trace.TimelineObserver - Workflow completed\
  \ -- rendering execution timeline\nFeb-20 07:31:44.093 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 07:31:44.111 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nanoseq%2F5kwe5cqt%2Flogs.tar.xz">0.217GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nanoseq%2F5kwe5cqt%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>1.044GiB</td>
						<td>687.7 seconds</td>
						<td>10800.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738261040">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.3.1">v0.3.1</a>
						</td>
						<td>snakemake</td>
						<td>446 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 29
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/i1nsuyq5ji/code/workflow/Snakefile\", line 29, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F9a952h5n%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F9a952h5n%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>2.0 seconds</td>
						<td>4.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743104464">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.4.1">2.4.1</a>
						</td>
						<td>nextflow</td>
						<td>446 days</td>
						<td>2023-02-19 04:30:43</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F9oxapupl%2Flogs.tar.xz">0.015GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2F9oxapupl%2Foutput.tar.xz">0.012GiB</a>
						</td>
						<td>1.678GiB</td>
						<td>703.7 seconds</td>
						<td>452.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741281120">
								<a href="https://github.com/thomasbtf/sars-cov-2-wbe">thomasbtf/sars-cov-2-wbe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/thomasbtf/sars-cov-2-wbe/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>445 days</td>
						<td>2023-02-19 04:41:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 7
rest: "The following environment variables are requested by the workflow but undefined.\
  \ Please make sure that they are correctly defined before running Snakemake:\nGISAID_API_TOKEN\n\
  \  File \"/home/azureuser/tmp/qz999ga4yd/code/workflow/Snakefile\", line 7, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/thomasbtf%252Fsars-cov-2-wbe%2Fz6kzzaqb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/thomasbtf%252Fsars-cov-2-wbe%2Fz6kzzaqb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.3 seconds</td>
						<td>2.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743111184">
								<a href="https://nf-co.re/fetchngs">fetchngs</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/fetchngs/tree/1.5">1.5</a>
						</td>
						<td>nextflow</td>
						<td>445 days</td>
						<td>2023-02-19 04:21:07</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "echo SRR9984183 > id.txt\nsra_ids_to_runinfo.py \\\n    id.txt\
  \ \\\n    SRR9984183.runinfo.tsv \\\n\n\ncat <<-END_VERSIONS > versions.yml\n\"\
  NFCORE_FETCHNGS:SRA:SRA_IDS_TO_RUNINFO\":\n    python: $(python --version | sed\
  \ 's/Python //g')\nEND_VERSIONS"
error: '[ERROR] The server couldn''t fulfill the request.

  [ERROR] Status: 400 Bad Request. Both list of IDs and query_key are empty'
exit_status: 1
kind: CalledProcessError
output: (empty)
process: NFCORE_FETCHNGS:SRA:SRA_IDS_TO_RUNINFO (SRR9984183)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- dc
- 9d9746424507b2f88edcf5300a1e81

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2F2gc5cz2n%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2F2gc5cz2n%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.660GiB</td>
						<td>42.6 seconds</td>
						<td>35.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738287328">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.3.2">v2.3.2</a>
						</td>
						<td>snakemake</td>
						<td>444 days</td>
						<td>2023-02-19 03:31:04</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/zj0w1txq0k/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/d3d504e3-6f8b-4fca-bfbf-41cb33922409\
  \ singularity  exec --home /home/azureuser/tmp/zj0w1txq0k/code/.test  /home/azureuser/tmp/zj0w1txq0k/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/zj0w1txq0k/code/.test/.snakemake/conda/3aed877f14cee4ba423450a06a10515d_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/zj0w1txq0k/code/.test/.snakemake/conda/3aed877f14cee4ba423450a06a10515d_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining pillow=6.2:   0%|          |\
  \ 0/9 [00:00<?, ?it/s]  \nExamining pandas=0.25:  11%|\u2588         | 1/9 [00:00<00:05,\
  \  1.46it/s]\nExamining pandas=0.25:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:02,\
  \  2.91it/s]\nExamining @/linux-64::__glibc==2.31=0:  22%|\u2588\u2588\u258F   \
  \    | 2/9 [00:03<00:02,  2.91it/s]\nExamining @/linux-64::__glibc==2.31=0:  33%|\u2588\
  \u2588\u2588\u258E      | 3/9 [00:03<00:07,  1.23s/it]\nExamining @/linux-64::__unix==0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:03<00:07,  1.23s/it]    \nExamining\
  \ matplotlib=3.1:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:03<00:06,  1.23s/it]\
  \         \nExamining goatools=0.9.7:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:05<00:04,  1.23s/it]\nExamining goatools=0.9.7:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:05<00:03,  1.03s/it]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:06<00:03,  1.03s/it]\n\
  Examining @/linux-64::__archspec==1=x86_64:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:06<00:01,  1.14it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:06<00:01,  1.14it/s]\
  \   \nExamining python=3.7:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:06<00:00,  1.14it/s]                   \n                     \
  \                                              \n\nDetermining conflicts:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining conflict for pillow pandas matplotlib\
  \ goatools python:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for\
  \ pillow matplotlib:  11%|\u2588         | 1/9 [00:05<00:43,  5.44s/it]        \
  \       \nExamining conflict for pillow matplotlib:  22%|\u2588\u2588\u258F    \
  \   | 2/9 [00:05<00:19,  2.72s/it]\nExamining conflict for pillow python:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:08<00:19,  2.72s/it]    \nExamining conflict for pillow\
  \ python:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:08<00:16,  2.78s/it]\nExamining\
  \ conflict for pillow pandas goatools matplotlib:  33%|\u2588\u2588\u2588\u258E\
  \      | 3/9 [00:09<00:16,  2.78s/it]\nExamining conflict for pillow pandas goatools\
  \ matplotlib:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:09<00:10,  2.05s/it]\n\
  Examining conflict for pillow pandas __glibc matplotlib python:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:15<00:10,  2.05s/it]\nExamining conflict for pillow\
  \ pandas __glibc matplotlib python:  56%|\u2588\u2588\u2588\u2588\u2588\u258C  \
  \  | 5/9 [00:15<00:13,  3.47s/it]\nExamining conflict for python pillow pandas matplotlib:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:19<00:13,  3.47s/it]  \
  \      \nExamining conflict for python pillow pandas matplotlib:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:19<00:10,  3.65s/it]\nExamining conflict\
  \ for pillow goatools python:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B  \
  \ | 6/9 [00:23<00:10,  3.65s/it]         \nExamining conflict for pillow goatools\
  \ python:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:23<00:07,\
  \  3.92s/it]\nExamining conflict for pandas goatools:  78%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258A  | 7/9 [00:25<00:07,  3.92s/it]       \nExamining conflict\
  \ for pandas goatools:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589\
  \ | 8/9 [00:25<00:03,  3.28s/it]\nExamining conflict for pandas goatools matplotlib:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:26<00:03,\
  \  3.28s/it]\nExamining conflict for pandas goatools matplotlib: 100%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:26<00:00,  2.68s/it]\n\
  Examining conflict for pandas matplotlib: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588| 9/9 [00:51<00:00,  2.68s/it]         \nExamining conflict\
  \ for pandas matplotlib: : 10it [00:51,  9.36s/it]                    \nExamining\
  \ conflict for python pandas matplotlib: : 10it [00:52,  9.36s/it]\nExamining conflict\
  \ for python pandas matplotlib: : 11it [00:52,  6.86s/it]\nExamining conflict for\
  \ goatools matplotlib: : 11it [00:52,  6.86s/it]     \nExamining conflict for goatools\
  \ matplotlib: : 12it [00:52,  4.86s/it]\n                                      \
  \                               \nfailed\nSolving environment: ...working... \n\
  Found conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining pillow=6.2:   0%|          | 0/9 [00:00<?, ?it/s]  \nExamining\
  \ pandas=0.25:  11%|\u2588         | 1/9 [00:01<00:08,  1.07s/it]\nExamining pandas=0.25:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:01<00:03,  1.87it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:03<00:03,  1.87it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:03<00:08,  1.41s/it]\nExamining @/linux-64::__unix==0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:03<00:08,  1.41s/it]    \nExamining\
  \ matplotlib=3.1:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:03<00:07,  1.41s/it]\
  \         \nExamining goatools=0.9.7:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:08<00:05,  1.41s/it]\nExamining goatools=0.9.7:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:08<00:04,  1.52s/it]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:08<00:04,  1.52s/it]\n\
  Examining @/linux-64::__archspec==1=x86_64:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:08<00:02,  1.28s/it]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:08<00:02,  1.28s/it]\
  \   \nExamining python=3.7:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2589 | 8/9 [00:08<00:01,  1.28s/it]                   \n                     \
  \                                              \n\nDetermining conflicts:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining conflict for pillow pandas matplotlib\
  \ goatools python:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for\
  \ pillow matplotlib:  11%|\u2588         | 1/9 [00:09<01:14,  9.34s/it]        \
  \       \nExamining conflict for pillow matplotlib:  22%|\u2588\u2588\u258F    \
  \   | 2/9 [00:09<00:32,  4.67s/it]\nExamining conflict for pillow python:  22%|\u2588\
  \u2588\u258F       | 2/9 [00:13<00:32,  4.67s/it]    \nExamining conflict for pillow\
  \ python:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:13<00:26,  4.38s/it]\nExamining\
  \ conflict for pillow pandas goatools matplotlib:  33%|\u2588\u2588\u2588\u258E\
  \      | 3/9 [00:14<00:26,  4.38s/it]\nExamining conflict for pillow pandas goatools\
  \ matplotlib:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:14<00:15,  3.12s/it]\n\
  Examining conflict for pillow pandas __glibc matplotlib python:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:24<00:15,  3.12s/it]\nExamining conflict for pillow\
  \ pandas __glibc matplotlib python:  56%|\u2588\u2588\u2588\u2588\u2588\u258C  \
  \  | 5/9 [00:24<00:21,  5.41s/it]\nExamining conflict for python pillow pandas matplotlib:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:28<00:21,  5.41s/it]  \
  \      \nExamining conflict for python pillow pandas matplotlib:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:28<00:14,  4.95s/it]\nExamining conflict\
  \ for pillow goatools python:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B  \
  \ | 6/9 [00:32<00:14,  4.95s/it]         \nExamining conflict for pillow goatools\
  \ python:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:32<00:09,\
  \  4.88s/it]\nExamining conflict for pandas goatools:  78%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258A  | 7/9 [00:35<00:09,  4.88s/it]       \nExamining conflict\
  \ for pandas goatools:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589\
  \ | 8/9 [00:35<00:04,  4.10s/it]\nExamining conflict for pandas goatools matplotlib:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:36<00:04,\
  \  4.10s/it]\nExamining conflict for pandas goatools matplotlib: 100%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:36<00:00,  3.30s/it]\n\
  Examining conflict for pandas matplotlib: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588| 9/9 [01:04<00:00,  3.30s/it]         \nExamining conflict\
  \ for pandas matplotlib: : 10it [01:04, 10.76s/it]                    \nExamining\
  \ conflict for python pandas matplotlib: : 10it [01:05, 10.76s/it]\nExamining conflict\
  \ for python pandas matplotlib: : 11it [01:05,  7.86s/it]\nExamining conflict for\
  \ goatools matplotlib: : 11it [01:05,  7.86s/it]     \nExamining conflict for goatools\
  \ matplotlib: : 12it [01:05,  5.56s/it]\n                                      \
  \                               \nfailed\n\nUnsatisfiableError: The following specifications\
  \ were found to be incompatible with each other:\n\nOutput in format: Requested\
  \ package -> Available versions\n\nPackage setuptools conflicts for:\npandas=0.25\
  \ -> setuptools[version='<60.0.0']\npython=3.7 -> pip -> setuptools\nmatplotlib=3.1\
  \ -> setuptools\n\nPackage numpy-base conflicts for:\nmatplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0']\
  \ -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0',build='py27h2b20989_6|py27hdbf6ddf_7|py27h2b20989_7|py36hdbf6ddf_7|py36h2b20989_7|py35h2b20989_7|py37h2b20989_7|py27h2b20989_7|py36h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_7|py36hdbf6ddf_8|py36h2b20989_8|py37h2b20989_8|py27h2b20989_8|py27hdbf6ddf_8|py35h2b20989_8|py27h7cdd4dd_9|py37h7cdd4dd_9|py37h3dfced4_9|py35h3dfced4_9|py27h3dfced4_9|py36h81de0dd_9|py27h74e8950_9|py35h74e8950_9|py37h74e8950_10|py35h74e8950_10|py36h74e8950_10|py27h81de0dd_10|py36h81de0dd_10|py27h2f8d375_11|py37hde5b4d6_11|py27hde5b4d6_11|py37hde5b4d6_12|py36hde5b4d6_12|py38h2f8d375_12|py35h0ea5e3f_1|py27h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py35h2b20989_0|py27hdbf6ddf_0|py36h2b20989_0|py27h2b20989_0|py27hdbf6ddf_0|py36h2b20989_1|py36hdbf6ddf_1|py37hdbf6ddf_1|py27h2b20989_1|py36h2b20989_2|py37hdbf6ddf_3|py36hdbf6ddf_3|py27h2b20989_3|py37h2b20989_4|py36h2b20989_4|py36hdbf6ddf_4|py35h2b20989_4|py37h2f8d375_0|py36hde5b4d6_0|py27h81de0dd_4|py36h2f8d375_4|py27h2f8d375_4|py38h2f8d375_4|py27h7cdd4dd_0|py35h7cdd4dd_0|py27h3dfced4_0|py35h3dfced4_0|py35h74e8950_0|py36h74e8950_0|py37h81de0dd_0|py35h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py35h81de0dd_0|py37h2f8d375_0|py36h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_1|py27h2f8d375_1|py37hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_1|py37hde5b4d6_1|py36hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py38hde5b4d6_0|py27hde5b4d6_0|py38h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py38hdc34a94_3|py39hdc34a94_3|py39ha8aedfd_4|py37h73d599e_4|py38h73d599e_4|py38hfa32c7d_0|py36h75fe3a5_0|py37hfa32c7d_0|py39h2ae0177_0|py39h0f7b65f_0|py39h4c65ebe_1|py38h21a3de8_1|py37h34387ca_0|py39h34387ca_0|py38h34387ca_0|py38he2ba247_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py39hfae3a4d_0|py37h74d4b33_0|py38h74d4b33_0|py39h79a1101_0|py37h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py38hb8be1f0_1|py37hb8be1f0_1|py310hf2716ce_1|py39hb8be1f0_2|py38hb8be1f0_2|py37hf524024_2|py37h1e6e340_3|py310hcba007f_3|py39ha15fc14_3|py310h9585f30_0|py38hb8be1f0_0|py39h1e6e340_0|py310h375b286_0|py310hcba007f_0|py38h31eccc5_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_1|py39h1e6e340_1|py310h8e6c178_1|py39h31eccc5_1|py310h375b286_0|py310h375b286_0|py39h1e6e340_0|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py38h31eccc5_1|py310h375b286_1|py310h8e6c178_0|py39h31eccc5_0|py38ha15fc14_0|py39ha15fc14_0|py38h1e6e340_0|py310hf2716ce_0|py39hb8be1f0_0|py38hf524024_0|py39hf524024_0|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py38h1e6e340_3|py38hf524024_2|py39hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py310hf2716ce_2|py310h9585f30_1|py37hf524024_1|py38hf524024_1|py39hf524024_1|py39hb8be1f0_1|py310h79a1101_0|py38h2b8c604_0|py39h2b8c604_0|py38h79a1101_0|py39h39b7dee_0|py37h39b7dee_0|py38h39b7dee_0|py39h74d4b33_0|py38hfae3a4d_0|py39h7d8b39e_0|py38h7d8b39e_0|py37h7d8b39e_0|py39h21a3de8_1|py37h21a3de8_1|py38h4c65ebe_1|py37h4c65ebe_1|py38hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py38h75fe3a5_0|py37h75fe3a5_0|py36hfa32c7d_0|py38h75fe3a5_0|py37hfa32c7d_0|py39h73d599e_4|py38ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py37h41b4c56_3|py39h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py27h2f8d375_0|py38h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_1|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py27h81de0dd_0|py36h2f8d375_0|py27h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py37h81de0dd_1|py36h81de0dd_1|py27h81de0dd_1|py27h2f8d375_1|py36h2f8d375_1|py36h81de0dd_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py27h74e8950_0|py37h3dfced4_0|py36h3dfced4_0|py37h7cdd4dd_0|py36h7cdd4dd_0|py36hde5b4d6_5|py37hde5b4d6_5|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py36h81de0dd_4|py37h81de0dd_4|py35h81de0dd_4|py35h2f8d375_4|py37h2f8d375_4|py37hde5b4d6_0|py36h2f8d375_0|py35hdbf6ddf_4|py37hdbf6ddf_4|py27hdbf6ddf_4|py27h2b20989_4|py27hdbf6ddf_3|py37h2b20989_3|py36h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27hdbf6ddf_2|py37h2b20989_2|py27h2b20989_2|py27hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36hdbf6ddf_0|py36h9be14a7_1|py27h0ea5e3f_1|py36h0ea5e3f_1|py38hde5b4d6_12|py27hde5b4d6_12|py36h2f8d375_12|py27h2f8d375_12|py37h2f8d375_12|py36hde5b4d6_11|py36h2f8d375_11|py37h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py36h2f8d375_10|py37h2f8d375_10|py35h81de0dd_10|py37h81de0dd_10|py27h74e8950_10|py35h81de0dd_9|py27h81de0dd_9|py36h74e8950_9|py37h74e8950_9|py37h81de0dd_9|py36h3dfced4_9|py35h7cdd4dd_9|py36h7cdd4dd_9|py35hdbf6ddf_8|py37hdbf6ddf_8|py37hdbf6ddf_7|py37h2b20989_7|py37hdbf6ddf_7|py35hdbf6ddf_7|py36hdbf6ddf_6|py27hdbf6ddf_6|py37hdbf6ddf_6|py37h2b20989_6|py36h2b20989_6']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> numpy-base[version='1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.17.0|1.17.0|1.17.0|1.17.0',build='py35h0ea5e3f_1|py27h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py36h2b20989_0|py35h2b20989_0|py27hdbf6ddf_0|py36h2b20989_0|py27h2b20989_0|py27hdbf6ddf_0|py36h2b20989_1|py36hdbf6ddf_1|py37hdbf6ddf_1|py27h2b20989_1|py36h2b20989_2|py37hdbf6ddf_3|py36hdbf6ddf_3|py27h2b20989_3|py37h2b20989_4|py36h2b20989_4|py36hdbf6ddf_4|py35h2b20989_4|py37h2f8d375_0|py36hde5b4d6_0|py27h81de0dd_4|py36h2f8d375_4|py27h2f8d375_4|py38h2f8d375_4|py27h7cdd4dd_0|py35h7cdd4dd_0|py27h3dfced4_0|py35h3dfced4_0|py35h74e8950_0|py36h74e8950_0|py37h81de0dd_0|py35h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py35h81de0dd_0|py37h2f8d375_0|py36h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_1|py27h2f8d375_1|py37hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_1|py37hde5b4d6_1|py36hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py38hde5b4d6_0|py27hde5b4d6_0|py38h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py38hdc34a94_3|py39hdc34a94_3|py39ha8aedfd_4|py37h73d599e_4|py38h73d599e_4|py38hfa32c7d_0|py36h75fe3a5_0|py37hfa32c7d_0|py39h2ae0177_0|py39h0f7b65f_0|py39h4c65ebe_1|py38h21a3de8_1|py37h34387ca_0|py39h34387ca_0|py38h34387ca_0|py38he2ba247_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py39hfae3a4d_0|py37h74d4b33_0|py38h74d4b33_0|py39h79a1101_0|py37h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py38hb8be1f0_1|py37hb8be1f0_1|py310hf2716ce_1|py39hb8be1f0_2|py38hb8be1f0_2|py37hf524024_2|py37h1e6e340_3|py310hcba007f_3|py39ha15fc14_3|py310h9585f30_0|py38hb8be1f0_0|py39h1e6e340_0|py310h375b286_0|py310hcba007f_0|py38h31eccc5_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_1|py39h1e6e340_1|py310h8e6c178_1|py39h31eccc5_1|py310h375b286_0|py310h375b286_0|py39h1e6e340_0|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py38h31eccc5_1|py310h375b286_1|py310h8e6c178_0|py39h31eccc5_0|py38ha15fc14_0|py39ha15fc14_0|py38h1e6e340_0|py310hf2716ce_0|py39hb8be1f0_0|py38hf524024_0|py39hf524024_0|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py38h1e6e340_3|py38hf524024_2|py39hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py310hf2716ce_2|py310h9585f30_1|py37hf524024_1|py38hf524024_1|py39hf524024_1|py39hb8be1f0_1|py310h79a1101_0|py38h2b8c604_0|py39h2b8c604_0|py38h79a1101_0|py39h39b7dee_0|py37h39b7dee_0|py38h39b7dee_0|py39h74d4b33_0|py38hfae3a4d_0|py39h7d8b39e_0|py38h7d8b39e_0|py37h7d8b39e_0|py39h21a3de8_1|py37h21a3de8_1|py38h4c65ebe_1|py37h4c65ebe_1|py38hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py38h75fe3a5_0|py37h75fe3a5_0|py36hfa32c7d_0|py38h75fe3a5_0|py37hfa32c7d_0|py39h73d599e_4|py38ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py37h41b4c56_3|py39h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py27h2f8d375_0|py38h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_1|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py27h81de0dd_0|py36h2f8d375_0|py27h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py37h81de0dd_1|py36h81de0dd_1|py27h81de0dd_1|py27h2f8d375_1|py36h2f8d375_1|py36h81de0dd_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py27h74e8950_0|py37h3dfced4_0|py36h3dfced4_0|py37h7cdd4dd_0|py36h7cdd4dd_0|py36hde5b4d6_5|py37hde5b4d6_5|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py36h81de0dd_4|py37h81de0dd_4|py35h81de0dd_4|py35h2f8d375_4|py37h2f8d375_4|py37hde5b4d6_0|py36h2f8d375_0|py35hdbf6ddf_4|py37hdbf6ddf_4|py27hdbf6ddf_4|py27h2b20989_4|py27hdbf6ddf_3|py37h2b20989_3|py36h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py27hdbf6ddf_2|py37h2b20989_2|py27h2b20989_2|py27hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py36hdbf6ddf_0|py36h9be14a7_1|py27h0ea5e3f_1|py36h0ea5e3f_1']\n\
  goatools=0.9.7 -> numpy -> numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.1|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.19.2|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.1|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.2|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.20.3|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.2|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.21.5|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.22.3|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.1|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.3|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.4|1.23.5|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|1.9.3|>=1.9.3,<2.0a0|1.17.0|1.17.0|1.17.0|1.17.0',build='py37h2f8d375_0|py36hde5b4d6_0|py27h2b20989_6|py36h2b20989_6|py37h2b20989_6|py36hdbf6ddf_6|py27h2b20989_7|py36hdbf6ddf_7|py36h2b20989_7|py35hdbf6ddf_7|py37h2b20989_7|py27h2b20989_7|py36h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_7|py36hdbf6ddf_8|py36h2b20989_8|py27h2b20989_8|py35h7cdd4dd_9|py27h7cdd4dd_9|py37h7cdd4dd_9|py37h3dfced4_9|py36h3dfced4_9|py27h3dfced4_9|py36h81de0dd_9|py37h81de0dd_9|py35h74e8950_9|py37h74e8950_10|py36h74e8950_10|py27h81de0dd_10|py37h81de0dd_10|py37h2f8d375_11|py27h2f8d375_11|py36hde5b4d6_12|py38hde5b4d6_12|py35h0ea5e3f_1|py27h2b20989_0|py36h2b20989_0|py27hdbf6ddf_0|py36h2b20989_0|py27h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_1|py36hdbf6ddf_1|py37hdbf6ddf_1|py36h2b20989_2|py27hdbf6ddf_2|py37hdbf6ddf_3|py36hdbf6ddf_3|py27h2b20989_3|py36h2b20989_4|py36hdbf6ddf_4|py35hdbf6ddf_4|py35h2b20989_4|py27h81de0dd_4|py36h2f8d375_4|py27h2f8d375_4|py38h2f8d375_4|py27h7cdd4dd_0|py35h7cdd4dd_0|py27h3dfced4_0|py35h3dfced4_0|py35h74e8950_0|py36h74e8950_0|py37h81de0dd_0|py35h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py35h81de0dd_0|py37h2f8d375_0|py36h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py36h81de0dd_0|py27hde5b4d6_0|py36hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_1|py27h2f8d375_1|py37hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_1|py37hde5b4d6_1|py36hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py27h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py38hde5b4d6_0|py27hde5b4d6_0|py38h41b4c56_3|py36h41b4c56_3|py36hdc34a94_3|py38hdc34a94_3|py39hdc34a94_3|py39ha8aedfd_4|py37h73d599e_4|py38h73d599e_4|py38hfa32c7d_0|py36h75fe3a5_0|py37hfa32c7d_0|py39h2ae0177_0|py39h0f7b65f_0|py39h4c65ebe_1|py38h21a3de8_1|py37h34387ca_0|py39h34387ca_0|py38h34387ca_0|py38he2ba247_0|py39he2ba247_0|py37hfae3a4d_0|py37he2ba247_0|py39hfae3a4d_0|py37h74d4b33_0|py38h74d4b33_0|py39h79a1101_0|py37h2b8c604_0|py37h79a1101_0|py310h2b8c604_0|py38hb8be1f0_1|py37hb8be1f0_1|py310hf2716ce_1|py39hb8be1f0_2|py38hb8be1f0_2|py37hf524024_2|py37h1e6e340_3|py310hcba007f_3|py39ha15fc14_3|py310h9585f30_0|py38hb8be1f0_0|py39h1e6e340_0|py310h375b286_0|py310hcba007f_0|py38h31eccc5_0|py38h1e6e340_0|py39h1e6e340_0|py310h375b286_0|py38h1e6e340_1|py39h1e6e340_1|py310h8e6c178_1|py39h31eccc5_1|py310h375b286_0|py310h375b286_0|py39h1e6e340_0|py38h31eccc5_0|py39h31eccc5_0|py310h8e6c178_0|py38h1e6e340_0|py39h1e6e340_0|py38h1e6e340_0|py310h8e6c178_0|py39h31eccc5_0|py38h31eccc5_0|py38h31eccc5_1|py310h375b286_1|py310h8e6c178_0|py39h31eccc5_0|py38ha15fc14_0|py39ha15fc14_0|py38h1e6e340_0|py310hf2716ce_0|py39hb8be1f0_0|py38hf524024_0|py39hf524024_0|py38ha15fc14_3|py37ha15fc14_3|py39h1e6e340_3|py310h375b286_3|py38h1e6e340_3|py38hf524024_2|py39hf524024_2|py310h9585f30_2|py37hb8be1f0_2|py310hf2716ce_2|py310h9585f30_1|py37hf524024_1|py38hf524024_1|py39hf524024_1|py39hb8be1f0_1|py310h79a1101_0|py38h2b8c604_0|py39h2b8c604_0|py38h79a1101_0|py39h39b7dee_0|py37h39b7dee_0|py38h39b7dee_0|py39h74d4b33_0|py38hfae3a4d_0|py39h7d8b39e_0|py38h7d8b39e_0|py37h7d8b39e_0|py39h21a3de8_1|py37h21a3de8_1|py38h4c65ebe_1|py37h4c65ebe_1|py38hfa32c7d_0|py36hfa32c7d_0|py36h75fe3a5_0|py37h75fe3a5_0|py38h75fe3a5_0|py37h75fe3a5_0|py36hfa32c7d_0|py38h75fe3a5_0|py37hfa32c7d_0|py39h73d599e_4|py38ha8aedfd_4|py37ha8aedfd_4|py37hdc34a94_3|py37h41b4c56_3|py39h41b4c56_3|py39h76555f2_1|py39hfb011de_1|py27h2f8d375_0|py38h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py27hde5b4d6_0|py37hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py27h2f8d375_0|py27hde5b4d6_0|py36h2f8d375_0|py27hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py36h2f8d375_0|py36hde5b4d6_1|py27hde5b4d6_1|py36h2f8d375_1|py27hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py27h2f8d375_0|py27h81de0dd_0|py36h2f8d375_0|py27h81de0dd_0|py36h2f8d375_0|py27h2f8d375_0|py37h2f8d375_1|py37h81de0dd_1|py36h81de0dd_1|py27h81de0dd_1|py27h2f8d375_1|py36h2f8d375_1|py36h81de0dd_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py37h81de0dd_0|py27h2f8d375_0|py37h2f8d375_0|py36h2f8d375_0|py35h2f8d375_0|py27h81de0dd_0|py36h81de0dd_0|py37h74e8950_0|py27h74e8950_0|py37h3dfced4_0|py36h3dfced4_0|py37h7cdd4dd_0|py36h7cdd4dd_0|py36hde5b4d6_5|py37hde5b4d6_5|py27hde5b4d6_5|py27h2f8d375_5|py37h2f8d375_5|py36h2f8d375_5|py38hde5b4d6_4|py36h81de0dd_4|py37h81de0dd_4|py35h81de0dd_4|py35h2f8d375_4|py37h2f8d375_4|py37hdbf6ddf_4|py37h2b20989_4|py27hdbf6ddf_4|py27h2b20989_4|py27hdbf6ddf_3|py37h2b20989_3|py36h2b20989_3|py36hdbf6ddf_2|py37hdbf6ddf_2|py37h2b20989_2|py27h2b20989_2|py27h2b20989_1|py27hdbf6ddf_1|py37h2b20989_1|py35hdbf6ddf_0|py36hdbf6ddf_0|py35h2b20989_0|py35h9be14a7_1|py27h9be14a7_1|py36h9be14a7_1|py27h0ea5e3f_1|py36h0ea5e3f_1|py38h2f8d375_12|py37hde5b4d6_12|py27hde5b4d6_12|py36h2f8d375_12|py27h2f8d375_12|py37h2f8d375_12|py27hde5b4d6_11|py37hde5b4d6_11|py36hde5b4d6_11|py36h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py36h2f8d375_10|py37h2f8d375_10|py35h81de0dd_10|py36h81de0dd_10|py27h74e8950_10|py35h74e8950_10|py35h81de0dd_9|py27h81de0dd_9|py36h74e8950_9|py37h74e8950_9|py27h74e8950_9|py35h3dfced4_9|py36h7cdd4dd_9|py35h2b20989_8|py35hdbf6ddf_8|py27hdbf6ddf_8|py37h2b20989_8|py37hdbf6ddf_8|py37hdbf6ddf_7|py37h2b20989_7|py37hdbf6ddf_7|py35h2b20989_7|py27hdbf6ddf_7|py27hdbf6ddf_6|py37hdbf6ddf_6|py37hde5b4d6_0|py36h2f8d375_0']\n\
  \nPackage pandas conflicts for:\npandas=0.25\ngoatools=0.9.7 -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\n\
  \nPackage pytz conflicts for:\nmatplotlib=3.1 -> pytz\npandas=0.25 -> pytz[version='>=2017.2']\n\
  \nPackage python conflicts for:\npython=3.7\npandas=0.25 -> numpy[version='>=1.14.6,<2.0a0']\
  \ -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|>=3.6|>=3|3.4.*']\n\
  pillow=6.2 -> olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  goatools=0.9.7 -> python\npillow=6.2 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  matplotlib=3.1 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  goatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  matplotlib=3.1 -> pyqt -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.6|>=3.0,<=3.9|>=3.7|>=3.8|>=3']\n\
  pandas=0.25 -> python[version='>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']\n\n\
  Package pypy3.6 conflicts for:\npillow=6.2 -> python[version='>=3.6,<3.7.0a0'] ->\
  \ pypy3.6[version='7.3.*|7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*']\npillow=6.2 -> pypy3.6[version='>=7.3.1']\n\
  \nPackage libgcc conflicts for:\ngoatools=0.9.7 -> scipy -> libgcc\nmatplotlib=3.1\
  \ -> pyqt -> libgcc\n\nPackage ca-certificates conflicts for:\ngoatools=0.9.7 ->\
  \ python -> ca-certificates\npython=3.7 -> openssl[version='>=1.1.1l,<1.1.2a'] ->\
  \ ca-certificates\npillow=6.2 -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n\
  \nPackage libpng conflicts for:\npillow=6.2 -> freetype[version='>=2.9.1,<3.0a0']\
  \ -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.37,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  matplotlib=3.1 -> libpng[version='>=1.6.37,<1.7.0a0']\nmatplotlib=3.1 -> matplotlib-base==3.1.1=py36he7580a8_1\
  \ -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\n\
  Package graalpy conflicts for:\ngoatools=0.9.7 -> python -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\
  matplotlib=3.1 -> python[version='>=3.8,<3.9.0a0'] -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\
  pillow=6.2 -> python[version='>=3.8,<3.9.0a0'] -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\
  pandas=0.25 -> python[version='>=3.8,<3.9.0a0'] -> graalpy[version='>=22.3.0,<22.3.1.0a0']\n\
  \nPackage _openmp_mutex conflicts for:\nmatplotlib=3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> _openmp_mutex[version='>=4.5']\npillow=6.2 -> libgcc-ng[version='>=7.5.0']\
  \ -> _openmp_mutex[version='>=4.5']\npython=3.7 -> libgcc-ng[version='>=9.4.0']\
  \ -> _openmp_mutex[version='>=4.5']\npandas=0.25 -> libgcc-ng[version='>=7.3.0']\
  \ -> _openmp_mutex[version='>=4.5']\n\nPackage gdbm conflicts for:\npython=3.7 ->\
  \ pypy3.7=7.3.7 -> gdbm[version='>=1.18,<1.19.0a0']\npillow=6.2 -> pypy3.6[version='>=7.3.1']\
  \ -> gdbm[version='>=1.18,<1.19.0a0']The following specifications were found to\
  \ be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - matplotlib=3.1 -> libgcc-ng[version='>=7.3.0']\
  \ -> __glibc[version='>=2.17']\n  - pandas=0.25 -> libgcc-ng[version='>=7.3.0']\
  \ -> __glibc[version='>=2.17']\n  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] ->\
  \ __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Faorh78k4%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Faorh78k4%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.239GiB</td>
						<td>1646.5 seconds</td>
						<td>2445.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738260800">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.4.0">v0.4.0</a>
						</td>
						<td>snakemake</td>
						<td>443 days</td>
						<td>2023-02-19 04:54:52</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- common.smk
kind: ValueError
line_no: 37
rest: "The truth value of an array with more than one element is ambiguous. Use a.any()\
  \ or a.all()\n  File \"/home/azureuser/tmp/i55b9wj2lb/code/workflow/Snakefile\"\
  , line 33, in <module>\n  File \"/home/azureuser/tmp/i55b9wj2lb/code/workflow/rules/common.smk\"\
  , line 37, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fz4ww82bs%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fz4ww82bs%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.104GiB</td>
						<td>2.0 seconds</td>
						<td>2.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738205408">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.1">1.4.1</a>
						</td>
						<td>snakemake</td>
						<td>440 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 10tx4adb6o
- code
- workflow
- rules
- blast.smk
kind: CalledProcessError
line_no: 20
rest: "Command 'set -euo pipefail;  /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/python3.10\
  \ /home/azureuser/tmp/10tx4adb6o/code/.tests/.snakemake/scripts/tmpt0nee8rk.filter_taxonomy.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/10tx4adb6o/code/workflow/rules/blast.smk\"\
  , line 20, in __rule_prep_taxonomy\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nActivating conda environment: .snakemake/conda/af63aadfae1a7ddeae0ce6b2f2ffebe2_\n\
  [Sun Feb 19 06:07:45 2023]\nFinished job 13.\n5 of 59 steps (8%) done\nRemoving\
  \ temporary output Lasagne_horse/trimmed/Lasagne_horse_noprimer5p_R1.fastq.gz.\n\
  Removing temporary output Lasagne_horse/trimmed/Lasagne_horse_noprimer5p_R2.fastq.gz.\n\
  Removing temporary output Lasagne_horse/trimmed/Lasagne_horse_noprimer3p_R1.fastq.gz.\n\
  Removing temporary output Lasagne_horse/trimmed/Lasagne_horse_noprimer3p_R2.fastq.gz.\n\
  Shutting down, this might take some time.\nExiting because a job execution failed.\
  \ Look above for error message\nComplete log: .snakemake/log/2023-02-19T060017.758056.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2F1ir9eu0j%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2F1ir9eu0j%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.971GiB</td>
						<td>308.9 seconds</td>
						<td>449.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739440208">
								<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover">IMS-Bio2Core-Facility/polya_liftover</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>440 days</td>
						<td>2023-02-19 03:53:56</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc_reads.smk
kind: MissingInputException
rest: "Missing input files for rule fastqc:\n    output: results/fastqc/1_sample1_R1.html,\
  \ results/fastqc/1_sample1_R1_fastqc.zip\n    wildcards: lane=1, sample=sample1,\
  \ read=R1\n    affected files:\n        path/to/your/data/r1.fq.gz"
rule: fastqc  in line 1

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2Fil5ncf2c%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2Fil5ncf2c%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.4 seconds</td>
						<td>1.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739322880">
								<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover">IMS-Bio2Core-Facility/polya_liftover</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v1.0.1">v1.0.1</a>
						</td>
						<td>snakemake</td>
						<td>439 days</td>
						<td>2023-02-19 04:31:48</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc_reads.smk
kind: MissingInputException
rest: "Missing input files for rule fastqc:\n    output: results/fastqc/1_sample1_R1.html,\
  \ results/fastqc/1_sample1_R1_fastqc.zip\n    wildcards: lane=1, sample=sample1,\
  \ read=R1\n    affected files:\n        path/to/your/data/r1.fq.gz"
rule: fastqc  in line 1

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2Fcmiulms1%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2Fcmiulms1%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.0 seconds</td>
						<td>1.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739287472">
								<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover">IMS-Bio2Core-Facility/polya_liftover</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>439 days</td>
						<td>2023-02-19 04:45:49</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc_reads.smk
kind: MissingInputException
rest: "Missing input files for rule fastqc:\n    output: results/fastqc/1_sample1_R1.html,\
  \ results/fastqc/1_sample1_R1_fastqc.zip\n    wildcards: lane=1, sample=sample1,\
  \ read=R1\n    affected files:\n        path/to/your/data/r1.fq.gz"
rule: fastqc  in line 1

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2F6tklmm18%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2F6tklmm18%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.061GiB</td>
						<td>1.6 seconds</td>
						<td>3.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738267040">
								<a href="https://github.com/mrvollger/StainedGlass">mrvollger/StainedGlass</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/StainedGlass/tree/v0.4">v0.4</a>
						</td>
						<td>snakemake</td>
						<td>438 days</td>
						<td>2023-02-19 09:08:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/m6pn2jsc6g/code/workflow/envs/env.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/m6pn2jsc6g/code/.snakemake/conda/bca8af08e312fca8eaf40f93af61f491_.yaml"
  --prefix "/home/azureuser/tmp/m6pn2jsc6g/code/.snakemake/conda/bca8af08e312fca8eaf40f93af61f491_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working...'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FStainedGlass%2Fb6tucqg4%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FStainedGlass%2Fb6tucqg4%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>8.082GiB</td>
						<td>1087.6 seconds</td>
						<td>1104.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738260560">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.4.1">v0.4.1</a>
						</td>
						<td>snakemake</td>
						<td>438 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- common.smk
kind: ValueError
line_no: 37
rest: "The truth value of an array with more than one element is ambiguous. Use a.any()\
  \ or a.all()\n  File \"/home/azureuser/tmp/zakmzlx2v8/code/workflow/Snakefile\"\
  , line 33, in <module>\n  File \"/home/azureuser/tmp/zakmzlx2v8/code/workflow/rules/common.smk\"\
  , line 37, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fwiy0g0f0%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fwiy0g0f0%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.105GiB</td>
						<td>2.1 seconds</td>
						<td>3.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743141552">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>437 days</td>
						<td>2023-02-19 04:12:26</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "DecoyDatabase -in test.fasta \\\n    -out test_decoy.fasta \\\n\
  \    -decoy_string DECOY_ \\\n    -decoy_string_position prefix\n\ncat <<-END_VERSIONS\
  \ > versions.yml\nOPENMS_DECOYDATABASE:\n    openms: $(echo $(FileInfo --help 2>&1)\
  \ | sed 's/^.*Version: //; s/-.*$//' | sed 's/ -*//; s/ .*$//')\nEND_VERSIONS"
error: 'WARNING: Skipping mount /home/azureuser/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/singularityce-3.10.3-z5qu45ho72nvsy4rpcyd4eqrop35tnbe/var/singularity/mnt/session/etc/resolv.conf
  [files]: /etc/resolv.conf doesn''t exist in container

  Error: File not found (the file ''test.fasta'' could not be found)

  Cannot read input file given from parameter ''-in''!'
exit_status: 1
kind: CalledProcessError
output: 'Error: File not found (the file ''test.fasta'' could not be found)'
process: NFCORE_MHCQUANT:MHCQUANT:OPENMS_DECOYDATABASE (4)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '73'
- 22a901b14586d559b7e7b3da69c8e7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fan6cemo9%2Flogs.tar.xz">0.016GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fan6cemo9%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.882GiB</td>
						<td>58.3 seconds</td>
						<td>64.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738260320">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.5.0">v0.5.0</a>
						</td>
						<td>snakemake</td>
						<td>437 days</td>
						<td>2023-02-19 03:50:48</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc.smk
kind: WorkflowError
line_no: 7
rest: Function did not return str or list of str.

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Ftf4bee1b%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Ftf4bee1b%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.108GiB</td>
						<td>2.4 seconds</td>
						<td>6.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738205168">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.2">1.4.2</a>
						</td>
						<td>snakemake</td>
						<td>433 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- oa352xnwpi
- code
- workflow
- rules
- blast.smk
kind: CalledProcessError
line_no: 20
rest: "Command 'set -euo pipefail;  /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/python3.10\
  \ /home/azureuser/tmp/oa352xnwpi/code/.tests/.snakemake/scripts/tmppy98skma.filter_taxonomy.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/oa352xnwpi/code/workflow/rules/blast.smk\"\
  , line 20, in __rule_prep_taxonomy\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Sun Feb 19 06:05:18 2023]\nFinished job 7.\n4 of 59 steps (7%)\
  \ done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-19T060017.850207.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Flhpt9jfl%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Flhpt9jfl%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.891GiB</td>
						<td>261.1 seconds</td>
						<td>304.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738260080">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.6.0">v0.6.0</a>
						</td>
						<td>snakemake</td>
						<td>433 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc.smk
kind: WorkflowError
line_no: 7
rest: Function did not return str or list of str.

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F1p2ry8pz%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F1p2ry8pz%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.108GiB</td>
						<td>2.5 seconds</td>
						<td>3.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738259840">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.7.0">v0.7.0</a>
						</td>
						<td>snakemake</td>
						<td>431 days</td>
						<td>2023-02-19 06:15:58</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc.smk
kind: WorkflowError
line_no: 7
rest: Function did not return str or list of str.

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Femgiaofe%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Femgiaofe%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.109GiB</td>
						<td>3.1 seconds</td>
						<td>4.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738259600">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.8.0">v0.8.0</a>
						</td>
						<td>snakemake</td>
						<td>431 days</td>
						<td>2023-02-19 04:51:27</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc.smk
kind: MissingInputException
rest: "Missing input files for rule fastqc:\n    output: results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1_fastqc.zip\n    wildcards: date=SEQUENCING_DATE,\
  \ sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq1\n        PATH/TO/fq2"
rule: fastqc  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fsqcn208a%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fsqcn208a%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.108GiB</td>
						<td>2.7 seconds</td>
						<td>5.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738259360">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.9.0">v0.9.0</a>
						</td>
						<td>snakemake</td>
						<td>430 days</td>
						<td>2023-02-19 04:16:10</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc.smk
kind: MissingInputException
rest: "Missing input files for rule fastqc:\n    output: results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1_fastqc.zip\n    wildcards: date=SEQUENCING_DATE,\
  \ sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq1\n        PATH/TO/fq2"
rule: fastqc  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fyeg8pjg7%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fyeg8pjg7%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.109GiB</td>
						<td>2.4 seconds</td>
						<td>3.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738259120">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.10.0">v0.10.0</a>
						</td>
						<td>snakemake</td>
						<td>429 days</td>
						<td>2023-02-19 04:45:39</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc.smk
kind: MissingInputException
rest: "Missing input files for rule fastqc:\n    output: results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1_fastqc.zip\n    wildcards: date=SEQUENCING_DATE,\
  \ sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq1\n        PATH/TO/fq2"
rule: fastqc  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F309u99pb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F309u99pb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.109GiB</td>
						<td>2.8 seconds</td>
						<td>4.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738300288">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.4.0">v3.4.0</a>
						</td>
						<td>snakemake</td>
						<td>429 days</td>
						<td>2023-02-19 04:30:36</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/77afkqtr43/singularity-tmp/tmpqy2um2a5.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/c941dcb6-0286-4298-a681-65e2fee2e1a5\
  \ singularity  exec --home /home/azureuser/tmp/77afkqtr43/code/.test  /home/azureuser/tmp/77afkqtr43/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/77afkqtr43/code/.test/.snakemake/conda/9c986af96a7a0c417ce1ac3d2b91c9ce_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/77afkqtr43/code/.test/.snakemake/conda/9c986af96a7a0c417ce1ac3d2b91c9ce_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining htslib==1.10:  20%|\u2588\u2588\
  \        | 1/5 [00:00<00:00, 6316.72it/s]     \nExamining @/linux-64::__unix==0=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 41.93it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00, 58.15it/s]\n\
  Examining @/linux-64::__archspec==1=x86_64:  80%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588  | 4/5 [00:00<00:00, 77.14it/s]\n                                 \
  \                                                        \n\nDetermining conflicts:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining conflict for __glibc htslib:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\n                                      \
  \                                      \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/5 [00:00<?,\
  \ ?it/s]\nExamining @/linux-64::__glibc==2.31=0:   0%|          | 0/5 [00:00<?,\
  \ ?it/s]\nExamining htslib==1.10:  20%|\u2588\u2588        | 1/5 [00:00<00:00, 7530.17it/s]\
  \     \nExamining @/linux-64::__unix==0=0:  40%|\u2588\u2588\u2588\u2588      |\
  \ 2/5 [00:00<00:00, 97.34it/s]\nExamining @/linux-64::__linux==5.15.0=0:  60%|\u2588\
  \u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00, 144.87it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  80%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 192.04it/s]\n\
  \                                                                              \
  \            \n\nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\n\
  Examining conflict for __glibc htslib:   0%|          | 0/5 [00:00<?, ?it/s]\n \
  \                                                                           \nfailed\n\
  \nUnsatisfiableError: The following specifications were found to be incompatible\
  \ with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n  - htslib==1.10\
  \ -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\nYour installed\
  \ version is: 2.31\n\nNote that strict channel priority may have removed packages\
  \ required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fr6swx1vg%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fr6swx1vg%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.317GiB</td>
						<td>1011.8 seconds</td>
						<td>1204.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743152352">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.5">3.5</a>
						</td>
						<td>nextflow</td>
						<td>429 days</td>
						<td>2023-02-19 06:00:19</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-19 07:00:19.952
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F1arq81lc%2Flogs.tar.xz">0.065GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F1arq81lc%2Foutput.tar.xz">0.005GiB</a>
						</td>
						<td>1.067GiB</td>
						<td>155.8 seconds</td>
						<td>3601.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738258880">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.10.1">v0.10.1</a>
						</td>
						<td>snakemake</td>
						<td>426 days</td>
						<td>2023-02-19 04:33:49</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc.smk
kind: MissingInputException
rest: "Missing input files for rule fastqc:\n    output: results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1_fastqc.zip\n    wildcards: date=SEQUENCING_DATE,\
  \ sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq2\n        PATH/TO/fq1"
rule: fastqc  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F35936elw%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F35936elw%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.109GiB</td>
						<td>2.7 seconds</td>
						<td>6.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738258640">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.10.2">v0.10.2</a>
						</td>
						<td>snakemake</td>
						<td>426 days</td>
						<td>2023-02-19 04:24:07</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc.smk
kind: MissingInputException
rest: "Missing input files for rule fastqc:\n    output: results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1_fastqc.zip\n    wildcards: date=SEQUENCING_DATE,\
  \ sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq1\n        PATH/TO/fq2"
rule: fastqc  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fg5jcbdwp%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fg5jcbdwp%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.109GiB</td>
						<td>2.5 seconds</td>
						<td>2.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738216208">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>426 days</td>
						<td>2023-02-19 09:08:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fpkw15bz8%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fpkw15bz8%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.979GiB</td>
						<td>430.3 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738211168">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>426 days</td>
						<td>2023-02-19 03:37:28</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Ft090ud18%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Ft090ud18%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.104GiB</td>
						<td>965.9 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738215968">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.0.1">v1.0.1</a>
						</td>
						<td>snakemake</td>
						<td>426 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2F8aogyzm3%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2F8aogyzm3%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.973GiB</td>
						<td>217.7 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738210928">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.0.1">v1.0.1</a>
						</td>
						<td>snakemake</td>
						<td>426 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F29iniwx2%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F29iniwx2%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.985GiB</td>
						<td>232.7 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743109504">
								<a href="https://nf-co.re/epitopeprediction">epitopeprediction</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/epitopeprediction/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>425 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "check_samplesheet.py \\\n    sample_sheet_variants.csv \\\n   \
  \ samplesheet.valid.csv\n\ncat <<-END_VERSIONS > versions.yml\n\"NFCORE_EPITOPEPREDICTION:EPITOPEPREDICTION:INPUT_CHECK:SAMPLESHEET_CHECK\"\
  :\n    python: $(python --version | sed 's/Python //g')\nEND_VERSIONS"
error: "WARNING: Skipping mount /home/azureuser/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/singularityce-3.10.3-z5qu45ho72nvsy4rpcyd4eqrop35tnbe/var/singularity/mnt/session/etc/resolv.conf\
  \ [files]: /etc/resolv.conf doesn't exist in container\nERROR: Please check samplesheet\
  \ header -> sample\talleles\tmhc_class\tfilename != sample\talleles\tfilename"
exit_status: 1
kind: CalledProcessError
output: "ERROR: Please check samplesheet header -> sample\talleles\tmhc_class\tfilename\
  \ != sample\talleles\tfilename"
process: NFCORE_EPITOPEPREDICTION:EPITOPEPREDICTION:INPUT_CHECK:SAMPLESHEET_CHECK
  (sample_sheet_variants.csv)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- d2
- 5adefbd37548526c79346ec9748174

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epitopeprediction%2Fhrwvwhx8%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epitopeprediction%2Fhrwvwhx8%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.708GiB</td>
						<td>34.7 seconds</td>
						<td>43.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738258400">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.11.0">v0.11.0</a>
						</td>
						<td>snakemake</td>
						<td>425 days</td>
						<td>2023-02-19 03:44:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc.smk
kind: MissingInputException
rest: "Missing input files for rule fastqc:\n    output: results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1_fastqc.zip\n    wildcards: date=SEQUENCING_DATE,\
  \ sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq2\n        PATH/TO/fq1"
rule: fastqc  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fldf680ff%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fldf680ff%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.109GiB</td>
						<td>2.8 seconds</td>
						<td>5.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739186464">
								<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover">IMS-Bio2Core-Facility/polya_liftover</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v2.0.0">v2.0.0</a>
						</td>
						<td>snakemake</td>
						<td>424 days</td>
						<td>2023-02-19 06:51:58</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- input_resources.smk
kind: MissingInputException
rest: "Missing input files for rule clean_names:\n    output: data/sample1_S1_L001_R1_001.fastq.gz\n\
  \    wildcards: lane=1, sample=sample1, read=R1\n    affected files:\n        path/to/your/data/r1.fq.gz"
rule: clean_names  in line 24

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2Fc4mb4g1d%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2Fc4mb4g1d%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.5 seconds</td>
						<td>1.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738258160">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.12.0">v0.12.0</a>
						</td>
						<td>snakemake</td>
						<td>422 days</td>
						<td>2023-02-19 04:48:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- qc.smk
kind: MissingInputException
rest: "Missing input files for rule fastqc:\n    output: results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/qc/fastqc/SAMPLE_NAME_1_fastqc.zip\n    wildcards: date=SEQUENCING_DATE,\
  \ sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq2\n        PATH/TO/fq1"
rule: fastqc  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F1mmpan7o%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2F1mmpan7o%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.108GiB</td>
						<td>2.5 seconds</td>
						<td>2.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738257920">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.13.0">v0.13.0</a>
						</td>
						<td>snakemake</td>
						<td>412 days</td>
						<td>2023-02-19 06:51:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- read_trimming.smk
kind: MissingInputException
rest: "Missing input files for rule fastp_pe:\n    output: results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.1.fastq.gz,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.2.fastq.gz, results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.fastp.json\n    wildcards:\
  \ date=SEQUENCING_DATE, sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq1\n\
  \        PATH/TO/fq2"
rule: fastp_pe  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fljyfoo09%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fljyfoo09%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.109GiB</td>
						<td>2.5 seconds</td>
						<td>2.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738257680">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.13.1">v0.13.1</a>
						</td>
						<td>snakemake</td>
						<td>411 days</td>
						<td>2023-02-19 04:17:03</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- read_trimming.smk
kind: MissingInputException
rest: "Missing input files for rule fastp_pe:\n    output: results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.1.fastq.gz,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.2.fastq.gz, results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.fastp.json\n    wildcards:\
  \ date=SEQUENCING_DATE, sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq1\n\
  \        PATH/TO/fq2"
rule: fastp_pe  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fnlpvgjun%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fnlpvgjun%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.108GiB</td>
						<td>2.8 seconds</td>
						<td>3.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738257440">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.13.2">v0.13.2</a>
						</td>
						<td>snakemake</td>
						<td>409 days</td>
						<td>2023-02-19 06:00:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- read_trimming.smk
kind: MissingInputException
rest: "Missing input files for rule fastp_pe:\n    output: results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.1.fastq.gz,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.2.fastq.gz, results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.fastp.json\n    wildcards:\
  \ date=SEQUENCING_DATE, sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq2\n\
  \        PATH/TO/fq1"
rule: fastp_pe  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fmyy90d7s%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fmyy90d7s%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.109GiB</td>
						<td>2.8 seconds</td>
						<td>3.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738300048">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.5.0">v3.5.0</a>
						</td>
						<td>snakemake</td>
						<td>405 days</td>
						<td>2023-02-19 04:06:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/823vbojznp/singularity-tmp/tmpb7nq35mo.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/c8463f83-025b-43a7-88b7-82f2996c4602\
  \ singularity  exec --home /home/azureuser/tmp/823vbojznp/code/.test  /home/azureuser/tmp/823vbojznp/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/823vbojznp/code/.test/.snakemake/conda/2a114852c575d40d019dda029465805b_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/823vbojznp/code/.test/.snakemake/conda/2a114852c575d40d019dda029465805b_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining bcftools=1.10:  17%|\u2588\u258B\
  \        | 1/6 [00:00<00:00, 6078.70it/s]    \nExamining @/linux-64::__archspec==1=x86_64:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 51.19it/s]\nExamining @/linux-64::__unix==0=0:\
  \  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00, 76.42it/s]       \
  \  \nExamining @/linux-64::__linux==5.15.0=0:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 4/6 [00:00<00:00, 101.60it/s]\nExamining curl:  83%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00, 126.68it/s]     \
  \                    \n                                                        \
  \      \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ conflict for __glibc curl bcftools:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ conflict for curl bcftools:  17%|\u2588\u258B        | 1/6 [00:00<00:01,  2.77it/s]\n\
  Examining conflict for curl bcftools:  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00,\
  \  5.54it/s]\n                                                                 \
  \                  \nfailed\nSolving environment: ...working... \nFound conflicts!\
  \ Looking for incompatible packages.\nThis can take several minutes.  Press CTRL-C\
  \ to abort.\n\nBuilding graph of deps:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ @/linux-64::__glibc==2.31=0:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ bcftools=1.10:  17%|\u2588\u258B        | 1/6 [00:00<00:00, 7121.06it/s]    \n\
  Examining @/linux-64::__archspec==1=x86_64:  33%|\u2588\u2588\u2588\u258E      |\
  \ 2/6 [00:00<00:00, 50.75it/s]\nExamining @/linux-64::__unix==0=0:  50%|\u2588\u2588\
  \u2588\u2588\u2588     | 3/6 [00:00<00:00, 75.85it/s]         \nExamining @/linux-64::__linux==5.15.0=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 4/6 [00:00<00:00, 100.90it/s]\n\
  Examining curl:  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6\
  \ [00:00<00:00, 125.85it/s]                         \n                         \
  \                                     \n\nDetermining conflicts:   0%|         \
  \ | 0/6 [00:00<?, ?it/s]\nExamining conflict for __glibc curl bcftools:   0%|  \
  \        | 0/6 [00:00<?, ?it/s]\nExamining conflict for curl bcftools:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:01,  2.83it/s]\nExamining conflict for curl bcftools:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00,  5.65it/s]\n          \
  \                                                                         \nfailed\n\
  \nUnsatisfiableError: The following specifications were found to be incompatible\
  \ with each other:\n\nOutput in format: Requested package -> Available versions\n\
  \nPackage curl conflicts for:\ncurl\nbcftools=1.10 -> htslib[version='>=1.10,<1.11.0a0']\
  \ -> curl[version='>=7.64.1,<8.0a0']The following specifications were found to be\
  \ incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n  -\
  \ bcftools=1.10 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n \
  \ - curl -> libgcc-ng[version='>=10.3.0'] -> __glibc[version='>=2.17']\n\nYour installed\
  \ version is: 2.31\n\nNote that strict channel priority may have removed packages\
  \ required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F3xmmax4n%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F3xmmax4n%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.242GiB</td>
						<td>662.0 seconds</td>
						<td>763.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738299808">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.6.0">v3.6.0</a>
						</td>
						<td>snakemake</td>
						<td>405 days</td>
						<td>2023-02-19 04:11:50</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/ki1zjzoa9q/singularity-tmp/tmpr78lupb5.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/9feab377-d731-47a0-98e5-7196581a2402\
  \ singularity  exec --home /home/azureuser/tmp/ki1zjzoa9q/code/.test  /home/azureuser/tmp/ki1zjzoa9q/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/ki1zjzoa9q/code/.test/.snakemake/conda/b68ecd1ec50ee84893458cf91944ee03_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/ki1zjzoa9q/code/.test/.snakemake/conda/b68ecd1ec50ee84893458cf91944ee03_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining gsl[version='>=2.5,<2.6.0a0']:\
  \  11%|\u2588         | 1/9 [00:00<00:00, 6636.56it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 92.35it/s]  \nExamining perl:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 137.54it/s]           \
  \             \nExamining libgcc-ng[version='>=7.3.0']:  44%|\u2588\u2588\u2588\u2588\
  \u258D     | 4/9 [00:00<00:00, 40.10it/s]\nExamining htslib[version='>=1.10,<1.11.0a0']:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 35.45it/s]\n\
  Examining htslib[version='>=1.10,<1.11.0a0']:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:00<00:00, 42.51it/s]\nExamining @/linux-64::__unix==0=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 42.51it/s]\
  \           \nExamining @/linux-64::__archspec==1=x86_64:  78%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258A  | 7/9 [00:00<00:00, 42.51it/s]\nExamining zlib[version='>=1.2.11,<1.3.0a0']:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00,\
  \ 42.51it/s]\n                                                                 \
  \                         \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for __glibc gsl perl libgcc-ng htslib zlib:   0%| \
  \         | 0/9 [00:00<?, ?it/s]\nExamining conflict for gsl perl libgcc-ng htslib\
  \ zlib:  11%|\u2588         | 1/9 [00:00<00:00,  8.54it/s]\nExamining conflict for\
  \ gsl perl libgcc-ng htslib zlib:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \ 17.01it/s]\nExamining conflict for zlib htslib:  22%|\u2588\u2588\u258F      \
  \ | 2/9 [00:00<00:00, 17.01it/s]                   \n                          \
  \                                                       \nfailed\nSolving environment:\
  \ ...working... \nFound conflicts! Looking for incompatible packages.\nThis can\
  \ take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:   0%|\
  \          | 0/9 [00:00<?, ?it/s]\nExamining gsl[version='>=2.5,<2.6.0a0']:  11%|\u2588\
  \         | 1/9 [00:00<00:00, 5706.54it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 59.19it/s]  \nExamining perl:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 68.30it/s]            \
  \             \nExamining libgcc-ng[version='>=7.3.0']:  44%|\u2588\u2588\u2588\u2588\
  \u258D     | 4/9 [00:00<00:00, 17.95it/s]\nExamining libgcc-ng[version='>=7.3.0']:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 21.94it/s]\n\
  Examining htslib[version='>=1.10,<1.11.0a0']:  56%|\u2588\u2588\u2588\u2588\u2588\
  \u258C    | 5/9 [00:00<00:00, 21.94it/s]\nExamining @/linux-64::__unix==0=0:  67%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 21.94it/s]          \
  \ \nExamining @/linux-64::__archspec==1=x86_64:  78%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2588\u258A  | 7/9 [00:00<00:00, 21.94it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00,\
  \ 15.75it/s]\nExamining zlib[version='>=1.2.11,<1.3.0a0']:  89%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00, 15.75it/s]\n          \
  \                                                                              \
  \  \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for __glibc gsl perl libgcc-ng htslib zlib:   0%|          | 0/9 [00:00<?, ?it/s]\n\
  Examining conflict for gsl perl libgcc-ng htslib zlib:  11%|\u2588         | 1/9\
  \ [00:00<00:01,  5.17it/s]\nExamining conflict for gsl perl libgcc-ng htslib zlib:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 10.33it/s]\nExamining conflict\
  \ for zlib htslib:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 10.33it/s]\
  \                   \n                                                         \
  \                        \nfailed\n\nUnsatisfiableError: The following specifications\
  \ were found to be incompatible with each other:\n\nOutput in format: Requested\
  \ package -> Available versions\n\nPackage libgcc-ng conflicts for:\nhtslib[version='>=1.10,<1.11.0a0']\
  \ -> bzip2[version='>=1.0.8,<2.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=9.3.0|>=9.4.0|>=11.2.0|>=4.9|>=7.2.0']\n\
  zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=10.3.0|>=12|>=7.5.0|>=7.3.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  perl -> libgcc-ng[version='>=11.2.0|>=4.9|>=7.3.0|>=7.5.0|>=9.3.0|>=9.4.0|>=7.2.0']\n\
  htslib[version='>=1.10,<1.11.0a0'] -> libgcc-ng[version='>=7.3.0|>=7.5.0']\nlibgcc-ng[version='>=7.3.0']\n\
  gsl[version='>=2.5,<2.6.0a0'] -> libgcc-ng[version='>=7.3.0']\ngsl[version='>=2.5,<2.6.0a0']\
  \ -> libblas[version='>=3.8.0,<4.0a0'] -> libgcc-ng[version='>=7.5.0|>=9.3.0']\n\
  \nPackage zlib conflicts for:\nhtslib[version='>=1.10,<1.11.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0']\n\
  zlib[version='>=1.2.11,<1.3.0a0']\nhtslib[version='>=1.10,<1.11.0a0'] -> libcurl[version='>=7.71.1,<8.0a0']\
  \ -> zlib[version='>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']The following specifications\
  \ were found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - gsl[version='>=2.5,<2.6.0a0'] -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - htslib[version='>=1.10,<1.11.0a0'] -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\
  \  - libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n  - perl -> libgcc-ng[version='>=9.4.0']\
  \ -> __glibc[version='>=2.17']\n  - zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=10.3.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Ft8z0g4vt%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Ft8z0g4vt%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.219GiB</td>
						<td>162.8 seconds</td>
						<td>224.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010737902272">
								<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing">IMS-Bio2Core-Facility/single_snake_sequencing</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/single_snake_sequencing/tree/v0.1.2">v0.1.2</a>
						</td>
						<td>snakemake</td>
						<td>402 days</td>
						<td>2023-02-19 06:52:55</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- resources.smk
kind: MissingInputException
rest: "Missing input files for rule clean_names:\n    output: data/sample1_S1_L001_R1_001.fastq.gz\n\
  \    wildcards: lane=1, sample=sample1, read=R1\n    affected files:\n        path/to/your/data/r1.fq.gz"
rule: clean_names  in line 56

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fsingle-snake-sequencing%2Fb9o77zb4%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fsingle-snake-sequencing%2Fb9o77zb4%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.4 seconds</td>
						<td>1.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738299568">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.6.1">v3.6.1</a>
						</td>
						<td>snakemake</td>
						<td>401 days</td>
						<td>2023-02-19 04:32:07</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/f76n67hzjd/code/workflow/rules/../envs/rbt.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/ddbd6e2d-e134-423a-b2b0-a6ca5d141d90\
  \ singularity  exec --home /home/azureuser/tmp/f76n67hzjd/code/.test  /home/azureuser/tmp/f76n67hzjd/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/f76n67hzjd/code/.test/.snakemake/conda/c146e6519a9639e42b0f3d82c49b9882_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/f76n67hzjd/code/.test/.snakemake/conda/c146e6519a9639e42b0f3d82c49b9882_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining rust-bio-tools=0.38:   0%|  \
  \        | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:00, 32.16it/s]\nExamining @/linux-64::__unix==0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 63.97it/s]    \nExamining\
  \ @/linux-64::__linux==5.15.0=0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 95.70it/s]\nExamining bcftools=1.12:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 4/6 [00:00<00:00, 127.30it/s]               \nExamining @/linux-64::__archspec==1=x86_64:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 118.15it/s]\n                                                                \
  \                          \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for rust-bio-tools bcftools:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for rust-bio-tools __glibc bcftools:  17%|\u2588\u258B\
  \        | 1/6 [00:00<00:01,  3.34it/s]\nExamining conflict for rust-bio-tools __glibc\
  \ bcftools:  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00,  6.69it/s]\n\
  \                                                                              \
  \                       \nfailed\nSolving environment: ...working... \nFound conflicts!\
  \ Looking for incompatible packages.\nThis can take several minutes.  Press CTRL-C\
  \ to abort.\n\nBuilding graph of deps:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ rust-bio-tools=0.38:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  17%|\u2588\u258B        | 1/6 [00:00<00:00, 20.15it/s]\nExamining @/linux-64::__unix==0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 40.13it/s]    \nExamining\
  \ @/linux-64::__linux==5.15.0=0:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00,\
  \ 59.97it/s]\nExamining bcftools=1.12:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 4/6 [00:00<00:00, 79.77it/s]                \nExamining @/linux-64::__archspec==1=x86_64:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 74.15it/s]\n                                                                 \
  \                        \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for rust-bio-tools bcftools:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for rust-bio-tools __glibc bcftools:  17%|\u2588\u258B\
  \        | 1/6 [00:00<00:02,  2.26it/s]\nExamining conflict for rust-bio-tools __glibc\
  \ bcftools:  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00,  4.52it/s]\n\
  \                                                                              \
  \                       \nfailed\n\nUnsatisfiableError: The following specifications\
  \ were found to be incompatible with each other:\n\nOutput in format: Requested\
  \ package -> Available versions\n\nPackage libgcc-ng conflicts for:\nrust-bio-tools=0.38\
  \ -> libgcc-ng[version='>=10.3.0|>=9.4.0']\nrust-bio-tools=0.38 -> gsl[version='>=2.7,<2.8.0a0']\
  \ -> libgcc-ng[version='>=11.2.0|>=9.3.0|>=7.5.0|>=7.3.0|>=12|>=4.9']\n\nPackage\
  \ libblas conflicts for:\nrust-bio-tools=0.38 -> gsl[version='>=2.7,<2.8.0a0'] ->\
  \ libblas[version='3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0.*|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0.*|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|>=3.8.0,<4.0a0',build='4_blis|4_mkl|5_blis|5_mkl|6_blis|6_openblas|6_mkl|7_openblas|8_mkl|9_openblas|10_openblas|11_blis|11_openblas|11_mkl|12_blis|12_mkl|13_blis|14_openblas|14_blis|15_blis|16_openblas|16_blis|19_mkl|20_mkl|21_mkl|0_openblas|1_openblas|2_blis|3_blis|4_blis|6_blis|6_mkl|7_blis|7_openblas|8_blis|8_mkl|9_openblas|10_blis|10_mkl|11_linux64_blis|11_linux64_openblas|11_linux64_mkl|12_linux64_blis|12_linux64_mkl|13_linux64_blis|14_linux64_openblas|15_linux64_blis|16_linux64_openblas|16_linux64_mkl|16_linux64_blis|15_linux64_mkl|15_linux64_openblas|14_linux64_mkl|14_linux64_blis|13_linux64_mkl|13_linux64_openblas|12_linux64_openblas|10_openblas|9_mkl|9_blis|8_openblas|7_mkl|6_openblas|5_mkl|5_openblas|5_blis|4_mkl|4_openblas|3_openblas|2_openblas|1_blis|0_blis|18_mkl|17_openblas|16_mkl|15_openblas|15_mkl|14_mkl|13_openblas|13_mkl|12_openblas|10_mkl|10_blis|9_mkl|9_blis|8_openblas|8_blis|7_mkl|7_blis|5_openblas|4_openblas']\n\
  bcftools=1.12 -> gsl[version='>=2.6,<2.7.0a0'] -> libblas[version='3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0.*|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0.*|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|3.9.0|>=3.8.0,<4.0a0',build='4_blis|4_mkl|5_blis|5_mkl|6_blis|6_openblas|6_mkl|7_openblas|8_mkl|9_openblas|10_openblas|11_blis|11_openblas|11_mkl|12_blis|12_mkl|13_blis|14_openblas|14_blis|15_blis|16_openblas|16_blis|19_mkl|20_mkl|21_mkl|0_openblas|1_openblas|2_blis|3_blis|4_blis|6_blis|6_mkl|7_blis|7_openblas|8_blis|8_mkl|9_openblas|10_blis|10_mkl|11_linux64_blis|11_linux64_openblas|11_linux64_mkl|12_linux64_blis|12_linux64_mkl|13_linux64_blis|14_linux64_openblas|15_linux64_blis|16_linux64_openblas|16_linux64_mkl|16_linux64_blis|15_linux64_mkl|15_linux64_openblas|14_linux64_mkl|14_linux64_blis|13_linux64_mkl|13_linux64_openblas|12_linux64_openblas|10_openblas|9_mkl|9_blis|8_openblas|7_mkl|6_openblas|5_mkl|5_openblas|5_blis|4_mkl|4_openblas|3_openblas|2_openblas|1_blis|0_blis|18_mkl|17_openblas|16_mkl|15_openblas|15_mkl|14_mkl|13_openblas|13_mkl|12_openblas|10_mkl|10_blis|9_mkl|9_blis|8_openblas|8_blis|7_mkl|7_blis|5_openblas|4_openblas']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - bcftools=1.12 -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fzjx15nd0%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fzjx15nd0%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.185GiB</td>
						<td>315.8 seconds</td>
						<td>272.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740639840">
								<a href="https://github.com/mrvollger/Rhodonite">mrvollger/Rhodonite</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/Rhodonite/tree/v0.7-alpha">v0.7-alpha</a>
						</td>
						<td>snakemake</td>
						<td>400 days</td>
						<td>2023-02-19 03:31:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: SystemExit
line_no: 38
rest: "1\n  File \"/home/azureuser/tmp/5v0twmx09z/code/workflow/Snakefile\", line\
  \ 38, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Fuboh5ieo%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Fuboh5ieo%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.098GiB</td>
						<td>2.0 seconds</td>
						<td>2.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743141312">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/2.2.0">2.2.0</a>
						</td>
						<td>nextflow</td>
						<td>400 days</td>
						<td>2023-02-19 03:51:30</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "DecoyDatabase -in test.fasta \\\n    -out test_decoy.fasta \\\n\
  \    -decoy_string DECOY_ \\\n    -decoy_string_position prefix\n\ncat <<-END_VERSIONS\
  \ > versions.yml\n\"NFCORE_MHCQUANT:MHCQUANT:OPENMS_DECOYDATABASE\":\n    openms:\
  \ $(echo $(FileInfo --help 2>&1) | sed 's/^.*Version: //; s/-.*$//' | sed 's/ -*//;\
  \ s/ .*$//')\nEND_VERSIONS"
error: 'WARNING: Skipping mount /home/azureuser/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/singularityce-3.10.3-z5qu45ho72nvsy4rpcyd4eqrop35tnbe/var/singularity/mnt/session/etc/resolv.conf
  [files]: /etc/resolv.conf doesn''t exist in container

  Error: File not found (the file ''test.fasta'' could not be found)

  Cannot read input file given from parameter ''-in''!'
exit_status: 1
kind: CalledProcessError
output: 'Error: File not found (the file ''test.fasta'' could not be found)'
process: NFCORE_MHCQUANT:MHCQUANT:OPENMS_DECOYDATABASE (4)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '36'
- a7d2bd539d6ba86a89eb03c77422b3

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fd8jz5gmz%2Flogs.tar.xz">0.016GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fd8jz5gmz%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.843GiB</td>
						<td>46.8 seconds</td>
						<td>61.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739085520">
								<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover">IMS-Bio2Core-Facility/polya_liftover</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IMS-Bio2Core-Facility/polya_liftover/tree/v2.0.1">v2.0.1</a>
						</td>
						<td>snakemake</td>
						<td>398 days</td>
						<td>2023-02-19 03:55:54</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- input_resources.smk
kind: MissingInputException
rest: "Missing input files for rule clean_names:\n    output: data/sample1_S1_L001_R1_001.fastq.gz\n\
  \    wildcards: lane=1, sample=sample1, read=R1\n    affected files:\n        path/to/your/data/r1.fq.gz"
rule: clean_names  in line 24

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2Fdw651sak%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IMS-Bio2Core-Facility%252Fpolya-liftover%2Fdw651sak%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.4 seconds</td>
						<td>1.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741229328">
								<a href="https://github.com/plycrsk/rna-seq-star-deseq2">plycrsk/rna-seq-star-deseq2</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/plycrsk/rna-seq-star-deseq2/tree/v1.2.1">v1.2.1</a>
						</td>
						<td>snakemake</td>
						<td>397 days</td>
						<td>2023-02-19 03:43:59</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 3yuboxnt7w
- code
- workflow
- Snakefile
kind: MissingInputException
rest: "Missing input files for rule all:\n    affected files:\n        results/pca.svg\n\
  \        results/deseq2/normcounts.symbol.tsv\n        results/counts/all.symbol.tsv\n\
  \        results/diffexp/treated-vs-untreated.diffexp.symbol.tsv"
rule: all  in line 36

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/plycrsk%252Frna-seq-star-deseq2%2Fm5xu8zoj%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/plycrsk%252Frna-seq-star-deseq2%2Fm5xu8zoj%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.104GiB</td>
						<td>2.0 seconds</td>
						<td>4.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743050448">
								<a href="https://nf-co.re/cutandrun">cutandrun</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/cutandrun/tree/1.1">1.1</a>
						</td>
						<td>nextflow</td>
						<td>396 days</td>
						<td>2023-02-20 04:31:42</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867594730\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\t\
  at nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\
  \tat nextflow.processor.TaskRun$isContainerEnabled$6.call(Unknown Source)\n\tat\
  \ nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867594730\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 07:31:42.596 [SIGTERM handler] DEBUG nextflow.Session\
  \ - The following nodes are still active:\n[process] NFCORE_CUTANDRUN:CUTANDRUN:PREPARE_GENOME:GUNZIP_FASTA\n\
  \n[process] NFCORE_CUTANDRUN:CUTANDRUN:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\n  status=ACTIVE\n\
  \  port 0: (value) OPEN  ; channel: fasta\n  port 1: (cntrl) -     ; channel: $\n\
  \n[process] NFCORE_CUTANDRUN:CUTANDRUN:PREPARE_GENOME:GET_SPIKEIN_CHROM_SIZES\n\
  \  status=ACTIVE\n  port 0: (value) OPEN  ; channel: fasta\n  port 1: (cntrl) -\
  \     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BOWTIE2_ALIGN\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) OPEN \
  \ ; channel: index\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BOWTIE2_SPIKEIN_ALIGN\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) OPEN \
  \ ; channel: index\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_STATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_FLAGSTAT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_IDXSTATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS_SPIKEIN:SAMTOOLS_SORT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS_SPIKEIN:SAMTOOLS_INDEX\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS_SPIKEIN:BAM_STATS_SAMTOOLS:SAMTOOLS_STATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS_SPIKEIN:BAM_STATS_SAMTOOLS:SAMTOOLS_FLAGSTAT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS_SPIKEIN:BAM_STATS_SAMTOOLS:SAMTOOLS_IDXSTATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:SAMTOOLS_VIEW_SORT_STATS:SAMTOOLS_VIEW\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound\
  \ ; channel: fasta\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:SAMTOOLS_VIEW_SORT_STATS:SAMTOOLS_SORT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:SAMTOOLS_VIEW_SORT_STATS:SAMTOOLS_INDEX\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:SAMTOOLS_VIEW_SORT_STATS:BAM_STATS_SAMTOOLS:SAMTOOLS_STATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:SAMTOOLS_VIEW_SORT_STATS:BAM_STATS_SAMTOOLS:SAMTOOLS_FLAGSTAT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:SAMTOOLS_VIEW_SORT_STATS:BAM_STATS_SAMTOOLS:SAMTOOLS_IDXSTATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:MARK_DUPLICATES_PICARD:PICARD_MARKDUPLICATES\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:MARK_DUPLICATES_PICARD:SAMTOOLS_INDEX\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:MARK_DUPLICATES_PICARD:BAM_STATS_SAMTOOLS:SAMTOOLS_STATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:MARK_DUPLICATES_PICARD:BAM_STATS_SAMTOOLS:SAMTOOLS_FLAGSTAT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:MARK_DUPLICATES_PICARD:BAM_STATS_SAMTOOLS:SAMTOOLS_IDXSTATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:DEDUP_PICARD:PICARD_MARKDUPLICATES\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:DEDUP_PICARD:SAMTOOLS_INDEX\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:DEDUP_PICARD:BAM_STATS_SAMTOOLS:SAMTOOLS_STATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:DEDUP_PICARD:BAM_STATS_SAMTOOLS:SAMTOOLS_FLAGSTAT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:DEDUP_PICARD:BAM_STATS_SAMTOOLS:SAMTOOLS_IDXSTATS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ANNOTATE_BT2_META:AWK_SCRIPT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound\
  \ ; channel: script\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ANNOTATE_BT2_SPIKEIN_META:AWK_SCRIPT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound\
  \ ; channel: script\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:ANNOTATE_DEDUP_META:AWK\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:BEDTOOLS_GENOMECOV\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound ; channel: sizes\n\
  \  port 2: (value) bound ; channel: extension\n  port 3: (cntrl) -     ; channel:\
  \ $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:BEDTOOLS_SORT\n  status=ACTIVE\n  port\
  \ 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound ; channel: extension\n\
  \  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:UCSC_BEDCLIP\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) OPEN \
  \ ; channel: sizes\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:UCSC_BEDGRAPHTOBIGWIG\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) OPEN \
  \ ; channel: sizes\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:SEACR_CALLPEAK\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound\
  \ ; channel: threshold\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:AWK_NAME_PEAK_BED\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CONSENSUS_PEAKS_ALL:SORT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CONSENSUS_PEAKS_ALL:BEDTOOLS_MERGE\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CONSENSUS_PEAKS_ALL:AWK\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CONSENSUS_PEAKS_ALL:PLOT_CONSENSUS_PEAKS\n\
  \  status=ACTIVE\n  port 0: (value) OPEN  ; channel: consensus_peaks\n  port 1:\
  \ (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CONSENSUS_PEAKS:SORT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CONSENSUS_PEAKS:BEDTOOLS_MERGE\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CONSENSUS_PEAKS:AWK\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] NFCORE_CUTANDRUN:CUTANDRUN:CONSENSUS_PEAKS:PLOT_CONSENSUS_PEAKS\n  status=ACTIVE\n\
  \  port 0: (value) OPEN  ; channel: consensus_peaks\n  port 1: (cntrl) -     ; channel:\
  \ $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CALCULATE_FRAGMENTS:SAMTOOLS_VIEW\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) bound\
  \ ; channel: fasta\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CALCULATE_FRAGMENTS:SAMTOOLS_SORT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CALCULATE_FRAGMENTS:BEDTOOLS_BAMTOBED\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CALCULATE_FRAGMENTS:AWK\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CALCULATE_FRAGMENTS:CUT\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:AWK_FRAG_BIN\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] NFCORE_CUTANDRUN:CUTANDRUN:SAMTOOLS_CUSTOMVIEW\n  status=ACTIVE\n  port\
  \ 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process]\
  \ NFCORE_CUTANDRUN:CUTANDRUN:IGV_SESSION\n  status=ACTIVE\n  port 0: (value) OPEN\
  \  ; channel: genome\n  port 1: (value) OPEN  ; channel: gtf\n  port 2: (value)\
  \ OPEN  ; channel: beds\n  port 3: (value) OPEN  ; channel: bigwig\n  port 4: (cntrl)\
  \ -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:AWK_EDIT_PEAK_BED\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:DEEPTOOLS_COMPUTEMATRIX_GENE\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (value) OPEN \
  \ ; channel: bed\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:DEEPTOOLS_PLOTHEATMAP_GENE\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:DEEPTOOLS_COMPUTEMATRIX_PEAKS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (queue) OPEN \
  \ ; channel: bed\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:DEEPTOOLS_PLOTHEATMAP_PEAKS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CALCULATE_FRIP\n  status=ACTIVE\n\
  \  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n\
  [process] NFCORE_CUTANDRUN:CUTANDRUN:CUT_CALC_REPROD\n  status=ACTIVE\n  port 0:\
  \ (queue) OPEN  ; channel: -\n  port 1: (cntrl) -     ; channel: $\n\n[process]\
  \ NFCORE_CUTANDRUN:CUTANDRUN:BEDTOOLS_INTERSECT\n  status=ACTIVE\n  port 0: (queue)\
  \ OPEN  ; channel: -\n  port 1: (value) bound ; channel: extension\n  port 2: (cntrl)\
  \ -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CALCULATE_PEAK_REPROD\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: -\n  port 1: (cntrl) -    \
  \ ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:EXPORT_META\n  status=ACTIVE\n\
  \  port 0: (value) OPEN  ; channel: meta\n  port 1: (value) bound ; channel: table_name\n\
  \  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:EXPORT_META_CTRL\n\
  \  status=ACTIVE\n  port 0: (value) OPEN  ; channel: meta\n  port 1: (value) bound\
  \ ; channel: table_name\n  port 2: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:GENERATE_REPORTS\n\
  \  status=ACTIVE\n  port 0: (value) OPEN  ; channel: meta_data\n  port 1: (value)\
  \ OPEN  ; channel: meta_data_ctrl\n  port 2: (value) OPEN  ; channel: raw_fragments\n\
  \  port 3: (value) OPEN  ; channel: bed_fragments\n  port 4: (value) OPEN  ; channel:\
  \ seacr_beds\n  port 5: (value) bound ; channel: frag_len_header_multiqc\n  port\
  \ 6: (cntrl) -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:CUSTOM_DUMPSOFTWAREVERSIONS\n\
  \  status=ACTIVE\n  port 0: (queue) OPEN  ; channel: versions\n  port 1: (cntrl)\
  \ -     ; channel: $\n\n[process] NFCORE_CUTANDRUN:CUTANDRUN:MULTIQC\n  status=ACTIVE\n\
  \  port 0: (value) bound ; channel: multiqc_config\n  port 1: (value) bound ; channel:\
  \ multiqc_custom_config\n  port 2: (value) OPEN  ; channel: software_versions\n\
  \  port 3: (value) OPEN  ; channel: software_versions_unique\n  port 4: (queue)\
  \ OPEN  ; channel: workflow_summary\n  port 5: (value) bound ; channel: fastqc/*\n\
  \  port 6: (value) bound ; channel: trimgalore/fastqc/*\n  port 7: (value) bound\
  \ ; channel: trimgalore/*\n  port 8: (value) OPEN  ; channel: bowtie2/*\n  port\
  \ 9: (value) OPEN  ; channel: bowtie2/*\n  port 10: (value) OPEN  ; channel: samtools/stats/*\n\
  \  port 11: (value) OPEN  ; channel: samtools/flagstat/*\n  port 12: (value) OPEN\
  \  ; channel: samtools/idxstats/*\n  port 13: (value) OPEN  ; channel: picard/markduplicates/*\n\
  \  port 14: (value) OPEN  ; channel: reports/*\n  port 15: (cntrl) -     ; channel:\
  \ $\n\nFeb-20 07:31:42.601 [main] DEBUG nextflow.Session - Session await > all processes\
  \ finished\nFeb-20 07:31:42.601 [main] DEBUG nextflow.Session - Session await >\
  \ all barriers passed\nFeb-20 07:31:42.745 [Actor Thread 11] DEBUG nextflow.sort.BigSort\
  \ - Sort completed -- entries: 4; slices: 1; internal sort time: 0.009 s; external\
  \ sort time: 0.001 s; total time: 0.01 s\nFeb-20 07:31:42.759 [Actor Thread 11]\
  \ DEBUG nextflow.file.FileCollector - Saved collect-files list to: /home/azureuser/tmp/avoyss1vbj/singularity-tmp/4f8d2f330b186da244495630d8e7d033.collect-file\n\
  Feb-20 07:31:42.768 [Actor Thread 11] DEBUG nextflow.file.FileCollector - Deleting\
  \ file collector temp dir: /home/azureuser/tmp/avoyss1vbj/singularity-tmp/nxf-5802214031459632750\n\
  Feb-20 07:31:42.797 [main] INFO  nextflow.Nextflow - -\e[0;35m[nf-core/cutandrun]\e\
  [0;31m Pipeline completed with errors\e[0m-\nFeb-20 07:31:42.803 [main] DEBUG nextflow.trace.WorkflowStatsObserver\
  \ - Workflow completed > WorkflowStats[succeededCount=5; failedCount=0; ignoredCount=0;\
  \ cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0;\
  \ abortedCount=0; succeedDuration=2m 6s; failedDuration=0ms; cachedDuration=0ms;loadCpus=0;\
  \ loadMemory=0; peakRunning=2; peakCpus=4; peakMemory=12 GB; ]\nFeb-20 07:31:42.804\
  \ [main] DEBUG nextflow.trace.TraceFileObserver - Workflow completed -- saving trace\
  \ file\nFeb-20 07:31:42.805 [main] DEBUG nextflow.trace.ReportObserver - Workflow\
  \ completed -- rendering execution report\nFeb-20 07:31:42.854 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Execution report summary data:\n  [{\"cpuUsage\":{\"mean\":21.8,\"min\":21.8,\"\
  q1\":21.8,\"q2\":21.8,\"q3\":21.8,\"max\":21.8,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\"},\"process\":\"SAMPLESHEET_CHECK\",\"mem\":{\"mean\"\
  :2990080,\"min\":2990080,\"q1\":2990080,\"q2\":2990080,\"q3\":2990080,\"max\":2990080,\"\
  minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK (test-GSE145187-small.csv)\"\
  ,\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK (test-GSE145187-small.csv)\"\
  ,\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK (test-GSE145187-small.csv)\"\
  ,\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK (test-GSE145187-small.csv)\"\
  ,\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK (test-GSE145187-small.csv)\"\
  },\"memUsage\":{\"mean\":0.05,\"min\":0.05,\"q1\":0.05,\"q2\":0.05,\"q3\":0.05,\"\
  max\":0.05,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\"},\"timeUsage\":null,\"vmem\":{\"mean\":11579392,\"\
  min\":11579392,\"q1\":11579392,\"q2\":11579392,\"q3\":11579392,\"max\":11579392,\"\
  minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK (test-GSE145187-small.csv)\"\
  ,\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK (test-GSE145187-small.csv)\"\
  ,\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK (test-GSE145187-small.csv)\"\
  ,\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK (test-GSE145187-small.csv)\"\
  ,\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK (test-GSE145187-small.csv)\"\
  },\"reads\":{\"mean\":674223,\"min\":674223,\"q1\":674223,\"q2\":674223,\"q3\":674223,\"\
  max\":674223,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\"},\"cpu\":{\"mean\":43.6,\"min\":43.6,\"q1\":43.6,\"\
  q2\":43.6,\"q3\":43.6,\"max\":43.6,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\"},\"time\":null,\"writes\":{\"mean\":856,\"min\":856,\"\
  q1\":856,\"q2\":856,\"q3\":856,\"max\":856,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (test-GSE145187-small.csv)\"}},{\"cpuUsage\":{\"mean\":26.15,\"min\":26,\"q1\"\
  :26.08,\"q2\":26.15,\"q3\":26.23,\"max\":26.3,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\"},\"process\":\"TRIMGALORE\",\"mem\":{\"mean\":2902016,\"min\"\
  :2830336,\"q1\":2866176,\"q2\":2902016,\"q3\":2937856,\"max\":2973696,\"minLabel\"\
  :\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE (h3k27me3_R1)\",\"maxLabel\"\
  :\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE (igg_R1)\",\"q1Label\"\
  :\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE (h3k27me3_R1)\",\"q2Label\"\
  :\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE (h3k27me3_R1)\",\"q3Label\"\
  :\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE (h3k27me3_R1)\"},\"memUsage\"\
  :{\"mean\":0.05,\"min\":0.04,\"q1\":0.04,\"q2\":0.05,\"q3\":0.05,\"max\":0.05,\"\
  minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE (h3k27me3_R1)\"\
  ,\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE (igg_R1)\"\
  ,\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE (h3k27me3_R1)\"\
  ,\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE (h3k27me3_R1)\"\
  ,\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE (h3k27me3_R1)\"\
  },\"timeUsage\":{\"mean\":0.11,\"min\":0.11,\"q1\":0.11,\"q2\":0.11,\"q3\":0.11,\"\
  max\":0.11,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\"},\"vmem\":{\"mean\":11579392,\"min\":11579392,\"q1\":11579392,\"q2\"\
  :11579392,\"q3\":11579392,\"max\":11579392,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\"},\"reads\":{\"mean\":58125704.5,\"min\":58047861,\"q1\":58086782.75,\"\
  q2\":58125704.5,\"q3\":58164626.25,\"max\":58203548,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\"},\"cpu\":{\"mean\":52.3,\"min\":52,\"q1\":52.15,\"q2\":52.3,\"q3\"\
  :52.45,\"max\":52.6,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\"},\"time\":{\"mean\":23500,\"min\":23000,\"q1\":23250,\"q2\":23500,\"\
  q3\":23750,\"max\":24000,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\"},\"writes\":{\"mean\":13136571.5,\"min\":13051976,\"q1\":13094273.75,\"\
  q2\":13136571.5,\"q3\":13178869.25,\"max\":13221167,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (h3k27me3_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\",\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:TRIMGALORE\
  \ (igg_R1)\"}},{\"cpuUsage\":{\"mean\":39.98,\"min\":39.1,\"q1\":39.54,\"q2\":39.98,\"\
  q3\":40.41,\"max\":40.85,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (igg_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (h3k27me3_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (igg_R1)\",\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (igg_R1)\"\
  ,\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (igg_R1)\"},\"\
  process\":\"FASTQC\",\"mem\":{\"mean\":2981888,\"min\":2973696,\"q1\":2977792,\"\
  q2\":2981888,\"q3\":2985984,\"max\":2990080,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (h3k27me3_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (igg_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  },\"memUsage\":{\"mean\":0.05,\"min\":0.05,\"q1\":0.05,\"q2\":0.05,\"q3\":0.05,\"\
  max\":0.05,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (igg_R1)\",\"\
  q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  },\"timeUsage\":{\"mean\":0.04,\"min\":0.04,\"q1\":0.04,\"q2\":0.04,\"q3\":0.04,\"\
  max\":0.04,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (igg_R1)\",\"\
  q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  },\"vmem\":{\"mean\":11579392,\"min\":11579392,\"q1\":11579392,\"q2\":11579392,\"\
  q3\":11579392,\"max\":11579392,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (h3k27me3_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (igg_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  },\"reads\":{\"mean\":21733508,\"min\":21731431,\"q1\":21732469.5,\"q2\":21733508,\"\
  q3\":21734546.5,\"max\":21735585,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (h3k27me3_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (igg_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  },\"cpu\":{\"mean\":79.95,\"min\":78.2,\"q1\":79.08,\"q2\":79.95,\"q3\":80.83,\"\
  max\":81.7,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (igg_R1)\"\
  ,\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (igg_R1)\",\"\
  q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (igg_R1)\",\"q3Label\"\
  :\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (igg_R1)\"},\"time\":{\"\
  mean\":8000,\"min\":8000,\"q1\":8000,\"q2\":8000,\"q3\":8000,\"max\":8000,\"minLabel\"\
  :\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\",\"maxLabel\"\
  :\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (igg_R1)\",\"q1Label\":\"\
  NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\",\"q2Label\"\
  :\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\",\"q3Label\"\
  :\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"},\"writes\"\
  :{\"mean\":3260060,\"min\":3256485,\"q1\":3258272.5,\"q2\":3260060,\"q3\":3261847.5,\"\
  max\":3263635,\"minLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (h3k27me3_R1)\",\"maxLabel\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC\
  \ (igg_R1)\",\"q1Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q2Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  ,\"q3Label\":\"NFCORE_CUTANDRUN:CUTANDRUN:FASTQC_TRIMGALORE:FASTQC (h3k27me3_R1)\"\
  }}]\nFeb-20 07:31:43.608 [main] DEBUG nextflow.trace.TimelineObserver - Workflow\
  \ completed -- rendering execution timeline\nFeb-20 07:31:44.506 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 07:31:44.527 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cutandrun%2Fewpei5ci%2Flogs.tar.xz">0.109GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cutandrun%2Fewpei5ci%2Foutput.tar.xz">0.004GiB</a>
						</td>
						<td>1.221GiB</td>
						<td>171.2 seconds</td>
						<td>10802.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738299328">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.6.2">v3.6.2</a>
						</td>
						<td>snakemake</td>
						<td>395 days</td>
						<td>2023-02-19 15:54:54</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/2hlyvzsbft/singularity-tmp/tmpboebpkcs.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/14a59a02-8f41-4b64-ab76-c1af5d709968\
  \ singularity  exec --home /home/azureuser/tmp/2hlyvzsbft/code/.test  /home/azureuser/tmp/2hlyvzsbft/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/2hlyvzsbft/code/.test/.snakemake/conda/b774f0a945c93d47f865b9735c2c3405_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/2hlyvzsbft/code/.test/.snakemake/conda/b774f0a945c93d47f865b9735c2c3405_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining parallel=20190522:   0%|    \
  \      | 0/9 [00:00<?, ?it/s]\nExamining bedtools[version='>=2.29']:  11%|\u2588\
  \         | 1/9 [00:00<00:00, 1354.31it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00, 29.57it/s] \nExamining bcftools=1.11:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 44.22it/s]            \
  \  \nExamining @/linux-64::__unix==0=0:  44%|\u2588\u2588\u2588\u2588\u258D    \
  \ | 4/9 [00:00<00:00, 53.65it/s]\nExamining freebayes=1.3.2:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:00<00:00, 66.92it/s]        \nExamining @/linux-64::__linux==5.15.0=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:00<00:00, 18.78it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:00<00:00, 21.90it/s]\nExamining sed=4.7:  78%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:00<00:00, 21.90it/s]           \
  \           \nExamining @/linux-64::__archspec==1=x86_64:  89%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00, 21.90it/s]\n          \
  \                                                                              \
  \ \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for freebayes parallel:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for parallel bedtools bcftools freebayes sed:  11%|\u2588         | 1/9 [00:00<00:00,\
  \ 2788.77it/s]\nExamining conflict for freebayes bcftools parallel:  22%|\u2588\u2588\
  \u258F       | 2/9 [00:00<00:00,  7.86it/s]               \nExamining conflict for\
  \ freebayes bcftools parallel:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00,\
  \ 11.78it/s]\nExamining conflict for bcftools parallel:  33%|\u2588\u2588\u2588\u258E\
  \      | 3/9 [00:00<00:00, 11.78it/s]          \nExamining conflict for freebayes\
  \ bedtools bcftools:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,\
  \ 11.78it/s]\nExamining conflict for freebayes bedtools bcftools:  56%|\u2588\u2588\
  \u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00,  6.96it/s]\nExamining conflict for\
  \ bedtools freebayes:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:01<00:00,\
  \  6.96it/s]         \nExamining conflict for bedtools freebayes:  67%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258B   | 6/9 [00:01<00:01,  2.27it/s]\nExamining conflict\
  \ for freebayes bedtools sed bcftools:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 6/9 [00:02<00:01,  2.27it/s]\nExamining conflict for freebayes bedtools\
  \ sed bcftools:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:02<00:00,\
  \  2.60it/s]\nExamining conflict for bedtools __glibc bcftools freebayes sed:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:02<00:00,  2.60it/s]\nExamining\
  \ conflict for bedtools __glibc bcftools freebayes sed:  89%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2589 | 8/9 [00:02<00:00,  2.49it/s]\nExamining conflict\
  \ for freebayes bcftools:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589\
  \ | 8/9 [00:02<00:00,  2.49it/s]                     \nExamining conflict for freebayes\
  \ bcftools: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9\
  \ [00:02<00:00,  2.79it/s]\n                                                   \
  \                                     \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining parallel=20190522:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ bedtools[version='>=2.29']:  11%|\u2588         | 1/9 [00:00<00:00, 1524.09it/s]\n\
  Examining @/linux-64::__glibc==2.31=0:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,\
  \ 29.13it/s] \nExamining bcftools=1.11:  33%|\u2588\u2588\u2588\u258E      | 3/9\
  \ [00:00<00:00, 43.56it/s]              \nExamining @/linux-64::__unix==0=0:  44%|\u2588\
  \u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00, 52.93it/s]\nExamining freebayes=1.3.2:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00, 66.02it/s]  \
  \      \nExamining @/linux-64::__linux==5.15.0=0:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:00<00:00, 18.50it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:00<00:00, 21.58it/s]\n\
  Examining sed=4.7:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9\
  \ [00:00<00:00, 21.58it/s]                      \nExamining @/linux-64::__archspec==1=x86_64:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:00<00:00,\
  \ 21.58it/s]\n                                                                 \
  \                        \n\nDetermining conflicts:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for freebayes parallel:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining conflict for parallel bedtools bcftools freebayes sed:  11%|\u2588\
  \         | 1/9 [00:00<00:00, 2928.98it/s]\nExamining conflict for freebayes bcftools\
  \ parallel:  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:00,  7.94it/s]       \
  \        \nExamining conflict for freebayes bcftools parallel:  33%|\u2588\u2588\
  \u2588\u258E      | 3/9 [00:00<00:00, 11.90it/s]\nExamining conflict for bcftools\
  \ parallel:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00, 11.90it/s]  \
  \        \nExamining conflict for freebayes bedtools bcftools:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:00<00:00, 11.90it/s]\nExamining conflict for freebayes\
  \ bedtools bcftools:  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:00<00:00,\
  \  6.95it/s]\nExamining conflict for bedtools freebayes:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:02<00:00,  6.95it/s]         \nExamining conflict\
  \ for bedtools freebayes:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9\
  \ [00:02<00:01,  2.04it/s]\nExamining conflict for freebayes bedtools sed bcftools:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:02<00:01,  2.04it/s]\n\
  Examining conflict for freebayes bedtools sed bcftools:  78%|\u2588\u2588\u2588\u2588\
  \u2588\u2588\u2588\u258A  | 7/9 [00:02<00:00,  2.36it/s]\nExamining conflict for\
  \ bedtools __glibc bcftools freebayes sed:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258A  | 7/9 [00:02<00:00,  2.36it/s]\nExamining conflict for bedtools __glibc\
  \ bcftools freebayes sed:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589\
  \ | 8/9 [00:02<00:00,  2.32it/s]\nExamining conflict for freebayes bcftools:  89%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:03<00:00,  2.32it/s]\
  \                     \nExamining conflict for freebayes bcftools: 100%|\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:03<00:00,  2.63it/s]\n\
  \                                                                              \
  \          \nfailed\n\nUnsatisfiableError: The following specifications were found\
  \ to be incompatible with each other:\n\nOutput in format: Requested package ->\
  \ Available versions\n\nPackage libgcc-ng conflicts for:\nsed=4.7 -> libgcc-ng[version='>=7.3.0']\n\
  parallel=20190522 -> perl -> libgcc-ng[version='>=11.2.0|>=4.9|>=7.3.0|>=7.5.0|>=9.3.0|>=9.4.0|>=7.2.0']\n\
  bcftools=1.11 -> gsl[version='>=2.6,<2.7.0a0'] -> libgcc-ng[version='>=10.3.0|>=11.2.0|>=4.9|>=7.3.0|>=9.3.0|>=9.4.0|>=7.2.0|>=12']\n\
  freebayes=1.3.2 -> libgcc-ng[version='>=7.3.0|>=7.5.0']\nfreebayes=1.3.2 -> bzip2[version='>=1.0.8,<2.0a0']\
  \ -> libgcc-ng[version='>=10.3.0|>=12|>=9.3.0|>=9.4.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  bcftools=1.11 -> libgcc-ng[version='>=7.5.0']\nbedtools[version='>=2.29'] -> libgcc-ng[version='>=10.3.0|>=9.4.0|>=9.3.0|>=7.5.0|>=7.3.0']\n\
  bedtools[version='>=2.29'] -> libzlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=11.2.0|>=12|>=4.9|>=7.2.0']\n\
  \nPackage _libgcc_mutex conflicts for:\nbcftools=1.11 -> libgcc-ng[version='>=7.5.0']\
  \ -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\nsed=4.7 ->\
  \ libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  freebayes=1.3.2 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  bedtools[version='>=2.29'] -> libgcc-ng[version='>=10.3.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main|main']\n\
  \nPackage openssl conflicts for:\nbcftools=1.11 -> htslib[version='>=1.11,<1.12.0a0']\
  \ -> openssl[version='>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a']\nbedtools[version='>=2.29']\
  \ -> curl[version='>=7.64.1,<8.0a0'] -> openssl[version='>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=3.0.8,<4.0a0|>=3.0.7,<4.0a0|>=3.0.5,<4.0a0|>=3.0.3,<4.0a0|>=3.0.2,<4.0a0|>=3.0.0,<4.0a0']\n\
  freebayes=1.3.2 -> htslib -> openssl[version='1.0.*|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=3.0.7,<4.0a0|>=1.1.1s,<1.1.2a|>=3.0.2,<4.0a0|>=3.0.0,<4.0a0|>=1.1.1f,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.0.2n,<1.0.3a|>=1.0.2m,<1.0.3a|>=1.0.2l,<1.0.3a']\n\
  \nPackage zlib conflicts for:\nfreebayes=1.3.2 -> zlib[version='>=1.2.11,<1.3.0a0']\n\
  freebayes=1.3.2 -> htslib -> zlib[version='1.2.*|1.2.11|1.2.11.*|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0|1.2.8']\n\
  \nPackage gdbm conflicts for:\nbcftools=1.11 -> perl -> gdbm[version='>=1.18|>=1.18,<1.19.0a0']\n\
  parallel=20190522 -> perl -> gdbm[version='>=1.18|>=1.18,<1.19.0a0']\n\nPackage\
  \ parallel conflicts for:\nparallel=20190522\nfreebayes=1.3.2 -> parallel\n\nPackage\
  \ libnsl conflicts for:\nparallel=20190522 -> perl -> libnsl[version='>=2.0.0,<2.1.0a0']\n\
  freebayes=1.3.2 -> python[version='>=3.8,<3.9.0a0'] -> libnsl[version='>=2.0.0,<2.1.0a0']\n\
  bcftools=1.11 -> perl -> libnsl[version='>=2.0.0,<2.1.0a0']\n\nPackage libstdcxx-ng\
  \ conflicts for:\nfreebayes=1.3.2 -> libstdcxx-ng[version='>=7.3.0|>=7.5.0']\nbedtools[version='>=2.29']\
  \ -> libstdcxx-ng[version='>=10.3.0|>=9.4.0|>=9.3.0|>=7.5.0|>=7.3.0']\nfreebayes=1.3.2\
  \ -> python[version='>=3.8,<3.9.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=9.3.0|>=9.4.0|>=4.9|>=7.2.0']\n\
  \nPackage htslib conflicts for:\nfreebayes=1.3.2 -> htslib\nbcftools=1.11 -> htslib[version='>=1.11,<1.12.0a0']\n\
  \nPackage xz conflicts for:\nfreebayes=1.3.2 -> htslib -> xz[version='5.2.*|>=5.2.3,<5.3.0a0|>=5.2.6,<5.3.0a0|>=5.2.5,<6.0a0|>=5.2.6,<6.0a0|>=5.2.10,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n\
  freebayes=1.3.2 -> xz[version='>=5.2.4,<5.3.0a0|>=5.2.5,<5.3.0a0']\n\nPackage bzip2\
  \ conflicts for:\nfreebayes=1.3.2 -> htslib -> bzip2[version='1.0.*|>=1.0.6,<2.0a0']\n\
  freebayes=1.3.2 -> bzip2[version='>=1.0.8,<2.0a0']The following specifications were\
  \ found to be incompatible with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n\
  \  - feature:|@/linux-64::__glibc==2.31=0\n  - bcftools=1.11 -> libgcc-ng[version='>=7.5.0']\
  \ -> __glibc[version='>=2.17']\n  - bedtools[version='>=2.29'] -> libgcc-ng[version='>=10.3.0']\
  \ -> __glibc[version='>=2.17']\n  - freebayes=1.3.2 -> libgcc-ng[version='>=7.5.0']\
  \ -> __glibc[version='>=2.17']\n  - sed=4.7 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \nYour installed version is: 2.31\n\nNote that strict channel priority may have\
  \ removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F59n2y5un%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F59n2y5un%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.248GiB</td>
						<td>313.2 seconds</td>
						<td>314.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743104224">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.4.2">2.4.2</a>
						</td>
						<td>nextflow</td>
						<td>391 days</td>
						<td>2023-02-19 06:27:13</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Feq5no5i6%2Flogs.tar.xz">0.015GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Feq5no5i6%2Foutput.tar.xz">0.012GiB</a>
						</td>
						<td>1.698GiB</td>
						<td>828.6 seconds</td>
						<td>378.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746488208">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/2.2.0">2.2.0</a>
						</td>
						<td>nextflow</td>
						<td>384 days</td>
						<td>2023-02-19 09:57:55</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-19 10:57:55.201
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fmupuxxfu%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fmupuxxfu%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.039GiB</td>
						<td>60.4 seconds</td>
						<td>3600.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743195264">
								<a href="https://nf-co.re/viralrecon">viralrecon</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/viralrecon/tree/2.3">2.3</a>
						</td>
						<td>nextflow</td>
						<td>380 days</td>
						<td>2023-02-20 05:34:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 08:34:09.160
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Foi781at6%2Flogs.tar.xz">0.081GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Foi781at6%2Foutput.tar.xz">0.011GiB</a>
						</td>
						<td>1.260GiB</td>
						<td>308.5 seconds</td>
						<td>10801.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741144576">
								<a href="https://github.com/hzi-bifo/snakemake-workflow-univec-removal">hzi-bifo/snakemake-workflow-univec-removal</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/hzi-bifo/snakemake-workflow-univec-removal/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>380 days</td>
						<td>2023-02-19 03:31:11</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hzi-bifo%252Fsnakemake-workflow-univec-removal%2Fdoql29qi%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hzi-bifo%252Fsnakemake-workflow-univec-removal%2Fdoql29qi%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.105GiB</td>
						<td>420.1 seconds</td>
						<td>460.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738268720">
								<a href="https://github.com/GooglingTheCancerGenome/sv-callers">GooglingTheCancerGenome/sv-callers</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.1.3">v1.1.3</a>
						</td>
						<td>snakemake</td>
						<td>375 days</td>
						<td>2023-02-19 06:15:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: ModuleNotFoundError
line_no: 7
rest: "No module named 'yatiml'\n  File \"/home/azureuser/tmp/wrhfb27wax/code/workflow/Snakefile\"\
  , line 7, in <module>\n  File \"/home/azureuser/tmp/wrhfb27wax/code/workflow/helper_functions.py\"\
  , line 5, in <module>\n  File \"/home/azureuser/tmp/wrhfb27wax/code/workflow/validator/__init__.py\"\
  , line 8, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2F83p96pgp%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2F83p96pgp%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.061GiB</td>
						<td>1.5 seconds</td>
						<td>2.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738299088">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.6.3">v3.6.3</a>
						</td>
						<td>snakemake</td>
						<td>374 days</td>
						<td>2023-02-19 04:33:52</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/coxtdwdsjg/singularity-tmp/tmp437u318s.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/8858288b-b993-42f8-96e6-9dbfa5d74aba\
  \ singularity  exec --home /home/azureuser/tmp/coxtdwdsjg/code/.test  /home/azureuser/tmp/coxtdwdsjg/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/coxtdwdsjg/code/.test/.snakemake/conda/5673a45a877993b8362b175d728701a5_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/coxtdwdsjg/code/.test/.snakemake/conda/5673a45a877993b8362b175d728701a5_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|\
  \          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:  20%|\u2588\
  \u2588        | 1/5 [00:00<00:00, 5023.12it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 5924.16it/s]\nExamining\
  \ delly==0.9.1:  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00,\
  \ 6529.79it/s]                    \nExamining @/linux-64::__linux==5.15.0=0:  80%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 75.94it/s]\n   \
  \                                                                              \
  \     \n\nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining\
  \ conflict for __glibc delly:   0%|          | 0/5 [00:00<?, ?it/s]\n          \
  \                                                                 \nfailed\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|\
  \          | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__glibc==2.31=0:  20%|\u2588\
  \u2588        | 1/5 [00:00<00:00, 4865.78it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 5518.82it/s]\nExamining\
  \ delly==0.9.1:  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5 [00:00<00:00,\
  \ 5943.75it/s]                    \nExamining @/linux-64::__linux==5.15.0=0:  80%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 68.15it/s]\n   \
  \                                                                              \
  \     \n\nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining\
  \ conflict for __glibc delly:   0%|          | 0/5 [00:00<?, ?it/s]\n          \
  \                                                                 \nfailed\n\nUnsatisfiableError:\
  \ The following specifications were found to be incompatible with your system:\n\
  \n  - feature:/linux-64::__glibc==2.31=0\n  - delly==0.9.1 -> libgcc-ng[version='>=10.3.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F4gffzv80%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F4gffzv80%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.231GiB</td>
						<td>467.5 seconds</td>
						<td>682.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738215728">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>374 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fgrzxse49%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fgrzxse49%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.125GiB</td>
						<td>414.3 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738210688">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>374 days</td>
						<td>2023-02-19 04:14:26</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/6bbjvhp934/code/workflow/rules/../envs/stats.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/6bbjvhp934/code/.test/.snakemake/conda/c42c9db2352ef78327e8b1a402265f74_.yaml"
  --prefix "/home/azureuser/tmp/6bbjvhp934/code/.test/.snakemake/conda/c42c9db2352ef78327e8b1a402265f74_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working... done


  [Errno 2] No such file or directory: ''/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/qt-main-5.15.8-h5d23da1_6/.cph_tmpwqot2896/pkg-qt-main-5.15.8-h5d23da1_6.tar.zst'''

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fodax7jeb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fodax7jeb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.160GiB</td>
						<td>900.0 seconds</td>
						<td>1325.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740539088">
								<a href="https://github.com/mrvollger/Rhodonite">mrvollger/Rhodonite</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/Rhodonite/tree/v0.12-alpha">v0.12-alpha</a>
						</td>
						<td>snakemake</td>
						<td>374 days</td>
						<td>2023-02-19 03:55:08</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: SystemExit
line_no: 38
rest: "1\n  File \"/home/azureuser/tmp/53dvwdunrt/code/workflow/Snakefile\", line\
  \ 38, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Fgt4n5thm%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FRhodonite%2Fgt4n5thm%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.9 seconds</td>
						<td>4.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743195024">
								<a href="https://nf-co.re/viralrecon">viralrecon</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/viralrecon/tree/2.3.1">2.3.1</a>
						</td>
						<td>nextflow</td>
						<td>369 days</td>
						<td>2023-02-19 23:32:30</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 02:32:30.521
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Fjihbws3r%2Flogs.tar.xz">0.081GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Fjihbws3r%2Foutput.tar.xz">0.011GiB</a>
						</td>
						<td>1.260GiB</td>
						<td>326.5 seconds</td>
						<td>10802.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738298848">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.7.0">v3.7.0</a>
						</td>
						<td>snakemake</td>
						<td>367 days</td>
						<td>2023-02-19 06:05:06</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/orihklga9b/singularity-tmp/tmpcvta124z.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/48de59af-cb84-4536-ae55-74a639b6bd60\
  \ singularity  exec --home /home/azureuser/tmp/orihklga9b/code/.test  /home/azureuser/tmp/orihklga9b/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/orihklga9b/code/.test/.snakemake/conda/b17f600313e1e2c3cd593683a69c2e3f_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/orihklga9b/code/.test/.snakemake/conda/b17f600313e1e2c3cd593683a69c2e3f_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining samtools==1.10:   0%|       \
  \   | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:  20%|\u2588\
  \u2588        | 1/5 [00:00<00:00, 21.59it/s]\nExamining @/linux-64::__unix==0=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 43.00it/s]      \nExamining\
  \ @/linux-64::__glibc==2.31=0:  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5\
  \ [00:00<00:00, 64.35it/s]\nExamining @/linux-64::__archspec==1=x86_64:  80%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 85.60it/s]\n   \
  \                                                                              \
  \        \n\nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining\
  \ conflict for samtools __glibc:   0%|          | 0/5 [00:00<?, ?it/s]\n       \
  \                                                                       \nfailed\n\
  Solving environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/5 [00:00<?, ?it/s]\nExamining samtools==1.10:   0%|       \
  \   | 0/5 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:  20%|\u2588\
  \u2588        | 1/5 [00:00<00:00, 34.78it/s]\nExamining @/linux-64::__unix==0=0:\
  \  40%|\u2588\u2588\u2588\u2588      | 2/5 [00:00<00:00, 69.04it/s]      \nExamining\
  \ @/linux-64::__glibc==2.31=0:  60%|\u2588\u2588\u2588\u2588\u2588\u2588    | 3/5\
  \ [00:00<00:00, 103.03it/s]\nExamining @/linux-64::__archspec==1=x86_64:  80%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588  | 4/5 [00:00<00:00, 136.80it/s]\n  \
  \                                                                              \
  \          \n\nDetermining conflicts:   0%|          | 0/5 [00:00<?, ?it/s]\nExamining\
  \ conflict for samtools __glibc:   0%|          | 0/5 [00:00<?, ?it/s]\n       \
  \                                                                       \nfailed\n\
  \nUnsatisfiableError: The following specifications were found to be incompatible\
  \ with your system:\n\n  - feature:/linux-64::__glibc==2.31=0\n  - samtools==1.10\
  \ -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n\nYour installed\
  \ version is: 2.31\n\nNote that strict channel priority may have removed packages\
  \ required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F2sbtqn58%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F2sbtqn58%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.282GiB</td>
						<td>262.1 seconds</td>
						<td>504.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742777264">
								<a href="https://github.com/solida-core/musta">solida-core/musta</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/musta/tree/v1.0">v1.0</a>
						</td>
						<td>snakemake</td>
						<td>366 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 8
rest: "Workflow defines configfile ../config/config.yaml but it is not present or\
  \ accessible.\n  File \"/home/azureuser/tmp/tkmurssup5/code/workflow/Snakefile\"\
  , line 8, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2Fxk6qplil%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2Fxk6qplil%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.6 seconds</td>
						<td>2.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743194784">
								<a href="https://nf-co.re/viralrecon">viralrecon</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/viralrecon/tree/2.4">2.4</a>
						</td>
						<td>nextflow</td>
						<td>363 days</td>
						<td>2023-02-20 04:31:43</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867579211\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\t\
  at nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\
  \tat nextflow.processor.TaskRun$isContainerEnabled$9.call(Unknown Source)\n\tat\
  \ nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867579211\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 07:31:43.745 [main] DEBUG nextflow.trace.WorkflowStatsObserver\
  \ - Workflow completed > WorkflowStats[succeededCount=18; failedCount=0; ignoredCount=0;\
  \ cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0;\
  \ abortedCount=0; succeedDuration=4m 48s; failedDuration=0ms; cachedDuration=0ms;loadCpus=0;\
  \ loadMemory=0; peakRunning=2; peakCpus=4; peakMemory=12 GB; ]\nFeb-20 07:31:43.746\
  \ [main] DEBUG nextflow.trace.TraceFileObserver - Workflow completed -- saving trace\
  \ file\nFeb-20 07:31:43.753 [main] DEBUG nextflow.trace.ReportObserver - Workflow\
  \ completed -- rendering execution report\nFeb-20 07:31:43.776 [Task submitter]\
  \ DEBUG n.processor.TaskPollingMonitor - %% executor local > tasks in the submission\
  \ queue: 2 -- tasks to be submitted are shown below\n~> TaskHandler[id: 25; name:\
  \ NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_IDXSTATS\
  \ (SAMPLE2_PE); status: NEW; exit: -; error: -; workDir: /home/azureuser/tmp/j95brtnwde/code/work/73/9941296866146cf075a4afb68236bf]\n\
  ~> TaskHandler[id: 28; name: NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:SAMTOOLS_FLAGSTAT\
  \ (SAMPLE1_PE); status: NEW; exit: -; error: -; workDir: /home/azureuser/tmp/j95brtnwde/code/work/52/d85309bc8c650315ebec1f9d9cf390]\n\
  Feb-20 07:31:43.848 [main] DEBUG nextflow.trace.ReportObserver - Execution report\
  \ summary data:\n  [{\"cpuUsage\":{\"mean\":25,\"min\":25,\"q1\":25,\"q2\":25,\"\
  q3\":25,\"max\":25,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\"},\"process\":\"SAMPLESHEET_CHECK\"\
  ,\"mem\":{\"mean\":5111808,\"min\":5111808,\"q1\":5111808,\"q2\":5111808,\"q3\"\
  :5111808,\"max\":5111808,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\"},\"memUsage\":{\"mean\":0.08,\"min\"\
  :0.08,\"q1\":0.08,\"q2\":0.08,\"q3\":0.08,\"max\":0.08,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\"},\"timeUsage\":null,\"vmem\":{\"mean\"\
  :14102528,\"min\":14102528,\"q1\":14102528,\"q2\":14102528,\"q3\":14102528,\"max\"\
  :14102528,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\"},\"reads\":{\"mean\":1512267,\"min\"\
  :1512267,\"q1\":1512267,\"q2\":1512267,\"q3\":1512267,\"max\":1512267,\"minLabel\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test_illumina_amplicon.csv)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test_illumina_amplicon.csv)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test_illumina_amplicon.csv)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test_illumina_amplicon.csv)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test_illumina_amplicon.csv)\"\
  },\"cpu\":{\"mean\":50,\"min\":50,\"q1\":50,\"q2\":50,\"q3\":50,\"max\":50,\"minLabel\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test_illumina_amplicon.csv)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test_illumina_amplicon.csv)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test_illumina_amplicon.csv)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test_illumina_amplicon.csv)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test_illumina_amplicon.csv)\"\
  },\"time\":null,\"writes\":{\"mean\":1094,\"min\":1094,\"q1\":1094,\"q2\":1094,\"\
  q3\":1094,\"max\":1094,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test_illumina_amplicon.csv)\"}},{\"cpuUsage\":{\"mean\":46.9,\"min\"\
  :46.9,\"q1\":46.9,\"q2\":46.9,\"q3\":46.9,\"max\":46.9,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\"},\"process\":\"COLLAPSE_PRIMERS\",\"mem\":{\"mean\":5185536,\"\
  min\":5185536,\"q1\":5185536,\"q2\":5185536,\"q3\":5185536,\"max\":5185536,\"minLabel\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS (nCoV-2019.primer.bed)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS (nCoV-2019.primer.bed)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS (nCoV-2019.primer.bed)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS (nCoV-2019.primer.bed)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS (nCoV-2019.primer.bed)\"\
  },\"memUsage\":{\"mean\":0.08,\"min\":0.08,\"q1\":0.08,\"q2\":0.08,\"q3\":0.08,\"\
  max\":0.08,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\"},\"timeUsage\":null,\"vmem\":{\"mean\":14102528,\"min\"\
  :14102528,\"q1\":14102528,\"q2\":14102528,\"q3\":14102528,\"max\":14102528,\"minLabel\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS (nCoV-2019.primer.bed)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS (nCoV-2019.primer.bed)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS (nCoV-2019.primer.bed)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS (nCoV-2019.primer.bed)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS (nCoV-2019.primer.bed)\"\
  },\"reads\":{\"mean\":1503676,\"min\":1503676,\"q1\":1503676,\"q2\":1503676,\"q3\"\
  :1503676,\"max\":1503676,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\"},\"cpu\":{\"mean\":93.8,\"min\":93.8,\"q1\":93.8,\"q2\"\
  :93.8,\"q3\":93.8,\"max\":93.8,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\"},\"time\":null,\"writes\":{\"mean\":4207,\"min\":4207,\"\
  q1\":4207,\"q2\":4207,\"q3\":4207,\"max\":4207,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:COLLAPSE_PRIMERS\
  \ (nCoV-2019.primer.bed)\"}},{\"cpuUsage\":{\"mean\":13.8,\"min\":13.8,\"q1\":13.8,\"\
  q2\":13.8,\"q3\":13.8,\"max\":13.8,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\"},\"process\":\"BEDTOOLS_GETFASTA\",\"mem\":{\"mean\"\
  :4907008,\"min\":4907008,\"q1\":4907008,\"q2\":4907008,\"q3\":4907008,\"max\":4907008,\"\
  minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA (nCoV-2019.primer.bed)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA (nCoV-2019.primer.bed)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA (nCoV-2019.primer.bed)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA (nCoV-2019.primer.bed)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA (nCoV-2019.primer.bed)\"\
  },\"memUsage\":{\"mean\":0.08,\"min\":0.08,\"q1\":0.08,\"q2\":0.08,\"q3\":0.08,\"\
  max\":0.08,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\"},\"timeUsage\":null,\"vmem\":{\"mean\":27783168,\"min\"\
  :27783168,\"q1\":27783168,\"q2\":27783168,\"q3\":27783168,\"max\":27783168,\"minLabel\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA (nCoV-2019.primer.bed)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA (nCoV-2019.primer.bed)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA (nCoV-2019.primer.bed)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA (nCoV-2019.primer.bed)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA (nCoV-2019.primer.bed)\"\
  },\"reads\":{\"mean\":86452,\"min\":86452,\"q1\":86452,\"q2\":86452,\"q3\":86452,\"\
  max\":86452,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\"},\"cpu\":{\"mean\":27.6,\"min\":27.6,\"q1\":27.6,\"q2\"\
  :27.6,\"q3\":27.6,\"max\":27.6,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\"},\"time\":null,\"writes\":{\"mean\":9818,\"min\":9818,\"\
  q1\":9818,\"q2\":9818,\"q3\":9818,\"max\":9818,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BEDTOOLS_GETFASTA\
  \ (nCoV-2019.primer.bed)\"}},{\"cpuUsage\":{\"mean\":17.4,\"min\":17.4,\"q1\":17.4,\"\
  q2\":17.4,\"q3\":17.4,\"max\":17.4,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\"},\"process\":\"CUSTOM_GETCHROMSIZES\",\"mem\":{\"\
  mean\":3256320,\"min\":3256320,\"q1\":3256320,\"q2\":3256320,\"q3\":3256320,\"max\"\
  :3256320,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\"},\"memUsage\":{\"mean\":0.05,\"min\":0.05,\"q1\"\
  :0.05,\"q2\":0.05,\"q3\":0.05,\"max\":0.05,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\"},\"timeUsage\":null,\"vmem\":{\"mean\":5644288,\"\
  min\":5644288,\"q1\":5644288,\"q2\":5644288,\"q3\":5644288,\"max\":5644288,\"minLabel\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES (nCoV-2019.reference.fasta)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES (nCoV-2019.reference.fasta)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES (nCoV-2019.reference.fasta)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES (nCoV-2019.reference.fasta)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES (nCoV-2019.reference.fasta)\"\
  },\"reads\":{\"mean\":136368,\"min\":136368,\"q1\":136368,\"q2\":136368,\"q3\":136368,\"\
  max\":136368,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\"},\"cpu\":{\"mean\":34.8,\"min\":34.8,\"q1\":34.8,\"\
  q2\":34.8,\"q3\":34.8,\"max\":34.8,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\"},\"time\":null,\"writes\":{\"mean\":4854,\"min\"\
  :4854,\"q1\":4854,\"q2\":4854,\"q3\":4854,\"max\":4854,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:CUSTOM_GETCHROMSIZES\
  \ (nCoV-2019.reference.fasta)\"}},{\"cpuUsage\":{\"mean\":21.8,\"min\":21.8,\"q1\"\
  :21.8,\"q2\":21.8,\"q3\":21.8,\"max\":21.8,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\"},\"process\":\"BLAST_MAKEBLASTDB\",\"mem\":{\"mean\"\
  :4726784,\"min\":4726784,\"q1\":4726784,\"q2\":4726784,\"q3\":4726784,\"max\":4726784,\"\
  minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB (nCoV-2019.reference.fasta)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB (nCoV-2019.reference.fasta)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB (nCoV-2019.reference.fasta)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB (nCoV-2019.reference.fasta)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB (nCoV-2019.reference.fasta)\"\
  },\"memUsage\":{\"mean\":0.07,\"min\":0.07,\"q1\":0.07,\"q2\":0.07,\"q3\":0.07,\"\
  max\":0.07,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\"},\"timeUsage\":{\"mean\":0.01,\"min\":0.01,\"q1\"\
  :0.01,\"q2\":0.01,\"q3\":0.01,\"max\":0.01,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\"},\"vmem\":{\"mean\":28913664,\"min\":28913664,\"\
  q1\":28913664,\"q2\":28913664,\"q3\":28913664,\"max\":28913664,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\"},\"reads\":{\"mean\":242394,\"min\":242394,\"q1\"\
  :242394,\"q2\":242394,\"q3\":242394,\"max\":242394,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\"},\"cpu\":{\"mean\":43.6,\"min\":43.6,\"q1\":43.6,\"\
  q2\":43.6,\"q3\":43.6,\"max\":43.6,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\"},\"time\":{\"mean\":2000,\"min\":2000,\"q1\":2000,\"\
  q2\":2000,\"q3\":2000,\"max\":2000,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\"},\"writes\":{\"mean\":58080,\"min\":58080,\"q1\"\
  :58080,\"q2\":58080,\"q3\":58080,\"max\":58080,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BLAST_MAKEBLASTDB\
  \ (nCoV-2019.reference.fasta)\"}},{\"cpuUsage\":{\"mean\":37.15,\"min\":37.15,\"\
  q1\":37.15,\"q2\":37.15,\"q3\":37.15,\"max\":37.15,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\"},\"process\":\"BOWTIE2_BUILD\",\"mem\":{\"mean\"\
  :17489920,\"min\":17489920,\"q1\":17489920,\"q2\":17489920,\"q3\":17489920,\"max\"\
  :17489920,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\"},\"memUsage\":{\"mean\":0.27,\"min\":0.27,\"q1\"\
  :0.27,\"q2\":0.27,\"q3\":0.27,\"max\":0.27,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\"},\"timeUsage\":null,\"vmem\":{\"mean\":22745088,\"\
  min\":22745088,\"q1\":22745088,\"q2\":22745088,\"q3\":22745088,\"max\":22745088,\"\
  minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD (nCoV-2019.reference.fasta)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD (nCoV-2019.reference.fasta)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD (nCoV-2019.reference.fasta)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD (nCoV-2019.reference.fasta)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD (nCoV-2019.reference.fasta)\"\
  },\"reads\":{\"mean\":3212856,\"min\":3212856,\"q1\":3212856,\"q2\":3212856,\"q3\"\
  :3212856,\"max\":3212856,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\"},\"cpu\":{\"mean\":74.3,\"min\":74.3,\"q1\":74.3,\"\
  q2\":74.3,\"q3\":74.3,\"max\":74.3,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\"},\"time\":{\"mean\":1000,\"min\":1000,\"q1\":1000,\"\
  q2\":1000,\"q3\":1000,\"max\":1000,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\"},\"writes\":{\"mean\":8694206,\"min\":8694206,\"\
  q1\":8694206,\"q2\":8694206,\"q3\":8694206,\"max\":8694206,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:PREPARE_GENOME:BOWTIE2_BUILD\
  \ (nCoV-2019.reference.fasta)\"}},{\"cpuUsage\":{\"mean\":51.68,\"min\":48.5,\"\
  q1\":50.09,\"q2\":51.68,\"q3\":53.26,\"max\":54.85,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\",\"q2Label\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE1_PE)\"},\"process\":\"FASTP\",\"mem\":{\"mean\":636579840,\"min\":59977728,\"\
  q1\":348278784,\"q2\":636579840,\"q3\":924880896,\"max\":1213181952,\"minLabel\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\",\"maxLabel\":\"\
  NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\"},\"\
  memUsage\":{\"mean\":9.88,\"min\":0.93,\"q1\":5.41,\"q2\":9.88,\"q3\":14.36,\"max\"\
  :18.83,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\",\"\
  q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\",\"q2Label\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE1_PE)\"},\"timeUsage\":{\"mean\":0.02,\"min\":0.01,\"q1\":0.02,\"q2\":0.02,\"\
  q3\":0.02,\"max\":0.03,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\",\"q2Label\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE2_PE)\"},\"vmem\":{\"mean\":794068992,\"min\":65286144,\"q1\":429677568,\"\
  q2\":794068992,\"q3\":1158460416,\"max\":1522851840,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\",\"q2Label\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE1_PE)\"},\"reads\":{\"mean\":22205999.5,\"min\":18700383,\"q1\":20453191.25,\"\
  q2\":22205999.5,\"q3\":23958807.75,\"max\":25711616,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\",\"q2Label\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE2_PE)\"},\"cpu\":{\"mean\":103.35,\"min\":97,\"q1\":100.18,\"q2\":103.35,\"\
  q3\":106.53,\"max\":109.7,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\",\"q2Label\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE1_PE)\"},\"time\":{\"mean\":4500,\"min\":3000,\"q1\":3750,\"q2\":4500,\"\
  q3\":5250,\"max\":6000,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\",\"q2Label\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE2_PE)\"},\"writes\":{\"mean\":5622984,\"min\":4952197,\"q1\":5287590.5,\"\
  q2\":5622984,\"q3\":5958377.5,\"max\":6293771,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE1_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\",\"q2Label\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTP\
  \ (SAMPLE2_PE)\"}},{\"cpuUsage\":{\"mean\":96.98,\"min\":90.05,\"q1\":93.51,\"q2\"\
  :96.98,\"q3\":100.44,\"max\":103.9,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\"},\"process\":\"FASTQC_RAW\",\"mem\":{\"mean\":417929216,\"min\"\
  :384200704,\"q1\":401064960,\"q2\":417929216,\"q3\":434793472,\"max\":451657728,\"\
  minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE1_PE)\",\"\
  maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE2_PE)\",\"\
  q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE1_PE)\",\"\
  q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE1_PE)\",\"\
  q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE1_PE)\"},\"\
  memUsage\":{\"mean\":6.49,\"min\":5.96,\"q1\":6.23,\"q2\":6.49,\"q3\":6.75,\"max\"\
  :7.01,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE1_PE)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE2_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE1_PE)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE1_PE)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE1_PE)\"\
  },\"timeUsage\":{\"mean\":0.03,\"min\":0.02,\"q1\":0.03,\"q2\":0.03,\"q3\":0.03,\"\
  max\":0.03,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE2_PE)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE1_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE2_PE)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE2_PE)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW (SAMPLE2_PE)\"\
  },\"vmem\":{\"mean\":3595632640,\"min\":3583623168,\"q1\":3589627904,\"q2\":3595632640,\"\
  q3\":3601637376,\"max\":3607642112,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\"},\"reads\":{\"mean\":28547449.5,\"min\":27507032,\"q1\":28027240.75,\"\
  q2\":28547449.5,\"q3\":29067658.25,\"max\":29587867,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\"},\"cpu\":{\"mean\":193.95,\"min\":180.1,\"q1\":187.03,\"q2\":193.95,\"\
  q3\":200.88,\"max\":207.8,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\"},\"time\":{\"mean\":6000,\"min\":5000,\"q1\":5500,\"q2\":6000,\"\
  q3\":6500,\"max\":7000,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\"},\"writes\":{\"mean\":4891785,\"min\":4887485,\"q1\":4889635,\"\
  q2\":4891785,\"q3\":4893935,\"max\":4896085,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_RAW\
  \ (SAMPLE1_PE)\"}},{\"cpuUsage\":{\"mean\":68.03,\"min\":58.5,\"q1\":63.26,\"q2\"\
  :68.03,\"q3\":72.79,\"max\":77.55,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\"},\"process\":\"FASTQC_TRIM\",\"mem\":{\"mean\":419915776,\"min\"\
  :413917184,\"q1\":416916480,\"q2\":419915776,\"q3\":422915072,\"max\":425914368,\"\
  minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE1_PE)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE2_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE1_PE)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE1_PE)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE1_PE)\"\
  },\"memUsage\":{\"mean\":6.52,\"min\":6.42,\"q1\":6.47,\"q2\":6.52,\"q3\":6.56,\"\
  max\":6.61,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE1_PE)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE2_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE1_PE)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE1_PE)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE1_PE)\"\
  },\"timeUsage\":{\"mean\":0.04,\"min\":0.03,\"q1\":0.03,\"q2\":0.04,\"q3\":0.04,\"\
  max\":0.05,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE2_PE)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE1_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE2_PE)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE2_PE)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM (SAMPLE2_PE)\"\
  },\"vmem\":{\"mean\":3582869504,\"min\":3582869504,\"q1\":3582869504,\"q2\":3582869504,\"\
  q3\":3582869504,\"max\":3582869504,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\"},\"reads\":{\"mean\":27029691.5,\"min\":26407728,\"q1\":26718709.75,\"\
  q2\":27029691.5,\"q3\":27340673.25,\"max\":27651655,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\"},\"cpu\":{\"mean\":136.05,\"min\":117,\"q1\":126.53,\"q2\":136.05,\"\
  q3\":145.57,\"max\":155.1,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\"},\"time\":{\"mean\":8000,\"min\":6000,\"q1\":7000,\"q2\":8000,\"\
  q3\":9000,\"max\":10000,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\"},\"writes\":{\"mean\":4682837,\"min\":4667332,\"q1\":4675084.5,\"\
  q2\":4682837,\"q3\":4690589.5,\"max\":4698342,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:FASTQC_FASTP:FASTQC_TRIM\
  \ (SAMPLE1_PE)\"}},{\"cpuUsage\":{\"mean\":80.38,\"min\":74.65,\"q1\":77.51,\"q2\"\
  :80.38,\"q3\":83.24,\"max\":86.1,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\"},\"process\":\"BOWTIE2_ALIGN\",\"mem\":{\"mean\":100096000,\"min\"\
  :96890880,\"q1\":98493440,\"q2\":100096000,\"q3\":101698560,\"max\":103301120,\"\
  minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN (SAMPLE1_PE)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN (SAMPLE2_PE)\"\
  ,\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN (SAMPLE1_PE)\"\
  ,\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN (SAMPLE1_PE)\"\
  ,\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN (SAMPLE1_PE)\"\
  },\"memUsage\":{\"mean\":1.55,\"min\":1.5,\"q1\":1.53,\"q2\":1.55,\"q3\":1.58,\"\
  max\":1.6,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\"},\"timeUsage\":{\"mean\":0.23,\"min\":0.23,\"q1\":0.23,\"q2\":0.23,\"\
  q3\":0.23,\"max\":0.23,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\"},\"vmem\":{\"mean\":751151104,\"min\":751149056,\"q1\":751150080,\"\
  q2\":751151104,\"q3\":751152128,\"max\":751153152,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\"},\"reads\":{\"mean\":47237725,\"min\":42828546,\"q1\":45033135.5,\"\
  q2\":47237725,\"q3\":49442314.5,\"max\":51646904,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\"},\"cpu\":{\"mean\":160.75,\"min\":149.3,\"q1\":155.03,\"q2\":160.75,\"\
  q3\":166.48,\"max\":172.2,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\"},\"time\":{\"mean\":50000,\"min\":50000,\"q1\":50000,\"q2\":50000,\"\
  q3\":50000,\"max\":50000,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\"},\"writes\":{\"mean\":38324540,\"min\":33799744,\"q1\":36062142,\"\
  q2\":38324540,\"q3\":40586938,\"max\":42849336,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BOWTIE2_ALIGN\
  \ (SAMPLE2_PE)\"}},{\"cpuUsage\":{\"mean\":33.85,\"min\":5.2,\"q1\":19.52,\"q2\"\
  :33.85,\"q3\":48.18,\"max\":62.5,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\"},\"process\":\"SAMTOOLS_SORT\",\"mem\":{\"mean\":39174144,\"min\"\
  :36364288,\"q1\":37769216,\"q2\":39174144,\"q3\":40579072,\"max\":41984000,\"minLabel\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT (SAMPLE2_PE)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\"},\"memUsage\":{\"mean\":0.61,\"min\":0.56,\"q1\":0.59,\"q2\":0.61,\"\
  q3\":0.63,\"max\":0.65,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\"},\"timeUsage\":{\"mean\":0.01,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"\
  q3\":0.01,\"max\":0.01,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\"},\"vmem\":{\"mean\":2114615296,\"min\":2081521664,\"q1\":2098068480,\"\
  q2\":2114615296,\"q3\":2131162112,\"max\":2147708928,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\"},\"reads\":{\"mean\":5627366,\"min\":4841750,\"q1\":5234558,\"\
  q2\":5627366,\"q3\":6020174,\"max\":6412982,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\"},\"cpu\":{\"mean\":67.7,\"min\":10.4,\"q1\":39.05,\"q2\":67.7,\"\
  q3\":96.35,\"max\":125,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\"},\"time\":{\"mean\":1500,\"min\":0,\"q1\":750,\"q2\":1500,\"q3\"\
  :2250,\"max\":3000,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\"},\"writes\":{\"mean\":2921891,\"min\":2536459,\"q1\":2729175,\"\
  q2\":2921891,\"q3\":3114607,\"max\":3307323,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT\
  \ (SAMPLE2_PE)\"}},{\"cpuUsage\":{\"mean\":22.88,\"min\":5.75,\"q1\":14.31,\"q2\"\
  :22.88,\"q3\":31.44,\"max\":40,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\"},\"process\":\"SAMTOOLS_INDEX\",\"mem\":{\"mean\":5144576,\"min\"\
  :3350528,\"q1\":4247552,\"q2\":5144576,\"q3\":6041600,\"max\":6938624,\"minLabel\"\
  :\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX (SAMPLE1_PE)\"\
  ,\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\"},\"memUsage\":{\"mean\":0.08,\"min\":0.05,\"q1\":0.07,\"q2\":0.08,\"\
  q3\":0.09,\"max\":0.11,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\"},\"timeUsage\":{\"mean\":0.01,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"\
  q3\":0.01,\"max\":0.01,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\"},\"vmem\":{\"mean\":118568960,\"min\":5644288,\"q1\":62106624,\"\
  q2\":118568960,\"q3\":175031296,\"max\":231493632,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\"},\"reads\":{\"mean\":3022580.5,\"min\":2637149,\"q1\":2829864.75,\"\
  q2\":3022580.5,\"q3\":3215296.25,\"max\":3408012,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\"},\"cpu\":{\"mean\":45.75,\"min\":11.5,\"q1\":28.63,\"q2\":45.75,\"\
  q3\":62.88,\"max\":80,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\"},\"time\":{\"mean\":500,\"min\":0,\"q1\":250,\"q2\":500,\"q3\"\
  :750,\"max\":1000,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\"},\"writes\":{\"mean\":4999,\"min\":4991,\"q1\":4995,\"q2\":4999,\"\
  q3\":5003,\"max\":5007,\"minLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"maxLabel\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE1_PE)\",\"q1Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q2Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\",\"q3Label\":\"NFCORE_VIRALRECON:ILLUMINA:ALIGN_BOWTIE2:BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX\
  \ (SAMPLE2_PE)\"}}]\nFeb-20 07:31:43.854 [Actor Thread 13] DEBUG nextflow.sort.BigSort\
  \ - Sort completed -- entries: 12; slices: 1; internal sort time: 0.028 s; external\
  \ sort time: 0.002 s; total time: 0.03 s\nFeb-20 07:31:43.873 [Actor Thread 13]\
  \ DEBUG nextflow.file.FileCollector - Saved collect-files list to: /home/azureuser/tmp/j95brtnwde/singularity-tmp/fa8e1cc217690ac4b9f07c64cc9a45a3.collect-file\n\
  Feb-20 07:31:43.879 [Actor Thread 13] DEBUG nextflow.file.FileCollector - Deleting\
  \ file collector temp dir: /home/azureuser/tmp/j95brtnwde/singularity-tmp/nxf-9732521837664505436\n\
  Feb-20 07:31:44.265 [main] DEBUG nextflow.trace.TimelineObserver - Workflow completed\
  \ -- rendering execution timeline\nFeb-20 07:31:45.530 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 07:31:45.579 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Flwan4l6d%2Flogs.tar.xz">0.081GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Flwan4l6d%2Foutput.tar.xz">0.011GiB</a>
						</td>
						<td>1.139GiB</td>
						<td>314.1 seconds</td>
						<td>10801.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742694000">
								<a href="https://github.com/solida-core/musta">solida-core/musta</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/musta/tree/v1.1">v1.1</a>
						</td>
						<td>snakemake</td>
						<td>363 days</td>
						<td>2023-02-19 04:36:48</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- call_variants.smk
kind: MissingInputException
rest: "Missing input files for rule get_sample_names:\n    output: results/tmp/sampleA_normal.samplename.txt,\
  \ results/tmp/sampleA_tumor.samplename.txt\n    wildcards: sample=sampleA\n    affected\
  \ files:\n        ../test-data-somatic/data/bam/A25.chr22.bam\n        ../test-data-somatic/data/bam/N1.chr22.bam"
rule: get_sample_names  in line 2

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2Fsd0ky8z6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2Fsd0ky8z6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.5 seconds</td>
						<td>1.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742611728">
								<a href="https://github.com/solida-core/musta">solida-core/musta</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/musta/tree/v1.1.1">v1.1.1</a>
						</td>
						<td>snakemake</td>
						<td>361 days</td>
						<td>2023-02-19 06:21:51</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- call_variants.smk
kind: MissingInputException
rest: "Missing input files for rule get_sample_names:\n    output: results/tmp/sampleA_normal.samplename.txt,\
  \ results/tmp/sampleA_tumor.samplename.txt\n    wildcards: sample=sampleA\n    affected\
  \ files:\n        ../test-data-somatic/data/bam/A25.chr22.bam\n        ../test-data-somatic/data/bam/N1.chr22.bam"
rule: get_sample_names  in line 2

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2Fjt9qslud%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2Fjt9qslud%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.6 seconds</td>
						<td>1.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738298608">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.7.1">v3.7.1</a>
						</td>
						<td>snakemake</td>
						<td>361 days</td>
						<td>2023-02-19 04:26:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/ifl2628bdf/singularity-tmp/tmpy7wqhw6l.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/185bb04c-81bf-4f96-b029-d8fbfe16f6b0\
  \ singularity  exec --home /home/azureuser/tmp/ifl2628bdf/code/.test  /home/azureuser/tmp/ifl2628bdf/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/ifl2628bdf/code/.test/.snakemake/conda/2441198f28f50ad3e83105e5bada1d19_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/ifl2628bdf/code/.test/.snakemake/conda/2441198f28f50ad3e83105e5bada1d19_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|\
  \          | 0/6 [00:00<?, ?it/s]\nExamining @/linux-64::__linux==5.15.0=0:  17%|\u2588\
  \u258B        | 1/6 [00:00<00:00, 7002.18it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 2/6 [00:00<00:00, 7869.24it/s]  \nExamining\
  \ bcftools=1.12:  50%|\u2588\u2588\u2588\u2588\u2588     | 3/6 [00:00<00:00, 8689.86it/s]\
  \              \nExamining ensembl-vep=105:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\
  \u258B   | 4/6 [00:00<00:00, 229.17it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  83%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00,\
  \ 123.04it/s]\n                                                                \
  \                          \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for __glibc bcftools:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining conflict for ensembl-vep bcftools:  17%|\u2588\u258B       \
  \ | 1/6 [00:00<00:00, 21.56it/s]\n                                             \
  \                                             \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/6 [00:00<?,\
  \ ?it/s]\nExamining @/linux-64::__unix==0=0:   0%|          | 0/6 [00:00<?, ?it/s]\n\
  Examining @/linux-64::__linux==5.15.0=0:  17%|\u2588\u258B        | 1/6 [00:00<00:00,\
  \ 8793.09it/s]\nExamining @/linux-64::__glibc==2.31=0:  33%|\u2588\u2588\u2588\u258E\
  \      | 2/6 [00:00<00:00, 9776.93it/s]  \nExamining bcftools=1.12:  50%|\u2588\u2588\
  \u2588\u2588\u2588     | 3/6 [00:00<00:00, 10708.86it/s]             \nExamining\
  \ ensembl-vep=105:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 4/6 [00:00<00:00,\
  \ 345.90it/s]\nExamining @/linux-64::__archspec==1=x86_64:  83%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u258E | 5/6 [00:00<00:00, 185.55it/s]\n         \
  \                                                                              \
  \   \n\nDetermining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ conflict for __glibc bcftools:   0%|          | 0/6 [00:00<?, ?it/s]\nExamining\
  \ conflict for ensembl-vep bcftools:  17%|\u2588\u258B        | 1/6 [00:00<00:00,\
  \ 26.76it/s]\n                                                                 \
  \                         \nfailed\n\nUnsatisfiableError: The following specifications\
  \ were found to be incompatible with each other:\n\nOutput in format: Requested\
  \ package -> Available versions\n\nPackage perl conflicts for:\nensembl-vep=105\
  \ -> perl[version='>=5.26.2,<5.26.3.0a0|>=5.32.1,<6.0a0',build=*_perl5]\nensembl-vep=105\
  \ -> perl-bio-db-hts[version='>=2.11'] -> perl[version='5.18.2.*|5.22.0.*|>=5.22.0,<5.23.0|>=5.26.2,<5.27.0a0|>=5.32.1,<5.33.0a0',build=*_perl5]\n\
  \nPackage htslib conflicts for:\nensembl-vep=105 -> perl-bio-db-hts[version='>=2.11']\
  \ -> htslib[version='>=1.10.2,<1.11.0a0|>=1.12,<1.13.0a0|>=1.14,<1.15.0a0|>=1.16,<1.17.0a0|>=1.9,<1.10.0a0']\n\
  ensembl-vep=105 -> htslib\n\nPackage libgcc-ng conflicts for:\nensembl-vep=105 ->\
  \ htslib -> libgcc-ng[version='>=10.3.0|>=12|>=9.4.0|>=9.3.0|>=7.5.0|>=7.3.0|>=4.9|>=11.2.0|>=7.2.0']\n\
  bcftools=1.12 -> libgcc-ng[version='>=7.5.0|>=9.3.0']\nbcftools=1.12 -> gsl[version='>=2.6,<2.7.0a0']\
  \ -> libgcc-ng[version='>=10.3.0|>=11.2.0|>=4.9|>=7.3.0|>=9.4.0|>=7.2.0|>=12']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - bcftools=1.12 -> libgcc-ng[version='>=9.3.0']\
  \ -> __glibc[version='>=2.17']\n\nYour installed version is: 2.31\n\nNote that strict\
  \ channel priority may have removed packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fnfjsnx8v%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fnfjsnx8v%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.251GiB</td>
						<td>973.4 seconds</td>
						<td>1245.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743194544">
								<a href="https://nf-co.re/viralrecon">viralrecon</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/viralrecon/tree/2.4.1">2.4.1</a>
						</td>
						<td>nextflow</td>
						<td>355 days</td>
						<td>2023-02-20 05:16:11</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 08:16:11.037
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Ftxhs9hjo%2Flogs.tar.xz">0.081GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2Ftxhs9hjo%2Foutput.tar.xz">0.011GiB</a>
						</td>
						<td>1.323GiB</td>
						<td>308.3 seconds</td>
						<td>10801.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738265360">
								<a href="https://github.com/epigen/atacseq_pipeline">epigen/atacseq_pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/epigen/atacseq_pipeline/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>354 days</td>
						<td>2023-02-19 06:12:36</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: FileNotFoundError
line_no: 50
rest: "[Errno 2] No such file or directory: '/research/home/sreichl/projects/atacseq_pipeline/test/BSA_0000_test_hg38_atac_atacseq_config.yaml'\n\
  \  File \"/home/azureuser/tmp/hatbnwnz1n/code/workflow/Snakefile\", line 171, in\
  \ <module>\n  File \"/home/azureuser/tmp/hatbnwnz1n/code/workflow/Snakefile\", line\
  \ 50, in configurator"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Fatacseq-pipeline%2Fa4nkr5et%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Fatacseq-pipeline%2Fa4nkr5et%2Foutput.tar.xz">0.015GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>11.5 seconds</td>
						<td>93.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738218608">
								<a href="https://github.com/sreichl/atacseq_pipeline">sreichl/atacseq_pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/sreichl/atacseq_pipeline/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>353 days</td>
						<td>2023-02-19 04:36:48</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: FileNotFoundError
line_no: 50
rest: "[Errno 2] No such file or directory: '/research/home/sreichl/projects/atacseq_pipeline/test/BSA_0000_test_hg38_atac_atacseq_config.yaml'\n\
  \  File \"/home/azureuser/tmp/6pxm6y0xvt/code/workflow/Snakefile\", line 171, in\
  \ <module>\n  File \"/home/azureuser/tmp/6pxm6y0xvt/code/workflow/Snakefile\", line\
  \ 50, in configurator"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sreichl%252Fatacseq-pipeline%2F89nm2iki%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sreichl%252Fatacseq-pipeline%2F89nm2iki%2Foutput.tar.xz">0.015GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>12.3 seconds</td>
						<td>105.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743152112">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.6">3.6</a>
						</td>
						<td>nextflow</td>
						<td>353 days</td>
						<td>2023-02-20 04:31:47</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867588298\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\t\
  at nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\
  \tat nextflow.processor.TaskRun$isContainerEnabled$6.call(Unknown Source)\n\tat\
  \ nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867588298\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 07:31:47.327 [main] DEBUG nextflow.Session\
  \ - Session await > all processes finished\nFeb-20 07:31:47.329 [main] DEBUG nextflow.Session\
  \ - Session await > all barriers passed\nFeb-20 07:31:47.333 [main] INFO  nextflow.Nextflow\
  \ - -\e[0;35m[nf-core/rnaseq]\e[0;31m Pipeline completed with errors\e[0m-\nFeb-20\
  \ 07:31:47.338 [main] DEBUG nextflow.trace.WorkflowStatsObserver - Workflow completed\
  \ > WorkflowStats[succeededCount=7; failedCount=0; ignoredCount=0; cachedCount=0;\
  \ pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0; abortedCount=0;\
  \ succeedDuration=4m 34s; failedDuration=0ms; cachedDuration=0ms;loadCpus=0; loadMemory=0;\
  \ peakRunning=2; peakCpus=4; peakMemory=12 GB; ]\nFeb-20 07:31:47.338 [main] DEBUG\
  \ nextflow.trace.TraceFileObserver - Workflow completed -- saving trace file\nFeb-20\
  \ 07:31:47.346 [main] DEBUG nextflow.trace.ReportObserver - Workflow completed --\
  \ rendering execution report\nFeb-20 07:31:47.462 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Execution report summary data:\n  [{\"cpuUsage\":{\"mean\":12.6,\"min\":12.6,\"\
  q1\":12.6,\"q2\":12.6,\"q3\":12.6,\"max\":12.6,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\"},\"process\":\"SAMPLESHEET_CHECK\",\"mem\":{\"mean\"\
  :13025280,\"min\":13025280,\"q1\":13025280,\"q2\":13025280,\"q3\":13025280,\"max\"\
  :13025280,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  },\"memUsage\":{\"mean\":0.2,\"min\":0.2,\"q1\":0.2,\"q2\":0.2,\"q3\":0.2,\"max\"\
  :0.2,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  },\"timeUsage\":{\"mean\":0.01,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"q3\":0.01,\"\
  max\":0.01,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  ,\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv)\"\
  },\"vmem\":{\"mean\":18321408,\"min\":18321408,\"q1\":18321408,\"q2\":18321408,\"\
  q3\":18321408,\"max\":18321408,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\"},\"reads\":{\"mean\":1515263,\"min\":1515263,\"q1\":1515263,\"\
  q2\":1515263,\"q3\":1515263,\"max\":1515263,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\"},\"cpu\":{\"mean\":25.2,\"min\":25.2,\"q1\":25.2,\"q2\"\
  :25.2,\"q3\":25.2,\"max\":25.2,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\"},\"time\":{\"mean\":1000,\"min\":1000,\"q1\":1000,\"\
  q2\":1000,\"q3\":1000,\"max\":1000,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\"},\"writes\":{\"mean\":1525,\"min\":1525,\"q1\":1525,\"\
  q2\":1525,\"q3\":1525,\"max\":1525,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (samplesheet_test.csv)\"}},{\"cpuUsage\":{\"mean\":35.37,\"min\":25.15,\"q1\"\
  :32.1,\"q2\":39.05,\"q3\":40.47,\"max\":41.9,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\"},\"process\":\"FASTQC\",\"mem\":{\"mean\":2897237.33,\"min\":2850816,\"\
  q1\":2877440,\"q2\":2904064,\"q3\":2920448,\"max\":2936832,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\"},\"memUsage\":{\"mean\":0.04,\"min\":0.04,\"q1\":0.04,\"q2\":0.05,\"\
  q3\":0.05,\"max\":0.05,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\"},\"timeUsage\":{\"mean\":0.05,\"min\":0.05,\"q1\":0.05,\"q2\":0.05,\"\
  q3\":0.05,\"max\":0.06,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\"},\"vmem\":{\"mean\":11579392,\"min\":11579392,\"q1\":11579392,\"\
  q2\":11579392,\"q3\":11579392,\"max\":11579392,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\"},\"reads\":{\"mean\":24833388,\"min\":22753628,\"q1\":24266127,\"\
  q2\":25778626,\"q3\":25873268,\"max\":25967910,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\"},\"cpu\":{\"mean\":70.73,\"min\":50.3,\"q1\":64.2,\"q2\":78.1,\"q3\"\
  :80.95,\"max\":83.8,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\"},\"time\":{\"mean\":11333.33,\"min\":11000,\"q1\":11000,\"q2\":11000,\"\
  q3\":11500,\"max\":12000,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\"},\"writes\":{\"mean\":3119194.33,\"min\":1868674,\"q1\"\
  :2804811.5,\"q2\":3740949,\"q3\":3744454.5,\"max\":3747960,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (WT_REP2)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_UNINDUCED_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:FASTQC\
  \ (RAP1_IAA_30M_REP1)\"}},{\"cpuUsage\":{\"mean\":24.95,\"min\":20.1,\"q1\":21.88,\"\
  q2\":23.65,\"q3\":27.38,\"max\":31.1,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (WT_REP2)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\"},\"process\":\"TRIMGALORE\",\"mem\":{\"mean\":185253888,\"\
  min\":145969152,\"q1\":169220096,\"q2\":192471040,\"q3\":204896256,\"max\":217321472,\"\
  minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (RAP1_IAA_30M_REP1)\"\
  ,\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (WT_REP2)\"\
  ,\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (RAP1_IAA_30M_REP1)\"\
  ,\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (RAP1_UNINDUCED_REP1)\"\
  ,\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (RAP1_UNINDUCED_REP1)\"\
  },\"memUsage\":{\"mean\":2.88,\"min\":2.27,\"q1\":2.63,\"q2\":2.99,\"q3\":3.18,\"\
  max\":3.37,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (WT_REP2)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\"},\"timeUsage\":{\"mean\":0.16,\"min\":0.11,\"q1\":0.13,\"\
  q2\":0.16,\"q3\":0.18,\"max\":0.2,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (WT_REP2)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (WT_REP2)\"},\"vmem\":{\"mean\":3228946432,\"min\":3138281472,\"q1\":3205722112,\"\
  q2\":3273162752,\"q3\":3274278912,\"max\":3275395072,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (WT_REP2)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\"},\"reads\":{\"mean\":145578676,\"min\":76555763,\"q1\"\
  :128107187.5,\"q2\":179658612,\"q3\":180090132.5,\"max\":180521653,\"minLabel\"\
  :\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (RAP1_UNINDUCED_REP1)\"\
  ,\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (RAP1_IAA_30M_REP1)\"\
  ,\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (RAP1_UNINDUCED_REP1)\"\
  ,\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (WT_REP2)\"\
  ,\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (WT_REP2)\"\
  },\"cpu\":{\"mean\":49.9,\"min\":40.2,\"q1\":43.75,\"q2\":47.3,\"q3\":54.75,\"max\"\
  :62.2,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (WT_REP2)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\"},\"time\":{\"mean\":34000,\"min\":24000,\"q1\":29000,\"\
  q2\":34000,\"q3\":39000,\"max\":44000,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (WT_REP2)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (WT_REP2)\"},\"writes\":{\"mean\":98080772.33,\"min\":42185228,\"q1\":83977088,\"\
  q2\":125768948,\"q3\":126028544.5,\"max\":126288141,\"minLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"maxLabel\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_IAA_30M_REP1)\",\"q1Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (RAP1_UNINDUCED_REP1)\",\"q2Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (WT_REP2)\",\"q3Label\":\"NFCORE_RNASEQ:RNASEQ:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE\
  \ (WT_REP2)\"}}]\nFeb-20 07:31:47.562 [Actor Thread 10] DEBUG nextflow.sort.BigSort\
  \ - Sort completed -- entries: 3; slices: 1; internal sort time: 0.008 s; external\
  \ sort time: 0.0 s; total time: 0.008 s\nFeb-20 07:31:47.580 [Actor Thread 10] DEBUG\
  \ nextflow.file.FileCollector - Saved collect-files list to: /home/azureuser/tmp/cs2hah6kly/singularity-tmp/b981b4bc78e5d49b4ae2d418d1546794.collect-file\n\
  Feb-20 07:31:47.592 [Actor Thread 10] DEBUG nextflow.file.FileCollector - Deleting\
  \ file collector temp dir: /home/azureuser/tmp/cs2hah6kly/singularity-tmp/nxf-12027394915773302859\n\
  Feb-20 07:31:48.204 [main] DEBUG nextflow.trace.TimelineObserver - Workflow completed\
  \ -- rendering execution timeline\nFeb-20 07:31:49.339 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 07:31:49.339 [main] INFO  org.pf4j.AbstractPluginManager\
  \ - Stop plugin 'nf-amazon@1.11.2'\nFeb-20 07:31:49.339 [main] DEBUG nextflow.plugin.BasePlugin\
  \ - Plugin stopped nf-amazon\nFeb-20 07:31:49.371 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fagrc2s33%2Flogs.tar.xz">0.065GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fagrc2s33%2Foutput.tar.xz">0.005GiB</a>
						</td>
						<td>1.079GiB</td>
						<td>167.4 seconds</td>
						<td>10802.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738265120">
								<a href="https://github.com/epigen/atacseq_pipeline">epigen/atacseq_pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/epigen/atacseq_pipeline/tree/v0.1.1">v0.1.1</a>
						</td>
						<td>snakemake</td>
						<td>345 days</td>
						<td>2023-02-19 06:32:06</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: FileNotFoundError
line_no: 50
rest: "[Errno 2] No such file or directory: '/research/home/sreichl/projects/atacseq_pipeline/test/BSA_0000_test_hg38_atac_atacseq_config.yaml'\n\
  \  File \"/home/azureuser/tmp/8xtu3ol9uj/code/workflow/Snakefile\", line 171, in\
  \ <module>\n  File \"/home/azureuser/tmp/8xtu3ol9uj/code/workflow/Snakefile\", line\
  \ 50, in configurator"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Fatacseq-pipeline%2Fr8oqxbt6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Fatacseq-pipeline%2Fr8oqxbt6%2Foutput.tar.xz">0.015GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>9.9 seconds</td>
						<td>90.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738218368">
								<a href="https://github.com/sreichl/atacseq_pipeline">sreichl/atacseq_pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/sreichl/atacseq_pipeline/tree/v0.1.1">v0.1.1</a>
						</td>
						<td>snakemake</td>
						<td>345 days</td>
						<td>2023-02-19 06:30:27</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: FileNotFoundError
line_no: 50
rest: "[Errno 2] No such file or directory: '/research/home/sreichl/projects/atacseq_pipeline/test/BSA_0000_test_hg38_atac_atacseq_config.yaml'\n\
  \  File \"/home/azureuser/tmp/2dtarkajij/code/workflow/Snakefile\", line 171, in\
  \ <module>\n  File \"/home/azureuser/tmp/2dtarkajij/code/workflow/Snakefile\", line\
  \ 50, in configurator"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sreichl%252Fatacseq-pipeline%2F9uvc3nxc%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sreichl%252Fatacseq-pipeline%2F9uvc3nxc%2Foutput.tar.xz">0.015GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>10.2 seconds</td>
						<td>155.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738210448">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.2.0">v1.2.0</a>
						</td>
						<td>snakemake</td>
						<td>342 days</td>
						<td>2023-02-19 23:32:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F4fuxgsih%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F4fuxgsih%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.028GiB</td>
						<td>319.8 seconds</td>
						<td>10800.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738298368">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.8.0">v3.8.0</a>
						</td>
						<td>snakemake</td>
						<td>342 days</td>
						<td>2023-02-19 04:40:22</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 8rrpukrvzc
- code
- workflow
- rules
- common.smk
kind: FileNotFoundError
line_no: 13
rest: "[Errno 2] No such file or directory: 'config/samples.tsv'\n  File \"/home/azureuser/tmp/8rrpukrvzc/code/workflow/Snakefile\"\
  , line 20, in <module>\n  File \"/home/azureuser/tmp/8rrpukrvzc/code/workflow/rules/common.smk\"\
  , line 13, in <module>\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 211, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 331, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 950, in read_csv\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 605, in _read\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1442, in __init__\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1735, in _make_engine\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/common.py\"\
  , line 856, in get_handle"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fzcta992k%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fzcta992k%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>2.1 seconds</td>
						<td>3.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738215488">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.2.0">v1.2.0</a>
						</td>
						<td>snakemake</td>
						<td>341 days</td>
						<td>2023-02-19 03:34:34</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2F9ih3njv2%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2F9ih3njv2%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.821GiB</td>
						<td>641.0 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738257200">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.14.0">v0.14.0</a>
						</td>
						<td>snakemake</td>
						<td>335 days</td>
						<td>2023-02-19 06:34:55</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- read_trimming.smk
kind: MissingInputException
rest: "Missing input files for rule fastp_pe:\n    output: results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.1.fastq.gz,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.2.fastq.gz, results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.fastp.json\n    wildcards:\
  \ date=SEQUENCING_DATE, sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq2\n\
  \        PATH/TO/fq1"
rule: fastp_pe  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fme0l3glz%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fme0l3glz%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.109GiB</td>
						<td>2.4 seconds</td>
						<td>4.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738298128">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.8.1">v3.8.1</a>
						</td>
						<td>snakemake</td>
						<td>332 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- xm6v4r5jok
- code
- workflow
- rules
- common.smk
kind: FileNotFoundError
line_no: 13
rest: "[Errno 2] No such file or directory: 'config/samples.tsv'\n  File \"/home/azureuser/tmp/xm6v4r5jok/code/workflow/Snakefile\"\
  , line 20, in <module>\n  File \"/home/azureuser/tmp/xm6v4r5jok/code/workflow/rules/common.smk\"\
  , line 13, in <module>\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 211, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 331, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 950, in read_csv\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 605, in _read\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1442, in __init__\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1735, in _make_engine\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/common.py\"\
  , line 856, in get_handle"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fiaqloy4i%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fiaqloy4i%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>2.0 seconds</td>
						<td>2.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743103984">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.4.3">2.4.3</a>
						</td>
						<td>nextflow</td>
						<td>332 days</td>
						<td>2023-02-19 03:31:06</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fnddj1cbo%2Flogs.tar.xz">0.014GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fnddj1cbo%2Foutput.tar.xz">0.011GiB</a>
						</td>
						<td>1.530GiB</td>
						<td>714.2 seconds</td>
						<td>759.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738256960">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.15.0">v0.15.0</a>
						</td>
						<td>snakemake</td>
						<td>331 days</td>
						<td>2023-02-19 03:31:58</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- read_trimming.smk
kind: MissingInputException
rest: "Missing input files for rule fastp_pe:\n    output: results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.1.fastq.gz,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.2.fastq.gz, results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.fastp.json\n    wildcards:\
  \ date=SEQUENCING_DATE, sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq2\n\
  \        PATH/TO/fq1"
rule: fastp_pe  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Feq43co9e%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Feq43co9e%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.110GiB</td>
						<td>2.5 seconds</td>
						<td>7.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738287088">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.4.0">v2.4.0</a>
						</td>
						<td>snakemake</td>
						<td>327 days</td>
						<td>2023-02-19 06:02:03</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: "Could not create conda environment from /home/azureuser/tmp/r95oy13ldk/code/workflow/rules/../envs/goatools.yaml:\n\
  Command:\nSINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/d668ddce-e39f-48d3-8bee-0463fb1b029f\
  \ singularity  exec --home /home/azureuser/tmp/r95oy13ldk/code/.test  /home/azureuser/tmp/r95oy13ldk/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ sh -c 'conda config --set channel_priority strict && conda env create --quiet\
  \ --file \"/home/azureuser/tmp/r95oy13ldk/code/.test/.snakemake/conda/11be4d40ce7bf24d582661379f585ec3_.yaml\"\
  \ --prefix \"/home/azureuser/tmp/r95oy13ldk/code/.test/.snakemake/conda/11be4d40ce7bf24d582661379f585ec3_\"\
  '\nOutput:\nCollecting package metadata (repodata.json): ...working... done\nSolving\
  \ environment: ...working... \nFound conflicts! Looking for incompatible packages.\n\
  This can take several minutes.  Press CTRL-C to abort.\n\nBuilding graph of deps:\
  \   0%|          | 0/9 [00:00<?, ?it/s]\nExamining goatools=0.9.7:   0%|       \
  \   | 0/9 [00:00<?, ?it/s]\nExamining @/linux-64::__archspec==1=x86_64:  11%|\u2588\
  \         | 1/9 [00:00<00:02,  3.23it/s]\nExamining @/linux-64::__archspec==1=x86_64:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:01,  6.46it/s]\nExamining @/linux-64::__glibc==2.31=0:\
  \  22%|\u2588\u2588\u258F       | 2/9 [00:00<00:01,  6.46it/s]     \nExamining @/linux-64::__unix==0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:00,  6.46it/s]    \nExamining\
  \ python=3.7:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,  6.46it/s]\
  \             \nExamining @/linux-64::__linux==5.15.0=0:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:01<00:00,  6.46it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:01<00:00,  3.41it/s]\n\
  Examining pandas=0.25:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:01<00:00,\
  \  3.41it/s]                  \nExamining matplotlib=3.1:  78%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258A  | 7/9 [00:03<00:00,  3.41it/s]\nExamining matplotlib=3.1:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:03<00:00,\
  \  2.27it/s]\nExamining pillow=6.2:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2589 | 8/9 [00:04<00:00,  2.27it/s]    \nExamining pillow=6.2: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:04<00:00,  1.44it/s]\n\
  \                                                                   \n\nDetermining\
  \ conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for matplotlib\
  \ goatools pillow pandas:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for matplotlib goatools pandas:  11%|\u2588         | 1/9 [00:07<01:02,  7.77s/it]\n\
  Examining conflict for matplotlib goatools pandas:  22%|\u2588\u2588\u258F     \
  \  | 2/9 [00:07<00:27,  3.89s/it]\nExamining conflict for goatools python pandas\
  \ matplotlib pillow:  22%|\u2588\u2588\u258F       | 2/9 [00:48<00:27,  3.89s/it]\n\
  Examining conflict for goatools python pandas matplotlib pillow:  33%|\u2588\u2588\
  \u2588\u258E      | 3/9 [00:48<01:54, 19.14s/it]\nExamining conflict for goatools\
  \ pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:50<01:54, 19.14s/it]    \
  \                     \nExamining conflict for goatools pandas:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [00:50<01:03, 12.65s/it]\nExamining conflict for goatools\
  \ matplotlib:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:50<01:03, 12.65s/it]\n\
  Examining conflict for goatools matplotlib:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [00:50<00:34,  8.57s/it]\nExamining conflict for goatools pillow python:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [00:53<00:34,  8.57s/it]\n\
  Examining conflict for goatools pillow python:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [00:53<00:19,  6.53s/it]\nExamining conflict for __glibc python\
  \ pandas matplotlib pillow:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   |\
  \ 6/9 [00:55<00:19,  6.53s/it]\nExamining conflict for __glibc python pandas matplotlib\
  \ pillow:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:55<00:10,\
  \  5.11s/it]\nExamining conflict for matplotlib pandas pillow python:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [00:56<00:10,  5.11s/it]     \
  \   \nExamining conflict for matplotlib pandas pillow python:  89%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:56<00:03,  3.96s/it]\nExamining conflict\
  \ for pillow python:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589\
  \ | 8/9 [01:01<00:03,  3.96s/it]                  \nExamining conflict for pillow\
  \ python: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9\
  \ [01:01<00:00,  4.02s/it]\nExamining conflict for matplotlib pandas python: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [01:01<00:00,  4.02s/it]\n\
  Examining conflict for matplotlib pandas python: : 10it [01:01,  2.93s/it]     \
  \               \nExamining conflict for pandas matplotlib: : 10it [01:01,  2.93s/it]\
  \       \nExamining conflict for pandas matplotlib: : 11it [01:01,  2.08s/it]\n\
  Examining conflict for pillow matplotlib: : 11it [01:02,  2.08s/it]\nExamining conflict\
  \ for pillow matplotlib: : 12it [01:02,  1.67s/it]\n                           \
  \                                        \nfailed\nSolving environment: ...working...\
  \ \nFound conflicts! Looking for incompatible packages.\nThis can take several minutes.\
  \  Press CTRL-C to abort.\n\nBuilding graph of deps:   0%|          | 0/9 [00:00<?,\
  \ ?it/s]\nExamining goatools=0.9.7:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining\
  \ @/linux-64::__archspec==1=x86_64:  11%|\u2588         | 1/9 [00:00<00:02,  2.92it/s]\n\
  Examining @/linux-64::__archspec==1=x86_64:  22%|\u2588\u2588\u258F       | 2/9\
  \ [00:00<00:01,  5.84it/s]\nExamining @/linux-64::__glibc==2.31=0:  22%|\u2588\u2588\
  \u258F       | 2/9 [00:00<00:01,  5.84it/s]     \nExamining @/linux-64::__unix==0=0:\
  \  33%|\u2588\u2588\u2588\u258E      | 3/9 [00:00<00:01,  5.84it/s]    \nExamining\
  \ python=3.7:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [00:00<00:00,  5.84it/s]\
  \             \nExamining @/linux-64::__linux==5.15.0=0:  56%|\u2588\u2588\u2588\
  \u2588\u2588\u258C    | 5/9 [00:02<00:00,  5.84it/s]\nExamining @/linux-64::__linux==5.15.0=0:\
  \  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:02<00:01,  2.27it/s]\n\
  Examining pandas=0.25:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   | 6/9 [00:02<00:01,\
  \  2.27it/s]                  \nExamining matplotlib=3.1:  78%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u258A  | 7/9 [00:05<00:00,  2.27it/s]\nExamining matplotlib=3.1:\
  \  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [00:05<00:00,\
  \  1.32it/s]\nExamining pillow=6.2:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\
  \u2588\u2589 | 8/9 [00:08<00:00,  1.32it/s]    \nExamining pillow=6.2: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [00:08<00:00,  1.22s/it]\n\
  \                                                                   \n\nDetermining\
  \ conflicts:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict for matplotlib\
  \ goatools pillow pandas:   0%|          | 0/9 [00:00<?, ?it/s]\nExamining conflict\
  \ for matplotlib goatools pandas:  11%|\u2588         | 1/9 [00:12<01:43, 12.98s/it]\n\
  Examining conflict for matplotlib goatools pandas:  22%|\u2588\u2588\u258F     \
  \  | 2/9 [00:12<00:45,  6.49s/it]\nExamining conflict for goatools python pandas\
  \ matplotlib pillow:  22%|\u2588\u2588\u258F       | 2/9 [01:01<00:45,  6.49s/it]\n\
  Examining conflict for goatools python pandas matplotlib pillow:  33%|\u2588\u2588\
  \u2588\u258E      | 3/9 [01:01<02:23, 23.99s/it]\nExamining conflict for goatools\
  \ pandas:  33%|\u2588\u2588\u2588\u258E      | 3/9 [01:04<02:23, 23.99s/it]    \
  \                     \nExamining conflict for goatools pandas:  44%|\u2588\u2588\
  \u2588\u2588\u258D     | 4/9 [01:04<01:20, 16.18s/it]\nExamining conflict for goatools\
  \ matplotlib:  44%|\u2588\u2588\u2588\u2588\u258D     | 4/9 [01:06<01:20, 16.18s/it]\n\
  Examining conflict for goatools matplotlib:  56%|\u2588\u2588\u2588\u2588\u2588\u258C\
  \    | 5/9 [01:06<00:44, 11.08s/it]\nExamining conflict for goatools pillow python:\
  \  56%|\u2588\u2588\u2588\u2588\u2588\u258C    | 5/9 [01:09<00:44, 11.08s/it]\n\
  Examining conflict for goatools pillow python:  67%|\u2588\u2588\u2588\u2588\u2588\
  \u2588\u258B   | 6/9 [01:09<00:25,  8.54s/it]\nExamining conflict for __glibc python\
  \ pandas matplotlib pillow:  67%|\u2588\u2588\u2588\u2588\u2588\u2588\u258B   |\
  \ 6/9 [01:11<00:25,  8.54s/it]\nExamining conflict for __glibc python pandas matplotlib\
  \ pillow:  78%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [01:11<00:13,\
  \  6.51s/it]\nExamining conflict for matplotlib pandas pillow python:  78%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u258A  | 7/9 [01:13<00:13,  6.51s/it]     \
  \   \nExamining conflict for matplotlib pandas pillow python:  89%|\u2588\u2588\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2589 | 8/9 [01:13<00:05,  5.13s/it]\nExamining conflict\
  \ for pillow python:  89%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2589\
  \ | 8/9 [01:21<00:05,  5.13s/it]                  \nExamining conflict for pillow\
  \ python: 100%|\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9\
  \ [01:21<00:00,  5.88s/it]\nExamining conflict for matplotlib pandas python: 100%|\u2588\
  \u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 9/9 [01:22<00:00,  5.88s/it]\n\
  Examining conflict for matplotlib pandas python: : 10it [01:22,  4.36s/it]     \
  \               \nExamining conflict for pandas matplotlib: : 10it [01:22,  4.36s/it]\
  \       \nExamining conflict for pandas matplotlib: : 11it [01:22,  3.14s/it]\n\
  Examining conflict for pillow matplotlib: : 11it [01:23,  3.14s/it]\nExamining conflict\
  \ for pillow matplotlib: : 12it [01:23,  2.56s/it]\n                           \
  \                                        \nfailed\n\nUnsatisfiableError: The following\
  \ specifications were found to be incompatible with each other:\n\nOutput in format:\
  \ Requested package -> Available versions\n\nPackage _libgcc_mutex conflicts for:\n\
  pandas=0.25 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  matplotlib=3.1 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main|main']\n\
  python=3.7 -> libgcc-ng[version='>=9.4.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main|main']\n\
  \nPackage ca-certificates conflicts for:\npillow=6.2 -> python[version='>=2.7,<2.8.0a0']\
  \ -> ca-certificates\npython=3.7 -> openssl[version='>=1.1.1l,<1.1.2a'] -> ca-certificates\n\
  goatools=0.9.7 -> python -> ca-certificates\n\nPackage python conflicts for:\npillow=6.2\
  \ -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n\
  goatools=0.9.7 -> numpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|3.4.*|>=3.7,<4.0|>=3.6,<4.0|>=3.6|>=3.4|>=2.7']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|>=3.6|>=3|3.4.*']\n\
  pillow=6.2 -> olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|3.7.*|3.8.*']\n\
  goatools=0.9.7 -> python\npython=3.7\npandas=0.25 -> python[version='>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']\n\
  \nPackage pypy3.6 conflicts for:\npillow=6.2 -> python[version='>=3.6,<3.7.0a0']\
  \ -> pypy3.6[version='7.3.*|7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*']\nmatplotlib=3.1 ->\
  \ python[version='>=3.6,<3.7.0a0'] -> pypy3.6[version='7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*|>=7.3.3|>=7.3.2|>=7.3.1']\n\
  goatools=0.9.7 -> numpy -> pypy3.6[version='7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*|>=7.3.1|>=7.3.2|>=7.3.3']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> pypy3.6[version='7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*|>=7.3.1|>=7.3.2|>=7.3.3']\n\
  pillow=6.2 -> pypy3.6[version='>=7.3.1']\n\nPackage expat conflicts for:\npython=3.7\
  \ -> pypy3.7=7.3.7 -> expat[version='>=2.2.9,<3.0.0a0|>=2.3.0,<3.0a0|>=2.4.1,<3.0a0']\n\
  pillow=6.2 -> pypy3.6[version='>=7.3.1'] -> expat[version='>=2.2.9,<3.0.0a0']\n\n\
  Package libpng conflicts for:\nmatplotlib=3.1 -> libpng[version='>=1.6.37,<1.7.0a0']\n\
  matplotlib=3.1 -> matplotlib-base==3.1.1=py36he7580a8_1 -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  pillow=6.2 -> freetype[version='>=2.9.1,<3.0a0'] -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.37,<1.7.0a0|>=1.6.39,<1.7.0a0']\n\
  \nPackage pypy3.9 conflicts for:\ngoatools=0.9.7 -> numpy -> pypy3.9[version='7.3.11.*|>=7.3.11|>=7.3.9|>=7.3.8|7.3.9.*|7.3.8.*']\n\
  pandas=0.25 -> numpy[version='>=1.14.6,<2.0a0'] -> pypy3.9[version='>=7.3.11|>=7.3.9|>=7.3.8']\n\
  matplotlib=3.1 -> kiwisolver -> pypy3.9[version='>=7.3.11|>=7.3.8|>=7.3.9']\n\n\
  Package pyparsing conflicts for:\nmatplotlib=3.1 -> pyparsing\ngoatools=0.9.7 ->\
  \ pydot -> pyparsing[version='>=2.1.4|>=2.1.4,<3']\n\nPackage pandas conflicts for:\n\
  pandas=0.25\ngoatools=0.9.7 -> statsmodels -> pandas[version='>=0.14|>=0.21|>=0.23|>=1.0|>=0.25']\n\
  \nPackage python-dateutil conflicts for:\nmatplotlib=3.1 -> python-dateutil\npandas=0.25\
  \ -> python-dateutil[version='>=2.6.1']\n\nPackage setuptools conflicts for:\nmatplotlib=3.1\
  \ -> setuptools\npython=3.7 -> pip -> setuptools\npandas=0.25 -> setuptools[version='<60.0.0']\n\
  \nPackage numpy conflicts for:\nmatplotlib=3.1 -> numpy[version='>=1.14.6,<2.0a0']\n\
  matplotlib=3.1 -> matplotlib-base==3.1.0=py37h5f35d83_0 -> numpy[version='>=1.11.3,<2.0a0']The\
  \ following specifications were found to be incompatible with your system:\n\n \
  \ - feature:/linux-64::__glibc==2.31=0\n  - feature:|@/linux-64::__glibc==2.31=0\n\
  \  - pandas=0.25 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n\
  \  - pillow=6.2 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n \
  \ - python=3.7 -> libgcc-ng[version='>=9.4.0'] -> __glibc[version='>=2.17']\n\n\
  Your installed version is: 2.31\n\nNote that strict channel priority may have removed\
  \ packages required for satisfiability."

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Ffbi07ac2%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Ffbi07ac2%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.291GiB</td>
						<td>1393.6 seconds</td>
						<td>1845.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740503680">
								<a href="https://github.com/TLenfers/multispecies_mitochondrial_variant_analysis">TLenfers/multispecies_mitochondrial_variant_analysis</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/TLenfers/multispecies_mitochondrial_variant_analysis/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>327 days</td>
						<td>2023-02-19 06:00:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: "\nMissingInputException: Missing input files for rule bwa_mem:\n    output:\
  \ results/mapped/human/A.bam\n    wildcards: reference=human, sample=A\n    affected\
  \ files:\n        data/A_R1.fastq.gz\n        data/A_R2.fastq.gz\nMissingInputException:\
  \ Missing input files for rule bwa_mem:\n    output: results/mapped/human/norm_A.bam\n\
  \    wildcards: reference=human, sample=norm_A\n    affected files:\n        data/norm_A_R2.fastq.gz\n\
  \        data/norm_A_R1.fastq.gz\n"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/TLenfers%252Fmultispecies-mitochondrial-variant-analysis%2Flcmoxs43%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/TLenfers%252Fmultispecies-mitochondrial-variant-analysis%2Flcmoxs43%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.099GiB</td>
						<td>2.2 seconds</td>
						<td>5.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740419072">
								<a href="https://github.com/TLenfers/multispecies_mitochondrial_variant_analysis">TLenfers/multispecies_mitochondrial_variant_analysis</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/TLenfers/multispecies_mitochondrial_variant_analysis/tree/v1.0.1">v1.0.1</a>
						</td>
						<td>snakemake</td>
						<td>326 days</td>
						<td>2023-02-19 06:55:28</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: "\nMissingInputException: Missing input files for rule bwa_mem:\n    output:\
  \ results/mapped/human/norm_A.bam\n    wildcards: reference=human, sample=norm_A\n\
  \    affected files:\n        data/norm_A_R1.fastq.gz\n        data/norm_A_R2.fastq.gz\n\
  MissingInputException: Missing input files for rule bwa_mem:\n    output: results/mapped/human/A.bam\n\
  \    wildcards: reference=human, sample=A\n    affected files:\n        data/A_R1.fastq.gz\n\
  \        data/A_R2.fastq.gz\n"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/TLenfers%252Fmultispecies-mitochondrial-variant-analysis%2Fy54f4ry1%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/TLenfers%252Fmultispecies-mitochondrial-variant-analysis%2Fy54f4ry1%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.099GiB</td>
						<td>2.0 seconds</td>
						<td>2.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746420512">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/2.3.0">2.3.0</a>
						</td>
						<td>nextflow</td>
						<td>322 days</td>
						<td>2023-02-20 05:01:30</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.NullPointerException
msg: Cannot invoke method getAt() on null object
rest: "at org.codehaus.groovy.runtime.NullObject.invokeMethod(NullObject.java:91)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:44)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.NullCallSite.call(NullCallSite.java:34)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_0e3dd523$_runScript_closure3$_closure5$_closure19.doCall(Script_0e3dd523:232)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:38)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:53)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat nextflow.extension.DataflowHelper$_subscribeImpl_closure2.doCall(DataflowHelper.groovy:285)\n\
  \tat jdk.internal.reflect.GeneratedMethodAccessor184.invoke(Unknown Source)\n\t\
  at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.onMessage(DataflowOperatorActor.java:108)\n\
  \tat groovyx.gpars.actor.impl.SDAClosure$1.call(SDAClosure.java:43)\n\tat groovyx.gpars.actor.AbstractLoopingActor.runEnhancedWithoutRepliesOnMessages(AbstractLoopingActor.java:293)\n\
  \tat groovyx.gpars.actor.AbstractLoopingActor.access$400(AbstractLoopingActor.java:30)\n\
  \tat groovyx.gpars.actor.AbstractLoopingActor$1.handleMessage(AbstractLoopingActor.java:93)\n\
  \tat groovyx.gpars.util.AsyncMessagingCore.run(AsyncMessagingCore.java:132)\n\t\
  at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nFeb-20 08:01:30.171 [main]\
  \ DEBUG nextflow.trace.ReportObserver - Workflow completed -- rendering execution\
  \ report\nFeb-20 08:01:30.194 [Actor Thread 59] DEBUG nextflow.file.SortFileCollector\
  \ - FileCollector temp dir not removed: null\nFeb-20 08:01:30.232 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Execution report summary data:\n  [{\"cpuUsage\":{\"mean\":60.55,\"min\":60.55,\"\
  q1\":60.55,\"q2\":60.55,\"q3\":60.55,\"max\":60.55,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"process\":\"METADATA_PAIRWISE\",\"mem\":{\"mean\":4468736,\"\
  min\":4468736,\"q1\":4468736,\"q2\":4468736,\"q3\":4468736,\"max\":4468736,\"minLabel\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"},\"memUsage\":{\"\
  mean\":0.07,\"min\":0.07,\"q1\":0.07,\"q2\":0.07,\"q3\":0.07,\"max\":0.07,\"minLabel\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"},\"timeUsage\":null,\"\
  vmem\":{\"mean\":54247424,\"min\":54247424,\"q1\":54247424,\"q2\":54247424,\"q3\"\
  :54247424,\"max\":54247424,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"reads\":{\"mean\":6085638,\"min\":6085638,\"q1\":6085638,\"\
  q2\":6085638,\"q3\":6085638,\"max\":6085638,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"cpu\":{\"mean\":121.1,\"min\":121.1,\"q1\":121.1,\"q2\":121.1,\"\
  q3\":121.1,\"max\":121.1,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"time\":null,\"writes\":{\"mean\":1120,\"min\":1120,\"q1\"\
  :1120,\"q2\":1120,\"q3\":1120,\"max\":1120,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"}},{\"cpuUsage\":{\"mean\":54.15,\"min\":54.15,\"q1\":54.15,\"\
  q2\":54.15,\"q3\":54.15,\"max\":54.15,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\"},\"process\":\"METADATA_ALL\",\"mem\":{\"mean\":4354048,\"min\"\
  :4354048,\"q1\":4354048,\"q2\":4354048,\"q3\":4354048,\"max\":4354048,\"minLabel\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"},\"memUsage\":{\"mean\"\
  :0.07,\"min\":0.07,\"q1\":0.07,\"q2\":0.07,\"q3\":0.07,\"max\":0.07,\"minLabel\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"},\"timeUsage\":null,\"\
  vmem\":{\"mean\":54247424,\"min\":54247424,\"q1\":54247424,\"q2\":54247424,\"q3\"\
  :54247424,\"max\":54247424,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\"},\"reads\":{\"mean\":6085540,\"min\":6085540,\"q1\":6085540,\"\
  q2\":6085540,\"q3\":6085540,\"max\":6085540,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\"},\"cpu\":{\"mean\":108.3,\"min\":108.3,\"q1\":108.3,\"q2\":108.3,\"\
  q3\":108.3,\"max\":108.3,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q1Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  },\"time\":null,\"writes\":{\"mean\":1109,\"min\":1109,\"q1\":1109,\"q2\":1109,\"\
  q3\":1109,\"max\":1109,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q1Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  }}]\nFeb-20 08:01:30.588 [main] DEBUG nextflow.trace.TimelineObserver - Workflow\
  \ completed -- rendering execution timeline\nFeb-20 08:01:30.797 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 08:01:30.816 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Frm36g1h5%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Frm36g1h5%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.027GiB</td>
						<td>70.0 seconds</td>
						<td>10800.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746369488">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/2.3.1">2.3.1</a>
						</td>
						<td>nextflow</td>
						<td>320 days</td>
						<td>2023-02-19 06:00:26</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-19 07:00:26.564
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fnvj3d8oo%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fnvj3d8oo%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.894GiB</td>
						<td>61.6 seconds</td>
						<td>3602.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743141072">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/2.3.0">2.3.0</a>
						</td>
						<td>nextflow</td>
						<td>319 days</td>
						<td>2023-02-19 06:02:03</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "DecoyDatabase -in test.fasta \\\n    -out test_decoy.fasta \\\n\
  \    -decoy_string DECOY_ \\\n    -decoy_string_position prefix\n\ncat <<-END_VERSIONS\
  \ > versions.yml\n\"NFCORE_MHCQUANT:MHCQUANT:OPENMS_DECOYDATABASE\":\n    openms:\
  \ $(echo $(FileInfo --help 2>&1) | sed 's/^.*Version: //; s/-.*$//' | sed 's/ -*//;\
  \ s/ .*$//')\nEND_VERSIONS"
error: 'WARNING: Skipping mount /home/azureuser/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/singularityce-3.10.3-z5qu45ho72nvsy4rpcyd4eqrop35tnbe/var/singularity/mnt/session/etc/resolv.conf
  [files]: /etc/resolv.conf doesn''t exist in container

  Error: File not found (the file ''test.fasta'' could not be found)

  Cannot read input file given from parameter ''-in''!'
exit_status: 1
kind: CalledProcessError
output: 'Error: File not found (the file ''test.fasta'' could not be found)'
process: NFCORE_MHCQUANT:MHCQUANT:OPENMS_DECOYDATABASE (4)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- db
- 13b6d9031e5905abeeda8b33546e25

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fmsu0jz30%2Flogs.tar.xz">0.027GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fmsu0jz30%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.709GiB</td>
						<td>54.7 seconds</td>
						<td>86.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738297888">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.9.0">v3.9.0</a>
						</td>
						<td>snakemake</td>
						<td>317 days</td>
						<td>2023-02-19 03:31:24</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 7df3scu1xx
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/7df3scu1xx/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F6a6lkfxq%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F6a6lkfxq%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.7 seconds</td>
						<td>3.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738204928">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.4">1.4.4</a>
						</td>
						<td>snakemake</td>
						<td>317 days</td>
						<td>2023-02-19 06:21:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- qrvm0tk9ay
- code
- workflow
- rules
- reports.smk
kind: CalledProcessError
line_no: 21
rest: "Command 'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate\
  \ '/home/azureuser/tmp/qrvm0tk9ay/code/.tests/.snakemake/conda/60836a4e660ddf763a775a295dde8a21_';\
  \ set -euo pipefail;  python /home/azureuser/tmp/qrvm0tk9ay/code/.tests/.snakemake/scripts/tmpf6rde94y.krona_table.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/qrvm0tk9ay/code/workflow/rules/reports.smk\"\
  , line 21, in __rule_krona_table\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Sun Feb 19 06:26:51 2023]\nFinished job 22.\n36 of 61 steps\
  \ (59%) done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-19T062110.168170.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fsl3nigs9%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fsl3nigs9%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.031GiB</td>
						<td>369.2 seconds</td>
						<td>342.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743103744">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.4.4">2.4.4</a>
						</td>
						<td>nextflow</td>
						<td>316 days</td>
						<td>2023-02-19 06:00:20</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fo5fboqtt%2Flogs.tar.xz">0.015GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fo5fboqtt%2Foutput.tar.xz">0.012GiB</a>
						</td>
						<td>1.913GiB</td>
						<td>760.2 seconds</td>
						<td>515.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738171856">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.5">1.4.5</a>
						</td>
						<td>snakemake</td>
						<td>314 days</td>
						<td>2023-02-19 03:36:06</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 3lm11z5wl1
- code
- workflow
- rules
- reports.smk
kind: CalledProcessError
line_no: 21
rest: "Command 'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate\
  \ '/home/azureuser/tmp/3lm11z5wl1/code/.tests/.snakemake/conda/088e7876344b50ce0ddee547533112d2_';\
  \ set -euo pipefail;  python /home/azureuser/tmp/3lm11z5wl1/code/.tests/.snakemake/scripts/tmp7gs2fmv_.krona_table.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/3lm11z5wl1/code/workflow/rules/reports.smk\"\
  , line 21, in __rule_krona_table\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Sun Feb 19 03:56:07 2023]\nFinished job 22.\n36 of 61 steps\
  \ (59%) done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-19T033608.908945.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fge84oobp%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fge84oobp%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.032GiB</td>
						<td>613.8 seconds</td>
						<td>1204.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742578048">
								<a href="https://github.com/solida-core/musta">solida-core/musta</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/musta/tree/1.0">1.0</a>
						</td>
						<td>snakemake</td>
						<td>303 days</td>
						<td>2023-02-19 04:10:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- call_variants.smk
kind: MissingInputException
rest: "Missing input files for rule get_sample_names:\n    output: /path/to/workdir/results/tmp/sampleA_normal.samplename.txt,\
  \ /path/to/workdir/results/tmp/sampleA_tumor.samplename.txt\n    wildcards: sample=sampleA\n\
  \    affected files:\n        ../test-data-somatic/data/bam/A25.chr22.bam\n    \
  \    ../test-data-somatic/data/bam/N1.chr22.bam"
rule: get_sample_names  in line 2

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2Fcdd76qgc%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2Fcdd76qgc%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.063GiB</td>
						<td>1.6 seconds</td>
						<td>1.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738297648">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.10.0">v3.10.0</a>
						</td>
						<td>snakemake</td>
						<td>299 days</td>
						<td>2023-02-19 04:29:33</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- gt2knzdhop
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/gt2knzdhop/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Faikwx1vy%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Faikwx1vy%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>0.9 seconds</td>
						<td>0.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742494640">
								<a href="https://github.com/solida-core/musta">solida-core/musta</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/musta/tree/v1.2">v1.2</a>
						</td>
						<td>snakemake</td>
						<td>297 days</td>
						<td>2023-02-19 04:19:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- call_variants.smk
kind: MissingInputException
rest: "Missing input files for rule get_sample_names:\n    output: /path/to/workdir/results/tmp/patientA_normal.samplename.txt,\
  \ /path/to/workdir/results/tmp/patientA_tumor.samplename.txt\n    wildcards: sample=patientA\n\
  \    affected files:\n        ../test-data-somatic/data/bam/N1.chr22.bam\n     \
  \   ../test-data-somatic/data/bam/A25.chr22.bam"
rule: get_sample_names  in line 2

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2F0sby6ioa%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2F0sby6ioa%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.063GiB</td>
						<td>1.5 seconds</td>
						<td>1.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742362176">
								<a href="https://github.com/solida-core/musta">solida-core/musta</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/musta/tree/v1.2.0.1">v1.2.0.1</a>
						</td>
						<td>snakemake</td>
						<td>296 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- call_variants.smk
kind: MissingInputException
rest: "Missing input files for rule get_sample_names:\n    output: /path/to/workdir/results/tmp/patientA_normal.samplename.txt,\
  \ /path/to/workdir/results/tmp/patientA_tumor.samplename.txt\n    wildcards: sample=patientA\n\
  \    affected files:\n        ../test-data-somatic/data/bam/N1.chr22.bam\n     \
  \   ../test-data-somatic/data/bam/A25.chr22.bam"
rule: get_sample_names  in line 2

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2F4knn5t0y%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2F4knn5t0y%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.063GiB</td>
						<td>1.2 seconds</td>
						<td>1.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738297408">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.10.1">v3.10.1</a>
						</td>
						<td>snakemake</td>
						<td>295 days</td>
						<td>2023-02-19 03:31:10</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- t21l28xr4a
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/t21l28xr4a/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F3jp7ixrb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F3jp7ixrb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.3 seconds</td>
						<td>1.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010763103024">
								<a href="https://nf-co.re/airrflow">airrflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/airrflow/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>294 days</td>
						<td>2023-02-20 04:31:42</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 07:31:42.533
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Fwajunl5o%2Flogs.tar.xz">0.021GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Fwajunl5o%2Foutput.tar.xz">0.018GiB</a>
						</td>
						<td>0.826GiB</td>
						<td>166.6 seconds</td>
						<td>10801.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738297168">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.10.2">v3.10.2</a>
						</td>
						<td>snakemake</td>
						<td>293 days</td>
						<td>2023-02-19 04:29:28</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 2qjt7tc1ue
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/2qjt7tc1ue/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fnn8wt5zb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fnn8wt5zb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>0.9 seconds</td>
						<td>0.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743148752">
								<a href="https://nf-co.re/quantms">quantms</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/quantms/tree/1.0">1.0</a>
						</td>
						<td>nextflow</td>
						<td>293 days</td>
						<td>2023-02-19 03:31:08</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: nextflow.exception.AbortOperationException
msg: 'Unknown configuration profile: ''test'''
rest: "Did you mean one of these?\n    oist\n\n\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.config.ConfigBuilder.checkValidProfile(ConfigBuilder.groovy:511)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.config.ConfigBuilder.buildConfig0(ConfigBuilder.groovy:385)\n\tat\
  \ java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat nextflow.config.ConfigBuilder.buildGivenFiles(ConfigBuilder.groovy:317)\n\t\
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.config.ConfigBuilder.buildConfigObject(ConfigBuilder.groovy:780)\n\
  \tat nextflow.config.ConfigBuilder.build(ConfigBuilder.groovy:793)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:307)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/quantms%2Fj7dznuaj%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/quantms%2Fj7dznuaj%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.408GiB</td>
						<td>16.4 seconds</td>
						<td>14.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743114784">
								<a href="https://nf-co.re/hicar">hicar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hicar/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>292 days</td>
						<td>2023-02-20 04:31:43</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 07:31:43.283
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hicar%2Fh65hcuce%2Flogs.tar.xz">0.193GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hicar%2Fh65hcuce%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>1.046GiB</td>
						<td>127.0 seconds</td>
						<td>10801.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743151872">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.7">3.7</a>
						</td>
						<td>nextflow</td>
						<td>292 days</td>
						<td>2023-02-19 04:16:13</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fg8qamao0%2Flogs.tar.xz">0.206GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fg8qamao0%2Foutput.tar.xz">0.031GiB</a>
						</td>
						<td>2.353GiB</td>
						<td>1455.3 seconds</td>
						<td>801.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738209488">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.3">v1.3.3</a>
						</td>
						<td>snakemake</td>
						<td>290 days</td>
						<td>2023-02-19 23:32:33</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fq01tpsn9%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fq01tpsn9%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.820GiB</td>
						<td>301.4 seconds</td>
						<td>10800.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738214528">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.3">v1.3.3</a>
						</td>
						<td>snakemake</td>
						<td>290 days</td>
						<td>2023-02-19 20:29:32</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2F1g4yehko%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2F1g4yehko%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.785GiB</td>
						<td>180.3 seconds</td>
						<td>7200.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738209968">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.1">v1.3.1</a>
						</td>
						<td>snakemake</td>
						<td>290 days</td>
						<td>2023-02-19 11:09:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fa0zv7grw%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fa0zv7grw%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.073GiB</td>
						<td>165.4 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738215248">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.0">v1.3.0</a>
						</td>
						<td>snakemake</td>
						<td>290 days</td>
						<td>2023-02-19 09:08:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fhrjzys7p%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fhrjzys7p%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.063GiB</td>
						<td>5.5 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738210208">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.0">v1.3.0</a>
						</td>
						<td>snakemake</td>
						<td>290 days</td>
						<td>2023-02-19 06:37:26</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/6sckbp5jtx/singularity-tmp/tmp4tk_r9r6.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/6sckbp5jtx/code/.test/.snakemake/conda/67ad6a6fbbffa26c70effe4b8a9426e2_.yaml"
  --prefix "/home/azureuser/tmp/6sckbp5jtx/code/.test/.snakemake/conda/67ad6a6fbbffa26c70effe4b8a9426e2_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working... done

  WARNING conda.gateways.disk.delete:unlink_or_rename_to_trash(140): Could not remove
  or rename /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/r-base-4.1.2-h2553ce4_1/lib/R/library/base/R/Rprofile.  Please
  remove this file manually (you may need to reboot to free file handles)

  WARNING conda.gateways.disk.delete:unlink_or_rename_to_trash(140): Could not remove
  or rename /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/r-base-4.1.2-h2553ce4_1/lib/R/library/base/DESCRIPTION.  Please
  remove this file manually (you may need to reboot to free file handles)

  WARNING conda.gateways.disk.delete:unlink_or_rename_to_trash(140): Could not remove
  or rename /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/r-base-4.1.2-h2553ce4_1/lib/R/library/base/INDEX.  Please
  remove this file manually (you may need to reboot to free file handles)

  WARNING conda.gateways.disk.delete:unlink_or_rename_to_trash(140): Could not remove
  or rename /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/r-base-4.1.2-h2553ce4_1/lib/R/library/base/CITATION.  Please
  remove this file manually (you may need to reboot to free file handles)

  WARNING conda.gateways.disk.delete:unlink_or_rename_to_trash(140): Could not remove
  or rename /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/r-base-4.1.2-h2553ce4_1/lib/R/COPYING.  Please
  remove this file manually (you may need to reboot to free file handles)

  WARNING conda.gateways.disk.delete:unlink_or_rename_to_trash(140): Could not remove
  or rename /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/r-base-4.1.2-h2553ce4_1/lib/R/SVN-REVISION.  Please
  remove this file manually (you may need to reboot to free file handles)


  [Errno 2] No such file or directory: ''/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/r-base-4.1.2-h2553ce4_1.tar.bz2'''

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fuwfyegsl%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fuwfyegsl%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.272GiB</td>
						<td>204.2 seconds</td>
						<td>205.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738215008">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.1">v1.3.1</a>
						</td>
						<td>snakemake</td>
						<td>290 days</td>
						<td>2023-02-19 06:05:15</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Ftijc0vz9%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Ftijc0vz9%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.128GiB</td>
						<td>222.5 seconds</td>
						<td>3600.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738209728">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.2">v1.3.2</a>
						</td>
						<td>snakemake</td>
						<td>290 days</td>
						<td>2023-02-19 04:49:22</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 5q7nhim36g
- code
- workflow
- rules
- download.smk
kind: InputFunctionException
line_no: 164
rest: "Error:\n  TypeError: 'functools.partial' object is not subscriptable\nWildcards:\n\
  \  benchmark=giab-na12878-exome\nTraceback:\n  File \"/home/azureuser/tmp/5q7nhim36g/code/workflow/rules/common.smk\"\
  , line 119, in inner\n  File \"/home/azureuser/tmp/5q7nhim36g/code/workflow/rules/download.smk\"\
  , line 173, in <lambda>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F5i5euh83%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F5i5euh83%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.4 seconds</td>
						<td>3.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738214768">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.2">v1.3.2</a>
						</td>
						<td>snakemake</td>
						<td>290 days</td>
						<td>2023-02-19 04:29:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- joyj80zn6o
- code
- workflow
- rules
- download.smk
kind: InputFunctionException
line_no: 164
rest: "Error:\n  TypeError: 'functools.partial' object is not subscriptable\nWildcards:\n\
  \  benchmark=giab-na12878-exome\nTraceback:\n  File \"/home/azureuser/tmp/joyj80zn6o/code/workflow/rules/common.smk\"\
  , line 119, in inner\n  File \"/home/azureuser/tmp/joyj80zn6o/code/workflow/rules/download.smk\"\
  , line 173, in <lambda>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fdfmjp5cz%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fdfmjp5cz%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>0.9 seconds</td>
						<td>0.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738171616">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.6">1.4.6</a>
						</td>
						<td>snakemake</td>
						<td>289 days</td>
						<td>2023-02-19 06:00:26</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 6p1dgqi88h
- code
- workflow
- rules
- reports.smk
kind: CalledProcessError
line_no: 21
rest: "Command 'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate\
  \ '/home/azureuser/tmp/6p1dgqi88h/code/.tests/.snakemake/conda/f544522396b62211963f91c7025f4913_';\
  \ set -euo pipefail;  python /home/azureuser/tmp/6p1dgqi88h/code/.tests/.snakemake/scripts/tmpvah3cjew.krona_table.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/6p1dgqi88h/code/workflow/rules/reports.smk\"\
  , line 21, in __rule_krona_table\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Sun Feb 19 06:14:43 2023]\nFinished job 30.\n35 of 61 steps\
  \ (57%) done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-19T060027.759431.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2F3rj6tc5v%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2F3rj6tc5v%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.969GiB</td>
						<td>466.0 seconds</td>
						<td>861.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738214288">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.4">v1.3.4</a>
						</td>
						<td>snakemake</td>
						<td>286 days</td>
						<td>2023-02-19 14:54:44</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fq06voxn7%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fq06voxn7%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.063GiB</td>
						<td>5.3 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738209248">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.4">v1.3.4</a>
						</td>
						<td>snakemake</td>
						<td>285 days</td>
						<td>2023-02-19 06:00:35</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F0u85za7i%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F0u85za7i%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.993GiB</td>
						<td>505.1 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743140832">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/2.3.1">2.3.1</a>
						</td>
						<td>nextflow</td>
						<td>285 days</td>
						<td>2023-02-19 04:23:04</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "DecoyDatabase -in test.fasta \\\n    -out test_decoy.fasta \\\n\
  \    -decoy_string DECOY_ \\\n    -decoy_string_position prefix\n\ncat <<-END_VERSIONS\
  \ > versions.yml\n\"NFCORE_MHCQUANT:MHCQUANT:OPENMS_DECOYDATABASE\":\n    openms:\
  \ $(echo $(FileInfo --help 2>&1) | sed 's/^.*Version: //; s/-.*$//' | sed 's/ -*//;\
  \ s/ .*$//')\nEND_VERSIONS"
error: 'WARNING: Skipping mount /home/azureuser/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/singularityce-3.10.3-z5qu45ho72nvsy4rpcyd4eqrop35tnbe/var/singularity/mnt/session/etc/resolv.conf
  [files]: /etc/resolv.conf doesn''t exist in container

  Error: File not found (the file ''test.fasta'' could not be found)

  Cannot read input file given from parameter ''-in''!'
exit_status: 1
kind: CalledProcessError
output: 'Error: File not found (the file ''test.fasta'' could not be found)'
process: NFCORE_MHCQUANT:MHCQUANT:OPENMS_DECOYDATABASE (4)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '05'
- 82a976d939b5eb16ac7263b55ab9a4

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fbty3kpwa%2Flogs.tar.xz">0.027GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fbty3kpwa%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.728GiB</td>
						<td>55.6 seconds</td>
						<td>40.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738296688">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.12.0">v3.12.0</a>
						</td>
						<td>snakemake</td>
						<td>285 days</td>
						<td>2023-02-19 06:58:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 4rfkfixstd
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/4rfkfixstd/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fjb1ml5lk%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fjb1ml5lk%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.5 seconds</td>
						<td>1.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738296928">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.11.0">v3.11.0</a>
						</td>
						<td>snakemake</td>
						<td>285 days</td>
						<td>2023-02-19 04:39:05</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- aejoeky9lx
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/aejoeky9lx/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fiq1rtqot%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fiq1rtqot%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.5 seconds</td>
						<td>3.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738296448">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.13.0">v3.13.0</a>
						</td>
						<td>snakemake</td>
						<td>283 days</td>
						<td>2023-02-19 04:42:06</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 4mqxuie09u
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/4mqxuie09u/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F2nlenxvs%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F2nlenxvs%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.6 seconds</td>
						<td>1.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738166336">
								<a href="https://github.com/snakemake-workflows/dna-seq-mtb">snakemake-workflows/dna-seq-mtb</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>283 days</td>
						<td>2023-02-19 04:30:50</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- d7bbkcpywy
- code
- workflow
- rules
- common.smk
kind: WorkflowError
line_no: 21
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/d7bbkcpywy/code/workflow/Snakefile\", line 1, in <module>\n\
  \  File \"/home/azureuser/tmp/d7bbkcpywy/code/workflow/rules/common.smk\", line\
  \ 21, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fjmn7swxg%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fjmn7swxg%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.0 seconds</td>
						<td>2.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738296208">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.13.1">v3.13.1</a>
						</td>
						<td>snakemake</td>
						<td>283 days</td>
						<td>2023-02-19 06:33:49</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- ug0dbyrgnt
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/ug0dbyrgnt/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fzi8ue7yk%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fzi8ue7yk%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.5 seconds</td>
						<td>2.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738166096">
								<a href="https://github.com/snakemake-workflows/dna-seq-mtb">snakemake-workflows/dna-seq-mtb</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.0.1">v1.0.1</a>
						</td>
						<td>snakemake</td>
						<td>283 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 3u82zbxp57
- code
- workflow
- rules
- common.smk
kind: WorkflowError
line_no: 21
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/3u82zbxp57/code/workflow/Snakefile\", line 1, in <module>\n\
  \  File \"/home/azureuser/tmp/3u82zbxp57/code/workflow/rules/common.smk\", line\
  \ 21, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fmsy0x4qe%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fmsy0x4qe%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.2 seconds</td>
						<td>1.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745351424">
								<a href="https://nf-co.re/atacseq">atacseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/atacseq/tree/1.2.2">1.2.2</a>
						</td>
						<td>nextflow</td>
						<td>282 days</td>
						<td>2023-02-19 06:59:52</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: nextflow.dag.MultipleInputChannelException
msg: Channel `design_multiple_samples` has been used as an input by more than a process
  or an operator
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:295)\n\
  \tat nextflow.dag.DAG.inbound(DAG.groovy:207)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat nextflow.dag.DAG.addVertex(DAG.groovy:145)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat nextflow.dag.DAG.addOperatorNode(DAG.groovy:116)\n\tat nextflow.dag.NodeMarker.addOperatorNode(NodeMarker.groovy:64)\n\
  \tat nextflow.extension.OpCall.addGraphNode(OpCall.groovy:178)\n\tat nextflow.extension.OpCall.invoke(OpCall.groovy:168)\n\
  \tat nextflow.extension.OpCall.call(OpCall.groovy:113)\n\tat nextflow.plugin.extension.PluginExtensionProvider.invokeExtensionMethod(PluginExtensionProvider.groovy:279)\n\
  \tat groovy.runtime.metaclass.NextflowDelegatingMetaClass.invokeMethod(NextflowDelegatingMetaClass.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.PojoMetaClassSite.call(PojoMetaClassSite.java:44)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_cf1875fb.runScript(Script_cf1875fb:377)\n\tat nextflow.script.BaseScript.runDsl1(BaseScript.groovy:164)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\
  \tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\
  \tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2F1h80tewb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2F1h80tewb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.726GiB</td>
						<td>580.3 seconds</td>
						<td>355.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743110944">
								<a href="https://nf-co.re/fetchngs">fetchngs</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/fetchngs/tree/1.6">1.6</a>
						</td>
						<td>nextflow</td>
						<td>278 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "echo SRR9984183 > id.txt\nsra_ids_to_runinfo.py \\\n    id.txt\
  \ \\\n    SRR9984183.runinfo.tsv \\\n\n\ncat <<-END_VERSIONS > versions.yml\n\"\
  NFCORE_FETCHNGS:SRA:SRA_IDS_TO_RUNINFO\":\n    python: $(python --version | sed\
  \ 's/Python //g')\nEND_VERSIONS"
error: '[ERROR] The server couldn''t fulfill the request.

  [ERROR] Status: 400 Bad Request. Both list of IDs and query_key are empty'
exit_status: 1
kind: CalledProcessError
output: (empty)
process: NFCORE_FETCHNGS:SRA:SRA_IDS_TO_RUNINFO (SRR9984183)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- fd
- 655e59bd81def01861085e1ad554fb

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fl3xu8a4r%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fl3xu8a4r%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.661GiB</td>
						<td>50.0 seconds</td>
						<td>54.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738171376">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.7">1.4.7</a>
						</td>
						<td>snakemake</td>
						<td>278 days</td>
						<td>2023-02-19 06:02:11</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- uwm26axoff
- code
- workflow
- rules
- reports.smk
kind: CalledProcessError
line_no: 21
rest: "Command 'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate\
  \ '/home/azureuser/tmp/uwm26axoff/code/.tests/.snakemake/conda/9a6c34784b2b6310268dab4e3dcc52f7_';\
  \ set -euo pipefail;  python /home/azureuser/tmp/uwm26axoff/code/.tests/.snakemake/scripts/tmpjzoh0zxc.krona_table.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/uwm26axoff/code/workflow/rules/reports.smk\"\
  , line 21, in __rule_krona_table\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Sun Feb 19 06:13:07 2023]\nFinished job 26.\n38 of 67 steps\
  \ (57%) done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-19T060212.716491.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2F87xx49f6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2F87xx49f6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.899GiB</td>
						<td>426.7 seconds</td>
						<td>660.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743149232">
								<a href="https://nf-co.re/rnafusion">rnafusion</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnafusion/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>276 days</td>
						<td>2023-02-19 03:59:42</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.cli.Launcher
date: Feb-19 04:00:01.247
kind: NextflowMiscError
process: main
rest: 'Unable to read script: ''/home/azureuser/tmp/cn9oowh3ia/code/./workflows/rnafusion.nf''
  -- cause: /home/azureuser/tmp/cn9oowh3ia/out/references/ensembl/Homo_sapiens.GRCh38.102.all.fa'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Flxdmumgz%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Flxdmumgz%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.541GiB</td>
						<td>26.0 seconds</td>
						<td>18.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738171136">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.8">1.4.8</a>
						</td>
						<td>snakemake</td>
						<td>275 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- byo6mnr9a7
- code
- workflow
- rules
- reports.smk
kind: CalledProcessError
line_no: 21
rest: "Command 'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate\
  \ '/home/azureuser/tmp/byo6mnr9a7/code/.tests/.snakemake/conda/ea2703fc29c7a0a2f7082ca0c475e3ea_';\
  \ set -euo pipefail;  python /home/azureuser/tmp/byo6mnr9a7/code/.tests/.snakemake/scripts/tmpbrxmup1r.krona_table.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/byo6mnr9a7/code/workflow/rules/reports.smk\"\
  , line 21, in __rule_krona_table\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Sun Feb 19 06:12:16 2023]\nFinished job 13.\n38 of 67 steps\
  \ (57%) done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-19T060017.544388.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fhlda5ofq%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fhlda5ofq%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.901GiB</td>
						<td>450.2 seconds</td>
						<td>722.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738295968">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.14.0">v3.14.0</a>
						</td>
						<td>snakemake</td>
						<td>275 days</td>
						<td>2023-02-19 04:35:39</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- kalpeayzyt
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/kalpeayzyt/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Femeasxmx%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Femeasxmx%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.3 seconds</td>
						<td>3.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738165856">
								<a href="https://github.com/snakemake-workflows/dna-seq-mtb">snakemake-workflows/dna-seq-mtb</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>275 days</td>
						<td>2023-02-19 04:35:47</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- gcjit64c0n
- code
- workflow
- rules
- common.smk
kind: WorkflowError
line_no: 21
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/gcjit64c0n/code/workflow/Snakefile\", line 1, in <module>\n\
  \  File \"/home/azureuser/tmp/gcjit64c0n/code/workflow/rules/common.smk\", line\
  \ 21, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fji5j1enk%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fji5j1enk%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.3 seconds</td>
						<td>2.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742160672">
								<a href="https://github.com/johnne/map_metaT">johnne/map_metaT</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/johnne/map_metaT/tree/v0.0.1">v0.0.1</a>
						</td>
						<td>snakemake</td>
						<td>272 days</td>
						<td>2023-02-19 03:31:08</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- cger668ak8
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/cger668ak8/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/johnne%252Fmap-metaT%2Felbvdkz2%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/johnne%252Fmap-metaT%2Felbvdkz2%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.1 seconds</td>
						<td>1.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742126992">
								<a href="https://github.com/johnne/map_metaT">johnne/map_metaT</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/johnne/map_metaT/tree/v0.0.2">v0.0.2</a>
						</td>
						<td>snakemake</td>
						<td>271 days</td>
						<td>2023-02-19 04:03:28</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- loq5bwj6bv
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/loq5bwj6bv/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/johnne%252Fmap-metaT%2Ffwmwl50u%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/johnne%252Fmap-metaT%2Ffwmwl50u%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.4 seconds</td>
						<td>3.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738170896">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.4.9">1.4.9</a>
						</td>
						<td>snakemake</td>
						<td>271 days</td>
						<td>2023-02-19 04:32:04</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Flazuagjx%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Flazuagjx%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.729GiB</td>
						<td>457.1 seconds</td>
						<td>424.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743151632">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.8">3.8</a>
						</td>
						<td>nextflow</td>
						<td>270 days</td>
						<td>2023-02-19 06:00:44</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F3nw5srcy%2Flogs.tar.xz">0.206GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F3nw5srcy%2Foutput.tar.xz">0.031GiB</a>
						</td>
						<td>2.385GiB</td>
						<td>1522.6 seconds</td>
						<td>1014.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743145392">
								<a href="https://nf-co.re/mnaseseq">mnaseseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mnaseseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>270 days</td>
						<td>2023-02-19 04:30:57</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: nextflow.dag.MultipleInputChannelException
msg: Channel `ch_blacklist` has been used as an input by more than a process or an
  operator
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:295)\n\
  \tat nextflow.dag.DAG.inbound(DAG.groovy:207)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat nextflow.dag.DAG.addVertex(DAG.groovy:145)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat nextflow.dag.DAG.addOperatorNode(DAG.groovy:116)\n\tat nextflow.dag.NodeMarker.addOperatorNode(NodeMarker.groovy:64)\n\
  \tat nextflow.extension.OpCall.addGraphNode(OpCall.groovy:178)\n\tat nextflow.extension.OpCall.invoke(OpCall.groovy:168)\n\
  \tat nextflow.extension.OpCall.call(OpCall.groovy:113)\n\tat nextflow.plugin.extension.PluginExtensionProvider.invokeExtensionMethod(PluginExtensionProvider.groovy:279)\n\
  \tat groovy.runtime.metaclass.NextflowDelegatingMetaClass.invokeMethod(NextflowDelegatingMetaClass.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.PojoMetaClassSite.call(PojoMetaClassSite.java:44)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)\n\
  \tat Script_325c6d19$_runScript_closure13.doCall(Script_325c6d19:403)\n\tat Script_325c6d19$_runScript_closure13.doCall(Script_325c6d19)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat nextflow.script.ProcessFactory.createProcessor(ProcessFactory.groovy:95)\n\
  \tat nextflow.script.ProcessFactory$createProcessor.call(Unknown Source)\n\tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)\n\
  \tat nextflow.script.ProcessFactory$createProcessor.call(Unknown Source)\n\tat nextflow.script.BaseScript.process(BaseScript.groovy:114)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)\n\
  \tat Script_325c6d19.runScript(Script_325c6d19:397)\n\tat nextflow.script.BaseScript.runDsl1(BaseScript.groovy:164)\n\
  \tat nextflow.script.BaseScript.run(BaseScript.groovy:217)\n\tat nextflow.script.ScriptParser.runScript(ScriptParser.groovy:230)\n\
  \tat nextflow.script.ScriptRunner.run(ScriptRunner.groovy:225)\n\tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:131)\n\
  \tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\
  \tat nextflow.cli.Launcher.main(Launcher.groovy:646)\nFeb-19 04:36:41.258 [FileTransfer-2]\
  \ DEBUG nextflow.file.FilePorter - Copying foreign file https://raw.githubusercontent.com/nf-core/test-datasets/chipseq/reference/genome.fa\
  \ to work dir: /home/azureuser/tmp/4c7ca9hffo/code/work/stage/a9/b745aad380c62381c701a1129ec16f/genome.fa"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mnaseseq%2Fnh0bp7fb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mnaseseq%2Fnh0bp7fb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.551GiB</td>
						<td>472.3 seconds</td>
						<td>343.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746334080">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/2.3.2">2.3.2</a>
						</td>
						<td>nextflow</td>
						<td>269 days</td>
						<td>2023-02-20 04:31:44</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867550576\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\t\
  at nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\
  \tat nextflow.processor.TaskRun$isContainerEnabled$6.call(Unknown Source)\n\tat\
  \ nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867550576\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 07:31:45.074 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Workflow completed -- rendering execution report\nFeb-20 07:31:45.131 [Actor\
  \ Thread 63] DEBUG nextflow.file.SortFileCollector - FileCollector temp dir not\
  \ removed: null\nFeb-20 07:31:45.165 [Actor Thread 59] INFO  nextflow.container.SingularityCache\
  \ - Pulling Singularity image https://depot.galaxyproject.org/singularity/multiqc:1.12--pyhdfd78af_0\
  \ [cache /home/azureuser/tmp/7euvbigwgy/singularity/depot.galaxyproject.org-singularity-multiqc-1.12--pyhdfd78af_0.img]\n\
  Feb-20 07:31:45.184 [main] DEBUG nextflow.trace.ReportObserver - Execution report\
  \ summary data:\n  [{\"cpuUsage\":{\"mean\":10.5,\"min\":10.5,\"q1\":10.5,\"q2\"\
  :10.5,\"q3\":10.5,\"max\":10.5,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\"},\"process\":\"METADATA_ALL\",\"mem\":{\"mean\":68591616,\"min\"\
  :68591616,\"q1\":68591616,\"q2\":68591616,\"q3\":68591616,\"max\":68591616,\"minLabel\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"},\"memUsage\":{\"mean\"\
  :1.06,\"min\":1.06,\"q1\":1.06,\"q2\":1.06,\"q3\":1.06,\"max\":1.06,\"minLabel\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"},\"timeUsage\":{\"mean\"\
  :0.01,\"min\":0.01,\"q1\":0.01,\"q2\":0.01,\"q3\":0.01,\"max\":0.01,\"minLabel\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"},\"vmem\":{\"mean\":575737856,\"\
  min\":575737856,\"q1\":575737856,\"q2\":575737856,\"q3\":575737856,\"max\":575737856,\"\
  minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"maxLabel\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"},\"reads\"\
  :{\"mean\":6085509,\"min\":6085509,\"q1\":6085509,\"q2\":6085509,\"q3\":6085509,\"\
  max\":6085509,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q1Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  },\"cpu\":{\"mean\":21,\"min\":21,\"q1\":21,\"q2\":21,\"q3\":21,\"max\":21,\"minLabel\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"},\"time\":{\"mean\":2000,\"\
  min\":2000,\"q1\":2000,\"q2\":2000,\"q3\":2000,\"max\":2000,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\"},\"writes\":{\"mean\":1116,\"min\":1116,\"q1\":1116,\"q2\":1116,\"\
  q3\":1116,\"max\":1116,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  ,\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q1Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\",\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL\
  \ (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_ALL (Metadata.tsv)\"\
  }},{\"cpuUsage\":{\"mean\":10.6,\"min\":10.6,\"q1\":10.6,\"q2\":10.6,\"q3\":10.6,\"\
  max\":10.6,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q1Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  },\"process\":\"METADATA_PAIRWISE\",\"mem\":{\"mean\":69120000,\"min\":69120000,\"\
  q1\":69120000,\"q2\":69120000,\"q3\":69120000,\"max\":69120000,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"memUsage\":{\"mean\":1.07,\"min\":1.07,\"q1\":1.07,\"q2\"\
  :1.07,\"q3\":1.07,\"max\":1.07,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"timeUsage\":{\"mean\":0.01,\"min\":0.01,\"q1\":0.01,\"q2\"\
  :0.01,\"q3\":0.01,\"max\":0.01,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"vmem\":{\"mean\":576151552,\"min\":576151552,\"q1\":576151552,\"\
  q2\":576151552,\"q3\":576151552,\"max\":576151552,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"reads\":{\"mean\":6085706,\"min\":6085706,\"q1\":6085706,\"\
  q2\":6085706,\"q3\":6085706,\"max\":6085706,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"cpu\":{\"mean\":21.2,\"min\":21.2,\"q1\":21.2,\"q2\":21.2,\"\
  q3\":21.2,\"max\":21.2,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"time\":{\"mean\":3000,\"min\":3000,\"q1\":3000,\"q2\":3000,\"\
  q3\":3000,\"max\":3000,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"},\"writes\":{\"mean\":1127,\"min\":1127,\"q1\":1127,\"q2\":1127,\"\
  q3\":1127,\"max\":1127,\"minLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\",\"maxLabel\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\"\
  ,\"q1Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q2Label\"\
  :\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE (Metadata.tsv)\",\"q3Label\":\"NFCORE_AMPLISEQ:AMPLISEQ:METADATA_PAIRWISE\
  \ (Metadata.tsv)\"}}]\nFeb-20 07:31:45.651 [main] DEBUG nextflow.trace.TimelineObserver\
  \ - Workflow completed -- rendering execution timeline\nFeb-20 07:31:45.858 [main]\
  \ DEBUG nextflow.cache.CacheDB - Closing CacheDB done\nFeb-20 07:31:45.923 [main]\
  \ DEBUG nextflow.script.ScriptRunner - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fnzt0t7dz%2Flogs.tar.xz">0.019GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Fnzt0t7dz%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.195GiB</td>
						<td>61.8 seconds</td>
						<td>10801.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743151392">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.8.1">3.8.1</a>
						</td>
						<td>nextflow</td>
						<td>268 days</td>
						<td>2023-02-19 06:31:01</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fwg4wa450%2Flogs.tar.xz">0.207GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fwg4wa450%2Foutput.tar.xz">0.031GiB</a>
						</td>
						<td>2.298GiB</td>
						<td>1502.1 seconds</td>
						<td>894.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744727152">
								<a href="https://nf-co.re/circdna">circdna</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/circdna/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>264 days</td>
						<td>2023-02-19 11:52:25</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/circdna%2F6tza02ii%2Flogs.tar.xz">0.952GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/circdna%2F6tza02ii%2Foutput.tar.xz">0.410GiB</a>
						</td>
						<td>3.147GiB</td>
						<td>3570.8 seconds</td>
						<td>1908.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743193584">
								<a href="https://nf-co.re/smrnaseq">smrnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/smrnaseq/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>263 days</td>
						<td>2023-02-19 06:00:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "export mirtracejar=$(dirname $(which mirtrace))\nfor i in C1-N1-R1_S4_L001_R1_001.fastq.gz\
  \ C1-N3-R1_S6_L001_R1_001.fastq.gz C9-N1-R1_S7_L001_R1_001.fastq.gz C9-N2-R1_S8_L001_R1_001.fastq.gz\
  \ C9-N3-R1_S9_L001_R1_001.fastq.gz Ctl-N1-R1_S1_L001_R1_001.fastq.gz Ctl-N2-R1_S2_L001_R1_001.fastq.gz\
  \ Ctl-N3-R1_S3_L001_R1_001.fastq.gz\ndo\n    path=$(realpath ${i})\n    prefix=$(echo\
  \ ${i} | sed -e 's/.gz//' -e 's/.fastq//' -e 's/.fq//' -e 's/_val_1//' -e 's/_trimmed//'\
  \ -e 's/_R1//' -e 's/.R1//')\n    echo $path\",\"$prefix\ndone > mirtrace_config\n\
  \njava -Xms6442450944 -Xmx6442450944 -jar $mirtracejar/mirtrace.jar --mirtrace-wrapper-name\
  \ mirtrace qc  \\\n    --species hsa \\\n     --adapter TGGAATTCTCGGGTGCCAAGG  \\\
  \n    --protocol illumina \\\n    --config mirtrace_config \\\n    --write-fasta\
  \ \\\n    --output-dir mirtrace \\\n    --force\n\ncat <<-END_VERSIONS > versions.yml\n\
  NFCORE_SMRNASEQ:SMRNASEQ:MIRTRACE:MIRTRACE_RUN\":\n    mirtrace: $(echo $(mirtrace\
  \ -v 2>&1))\nEND_VERSIONS"
error: 'miRTrace version 1.0.1 starting. Processing 8 sample(s).

  Failed parsing FASTQ file ''/home/azureuser/tmp/ndiqmlsb12/code/work/stage/6f/7ce558494e72ba34755053ece2bdb0/C9-N2-R1_S8_L001_R1_001.fastq.gz'':
  file not found.

  Error parsing FASTQ file, aborting.'
exit_status: 255
kind: CalledProcessError
output: (empty)
process: NFCORE_SMRNASEQ:SMRNASEQ:MIRTRACE:MIRTRACE_RUN
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '71'
- 0d4d901066ebced317d930a5fb79bc

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/smrnaseq%2Faomy3773%2Flogs.tar.xz">0.802GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/smrnaseq%2Faomy3773%2Foutput.tar.xz">0.014GiB</a>
						</td>
						<td>0.905GiB</td>
						<td>709.1 seconds</td>
						<td>552.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738170656">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.5.0">1.5.0</a>
						</td>
						<td>snakemake</td>
						<td>262 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2F57kr5v6l%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2F57kr5v6l%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.793GiB</td>
						<td>635.4 seconds</td>
						<td>888.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010761613616">
								<a href="https://nf-co.re/airrflow">airrflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/airrflow/tree/2.2.0">2.2.0</a>
						</td>
						<td>nextflow</td>
						<td>262 days</td>
						<td>2023-02-20 05:30:54</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 08:30:54.210
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Fpo60znzk%2Flogs.tar.xz">0.021GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Fpo60znzk%2Foutput.tar.xz">0.018GiB</a>
						</td>
						<td>0.823GiB</td>
						<td>145.9 seconds</td>
						<td>10800.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738170416">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.5.1">1.5.1</a>
						</td>
						<td>snakemake</td>
						<td>261 days</td>
						<td>2023-02-19 06:37:24</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fezas5oec%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fezas5oec%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.926GiB</td>
						<td>550.5 seconds</td>
						<td>525.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738165616">
								<a href="https://github.com/snakemake-workflows/dna-seq-mtb">snakemake-workflows/dna-seq-mtb</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.2.0">v1.2.0</a>
						</td>
						<td>snakemake</td>
						<td>256 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 746olzs12z
- code
- workflow
- rules
- common.smk
kind: WorkflowError
line_no: 20
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/746olzs12z/code/workflow/Snakefile\", line 1, in <module>\n\
  \  File \"/home/azureuser/tmp/746olzs12z/code/workflow/rules/common.smk\", line\
  \ 20, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2F19ru8o4v%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2F19ru8o4v%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.5 seconds</td>
						<td>1.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738209008">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.5">v1.3.5</a>
						</td>
						<td>snakemake</td>
						<td>256 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fxhrwvb5d%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fxhrwvb5d%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.975GiB</td>
						<td>425.1 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738214048">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.5">v1.3.5</a>
						</td>
						<td>snakemake</td>
						<td>256 days</td>
						<td>2023-02-19 03:30:59</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Faeypxiok%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Faeypxiok%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.828GiB</td>
						<td>419.9 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743050208">
								<a href="https://nf-co.re/cutandrun">cutandrun</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/cutandrun/tree/2.0">2.0</a>
						</td>
						<td>nextflow</td>
						<td>256 days</td>
						<td>2023-02-19 06:46:02</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "mkdir output\n\ntar \\\n    -C output --strip-components 1 \\\n\
  \    -xzvf \\\n     \\\n    hg38-chr20-bowtie2.tar.gz \\\n    --no-same-owner\n\n\
  mv output hg38-chr20-bowtie2\n\ncat <<-END_VERSIONS > versions.yml\n\"NFCORE_CUTANDRUN:CUTANDRUN:PREPARE_GENOME:UNTAR_INDEX_TARGET\"\
  :\n    untar: $(echo $(tar --version 2>&1) | sed 's/^.*(GNU tar) //; s/ Copyright.*$//')\n\
  END_VERSIONS"
error: 'gzip: stdin: unexpected end of file

  hg38-chr20-bowtie2/hg38-chr20.4.bt2

  tar: Unexpected EOF in archive

  tar: Unexpected EOF in archive

  tar: Error is not recoverable: exiting now'
exit_status: 2
kind: CalledProcessError
output: hg38-chr20-bowtie2/hg38-chr20.4.bt2
process: NFCORE_CUTANDRUN:CUTANDRUN:PREPARE_GENOME:UNTAR_INDEX_TARGET (hg38-chr20-bowtie2.tar.gz)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '10'
- ff7ec9be537d12cd9d72469660ff57

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cutandrun%2F7cuau0ym%2Flogs.tar.xz">0.041GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cutandrun%2F7cuau0ym%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>1.036GiB</td>
						<td>60.0 seconds</td>
						<td>52.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738170176">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.6.0">1.6.0</a>
						</td>
						<td>snakemake</td>
						<td>255 days</td>
						<td>2023-02-19 03:32:29</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fh7v1anfv%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fh7v1anfv%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.228GiB</td>
						<td>701.9 seconds</td>
						<td>1159.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743190224">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/2.7.2">2.7.2</a>
						</td>
						<td>nextflow</td>
						<td>254 days</td>
						<td>2023-02-19 04:51:40</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: nextflow.exception.AbortOperationException
msg: 'DAG file already exists: /home/azureuser/tmp/zlvp1hjcjl/out/pipeline_info/pipeline_dag_2023-02-19_04-51-49.svg
  -- enable `dag.overwrite` in your config file to overwrite existing DAG files'
rest: "at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native\
  \ Method)\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\
  \tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)\n\
  \tat org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)\n\
  \tat org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)\n\
  \tat nextflow.trace.GraphObserver.onFlowCreate(GraphObserver.groovy:77)\n\tat nextflow.Session$_notifyFlowCreate_closure22.doCall(Session.groovy:1055)\n\
  \tat nextflow.Session$_notifyFlowCreate_closure22.call(Session.groovy)\n\tat org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2359)\n\
  \tat org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2344)\n\
  \tat org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2385)\n\
  \tat nextflow.Session.notifyFlowCreate(Session.groovy:1055)\n\tat nextflow.Session.start(Session.groovy:517)\n\
  \tat nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:126)\n\tat nextflow.cli.CmdRun.run(CmdRun.groovy:354)\n\
  \tat nextflow.cli.Launcher.run(Launcher.groovy:487)\n\tat nextflow.cli.Launcher.main(Launcher.groovy:646)"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fw94lki90%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fw94lki90%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.316GiB</td>
						<td>16.7 seconds</td>
						<td>9.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743118624">
								<a href="https://nf-co.re/mag">mag</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mag/tree/2.2.0">2.2.0</a>
						</td>
						<td>nextflow</td>
						<td>250 days</td>
						<td>2023-02-19 17:00:04</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fbw1kgjg9%2Flogs.tar.xz">0.463GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fbw1kgjg9%2Foutput.tar.xz">0.032GiB</a>
						</td>
						<td>1.133GiB</td>
						<td>2359.9 seconds</td>
						<td>989.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738256240">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.3">1.3</a>
						</td>
						<td>snakemake</td>
						<td>250 days</td>
						<td>2023-02-19 03:42:36</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 12
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/gv8c57nkud/code/workflow/Snakefile\", line 12, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fpbhi4g1z%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fpbhi4g1z%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>2.0 seconds</td>
						<td>5.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743188544">
								<a href="https://nf-co.re/rnavar">rnavar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnavar/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>248 days</td>
						<td>2023-02-19 04:22:09</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnavar%2Fxdo5l7ud%2Flogs.tar.xz">0.004GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnavar%2Fxdo5l7ud%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>1.954GiB</td>
						<td>440.0 seconds</td>
						<td>340.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738169936">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.6.1">1.6.1</a>
						</td>
						<td>snakemake</td>
						<td>247 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fjhf8nslx%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fjhf8nslx%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.982GiB</td>
						<td>690.6 seconds</td>
						<td>1041.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743192384">
								<a href="https://nf-co.re/scrnaseq">scrnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/scrnaseq/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>247 days</td>
						<td>2023-02-19 03:39:35</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/scrnaseq%2F5fxv54rb%2Flogs.tar.xz">0.280GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/scrnaseq%2F5fxv54rb%2Foutput.tar.xz">0.241GiB</a>
						</td>
						<td>0.872GiB</td>
						<td>264.7 seconds</td>
						<td>149.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743145632">
								<a href="https://nf-co.re/nanoseq">nanoseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/nanoseq/tree/3.0.0">3.0.0</a>
						</td>
						<td>nextflow</td>
						<td>245 days</td>
						<td>2023-02-20 05:18:54</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 08:18:54.620
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nanoseq%2Fnrfwidcl%2Flogs.tar.xz">0.220GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nanoseq%2Fnrfwidcl%2Foutput.tar.xz">0.004GiB</a>
						</td>
						<td>1.005GiB</td>
						<td>582.3 seconds</td>
						<td>10800.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738169696">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.6.2">1.6.2</a>
						</td>
						<td>snakemake</td>
						<td>244 days</td>
						<td>2023-02-19 04:34:09</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fpy0yag59%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fpy0yag59%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.845GiB</td>
						<td>761.8 seconds</td>
						<td>1181.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744626624">
								<a href="https://nf-co.re/circdna">circdna</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/circdna/tree/1.0.1">1.0.1</a>
						</td>
						<td>nextflow</td>
						<td>242 days</td>
						<td>2023-02-19 03:48:08</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/circdna%2F631nz7gu%2Flogs.tar.xz">0.953GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/circdna%2F631nz7gu%2Foutput.tar.xz">0.410GiB</a>
						</td>
						<td>2.728GiB</td>
						<td>4187.0 seconds</td>
						<td>2026.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738268480">
								<a href="https://github.com/GooglingTheCancerGenome/sv-callers">GooglingTheCancerGenome/sv-callers</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.2.0">v1.2.0</a>
						</td>
						<td>snakemake</td>
						<td>241 days</td>
						<td>2023-02-19 03:31:13</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: ModuleNotFoundError
line_no: 7
rest: "No module named 'yatiml'\n  File \"/home/azureuser/tmp/5018p5k5a0/code/workflow/Snakefile\"\
  , line 7, in <module>\n  File \"/home/azureuser/tmp/5018p5k5a0/code/workflow/helper_functions.py\"\
  , line 5, in <module>\n  File \"/home/azureuser/tmp/5018p5k5a0/code/workflow/validator/__init__.py\"\
  , line 8, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2F5577zt9w%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2F5577zt9w%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.061GiB</td>
						<td>1.6 seconds</td>
						<td>1.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743118144">
								<a href="https://nf-co.re/isoseq">isoseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/isoseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>236 days</td>
						<td>2023-02-19 06:34:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/isoseq%2F75a8g5zu%2Flogs.tar.xz">0.190GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/isoseq%2F75a8g5zu%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>2.633GiB</td>
						<td>1114.2 seconds</td>
						<td>969.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738255760">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.5.1-dev">1.5.1-dev</a>
						</td>
						<td>snakemake</td>
						<td>235 days</td>
						<td>2023-02-20 05:02:35</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- input_check.smk
kind: CalledProcessError
line_no: 23
rest: "Command ' singularity  exec --home /home/azureuser/tmp/5x41axirtc/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/5x41axirtc/code/.snakemake/singularity/4116f096d6b74a20636c9be9d8f6e48c.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/cd15c940fe1181e1751e02b25c7fcfb7'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/5x41axirtc/code/.snakemake/scripts/tmp8w6_09kf.generate_exclude_file.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/5x41axirtc/code/workflow/rules/input_check.smk\"\
  , line 23, in __rule_generate_exclude_file_for_mosaic_count\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 05:09:25 2023]\nError in rule determine_sex_per_cell:\n\
  \    jobid: 48\n    input: .tests/data/RPE-BM510/selected\n    output: .tests/output/config/RPE-BM510/sex_analysis_cells.tsv,\
  \ .tests/output/config/RPE-BM510/sex_analysis_sample.txt\n    log: .tests/output/log/strandphaser/determine_sex_per_cell/RPE-BM510.log\
  \ (check log file(s) for error message)\n    conda-env: /conda-envs/cd15c940fe1181e1751e02b25c7fcfb7\n\
  \nRuleException:\nCalledProcessError in line 47 of /home/azureuser/tmp/5x41axirtc/code/workflow/rules/strandphaser.smk:\n\
  Command ' singularity  exec --home /home/azureuser/tmp/5x41axirtc/code  --bind /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/5x41axirtc/code/.snakemake/singularity/4116f096d6b74a20636c9be9d8f6e48c.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/cd15c940fe1181e1751e02b25c7fcfb7'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/5x41axirtc/code/.snakemake/scripts/tmp11o8prdb.chrxy_analysis.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/5x41axirtc/code/workflow/rules/strandphaser.smk\"\
  , line 47, in __rule_determine_sex_per_cell\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T050236.404248.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Frfv01fxw%2Flogs.tar.xz">2.385GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Frfv01fxw%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.673GiB</td>
						<td>576.7 seconds</td>
						<td>409.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740080768">
								<a href="https://github.com/solida-core/dima">solida-core/dima</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/dima/tree/v2.0">v2.0</a>
						</td>
						<td>snakemake</td>
						<td>233 days</td>
						<td>2023-02-19 03:47:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- trimming_se.smk
kind: MissingInputException
rest: "Missing input files for rule pre_rename_fastq_se:\n    output: /path/to/results_dir/reads/untrimmed/se/HSQ1008_141.L001.NA20804-R1.fq.gz\n\
  \    wildcards: unit=HSQ1008_141.L001.NA20804\n    affected files:\n        /home/matteo/Scrivania/crs4_pipelines/solida-core/dima/test-data-DNA/data/fastq/NA20804.L001.R1.fq.gz"
rule: pre_rename_fastq_se  in line 2

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2Fkp3m9ddm%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2Fkp3m9ddm%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>2.4 seconds</td>
						<td>5.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743110704">
								<a href="https://nf-co.re/fetchngs">fetchngs</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/fetchngs/tree/1.7">1.7</a>
						</td>
						<td>nextflow</td>
						<td>233 days</td>
						<td>2023-02-19 06:00:47</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fnk8lglak%2Flogs.tar.xz">0.033GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fnk8lglak%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>0.764GiB</td>
						<td>59.8 seconds</td>
						<td>101.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738256000">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.5.1">1.5.1</a>
						</td>
						<td>snakemake</td>
						<td>229 days</td>
						<td>2023-02-19 06:44:50</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- Snakefile
kind: ModuleNotFoundError
line_no: 71
rest: "No module named 'pysam'\n  File \"/home/azureuser/tmp/hmj0rcn4cl/code/Snakefile\"\
  , line 71, in <module>\n  File \"/home/azureuser/tmp/hmj0rcn4cl/code/workflow/scripts/utils/handle_input.py\"\
  , line 5, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fd4mazq2i%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fd4mazq2i%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.098GiB</td>
						<td>1.3 seconds</td>
						<td>11.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740012592">
								<a href="https://github.com/solida-core/dima">solida-core/dima</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/dima/tree/v2.0.1">v2.0.1</a>
						</td>
						<td>snakemake</td>
						<td>228 days</td>
						<td>2023-02-19 03:31:40</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- picard.smk
kind: AttributeError
line_no: 79
rest: "'NoneType' object has no attribute 'get'\n  File \"/home/azureuser/tmp/q09gyw8enl/code/workflow/Snakefile\"\
  , line 19, in <module>\n  File \"/home/azureuser/tmp/q09gyw8enl/code/workflow/rules/picard.smk\"\
  , line 79, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2Ft9m6903y%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2Ft9m6903y%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>2.1 seconds</td>
						<td>5.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742311776">
								<a href="https://github.com/solida-core/musta">solida-core/musta</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/musta/tree/v1.2.1">v1.2.1</a>
						</td>
						<td>snakemake</td>
						<td>227 days</td>
						<td>2023-02-19 03:31:14</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2F1534hp9f%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2F1534hp9f%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>0.063GiB</td>
						<td>1.7 seconds</td>
						<td>2.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743117904">
								<a href="https://nf-co.re/isoseq">isoseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/isoseq/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>222 days</td>
						<td>2023-02-19 06:18:13</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/isoseq%2Fcbgc6xyv%2Flogs.tar.xz">0.088GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/isoseq%2Fcbgc6xyv%2Foutput.tar.xz">0.030GiB</a>
						</td>
						<td>2.644GiB</td>
						<td>915.7 seconds</td>
						<td>1032.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743148992">
								<a href="https://nf-co.re/rnafusion">rnafusion</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnafusion/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>222 days</td>
						<td>2023-02-19 03:33:48</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.cli.Launcher
date: Feb-19 03:34:08.208
kind: NextflowMiscError
process: main
rest: 'Unable to read script: ''/home/azureuser/tmp/auooxnsnkf/code/./workflows/rnafusion.nf''
  -- cause: /home/azureuser/tmp/auooxnsnkf/out/references/ensembl/Homo_sapiens.GRCh38.102.all.fa'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Feuo8n2t8%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnafusion%2Feuo8n2t8%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.526GiB</td>
						<td>19.4 seconds</td>
						<td>20.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743194304">
								<a href="https://nf-co.re/viralrecon">viralrecon</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/viralrecon/tree/2.5">2.5</a>
						</td>
						<td>nextflow</td>
						<td>221 days</td>
						<td>2023-02-19 10:39:24</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2F1cggal03%2Flogs.tar.xz">0.192GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/viralrecon%2F1cggal03%2Foutput.tar.xz">0.030GiB</a>
						</td>
						<td>1.197GiB</td>
						<td>1592.0 seconds</td>
						<td>570.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739977136">
								<a href="https://github.com/solida-core/dima">solida-core/dima</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/dima/tree/v2.0.2">v2.0.2</a>
						</td>
						<td>snakemake</td>
						<td>221 days</td>
						<td>2023-02-19 07:00:45</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- picard.smk
kind: AttributeError
line_no: 137
rest: "'NoneType' object has no attribute 'get'\n  File \"/home/azureuser/tmp/0qq4gk4oeu/code/workflow/Snakefile\"\
  , line 17, in <module>\n  File \"/home/azureuser/tmp/0qq4gk4oeu/code/workflow/rules/picard.smk\"\
  , line 137, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2F5ndko7zf%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fdima%2F5ndko7zf%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>2.0 seconds</td>
						<td>4.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742245568">
								<a href="https://github.com/solida-core/musta">solida-core/musta</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/solida-core/musta/tree/v1.2.2">v1.2.2</a>
						</td>
						<td>snakemake</td>
						<td>220 days</td>
						<td>2023-02-19 06:51:21</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2Fwitn8bjq%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/solida-core%252Fmusta%2Fwitn8bjq%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>2.2 seconds</td>
						<td>5.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738213808">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.3.6">v1.3.6</a>
						</td>
						<td>snakemake</td>
						<td>214 days</td>
						<td>2023-02-19 23:32:28</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fkcww5c26%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fkcww5c26%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.780GiB</td>
						<td>265.0 seconds</td>
						<td>10800.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738208768">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.3.6">v1.3.6</a>
						</td>
						<td>snakemake</td>
						<td>214 days</td>
						<td>2023-02-19 13:52:04</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.WorkflowTimeoutError
kind: timeout
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F85xtaqa0%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F85xtaqa0%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.125GiB</td>
						<td>359.6 seconds</td>
						<td>3600.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743189984">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/3.0">3.0</a>
						</td>
						<td>nextflow</td>
						<td>213 days</td>
						<td>2023-02-19 06:15:41</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fr1p7hmoy%2Flogs.tar.xz">0.126GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fr1p7hmoy%2Foutput.tar.xz">0.025GiB</a>
						</td>
						<td>2.347GiB</td>
						<td>246.1 seconds</td>
						<td>202.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738169216">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.0">1.0</a>
						</td>
						<td>snakemake</td>
						<td>213 days</td>
						<td>2023-02-19 04:03:49</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: "\nWorkflowError:\n    MissingInputException: Missing input files for rule bwa_strandseq_to_reference_alignment:\n\
  \        output: .tests/data_CHR21/RPE-BM510/all/BM510x3PE20401.bam\n        wildcards:\
  \ path=.tests/data_CHR21, sample=RPE-BM510, cell=BM510x3PE20401\n        affected\
  \ files:\n            sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n\
  \    MissingInputException: Missing input files for rule bwa_strandseq_to_reference_alignment:\n\
  \        output: .tests/data_CHR21/RPE-BM510/all/BM510x3PE20401.sort.bam\n     \
  \   wildcards: path=.tests/data_CHR21, sample=RPE-BM510, cell=BM510x3PE20401.sort\n\
  \        affected files:\n            .tests/data_CHR21/RPE-BM510/fastq/BM510x3PE20401.sort.2.fastq.gz\n\
  \            sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n\
  \            .tests/data_CHR21/RPE-BM510/fastq/BM510x3PE20401.sort.1.fastq.gz\n\
  MissingInputException: Missing input files for rule bwa_strandseq_to_reference_alignment:\n\
  \    output: .tests/data_CHR21/RPE-BM510/all/BM510x3PE20401.sort.mdup.bam\n    wildcards:\
  \ path=.tests/data_CHR21, sample=RPE-BM510, cell=BM510x3PE20401.sort.mdup\n    affected\
  \ files:\n        .tests/data_CHR21/RPE-BM510/fastq/BM510x3PE20401.sort.mdup.1.fastq.gz\n\
  \        sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n\
  \        .tests/data_CHR21/RPE-BM510/fastq/BM510x3PE20401.sort.mdup.2.fastq.gz\n"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fmw8hhg1s%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fmw8hhg1s%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>2.0 seconds</td>
						<td>3.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738168976">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.1">1.1</a>
						</td>
						<td>snakemake</td>
						<td>213 days</td>
						<td>2023-02-19 04:15:59</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: "\nMissingInputException: Missing input files for rule bwa_strandseq_to_reference_alignment:\n\
  \    output: .tests/data_CHR21/RPE-BM510/all/BM510x3PE20401.sort.mdup.bam\n    wildcards:\
  \ path=.tests/data_CHR21, sample=RPE-BM510, cell=BM510x3PE20401.sort.mdup\n    affected\
  \ files:\n        .tests/data_CHR21/RPE-BM510/fastq/BM510x3PE20401.sort.mdup.2.fastq.gz\n\
  \        .tests/data_CHR21/RPE-BM510/fastq/BM510x3PE20401.sort.mdup.1.fastq.gz\n\
  \        sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n\
  WorkflowError:\n    MissingInputException: Missing input files for rule bwa_strandseq_to_reference_alignment:\n\
  \        output: .tests/data_CHR21/RPE-BM510/all/BM510x3PE20401.sort.bam\n     \
  \   wildcards: path=.tests/data_CHR21, sample=RPE-BM510, cell=BM510x3PE20401.sort\n\
  \        affected files:\n            .tests/data_CHR21/RPE-BM510/fastq/BM510x3PE20401.sort.1.fastq.gz\n\
  \            .tests/data_CHR21/RPE-BM510/fastq/BM510x3PE20401.sort.2.fastq.gz\n\
  \            sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n\
  \    MissingInputException: Missing input files for rule bwa_strandseq_to_reference_alignment:\n\
  \        output: .tests/data_CHR21/RPE-BM510/all/BM510x3PE20401.bam\n        wildcards:\
  \ path=.tests/data_CHR21, sample=RPE-BM510, cell=BM510x3PE20401\n        affected\
  \ files:\n            sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F38tz7gs8%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F38tz7gs8%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.103GiB</td>
						<td>1.9 seconds</td>
						<td>3.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738295728">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.14.1">v3.14.1</a>
						</td>
						<td>snakemake</td>
						<td>209 days</td>
						<td>2023-02-19 04:47:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- bqxa6gnfmd
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/bqxa6gnfmd/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fkkqhkm1t%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fkkqhkm1t%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.1 seconds</td>
						<td>1.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740945472">
								<a href="https://github.com/vembrane/vembrane-benchmark">vembrane/vembrane-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/vembrane/vembrane-benchmark/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>207 days</td>
						<td>2023-02-19 03:31:08</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/m9sviisdha/singularity-tmp/tmpy499k5p9.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/m9sviisdha/code/.snakemake/conda/5bbfd264b173cf328b078f14f3946330_.yaml"
  --prefix "/home/azureuser/tmp/m9sviisdha/code/.snakemake/conda/5bbfd264b173cf328b078f14f3946330_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working...'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/vembrane%252Fvembrane-benchmark%2Fx2zix3ev%2Flogs.tar.xz">0.628GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/vembrane%252Fvembrane-benchmark%2Fx2zix3ev%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>11.400GiB</td>
						<td>1064.3 seconds</td>
						<td>1110.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738168736">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.2.0">1.2.0</a>
						</td>
						<td>snakemake</td>
						<td>207 days</td>
						<td>2023-02-19 04:45:51</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: "\nMissingInputException: Missing input files for rule bwa_index:\n    output:\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.amb,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.ann,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.bwt,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.pac,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.sa\n\
  \    affected files:\n        sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n\
  WorkflowError:\n    MissingInputException: Missing input files for rule bwa_index:\n\
  \        output: sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.amb,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.ann,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.bwt,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.pac,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.sa\n\
  \        affected files:\n            sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n\
  \    MissingInputException: Missing input files for rule bwa_index:\n        output:\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.amb,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.ann,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.bwt,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.pac,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.sa\n\
  \        affected files:\n            sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fc3w4ouga%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fc3w4ouga%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>2.0 seconds</td>
						<td>4.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741990400">
								<a href="https://github.com/dbespiatykh/TBvar">dbespiatykh/TBvar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/dbespiatykh/TBvar/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>206 days</td>
						<td>2023-02-19 03:51:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- reference.smk
kind: WorkflowError
line_no: 1
rest: "The Python package 'biopython' needs to be installed to use NCBI Entrez remote()\
  \ file functionality. No module named 'Bio'\n  File \"/home/azureuser/tmp/xvfodchfsj/code/workflow/Snakefile\"\
  , line 14, in <module>\n  File \"/home/azureuser/tmp/xvfodchfsj/code/workflow/rules/reference.smk\"\
  , line 1, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2Fm0lm3c2w%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2Fm0lm3c2w%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>1.7 seconds</td>
						<td>2.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741028592">
								<a href="https://github.com/SilkeAllmannLab/pacbio_snakemake">SilkeAllmannLab/pacbio_snakemake</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/SilkeAllmannLab/pacbio_snakemake/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>205 days</td>
						<td>2023-02-19 04:49:32</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- rdpiipgl1d
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 19
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/rdpiipgl1d/code/workflow/Snakefile\", line 19, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/SilkeAllmannLab%252Fpacbio-snakemake%2F6bmmm8et%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/SilkeAllmannLab%252Fpacbio-snakemake%2F6bmmm8et%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.7 seconds</td>
						<td>3.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738255520">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.6.0">1.6.0</a>
						</td>
						<td>snakemake</td>
						<td>205 days</td>
						<td>2023-02-19 06:01:34</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: '

  Unable to find environment in container image. Maybe a conda environment was modified
  without containerizing again (see snakemake --containerize)?

  Details:

  Command ''SINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/f7e4bc8e-ca2a-4e7f-9411-7541731567db
  singularity --quiet --silent exec --home /home/azureuser/tmp/mpx9s0vdfg/code  /home/azureuser/tmp/mpx9s0vdfg/code/.snakemake/singularity/05142899ca28933c11364ad46012383f.simg
  sh -c ''[ -d ''\''''/conda-envs/06d3458a8e9d69aa9f45e965a1601670''\'''' ]'''' returned
  non-zero exit status 1.


  '

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2F6pvpnqun%2Flogs.tar.xz">2.281GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2F6pvpnqun%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.721GiB</td>
						<td>597.3 seconds</td>
						<td>561.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738266800">
								<a href="https://github.com/mrvollger/StainedGlass">mrvollger/StainedGlass</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/mrvollger/StainedGlass/tree/v0.5">v0.5</a>
						</td>
						<td>snakemake</td>
						<td>201 days</td>
						<td>2023-02-19 16:18:41</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FStainedGlass%2Fg7rrc7u4%2Flogs.tar.xz">0.194GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mrvollger%252FStainedGlass%2Fg7rrc7u4%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>11.277GiB</td>
						<td>2029.3 seconds</td>
						<td>2144.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743054288">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.4.5">2.4.5</a>
						</td>
						<td>nextflow</td>
						<td>201 days</td>
						<td>2023-02-19 03:31:07</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fug0pmwiy%2Flogs.tar.xz">0.015GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fug0pmwiy%2Foutput.tar.xz">0.012GiB</a>
						</td>
						<td>1.661GiB</td>
						<td>725.5 seconds</td>
						<td>548.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738168496">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.2.1">1.2.1</a>
						</td>
						<td>snakemake</td>
						<td>201 days</td>
						<td>2023-02-19 03:43:48</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: "\nWorkflowError:\n    MissingInputException: Missing input files for rule bwa_index:\n\
  \        output: sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.amb,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.ann,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.bwt,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.pac,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.sa\n\
  \        affected files:\n            sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n\
  \    MissingInputException: Missing input files for rule bwa_index:\n        output:\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.amb,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.ann,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.bwt,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.pac,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.sa\n\
  \        affected files:\n            sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n\
  MissingInputException: Missing input files for rule bwa_index:\n    output: sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.amb,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.ann,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.bwt,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.pac,\
  \ sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.sa\n\
  \    affected files:\n        sandbox.zenodo.org/record/1074721/files/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna\n"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F9j063d2r%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F9j063d2r%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>1.9 seconds</td>
						<td>3.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738217168">
								<a href="https://github.com/dbespiatykh/RDscan">dbespiatykh/RDscan</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/dbespiatykh/RDscan/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>199 days</td>
						<td>2023-02-19 03:31:23</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- reference.smk
kind: WorkflowError
line_no: 1
rest: "The Python package 'biopython' needs to be installed to use NCBI Entrez remote()\
  \ file functionality. No module named 'Bio'\n  File \"/home/azureuser/tmp/yai1bocsaf/code/workflow/Snakefile\"\
  , line 19, in <module>\n  File \"/home/azureuser/tmp/yai1bocsaf/code/workflow/rules/reference.smk\"\
  , line 1, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FRDscan%2F08025nzs%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FRDscan%2F08025nzs%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>2.0 seconds</td>
						<td>2.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743109264">
								<a href="https://nf-co.re/epitopeprediction">epitopeprediction</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/epitopeprediction/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>199 days</td>
						<td>2023-02-19 04:10:46</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epitopeprediction%2Fb5zrw9d8%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epitopeprediction%2Fb5zrw9d8%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.932GiB</td>
						<td>410.2 seconds</td>
						<td>283.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738295488">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.15.0">v3.15.0</a>
						</td>
						<td>snakemake</td>
						<td>195 days</td>
						<td>2023-02-19 04:36:32</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- bofvhmnr99
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/bofvhmnr99/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F49oij28r%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F49oij28r%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.4 seconds</td>
						<td>2.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740928176">
								<a href="https://github.com/vembrane/vembrane-benchmark">vembrane/vembrane-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/vembrane/vembrane-benchmark/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>193 days</td>
						<td>2023-02-19 13:41:50</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/os6b5mmm09/singularity-tmp/tmpxopn4r18.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/os6b5mmm09/code/.snakemake/conda/7c919ec979cc84461d9998d05f620a4c_.yaml"
  --prefix "/home/azureuser/tmp/os6b5mmm09/code/.snakemake/conda/7c919ec979cc84461d9998d05f620a4c_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working...'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/vembrane%252Fvembrane-benchmark%2Fjhpzb092%2Flogs.tar.xz">0.804GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/vembrane%252Fvembrane-benchmark%2Fjhpzb092%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>10.387GiB</td>
						<td>1994.0 seconds</td>
						<td>2067.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743189744">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/3.0.1">3.0.1</a>
						</td>
						<td>nextflow</td>
						<td>185 days</td>
						<td>2023-02-19 03:39:02</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2F56svhbco%2Flogs.tar.xz">0.126GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2F56svhbco%2Foutput.tar.xz">0.025GiB</a>
						</td>
						<td>2.354GiB</td>
						<td>253.7 seconds</td>
						<td>160.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738213568">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.4.0">v1.4.0</a>
						</td>
						<td>snakemake</td>
						<td>178 days</td>
						<td>2023-02-19 06:16:08</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2F7rjvr5jv%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2F7rjvr5jv%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.594GiB</td>
						<td>1329.3 seconds</td>
						<td>1528.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738208528">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.4.0">v1.4.0</a>
						</td>
						<td>snakemake</td>
						<td>178 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/xyikzvj56d/code/workflow/rules/../envs/stats.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/xyikzvj56d/code/.test/.snakemake/conda/dc610de179f6a289665a3017f3b1addf_.yaml"
  --prefix "/home/azureuser/tmp/xyikzvj56d/code/.test/.snakemake/conda/dc610de179f6a289665a3017f3b1addf_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working... done


  [Errno 2] No such file or directory: ''/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/altair-4.1.0-py_1.tar.bz2''

  [Errno 2] No such file or directory: ''/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/nodejs-18.12.1-h8839609_0.tar.bz2'''

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fji6cel59%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fji6cel59%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.833GiB</td>
						<td>263.8 seconds</td>
						<td>308.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743118384">
								<a href="https://nf-co.re/mag">mag</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mag/tree/2.2.1">2.2.1</a>
						</td>
						<td>nextflow</td>
						<td>178 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fhob40wct%2Flogs.tar.xz">0.462GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mag%2Fhob40wct%2Foutput.tar.xz">0.032GiB</a>
						</td>
						<td>1.181GiB</td>
						<td>2443.6 seconds</td>
						<td>1700.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738213328">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.4.1">v1.4.1</a>
						</td>
						<td>snakemake</td>
						<td>178 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fv1aogrsb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fv1aogrsb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.833GiB</td>
						<td>1349.5 seconds</td>
						<td>2193.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738208288">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.4.1">v1.4.1</a>
						</td>
						<td>snakemake</td>
						<td>178 days</td>
						<td>2023-02-19 03:30:59</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/re3ndc1e13/singularity-tmp/tmpddj1n9b8.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/re3ndc1e13/code/.test/.snakemake/conda/404e7c919d55600da68da045e551e1a9_.yaml"
  --prefix "/home/azureuser/tmp/re3ndc1e13/code/.test/.snakemake/conda/404e7c919d55600da68da045e551e1a9_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working... done


  [Errno 2] No such file or directory: ''/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/htslib-1.14-h9753748_2.tar.bz2'''

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F6cd0fmri%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F6cd0fmri%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.922GiB</td>
						<td>227.3 seconds</td>
						<td>240.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738295248">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.16.0">v3.16.0</a>
						</td>
						<td>snakemake</td>
						<td>171 days</td>
						<td>2023-02-19 04:40:32</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- f9yuw5l39a
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/f9yuw5l39a/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fibxmvden%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fibxmvden%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.3 seconds</td>
						<td>1.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738268240">
								<a href="https://github.com/GooglingTheCancerGenome/sv-callers">GooglingTheCancerGenome/sv-callers</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.2.1">v1.2.1</a>
						</td>
						<td>snakemake</td>
						<td>166 days</td>
						<td>2023-02-19 04:18:47</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: ModuleNotFoundError
line_no: 7
rest: "No module named 'yatiml'\n  File \"/home/azureuser/tmp/lp153dey0x/code/workflow/Snakefile\"\
  , line 7, in <module>\n  File \"/home/azureuser/tmp/lp153dey0x/code/workflow/helper_functions.py\"\
  , line 5, in <module>\n  File \"/home/azureuser/tmp/lp153dey0x/code/workflow/validator/__init__.py\"\
  , line 8, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2Foe6ppsmz%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2Foe6ppsmz%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.061GiB</td>
						<td>1.3 seconds</td>
						<td>1.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746200704">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/2.4.0">2.4.0</a>
						</td>
						<td>nextflow</td>
						<td>165 days</td>
						<td>2023-02-19 04:12:00</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Ftd8rhdoh%2Flogs.tar.xz">0.058GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Ftd8rhdoh%2Foutput.tar.xz">0.009GiB</a>
						</td>
						<td>4.803GiB</td>
						<td>3016.7 seconds</td>
						<td>1187.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738255280">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.7.0">1.7.0</a>
						</td>
						<td>snakemake</td>
						<td>164 days</td>
						<td>2023-02-20 05:13:11</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- input_check.smk
kind: CalledProcessError
line_no: 26
rest: "Command ' singularity  exec --home /home/azureuser/tmp/w8poc62f4r/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/w8poc62f4r/code/.snakemake/singularity/14400fdfd76f75691dcfa360e6e40331.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/2a0dec58a5a836a46ab65c71fc2ee5f9'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/w8poc62f4r/code/.snakemake/scripts/tmpsblkg4jz.generate_exclude_file.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/w8poc62f4r/code/workflow/rules/input_check.smk\"\
  , line 26, in __rule_generate_exclude_file_for_mosaic_count\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 05:21:35 2023]\nFinished job 8.\n1 of 25 steps (4%)\
  \ done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T051312.129113.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fqlfhstod%2Flogs.tar.xz">3.358GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fqlfhstod%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.091GiB</td>
						<td>969.7 seconds</td>
						<td>504.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738219808">
								<a href="https://github.com/epigen/unsupervised_analysis">epigen/unsupervised_analysis</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/epigen/unsupervised_analysis/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>163 days</td>
						<td>2023-02-19 04:30:33</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: FileNotFoundError
line_no: 23
rest: "[Errno 2] No such file or directory: '/research/home/sreichl/projects/macroIC/config/AKsmall_unsupervised_analysis_annotation.csv'\n\
  \  File \"/home/azureuser/tmp/a0xfrl5ho6/code/workflow/Snakefile\", line 23, in\
  \ <module>\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 211, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 331, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 950, in read_csv\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 605, in _read\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1442, in __init__\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1735, in _make_engine\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/common.py\"\
  , line 856, in get_handle"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Funsupervised-analysis%2Fzc7bouqu%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Funsupervised-analysis%2Fzc7bouqu%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.098GiB</td>
						<td>1.2 seconds</td>
						<td>1.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738169456">
								<a href="https://github.com/CVUA-RRW/FooDMe">CVUA-RRW/FooDMe</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/CVUA-RRW/FooDMe/tree/1.6.3">1.6.3</a>
						</td>
						<td>snakemake</td>
						<td>159 days</td>
						<td>2023-02-19 04:42:15</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fq8zyrlwz%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/CVUA-RRW%252FFooDMe%2Fq8zyrlwz%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.036GiB</td>
						<td>734.6 seconds</td>
						<td>760.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739876144">
								<a href="https://github.com/snakemake-workflows/cyrcular-calling">snakemake-workflows/cyrcular-calling</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/cyrcular-calling/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>158 days</td>
						<td>2023-02-19 06:26:11</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: '

  Failed to open source file /home/azureuser/tmp/k18gvqyyul/code/workflow/rules/../envs/pigz.yaml

  FileNotFoundError: [Errno 2] No such file or directory: ''/home/azureuser/tmp/k18gvqyyul/code/workflow/rules/../envs/pigz.yaml''

  '

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fcyrcular-calling%2Fb0wtf025%2Flogs.tar.xz">0.121GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fcyrcular-calling%2Fb0wtf025%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.955GiB</td>
						<td>226.4 seconds</td>
						<td>234.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010763176784">
								<a href="https://nf-co.re/airrflow">airrflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/airrflow/tree/2.3.0">2.3.0</a>
						</td>
						<td>nextflow</td>
						<td>157 days</td>
						<td>2023-02-20 04:31:43</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 07:31:43.069
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Fstacghwv%2Flogs.tar.xz">0.021GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Fstacghwv%2Foutput.tar.xz">0.018GiB</a>
						</td>
						<td>0.760GiB</td>
						<td>157.2 seconds</td>
						<td>10800.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738265840">
								<a href="https://github.com/sneuensc/mapache">sneuensc/mapache</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/sneuensc/mapache/tree/v0.2.0">v0.2.0</a>
						</td>
						<td>snakemake</td>
						<td>149 days</td>
						<td>2023-02-19 04:29:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- scripts
- utils.py
kind: IndexError
line_no: 665
rest: "list index out of range\n  File \"/home/azureuser/tmp/rjpmx8d3pr/code/workflow/Snakefile\"\
  , line 179, in <module>\n  File \"/home/azureuser/tmp/rjpmx8d3pr/code/workflow/scripts/utils.py\"\
  , line 665, in set_sex_inference"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sneuensc%252Fmapache%2Fv5t44vam%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sneuensc%252Fmapache%2Fv5t44vam%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.104GiB</td>
						<td>1.4 seconds</td>
						<td>1.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738295008">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.17.0">v3.17.0</a>
						</td>
						<td>snakemake</td>
						<td>148 days</td>
						<td>2023-02-19 06:00:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- uy9qaig08n
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/uy9qaig08n/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fte6lmrzl%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fte6lmrzl%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.4 seconds</td>
						<td>2.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743189504">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/3.0.2">3.0.2</a>
						</td>
						<td>nextflow</td>
						<td>146 days</td>
						<td>2023-02-19 06:58:55</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2F3kgjj4df%2Flogs.tar.xz">0.126GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2F3kgjj4df%2Foutput.tar.xz">0.025GiB</a>
						</td>
						<td>2.528GiB</td>
						<td>266.9 seconds</td>
						<td>249.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738294768">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.17.1">v3.17.1</a>
						</td>
						<td>snakemake</td>
						<td>145 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- igrr6u2eob
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/igrr6u2eob/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F4j95m35v%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F4j95m35v%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.2 seconds</td>
						<td>1.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743117664">
								<a href="https://nf-co.re/isoseq">isoseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/isoseq/tree/1.1.1">1.1.1</a>
						</td>
						<td>nextflow</td>
						<td>145 days</td>
						<td>2023-02-19 04:00:05</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/isoseq%2Flh1qnbdg%2Flogs.tar.xz">0.088GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/isoseq%2Flh1qnbdg%2Foutput.tar.xz">0.030GiB</a>
						</td>
						<td>2.279GiB</td>
						<td>911.8 seconds</td>
						<td>765.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738294528">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.0">v3.18.0</a>
						</td>
						<td>snakemake</td>
						<td>144 days</td>
						<td>2023-02-19 04:22:59</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- skj9l6mivs
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/skj9l6mivs/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F9jalpp32%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F9jalpp32%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.4 seconds</td>
						<td>1.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738294288">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.1">v3.18.1</a>
						</td>
						<td>snakemake</td>
						<td>144 days</td>
						<td>2023-02-19 04:22:33</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- q8funcq397
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/q8funcq397/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fzym18p86%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fzym18p86%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.5 seconds</td>
						<td>1.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743151152">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.9">3.9</a>
						</td>
						<td>nextflow</td>
						<td>141 days</td>
						<td>2023-02-19 04:37:15</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F4b871meg%2Flogs.tar.xz">0.206GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F4b871meg%2Foutput.tar.xz">0.031GiB</a>
						</td>
						<td>2.165GiB</td>
						<td>1530.8 seconds</td>
						<td>882.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744202528">
								<a href="https://nf-co.re/chipseq">chipseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/chipseq/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>139 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2F412wkhd4%2Flogs.tar.xz">0.507GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/chipseq%2F412wkhd4%2Foutput.tar.xz">0.085GiB</a>
						</td>
						<td>2.480GiB</td>
						<td>2645.2 seconds</td>
						<td>1548.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738165376">
								<a href="https://github.com/snakemake-workflows/dna-seq-mtb">snakemake-workflows/dna-seq-mtb</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.3.0">v1.3.0</a>
						</td>
						<td>snakemake</td>
						<td>137 days</td>
						<td>2023-02-19 06:46:14</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- sjp34tppf7
- code
- workflow
- rules
- common.smk
kind: WorkflowError
line_no: 20
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/sjp34tppf7/code/workflow/Snakefile\", line 1, in <module>\n\
  \  File \"/home/azureuser/tmp/sjp34tppf7/code/workflow/rules/common.smk\", line\
  \ 20, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fknrxp6r9%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fknrxp6r9%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.4 seconds</td>
						<td>1.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743192144">
								<a href="https://nf-co.re/scrnaseq">scrnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/scrnaseq/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>136 days</td>
						<td>2023-02-19 04:11:41</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/scrnaseq%2Ffnxuq9xo%2Flogs.tar.xz">0.525GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/scrnaseq%2Ffnxuq9xo%2Foutput.tar.xz">0.498GiB</a>
						</td>
						<td>2.695GiB</td>
						<td>1605.5 seconds</td>
						<td>991.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743051408">
								<a href="https://nf-co.re/demultiplex">demultiplex</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/demultiplex/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>136 days</td>
						<td>2023-02-19 03:59:23</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/demultiplex%2Fnqins27b%2Flogs.tar.xz">0.182GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/demultiplex%2Fnqins27b%2Foutput.tar.xz">0.035GiB</a>
						</td>
						<td>2.560GiB</td>
						<td>119.7 seconds</td>
						<td>86.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738219568">
								<a href="https://github.com/epigen/unsupervised_analysis">epigen/unsupervised_analysis</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/epigen/unsupervised_analysis/tree/v0.2.0">v0.2.0</a>
						</td>
						<td>snakemake</td>
						<td>130 days</td>
						<td>2023-02-19 15:33:26</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeInternalError
kind: SnakemakeInternalError
rest: "File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/conda\"\
  , line 12, in <module>\n    from conda.cli import main\nModuleNotFoundError: No\
  \ module named 'conda'\n[Sun Feb 19 16:16:45 2023]\nError in rule env_export:\n\
  \    jobid: 16\n    output: .test/results/envs/unsupervised_analysis/sklearn.yaml\n\
  \    log: logs/rules/env_sklearn.log (check log file(s) for error message)\n   \
  \ conda-env: /home/azureuser/tmp/c1jg2cv7n3/code/.snakemake/conda/16e1cb28a6444ccb3929f923ce3cdfa3_\n\
  \    shell:\n        \n        conda env export > .test/results/envs/unsupervised_analysis/sklearn.yaml\n\
  \        \n        (one of the commands exited with non-zero exit code; note that\
  \ snakemake uses bash strict mode!)\n\nRemoving output files of failed job env_export\
  \ since they might be corrupted:\n.test/results/envs/unsupervised_analysis/sklearn.yaml\n\
  Shutting down, this might take some time.\nExiting because a job execution failed.\
  \ Look above for error message\nComplete log: .snakemake/log/2023-02-19T153327.397978.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Funsupervised-analysis%2Fmne28ili%2Flogs.tar.xz">0.622GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Funsupervised-analysis%2Fmne28ili%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>6.382GiB</td>
						<td>2370.1 seconds</td>
						<td>2599.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743115024">
								<a href="https://nf-co.re/hlatyping">hlatyping</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hlatyping/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>124 days</td>
						<td>2023-02-19 06:00:26</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fagys3jdk%2Flogs.tar.xz">0.033GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hlatyping%2Fagys3jdk%2Foutput.tar.xz">0.023GiB</a>
						</td>
						<td>1.637GiB</td>
						<td>114.8 seconds</td>
						<td>167.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738168256">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.0">1.3.0</a>
						</td>
						<td>snakemake</td>
						<td>122 days</td>
						<td>2023-02-20 04:33:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- rules.smk
kind: CalledProcessError
line_no: 32
rest: "Command ' singularity  exec --home /home/azureuser/tmp/93i2z8eygm/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/93i2z8eygm/code/.snakemake/singularity/e27977755b4115e9e3788cddd3c7d374.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/08d4368302a4bdf7eda6b536495efe7d'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/93i2z8eygm/code/.snakemake/scripts/tmprdtcjdm_.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/93i2z8eygm/code/workflow/rules/rules.smk\"\
  , line 32, in __rule_fastqc\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nActivating singularity image /home/azureuser/tmp/93i2z8eygm/code/.snakemake/singularity/e27977755b4115e9e3788cddd3c7d374.simg\n\
  Activating conda environment: ../../../../../conda-envs/08d4368302a4bdf7eda6b536495efe7d\n\
  [Mon Feb 20 04:42:36 2023]\nError in rule fastqc:\n    jobid: 90\n    input: .tests/data_CHR17/RPE-BM510/fastq/BM510x04_PE20305.2.fastq.gz\n\
  \    output: .tests/data_CHR17/RPE-BM510/fastqc/BM510x04_PE20305_2_fastqc.html,\
  \ .tests/data_CHR17/RPE-BM510/fastqc/BM510x04_PE20305_2_fastqc.zip\n    log: .tests/data_CHR17/log/fastqc/RPE-BM510/BM510x04_PE20305_2.log\
  \ (check log file(s) for error message)\n    conda-env: /conda-envs/08d4368302a4bdf7eda6b536495efe7d\n\
  \nRuleException:\nCalledProcessError in line 32 of /home/azureuser/tmp/93i2z8eygm/code/workflow/rules/rules.smk:\n\
  Command ' singularity  exec --home /home/azureuser/tmp/93i2z8eygm/code  --bind /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/93i2z8eygm/code/.snakemake/singularity/e27977755b4115e9e3788cddd3c7d374.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/08d4368302a4bdf7eda6b536495efe7d'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/93i2z8eygm/code/.snakemake/scripts/tmpzejzwnmi.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/93i2z8eygm/code/workflow/rules/rules.smk\"\
  , line 32, in __rule_fastqc\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T043315.276389.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fwgs1jbuy%2Flogs.tar.xz">2.739GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fwgs1jbuy%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.924GiB</td>
						<td>835.5 seconds</td>
						<td>567.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738168016">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.1">1.3.1</a>
						</td>
						<td>snakemake</td>
						<td>122 days</td>
						<td>2023-02-20 05:06:01</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- rules.smk
kind: CalledProcessError
line_no: 32
rest: "Command ' singularity  exec --home /home/azureuser/tmp/lv9tnah0xi/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/lv9tnah0xi/code/.snakemake/singularity/f193cea8f006dca7f291c0844aac462f.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/08d4368302a4bdf7eda6b536495efe7d'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/lv9tnah0xi/code/.snakemake/scripts/tmpl1gvhlka.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/lv9tnah0xi/code/workflow/rules/rules.smk\"\
  , line 32, in __rule_fastqc\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 05:09:40 2023]\nError in rule fastqc:\n    jobid:\
  \ 89\n    input: .tests/data_CHR17/RPE-BM510/fastq/BM510x04_PE20305.1.fastq.gz\n\
  \    output: .tests/data_CHR17/RPE-BM510/fastqc/BM510x04_PE20305_1_fastqc.html,\
  \ .tests/data_CHR17/RPE-BM510/fastqc/BM510x04_PE20305_1_fastqc.zip\n    log: .tests/data_CHR17/log/fastqc/RPE-BM510/BM510x04_PE20305_1.log\
  \ (check log file(s) for error message)\n    conda-env: /conda-envs/08d4368302a4bdf7eda6b536495efe7d\n\
  \nRuleException:\nCalledProcessError in line 32 of /home/azureuser/tmp/lv9tnah0xi/code/workflow/rules/rules.smk:\n\
  Command ' singularity  exec --home /home/azureuser/tmp/lv9tnah0xi/code  --bind /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/lv9tnah0xi/code/.snakemake/singularity/f193cea8f006dca7f291c0844aac462f.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/08d4368302a4bdf7eda6b536495efe7d'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/lv9tnah0xi/code/.snakemake/scripts/tmpw7h4atui.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/lv9tnah0xi/code/workflow/rules/rules.smk\"\
  , line 32, in __rule_fastqc\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T050601.738260.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fcsa9229b%2Flogs.tar.xz">0.960GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fcsa9229b%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.591GiB</td>
						<td>460.1 seconds</td>
						<td>219.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738255040">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.8.1">1.8.1</a>
						</td>
						<td>snakemake</td>
						<td>120 days</td>
						<td>2023-02-20 04:32:58</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- count.smk
kind: CalledProcessError
line_no: 20
rest: "Command ' singularity  exec --home /home/azureuser/tmp/s4lgu8trmq/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/s4lgu8trmq/code/.snakemake/singularity/59116dfc3cab1dcf2fab519548b65498.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/2a0dec58a5a836a46ab65c71fc2ee5f9'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/s4lgu8trmq/code/.snakemake/scripts/tmpm2hq3mrv.generate_exclude_file.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/s4lgu8trmq/code/workflow/rules/count.smk\"\
  , line 20, in __rule_generate_exclude_file_for_mosaic_count\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 04:43:29 2023]\nFinished job 8.\n1 of 25 steps (4%)\
  \ done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T043259.797940.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fcqqmbgsm%2Flogs.tar.xz">2.680GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fcqqmbgsm%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.959GiB</td>
						<td>849.1 seconds</td>
						<td>632.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743112864">
								<a href="https://nf-co.re/hgtseq">hgtseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hgtseq/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>118 days</td>
						<td>2023-02-19 03:33:03</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hgtseq%2Fjr5isltb%2Flogs.tar.xz">0.060GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hgtseq%2Fjr5isltb%2Foutput.tar.xz">0.011GiB</a>
						</td>
						<td>1.419GiB</td>
						<td>420.6 seconds</td>
						<td>689.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743193344">
								<a href="https://nf-co.re/smrnaseq">smrnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/smrnaseq/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>118 days</td>
						<td>2023-02-19 12:51:48</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/smrnaseq%2Fddujwqug%2Flogs.tar.xz">5.341GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/smrnaseq%2Fddujwqug%2Foutput.tar.xz">2.135GiB</a>
						</td>
						<td>1.377GiB</td>
						<td>4143.5 seconds</td>
						<td>2171.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743147072">
								<a href="https://nf-co.re/nascent">nascent</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/nascent/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>118 days</td>
						<td>2023-02-19 04:19:18</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nascent%2F5eun4hab%2Flogs.tar.xz">0.148GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nascent%2F5eun4hab%2Foutput.tar.xz">0.083GiB</a>
						</td>
						<td>1.917GiB</td>
						<td>775.5 seconds</td>
						<td>531.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738294048">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.2">v3.18.2</a>
						</td>
						<td>snakemake</td>
						<td>117 days</td>
						<td>2023-02-19 06:14:50</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- mrnx15yneb
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/mrnx15yneb/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fduv0ptla%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fduv0ptla%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.4 seconds</td>
						<td>3.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738213088">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.5.0">v1.5.0</a>
						</td>
						<td>snakemake</td>
						<td>115 days</td>
						<td>2023-02-19 06:40:19</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fl9ea4dbp%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fl9ea4dbp%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.833GiB</td>
						<td>1279.8 seconds</td>
						<td>1381.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738208048">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.5.0">v1.5.0</a>
						</td>
						<td>snakemake</td>
						<td>115 days</td>
						<td>2023-02-19 06:00:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeCondaError
kind: CreateCondaEnvironmentException
rest: 'Could not create conda environment from /home/azureuser/tmp/hme88ta3z3/singularity-tmp/tmp_mxum5tz.yaml:

  Command:

  conda env create --quiet --file "/home/azureuser/tmp/hme88ta3z3/code/.test/.snakemake/conda/26458f0826946275be522f243061bf4e_.yaml"
  --prefix "/home/azureuser/tmp/hme88ta3z3/code/.test/.snakemake/conda/26458f0826946275be522f243061bf4e_"

  Output:

  Collecting package metadata (repodata.json): ...working... done

  Solving environment: ...working... done


  [Errno 2] No such file or directory: ''/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/pkgs/openjdk-17.0.3-h58dac75_5.conda'''

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Foaoxoi6s%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Foaoxoi6s%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.829GiB</td>
						<td>157.9 seconds</td>
						<td>196.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743049968">
								<a href="https://nf-co.re/cutandrun">cutandrun</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/cutandrun/tree/3.0">3.0</a>
						</td>
						<td>nextflow</td>
						<td>115 days</td>
						<td>2023-02-19 04:31:06</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cutandrun%2F7k7eta1a%2Flogs.tar.xz">0.208GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/cutandrun%2F7k7eta1a%2Foutput.tar.xz">0.020GiB</a>
						</td>
						<td>1.809GiB</td>
						<td>435.3 seconds</td>
						<td>301.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738254800">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.8.2">1.8.2</a>
						</td>
						<td>snakemake</td>
						<td>115 days</td>
						<td>2023-02-20 04:51:59</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- count.smk
kind: CalledProcessError
line_no: 20
rest: "Command ' singularity  exec --home /home/azureuser/tmp/ct2tur3sui/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/ct2tur3sui/code/.snakemake/singularity/6ecacc57daea3a1fadb91ae90619160a.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/02d721d968887d6d80a06bbd7ca09642'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/ct2tur3sui/code/.snakemake/scripts/tmpp83aghgx.generate_exclude_file.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/ct2tur3sui/code/workflow/rules/count.smk\"\
  , line 20, in __rule_generate_exclude_file_for_mosaic_count\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 04:58:57 2023]\nFinished job 8.\n1 of 25 steps (4%)\
  \ done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T045200.017900.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fv3hbe7v6%2Flogs.tar.xz">2.741GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fv3hbe7v6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.083GiB</td>
						<td>790.0 seconds</td>
						<td>419.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010742194928">
								<a href="https://github.com/crs4/fair-crcc-get-data">crs4/fair-crcc-get-data</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/crs4/fair-crcc-get-data/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>114 days</td>
						<td>2023-02-19 06:18:03</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- common.smk
kind: WorkflowError
line_no: 13
rest: "Error validating config file.\nValidationError: 'recipient_key' is a required\
  \ property\n\nFailed validating 'required' in schema:\n    OrderedDict([('$schema',\n\
  \                  'https://json-schema.org/draft/2020-12/schema'),\n          \
  \       ('description',\n                  'Configuration schema for get-data workflow.'),\n\
  \                 ('properties',\n                  OrderedDict([('recipient_key',\n\
  \                                OrderedDict([('type', 'string'),\n            \
  \                                 ('description',\n                            \
  \                  \"Path to recipient's c4gh \"\n                             \
  \                 'private key.  Must be '\n                                   \
  \           'within working directory '\n                                      \
  \        'tree.\\n')])),\n                               ('source',\n          \
  \                      OrderedDict([('type', 'object'),\n                      \
  \                       ('description',\n                                      \
  \        'Source of encrypted data.'),\n                                       \
  \      ('required',\n                                              ['type', 'connection']),\n\
  \                                             ('properties',\n                 \
  \                             OrderedDict([('type',\n                          \
  \                                  OrderedDict([('description',\n              \
  \                                                            'Any '\n          \
  \                                                                'of '\n       \
  \                                                                   'the '\n   \
  \                                                                       '[remote\
  \ '\n                                                                          'providers\
  \ '\n                                                                          'supported\
  \ '\n                                                                          'by\
  \ '\n                                                                          'snakemake](https://snakemake.readthedocs.io/en/v6.13.1/snakefiles/remote_files.html#remote-files).\\\
  n'),\n                                                                         ('allOf',\n\
  \                                                                          [OrderedDict([('$ref',\n\
  \                                                                              \
  \           '#/$defs/connection_type')])])])),\n                               \
  \                            ('root_path',\n                                   \
  \                         OrderedDict([('allOf',\n                             \
  \                                             [OrderedDict([('$ref',\n         \
  \                                                                              \
  \  '#/$defs/root_path')]),\n                                                   \
  \                        OrderedDict([('description',\n                        \
  \                                                                 'Prefix '\n  \
  \                                                                              \
  \         'path '\n                                                            \
  \                             'for '\n                                         \
  \                                                'source. '\n                  \
  \                                                                       'The '\n\
  \                                                                              \
  \           'workflow '\n                                                      \
  \                                   'will '\n                                  \
  \                                                       'look '\n              \
  \                                                                           'for\
  \ '\n                                                                          \
  \               'files '\n                                                     \
  \                                    'under '\n                                \
  \                                                         'this '\n            \
  \                                                                             'path.\\\
  n')])])])),\n                                                           ('connection',\n\
  \                                                            OrderedDict([('$ref',\n\
  \                                                                          '#/$defs/connection')]))]))])),\n\
  \                               ('destination',\n                              \
  \  OrderedDict([('type', 'object'),\n                                          \
  \   ('description',\n                                              'Destination\
  \ for decrypted '\n                                              'data.\\n'),\n\
  \                                             ('required', ['type']),\n        \
  \                                     ('properties',\n                         \
  \                     OrderedDict([('type',\n                                  \
  \                          OrderedDict([('oneOf',\n                            \
  \                                              [OrderedDict([('type',\n        \
  \                                                                              \
  \   'string'),\n                                                               \
  \                         ('enum',\n                                           \
  \                                              ['local'])]),\n                 \
  \                                                          OrderedDict([('$ref',\n\
  \                                                                              \
  \           '#/$defs/connection_type')])]),\n                                  \
  \                                       ('default',\n                          \
  \                                                'local'),\n                   \
  \                                                      ('description',\n       \
  \                                                                   'Any '\n   \
  \                                                                       'of '\n\
  \                                                                          'the\
  \ '\n                                                                          '[remote\
  \ '\n                                                                          'providers\
  \ '\n                                                                          'supported\
  \ '\n                                                                          'by\
  \ '\n                                                                          'snakemake](https://snakemake.readthedocs.io/en/v6.13.1/snakefiles/remote_files.html#remote-files)\
  \ '\n                                                                          'plus\
  \ '\n                                                                          '\"\
  local\". '\n                                                                   \
  \       'Default '\n                                                           \
  \               'value: '\n                                                    \
  \                      '\"local\"\\n')])),\n                                   \
  \                        ('root_path',\n                                       \
  \                     OrderedDict([('allOf',\n                                 \
  \                                         [OrderedDict([('$ref',\n             \
  \                                                                            '#/$defs/root_path')]),\n\
  \                                                                           OrderedDict([('description',\n\
  \                                                                              \
  \           'Prefix '\n                                                        \
  \                                 'path '\n                                    \
  \                                                     'for '\n                 \
  \                                                                        'destination.\
  \  '\n                                                                         \
  \                'The '\n                                                      \
  \                                   'workflow '\n                              \
  \                                                           'will '\n          \
  \                                                                              \
  \ 're-create '\n                                                               \
  \                          'the '\n                                            \
  \                                             'original '\n                    \
  \                                                                     'source '\n\
  \                                                                              \
  \           'directory '\n                                                     \
  \                                    'structure '\n                            \
  \                                                             'at '\n          \
  \                                                                              \
  \ 'the '\n                                                                     \
  \                    'destination '\n                                          \
  \                                               'storage '\n                   \
  \                                                                      'and '\n\
  \                                                                              \
  \           'under '\n                                                         \
  \                                'this '\n                                     \
  \                                                    'path.\\n')])])])),\n     \
  \                                                      ('connection',\n        \
  \                                                    OrderedDict([('$ref',\n   \
  \                                                                       '#/$defs/connection')]))]))])),\n\
  \                               ('filters',\n                                OrderedDict([('type',\
  \ 'array'),\n                                             ('items',\n          \
  \                                    OrderedDict([('$ref',\n                   \
  \                                         '#/$defs/rule')])),\n                \
  \                             ('description',\n                                \
  \              'Optional list of of filters '\n                                \
  \              'to be applied to '\n                                           \
  \   'destination file names. '\n                                              'These\
  \ are the original '\n                                              'names of the\
  \ files and the '\n                                              'names with which\
  \ they will '\n                                              'be saved in the specified\
  \ '\n                                              'destination.  The filters '\n\
  \                                              'follow a logic similar to '\n  \
  \                                            'rsync (for those who are '\n     \
  \                                         'familiar with that tool).\\n'\n     \
  \                                         'Each filter specifies a '\n         \
  \                                     'glob expression for '\n                 \
  \                             'matching files and an '\n                       \
  \                       '\"include\" or \"exclude\" '\n                        \
  \                      'action that they apply, if '\n                         \
  \                     'they match a file.  The '\n                             \
  \                 'filters are tried in the '\n                                \
  \              'order they are specified: '\n                                  \
  \            'the first one that matches '\n                                   \
  \           'is applied; if no filters '\n                                     \
  \         'match a file name, the file '\n                                     \
  \         'is included.\\n')]))])),\n                 ('required', ['recipient_key',\
  \ 'source', 'destination']),\n                 ('examples',\n                  OrderedDict([('recipient_key',\
  \ './recipient_key'),\n                               ('repository',\n         \
  \                       OrderedDict([('path',\n                                \
  \              '/mnt/rbd/data/sftp/fair-crcc/'),\n                             \
  \                ('private_key', 'bbmri-key'),\n                               \
  \              ('public_key',\n                                              'bbmri-key.pub')])),\n\
  \                               ('sources',\n                                OrderedDict([('glob_extension',\n\
  \                                              '.tiff.c4gh'),\n                \
  \                             ('items',\n                                      \
  \        ['some/directory/to/glob',\n                                          \
  \     'another/individual/file.tiff.c4gh'])])),\n                              \
  \ ('destination',\n                                OrderedDict([('type', 'S3'),\n\
  \                                             ('root_path',\n                  \
  \                            'bucket/subpath/'),\n                             \
  \                ('connection',\n                                              OrderedDict([('access_key_id',\n\
  \                                                            'MYACCESSKEY'),\n \
  \                                                          ('secret_access_key',\n\
  \                                                            'MYSECRET'),\n    \
  \                                                       ('host',\n             \
  \                                               'http://localhost:9000'),\n    \
  \                                                       ('verify',\n           \
  \                                                 False)]))])),\n              \
  \                 ('filters',\n                                [OrderedDict([('action',\
  \ 'exclude'),\n                                              ('pattern',\n     \
  \                                          'test-file_2.txt')])])])),\n        \
  \         ('$defs',\n                  OrderedDict([('rule',\n                 \
  \               OrderedDict([('type', 'object'),\n                             \
  \                ('properties',\n                                              OrderedDict([('pattern',\n\
  \                                                            OrderedDict([('type',\n\
  \                                                                          'string')])),\n\
  \                                                           ('action',\n       \
  \                                                     OrderedDict([('type',\n  \
  \                                                                        'string'),\n\
  \                                                                         ('enum',\n\
  \                                                                          ['include',\n\
  \                                                                           'exclude'])]))])),\n\
  \                                             ('required',\n                   \
  \                           ['action', 'pattern'])])),\n                       \
  \        ('connection_type',\n                                OrderedDict([('type',\
  \ 'string'),\n                                             ('enum',\n          \
  \                                    ['AzBlob',\n                              \
  \                 'dropbox',\n                                               'EGA',\n\
  \                                               'FTP',\n                       \
  \                        'gfal',\n                                             \
  \  'gridftp',\n                                               'GS',\n          \
  \                                     'HTTP',\n                                \
  \               'iRODS',\n                                               'NCBI',\n\
  \                                               'S3',\n                        \
  \                       'SFTP',\n                                              \
  \ 'webdav',\n                                               'XRootD'])])),\n   \
  \                            ('connection',\n                                OrderedDict([('type',\
  \ 'object'),\n                                             ('description',\n   \
  \                                           'Connection parameters for '\n     \
  \                                         '[Snakemake remote '\n               \
  \                               'provider](https://snakemake.readthedocs.io/en/v6.13.1/snakefiles/remote_files.html#remote-files).\
  \ '\n                                              'Keys required/supported '\n\
  \                                              'vary depending on the '\n      \
  \                                        'provider type.  See the '\n          \
  \                                    'snakemake documentation for '\n          \
  \                                    'details.\\n'),\n                         \
  \                    ('example',\n                                             \
  \ OrderedDict([('access_key_id',\n                                             \
  \               'MYACCESSKEY'),\n                                              \
  \             ('secret_access_key',\n                                          \
  \                  'MYSECRET'),\n                                              \
  \             ('host',\n                                                       \
  \     'http://localhost:9000'),\n                                              \
  \             ('verify',\n                                                     \
  \       False)]))])),\n                               ('root_path',\n          \
  \                      OrderedDict([('type', 'string'),\n                      \
  \                       ('default', '/')]))]))])\n\nOn instance:\n    {}\n  File\
  \ \"/home/azureuser/tmp/8uldmlzdyz/code/workflow/Snakefile\", line 17, in <module>\n\
  \  File \"/home/azureuser/tmp/8uldmlzdyz/code/workflow/rules/common.smk\", line\
  \ 13, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/crs4%252Ffair-crcc-get-data%2Fc8xr8yg2%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/crs4%252Ffair-crcc-get-data%2Fc8xr8yg2%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.061GiB</td>
						<td>1.5 seconds</td>
						<td>2.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739693216">
								<a href="https://github.com/aryarm/admixtools">aryarm/admixtools</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/aryarm/admixtools/tree/v0.0.1">v0.0.1</a>
						</td>
						<td>snakemake</td>
						<td>109 days</td>
						<td>2023-02-19 03:32:38</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2Fmbyjvuuc%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2Fmbyjvuuc%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738167776">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.2">1.3.2</a>
						</td>
						<td>snakemake</td>
						<td>108 days</td>
						<td>2023-02-20 04:31:49</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- rules.smk
kind: CalledProcessError
line_no: 51
rest: "Command ' singularity  exec --home /home/azureuser/tmp/iiipo7ge5i/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/iiipo7ge5i/code/.snakemake/singularity/40c7499b3d8bfab33b3f72f6dc8c4317.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/08d4368302a4bdf7eda6b536495efe7d'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/iiipo7ge5i/code/.snakemake/scripts/tmp52b3h5c4.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/iiipo7ge5i/code/workflow/rules/rules.smk\"\
  , line 51, in __rule_fastqc\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 04:36:54 2023]\nError in rule fastqc:\n    jobid:\
  \ 9\n    input: .tests/data_CHR17/RPE-BM510/fastq/BM510x04_PE20305.1.fastq.gz\n\
  \    output: .tests/data_CHR17/RPE-BM510/fastqc/BM510x04_PE20305_1_fastqc.html,\
  \ .tests/data_CHR17/RPE-BM510/fastqc/BM510x04_PE20305_1_fastqc.zip\n    log: .tests/data_CHR17/log/fastqc/RPE-BM510/BM510x04_PE20305_1.log\
  \ (check log file(s) for error message)\n    conda-env: /conda-envs/08d4368302a4bdf7eda6b536495efe7d\n\
  \nRuleException:\nCalledProcessError in line 51 of /home/azureuser/tmp/iiipo7ge5i/code/workflow/rules/rules.smk:\n\
  Command ' singularity  exec --home /home/azureuser/tmp/iiipo7ge5i/code  --bind /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/iiipo7ge5i/code/.snakemake/singularity/40c7499b3d8bfab33b3f72f6dc8c4317.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/08d4368302a4bdf7eda6b536495efe7d'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/iiipo7ge5i/code/.snakemake/scripts/tmpenpwymwo.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/iiipo7ge5i/code/workflow/rules/rules.smk\"\
  , line 51, in __rule_fastqc\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T043150.448301.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fvelbdp7x%2Flogs.tar.xz">0.953GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fvelbdp7x%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.470GiB</td>
						<td>480.4 seconds</td>
						<td>305.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738167536">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.3">1.3.3</a>
						</td>
						<td>snakemake</td>
						<td>108 days</td>
						<td>2023-02-20 04:31:52</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: '

  Failed to pull singularity image from docker://weber8thomas/ashleys-qc-pipeline:1.3.3:

  FATAL:   While making image from oci registry: error fetching image to cache: failed
  to get checksum for docker://weber8thomas/ashleys-qc-pipeline:1.3.3: reading manifest
  1.3.3 in docker.io/weber8thomas/ashleys-qc-pipeline: manifest unknown: manifest
  unknown


  '

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F216px9bh%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F216px9bh%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.106GiB</td>
						<td>1.7 seconds</td>
						<td>3.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010739788832">
								<a href="https://github.com/aryarm/admixtools">aryarm/admixtools</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/aryarm/admixtools/tree/v0.0.2">v0.0.2</a>
						</td>
						<td>snakemake</td>
						<td>107 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2F69rhoh8f%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2F69rhoh8f%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738219088">
								<a href="https://github.com/akcorut/kGWASflow">akcorut/kGWASflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/akcorut/kGWASflow/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>107 days</td>
						<td>2023-02-19 06:40:06</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- hzvk53rvti
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 13
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/hzvk53rvti/code/workflow/Snakefile\", line 13, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/akcorut%252FkGWASflow%2F50ajdzvl%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/akcorut%252FkGWASflow%2F50ajdzvl%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.4 seconds</td>
						<td>1.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010744879120">
								<a href="https://nf-co.re/coproid">coproid</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/coproid/tree/1.1.1">1.1.1</a>
						</td>
						<td>nextflow</td>
						<td>107 days</td>
						<td>2023-02-19 06:09:49</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/coproid%2F2y79rfz6%2Flogs.tar.xz">0.324GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/coproid%2F2y79rfz6%2Foutput.tar.xz">0.029GiB</a>
						</td>
						<td>3.125GiB</td>
						<td>1093.0 seconds</td>
						<td>827.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738254560">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.8.3">1.8.3</a>
						</td>
						<td>snakemake</td>
						<td>105 days</td>
						<td>2023-02-20 04:48:35</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- count.smk
kind: CalledProcessError
line_no: 20
rest: "Command ' singularity  exec --home /home/azureuser/tmp/zjbwsqmjnj/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/zjbwsqmjnj/code/.snakemake/singularity/ae0d8251a7513a8ac081b20d26a77162.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/02d721d968887d6d80a06bbd7ca09642'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/zjbwsqmjnj/code/.snakemake/scripts/tmpjk88c3qz.generate_exclude_file.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/zjbwsqmjnj/code/workflow/rules/count.smk\"\
  , line 20, in __rule_generate_exclude_file_for_mosaic_count\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 04:56:38 2023]\nFinished job 8.\n1 of 25 steps (4%)\
  \ done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T044835.611066.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fbwr7p4ps%2Flogs.tar.xz">2.973GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fbwr7p4ps%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.110GiB</td>
						<td>918.4 seconds</td>
						<td>483.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738167296">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.4">1.3.4</a>
						</td>
						<td>snakemake</td>
						<td>105 days</td>
						<td>2023-02-20 05:27:41</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- rules.smk
kind: CalledProcessError
line_no: 51
rest: "Command ' singularity  exec --home /home/azureuser/tmp/fg3fynoadd/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/fg3fynoadd/code/.snakemake/singularity/232465a72965a52a973b7d3e654a401e.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/08d4368302a4bdf7eda6b536495efe7d'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/fg3fynoadd/code/.snakemake/scripts/tmphqtdhu93.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/fg3fynoadd/code/workflow/rules/rules.smk\"\
  , line 51, in __rule_fastqc\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 05:31:12 2023]\nError in rule fastqc:\n    jobid:\
  \ 10\n    input: .tests/data_CHR17/RPE-BM510/fastq/BM510x04_PE20305.2.fastq.gz\n\
  \    output: .tests/data_CHR17/RPE-BM510/fastqc/BM510x04_PE20305_2_fastqc.html,\
  \ .tests/data_CHR17/RPE-BM510/fastqc/BM510x04_PE20305_2_fastqc.zip\n    log: .tests/data_CHR17/log/fastqc/RPE-BM510/BM510x04_PE20305_2.log\
  \ (check log file(s) for error message)\n    conda-env: /conda-envs/08d4368302a4bdf7eda6b536495efe7d\n\
  \nRuleException:\nCalledProcessError in line 51 of /home/azureuser/tmp/fg3fynoadd/code/workflow/rules/rules.smk:\n\
  Command ' singularity  exec --home /home/azureuser/tmp/fg3fynoadd/code  --bind /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/fg3fynoadd/code/.snakemake/singularity/232465a72965a52a973b7d3e654a401e.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/08d4368302a4bdf7eda6b536495efe7d'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/fg3fynoadd/code/.snakemake/scripts/tmpxhlwft4r.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/fg3fynoadd/code/workflow/rules/rules.smk\"\
  , line 51, in __rule_fastqc\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T052742.346470.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F3a599laj%2Flogs.tar.xz">0.994GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F3a599laj%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.621GiB</td>
						<td>486.5 seconds</td>
						<td>210.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738254320">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.8.4">1.8.4</a>
						</td>
						<td>snakemake</td>
						<td>105 days</td>
						<td>2023-02-20 04:33:12</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- count.smk
kind: CalledProcessError
line_no: 20
rest: "Command ' singularity  exec --home /home/azureuser/tmp/phzsatc8cf/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/phzsatc8cf/code/.snakemake/singularity/81d60ed7c2cf90e244b658b9d4d58014.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/02d721d968887d6d80a06bbd7ca09642'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/phzsatc8cf/code/.snakemake/scripts/tmpqs3mcz0n.generate_exclude_file.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/phzsatc8cf/code/workflow/rules/count.smk\"\
  , line 20, in __rule_generate_exclude_file_for_mosaic_count\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 04:44:25 2023]\nFinished job 8.\n1 of 25 steps (4%)\
  \ done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T043314.526233.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fs7rj08qn%2Flogs.tar.xz">3.084GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fs7rj08qn%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.257GiB</td>
						<td>1002.4 seconds</td>
						<td>672.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738220288">
								<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow">SchlossLab/mikropml-snakemake-workflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>104 days</td>
						<td>2023-02-19 04:06:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- learn.smk
kind: CalledProcessError
line_no: 68
rest: "Command 'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate\
  \ '/home/azureuser/tmp/mzfmy02ilc/code/.snakemake/conda/380f5f8d8fcb955186f723a8f3ea1525_';\
  \ set -euo pipefail;  Rscript --vanilla /home/azureuser/tmp/mzfmy02ilc/code/.snakemake/scripts/tmpd4b78l1z.find_feature_importance.R'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/mzfmy02ilc/code/workflow/rules/learn.smk\"\
  , line 68, in __rule_find_feature_importance\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-19T040630.717357.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/SchlossLab%252Fmikropml-snakemake-workflow%2Fyom9jh2u%2Flogs.tar.xz">0.299GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/SchlossLab%252Fmikropml-snakemake-workflow%2Fyom9jh2u%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>1.779GiB</td>
						<td>286.9 seconds</td>
						<td>417.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741621200">
								<a href="https://github.com/aryarm/admixtools">aryarm/admixtools</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/aryarm/admixtools/tree/v0.0.3">v0.0.3</a>
						</td>
						<td>snakemake</td>
						<td>103 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2Fgi2n7rpc%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2Fgi2n7rpc%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743110464">
								<a href="https://nf-co.re/fetchngs">fetchngs</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/fetchngs/tree/1.8">1.8</a>
						</td>
						<td>nextflow</td>
						<td>103 days</td>
						<td>2023-02-19 06:29:06</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Ftmj05wr5%2Flogs.tar.xz">0.033GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Ftmj05wr5%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>0.680GiB</td>
						<td>60.1 seconds</td>
						<td>72.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743138432">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>102 days</td>
						<td>2023-02-19 04:22:38</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.processor.TaskProcessor
date: Feb-19 04:25:11.255
kind: NextflowMiscError
process: Actor Thread 21
rest: Error executing process > 'NFCORE_METHYLSEQ:METHYLSEQ:ALIGNER:BISMARK_SUMMARY'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fhqqk20we%2Flogs.tar.xz">0.040GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fhqqk20we%2Foutput.tar.xz">0.007GiB</a>
						</td>
						<td>0.960GiB</td>
						<td>186.0 seconds</td>
						<td>162.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738286848">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.4.1">v2.4.1</a>
						</td>
						<td>snakemake</td>
						<td>101 days</td>
						<td>2023-02-20 04:31:52</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- trim.smk
kind: CalledProcessError
line_no: 13
rest: "Command ' singularity  exec --home /home/azureuser/tmp/50ifalyqnu/code/.test\
  \  --bind /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/50ifalyqnu/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/home/azureuser/tmp/50ifalyqnu/code/.test/.snakemake/conda/5cbfce52ea40cdaf1e914ab39d2b70f9_'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/50ifalyqnu/code/.test/.snakemake/scripts/tmpyb54zzil.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/50ifalyqnu/code/workflow/rules/trim.smk\"\
  , line 13, in __rule_cutadapt_pe\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nTraceback (most recent call last):\n  File \"/home/azureuser/tmp/50ifalyqnu/code/.test/.snakemake/scripts/tmpq_vv4_29.wrapper.py\"\
  , line 3, in <module>\n    import sys; sys.path.extend(['/mnt/snakemake', '/home/azureuser/.cache/snakemake/snakemake/source-cache/runtime-cache/tmpvqc5dmrz/https/github.com/snakemake/snakemake-wrappers/raw/0.31.1/bio/cutadapt/pe']);\
  \ import pickle; snakemake = pickle.loads(b\"\\x80\\x04\\x95P\\n\\x00\\x00\\x00\\\
  x00\\x00\\x00\\x8c\\x10snakemake.script\\x94\\x8c\\tSnakemake\\x94\\x93\\x94)\\\
  x81\\x94}\\x94(\\x8c\\x05input\\x94\\x8c\\x0csnakemake.io\\x94\\x8c\\nInputFiles\\\
  x94\\x93\\x94)\\x81\\x94(\\x8c ngs-test-data/reads/b.chr21.1.fq\\x94\\x8c ngs-test-data/reads/b.chr21.2.fq\\\
  x94e}\\x94(\\x8c\\x06_names\\x94}\\x94\\x8c\\x12_allowed_overrides\\x94]\\x94(\\\
  x8c\\x05index\\x94\\x8c\\x04sort\\x94eh\\x11\\x8c\\tfunctools\\x94\\x8c\\x07partial\\\
  x94\\x93\\x94h\\x06\\x8c\\x19Namedlist._used_attribute\\x94\\x93\\x94\\x85\\x94R\\\
  x94(h\\x17)}\\x94\\x8c\\x05_name\\x94h\\x11sNt\\x94bh\\x12h\\x15h\\x17\\x85\\x94R\\\
  x94(h\\x17)}\\x94h\\x1bh\\x12sNt\\x94bub\\x8c\\x06output\\x94h\\x06\\x8c\\x0bOutputFiles\\\
  x94\\x93\\x94)\\x81\\x94(\\x8c\\x1eresults/trimmed/B-1.1.fastq.gz\\x94\\x8c\\x1eresults/trimmed/B-1.2.fastq.gz\\\
  x94\\x8c\\x1aresults/trimmed/B-1.qc.txt\\x94e}\\x94(h\\r}\\x94(\\x8c\\x06fastq1\\\
  x94K\\x00N\\x86\\x94\\x8c\\x06fastq2\\x94K\\x01N\\x86\\x94\\x8c\\x02qc\\x94K\\x02N\\\
  x86\\x94uh\\x0f]\\x94(h\\x11h\\x12eh\\x11h\\x15h\\x17\\x85\\x94R\\x94(h\\x17)}\\\
  x94h\\x1bh\\x11sNt\\x94bh\\x12h\\x15h\\x17\\x85\\x94R\\x94(h\\x17)}\\x94h\\x1bh\\\
  x12sNt\\x94bh*h%h,h&h.h'ub\\x8c\\x06params\\x94h\\x06\\x8c\\x06Params\\x94\\x93\\\
  x94)\\x81\\x94\\x8c\\x8c-a ACGGATCGATCGATCGATCGAT -g GGATCGATCGATCGATCGAT -A ACGGATCGATCGATCGATCGAT\
  \ -G GGATCGATCGATCGATCGAT --minimum-length 33 -e 0.005 --overlap 7\\x94a}\\x94(h\\\
  r}\\x94h\\x0f]\\x94(h\\x11h\\x12eh\\x11h\\x15h\\x17\\x85\\x94R\\x94(h\\x17)}\\x94h\\\
  x1bh\\x11sNt\\x94bh\\x12h\\x15h\\x17\\x85\\x94R\\x94(h\\x17)}\\x94h\\x1bh\\x12sNt\\\
  x94bub\\x8c\\twildcards\\x94h\\x06\\x8c\\tWildcards\\x94\\x93\\x94)\\x81\\x94(\\\
  x8c\\x01B\\x94\\x8c\\x011\\x94e}\\x94(h\\r}\\x94(\\x8c\\x06sample\\x94K\\x00N\\\
  x86\\x94\\x8c\\x04unit\\x94K\\x01N\\x86\\x94uh\\x0f]\\x94(h\\x11h\\x12eh\\x11h\\\
  x15h\\x17\\x85\\x94R\\x94(h\\x17)}\\x94h\\x1bh\\x11sNt\\x94bh\\x12h\\x15h\\x17\\\
  x85\\x94R\\x94(h\\x17)}\\x94h\\x1bh\\x12sNt\\x94b\\x8c\\x06sample\\x94hM\\x8c\\\
  x04unit\\x94hNub\\x8c\\x07threads\\x94K\\x01\\x8c\\tresources\\x94h\\x06\\x8c\\\
  tResources\\x94\\x93\\x94)\\x81\\x94(K\\x01K\\x01\\x8c./home/azureuser/tmp/50ifalyqnu/singularity-tmp\\\
  x94e}\\x94(h\\r}\\x94(\\x8c\\x06_cores\\x94K\\x00N\\x86\\x94\\x8c\\x06_nodes\\x94K\\\
  x01N\\x86\\x94\\x8c\\x06tmpdir\\x94K\\x02N\\x86\\x94uh\\x0f]\\x94(h\\x11h\\x12eh\\\
  x11h\\x15h\\x17\\x85\\x94R\\x94(h\\x17)}\\x94h\\x1bh\\x11sNt\\x94bh\\x12h\\x15h\\\
  x17\\x85\\x94R\\x94(h\\x17)}\\x94h\\x1bh\\x12sNt\\x94bhhK\\x01hjK\\x01hlheub\\x8c\\\
  x03log\\x94h\\x06\\x8c\\x03Log\\x94\\x93\\x94)\\x81\\x94\\x8c\\x1dresults/logs/cutadapt/B-1.log\\\
  x94a}\\x94(h\\r}\\x94h\\x0f]\\x94(h\\x11h\\x12eh\\x11h\\x15h\\x17\\x85\\x94R\\x94(h\\\
  x17)}\\x94h\\x1bh\\x11sNt\\x94bh\\x12h\\x15h\\x17\\x85\\x94R\\x94(h\\x17)}\\x94h\\\
  x1bh\\x12sNt\\x94bub\\x8c\\x06config\\x94}\\x94(\\x8c\\x07samples\\x94\\x8c\\x12config/samples.tsv\\\
  x94\\x8c\\x05units\\x94\\x8c\\x10config/units.tsv\\x94\\x8c\\tresources\\x94}\\\
  x94(\\x8c\\x03ref\\x94}\\x94(\\x8c\\x07species\\x94\\x8c\\x0chomo_sapiens\\x94\\\
  x8c\\x07release\\x94\\x8c\\x03104\\x94\\x8c\\x05build\\x94\\x8c\\x06GRCh38\\x94\\\
  x8c\\x12species_db_version\\x94\\x8c\\x043.13\\x94\\x8c\\x04pfam\\x94\\x8c\\x0433.0\\\
  x94\\x8c\\x1arepresentative_transcripts\\x94\\x8c\\tcanonical\\x94u\\x8c\\x08ontology\\\
  x94}\\x94\\x8c\\rgene_ontology\\x94\\x8c5http://current.geneontology.org/ontology/go-basic.obo\\\
  x94su\\x8c\\x03pca\\x94}\\x94\\x8c\\x06labels\\x94]\\x94\\x8c\\tcondition\\x94as\\\
  x8c\\x07scatter\\x94}\\x94\\x8c\\x08activate\\x94\\x88s\\x8c\\x07diffexp\\x94}\\\
  x94(\\x8c\\x07exclude\\x94N\\x8c\\x06models\\x94}\\x94\\x8c\\x07model_X\\x94}\\\
  x94(\\x8c\\x04full\\x94\\x8c\\x19~condition + batch_effect\\x94\\x8c\\x07reduced\\\
  x94\\x8c\\r~batch_effect\\x94\\x8c\\x10primary_variable\\x94\\x8c\\tcondition\\\
  x94\\x8c\\nbase_level\\x94\\x8c\\tuntreated\\x94us\\x8c\\tsig-level\\x94}\\x94(\\\
  x8c\\x0cvolcano-plot\\x94G?\\xa9\\x99\\x99\\x99\\x99\\x99\\x9a\\x8c\\x07ma-plot\\\
  x94G?\\xa9\\x99\\x99\\x99\\x99\\x99\\x9a\\x8c\\x07qq-plot\\x94G?\\xa9\\x99\\x99\\\
  x99\\x99\\x99\\x9auu\\x8c\\ndiffsplice\\x94}\\x94(\\x8c\\x08activate\\x94\\x88\\\
  x8c\\rcoding_cutoff\\x94G?\\xe7333333\\x8c\\x15remove_noncoding_orfs\\x94\\x89\\\
  x8c\\x03fdr\\x94G?\\xf0\\x00\\x00\\x00\\x00\\x00\\x00\\x8c\\x0fmin_effect_size\\\
  x94G\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00u\\x8c\\nenrichment\\x94}\\x94(\\x8c\\\
  x08goatools\\x94}\\x94(\\x8c\\x08activate\\x94\\x88\\x8c\\tfdr_genes\\x94G?\\xa9\\\
  x99\\x99\\x99\\x99\\x99\\x9a\\x8c\\x0cfdr_go_terms\\x94G?\\xa9\\x99\\x99\\x99\\\
  x99\\x99\\x9au\\x8c\\x05fgsea\\x94}\\x94(\\x8c\\x0egene_sets_file\\x94\\x8c\\x1bngs-test-data/ref/dummy.gmt\\\
  x94\\x8c\\x08activate\\x94\\x88\\x8c\\x0cfdr_gene_set\\x94G?\\xa9\\x99\\x99\\x99\\\
  x99\\x99\\x9a\\x8c\\x03eps\\x94G?\\x1a6\\xe2\\xeb\\x1cC-u\\x8c\\x04spia\\x94}\\\
  x94(\\x8c\\x08activate\\x94\\x88\\x8c\\x10pathway_database\\x94\\x8c\\x07panther\\\
  x94uu\\x8c\\x0fbootstrap_plots\\x94}\\x94(\\x8c\\x03FDR\\x94G?\\x84z\\xe1G\\xae\\\
  x14{\\x8c\\x05top_n\\x94K\\x03\\x8c\\x08color_by\\x94\\x8c\\tcondition\\x94\\x8c\\\
  x11genes_of_interest\\x94]\\x94\\x8c\\x06A4galt\\x94au\\x8c\\tplot_vars\\x94}\\\
  x94\\x8c\\tsig_level\\x94G?\\xb9\\x99\\x99\\x99\\x99\\x99\\x9as\\x8c\\x06params\\\
  x94}\\x94(\\x8c\\x08kallisto\\x94\\x8c\\x06-b 100\\x94\\x8c\\x0bcutadapt-se\\x94\\\
  x8c\\x00\\x94\\x8c\\x0bcutadapt-pe\\x94h=uu\\x8c\\x04rule\\x94\\x8c\\x0bcutadapt_pe\\\
  x94\\x8c\\x0fbench_iteration\\x94N\\x8c\\tscriptdir\\x94\\x8cJhttps://github.com/snakemake/snakemake-wrappers/raw/0.31.1/bio/cutadapt/pe\\\
  x94ub.\"); from snakemake.logging import logger; logger.printshellcmds = False;\
  \ __real_file__ = __file__; __file__ = 'https://github.com/snakemake/snakemake-wrappers/raw/0.31.1/bio/cutadapt/pe/wrapper.py';\n\
  ModuleNotFoundError: No module named 'snakemake.script'\n[Mon Feb 20 05:12:54 2023]\n\
  Error in rule cutadapt_pe:\n    jobid: 12\n    input: ngs-test-data/reads/b.chr21.1.fq,\
  \ ngs-test-data/reads/b.chr21.2.fq\n    output: results/trimmed/B-1.1.fastq.gz,\
  \ results/trimmed/B-1.2.fastq.gz, results/trimmed/B-1.qc.txt\n    log: results/logs/cutadapt/B-1.log\
  \ (check log file(s) for error message)\n    conda-env: /home/azureuser/tmp/50ifalyqnu/code/.test/.snakemake/conda/5cbfce52ea40cdaf1e914ab39d2b70f9_\n\
  \nRuleException:\nCalledProcessError in line 13 of /home/azureuser/tmp/50ifalyqnu/code/workflow/rules/trim.smk:\n\
  Command ' singularity  exec --home /home/azureuser/tmp/50ifalyqnu/code/.test  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/50ifalyqnu/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/home/azureuser/tmp/50ifalyqnu/code/.test/.snakemake/conda/5cbfce52ea40cdaf1e914ab39d2b70f9_'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/50ifalyqnu/code/.test/.snakemake/scripts/tmpq_vv4_29.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/50ifalyqnu/code/workflow/rules/trim.smk\"\
  , line 13, in __rule_cutadapt_pe\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T043157.707803.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2F1opzcym3%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2F1opzcym3%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.261GiB</td>
						<td>1959.5 seconds</td>
						<td>2463.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743138192">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>101 days</td>
						<td>2023-02-19 04:19:58</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowMiscError
class_: nextflow.processor.TaskProcessor
date: Feb-19 04:23:10.363
kind: NextflowMiscError
process: Actor Thread 25
rest: Error executing process > 'NFCORE_METHYLSEQ:METHYLSEQ:ALIGNER:BISMARK_SUMMARY'

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Ffuy0wbzt%2Flogs.tar.xz">0.040GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Ffuy0wbzt%2Foutput.tar.xz">0.007GiB</a>
						</td>
						<td>0.775GiB</td>
						<td>190.7 seconds</td>
						<td>195.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738322016">
								<a href="https://github.com/franciscozorrilla/metaGEM">franciscozorrilla/metaGEM</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/franciscozorrilla/metaGEM/tree/v1.0.2">v1.0.2</a>
						</td>
						<td>snakemake</td>
						<td>101 days</td>
						<td>2023-02-19 04:30:43</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/franciscozorrilla%252FmetaGEM%2Firyrwt1n%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/franciscozorrilla%252FmetaGEM%2Firyrwt1n%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.063GiB</td>
						<td>2.8 seconds</td>
						<td>3.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738293808">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.3">v3.18.3</a>
						</td>
						<td>snakemake</td>
						<td>100 days</td>
						<td>2023-02-19 04:48:14</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 7boifm6kcg
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/7boifm6kcg/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fm0g5u5c9%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fm0g5u5c9%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.3 seconds</td>
						<td>1.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738293568">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.4">v3.18.4</a>
						</td>
						<td>snakemake</td>
						<td>100 days</td>
						<td>2023-02-19 03:32:37</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- qchvospifp
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/qchvospifp/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fxwzxgnn6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fxwzxgnn6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.4 seconds</td>
						<td>1.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738212848">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.5.1">v1.5.1</a>
						</td>
						<td>snakemake</td>
						<td>100 days</td>
						<td>2023-02-19 04:29:29</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fufo0jktx%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fufo0jktx%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.529GiB</td>
						<td>942.0 seconds</td>
						<td>869.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738207808">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.5.1">v1.5.1</a>
						</td>
						<td>snakemake</td>
						<td>100 days</td>
						<td>2023-02-19 03:56:25</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F08ibjmyl%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F08ibjmyl%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.650GiB</td>
						<td>1320.5 seconds</td>
						<td>1564.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738167056">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.5">1.3.5</a>
						</td>
						<td>snakemake</td>
						<td>97 days</td>
						<td>2023-02-20 05:22:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- rules.smk
kind: CalledProcessError
line_no: 52
rest: "Command ' singularity  exec --home /home/azureuser/tmp/nfiv98m6pc/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/nfiv98m6pc/code/.snakemake/singularity/3b5909a4d5be3839879a7a7d3b629434.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/08d4368302a4bdf7eda6b536495efe7d'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/nfiv98m6pc/code/.snakemake/scripts/tmpubjew41n.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/nfiv98m6pc/code/workflow/rules/rules.smk\"\
  , line 52, in __rule_fastqc\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 05:26:09 2023]\nError in rule fastqc:\n    jobid:\
  \ 10\n    input: .tests/data_CHR17/RPE-BM510/fastq/BM510x04_PE20305.1.fastq.gz\n\
  \    output: .tests/data_CHR17/RPE-BM510/fastqc/BM510x04_PE20305_1_fastqc.html,\
  \ .tests/data_CHR17/RPE-BM510/fastqc/BM510x04_PE20305_1_fastqc.zip\n    log: .tests/data_CHR17/log/fastqc/RPE-BM510/BM510x04_PE20305_1.log\
  \ (check log file(s) for error message)\n    conda-env: /conda-envs/08d4368302a4bdf7eda6b536495efe7d\n\
  \nRuleException:\nCalledProcessError in line 52 of /home/azureuser/tmp/nfiv98m6pc/code/workflow/rules/rules.smk:\n\
  Command ' singularity  exec --home /home/azureuser/tmp/nfiv98m6pc/code  --bind /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/nfiv98m6pc/code/.snakemake/singularity/3b5909a4d5be3839879a7a7d3b629434.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/08d4368302a4bdf7eda6b536495efe7d'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/nfiv98m6pc/code/.snakemake/scripts/tmp8mlrxe72.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/nfiv98m6pc/code/workflow/rules/rules.smk\"\
  , line 52, in __rule_fastqc\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T052217.477682.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F3pacjrgv%2Flogs.tar.xz">0.999GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F3pacjrgv%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.623GiB</td>
						<td>493.2 seconds</td>
						<td>232.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738293328">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.5">v3.18.5</a>
						</td>
						<td>snakemake</td>
						<td>97 days</td>
						<td>2023-02-19 04:45:57</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 8hmsqpsa4w
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/8hmsqpsa4w/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F1gnk0668%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F1gnk0668%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.5 seconds</td>
						<td>3.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738165136">
								<a href="https://github.com/snakemake-workflows/dna-seq-mtb">snakemake-workflows/dna-seq-mtb</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.3.1">v1.3.1</a>
						</td>
						<td>snakemake</td>
						<td>97 days</td>
						<td>2023-02-19 04:30:34</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- i4s32xlkgt
- code
- workflow
- rules
- common.smk
kind: WorkflowError
line_no: 20
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/i4s32xlkgt/code/workflow/Snakefile\", line 1, in <module>\n\
  \  File \"/home/azureuser/tmp/i4s32xlkgt/code/workflow/rules/common.smk\", line\
  \ 20, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2F8jbx5k5z%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2F8jbx5k5z%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.8 seconds</td>
						<td>2.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738293088">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.6">v3.18.6</a>
						</td>
						<td>snakemake</td>
						<td>96 days</td>
						<td>2023-02-19 04:36:05</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 9xhtqo6mug
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/9xhtqo6mug/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fhyy9pvqb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fhyy9pvqb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.3 seconds</td>
						<td>2.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743189264">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/3.1">3.1</a>
						</td>
						<td>nextflow</td>
						<td>95 days</td>
						<td>2023-02-19 04:02:55</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fc5lii8sk%2Flogs.tar.xz">0.094GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2Fc5lii8sk%2Foutput.tar.xz">0.032GiB</a>
						</td>
						<td>2.456GiB</td>
						<td>273.1 seconds</td>
						<td>166.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743054048">
								<a href="https://nf-co.re/eager">eager</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/eager/tree/2.4.6">2.4.6</a>
						</td>
						<td>nextflow</td>
						<td>94 days</td>
						<td>2023-02-19 06:00:22</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fe0uvni6s%2Flogs.tar.xz">0.015GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/eager%2Fe0uvni6s%2Foutput.tar.xz">0.012GiB</a>
						</td>
						<td>1.707GiB</td>
						<td>740.2 seconds</td>
						<td>608.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743189024">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/3.1.1">3.1.1</a>
						</td>
						<td>nextflow</td>
						<td>90 days</td>
						<td>2023-02-19 03:31:18</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2F8oic07h9%2Flogs.tar.xz">0.087GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2F8oic07h9%2Foutput.tar.xz">0.025GiB</a>
						</td>
						<td>2.365GiB</td>
						<td>274.7 seconds</td>
						<td>333.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738268000">
								<a href="https://github.com/GooglingTheCancerGenome/sv-callers">GooglingTheCancerGenome/sv-callers</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/GooglingTheCancerGenome/sv-callers/tree/v1.2.2">v1.2.2</a>
						</td>
						<td>snakemake</td>
						<td>88 days</td>
						<td>2023-02-19 04:39:38</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: ModuleNotFoundError
line_no: 7
rest: "No module named 'yatiml'\n  File \"/home/azureuser/tmp/7qfk4uy5mm/code/workflow/Snakefile\"\
  , line 7, in <module>\n  File \"/home/azureuser/tmp/7qfk4uy5mm/code/workflow/helper_functions.py\"\
  , line 5, in <module>\n  File \"/home/azureuser/tmp/7qfk4uy5mm/code/workflow/validator/__init__.py\"\
  , line 8, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2Fqzw0hsso%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-callers%2Fqzw0hsso%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.061GiB</td>
						<td>1.3 seconds</td>
						<td>1.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738292848">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.18.7">v3.18.7</a>
						</td>
						<td>snakemake</td>
						<td>87 days</td>
						<td>2023-02-19 06:33:05</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- uayjc5hg68
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/uayjc5hg68/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F89fzjmst%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F89fzjmst%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.4 seconds</td>
						<td>2.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738254080">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.8.5">1.8.5</a>
						</td>
						<td>snakemake</td>
						<td>84 days</td>
						<td>2023-02-20 04:55:39</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- count.smk
kind: CalledProcessError
line_no: 20
rest: "Command ' singularity  exec --home /home/azureuser/tmp/5c635sn8h4/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/5c635sn8h4/code/.snakemake/singularity/5afcbe3f77d743e19a7c9d3055a3b549.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/02d721d968887d6d80a06bbd7ca09642'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/5c635sn8h4/code/.snakemake/scripts/tmpj5vy537y.generate_exclude_file.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/5c635sn8h4/code/workflow/rules/count.smk\"\
  , line 20, in __rule_generate_exclude_file_for_mosaic_count\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 05:05:33 2023]\nFinished job 13.\n2 of 25 steps (8%)\
  \ done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T045539.807888.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2F0l5vz99d%2Flogs.tar.xz">3.105GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2F0l5vz99d%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.091GiB</td>
						<td>1006.9 seconds</td>
						<td>594.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738266560">
								<a href="https://github.com/zavolanlab/zarp">zavolanlab/zarp</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/zavolanlab/zarp/tree/v0.4.0">v0.4.0</a>
						</td>
						<td>snakemake</td>
						<td>82 days</td>
						<td>2023-02-19 04:31:21</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: KeyError
line_no: 13
rest: "'samples'\n  File \"/home/azureuser/tmp/w1zzblg6nc/code/workflow/Snakefile\"\
  , line 13, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/zavolanlab%252Fzarp%2Fkeagivr5%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/zavolanlab%252Fzarp%2Fkeagivr5%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.098GiB</td>
						<td>1.9 seconds</td>
						<td>2.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743137952">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/2.2.0">2.2.0</a>
						</td>
						<td>nextflow</td>
						<td>82 days</td>
						<td>2023-02-19 04:40:36</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2F7abb93h2%2Flogs.tar.xz">0.046GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2F7abb93h2%2Foutput.tar.xz">0.013GiB</a>
						</td>
						<td>0.846GiB</td>
						<td>228.8 seconds</td>
						<td>196.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010745299920">
								<a href="https://nf-co.re/atacseq">atacseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/atacseq/tree/2.0">2.0</a>
						</td>
						<td>nextflow</td>
						<td>80 days</td>
						<td>2023-02-19 06:19:44</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "printf \"%s %s\\n\" SRR1822153_1.fastq.gz OSMOTIC_STRESS_T100_SE_REP1_T1.gz\
  \ | while read old_name new_name; do\n    [ -f \"${new_name}\" ] || ln -s $old_name\
  \ $new_name\ndone\nfastqc --quiet --threads 2 OSMOTIC_STRESS_T100_SE_REP1_T1.gz\n\
  \ncat <<-END_VERSIONS > versions.yml\n\"NFCORE_ATACSEQ:ATACSEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC\"\
  :\n    fastqc: $( fastqc --version | sed -e \"s/FastQC v//g\" )\nEND_VERSIONS"
error: 'WARNING: Skipping mount /home/azureuser/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/singularityce-3.10.3-z5qu45ho72nvsy4rpcyd4eqrop35tnbe/var/singularity/mnt/session/etc/resolv.conf
  [files]: /etc/resolv.conf doesn''t exist in container

  Skipping ''OSMOTIC_STRESS_T100_SE_REP1_T1.gz'' which didn''t exist, or couldn''t
  be read'
exit_status: 0
kind: CalledProcessError
output: (empty)
process: Missing output file(s) `*.html` expected by process `NFCORE_ATACSEQ:ATACSEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC
  (OSMOTIC_STRESS_T100_SE_REP1_T1)`
work_dir: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 2phf4f6ipc
- code
- work
- 3d
- 424c4da0eb99c93d054a3380061803

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2Fstbehp35%2Flogs.tar.xz">0.058GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/atacseq%2Fstbehp35%2Foutput.tar.xz">0.004GiB</a>
						</td>
						<td>1.055GiB</td>
						<td>120.0 seconds</td>
						<td>94.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738256720">
								<a href="https://github.com/IKIM-Essen/uncovar">IKIM-Essen/uncovar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/IKIM-Essen/uncovar/tree/v0.15.1">v0.15.1</a>
						</td>
						<td>snakemake</td>
						<td>80 days</td>
						<td>2023-02-19 03:30:59</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- read_trimming.smk
kind: MissingInputException
rest: "Missing input files for rule fastp_pe:\n    output: results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.1.fastq.gz,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.2.fastq.gz, results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.html,\
  \ results/SEQUENCING_DATE/trimmed/fastp-pe/SAMPLE_NAME_1.fastp.json\n    wildcards:\
  \ date=SEQUENCING_DATE, sample=SAMPLE_NAME_1\n    affected files:\n        PATH/TO/fq1\n\
  \        PATH/TO/fq2"
rule: fastp_pe  in line 7

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fv4ffdg4n%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/IKIM-Essen%252Funcovar%2Fv4ffdg4n%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.110GiB</td>
						<td>2.9 seconds</td>
						<td>4.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743140592">
								<a href="https://nf-co.re/mhcquant">mhcquant</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/mhcquant/tree/2.4.0">2.4.0</a>
						</td>
						<td>nextflow</td>
						<td>80 days</td>
						<td>2023-02-19 04:29:58</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "DecoyDatabase -in test.fasta \\\n    -out test_decoy.fasta \\\n\
  \    -decoy_string DECOY_ \\\n    -decoy_string_position prefix\n\ncat <<-END_VERSIONS\
  \ > versions.yml\n\"NFCORE_MHCQUANT:MHCQUANT:OPENMS_DECOYDATABASE\":\n    openms:\
  \ $(echo $(FileInfo --help 2>&1) | sed 's/^.*Version: //; s/-.*$//' | sed 's/ -*//;\
  \ s/ .*$//')\nEND_VERSIONS"
error: 'Error: File not found (the file ''test.fasta'' could not be found)

  Cannot read input file given from parameter ''-in''!'
exit_status: 1
kind: CalledProcessError
output: 'Error: File not found (the file ''test.fasta'' could not be found)'
process: NFCORE_MHCQUANT:MHCQUANT:OPENMS_DECOYDATABASE (4)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- '30'
- e4373d9edca7ceccc74127ea51f270

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fumo370r6%2Flogs.tar.xz">0.034GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/mhcquant%2Fumo370r6%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.892GiB</td>
						<td>56.5 seconds</td>
						<td>34.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743826128">
								<a href="https://github.com/dbespiatykh/TBvar">dbespiatykh/TBvar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/dbespiatykh/TBvar/tree/v1.0.1">v1.0.1</a>
						</td>
						<td>snakemake</td>
						<td>79 days</td>
						<td>2023-02-19 03:31:20</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- reference.smk
kind: WorkflowError
line_no: 1
rest: "The Python package 'biopython' needs to be installed to use NCBI Entrez remote()\
  \ file functionality. No module named 'Bio'\n  File \"/home/azureuser/tmp/7sgyu3usqc/code/workflow/Snakefile\"\
  , line 15, in <module>\n  File \"/home/azureuser/tmp/7sgyu3usqc/code/workflow/rules/reference.smk\"\
  , line 1, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2F8h449es6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2F8h449es6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.101GiB</td>
						<td>1.9 seconds</td>
						<td>3.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738266320">
								<a href="https://github.com/zavolanlab/zarp">zavolanlab/zarp</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/zavolanlab/zarp/tree/v0.4.1">v0.4.1</a>
						</td>
						<td>snakemake</td>
						<td>76 days</td>
						<td>2023-02-19 03:59:32</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeRuleError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: MissingInputException
rest: "Missing input files for rule start:\n    output: results/samples/synthetic_10_reads_paired_synthetic_10_reads_paired/start/synthetic_10_reads_paired_synthetic_10_reads_paired.fq1.fastq.gz\n\
  \    wildcards: sample=synthetic_10_reads_paired_synthetic_10_reads_paired, mate=fq1\n\
  \    affected files:\n        ../input_files/project1/synthetic.mate_1.fastq.gz"
rule: start  in line 178

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/zavolanlab%252Fzarp%2F8wcxh7la%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/zavolanlab%252Fzarp%2F8wcxh7la%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.103GiB</td>
						<td>2.1 seconds</td>
						<td>4.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010762278688">
								<a href="https://nf-co.re/airrflow">airrflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/airrflow/tree/2.4.0">2.4.0</a>
						</td>
						<td>nextflow</td>
						<td>76 days</td>
						<td>2023-02-20 04:31:44</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowJavaError
kind: java.lang.IllegalStateException
msg: 'java.lang.IllegalStateException: Failed to pull singularity image'
rest: "command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867546979\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.getCachePathFor(SingularityCache.groovy:348)\n\
  \tat nextflow.container.ContainerHandler.createSingularityCache(ContainerHandler.groovy:97)\n\
  \tat nextflow.container.ContainerHandler.normalizeImageName(ContainerHandler.groovy:81)\n\
  \tat nextflow.container.resolver.DefaultContainerResolver.resolveImage(DefaultContainerResolver.groovy:44)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:170)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:73)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)\n\
  \tat nextflow.processor.TaskRun.memoizedMethodPriv$getContainerInfo0(TaskRun.groovy:600)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:362)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)\n\
  \tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\tat nextflow.processor.TaskRun$_closure3.doCall(TaskRun.groovy)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat groovy.lang.Closure.call(Closure.java:412)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)\n\
  \tat org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)\n\
  \tat org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)\n\
  \tat groovy.lang.Closure.call(Closure.java:406)\n\tat java_util_concurrent_Callable$call.call(Unknown\
  \ Source)\n\tat nextflow.processor.TaskRun.getContainerInfo0(TaskRun.groovy)\n\t\
  at nextflow.processor.TaskRun.getContainer(TaskRun.groovy:608)\n\tat nextflow.processor.TaskRun.isContainerEnabled(TaskRun.groovy:641)\n\
  \tat nextflow.processor.TaskRun$isContainerEnabled$2.call(Unknown Source)\n\tat\
  \ nextflow.processor.TaskProcessor.createTaskHashKey(TaskProcessor.groovy:2013)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)\n\
  \tat org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)\n\
  \tat org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)\n\
  \tat nextflow.processor.TaskProcessor.invokeTask(TaskProcessor.groovy:599)\n\tat\
  \ nextflow.processor.InvokeTaskAdapter.call(InvokeTaskAdapter.groovy:53)\n\tat groovyx.gpars.dataflow.operator.DataflowOperatorActor.startTask(DataflowOperatorActor.java:120)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor.access$001(ForkingDataflowOperatorActor.java:35)\n\
  \tat groovyx.gpars.dataflow.operator.ForkingDataflowOperatorActor$1.run(ForkingDataflowOperatorActor.java:58)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\
  \tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\
  \tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.IllegalStateException:\
  \ Failed to pull singularity image\n  command: singularity pull  --name containers.biocontainers.pro-s3-SingImgsRepo-biocontainers-v1.2.0_cv1-biocontainers_v1.2.0_cv1.img.img.pulling.1676867546979\
  \ https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img\
  \ > /dev/null\n  status : 143\n  message:\n    INFO:    Downloading network image\n\
  \n\tat nextflow.container.SingularityCache.runCommand(SingularityCache.groovy:300)\n\
  \tat nextflow.container.SingularityCache.downloadSingularityImage0(SingularityCache.groovy:267)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.doCall(SingularityCache.groovy:235)\n\
  \tat nextflow.container.SingularityCache$_downloadSingularityImage_closure1.call(SingularityCache.groovy)\n\
  \tat nextflow.file.FileMutex.lock(FileMutex.groovy:107)\n\tat nextflow.container.SingularityCache.downloadSingularityImage(SingularityCache.groovy:235)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\
  \tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\
  \tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\
  \tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)\n\
  \tat groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)\n\tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)\n\
  \tat groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)\n\tat org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)\n\
  \tat org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.doCall(SingularityCache.groovy:325)\n\
  \tat nextflow.container.SingularityCache$_getLazyImagePath_closure2.call(SingularityCache.groovy)\n\
  \tat groovyx.gpars.dataflow.LazyDataflowVariable$1.run(LazyDataflowVariable.java:70)\n\
  \t... 3 common frames omitted\nFeb-20 07:31:44.545 [main] DEBUG nextflow.trace.WorkflowStatsObserver\
  \ - Workflow completed > WorkflowStats[succeededCount=17; failedCount=0; ignoredCount=0;\
  \ cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0;\
  \ abortedCount=0; succeedDuration=2m 46s; failedDuration=0ms; cachedDuration=0ms;loadCpus=0;\
  \ loadMemory=0; peakRunning=2; peakCpus=4; peakMemory=12 GB; ]\nFeb-20 07:31:44.560\
  \ [main] DEBUG nextflow.trace.TraceFileObserver - Workflow completed -- saving trace\
  \ file\nFeb-20 07:31:44.561 [main] DEBUG nextflow.trace.ReportObserver - Workflow\
  \ completed -- rendering execution report\nFeb-20 07:31:44.604 [main] DEBUG nextflow.trace.ReportObserver\
  \ - Execution report summary data:\n  [{\"cpuUsage\":{\"mean\":31.7,\"min\":31.7,\"\
  q1\":31.7,\"q2\":31.7,\"q3\":31.7,\"max\":31.7,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\"},\"process\":\"SAMPLESHEET_CHECK\",\"mem\":{\"mean\":66973696,\"\
  min\":66973696,\"q1\":66973696,\"q2\":66973696,\"q3\":66973696,\"max\":66973696,\"\
  minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK (Metadata_test.tsv)\"\
  ,\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK (Metadata_test.tsv)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK (Metadata_test.tsv)\"\
  ,\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK (Metadata_test.tsv)\"\
  ,\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK (Metadata_test.tsv)\"\
  },\"memUsage\":{\"mean\":1.04,\"min\":1.04,\"q1\":1.04,\"q2\":1.04,\"q3\":1.04,\"\
  max\":1.04,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\"},\"timeUsage\":{\"mean\":0.06,\"min\":0.06,\"q1\":0.06,\"\
  q2\":0.06,\"q3\":0.06,\"max\":0.06,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\"},\"vmem\":{\"mean\":567058432,\"min\":567058432,\"q1\":567058432,\"\
  q2\":567058432,\"q3\":567058432,\"max\":567058432,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\"},\"reads\":{\"mean\":22559082,\"min\":22559082,\"q1\":22559082,\"\
  q2\":22559082,\"q3\":22559082,\"max\":22559082,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\"},\"cpu\":{\"mean\":31.7,\"min\":31.7,\"q1\":31.7,\"q2\"\
  :31.7,\"q3\":31.7,\"max\":31.7,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\"},\"time\":{\"mean\":8000,\"min\":8000,\"q1\":8000,\"q2\"\
  :8000,\"q3\":8000,\"max\":8000,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\"},\"writes\":{\"mean\":3464,\"min\":3464,\"q1\":3464,\"q2\"\
  :3464,\"q3\":3464,\"max\":3464,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:INPUT_CHECK:SAMPLESHEET_CHECK\
  \ (Metadata_test.tsv)\"}},{\"cpuUsage\":{\"mean\":11.95,\"min\":9.4,\"q1\":10.53,\"\
  q2\":11.2,\"q3\":12.69,\"max\":17.65,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample7)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample4)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample6)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample5)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample8)\"},\"process\":\"FASTP\",\"mem\":{\"mean\":1005267456,\"min\":274411520,\"\
  q1\":1097218048,\"q2\":1121480704,\"q3\":1124166656,\"max\":1125879808,\"minLabel\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample4)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample1)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample7)\"\
  ,\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample6)\",\"q3Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample2)\"},\"memUsage\":{\"mean\"\
  :15.6,\"min\":4.26,\"q1\":17.03,\"q2\":17.41,\"q3\":17.45,\"max\":17.48,\"minLabel\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample4)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample1)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample7)\"\
  ,\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample6)\",\"q3Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample2)\"},\"timeUsage\":{\"mean\"\
  :0.01,\"min\":0,\"q1\":0.01,\"q2\":0.01,\"q3\":0.01,\"max\":0.01,\"minLabel\":\"\
  NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample4)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample7)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample2)\"\
  ,\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample5)\",\"q3Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample3)\"},\"vmem\":{\"mean\":1308305920,\"\
  min\":1103048704,\"q1\":1308276736,\"q2\":1376714752,\"q3\":1376732160,\"max\":1376768000,\"\
  minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample4)\",\"maxLabel\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample8)\",\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample7)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample1)\"\
  ,\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample2)\"},\"reads\"\
  :{\"mean\":2719721,\"min\":2359218,\"q1\":2509671.5,\"q2\":2697795,\"q3\":2968189.5,\"\
  max\":3075364,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample4)\"\
  ,\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample1)\",\"q1Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample6)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample5)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample3)\"\
  },\"cpu\":{\"mean\":23.9,\"min\":18.8,\"q1\":21.05,\"q2\":22.4,\"q3\":25.38,\"max\"\
  :35.3,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample7)\",\"\
  maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample4)\",\"q1Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample6)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample5)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample8)\"\
  },\"time\":{\"mean\":2250,\"min\":1000,\"q1\":2000,\"q2\":2000,\"q3\":3000,\"max\"\
  :3000,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample4)\",\"\
  maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample7)\",\"q1Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample2)\",\"q2Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample5)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample3)\"\
  },\"writes\":{\"mean\":1488880.38,\"min\":1388993,\"q1\":1429997,\"q2\":1478305.5,\"\
  q3\":1562769.75,\"max\":1591854,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample4)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample1)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample6)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP (Sample5)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:FASTP\
  \ (Sample3)\"}},{\"cpuUsage\":{\"mean\":15.85,\"min\":9,\"q1\":11.51,\"q2\":16.38,\"\
  q3\":20.64,\"max\":22.05,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample3)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample6)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample1)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample2)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample8)\"},\"process\":\"MERGE_UMI\",\"mem\":{\"mean\":28861952,\"min\":28745728,\"\
  q1\":28803072,\"q2\":28856320,\"q3\":28904448,\"max\":28991488,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample4)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample5)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample3)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample1)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample2)\"},\"memUsage\":{\"mean\":0.45,\"min\":0.45,\"q1\":0.45,\"q2\":0.45,\"\
  q3\":0.45,\"max\":0.45,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample4)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample5)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample3)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample1)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample2)\"},\"timeUsage\":{\"mean\":0.05,\"min\":0.03,\"q1\":0.03,\"q2\":0.05,\"\
  q3\":0.07,\"max\":0.08,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample6)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample1)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample8)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample7)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample4)\"},\"vmem\":{\"mean\":73686528,\"min\":54300672,\"q1\":74833920,\"q2\"\
  :75911168,\"q3\":76988416,\"max\":79724544,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample7)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample4)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample1)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample6)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample2)\"},\"reads\":{\"mean\":10306883.88,\"min\":10254040,\"q1\":10274262.75,\"\
  q2\":10305889.5,\"q3\":10339923.75,\"max\":10360244,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample4)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample1)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample6)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample5)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample3)\"},\"cpu\":{\"mean\":31.7,\"min\":18,\"q1\":23.03,\"q2\":32.75,\"q3\"\
  :41.28,\"max\":44.1,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample3)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample6)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample1)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample2)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample8)\"},\"time\":{\"mean\":7625,\"min\":4000,\"q1\":5000,\"q2\":7000,\"q3\"\
  :9750,\"max\":12000,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample6)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample1)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample8)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample7)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample4)\"},\"writes\":{\"mean\":361485.25,\"min\":307865,\"q1\":332074.25,\"\
  q2\":359727.5,\"q3\":395853.25,\"max\":412432,\"minLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample4)\",\"maxLabel\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample1)\"\
  ,\"q1Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample6)\",\"q2Label\"\
  :\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI (Sample5)\",\"q3Label\":\"NFCORE_AIRRFLOW:BCELLMAGIC:PRESTO_UMI:MERGE_UMI\
  \ (Sample3)\"}}]\nFeb-20 07:31:44.609 [Actor Thread 61] DEBUG nextflow.sort.BigSort\
  \ - Sort completed -- entries: 17; slices: 1; internal sort time: 0.029 s; external\
  \ sort time: 0.002 s; total time: 0.031 s\nFeb-20 07:31:44.629 [Actor Thread 61]\
  \ DEBUG nextflow.file.FileCollector - Saved collect-files list to: /home/azureuser/tmp/4w8i3i4df7/singularity-tmp/a620c7f5076b29428ad9fa47193d8352.collect-file\n\
  Feb-20 07:31:44.632 [Actor Thread 61] DEBUG nextflow.file.FileCollector - Deleting\
  \ file collector temp dir: /home/azureuser/tmp/4w8i3i4df7/singularity-tmp/nxf-8060261275996223310\n\
  Feb-20 07:31:45.022 [main] DEBUG nextflow.trace.TimelineObserver - Workflow completed\
  \ -- rendering execution timeline\nFeb-20 07:31:45.201 [main] DEBUG nextflow.cache.CacheDB\
  \ - Closing CacheDB done\nFeb-20 07:31:45.241 [main] DEBUG nextflow.script.ScriptRunner\
  \ - > Execution complete -- Goodbye"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Fe1ab8ajt%2Flogs.tar.xz">0.016GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/airrflow%2Fe1ab8ajt%2Foutput.tar.xz">0.007GiB</a>
						</td>
						<td>1.048GiB</td>
						<td>89.8 seconds</td>
						<td>10800.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010746065440">
								<a href="https://nf-co.re/ampliseq">ampliseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/ampliseq/tree/2.4.1">2.4.1</a>
						</td>
						<td>nextflow</td>
						<td>75 days</td>
						<td>2023-02-20 04:35:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowSigterm
date: Feb-20 07:35:16.634
kind: Sigterm
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Faiwshqus%2Flogs.tar.xz">0.029GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/ampliseq%2Faiwshqus%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>2.068GiB</td>
						<td>1229.6 seconds</td>
						<td>10800.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738265600">
								<a href="https://github.com/sneuensc/mapache">sneuensc/mapache</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/sneuensc/mapache/tree/v0.3.0">v0.3.0</a>
						</td>
						<td>snakemake</td>
						<td>68 days</td>
						<td>2023-02-19 04:19:13</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- scripts
- utils.py
kind: SystemExit
line_no: 432
rest: "1\n  File \"/home/azureuser/tmp/fod63rk08m/code/workflow/Snakefile\", line\
  \ 82, in <module>\n  File \"/home/azureuser/tmp/fod63rk08m/code/workflow/scripts/utils.py\"\
  , line 432, in test_SAMPLES"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sneuensc%252Fmapache%2Fpwsqngyw%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sneuensc%252Fmapache%2Fpwsqngyw%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.104GiB</td>
						<td>2.2 seconds</td>
						<td>2.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738292608">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.0">v3.19.0</a>
						</td>
						<td>snakemake</td>
						<td>68 days</td>
						<td>2023-02-19 04:15:45</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- jwip7w8dqw
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/jwip7w8dqw/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fds8wfkq7%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fds8wfkq7%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.1 seconds</td>
						<td>1.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743137712">
								<a href="https://nf-co.re/methylseq">methylseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/methylseq/tree/2.3.0">2.3.0</a>
						</td>
						<td>nextflow</td>
						<td>64 days</td>
						<td>2023-02-19 04:25:32</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fso5tj5r0%2Flogs.tar.xz">0.046GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/methylseq%2Fso5tj5r0%2Foutput.tar.xz">0.013GiB</a>
						</td>
						<td>0.830GiB</td>
						<td>229.9 seconds</td>
						<td>169.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738270160">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.4.2">v2.4.2</a>
						</td>
						<td>snakemake</td>
						<td>62 days</td>
						<td>2023-02-20 04:31:47</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- trim.smk
kind: CalledProcessError
line_no: 13
rest: "Command ' singularity  exec --home /home/azureuser/tmp/m31fucwrbo/code/.test\
  \  --bind /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/m31fucwrbo/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/home/azureuser/tmp/m31fucwrbo/code/.test/.snakemake/conda/1f12dbbb3a74563ca0b2467603494776_'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/m31fucwrbo/code/.test/.snakemake/scripts/tmpgbmnhylo.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/m31fucwrbo/code/workflow/rules/trim.smk\"\
  , line 13, in __rule_cutadapt_pe\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 05:00:58 2023]\nError in rule cutadapt_pe:\n    jobid:\
  \ 17\n    input: ngs-test-data/reads/b.chr21.1.fq, ngs-test-data/reads/b.chr21.2.fq\n\
  \    output: results/trimmed/D-1.1.fastq.gz, results/trimmed/D-1.2.fastq.gz, results/trimmed/D-1.qc.txt\n\
  \    log: results/logs/cutadapt/D-1.log (check log file(s) for error message)\n\
  \    conda-env: /home/azureuser/tmp/m31fucwrbo/code/.test/.snakemake/conda/1f12dbbb3a74563ca0b2467603494776_\n\
  \nRuleException:\nCalledProcessError in line 13 of /home/azureuser/tmp/m31fucwrbo/code/workflow/rules/trim.smk:\n\
  Command ' singularity  exec --home /home/azureuser/tmp/m31fucwrbo/code/.test  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/m31fucwrbo/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/home/azureuser/tmp/m31fucwrbo/code/.test/.snakemake/conda/1f12dbbb3a74563ca0b2467603494776_'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/m31fucwrbo/code/.test/.snakemake/scripts/tmpvi9m9923.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/m31fucwrbo/code/workflow/rules/trim.smk\"\
  , line 13, in __rule_cutadapt_pe\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T043149.117993.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Ff8togiuu%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Ff8togiuu%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.243GiB</td>
						<td>1478.2 seconds</td>
						<td>1751.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743694336">
								<a href="https://github.com/dbespiatykh/TBvar">dbespiatykh/TBvar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/dbespiatykh/TBvar/tree/v1.1.1">v1.1.1</a>
						</td>
						<td>snakemake</td>
						<td>61 days</td>
						<td>2023-02-19 03:38:34</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2Fn6ti1pmw%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2Fn6ti1pmw%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.004GiB</td>
						<td>901.6 seconds</td>
						<td>1026.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743110224">
								<a href="https://nf-co.re/fetchngs">fetchngs</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/fetchngs/tree/1.9">1.9</a>
						</td>
						<td>nextflow</td>
						<td>60 days</td>
						<td>2023-02-19 06:28:19</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fukf9mcrc%2Flogs.tar.xz">0.033GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/fetchngs%2Fukf9mcrc%2Foutput.tar.xz">0.002GiB</a>
						</td>
						<td>0.785GiB</td>
						<td>59.6 seconds</td>
						<td>63.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743577280">
								<a href="https://github.com/dbespiatykh/TBvar">dbespiatykh/TBvar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/dbespiatykh/TBvar/tree/v1.1.2">v1.1.2</a>
						</td>
						<td>snakemake</td>
						<td>60 days</td>
						<td>2023-02-19 03:31:10</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2Fqxpdmq63%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2Fqxpdmq63%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.939GiB</td>
						<td>1048.7 seconds</td>
						<td>1317.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743150912">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.10">3.10</a>
						</td>
						<td>nextflow</td>
						<td>60 days</td>
						<td>2023-02-19 03:35:56</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fj1903pyz%2Flogs.tar.xz">0.216GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2Fj1903pyz%2Foutput.tar.xz">0.031GiB</a>
						</td>
						<td>2.226GiB</td>
						<td>1596.0 seconds</td>
						<td>1630.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738292368">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.1">v3.19.1</a>
						</td>
						<td>snakemake</td>
						<td>60 days</td>
						<td>2023-02-19 03:59:31</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- cm4zrw3bus
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/cm4zrw3bus/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F7z5lbqev%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F7z5lbqev%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.5 seconds</td>
						<td>1.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738166816">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.3.6">1.3.6</a>
						</td>
						<td>snakemake</td>
						<td>59 days</td>
						<td>2023-02-20 05:17:09</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- rules.smk
kind: CalledProcessError
line_no: 97
rest: "Command ' singularity  exec --home /home/azureuser/tmp/iuy5br1uju/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/iuy5br1uju/code/.snakemake/singularity/01891d672e0245d9a0405b3b87aa0edf.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/5681728a49bd83ceed09ba194330c858'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/iuy5br1uju/code/.snakemake/scripts/tmp9ia4b9hw.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/iuy5br1uju/code/workflow/rules/rules.smk\"\
  , line 97, in __rule_bwa_index\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T051710.117492.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fd7y25j13%2Flogs.tar.xz">1.004GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2Fd7y25j13%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.633GiB</td>
						<td>514.9 seconds</td>
						<td>276.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741945760">
								<a href="https://github.com/zifornd/arrays">zifornd/arrays</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/zifornd/arrays/tree/1.0.0">1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>50 days</td>
						<td>2023-02-19 06:05:46</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/zifornd%252Farrays%2Ftknuphsc%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/zifornd%252Farrays%2Ftknuphsc%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.565GiB</td>
						<td>1056.1 seconds</td>
						<td>805.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010740861632">
								<a href="https://github.com/zifornd/array">zifornd/array</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/zifornd/array/tree/1.0.0">1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>50 days</td>
						<td>2023-02-19 04:05:20</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/zifornd%252Farray%2Fiscr4d5c%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/zifornd%252Farray%2Fiscr4d5c%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.668GiB</td>
						<td>1098.8 seconds</td>
						<td>1186.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743150672">
								<a href="https://nf-co.re/rnaseq">rnaseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/rnaseq/tree/3.10.1">3.10.1</a>
						</td>
						<td>nextflow</td>
						<td>45 days</td>
						<td>2023-02-19 03:56:14</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F1xgp2ods%2Flogs.tar.xz">0.216GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/rnaseq%2F1xgp2ods%2Foutput.tar.xz">0.031GiB</a>
						</td>
						<td>2.445GiB</td>
						<td>1538.9 seconds</td>
						<td>747.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743188784">
								<a href="https://nf-co.re/sarek">sarek</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/sarek/tree/3.1.2">3.1.2</a>
						</td>
						<td>nextflow</td>
						<td>45 days</td>
						<td>2023-02-19 06:34:47</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2F61ctqf9c%2Flogs.tar.xz">0.087GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/sarek%2F61ctqf9c%2Foutput.tar.xz">0.025GiB</a>
						</td>
						<td>2.376GiB</td>
						<td>279.7 seconds</td>
						<td>164.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738217408">
								<a href="https://github.com/epigen/dea_limma">epigen/dea_limma</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/epigen/dea_limma/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>44 days</td>
						<td>2023-02-19 06:00:15</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: FileNotFoundError
line_no: 24
rest: "[Errno 2] No such file or directory: '/research/home/sreichl/projects/macroIC/config/CC001_dea_limma_annotation.csv'\n\
  \  File \"/home/azureuser/tmp/fdmedqqimn/code/workflow/Snakefile\", line 24, in\
  \ <module>\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 211, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 331, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 950, in read_csv\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 605, in _read\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1442, in __init__\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1735, in _make_engine\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/common.py\"\
  , line 856, in get_handle"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Fdea-limma%2Fg1bz5ga6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Fdea-limma%2Fg1bz5ga6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.098GiB</td>
						<td>2.0 seconds</td>
						<td>2.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738217648">
								<a href="https://github.com/epigen/genome_tracks">epigen/genome_tracks</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/epigen/genome_tracks/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>44 days</td>
						<td>2023-02-19 03:30:59</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: FileNotFoundError
line_no: 23
rest: "[Errno 2] No such file or directory: '/path/to/myData_genome_tracks_annotation.csv'\n\
  \  File \"/home/azureuser/tmp/zjyfhaw02z/code/workflow/Snakefile\", line 23, in\
  \ <module>\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 211, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 331, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 950, in read_csv\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 605, in _read\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1442, in __init__\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1735, in _make_engine\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/common.py\"\
  , line 856, in get_handle"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Fgenome-tracks%2F73dtc7lm%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Fgenome-tracks%2F73dtc7lm%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.098GiB</td>
						<td>1.7 seconds</td>
						<td>2.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738207568">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.6.0">v1.6.0</a>
						</td>
						<td>snakemake</td>
						<td>40 days</td>
						<td>2023-02-19 03:53:51</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Froibwbhv%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Froibwbhv%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.896GiB</td>
						<td>1030.5 seconds</td>
						<td>1126.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738212608">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.6.0">v1.6.0</a>
						</td>
						<td>snakemake</td>
						<td>40 days</td>
						<td>2023-02-19 03:49:25</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fhwecbz6z%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fhwecbz6z%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.823GiB</td>
						<td>1216.0 seconds</td>
						<td>1579.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738207328">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.6.1">v1.6.1</a>
						</td>
						<td>snakemake</td>
						<td>39 days</td>
						<td>2023-02-19 04:09:25</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- bxhdgkmyui
- code
- workflow
- rules
- eval.smk
kind: CalledProcessError
line_no: 161
rest: "Command 'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate\
  \ '/home/azureuser/tmp/bxhdgkmyui/code/.test/.snakemake/conda/170f7b276dafd14c1f9cee6ea57cff9c_';\
  \ set -euo pipefail;  python /home/azureuser/tmp/bxhdgkmyui/code/.test/.snakemake/scripts/tmpsikgjmo6.collect-precision-recall.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/bxhdgkmyui/code/workflow/rules/eval.smk\"\
  , line 161, in __rule_collect_precision_recall\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Sun Feb 19 04:21:28 2023]\nError in rule collect_precision_recall:\n\
  \    jobid: 41\n    input: results/precision-recall/callsets/dummy-chm.indels.tsv\n\
  \    output: results/precision-recall/benchmarks/chm-eval.indels.tsv\n    log: logs/collect-precision-recall/chm-eval/indels.log\
  \ (check log file(s) for error message)\n    conda-env: /home/azureuser/tmp/bxhdgkmyui/code/.test/.snakemake/conda/170f7b276dafd14c1f9cee6ea57cff9c_\n\
  \nRuleException:\nCalledProcessError in line 161 of /home/azureuser/tmp/bxhdgkmyui/code/workflow/rules/eval.smk:\n\
  Command 'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate\
  \ '/home/azureuser/tmp/bxhdgkmyui/code/.test/.snakemake/conda/170f7b276dafd14c1f9cee6ea57cff9c_';\
  \ set -euo pipefail;  python /home/azureuser/tmp/bxhdgkmyui/code/.test/.snakemake/scripts/tmpxw1w626o.collect-precision-recall.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/bxhdgkmyui/code/workflow/rules/eval.smk\"\
  , line 161, in __rule_collect_precision_recall\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-19T040926.519236.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Figatmgt1%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Figatmgt1%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.980GiB</td>
						<td>672.6 seconds</td>
						<td>722.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738212368">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.6.1">v1.6.1</a>
						</td>
						<td>snakemake</td>
						<td>39 days</td>
						<td>2023-02-19 03:47:53</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- clc5qtlcfs
- code
- workflow
- rules
- eval.smk
kind: CalledProcessError
line_no: 161
rest: "Command 'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate\
  \ '/home/azureuser/tmp/clc5qtlcfs/code/.test/.snakemake/conda/5987075987fe13661dc2a0ebfbd253f9_';\
  \ set -euo pipefail;  python /home/azureuser/tmp/clc5qtlcfs/code/.test/.snakemake/scripts/tmp98n4pgw1.collect-precision-recall.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/clc5qtlcfs/code/workflow/rules/eval.smk\"\
  , line 161, in __rule_collect_precision_recall\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Sun Feb 19 04:05:42 2023]\nError in rule collect_precision_recall:\n\
  \    jobid: 41\n    input: results/precision-recall/callsets/dummy-chm.indels.tsv\n\
  \    output: results/precision-recall/benchmarks/chm-eval.indels.tsv\n    log: logs/collect-precision-recall/chm-eval/indels.log\
  \ (check log file(s) for error message)\n    conda-env: /home/azureuser/tmp/clc5qtlcfs/code/.test/.snakemake/conda/5987075987fe13661dc2a0ebfbd253f9_\n\
  \nRuleException:\nCalledProcessError in line 161 of /home/azureuser/tmp/clc5qtlcfs/code/workflow/rules/eval.smk:\n\
  Command 'source /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/bin/activate\
  \ '/home/azureuser/tmp/clc5qtlcfs/code/.test/.snakemake/conda/5987075987fe13661dc2a0ebfbd253f9_';\
  \ set -euo pipefail;  python /home/azureuser/tmp/clc5qtlcfs/code/.test/.snakemake/scripts/tmpxtham_sm.collect-precision-recall.py'\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/clc5qtlcfs/code/workflow/rules/eval.smk\"\
  , line 161, in __rule_collect_precision_recall\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/d4uka3meqsxj3wwh2lzivhcbhchs2sqq/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-19T034754.319156.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fjffgpno5%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fjffgpno5%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.824GiB</td>
						<td>846.9 seconds</td>
						<td>1069.1 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743117424">
								<a href="https://nf-co.re/isoseq">isoseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/isoseq/tree/1.1.2">1.1.2</a>
						</td>
						<td>nextflow</td>
						<td>39 days</td>
						<td>2023-02-19 06:41:47</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/isoseq%2Fkh4ji190%2Flogs.tar.xz">0.088GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/isoseq%2Fkh4ji190%2Foutput.tar.xz">0.030GiB</a>
						</td>
						<td>2.630GiB</td>
						<td>873.2 seconds</td>
						<td>667.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743528416">
								<a href="https://github.com/dbespiatykh/TBvar">dbespiatykh/TBvar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/dbespiatykh/TBvar/tree/v1.1.3">v1.1.3</a>
						</td>
						<td>snakemake</td>
						<td>38 days</td>
						<td>2023-02-19 06:12:15</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2Fvi9ysjac%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2Fvi9ysjac%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.833GiB</td>
						<td>980.6 seconds</td>
						<td>1101.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738221008">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.9.0">1.9.0</a>
						</td>
						<td>snakemake</td>
						<td>38 days</td>
						<td>2023-02-20 04:33:54</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- count.smk
kind: CalledProcessError
line_no: 20
rest: "Command ' singularity  exec --home /home/azureuser/tmp/y4tbfwv28t/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/y4tbfwv28t/code/.snakemake/singularity/34b4bea3b670d47819133863ba26c281.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/c80307395eddf442c2fb6870f40d822b'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/y4tbfwv28t/code/.snakemake/scripts/tmplhny_vzr.generate_exclude_file.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/y4tbfwv28t/code/workflow/rules/count.smk\"\
  , line 20, in __rule_generate_exclude_file_for_mosaic_count\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 04:48:48 2023]\nFinished job 8.\n1 of 25 steps (4%)\
  \ done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T043355.664609.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Ftf2y32pr%2Flogs.tar.xz">2.884GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Ftf2y32pr%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.055GiB</td>
						<td>951.0 seconds</td>
						<td>894.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738207088">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.6.2">v1.6.2</a>
						</td>
						<td>snakemake</td>
						<td>37 days</td>
						<td>2023-02-19 03:55:26</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F708zjohy%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2F708zjohy%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.877GiB</td>
						<td>1004.2 seconds</td>
						<td>978.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738212128">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.6.2">v1.6.2</a>
						</td>
						<td>snakemake</td>
						<td>37 days</td>
						<td>2023-02-19 03:52:50</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Ftt7fzz9q%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Ftt7fzz9q%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.660GiB</td>
						<td>1226.2 seconds</td>
						<td>1440.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738292128">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.2">v3.19.2</a>
						</td>
						<td>snakemake</td>
						<td>37 days</td>
						<td>2023-02-19 06:05:26</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- ps40mvffr0
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/ps40mvffr0/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fc950i23x%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fc950i23x%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.1 seconds</td>
						<td>8.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738164896">
								<a href="https://github.com/snakemake-workflows/dna-seq-mtb">snakemake-workflows/dna-seq-mtb</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.3.2">v1.3.2</a>
						</td>
						<td>snakemake</td>
						<td>35 days</td>
						<td>2023-02-19 06:04:00</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- c8otgdauek
- code
- workflow
- rules
- common.smk
kind: WorkflowError
line_no: 20
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/c8otgdauek/code/workflow/Snakefile\", line 1, in <module>\n\
  \  File \"/home/azureuser/tmp/c8otgdauek/code/workflow/rules/common.smk\", line\
  \ 20, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fr6dlcaw3%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fr6dlcaw3%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.5 seconds</td>
						<td>1.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738291888">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.3">v3.19.3</a>
						</td>
						<td>snakemake</td>
						<td>35 days</td>
						<td>2023-02-19 04:17:14</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- qsgnvninem
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/qsgnvninem/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fj88bwlxj%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fj88bwlxj%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.3 seconds</td>
						<td>1.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738219328">
								<a href="https://github.com/epigen/enrichment_analysis">epigen/enrichment_analysis</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/epigen/enrichment_analysis/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>35 days</td>
						<td>2023-02-19 04:30:17</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: FileNotFoundError
line_no: 28
rest: "[Errno 2] No such file or directory: '/path/to/enrichment_analysis_annotation.csv'\n\
  \  File \"/home/azureuser/tmp/vbpdnixptx/code/workflow/Snakefile\", line 28, in\
  \ <module>\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 211, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/util/_decorators.py\"\
  , line 331, in wrapper\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 950, in read_csv\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 605, in _read\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1442, in __init__\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/parsers/readers.py\"\
  , line 1735, in _make_engine\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/io/common.py\"\
  , line 856, in get_handle"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Fenrichment-analysis%2Fv0lq9uas%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/epigen%252Fenrichment-analysis%2Fv0lq9uas%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.098GiB</td>
						<td>2.0 seconds</td>
						<td>4.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738164656">
								<a href="https://github.com/snakemake-workflows/dna-seq-mtb">snakemake-workflows/dna-seq-mtb</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.3.3">v1.3.3</a>
						</td>
						<td>snakemake</td>
						<td>34 days</td>
						<td>2023-02-19 03:32:56</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- r8y5qpq93y
- code
- workflow
- rules
- common.smk
kind: WorkflowError
line_no: 24
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/r8y5qpq93y/code/workflow/Snakefile\", line 1, in <module>\n\
  \  File \"/home/azureuser/tmp/r8y5qpq93y/code/workflow/rules/common.smk\", line\
  \ 24, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fmcthrwfa%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fmcthrwfa%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.4 seconds</td>
						<td>3.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738217888">
								<a href="https://github.com/GooglingTheCancerGenome/sv-gen">GooglingTheCancerGenome/sv-gen</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/GooglingTheCancerGenome/sv-gen/tree/v1.1.0">v1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>31 days</td>
						<td>2023-02-19 04:54:33</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: ModuleNotFoundError
line_no: 1
rest: "No module named 'pyfaidx'\n  File \"/home/azureuser/tmp/8j15m8kvzk/code/workflow/Snakefile\"\
  , line 1, in <module>\n  File \"/home/azureuser/tmp/8j15m8kvzk/code/workflow/helper_functions.py\"\
  , line 5, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-gen%2F7ttnpyar%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/GooglingTheCancerGenome%252Fsv-gen%2F7ttnpyar%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.4 seconds</td>
						<td>2.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738166576">
								<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline">friendsofstrandseq/ashleys-qc-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/ashleys-qc-pipeline/tree/1.4.1">1.4.1</a>
						</td>
						<td>snakemake</td>
						<td>31 days</td>
						<td>2023-02-20 05:01:36</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- rules.smk
kind: CalledProcessError
line_no: 97
rest: "Command ' singularity  exec --home /home/azureuser/tmp/fq268i0s7j/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/fq268i0s7j/code/.snakemake/singularity/8fcc2a64fe20f687830bd2036398cb20.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/5681728a49bd83ceed09ba194330c858'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/fq268i0s7j/code/.snakemake/scripts/tmp6l2pj_l0.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/fq268i0s7j/code/workflow/rules/rules.smk\"\
  , line 97, in __rule_bwa_index\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T050137.497623.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F91eza9cr%2Flogs.tar.xz">0.999GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fashleys-qc-pipeline%2F91eza9cr%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.604GiB</td>
						<td>525.4 seconds</td>
						<td>359.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738220048">
								<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow">SchlossLab/mikropml-snakemake-workflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/SchlossLab/mikropml-snakemake-workflow/tree/v1.2.0">v1.2.0</a>
						</td>
						<td>snakemake</td>
						<td>30 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: '

  Unable to find environment in container image. Maybe a conda environment was modified
  without containerizing again (see snakemake --containerize)?

  Details:

  Command ''SINGULARITYENV_CONDA_PKGS_DIRS=/tmp/conda/cbecbdf8-40ce-43a9-91b8-8379c0b0ac72
  singularity --quiet --silent exec --home /home/azureuser/tmp/qo58fw9v0x/code  /home/azureuser/tmp/qo58fw9v0x/code/.snakemake/singularity/0c5b3ddbad4071ce8cdb3cdee75d09f0.simg
  sh -c ''[ -d ''\''''/conda-envs/289feab62f93e0fbbc4ff01e415879cf''\'''' ]'''' returned
  non-zero exit status 1.


  '

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/SchlossLab%252Fmikropml-snakemake-workflow%2F2h8jo7y7%2Flogs.tar.xz">0.704GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/SchlossLab%252Fmikropml-snakemake-workflow%2F2h8jo7y7%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.479GiB</td>
						<td>413.4 seconds</td>
						<td>319.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738220768">
								<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline">friendsofstrandseq/mosaicatcher-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/friendsofstrandseq/mosaicatcher-pipeline/tree/1.9.1">1.9.1</a>
						</td>
						<td>snakemake</td>
						<td>30 days</td>
						<td>2023-02-20 05:15:50</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- count.smk
kind: CalledProcessError
line_no: 20
rest: "Command ' singularity  exec --home /home/azureuser/tmp/ed2m39q1y1/code  --bind\
  \ /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/ed2m39q1y1/code/.snakemake/singularity/c81abd2343fdbdc73088d420b3c2c87c.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/conda-envs/c80307395eddf442c2fb6870f40d822b'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/ed2m39q1y1/code/.snakemake/scripts/tmpkhlwphba.generate_exclude_file.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/ed2m39q1y1/code/workflow/rules/count.smk\"\
  , line 20, in __rule_generate_exclude_file_for_mosaic_count\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\n[Mon Feb 20 05:30:36 2023]\nFinished job 13.\n2 of 25 steps (8%)\
  \ done\nShutting down, this might take some time.\nExiting because a job execution\
  \ failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T051551.688302.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fj3vxyemx%2Flogs.tar.xz">2.885GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/friendsofstrandseq%252Fmosaicatcher-pipeline%2Fj3vxyemx%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.067GiB</td>
						<td>950.8 seconds</td>
						<td>886.5 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743113104">
								<a href="https://nf-co.re/hic">hic</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/hic/tree/2.0.0">2.0.0</a>
						</td>
						<td>nextflow</td>
						<td>29 days</td>
						<td>2023-02-19 04:39:20</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2Fen22clmp%2Flogs.tar.xz">0.216GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/hic%2Fen22clmp%2Foutput.tar.xz">0.038GiB</a>
						</td>
						<td>0.825GiB</td>
						<td>455.9 seconds</td>
						<td>241.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738216928">
								<a href="https://github.com/dbespiatykh/RDscan">dbespiatykh/RDscan</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/dbespiatykh/RDscan/tree/v1.0.1">v1.0.1</a>
						</td>
						<td>snakemake</td>
						<td>29 days</td>
						<td>2023-02-19 06:22:12</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakeWorkflowError
kind: SnakemakeError
rest: "\nInputFunctionException (rule bwa_mem, line 2, /home/azureuser/tmp/i7h1flflmy/code/workflow/rules/mapping.smk):\n\
  \    Error:\n      KeyError: 'SRR2024925.proportion'\n    Wildcards:\n      sample=SRR2024925.proportion\n\
  \    Traceback:\n      File \"/home/azureuser/tmp/i7h1flflmy/code/workflow/rules/common.smk\"\
  , line 19, in get_fastq\n      File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/core/indexing.py\"\
  , line 1073, in __getitem__\n      File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/core/indexing.py\"\
  , line 1312, in _getitem_axis\n      File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/core/indexing.py\"\
  , line 1260, in _get_label\n      File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/core/generic.py\"\
  , line 4056, in xs\n      File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages/pandas/core/indexes/base.py\"\
  , line 3804, in get_loc\nMissingInputException: Missing input files for rule bwa_mem:\n\
  \    output: mapped/SRR2024925.bam, mapped/SRR2024925.bam.bai\n    wildcards: sample=SRR2024925\n\
  \    affected files:\n        /path/to/SRR2024925_1.fastq.gz\n        /path/to/SRR2024925_2.fastq.gz\n"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FRDscan%2F1121yczo%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FRDscan%2F1121yczo%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.102GiB</td>
						<td>1.9 seconds</td>
						<td>4.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738291648">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.4">v3.19.4</a>
						</td>
						<td>snakemake</td>
						<td>27 days</td>
						<td>2023-02-19 06:16:50</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 4fib8820xb
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/4fib8820xb/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F0bsu9d99%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F0bsu9d99%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.6 seconds</td>
						<td>7.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738216688">
								<a href="https://github.com/dbespiatykh/RDscan">dbespiatykh/RDscan</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/dbespiatykh/RDscan/tree/v1.0.3r">v1.0.3r</a>
						</td>
						<td>snakemake</td>
						<td>27 days</td>
						<td>2023-02-19 06:58:18</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FRDscan%2F53co9mbc%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FRDscan%2F53co9mbc%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.962GiB</td>
						<td>567.0 seconds</td>
						<td>492.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743051168">
								<a href="https://nf-co.re/demultiplex">demultiplex</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/demultiplex/tree/1.1.0">1.1.0</a>
						</td>
						<td>nextflow</td>
						<td>26 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/demultiplex%2Fbbgdb7zh%2Flogs.tar.xz">0.182GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/demultiplex%2Fbbgdb7zh%2Foutput.tar.xz">0.035GiB</a>
						</td>
						<td>2.445GiB</td>
						<td>133.7 seconds</td>
						<td>219.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743053568">
								<a href="https://nf-co.re/differentialabundance">differentialabundance</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/differentialabundance/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>26 days</td>
						<td>2023-02-19 03:30:59</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/differentialabundance%2F7lfq1utr%2Flogs.tar.xz">0.051GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/differentialabundance%2F7lfq1utr%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>1.006GiB</td>
						<td>246.5 seconds</td>
						<td>440.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743053328">
								<a href="https://nf-co.re/differentialabundance">differentialabundance</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/differentialabundance/tree/1.0.1">1.0.1</a>
						</td>
						<td>nextflow</td>
						<td>25 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/differentialabundance%2Fsppcjtvy%2Flogs.tar.xz">0.051GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/differentialabundance%2Fsppcjtvy%2Foutput.tar.xz">0.003GiB</a>
						</td>
						<td>1.008GiB</td>
						<td>272.2 seconds</td>
						<td>408.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738218848">
								<a href="https://github.com/akcorut/kGWASflow">akcorut/kGWASflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/akcorut/kGWASflow/tree/v.1.1.0">v.1.1.0</a>
						</td>
						<td>snakemake</td>
						<td>23 days</td>
						<td>2023-02-19 06:16:14</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 6jilrcjny2
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 13
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/6jilrcjny2/code/workflow/Snakefile\", line 13, in\
  \ <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/akcorut%252FkGWASflow%2F619z4g50%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/akcorut%252FkGWASflow%2F619z4g50%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.3 seconds</td>
						<td>1.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738321776">
								<a href="https://github.com/franciscozorrilla/metaGEM">franciscozorrilla/metaGEM</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/franciscozorrilla/metaGEM/tree/v1.0.3">v1.0.3</a>
						</td>
						<td>snakemake</td>
						<td>22 days</td>
						<td>2023-02-19 03:45:29</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- Snakefile
kind: WorkflowError
line_no: 1
rest: "Workflow defines configfile ../config/config.yaml but it is not present or\
  \ accessible.\n  File \"/home/azureuser/tmp/40bpkj89d0/code/workflow/Snakefile\"\
  , line 1, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/franciscozorrilla%252FmetaGEM%2Fxha2ai7b%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/franciscozorrilla%252FmetaGEM%2Fxha2ai7b%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.5 seconds</td>
						<td>4.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738216448">
								<a href="https://github.com/tahiri-lab/aPhyloGeo-pipeline">tahiri-lab/aPhyloGeo-pipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/tahiri-lab/aPhyloGeo-pipeline/tree/v1.0">v1.0</a>
						</td>
						<td>snakemake</td>
						<td>22 days</td>
						<td>2023-02-19 06:45:34</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 2n14y2a01c
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 9
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/2n14y2a01c/code/workflow/Snakefile\", line 9, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/tahiri-lab%252FaPhyloGeo-pipeline%2Fykxg0pj8%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/tahiri-lab%252FaPhyloGeo-pipeline%2Fykxg0pj8%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.4 seconds</td>
						<td>1.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738291408">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.5">v3.19.5</a>
						</td>
						<td>snakemake</td>
						<td>20 days</td>
						<td>2023-02-19 06:00:18</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- rd28dxrr4c
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/rd28dxrr4c/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fb7d9k7do%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fb7d9k7do%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.6 seconds</td>
						<td>2.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741895216">
								<a href="https://github.com/simonmoulds/uk-decadal-flood-prediction">simonmoulds/uk-decadal-flood-prediction</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/simonmoulds/uk-decadal-flood-prediction/tree/v1.0.0">v1.0.0</a>
						</td>
						<td>snakemake</td>
						<td>19 days</td>
						<td>2023-02-19 04:36:59</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- xnxu34p6af
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 7
rest: "Workflow defines configfile config/config.yml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/xnxu34p6af/code/workflow/Snakefile\", line 7, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/simonmoulds%252Fuk-decadal-flood-prediction%2Ft9dwqxyh%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/simonmoulds%252Fuk-decadal-flood-prediction%2Ft9dwqxyh%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.097GiB</td>
						<td>1.8 seconds</td>
						<td>4.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738291168">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.19.6">v3.19.6</a>
						</td>
						<td>snakemake</td>
						<td>18 days</td>
						<td>2023-02-19 06:00:17</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- mg4g76xcyc
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/mg4g76xcyc/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fmcqvcnlt%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fmcqvcnlt%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>1.2 seconds</td>
						<td>1.3 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741585792">
								<a href="https://github.com/aryarm/admixtools">aryarm/admixtools</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/aryarm/admixtools/tree/v0.1.0">v0.1.0</a>
						</td>
						<td>snakemake</td>
						<td>17 days</td>
						<td>2023-02-19 04:54:36</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2Ffw25wvgb%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2Ffw25wvgb%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741534096">
								<a href="https://github.com/aryarm/admixtools">aryarm/admixtools</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/aryarm/admixtools/tree/v0.1.1">v0.1.1</a>
						</td>
						<td>snakemake</td>
						<td>17 days</td>
						<td>2023-02-19 06:20:44</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2F3xxfingg%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2F3xxfingg%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743002112">
								<a href="https://nf-co.re/crisprseq">crisprseq</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/crisprseq/tree/1.0">1.0</a>
						</td>
						<td>nextflow</td>
						<td>17 days</td>
						<td>2023-02-19 06:42:57</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/crisprseq%2Fms7kwhg0%2Flogs.tar.xz">0.037GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/crisprseq%2Fms7kwhg0%2Foutput.tar.xz">0.017GiB</a>
						</td>
						<td>1.079GiB</td>
						<td>593.9 seconds</td>
						<td>454.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741465776">
								<a href="https://github.com/aryarm/admixtools">aryarm/admixtools</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/aryarm/admixtools/tree/v0.1.2">v0.1.2</a>
						</td>
						<td>snakemake</td>
						<td>16 days</td>
						<td>2023-02-19 03:56:03</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2Fvpgc7tlv%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2Fvpgc7tlv%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743477968">
								<a href="https://github.com/dbespiatykh/TBvar">dbespiatykh/TBvar</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/dbespiatykh/TBvar/tree/v1.1.4">v1.1.4</a>
						</td>
						<td>snakemake</td>
						<td>12 days</td>
						<td>2023-02-19 03:31:29</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2Fnj0qasv0%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/dbespiatykh%252FTBvar%2Fnj0qasv0%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.067GiB</td>
						<td>1027.7 seconds</td>
						<td>1654.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743201264">
								<a href="https://github.com/DSiB-G2/G2-CovPipeline">DSiB-G2/G2-CovPipeline</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/DSiB-G2/G2-CovPipeline/tree/v1.0">v1.0</a>
						</td>
						<td>snakemake</td>
						<td>10 days</td>
						<td>2023-02-19 03:33:29</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/DSiB-G2%252FG2-CovPipeline%2Fb455yagu%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/DSiB-G2%252FG2-CovPipeline%2Fb455yagu%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>4.708GiB</td>
						<td>1889.5 seconds</td>
						<td>2671.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738269920">
								<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth">snakemake-workflows/rna-seq-kallisto-sleuth</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/v2.4.3">v2.4.3</a>
						</td>
						<td>snakemake</td>
						<td>10 days</td>
						<td>2023-02-20 04:31:48</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- workflow
- rules
- trim.smk
kind: CalledProcessError
line_no: 15
rest: "Command ' singularity  exec --home /home/azureuser/tmp/vxhhhdfufd/code/.test\
  \  --bind /home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/view/lib/python3.10/site-packages:/mnt/snakemake\
  \ /home/azureuser/tmp/vxhhhdfufd/code/.test/.snakemake/singularity/89e397d32d0575ad5d1e8e61ae29b7da.simg\
  \ bash -c 'source /opt/conda/bin/activate '\\''/home/azureuser/tmp/vxhhhdfufd/code/.test/.snakemake/conda/dde4e9ec93176343728a5de1966d09ad_'\\\
  ''; set -euo pipefail;  python /home/azureuser/tmp/vxhhhdfufd/code/.test/.snakemake/scripts/tmptm69acad.wrapper.py''\
  \ returned non-zero exit status 1.\n  File \"/home/azureuser/tmp/vxhhhdfufd/code/workflow/rules/trim.smk\"\
  , line 15, in __rule_cutadapt_pe\n  File \"/home/azureuser/spack/var/spack/environments/wf-reg-test/.spack-env/._view/tykdg6bptdy64dwkwitmue6qomekey7e/lib/python3.10/concurrent/futures/thread.py\"\
  , line 58, in run\nShutting down, this might take some time.\nExiting because a\
  \ job execution failed. Look above for error message\nComplete log: .snakemake/log/2023-02-20T043154.221653.snakemake.log"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fhkg93ava%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Frna-seq-kallisto-sleuth%2Fhkg93ava%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>3.132GiB</td>
						<td>1187.5 seconds</td>
						<td>1340.4 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743148272">
								<a href="https://nf-co.re/proteinfold">proteinfold</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/proteinfold/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>9 days</td>
						<td>2023-02-19 03:31:01</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/proteinfold%2Fun6jyg1r%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/proteinfold%2Fun6jyg1r%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.836GiB</td>
						<td>50.8 seconds</td>
						<td>63.6 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738290928">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.20.0">v3.20.0</a>
						</td>
						<td>snakemake</td>
						<td>9 days</td>
						<td>2023-02-19 04:35:54</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- 4mymsq2l99
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/4mymsq2l99/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F0sw7w20v%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2F0sw7w20v%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.060GiB</td>
						<td>1.3 seconds</td>
						<td>1.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010741332144">
								<a href="https://github.com/aryarm/admixtools">aryarm/admixtools</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/aryarm/admixtools/tree/v0.1.3">v0.1.3</a>
						</td>
						<td>snakemake</td>
						<td>8 days</td>
						<td>2023-02-19 03:31:00</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NoSnakefileError
kind: NoSnakefileError
</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2Fwlaz0hc4%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/aryarm%252Fadmixtools%2Fwlaz0hc4%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.002GiB</td>
						<td>0.0 seconds</td>
						<td>0.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738211888">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.6.3">v1.6.3</a>
						</td>
						<td>snakemake</td>
						<td>7 days</td>
						<td>2023-02-19 11:35:18</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fj75nj7jy%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2Fj75nj7jy%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.593GiB</td>
						<td>886.2 seconds</td>
						<td>784.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738206848">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.6.3">v1.6.3</a>
						</td>
						<td>snakemake</td>
						<td>6 days</td>
						<td>2023-02-19 06:15:21</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fm5e3e4qi%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fm5e3e4qi%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.971GiB</td>
						<td>1228.0 seconds</td>
						<td>1306.0 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738206608">
								<a href="https://github.com/snakemake-workflows/benchmark-giab">snakemake-workflows/benchmark-giab</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/benchmark-giab/tree/v1.6.4">v1.6.4</a>
						</td>
						<td>snakemake</td>
						<td>6 days</td>
						<td>2023-02-19 06:00:16</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fqkvoa46z%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fbenchmark-giab%2Fqkvoa46z%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>2.829GiB</td>
						<td>1050.1 seconds</td>
						<td>1208.2 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738211648">
								<a href="https://github.com/snakemake-workflows/dna-seq-benchmark">snakemake-workflows/dna-seq-benchmark</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-benchmark/tree/v1.6.4">v1.6.4</a>
						</td>
						<td>snakemake</td>
						<td>6 days</td>
						<td>2023-02-19 04:15:34</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2F3wjb5sej%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-benchmark%2F3wjb5sej%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>1.980GiB</td>
						<td>901.2 seconds</td>
						<td>809.7 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743252592">
								<a href="https://github.com/seankmartin/atn-sub-lfp-workflow">seankmartin/atn-sub-lfp-workflow</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/seankmartin/atn-sub-lfp-workflow/tree/23.02.0">23.02.0</a>
						</td>
						<td>snakemake</td>
						<td>6 days</td>
						<td>2023-02-19 03:35:16</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- o7mew9by3r
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 2
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/o7mew9by3r/code/workflow/Snakefile\", line 2, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/seankmartin%252Fatn-sub-lfp-workflow%2Fks4xxjnu%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/seankmartin%252Fatn-sub-lfp-workflow%2Fks4xxjnu%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>0.9 seconds</td>
						<td>0.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738290688">
								<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor">snakemake-workflows/dna-seq-varlociraptor</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-varlociraptor/tree/v3.20.1">v3.20.1</a>
						</td>
						<td>snakemake</td>
						<td>5 days</td>
						<td>2023-02-19 06:21:05</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- zeangs3vje
- code
- workflow
- Snakefile
kind: WorkflowError
line_no: 6
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/zeangs3vje/code/workflow/Snakefile\", line 6, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fwksrr6q6%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-varlociraptor%2Fwksrr6q6%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.059GiB</td>
						<td>0.8 seconds</td>
						<td>0.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010738164416">
								<a href="https://github.com/snakemake-workflows/dna-seq-mtb">snakemake-workflows/dna-seq-mtb</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/snakemake-workflows/dna-seq-mtb/tree/v1.3.4">v1.3.4</a>
						</td>
						<td>snakemake</td>
						<td>4 days</td>
						<td>2023-02-19 03:35:11</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.SnakemakePythonError
file: !!python/object/apply:pathlib.PosixPath
- /
- home
- azureuser
- tmp
- m5r580r0yd
- code
- workflow
- rules
- common.smk
kind: WorkflowError
line_no: 24
rest: "Workflow defines configfile config/config.yaml but it is not present or accessible.\n\
  \  File \"/home/azureuser/tmp/m5r580r0yd/code/workflow/Snakefile\", line 1, in <module>\n\
  \  File \"/home/azureuser/tmp/m5r580r0yd/code/workflow/rules/common.smk\", line\
  \ 24, in <module>"

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fmv66m1an%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/snakemake-workflows%252Fdna-seq-mtb%2Fmv66m1an%2Foutput.tar.xz">0.000GiB</a>
						</td>
						<td>0.062GiB</td>
						<td>1.6 seconds</td>
						<td>3.9 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743112624">
								<a href="https://nf-co.re/funcscan">funcscan</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/funcscan/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>4 days</td>
						<td>2023-02-19 04:06:56</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/funcscan%2Fi92w9tde%2Flogs.tar.xz">0.129GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/funcscan%2Fi92w9tde%2Foutput.tar.xz">0.009GiB</a>
						</td>
						<td>1.295GiB</td>
						<td>3951.8 seconds</td>
						<td>1105.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743146832">
								<a href="https://nf-co.re/nascent">nascent</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/nascent/tree/2.1.0">2.1.0</a>
						</td>
						<td>nextflow</td>
						<td>4 days</td>
						<td>2023-02-19 06:44:42</td>
						<td>
							<div>
								<p>
									<span></span>
								</p>
								<p>
									<details>
										<summary>show error</summary>
										<code>
											<pre>!!python/object:wf_reg_test.engines.NextflowCommandError
command_executed: "samtools \\\n    faidx \\\n     \\\n    GRCh38_chr21.fa\n\ncat\
  \ <<-END_VERSIONS > versions.yml\n\"NFCORE_NASCENT:NASCENT:PREPARE_GENOME:SAMTOOLS_FAIDX\"\
  :\n    samtools: $(echo $(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*$//')\n\
  END_VERSIONS"
error: '[E::fai_build3_core] Failed to open the file GRCh38_chr21.fa

  [faidx] Could not build fai index GRCh38_chr21.fa.fai'
exit_status: 1
kind: CalledProcessError
output: (empty)
process: NFCORE_NASCENT:NASCENT:PREPARE_GENOME:SAMTOOLS_FAIDX (GRCh38_chr21.fa)
work_dir: !!python/object/apply:pathlib.PosixPath
- work
- fc
- e23fbdd2bc09a131b8e37f86403dd6

											</pre>
										</code>
									</details>
								</p>
							</div>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nascent%2Fzoambtxd%2Flogs.tar.xz">0.017GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/nascent%2Fzoambtxd%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.873GiB</td>
						<td>51.0 seconds</td>
						<td>35.8 seconds</td>
					</tr>
					<tr>
						<td>
							<span id="140010743148032">
								<a href="https://nf-co.re/phyloplace">phyloplace</a>
							</span>
						</td>
						<td>
							<a href="https://github.com/nf-core/phyloplace/tree/1.0.0">1.0.0</a>
						</td>
						<td>nextflow</td>
						<td>4 days</td>
						<td>2023-02-19 06:34:00</td>
						<td>
							<span></span>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/phyloplace%2Fve407xme%2Flogs.tar.xz">0.000GiB</a>
						</td>
						<td>
							<a href="https://wfregtest.blob.core.windows.net/data3/phyloplace%2Fve407xme%2Foutput.tar.xz">0.001GiB</a>
						</td>
						<td>0.662GiB</td>
						<td>42.6 seconds</td>
						<td>23.6 seconds</td>
					</tr>
				</tbody>
			</table>
		</div>
	</body>
</html>